{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Get the reactions in carveme model into a format that can be uploaded into kbase"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "check out Escher visualization https://escher.readthedocs.io/en/latest/getting_started.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2020-12-15T09:30:48.353507Z",
     "start_time": "2020-12-15T09:30:43.130260Z"
    }
   },
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import cobra\n",
    "import numpy as np\n",
    "import seaborn as sns\n",
    "import itertools\n",
    "from matplotlib.colors import LogNorm, Normalize\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['README.txt', 'HOT1A3_ncbi_nogapfill.xml', 'HOT1A3_ncbi.xml']"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "dpath = os.path.join('..','Models')\n",
    "os.listdir(dpath)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from cobra.flux_analysis import flux_variability_analysis\n",
    "from cobra.flux_analysis import production_envelope\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_elements(rid):\n",
    "    r = model.reactions.get_by_id(rid)\n",
    "    m = r.reactants[0]    \n",
    "    elements = m.elements.copy()\n",
    "    \n",
    "    res = {\n",
    "        'reaction' : rid,\n",
    "        'metabolite' : m.name,\n",
    "        'mid' : m.id,\n",
    "        'N' : 'N' in elements,\n",
    "        'C' : 'C' in elements,\n",
    "        \n",
    "    }\n",
    "    return pd.Series(res)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "model_dpath = os.path.join('..', 'Models')\n",
    "model_fname = 'HOT1A3_ncbi.xml'\n",
    "model_nogap_fname = 'HOT1A3_ncbi_nogapfill.xml'\n",
    "\n",
    "model_fpath = os.path.join(model_dpath, model_fname)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Set parameter Username\n",
      "Academic license - for non-commercial use only - expires 2023-07-13\n"
     ]
    }
   ],
   "source": [
    "model = cobra.io.read_sbml_model(model_fpath)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_nogap = cobra.io.read_sbml_model(os.path.join(model_dpath, model_nogap_fname))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>HOT1A3_ncbi</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x07f5944018490</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>1546</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>2341</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>1.0*Growth - 1.0*Growth_reverse_699ae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>cytosol, periplasm, extracellular space</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model HOT1A3_ncbi at 0x7f5944018490>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>HOT1A3_ncbi_nogapfill</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x07f5944018e80</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>1546</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>2340</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>1.0*Growth - 1.0*Growth_reverse_699ae</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>cytosol, periplasm, extracellular space</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model HOT1A3_ncbi_nogapfill at 0x7f5944018e80>"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model_nogap"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "r = model.reactions[25]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'sbo': 'SBO:0000185',\n",
       " 'metanetx.reaction': 'MNXR94753',\n",
       " 'seed.reaction': 'rxn07942'}"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r.annotation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "frozenset()"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r.genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Reaction identifier</strong></td><td>2AGPE160tipp</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>2-Acyl-sn-glycero-3-phosphoethanolamine (n-C16:0) transporter via facilitated diffusion (periplasm)</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x7f594398dcd0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Stoichiometry</strong></td>\n",
       "                <td>\n",
       "                    <p style='text-align:right'>2agpe160_p --> 2agpe160_c</p>\n",
       "                    <p style='text-align:right'>2-Acyl-sn-glycero-3-phosphoethanolamine (n-C16:0) --> 2-Acyl-sn-glycero-3-phosphoethanolamine (n-C16:0)</p>\n",
       "                </td>\n",
       "            </tr><tr>\n",
       "                <td><strong>GPR</strong></td><td></td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Lower bound</strong></td><td>0.0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Upper bound</strong></td><td>1000.0</td>\n",
       "            </tr>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<Reaction 2AGPE160tipp at 0x7f594398dcd0>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "def _r_to_dict(r):\n",
    "    res = {\n",
    "        'reaction': r.name,\n",
    "        'rid': r.id, \n",
    "        'genes' : [g.name for g in r.genes],\n",
    "        'genes_number' : len(r.genes),\n",
    "        \n",
    "    }\n",
    "    res.update(r.annotation)\n",
    "    return res\n",
    "    \n",
    "\n",
    "rlist = [_r_to_dict(r) for r in model_nogap.reactions]\n",
    " "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "def _r_to_dict(r):\n",
    "    res = {\n",
    "        'reaction': r.name,\n",
    "        'rid': r.id, \n",
    "        'genes' : [g.name for g in r.genes],\n",
    "        'genes_number' : len(r.genes),\n",
    "        \n",
    "    }\n",
    "    res.update(r.annotation)\n",
    "    return res\n",
    "    \n",
    "\n",
    "rgaplist = [_r_to_dict(r) for r in model.reactions]\n",
    " "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.DataFrame(rlist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfgap = pd.DataFrame(rgaplist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>reaction</th>\n",
       "      <th>rid</th>\n",
       "      <th>genes</th>\n",
       "      <th>genes_number</th>\n",
       "      <th>sbo</th>\n",
       "      <th>metanetx.reaction</th>\n",
       "      <th>biocyc</th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>ec-code</th>\n",
       "      <th>rhea</th>\n",
       "      <th>kegg.reaction</th>\n",
       "      <th>reactome</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR120tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94675</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR140tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94676</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR141tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94677</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR161tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94679</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR180tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94680</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2335</th>\n",
       "      <td>R_EX_xtsn_e</td>\n",
       "      <td>EX_xtsn_e</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000627</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2336</th>\n",
       "      <td>R_EX_xyl__D_e</td>\n",
       "      <td>EX_xyl__D_e</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000627</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2337</th>\n",
       "      <td>R_EX_zn2_e</td>\n",
       "      <td>EX_zn2_e</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000627</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2338</th>\n",
       "      <td>Biomass reaction</td>\n",
       "      <td>Growth</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000629</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2339</th>\n",
       "      <td>R_ATPM</td>\n",
       "      <td>ATPM</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000630</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2340 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               reaction           rid genes  \\\n",
       "0     1,2 diacylglycerol transport via flipping (per...  12DGR120tipp    []   \n",
       "1     1,2 diacylglycerol transport via flipping (per...  12DGR140tipp    []   \n",
       "2     1,2 diacylglycerol transport via flipping (per...  12DGR141tipp    []   \n",
       "3     1,2 diacylglycerol transport via flipping (per...  12DGR161tipp    []   \n",
       "4     1,2 diacylglycerol transport via flipping (per...  12DGR180tipp    []   \n",
       "...                                                 ...           ...   ...   \n",
       "2335                                        R_EX_xtsn_e     EX_xtsn_e    []   \n",
       "2336                                      R_EX_xyl__D_e   EX_xyl__D_e    []   \n",
       "2337                                         R_EX_zn2_e      EX_zn2_e    []   \n",
       "2338                                   Biomass reaction        Growth    []   \n",
       "2339                                             R_ATPM          ATPM    []   \n",
       "\n",
       "      genes_number          sbo metanetx.reaction biocyc seed.reaction  \\\n",
       "0                0  SBO:0000185         MNXR94675    NaN           NaN   \n",
       "1                0  SBO:0000185         MNXR94676    NaN           NaN   \n",
       "2                0  SBO:0000185         MNXR94677    NaN           NaN   \n",
       "3                0  SBO:0000185         MNXR94679    NaN           NaN   \n",
       "4                0  SBO:0000185         MNXR94680    NaN           NaN   \n",
       "...            ...          ...               ...    ...           ...   \n",
       "2335             0  SBO:0000627               NaN    NaN           NaN   \n",
       "2336             0  SBO:0000627               NaN    NaN           NaN   \n",
       "2337             0  SBO:0000627               NaN    NaN           NaN   \n",
       "2338             0  SBO:0000629               NaN    NaN           NaN   \n",
       "2339             0  SBO:0000630               NaN    NaN           NaN   \n",
       "\n",
       "     ec-code rhea kegg.reaction reactome  \n",
       "0        NaN  NaN           NaN      NaN  \n",
       "1        NaN  NaN           NaN      NaN  \n",
       "2        NaN  NaN           NaN      NaN  \n",
       "3        NaN  NaN           NaN      NaN  \n",
       "4        NaN  NaN           NaN      NaN  \n",
       "...      ...  ...           ...      ...  \n",
       "2335     NaN  NaN           NaN      NaN  \n",
       "2336     NaN  NaN           NaN      NaN  \n",
       "2337     NaN  NaN           NaN      NaN  \n",
       "2338     NaN  NaN           NaN      NaN  \n",
       "2339     NaN  NaN           NaN      NaN  \n",
       "\n",
       "[2340 rows x 12 columns]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1     1139\n",
       "0      544\n",
       "2      390\n",
       "3      170\n",
       "4       42\n",
       "5       35\n",
       "7        8\n",
       "8        5\n",
       "6        5\n",
       "12       1\n",
       "13       1\n",
       "Name: genes_number, dtype: int64"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.genes_number.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "reaction             2340\n",
       "rid                  2340\n",
       "genes                2340\n",
       "genes_number         2340\n",
       "sbo                  2340\n",
       "metanetx.reaction    1588\n",
       "biocyc                805\n",
       "seed.reaction        1169\n",
       "ec-code               882\n",
       "rhea                  864\n",
       "kegg.reaction         649\n",
       "reactome              245\n",
       "dtype: int64"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "(~df.isna()).sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['reaction', 'rid', 'genes', 'genes_number', 'sbo', 'metanetx.reaction',\n",
       "       'biocyc', 'seed.reaction', 'ec-code', 'rhea', 'kegg.reaction',\n",
       "       'reactome'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>ec-code</th>\n",
       "      <th>kegg.reaction</th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>1040</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>87</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>43</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>415</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>149</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>603</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   seed.reaction  ec-code  kegg.reaction     0\n",
       "0          False    False          False  1040\n",
       "1          False    False           True     1\n",
       "2          False     True          False    87\n",
       "3          False     True           True    43\n",
       "4           True    False          False   415\n",
       "5           True    False           True     2\n",
       "6           True     True          False   149\n",
       "7           True     True           True   603"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "(~df.isna()).groupby([ 'seed.reaction', 'ec-code', 'kegg.reaction',]).size().reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>metanetx.reaction</th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>752</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>419</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>1169</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   metanetx.reaction  seed.reaction     0\n",
       "0              False          False   752\n",
       "1               True          False   419\n",
       "2               True           True  1169"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "(~df.isna()).groupby(['metanetx.reaction', 'seed.reaction', ]).size().reset_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>reaction</th>\n",
       "      <th>rid</th>\n",
       "      <th>genes</th>\n",
       "      <th>genes_number</th>\n",
       "      <th>sbo</th>\n",
       "      <th>metanetx.reaction</th>\n",
       "      <th>biocyc</th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>ec-code</th>\n",
       "      <th>rhea</th>\n",
       "      <th>kegg.reaction</th>\n",
       "      <th>reactome</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR120tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94675</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR140tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94676</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR141tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94677</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR161tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94679</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR180tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94680</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            reaction           rid genes  \\\n",
       "0  1,2 diacylglycerol transport via flipping (per...  12DGR120tipp    []   \n",
       "1  1,2 diacylglycerol transport via flipping (per...  12DGR140tipp    []   \n",
       "2  1,2 diacylglycerol transport via flipping (per...  12DGR141tipp    []   \n",
       "3  1,2 diacylglycerol transport via flipping (per...  12DGR161tipp    []   \n",
       "4  1,2 diacylglycerol transport via flipping (per...  12DGR180tipp    []   \n",
       "\n",
       "   genes_number          sbo metanetx.reaction biocyc seed.reaction ec-code  \\\n",
       "0             0  SBO:0000185         MNXR94675    NaN           NaN     NaN   \n",
       "1             0  SBO:0000185         MNXR94676    NaN           NaN     NaN   \n",
       "2             0  SBO:0000185         MNXR94677    NaN           NaN     NaN   \n",
       "3             0  SBO:0000185         MNXR94679    NaN           NaN     NaN   \n",
       "4             0  SBO:0000185         MNXR94680    NaN           NaN     NaN   \n",
       "\n",
       "  rhea kegg.reaction reactome  \n",
       "0  NaN           NaN      NaN  \n",
       "1  NaN           NaN      NaN  \n",
       "2  NaN           NaN      NaN  \n",
       "3  NaN           NaN      NaN  \n",
       "4  NaN           NaN      NaN  "
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['1,2 diacylglycerol transport via flipping (periplasm to cytoplasm, n-C12:0)',\n",
       "       '1,2 diacylglycerol transport via flipping (periplasm to cytoplasm, n-C14:0)',\n",
       "       '1,2 diacylglycerol transport via flipping (periplasm to cytoplasm, n-C14:1)',\n",
       "       '1,2 diacylglycerol transport via flipping (periplasm to cytoplasm, n-C16:1)',\n",
       "       '1,2 diacylglycerol transport via flipping (periplasm to cytoplasm, n-C18:0)',\n",
       "       '(R)-Propane-1,2-diol transport via diffusion (extracellular to periplasm)',\n",
       "       '(S)-Propane-1,2-diol transport via diffusion (extracellular to periplasm)',\n",
       "       '1,4-alpha-D-glucan transport via diffusion (extracellular to periplasm) irreversible',\n",
       "       '2 5 diketo D gluconate transport via proton symport  reversible  periplasm ',\n",
       "       '2 5 diketo D gluconate transport via diffusion  extracellular to periplasm ',\n",
       "       '2-Acyl-sn-glycero-3-phosphatidate (n-C16:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphatidate (n-C16:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphatidate (n-C18:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C12:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C14:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C14:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C16:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C16:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C18:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoethanolamine (n-C18:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C12:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C14:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C14:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C16:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C16:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C18:0) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-Acyl-sn-glycero-3-phosphoglycerol (n-C18:1) transporter via facilitated diffusion (periplasm)',\n",
       "       '2-dehydro-D-gluconate reductase (NADH)',\n",
       "       '2-Dehydro-L-gulonate reductase to gluconate (NADH)',\n",
       "       '3 Hydroxyacyl ACPCoA Transacylase',\n",
       "       '3R-3-Hydroxybutanoyl-acyl-carrier-protein hydro-lyase ',\n",
       "       '4 hydroxy benzyl alcohol dehydrogenase', 'MNXR68734',\n",
       "       '5-Dehydro-D-gluconate transport via diffusion (extracellular to periplasm)',\n",
       "       '6 acetylthiohexanoic acid transport via diffusion  extracellular to periplasm ',\n",
       "       '', '4 acetamidobutyrate deacetylase',\n",
       "       'Aryl alcohol dehydrogenase',\n",
       "       'L arabinitol transport via passive diffusion',\n",
       "       'Acetoacetate transport via diffusion (extracellular to periplasm)',\n",
       "       'N-Acetyl-D-glucosamine transport via diffusion (extracellular to periplasm)',\n",
       "       'Acyl CoA dehydrogenase  isobutyryl CoA ',\n",
       "       'O3-Acetyl-L-serine acetate-lyase (adding hydrogen sulfide), cytosol',\n",
       "       'Adenine transport via diffusion (extracellular to periplasm)',\n",
       "       'Adenosine transport via diffusion (extracellular to periplasm)',\n",
       "       '2 aminoethylphosphonate pyruvate transaminase',\n",
       "       'AEP transport via diffusion extracellular to periplasm ',\n",
       "       'Agmatine/putrescine antiporter (periplasm)',\n",
       "       'Ai2 transport, outer membrane',\n",
       "       'L Alaninylhistidine transport via diffusion  extracellular to periplasm ',\n",
       "       'L Alaninylleucine transport via diffusion  extracellular to periplasm ',\n",
       "       'L Alaninylthreonine transport via diffusion  extracellular to periplasm ',\n",
       "       'L Alaninyltryptophan transport via diffusion  extracellular to periplasm ',\n",
       "       'L-alanine export via facilited diffusion',\n",
       "       'Aldehyde dehydrogenase (butanal, NAD)',\n",
       "       'Aldose reductase (acetol)', 'Ampicilin transport  extracellular ',\n",
       "       'Ampicilin transport  periplasm ',\n",
       "       '5 aminopentanamide amidohydrolase',\n",
       "       '5-aminopentanoate:2-oxoglutarate aminotransferase',\n",
       "       'L-arabinose transport via proton symport (periplasm)',\n",
       "       'Arginine/agmatine antiport (periplasm)',\n",
       "       'L-asparagine transport via diffusion (extracellular to periplasm)',\n",
       "       'Arsenite transport via diffusion (extracellular to periplasm)',\n",
       "       'Arsenate reversible transport via symport (periplasm)',\n",
       "       'Benzoate CoA ligase', '4-hydroxybenzoate--coenzyme A ligase',\n",
       "       'Benzoylsuccinyl-CoA thiolase',\n",
       "       'Release of biotin across the basolatral membrane',\n",
       "       'Butanol synthase (NADPH)',\n",
       "       'Benzoate transport in via proton symport  2H ',\n",
       "       'Cadmium (+2) transport in via permease (no H+)',\n",
       "       'Cellulose export system', '6-carboxyhexanoate-CoA ligase',\n",
       "       'Cinnamoyl CoA Hydrolase',\n",
       "       'Cinnamic acid transport  diffusion  extra organism ',\n",
       "       'Cinnamic acid transport  diffusion  periplasm ',\n",
       "       'Cellobiose transport transport via diffusion extracellular to periplasm ',\n",
       "       'Cardiolipin hydrolase (periplasm, n-C12:0)',\n",
       "       'Cardiolipin hydrolase (periplasm, n-C14:0)',\n",
       "       'Cardiolipin hydrolase (periplasm, n-C14:1)',\n",
       "       'Cardiolipin hydrolase (periplasm, n-C16:1)',\n",
       "       'Cardiolipin hydrolase (periplasm, n-C18:0)',\n",
       "       'Chloramphenicol transport via diffusion  periplasm to cytosol ',\n",
       "       'Chloramphenicol transport via diffusion (extracellular to periplasm)',\n",
       "       'Carnitine 3 dehydrogenase',\n",
       "       'Chromate transport out via proton antiport', 'Cytosine deaminase',\n",
       "       'Carboxyspermidine dehydrogenase',\n",
       "       'Cis trans enoyl CoA isomerase  cis tetradec 2 enoyl CoA ',\n",
       "       'Cyanate transport via proton symport  periplasm ',\n",
       "       'Cyanide transport via diffusion (extracellular to periplasm)',\n",
       "       'Menaquinol oxidase  71 protons ',\n",
       "       'Cytidine transport via diffusion (extracellular to periplasm)',\n",
       "       'Cytosine 5 methyltransferase',\n",
       "       'Major Facilitator(MFS) TCDB:2.A.1.14.6', 'Deoxyadenosine kinase',\n",
       "       'Deoxyadenosine transport via diffusion (extracellular to periplasm)',\n",
       "       'D-alanine transport via ABC peptide transporter',\n",
       "       '1,5-Diaminopentane transport via diffusion (extracellular to periplasm)',\n",
       "       'D-arabinose transport via proton symport (periplasm)',\n",
       "       'Decanoate transport via diffusion (extracellular to periplasm)',\n",
       "       'Deoxycytidine transport via diffusion (extracellular to periplasm)',\n",
       "       'PG160 desaturase  pg C160  pg C161d9 ',\n",
       "       \"ATP deoxyguanosine 5'-phosphotransferase \",\n",
       "       'Deoxyguanosine transport via diffusion (extracellular to periplasm)',\n",
       "       'Dihydroxy acid dehydratase  2 3 dihydroxy 3 methylpentanoate ',\n",
       "       '1,4-dihydroxy-2-naphthoate octaprenyltransferase',\n",
       "       '1 4 dihydroxy 2 naphthoate octaprenyltransferase',\n",
       "       '1,4-dihydroxy-2-napthoyl-CoA synthase',\n",
       "       'Dihydroneopterin aldolase reversible', 'Dihydropyrimidinase',\n",
       "       'Diaminohydroxyphosphoribosylaminopyrimidine deaminase',\n",
       "       'Deoxyinosine transport via diffusion (extracellular to periplasm)',\n",
       "       'Cysteine racemase',\n",
       "       'Doxorubicin transport via diffusion (extracellular to periplasm)',\n",
       "       '5-Deoxy-D-ribose transport by diffusion',\n",
       "       'Deoxyribose-phosphate aldolase',\n",
       "       'Dihydrouracil dehydrogenase (NADP)',\n",
       "       'Deoxyuridine transport via diffusion (extracellular to periplasm)',\n",
       "       'Enoyl-[acyl-carrier-protein] reductase (NADH) (n-C14:0)',\n",
       "       'Ethanolamine transport via diffusion (extracellular)',\n",
       "       'Linoleoyl-CoA hydrolase (18:2(9Z,12Z))',\n",
       "       'Fatty acid CoA thioesterase  pentanoate ',\n",
       "       'Fatty acid CoA thioesterase  heptanoate ',\n",
       "       'Fatty acid CoA thioesterase  nonanoate ',\n",
       "       'Fatty-acid--CoA ligase (isobutanoate transport via vectoral Co-A coupling)',\n",
       "       'Fatty acid synthase  n C120 ',\n",
       "       'Iron transport from ferric dicitrate via ABC system  periplasm ',\n",
       "       'Ferrypyoverdine  dissociation reduction  Pputida specific ',\n",
       "       'Fe-enterobactin reduction (Fe(III)-unloading)',\n",
       "       'Fe-enterobactin transport via ABC system (periplasm)',\n",
       "       'Beta-fructofuranosidase', 'Dihydropteroate synthase', 'FORAMD',\n",
       "       'L-Fucose transport (transport mechanism unknown - evidence from biolog data)',\n",
       "       'L-fucose transport via diffusion (extracellular to periplasm)',\n",
       "       'Fumarylacetoacetase',\n",
       "       'Fusidic acid transport via diffusion (extracellular to periplasm)',\n",
       "       'Glycero-3-phosphocholine transport via diffusion (extracellular to periplasm)',\n",
       "       'Glycero-3-phospho-1-inositol transport via diffusion (extracellular to periplasm)',\n",
       "       'Glycerophosphserine transport via diffusion (extracellular to periplasm)',\n",
       "       'D-galactose transport via diffusion (extracellular to periplasm)',\n",
       "       'Glycogen synthase  ADPGlc   4 units ',\n",
       "       'Glycogen debranching enzyme (bglycogen -> glycogen)',\n",
       "       'L-glutamine transport via diffusion (extracellular to periplasm)',\n",
       "       'Galactitol-1-phosphate dehydrogenase',\n",
       "       '4-aminobutyrate/glutamate antiport (periplasm)',\n",
       "       'Glyoxalate carboligase',\n",
       "       'Glyceraldehyde transport via diffusion (extracellular to periplasm)',\n",
       "       'Glycine betaine CoA thioesterase',\n",
       "       'Glycine betaine transport via diffusion (extracellular to periplasm)',\n",
       "       'Sn-Glycerol 3-phosphate transport via ABC system (periplasm)',\n",
       "       'Glycerate kinase', 'Glycol transport difusion',\n",
       "       'L Glycinylglutamine transport via diffusion  extracellular to periplasm ',\n",
       "       'L Histidinylglycine transport via diffusion  extracellular to periplasm ',\n",
       "       'L Glycinylglycine transport via diffusion  extracellular to periplasm ',\n",
       "       'L Glycinylmethionine transport via diffusion  extracellular to periplasm ',\n",
       "       'L GlyPhe transport via diffusion  extracellular to periplasm ',\n",
       "       'L Glycinylserine transport via diffusion  extracellular to periplasm ',\n",
       "       'Guanosine transport via diffusion (extracellular to periplasm)',\n",
       "       'H2s transport via diffusion (extracellular to periplasm)',\n",
       "       'S 3 Hydroxyacetylthio acid transport diffusion C40  extracellularperiplasm ',\n",
       "       'S 3 Hydroxyacetylthio acid transport diffusion C40   periplasm ',\n",
       "       '3 Hydroxy 4 Acetylthiobutanoyl CoA Hydrolase',\n",
       "       'L Histidinylhistidine transport via diffusion  extracellular to periplasm ',\n",
       "       'Palmitoyl Coenzyme A Hydrolase',\n",
       "       '3-Mercaptopyruvate:Cyanide Sulfurtransferase',\n",
       "       'L-homoserine transport via diffusion (extracellular to periplasm)',\n",
       "       'Heptanoate transport via diffusion  extracellular to periplasm ',\n",
       "       '3-hydroxypropionyl-CoA:NADP+ oxidoreductase',\n",
       "       'Trans-4-Hydroxy-L-proline:NAD+ 5-oxidoreductase Arginine and proline metabolism EC:1.5.1.2',\n",
       "       'Hexanoate transport via diffusion (extracellular to periplasm)',\n",
       "       'Indole 3 acetate transport  exterior ',\n",
       "       'Indole transport via diffusion (extracellular to periplasm)',\n",
       "       'Myo-Inositol transport (transport mechanism unknown - evidence from biolog data)',\n",
       "       'Inosine transport via diffusion (extracellular to periplasm)',\n",
       "       'MNXR84838',\n",
       "       'L-alanyl-L-glutamate transport via diffusion (extracellular to periplasm)',\n",
       "       'Lactose transport via proton symport',\n",
       "       'Leucylleucine transport via diffusion  extracellular to periplasm ',\n",
       "       'Linoleic acid  n C182  transport periplasm',\n",
       "       'L-Tryptophan decarboxy-lyase', 'Lysine oxygenase',\n",
       "       'Manose epimerase', '2-methylcitrate dehydratase',\n",
       "       'Melibiose transport in via symport',\n",
       "       'Melibiose transport in via symport (periplasm)',\n",
       "       'L-methionine transport via diffusion (extracellular to periplasm)',\n",
       "       'Myo Inositol 1 phosphate synthase',\n",
       "       'D-Mannitol 1-phosphate transport (transport mechanism unknown - evidence from biolog data)',\n",
       "       'D-mannonate hydrolyase', 'Passive diffusion',\n",
       "       'M xylene  outer membrane porin transport', 'NAD nucleosidase',\n",
       "       'NAPRTase (rev)', 'N Formimino L glutamate formiminohydrolase',\n",
       "       'NMN synthetase',\n",
       "       'Nonanoate transport via diffusion  extracellular to periplasm ',\n",
       "       '3-oxopropionyl-CoA:NAD+ oxidoreductase',\n",
       "       'Octanoate transport via diffusion (extracellular to periplasm)',\n",
       "       'Phenylacetyl-CoA thioesterase',\n",
       "       'Phosphatidate phosphatase (periplasmic, n-C12:0)',\n",
       "       'Phosphatidate phosphatase (periplasmic, n-C14:0)',\n",
       "       'Phosphatidate phosphatase (periplasmic, n-C14:1)',\n",
       "       'Phosphatidate phosphatase (periplasmic, n-C16:1)',\n",
       "       'Phosphatidate phosphatase (periplasmic, n-C18:0)',\n",
       "       'Pyruvate dehydrogenase (dihydrolipoamide) reversible',\n",
       "       'Phenylethanol transport difusion',\n",
       "       '1  Phenyldecanoic acid transport via diffusion  extracellular to periplasm ',\n",
       "       '7 Phenylheptanoic acid transport via diffusion  extracellular to periplasm ',\n",
       "       '6 Phenylhexanoic transport via diffusion  extracellular to periplasm ',\n",
       "       '9 Phenylnonanoic acid transport via diffusion  extracellular to periplasm ',\n",
       "       '8 Phenyloctanoic acid transport via diffusion  extracellular to periplasm ',\n",
       "       '5 Phenylvaleric acid transport via diffusion  extracellular to periplasm ',\n",
       "       'PIMEtr', 'Phosphomethylpyrimidine kinase',\n",
       "       'Diphosphate transporter  periplasm',\n",
       "       'Propanol transport in via proton symport (periplasm)',\n",
       "       'L-Prolinylglycine transport via diffusion (extracellular to periplasm)',\n",
       "       'L-proline transport via diffusion (extracellular to periplasm)',\n",
       "       'Phospho-threonine phosphatase',\n",
       "       'Putrescine transport via diffusion (extracellular to periplasm)',\n",
       "       ' 6E  Octadec 6 enoic acid transport via facilitated irreversible diffusion  extracellular to periplasm ',\n",
       "       'Purine-nucleoside phosphorylase (Deoxyinosine)',\n",
       "       'Pyridoxal kinase', 'P methyltoluene passive diffusion',\n",
       "       'P xylene  outer membrane porin transport',\n",
       "       'Quinol monooxygenase (menaquinol 8)',\n",
       "       'Raffinose transport in via proton symport',\n",
       "       'R 3 Hydroxy acid transport diffusion C40  extracellular to periplasm ',\n",
       "       'R hydroxy acyl CoA thioesterase  C100 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C120 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C121 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C121d6 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C140 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C141 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C142 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C160 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C50 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C60 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C70 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C80 ',\n",
       "       'R hydroxy acyl CoA thioesterase  C90 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C100 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  C101 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C40 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C50 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C60 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C70 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C80 ',\n",
       "       'R hydroxy phenyl acyl CoA thioesterase  Phenyl C90 ',\n",
       "       'R hydroxy acetylthio acyl CoA thioesterase  Acetylthio C40 ',\n",
       "       'R hydroxy acetylthio acyl CoA thioesterase  Acetylthio C60 ',\n",
       "       ' 3R  3 Hydroxyacyl CoANADP oxidoreductase',\n",
       "       'S-ribosylhomocysteine cleavage enzyme',\n",
       "       'Ribose transport via diffusion (extracellular to periplasm)',\n",
       "       'Salmochelin S4 Fe III reduction Fe III unloading ',\n",
       "       'Salmochelin S4 Fe III transport via ABC system periplasm ',\n",
       "       'Salmochelin S4 transport secretion periplasm ',\n",
       "       'Sarcosine oxidase',\n",
       "       'D sorbitol dehydrogenase  D fructose producing ',\n",
       "       'Selenite transport via diffusion, cytosol',\n",
       "       'Selenate transport via diffusion (extracellular to periplasm)',\n",
       "       'Siderophore Mediated Iron Acquisition: Staphyloferrin A transport via ABC',\n",
       "       'Spermidine transport in via proton antiport',\n",
       "       'Ferredoxin-sulfite reductase', 'Tagaturonate reductase',\n",
       "       'Thiocyanate transport via diffusion (extracellular to periplasm)',\n",
       "       'Tagatose-bisphosphate aldolase',\n",
       "       'Thymidine transport via diffusion (extracellular to periplasm)',\n",
       "       'Thiamin phosphatase',\n",
       "       'Thymine transport via diffusion (extracellular to periplasm)',\n",
       "       'Thymidine phosphorylase',\n",
       "       'Trinitrotoluene transport via diffusion  extracellular to periplasm  irreversible',\n",
       "       'Trinitrotoluene transport via diffusion  periplasm to cytosol  irreversible',\n",
       "       'Toluene transport via diffusion',\n",
       "       'Trehalose transport via diffusion (extracellular to periplasm)',\n",
       "       'Tartronate semialdehyde reductase', 'Tetracycline transport',\n",
       "       'Tetracycline transport via diffusion (extracellular to periplasm)',\n",
       "       'UDP N acetylglucosamine diphosphorylase',\n",
       "       'UDP-N-acetylmuramoyl-L-alanyl-D-gamma-glutamyl-meso-2,6-diaminopimelate-D-alanine L,D-carboxypeptidase',\n",
       "       'Uridine transport via diffusion (extracellular to periplasm)',\n",
       "       'Xanthine reversible transport (periplasm)',\n",
       "       'Xanthosine transport via diffusion (extracellular to periplasm)',\n",
       "       'D xylose reversible transport', 'R_sink_2ohph_c',\n",
       "       'R_sink_4crsol_c', 'R_sink_amob_c', 'R_sink_hemeO_c',\n",
       "       'R_sink_mobd_c', 'R_EX_12ppd__R_e', 'R_EX_12ppd__S_e',\n",
       "       'R_EX_14glucan_e', 'R_EX_15dap_e', 'R_EX_25dkglcn_e',\n",
       "       'R_EX_2ameph_e', 'R_EX_2m35mdntha_e', 'R_EX_35dnta_e',\n",
       "       'R_EX_3h4atb_e', 'R_EX_3mb_e', 'R_EX_4hba_e', 'R_EX_4hbald_e',\n",
       "       'R_EX_4hbz_e', 'R_EX_5dglcn_e', 'R_EX_5drib_e', 'R_EX_6atha_e',\n",
       "       'R_EX_LalaLglu_e', 'R_EX_abg4_e', 'R_EX_abt_e', 'R_EX_ac_e',\n",
       "       'R_EX_acac_e', 'R_EX_acald_e', 'R_EX_acgam_e', 'R_EX_actn__R_e',\n",
       "       'R_EX_ade_e', 'R_EX_adn_e', 'R_EX_ala_L_asp__L_e',\n",
       "       'R_EX_ala_L_glu__L_e', 'R_EX_ala_L_thr__L_e', 'R_EX_ala__D_e',\n",
       "       'R_EX_ala__L_e', 'R_EX_ala_gln_e', 'R_EX_ala_his_e',\n",
       "       'R_EX_ala_leu_e', 'R_EX_alagly_e', 'R_EX_alahis_e',\n",
       "       'R_EX_alaleu_e', 'R_EX_alathr_e', 'R_EX_alatrp_e', 'R_EX_alltn_e',\n",
       "       'R_EX_arab__L_e', 'R_EX_arbt6p_e', 'R_EX_arbt_e', 'R_EX_arg__L_e',\n",
       "       'R_EX_asn__L_e', 'R_EX_aso3_e', 'R_EX_aso4_e', 'R_EX_asp__L_e',\n",
       "       'R_EX_bhb_e', 'R_EX_btn_e', 'R_EX_but_e', 'R_EX_bz_e',\n",
       "       'R_EX_ca2_e', 'R_EX_cell4_e', 'R_EX_cellb_e', 'R_EX_cgly_e',\n",
       "       'R_EX_chol_e', 'R_EX_cinnm_e', 'R_EX_cit_e', 'R_EX_cl_e',\n",
       "       'R_EX_co2_e', 'R_EX_co_e', 'R_EX_cobalt2_e', 'R_EX_crn_e',\n",
       "       'R_EX_csn_e', 'R_EX_cu2_e', 'R_EX_cyan_e', 'R_EX_cys__L_e',\n",
       "       'R_EX_cytd_e', 'R_EX_dad_2_e', 'R_EX_dca_e', 'R_EX_dcyt_e',\n",
       "       'R_EX_ddca_e', 'R_EX_dextrin_e', 'R_EX_dgsn_e', 'R_EX_din_e',\n",
       "       'R_EX_drib_e', 'R_EX_duri_e', 'R_EX_enter_e', 'R_EX_etha_e',\n",
       "       'R_EX_etoh_e', 'R_EX_fald_e', 'R_EX_fe2_e', 'R_EX_fe3_e',\n",
       "       'R_EX_fe3dcit_e', 'R_EX_fe3pyovd_kt_e', 'R_EX_feenter_e',\n",
       "       'R_EX_for_e', 'R_EX_fru_e', 'R_EX_fuc__L_e', 'R_EX_fuc_e',\n",
       "       'R_EX_fum_e', 'R_EX_g3pc_e', 'R_EX_g3pi_e', 'R_EX_g3ps_e',\n",
       "       'R_EX_gal_e', 'R_EX_galt_e', 'R_EX_galur_e', 'R_EX_gam_e',\n",
       "       'R_EX_glc__D_e', 'R_EX_glcur_e', 'R_EX_gln__L_e', 'R_EX_glu__L_e',\n",
       "       'R_EX_gly_asn__L_e', 'R_EX_gly_asp__L_e', 'R_EX_gly_cys_e',\n",
       "       'R_EX_gly_e', 'R_EX_gly_gln_e', 'R_EX_gly_glu__L_e',\n",
       "       'R_EX_gly_leu_e', 'R_EX_gly_met_e', 'R_EX_gly_phe_e',\n",
       "       'R_EX_gly_pro__L_e', 'R_EX_gly_tyr_e', 'R_EX_glyald_e',\n",
       "       'R_EX_glyb_e', 'R_EX_glyc_e', 'R_EX_glycol_e', 'R_EX_glygln_e',\n",
       "       'R_EX_glyglu_e', 'R_EX_glygly_e', 'R_EX_glyglygln_e',\n",
       "       'R_EX_glymet_e', 'R_EX_glyphe_e', 'R_EX_glyser_e', 'R_EX_gsn_e',\n",
       "       'R_EX_gthrd_e', 'R_EX_gua_e', 'R_EX_h2_e', 'R_EX_h2o2_e',\n",
       "       'R_EX_h2o_e', 'R_EX_h2s_e', 'R_EX_h_e', 'R_EX_hdca_e',\n",
       "       'R_EX_hdcea_e', 'R_EX_hisgly_e', 'R_EX_hishis_e', 'R_EX_hom__L_e',\n",
       "       'R_EX_hpta_e', 'R_EX_hxa_e', 'R_EX_hxan_e', 'R_EX_ibt_e',\n",
       "       'R_EX_ile__L_e', 'R_EX_ind3ac_e', 'R_EX_indole_e', 'R_EX_inost_e',\n",
       "       'R_EX_ins_e', 'R_EX_istfrnB_e', 'R_EX_k_e', 'R_EX_lac__D_e',\n",
       "       'R_EX_lac__L_e', 'R_EX_lcts_e', 'R_EX_leu__L_e', 'R_EX_leuleu_e',\n",
       "       'R_EX_lnlc_e', 'R_EX_lys__L_e', 'R_EX_lysglugly_e', 'R_EX_m_xyl_e',\n",
       "       'R_EX_mal__L_e', 'R_EX_man_e', 'R_EX_melib_e', 'R_EX_meoh_e',\n",
       "       'R_EX_met_L_ala__L_e', 'R_EX_met__D_e', 'R_EX_met__L_e',\n",
       "       'R_EX_mg2_e', 'R_EX_mn2_e', 'R_EX_mnl1p_e', 'R_EX_mnl_e',\n",
       "       'R_EX_mobd_e', 'R_EX_mththf_e', 'R_EX_nh4_e', 'R_EX_no2_e',\n",
       "       'R_EX_no3_e', 'R_EX_nona_e', 'R_EX_o2_e', 'R_EX_ocdca_e',\n",
       "       'R_EX_ocdcea_e', 'R_EX_octa_e', 'R_EX_oxa_e', 'R_EX_p_xyl_e',\n",
       "       'R_EX_pea_e', 'R_EX_phedca_e', 'R_EX_phehpa_e', 'R_EX_phehxa_e',\n",
       "       'R_EX_pheme_e', 'R_EX_phenona_e', 'R_EX_pheocta_e', 'R_EX_phept_e',\n",
       "       'R_EX_pi_e', 'R_EX_pime_e', 'R_EX_ppa_e', 'R_EX_ppi_e',\n",
       "       'R_EX_ppoh_e', 'R_EX_pro__L_e', 'R_EX_progly_e', 'R_EX_pta_e',\n",
       "       'R_EX_ptrc_e', 'R_EX_ptsla_e', 'R_EX_pyovd_kt_e', 'R_EX_pyr_e',\n",
       "       'R_EX_quin_e', 'R_EX_raffin_e', 'R_EX_rib__D_e', 'R_EX_s_e',\n",
       "       'R_EX_salchs4_e', 'R_EX_salchs4fe_e', 'R_EX_sbt__D_e',\n",
       "       'R_EX_sel_e', 'R_EX_serglugly_e', 'R_EX_sheme_e', 'R_EX_slnt_e',\n",
       "       'R_EX_so3_e', 'R_EX_so4_e', 'R_EX_spmd_e', 'R_EX_starch_e',\n",
       "       'R_EX_stfrnB_e', 'R_EX_succ_e', 'R_EX_sucr_e', 'R_EX_tcynt_e',\n",
       "       'R_EX_thym_e', 'R_EX_thymd_e', 'R_EX_tnt_e', 'R_EX_tol_e',\n",
       "       'R_EX_tre_e', 'R_EX_tsul_e', 'R_EX_ttdca_e', 'R_EX_ttdcea_e',\n",
       "       'R_EX_udcpdp_e', 'R_EX_udcpp_e', 'R_EX_ura_e', 'R_EX_urea_e',\n",
       "       'R_EX_uri_e', 'R_EX_val__L_e', 'R_EX_xan_e', 'R_EX_xtsn_e',\n",
       "       'R_EX_xyl__D_e', 'R_EX_zn2_e', 'Biomass reaction', 'R_ATPM'],\n",
       "      dtype=object)"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.loc[df.genes_number == 0, 'reaction'].unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>reaction</th>\n",
       "      <th>rid</th>\n",
       "      <th>genes</th>\n",
       "      <th>genes_number</th>\n",
       "      <th>sbo</th>\n",
       "      <th>metanetx.reaction</th>\n",
       "      <th>biocyc</th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>ec-code</th>\n",
       "      <th>rhea</th>\n",
       "      <th>kegg.reaction</th>\n",
       "      <th>reactome</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2340</th>\n",
       "      <td>Phenylpyruvate oxidation chemical</td>\n",
       "      <td>PHPYROX</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000176</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                reaction      rid genes  genes_number  \\\n",
       "2340  Phenylpyruvate oxidation chemical   PHPYROX    []             0   \n",
       "\n",
       "              sbo metanetx.reaction biocyc seed.reaction ec-code rhea  \\\n",
       "2340  SBO:0000176               NaN    NaN           NaN     NaN  NaN   \n",
       "\n",
       "     kegg.reaction reactome  \n",
       "2340           NaN      NaN  "
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfgap.loc[~dfgap.rid.isin(df.rid)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Reaction identifier</strong></td><td>PHPYROX</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>Phenylpyruvate oxidation chemical</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x7f5942357fd0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Stoichiometry</strong></td>\n",
       "                <td>\n",
       "                    <p style='text-align:right'>o2_c + phpyr_c --> bzal_c + h_c + oxa_c</p>\n",
       "                    <p style='text-align:right'>O2 O2 + Phenylpyruvate --> Benzaldehyde + H+ + Oxalate</p>\n",
       "                </td>\n",
       "            </tr><tr>\n",
       "                <td><strong>GPR</strong></td><td></td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Lower bound</strong></td><td>0.0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Upper bound</strong></td><td>1000.0</td>\n",
       "            </tr>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<Reaction PHPYROX at 0x7f5942357fd0>"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.reactions.PHPYROX"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['12DGR120tipp', '12DGR140tipp', '12DGR141tipp', ..., 'EX_zn2_e',\n",
       "       'Growth', 'ATPM'], dtype=object)"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.rid.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "jfpath = '../../../../tools/ModelSEEDDatabase/Biochemistry/reactions.json'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "with open(jfpath) as user_file:\n",
    "  parsed_json = json.load(user_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'abbreviation': 'R00004',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: INORGPYROPHOSPHAT-RXN',\n",
       "   'BiGG: IPP1; PPA; PPA_1; PPAm',\n",
       "   'BrachyCyc: INORGPYROPHOSPHAT-RXN',\n",
       "   'KEGG: R00004',\n",
       "   'MetaCyc: INORGPYROPHOSPHAT-RXN',\n",
       "   'Name: Diphosphate phosphohydrolase; Inorganic diphosphatase; Inorganic pyrophosphatase; Pyrophosphate phosphohydrolase; diphosphate phosphohydrolase; inorganic diphosphatase; inorganic diphosphatase (one proton translocation); inorganicdiphosphatase; pyrophosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00012[0] <=> (2) cpd00009[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00012;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) PPi[0] <=> (2) Phosphate[0] + (1) H+[0]',\n",
       "  'deltag': -3.46,\n",
       "  'deltagerr': 0.05,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.6.1.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00012[0] <=> (2) cpd00009[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00001',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27946;rxn27947;rxn27948;rxn32487;rxn38157;rxn38158',\n",
       "  'name': 'diphosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Glyphosate-Degradation (glyphosate degradation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-7805 ((aminomethyl)phosphonate degradation); PWY-7807 (glyphosate degradation III); Phosphorus-Compounds (Phosphorus Compound Metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00012:0:0:\"PPi\";2:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00005',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ALLOPHANATE-HYDROLASE-RXN',\n",
       "   'BiGG: ALPHNH; DUR1_2',\n",
       "   'ChlamyCyc: ALLOPHANATE-HYDROLASE-RXN',\n",
       "   'KEGG: R00005',\n",
       "   'MetaCyc: ALLOPHANATE-HYDROLASE-RXN',\n",
       "   'Name: Allophanate hydrolase; AtzF; TrzF; Urea-1-carboxylate amidohydrolase; allophanate hydrolase; allophanate lyase; urea-1-carboxylate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00742[0] <=> (2) cpd00011[0] + (2) cpd00013[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00067;cpd00742',\n",
       "  'definition': '(1) H2O[0] + (3) H+[0] + (1) Allophanate[0] => (2) CO2[0] + (2) NH3[0]',\n",
       "  'deltag': -20.14,\n",
       "  'deltagerr': 1.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.54'],\n",
       "  'equation': '(1) cpd00001[0] + (3) cpd00067[0] + (1) cpd00742[0] => (2) cpd00011[0] + (2) cpd00013[0]',\n",
       "  'id': 'rxn00002',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30346;rxn35525',\n",
       "  'name': 'urea-1-carboxylate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALLANTOINDEG-PWY (superpathway of allantoin degradation in yeast); AMINE-DEG (Amine and Polyamine Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Allantoin-degradation (Allantoin Degradation); Atrazine-Degradation (Atrazine Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5169 (cyanurate degradation); PWY-5703 (urea degradation I); PWY-5724 (superpathway of atrazine degradation); Urea-Degradation (Urea Degradation); s-Triazine-Degradation (<i>s</i>-Triazine Degredation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn00791 (Atrazine degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-3:cpd00067:0:0:\"H+\";-1:cpd00742:0:0:\"Allophanate\";2:cpd00011:0:0:\"CO2\";2:cpd00013:0:0:\"NH3\"'},\n",
       " {'abbreviation': 'R00006',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00006_c',\n",
       "   'AraGEM: R_R00006_c',\n",
       "   'BiGG: ILV2_2',\n",
       "   'DF_Athaliana: reac_140',\n",
       "   'KEGG: R00006',\n",
       "   'Name: 2-acetolactate pyruvate-lyase (carboxylating); Acetolactate synthase; Acetolactate synthase catalytic subunit; acetolactate synthase (Also catalyzes ACHBS); acetolactate synthase large subunit; acetolactate synthase small subunit; pyruvate:pyruvate acetaldehydetransferase (decarboxylating)'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00668[0] <=> (2) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00020;cpd00067;cpd00668',\n",
       "  'definition': '(1) CO2[0] + (1) ALCTT[0] <= (2) Pyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 8.27,\n",
       "  'deltagerr': 0.9,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.2.1.6'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00668[0] <= (2) cpd00020[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00003',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30144;rxn33164',\n",
       "  'name': 'pyruvate:pyruvate acetaldehydetransferase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00668:0:0:\"ALCTT\";2:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00008',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00008',\n",
       "   'Name: 4-Hydroxy-4-methyl-2-oxoglutarate pyruvate-lyase; 4-hydroxy-4-methyl-2-oxoglutarate pyruvate-lyase (pyruvate-forming)'],\n",
       "  'code': '(1) cpd02570[0] <=> (2) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00020;cpd02570',\n",
       "  'definition': '(1) Parapyruvate[0] <=> (2) Pyruvate[0]',\n",
       "  'deltag': 4.49,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.17'],\n",
       "  'equation': '(1) cpd02570[0] <=> (2) cpd00020[0]',\n",
       "  'id': 'rxn00004',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxy-4-methyl-2-oxoglutarate pyruvate-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation); rn00660 (C5-Branched dibasic acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02570:0:0:\"Parapyruvate\";2:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00009',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CATAL-RXN',\n",
       "   'BiGG: CAT; CATp; CTA1; CTT1',\n",
       "   'BrachyCyc: CATAL-RXN',\n",
       "   'KEGG: R00009',\n",
       "   'MetaCyc: CATAL-RXN; RXN-12121',\n",
       "   'Name: Catalase; caperase; catalase A; catalase-peroxidase; donor:hydrogen-peroxide oxidoreductase; equilase; hydrogen-peroxide:hydrogen-peroxide oxidoreductase; katG (gene name); optidase'],\n",
       "  'code': '(2) cpd00025[0] <=> (2) cpd00001[0] + (1) cpd00007[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025',\n",
       "  'definition': '(2) H2O2[0] => (2) H2O[0] + (1) O2[0]',\n",
       "  'deltag': -46.06,\n",
       "  'deltagerr': 1.64,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.21', '1.11.1.6'],\n",
       "  'equation': '(2) cpd00025[0] => (2) cpd00001[0] + (1) cpd00007[0]',\n",
       "  'id': 'rxn00006',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19264;rxn22404;rxn27744;rxn31381',\n",
       "  'name': 'hydrogen-peroxide:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); DETOX1-PWY (superoxide radicals degradation); DETOX1-PWY-1 (reactive oxygen species degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); Methanol-Oxidation (Methanol Oxidation); PWY-5506 (methanol oxidation to formaldehyde IV); REACTIVE-OXYGEN-SPECIES-DEGRADATION (Reactive Oxygen Specie Degradation)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00025:0:0:\"H2O2\";2:cpd00001:0:0:\"H2O\";1:cpd00007:0:0:\"O2\"'},\n",
       " {'abbreviation': 'R00010',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00010_c',\n",
       "   'AraGEM: R_R00010_c',\n",
       "   'BiGG: ATH1; NTH1; NTH2; TREH; TREHe; TREHpp; TREHv',\n",
       "   'KEGG: R00010; R06103',\n",
       "   'Maize_C4GEM: R_R00010_c',\n",
       "   'Name: alpha,alpha-Trehalose glucohydrolase; alpha,alpha-trehalase; alpha,alpha-trehalose glucohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00794[0] <=> (2) cpd00027[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00794',\n",
       "  'definition': '(1) H2O[0] + (1) TRHL[0] => (2) D-Glucose[0]',\n",
       "  'deltag': -2.79,\n",
       "  'deltagerr': 0.26,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.28'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00794[0] => (2) cpd00027[0]',\n",
       "  'id': 'rxn00007',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29977;rxn30219;rxn30442',\n",
       "  'name': 'alpha,alpha-trehalose glucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00794:0:0:\"TRHL\";2:cpd00027:0:0:\"D-Glucose\"'},\n",
       " {'abbreviation': 'R00011',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00011',\n",
       "   'MetaCyc: MANGANESE-PEROXIDASE-RXN',\n",
       "   'Name: Mn(II):hydrogen-peroxide oxidoreductase; Mn-dependent (NADH-oxidizing) peroxidase; manganese peroxidase; peroxidase-M2'],\n",
       "  'code': '(2) cpd00001[0] <=> (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00025;cpd00067',\n",
       "  'definition': '(2) H2O[0] <=> (1) H2O2[0] + (2) H+[0]',\n",
       "  'deltag': 62.53,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.11.1.13'],\n",
       "  'equation': '(2) cpd00001[0] <=> (1) cpd00025[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00008',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Mn(II):hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Manganese-Oxidation (Manganese Oxidation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-6591 (manganese oxidation I)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";1:cpd00025:0:0:\"H2O2\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00012',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00012',\n",
       "   'MetaCyc: 2.7.7.45-RXN',\n",
       "   'TS_Athaliana: rxn555_r; rxn555_x',\n",
       "   'Name: GTP-GTP guanylyltransferase; GTP:GTP guanylyltransferase; Gp4G synthetase; diguanosine tetraphosphate synthetase; guanosine triphosphate-guanose triphosphate guanylyltransferase; guanosine-triphosphate guanylyltransferase'],\n",
       "  'code': '(2) cpd00038[0] <=> (1) cpd00012[0] + (1) cpd00925[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00038;cpd00067;cpd00925',\n",
       "  'definition': '(2) GTP[0] => (1) PPi[0] + (1) H+[0] + (1) GppppG[0]',\n",
       "  'deltag': 2.7,\n",
       "  'deltagerr': 1.52,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.7.45'],\n",
       "  'equation': '(2) cpd00038[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00925[0]',\n",
       "  'id': 'rxn00009',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33150;rxn33152',\n",
       "  'name': 'GTP:GTP guanylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00038:0:0:\"GTP\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd00925:0:0:\"GppppG\"'},\n",
       " {'abbreviation': 'R00013',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GLXCBL; GLXCL',\n",
       "   'EcoCyc: GLYOCARBOLIG-RXN',\n",
       "   'KEGG: R00013',\n",
       "   'MetaCyc: GLYOCARBOLIG-RXN',\n",
       "   'RiceCyc: GLYOCARBOLIG-RXN',\n",
       "   'Name: Glyoxylate carboxy-lyase (dimerizing); glyoxalate carboligase; glyoxylate carbo-ligase; glyoxylate carboxy-lyase (dimerizing); glyoxylate carboxy-lyase (dimerizing, tartronate-semialdehyde-forming); glyoxylate carboxy-lyase (dimerizing; tartronate-semialdehyde-forming); glyoxylic carbo-ligase; hydroxymalonic semialdehyde carboxylase; tartronate semialdehyde carboxylase; tartronate-semialdehyde synthase; tartronic semialdehyde carboxylase'],\n",
       "  'code': '(2) cpd00040[0] <=> (1) cpd00011[0] + (1) cpd00843[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00040;cpd00067;cpd00843',\n",
       "  'definition': '(2) Glyoxalate[0] + (1) H+[0] => (1) CO2[0] + (1) Tartronate semialdehyde[0]',\n",
       "  'deltag': -4.57,\n",
       "  'deltagerr': 1.08,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.47'],\n",
       "  'equation': '(2) cpd00040[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00843[0]',\n",
       "  'id': 'rxn00010',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glyoxylate carboxy-lyase (dimerizing; tartronate-semialdehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); GLYCOL-GLYOXDEG-PWY (superpathway of glycol metabolism and degradation); GLYCOLATEMET-PWY (glycolate and glyoxylate degradation I); Glycolate-Degradation (Glycolate Degradation)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00040:0:0:\"Glyoxalate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00843:0:0:\"Tartronate semialdehyde\"'},\n",
       " {'abbreviation': 'R00014',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-12583',\n",
       "   'CornCyc: RXN-12583',\n",
       "   'EcoCyc: RXN-12583',\n",
       "   'KEGG: R00014',\n",
       "   'MetaCyc: RXN-12583',\n",
       "   'Name: pyruvate dehydrogenase E1 component alpha subunit; pyruvate:thiamin diphosphate acetaldehydetransferase; pyruvate:thiamin diphosphate acetaldehydetransferase (decarboxylating)'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd03049[0] <=> (1) cpd00020[0] + (1) cpd00056[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00020;cpd00056;cpd00067;cpd03049',\n",
       "  'definition': '(1) CO2[0] + (1) 2-Hydroxyethyl-ThPP[0] => (1) Pyruvate[0] + (1) TPP[0] + (1) H+[0]',\n",
       "  'deltag': 7.23,\n",
       "  'deltagerr': 0.84,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.4.1', '2.2.1.6', '4.1.1.1'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd03049[0] => (1) cpd00020[0] + (1) cpd00056[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00011',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31360;rxn31361',\n",
       "  'name': 'pyruvate:thiamin diphosphate acetaldehydetransferase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd03049:0:0:\"2-Hydroxyethyl-ThPP\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00056:0:0:\"TPP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00015',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00015',\n",
       "   'MetaCyc: RXN-1781',\n",
       "   \"Name: Sucrose:sucrose 1F-beta-D-fructosyltransferase; sucrose:sucrose 1'-beta-D-fructosyltransferase; sucrose:sucrose 1F-beta-D-fructosyltransferase\"],\n",
       "  'code': '(2) cpd00076[0] <=> (1) cpd00027[0] + (1) cpd02298[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00076;cpd02298',\n",
       "  'definition': '(2) Sucrose[0] <=> (1) D-Glucose[0] + (1) 1F-beta-D-Fructosylsucrose[0]',\n",
       "  'deltag': -1.36,\n",
       "  'deltagerr': 1.32,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.99'],\n",
       "  'equation': '(2) cpd00076[0] <=> (1) cpd00027[0] + (1) cpd02298[0]',\n",
       "  'id': 'rxn00012',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"sucrose:sucrose 1'-beta-D-fructosyltransferase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Glycan-Biosynthesis (Glycan Biosynthesis); Glycan-Pathways (Glycan Pathways); PWY-822 (fructan biosynthesis); Polysaccharides-Biosynthesis (Polysaccharide Biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00076:0:0:\"Sucrose\";1:cpd00027:0:0:\"D-Glucose\";1:cpd02298:0:0:\"1F-beta-D-Fructosylsucrose\"'},\n",
       " {'abbreviation': 'G1PPDM',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00016',\n",
       "   'MetaCyc: GLUCOSE-1-PHOSPHATE-PHOSPHODISMUTASE-RXN',\n",
       "   'iAG612: rbs_18',\n",
       "   'iYO844: G1PPDM',\n",
       "   'Name: D-Glucose-1-phosphate:D-glucose-1-phosphate 6-phosphotransferase; glucose-1-phosphate phosphodismutase'],\n",
       "  'code': '(2) cpd00089[0] <=> (1) cpd00027[0] + (1) cpd00499[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00089;cpd00499',\n",
       "  'definition': '(2) Glucose-1-phosphate[0] <=> (1) D-Glucose[0] + (1) D-glucose-1,6-biphosphate[0]',\n",
       "  'deltag': -3.73,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.41'],\n",
       "  'equation': '(2) cpd00089[0] <=> (1) cpd00027[0] + (1) cpd00499[0]',\n",
       "  'id': 'rxn00013',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose-1-phosphate:D-glucose-1-phosphate 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00499:0:0:\"D-glucose-1,6-biphosphate\"'},\n",
       " {'abbreviation': 'R00017',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CCP; CCP2m; CYCPO',\n",
       "   'JP_Creinhardtii_MSB: R_CCP2m',\n",
       "   'KEGG: R00017',\n",
       "   'iIT341: CCP',\n",
       "   'iMM904: CCP2m',\n",
       "   'Name: Cytochrome c peroxidase; Cytochromecperoxidase; Ferrocytochrome-c:hydrogen-peroxide oxidoreductase; ferrocytochrome-c:hydrogen-peroxide oxidoreductase'],\n",
       "  'code': '(1) cpd00025[0] + (2) cpd00110[0] <=> (2) cpd00001[0] + (2) cpd00109[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00025;cpd00067;cpd00109;cpd00110',\n",
       "  'definition': '(1) H2O2[0] + (2) H+[0] + (2) Cytochrome c2+[0] => (2) H2O[0] + (2) Cytochrome c3+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.5'],\n",
       "  'equation': '(1) cpd00025[0] + (2) cpd00067[0] + (2) cpd00110[0] => (2) cpd00001[0] + (2) cpd00109[0]',\n",
       "  'id': 'rxn00014',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30356',\n",
       "  'name': 'ferrocytochrome-c:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00025:0:0:\"H2O2\";-2:cpd00067:0:0:\"H+\";-2:cpd00110:0:0:\"Cytochrome c2+\";2:cpd00001:0:0:\"H2O\";2:cpd00109:0:0:\"Cytochrome c3+\"'},\n",
       " {'abbreviation': 'R00018',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: HSPMS',\n",
       "   'KEGG: R00018',\n",
       "   'MetaCyc: 2.5.1.44-RXN',\n",
       "   'iAF692: HSPMS',\n",
       "   'Name: Putrescine:putrescine 4-aminobutyltransferase (ammonia-forming); homospermidine synthase; putrescine:putrescine 4-aminobutyltransferase (ammonia-forming)'],\n",
       "  'code': '(2) cpd00118[0] <=> (1) cpd00013[0] + (1) cpd03802[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00118;cpd03802',\n",
       "  'definition': '(2) Putrescine[0] <=> (1) NH3[0] + (1) Homospermidine[0]',\n",
       "  'deltag': -1.2,\n",
       "  'deltagerr': 1.08,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.5.1.44'],\n",
       "  'equation': '(2) cpd00118[0] <=> (1) cpd00013[0] + (1) cpd03802[0]',\n",
       "  'id': 'rxn00015',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30077',\n",
       "  'name': 'putrescine:putrescine 4-aminobutyltransferase (ammonia-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00118:0:0:\"Putrescine\";1:cpd00013:0:0:\"NH3\";1:cpd03802:0:0:\"Homospermidine\"'},\n",
       " {'abbreviation': 'R00022',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-12625',\n",
       "   'BiGG: HXAD',\n",
       "   'CornCyc: RXN-12625',\n",
       "   'KEGG: R00022',\n",
       "   'MetaCyc: RXN-12625',\n",
       "   \"Name: Hexosaminidase; N,N'-diacetylchitobiose beta-N-acetylglucosaminidase; chitobiose N-acetylglucosaminohydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01157[0] <=> (2) cpd00122[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00122;cpd01157',\n",
       "  'definition': '(1) H2O[0] + (1) Chitobiose[0] => (2) N-Acetyl-D-glucosamine[0]',\n",
       "  'deltag': -3.2,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.52'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01157[0] => (2) cpd00122[0]',\n",
       "  'id': 'rxn00016',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn23055',\n",
       "  'name': 'chitobiose N-acetylglucosaminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01157:0:0:\"Chitobiose\";2:cpd00122:0:0:\"N-Acetyl-D-glucosamine\"'},\n",
       " {'abbreviation': 'R00023',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00023',\n",
       "   'MetaCyc: HYPONITRITE-REDUCTASE-RXN',\n",
       "   'Name: NADH2:hyponitrite oxidoreductase; NADH:hyponitrite oxidoreductase; hydroxylamine:NAD+ oxidoreductase; hyponitrite reductase'],\n",
       "  'code': '(2) cpd00003[0] + (2) cpd00165[0] <=> (2) cpd00004[0] + (1) cpd01252[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00165;cpd01252',\n",
       "  'definition': '(2) NAD[0] + (2) Hydroxylamine[0] <= (2) NADH[0] + (2) H+[0] + (1) Hyponitrite[0]',\n",
       "  'deltag': 18.48,\n",
       "  'deltagerr': 13.84,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.7.1.5'],\n",
       "  'equation': '(2) cpd00003[0] + (2) cpd00165[0] <= (2) cpd00004[0] + (2) cpd00067[0] + (1) cpd01252[0]',\n",
       "  'id': 'rxn00017',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'hydroxylamine:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00003:0:0:\"NAD\";-2:cpd00165:0:0:\"Hydroxylamine\";2:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd01252:0:0:\"Hyponitrite\"'},\n",
       " {'abbreviation': 'R00024',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RIBULOSE-BISPHOSPHATE-CARBOXYLASE-RXN',\n",
       "   'BiGG: RBPC',\n",
       "   'BrachyCyc: RIBULOSE-BISPHOSPHATE-CARBOXYLASE-RXN',\n",
       "   'KEGG: R00024',\n",
       "   'MetaCyc: RIBULOSE-BISPHOSPHATE-CARBOXYLASE-RXN',\n",
       "   'Name: 3-phospho-D-glycerate carboxy-lyase (dimerizing); 3-phospho-D-glycerate carboxy-lyase (dimerizing, D-ribulose-1,5-bisphosphate-forming); 3-phospho-D-glycerate carboxy-lyase (dimerizing; D-ribulose-1,5-bisphosphate-forming); D-Ribulose 1,5-bisphosphate carboxy-lyase (dimerizing); D-ribulose 1,5-diphosphate carboxylase; D-ribulose-1,5-bisphosphate carboxylase; RuBP carboxylase; carboxydismutase; diphosphoribulose carboxylase; ribulose 1,5-bisphosphate carboxylase; ribulose 1,5-bisphosphate carboxylase/oxygenase; ribulose 1,5-diphosphate carboxylase; ribulose 1,5-diphosphate carboxylase/oxygenase; ribulose bisphosphate carboxylase/oxygenase; ribulose diphosphate carboxylase; ribulose diphosphate carboxylase/oxygenase; ribulose-bisphosphate carboxylase; rubisco'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00871[0] <=> (2) cpd00169[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00067;cpd00169;cpd00871',\n",
       "  'definition': '(1) H2O[0] + (1) CO2[0] + (1) D-Ribulose 1,5-bisphosphate[0] <= (2) H+[0] + (2) 3-Phosphoglycerate[0]',\n",
       "  'deltag': -6.24,\n",
       "  'deltagerr': 0.92,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['4.1.1.39'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00871[0] <= (2) cpd00067[0] + (2) cpd00169[0]',\n",
       "  'id': 'rxn00018',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20788;rxn38286',\n",
       "  'name': '3-phospho-D-glycerate carboxy-lyase (dimerizing; D-ribulose-1,5-bisphosphate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CALVIN-PWY (Calvin-Benson-Bassham cycle); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethylene-Biosynthesis (Ethylene Biosynthesis); Fermentation (); HORMONE-SYN (Hormone Biosynthesis); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PHOTOALL-PWY (oxygenic photosynthesis); PWY-5532 (nucleoside and nucleotide degradation (archaea)); PWY-5723 (Rubisco shunt); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); Photosynthesis (); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis)',\n",
       "   'KEGG: rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00011:0:0:\"CO2\";-1:cpd00871:0:0:\"D-Ribulose 1,5-bisphosphate\";2:cpd00067:0:0:\"H+\";2:cpd00169:0:0:\"3-Phosphoglycerate\"'},\n",
       " {'abbreviation': '2NPDO',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['iMO1053-PAO1: 2NPDO',\n",
       "   'iMO1056: 2NPDO',\n",
       "   'Name: 2-Nitropropane:oxygen 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd01433[0] <=> (2) cpd00075[0] + (2) cpd00178[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00075;cpd00178;cpd01433',\n",
       "  'definition': '(1) O2[0] + (2) 2-Nitropropane[0] <=> (2) Nitrite[0] + (2) Acetone[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.13.11.32'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd01433[0] <=> (2) cpd00075[0] + (2) cpd00178[0]',\n",
       "  'id': 'rxn00019',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Nitropropane:oxygen 2-oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd01433:0:0:\"2-Nitropropane\";2:cpd00075:0:0:\"Nitrite\";2:cpd00178:0:0:\"Acetone\"'},\n",
       " {'abbreviation': 'R00026',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraGEM: R_R00026_c',\n",
       "   'KEGG: R00026; R06092',\n",
       "   'Maize_C4GEM: R_R00026_c',\n",
       "   'TS_Athaliana: rxn1059_r',\n",
       "   'iAO358: rll_74',\n",
       "   'Name: beta-D-Glucoside glucohydrolase; cellobiose glucohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00158[0] <=> (2) cpd00190[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00158;cpd00190',\n",
       "  'definition': '(1) H2O[0] + (1) CELB[0] => (2) beta-D-Glucose[0]',\n",
       "  'deltag': -2.63,\n",
       "  'deltagerr': 0.42,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00158[0] => (2) cpd00190[0]',\n",
       "  'id': 'rxn00020',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33580',\n",
       "  'name': 'cellobiose glucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00158:0:0:\"CELB\";2:cpd00190:0:0:\"beta-D-Glucose\"'},\n",
       " {'abbreviation': 'R00027',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00027',\n",
       "   'Name: 2-Hydroxy-1,2-diphenylethanone benzaldehyde-lyase; 2-hydroxy-1,2-diphenylethanone benzaldehyde-lyase (benzaldehyde-forming)'],\n",
       "  'code': '(1) cpd01008[0] <=> (2) cpd00225[0]',\n",
       "  'compound_ids': 'cpd00225;cpd01008',\n",
       "  'definition': '(1) Benzoin[0] <=> (2) Benzaldehyde[0]',\n",
       "  'deltag': 2.81,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.38'],\n",
       "  'equation': '(1) cpd01008[0] <=> (2) cpd00225[0]',\n",
       "  'id': 'rxn00021',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-hydroxy-1,2-diphenylethanone benzaldehyde-lyase (benzaldehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd01008:0:0:\"Benzoin\";2:cpd00225:0:0:\"Benzaldehyde\"'},\n",
       " {'abbreviation': 'R00028',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00028_c',\n",
       "   'BiGG: FSP2_1; MAL12; MAL32; MALT; YGR287C; YIL172C; YJL216C',\n",
       "   'KEGG: R00028; R06084',\n",
       "   'MetaCyc: RXN-15910',\n",
       "   'iAG612: rbs_447; rbs_448; rbs_449',\n",
       "   'Name: Probable alpha-glucosidase FSP2; alpha-D-Glucoside glucohydrolase; alpha-glucosidase; maltose glucohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00179[0] <=> (2) cpd00027[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00179',\n",
       "  'definition': '(1) H2O[0] + (1) Maltose[0] => (2) D-Glucose[0]',\n",
       "  'deltag': -3.48,\n",
       "  'deltagerr': 0.38,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.20'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00179[0] => (2) cpd00027[0]',\n",
       "  'id': 'rxn00022',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30130',\n",
       "  'name': 'maltose glucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00179:0:0:\"Maltose\";2:cpd00027:0:0:\"D-Glucose\"'},\n",
       " {'abbreviation': 'R00029',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00029',\n",
       "   'Name: Thiosulfate:ferricytochrome-c oxidoreductase; thiosulfate:ferricytochrome-c oxidoreductase'],\n",
       "  'code': '(2) cpd00109[0] + (2) cpd00268[0] <=> (2) cpd00110[0] + (1) cpd01414[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00109;cpd00110;cpd00268;cpd01414',\n",
       "  'definition': '(2) Cytochrome c3+[0] + (2) H2S2O3[0] <=> (2) H+[0] + (2) Cytochrome c2+[0] + (1) Tetrathionate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.2.2'],\n",
       "  'equation': '(2) cpd00109[0] + (2) cpd00268[0] <=> (2) cpd00067[0] + (2) cpd00110[0] + (1) cpd01414[0]',\n",
       "  'id': 'rxn00023',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'thiosulfate:ferricytochrome-c oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00109:0:0:\"Cytochrome c3+\";-2:cpd00268:0:0:\"H2S2O3\";2:cpd00067:0:0:\"H+\";2:cpd00110:0:0:\"Cytochrome c2+\";1:cpd01414:0:0:\"Tetrathionate\"'},\n",
       " {'abbreviation': 'R00031',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00031',\n",
       "   'MetaCyc: RXN-5861',\n",
       "   'iRS1563: R_R00031_c',\n",
       "   'Name: 1,2-Benzenediol:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00069[0] <=> (2) cpd00291[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00069;cpd00291',\n",
       "  'definition': '(1) O2[0] + (2) L-Tyrosine[0] => (2) L-Dopa[0]',\n",
       "  'deltag': -125.98,\n",
       "  'deltagerr': 2.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.10.3.1'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00069[0] => (2) cpd00291[0]',\n",
       "  'id': 'rxn00024',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '1,2-Benzenediol:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); Antibiotic-Biosynthesis (Antibiotic Biosynthesis); BETALAIN-ALKALOIDS (Betalaine Alkaloid Biosynthesis); ISOQUINOLINE-ALKALOIDS (Isoquinoline and Benzylisoquinoline Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-3581 ((<i>S</i>)-reticuline biosynthesis I); PWY-5399 (betacyanin biosynthesis); PWY-5405 (superpathway of betalain biosynthesis); PWY-6133 ((S)-reticuline biosynthesis II); PWY-6955 (lincomycin biosynthesis); S-Reticuline-Biosynthesis ((<i>S</i>)-reticuline biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism); rn00950 (Isoquinoline alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00069:0:0:\"L-Tyrosine\";2:cpd00291:0:0:\"L-Dopa\"'},\n",
       " {'abbreviation': 'R00032',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.14.99.36-RXN; BETA-CAROTENE-1515-DIOXYGENASE-RXN',\n",
       "   'BrachyCyc: BETA-CAROTENE-1515-DIOXYGENASE-RXN',\n",
       "   'CornCyc: 1.14.99.36-RXN',\n",
       "   'KEGG: R00032',\n",
       "   'MetaCyc: 1.14.99.36-RXN; RXN-13374',\n",
       "   \"Name: beta-Carotene:oxygen 15,15'-oxidoreductase (bond-cleaving); beta-carotene 15,15'-dioxygenase, carotene dioxygenase; beta-carotene 15,15'-monooxygenase; beta-carotene:oxygen 15,15'-oxidoreductase (bond-cleaving); carotene 15,15'-dioxygenase\"],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd01420[0] <=> (2) cpd00304[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00304;cpd01420',\n",
       "  'definition': '(1) O2[0] + (1) beta-Carotene[0] => (2) Retinal[0]',\n",
       "  'deltag': -86.46,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.63', '1.14.99.36'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd01420[0] => (2) cpd00304[0]',\n",
       "  'id': 'rxn00025',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35456',\n",
       "  'name': \"beta-Carotene:oxygen 15,15'-oxidoreductase (bond-cleaving)\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-6857 (retinol biosynthesis); Vitamin-A-Biosynthesis (Vitamin A Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00830 (Retinol metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd01420:0:0:\"beta-Carotene\";2:cpd00304:0:0:\"Retinal\"'},\n",
       " {'abbreviation': 'R00033',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00033', 'Name: thiamine:oxygen 5-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00305[0] <=> (1) cpd00025[0] + (1) cpd03484[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00305;cpd03484',\n",
       "  'definition': '(1) O2[0] + (1) Thiamin[0] => (1) H2O2[0] + (1) Thiamine aldehyde[0]',\n",
       "  'deltag': -26.84,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.23'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00305[0] => (1) cpd00025[0] + (1) cpd03484[0]',\n",
       "  'id': 'rxn00026',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'thiamine:oxygen 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00305:0:0:\"Thiamin\";1:cpd00025:0:0:\"H2O2\";1:cpd03484:0:0:\"Thiamine aldehyde\"'},\n",
       " {'abbreviation': 'R00034',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00034',\n",
       "   'Name: Cyclobutadipyrimidine (in DNA) pyrimidine-lyase'],\n",
       "  'code': '(1) cpd01990[0] <=> (2) cpd05167[0]',\n",
       "  'compound_ids': 'cpd00067;cpd01990;cpd05167',\n",
       "  'definition': \"(1) Cyclobutadipyrimidine[0] => (8) H+[0] + (2) Pyrimidine 5'-deoxynucleotide[0]\",\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.99.3'],\n",
       "  'equation': '(1) cpd01990[0] => (8) cpd00067[0] + (2) cpd05167[0]',\n",
       "  'id': 'rxn00027',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Cyclobutadipyrimidine (in DNA) pyrimidine-lyase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'CI:8',\n",
       "  'stoichiometry': '-1:cpd01990:0:0:\"Cyclobutadipyrimidine\";8:cpd00067:0:0:\"H+\";2:cpd05167:0:0:\"Pyrimidine 5\\'-deoxynucleotide\"'},\n",
       " {'abbreviation': 'R00068',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: L-ASCORBATE-OXIDASE-RXN',\n",
       "   'ChlamyCyc: L-ASCORBATE-OXIDASE-RXN',\n",
       "   'KEGG: R00068',\n",
       "   'MaizeCyc: L-ASCORBATE-OXIDASE-RXN',\n",
       "   'PoplarCyc: L-ASCORBATE-OXIDASE-RXN',\n",
       "   'Name: AA oxidase; AAO; L-Ascorbate:oxygen oxidoreductase; L-ascorbate oxidase; L-ascorbate:O2 oxidoreductase; L-ascorbate:oxygen oxidoreductase; L-ascorbic acid oxidase; ascorbase; ascorbate dehydrogenase; ascorbate oxidase; ascorbic acid oxidase; ascorbic oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00059[0] <=> (2) cpd00001[0] + (2) cpd00335[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00059;cpd00067;cpd00335',\n",
       "  'definition': '(1) O2[0] + (2) L-Ascorbate[0] + (2) H+[0] => (2) H2O[0] + (2) Dehydroascorbate[0]',\n",
       "  'deltag': -103.9,\n",
       "  'deltagerr': 3.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.10.3.3'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00059[0] + (2) cpd00067[0] => (2) cpd00001[0] + (2) cpd00335[0]',\n",
       "  'id': 'rxn00028',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33449;rxn33465;rxn38163',\n",
       "  'name': 'L-ascorbate:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00053 (Ascorbate and aldarate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00059:0:0:\"L-Ascorbate\";-2:cpd00067:0:0:H+;2:cpd00001:0:0:\"H2O\";2:cpd00335:0:0:\"Dehydroascorbate\"'},\n",
       " {'abbreviation': 'R00036',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PORPHOBILSYNTH-RXN',\n",
       "   'BiGG: HEM2; PPBNGS',\n",
       "   'BrachyCyc: PORPHOBILSYNTH-RXN',\n",
       "   'KEGG: R00036',\n",
       "   'MetaCyc: PORPHOBILSYNTH-RXN',\n",
       "   'Name: 5-Aminolevulinate hydro-lyase(adding 5-aminolevulinate and; 5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing); 5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing, porphobilinogen-forming); 5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing; porphobilinogen-forming); 5-levulinic acid dehydratase; Porphobilinogen synthase; aminolevulinate dehydratase; aminolevulinic dehydratase; cyclizing); delta-aminolevulinate dehydratase; delta-aminolevulinic acid dehydrase; delta-aminolevulinic acid dehydratase; delta-aminolevulinic dehydratase; porphobilinogen synthase; porphobilinogensynthase'],\n",
       "  'code': '(2) cpd00338[0] <=> (2) cpd00001[0] + (1) cpd00689[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00338;cpd00689',\n",
       "  'definition': '(2) 5-Aminolevulinate[0] => (2) H2O[0] + (1) H+[0] + (1) Porphobilinogen[0]',\n",
       "  'deltag': -24.58,\n",
       "  'deltagerr': 0.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.24'],\n",
       "  'equation': '(2) cpd00338[0] => (2) cpd00001[0] + (1) cpd00067[0] + (1) cpd00689[0]',\n",
       "  'id': 'rxn00029',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28081',\n",
       "  'name': '5-aminolevulinate hydro-lyase (adding 5-aminolevulinate and cyclizing; porphobilinogen-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenosylcobamide-Biosynthesis (Adenosylcobamide Biosynthesis); Chlorophyll-Biosynthesis (Chlorophyll Biosynthesis); Chlorophyll-a-Biosynthesis (Chlorophyll <i>a</i> Biosynthesis); Cobalamin-Biosynthesis (Cobamide Biosynthesis); Cobamide-De-Novo-Biosynthesis (Cobamide <i>de novo</i> Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); De-Novo-Adenosylcobalamin-Biosynthesis (Adenosylcobalamin <i>de novo</i> Biosynthesis); HEME-SYN (Heme Biosynthesis); Heme-b-Biosynthesis (Heme <i>b</i> Biosynthesis); P381-PWY (adenosylcobalamin biosynthesis II (aerobic)); PWY-5188 (tetrapyrrole biosynthesis I (from glutamate)); PWY-5189 (tetrapyrrole biosynthesis II (from glycine)); PWY-5507 (adenosylcobalamin biosynthesis I (anaerobic)); PWY-5529 (superpathway of bacteriochlorophyll <i>a</i> biosynthesis); PWY-5918 (superpathay of heme <i>b</i> biosynthesis from glutamate); PWY-5920 (superpathway of <i>b</i> heme biosynthesis from glycine); Porphyrin-Compounds-Biosynthesis (Porphyrin Compound Biosynthesis); Tetrapyrrole-Biosynthesis (Tetrapyrrole Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00860 (Porphyrin and chlorophyll metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00338:0:0:\"5-Aminolevulinate\";2:cpd00001:0:0:\"H2O\";1:cpd00067:0:0:\"H+\";1:cpd00689:0:0:\"Porphobilinogen\"'},\n",
       " {'abbreviation': 'R00038',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00038',\n",
       "   'MetaCyc: PROPIOIN-SYNTHASE-RXN',\n",
       "   'Name: 4-Hydroxy-3-hexanone propanal-lyase; 4-hydroxy-3-hexanone aldolase; 4-hydroxy-3-hexanone propanal-lyase; 4-hydroxy-3-hexanone propanal-lyase (propanal-forming); propioin synthase'],\n",
       "  'code': '(2) cpd00371[0] <=> (1) cpd01891[0]',\n",
       "  'compound_ids': 'cpd00371;cpd01891',\n",
       "  'definition': '(2) Propanal[0] <=> (1) Propioin[0]',\n",
       "  'deltag': -2.83,\n",
       "  'deltagerr': 0.26,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.35'],\n",
       "  'equation': '(2) cpd00371[0] <=> (1) cpd01891[0]',\n",
       "  'id': 'rxn00030',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxy-3-hexanone propanal-lyase (propanal-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00371:0:0:\"Propanal\";1:cpd01891:0:0:\"Propioin\"'},\n",
       " {'abbreviation': 'R00039',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00039',\n",
       "   'MetaCyc: 2.4.1.166-RXN',\n",
       "   'Name: Raffinose:raffinose alpha-D-galactosyltransferase; raffinose (raffinose donor) galactosyltransferase; raffinose-raffinose alpha-galactosyltransferase; raffinose-raffinose alpha-galactotransferase; raffinose:raffinose 1F-alpha-D-galactosyltransferase; raffinose:raffinose alpha-D-galactosyltransferase; raffinose:raffinose alpha-galactosyltransferase'],\n",
       "  'code': '(2) cpd00382[0] <=> (1) cpd00076[0] + (1) cpd02464[0]',\n",
       "  'compound_ids': 'cpd00076;cpd00382;cpd02464',\n",
       "  'definition': '(2) Melitose[0] <=> (1) Sucrose[0] + (1) 1F-alpha-D-Galactosylraffinose[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 10.63,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.166'],\n",
       "  'equation': '(2) cpd00382[0] <=> (1) cpd00076[0] + (1) cpd02464[0]',\n",
       "  'id': 'rxn00031',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'raffinose:raffinose 1F-alpha-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00382:0:0:\"Melitose\";1:cpd00076:0:0:\"Sucrose\";1:cpd02464:0:0:\"1F-alpha-D-Galactosylraffinose\"'},\n",
       " {'abbreviation': 'Choline:oxygen 1-oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Choline:oxygen 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00098[0] <=> (2) cpd00001[0] + (2) cpd00447[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00098;cpd00447',\n",
       "  'definition': '(1) O2[0] + (2) Choline[0] <=> (2) H2O[0] + (2) Betaine aldehyde[0]',\n",
       "  'deltag': -101.23,\n",
       "  'deltagerr': 0.92,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.3.17'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00098[0] <=> (2) cpd00001[0] + (2) cpd00447[0]',\n",
       "  'id': 'rxn00032',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Choline:oxygen 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00098:0:0:\"Choline\";2:cpd00001:0:0:\"H2O\";2:cpd00447:0:0:\"Betaine aldehyde\"'},\n",
       " {'abbreviation': 'R00042',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00042',\n",
       "   'MetaCyc: 1.2.3.13-RXN',\n",
       "   'Name: 4-Hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating); 4-hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating)'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00868[0] <=> (2) cpd00011[0] + (2) cpd00489[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00489;cpd00868',\n",
       "  'definition': '(1) O2[0] + (2) p-hydroxyphenylpyruvate[0] => (2) CO2[0] + (2) 4-Hydroxyphenylacetate[0]',\n",
       "  'deltag': -159.92,\n",
       "  'deltagerr': 2.75,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.3.13'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00868[0] => (2) cpd00011[0] + (2) cpd00489[0]',\n",
       "  'id': 'rxn00033',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxyphenylpyruvate:oxygen oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5151 (L-tyrosine degradation II); PWY-7514 (L-tyrosine degradation IV (to 4-methylphenol)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TYROSINE-DEG (L-tyrosine Degradation)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00868:0:0:\"p-hydroxyphenylpyruvate\";2:cpd00011:0:0:\"CO2\";2:cpd00489:0:0:\"4-Hydroxyphenylacetate\"'},\n",
       " {'abbreviation': 'R00043',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00043',\n",
       "   'MetaCyc: LIGNOSTILBENE-ALPHA-BETA-DIOXYGENASE-RXN',\n",
       "   'Name: (alpha-beta-bond-cleaving); 1,2-Bis(4-hydroxy-3-methoxyphenyl)ethylene:oxygen oxidoreductase; 1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene:oxygen oxidoreductase (alpha-beta-bond-cleaving); 1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene:oxygen oxidoreductase (alphabeta-bond-cleaving); lignostilbene alphabeta-dioxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd02768[0] <=> (2) cpd00561[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00561;cpd02768',\n",
       "  'definition': '(1) O2[0] + (1) 1,2-Bis(4-hydroxy-3-methoxyphenyl)ethylene[0] => (2) Vanillin[0]',\n",
       "  'deltag': -87.92,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.43'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd02768[0] => (2) cpd00561[0]',\n",
       "  'id': 'rxn00034',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '1,2-bis(4-hydroxy-3-methoxyphenyl)ethylene:oxygen oxidoreductase (alpha-beta-bond-cleaving)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd02768:0:0:\"1,2-Bis(4-hydroxy-3-methoxyphenyl)ethylene\";2:cpd00561:0:0:\"Vanillin\"'},\n",
       " {'abbreviation': 'R00044',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00044',\n",
       "   'MetaCyc: COLUMBAMINE-OXIDASE-RXN',\n",
       "   'Name: Columbamine:oxygen oxidoreductase (cyclizing); berberine synthase; columbamine oxidase; columbamine:oxygen oxidoreductase (cyclizing)'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd01238[0] <=> (2) cpd00001[0] + (2) cpd00563[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00563;cpd01238',\n",
       "  'definition': '(1) O2[0] + (2) Columbamine[0] => (2) H2O[0] + (2) Berberine[0]',\n",
       "  'deltag': -119.25,\n",
       "  'deltagerr': 1.85,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.21.3.2'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd01238[0] => (2) cpd00001[0] + (2) cpd00563[0]',\n",
       "  'id': 'rxn00035',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'columbamine:oxygen oxidoreductase (cyclizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00950 (Isoquinoline alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd01238:0:0:\"Columbamine\";2:cpd00001:0:0:\"H2O\";2:cpd00563:0:0:\"Berberine\"'},\n",
       " {'abbreviation': 'R00045',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00045',\n",
       "   'MetaCyc: RXN-13061',\n",
       "   'iRS1563: R_R00045_c',\n",
       "   'Name: 3,4-dihydroxy-L-phenylalanine:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00291[0] <=> (2) cpd00001[0] + (2) cpd00612[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00291;cpd00612',\n",
       "  'definition': '(1) O2[0] + (2) L-Dopa[0] => (2) H2O[0] + (2) Dopaquinone[0]',\n",
       "  'deltag': -82.41,\n",
       "  'deltagerr': 3.41,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.10.3.1'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00291[0] => (2) cpd00001[0] + (2) cpd00612[0]',\n",
       "  'id': 'rxn00036',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3,4-dihydroxy-L-phenylalanine:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); BETALAIN-ALKALOIDS (Betalaine Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-5399 (betacyanin biosynthesis); PWY-5405 (superpathway of betalain biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00291:0:0:\"L-Dopa\";2:cpd00001:0:0:\"H2O\";2:cpd00612:0:0:\"Dopaquinone\"'},\n",
       " {'abbreviation': 'R_R00047_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00047_c',\n",
       "   'AraGEM: R_R00047_c',\n",
       "   'Maize_C4GEM: R_R00047_c',\n",
       "   'iRS1563: R_R00047_c',\n",
       "   'iRS1597: R00047_c',\n",
       "   'Name: (5-hydroxylating); 6-Hydroxynicotinate, hydrogen-donor:oxygen oxidoreductase; omega-3 fatty acid desaturase(FAD7) (FADD)'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00752[0] <=> (2) cpd00011[0] + (2) cpd00778[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00067;cpd00752;cpd00778',\n",
       "  'definition': '(1) O2[0] + (2) H+[0] + (2) 6-Hydroxynicotinate[0] => (2) CO2[0] + (2) 2,5-Dihydroxypyridine[0]',\n",
       "  'deltag': -107.75,\n",
       "  'deltagerr': 2.01,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.99.-'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00067[0] + (2) cpd00752[0] => (2) cpd00011[0] + (2) cpd00778[0]',\n",
       "  'id': 'rxn00037',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'omega-3 fatty acid desaturase, chloroplast (FAD7) (FADD)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-2:cpd00752:0:0:\"6-Hydroxynicotinate\";2:cpd00011:0:0:\"CO2\";2:cpd00778:0:0:\"2,5-Dihydroxypyridine\"'},\n",
       " {'abbreviation': 'R00048',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00048',\n",
       "   'MetaCyc: HYDROXYBUTYRATE-DIMER-HYDROLASE-RXN',\n",
       "   'Name: (R)-3-((R)-3-Hydroxybutanoyloxy)-butanoate hydroxybutanoylhydrolase; (R)-3-((R)-3-hydroxybutanoyloxy)butanoate hydroxybutanoylhydrolase; D-(-)-3-hydroxybutyrate-dimer hydrolase; hydroxybutyrate-dimer hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02767[0] <=> (2) cpd00797[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00797;cpd02767',\n",
       "  'definition': '(1) H2O[0] + (1) (R)-3-((R)-3-Hydroxybutanoyloxy)butanoate[0] <=> (1) H+[0] + (2) (R)-3-Hydroxybutanoate[0]',\n",
       "  'deltag': -11.98,\n",
       "  'deltagerr': 1.19,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02767[0] <=> (1) cpd00067[0] + (2) cpd00797[0]',\n",
       "  'id': 'rxn00038',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(R)-3-((R)-3-Hydroxybutanoyloxy)-butanoate hydroxybutanoylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02767:0:0:\"(R)-3-((R)-3-Hydroxybutanoyloxy)butanoate\";1:cpd00067:0:0:\"H+\";2:cpd00797:0:0:\"(R)-3-Hydroxybutanoate\"'},\n",
       " {'abbreviation': 'R00049',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00049',\n",
       "   'MetaCyc: 2.3.1.90-RXN',\n",
       "   'Name: 1-O-Galloyl-beta-D-glucose:1-O-galloyl-beta-D-glucose; 1-O-galloyl-beta-D-glucose:1-O-galloyl-beta-D-glucose O-galloyltransferase; O-galloyltransferase'],\n",
       "  'code': '(2) cpd00852[0] <=> (1) cpd00027[0] + (1) cpd02532[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00852;cpd02532',\n",
       "  'definition': '(2) beta-Glucogallin[0] <=> (1) D-Glucose[0] + (1) 1-O,6-O-Digalloyl-beta-D-glucose[0]',\n",
       "  'deltag': -2.55,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.90'],\n",
       "  'equation': '(2) cpd00852[0] <=> (1) cpd00027[0] + (1) cpd02532[0]',\n",
       "  'id': 'rxn00039',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '1-O-galloyl-beta-D-glucose:1-O-galloyl-beta-D-glucose O-galloyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PWY-5475 (pentagalloylglucose biosynthesis); PWY-5478 (superpathway of hydrolyzable tannin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TANNIN-SYN (Hydrolyzable Tannin Biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00852:0:0:\"beta-Glucogallin\";1:cpd00027:0:0:\"D-Glucose\";1:cpd02532:0:0:\"1-O,6-O-Digalloyl-beta-D-glucose\"'},\n",
       " {'abbreviation': 'R00053',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00053',\n",
       "   'MetaCyc: TANNASE-RXN',\n",
       "   'Name: Tannin acylhydrolase; digallate acylhydrolase; tannase; tannase S; tannin acetylhydrolase; tannin acylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01108[0] <=> (2) cpd01020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd01020;cpd01108',\n",
       "  'definition': '(1) H2O[0] + (1) Digallate[0] <=> (1) H+[0] + (2) Gallate[0]',\n",
       "  'deltag': -13.77,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.20'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01108[0] <=> (1) cpd00067[0] + (2) cpd01020[0]',\n",
       "  'id': 'rxn00040',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'digallate acylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01108:0:0:\"Digallate\";1:cpd00067:0:0:\"H+\";2:cpd01020:0:0:\"Gallate\"'},\n",
       " {'abbreviation': 'R00054',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00054',\n",
       "   'MetaCyc: ORSELLINATE-DEPSIDE-HYDROLASE-RXN',\n",
       "   'Name: Orsellinate-depside hydrolase; lecanorate hydrolase; orsellinate-depside hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01843[0] <=> (2) cpd01265[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd01265;cpd01843',\n",
       "  'definition': '(1) H2O[0] + (1) Lecanoric acid[0] <=> (1) H+[0] + (2) Orsellinate[0]',\n",
       "  'deltag': -13.87,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.40'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01843[0] <=> (1) cpd00067[0] + (2) cpd01265[0]',\n",
       "  'id': 'rxn00041',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'orsellinate-depside hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01843:0:0:\"Lecanoric acid\";1:cpd00067:0:0:\"H+\";2:cpd01265:0:0:\"Orsellinate\"'},\n",
       " {'abbreviation': 'R00055',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00055',\n",
       "   'MetaCyc: RXN-7802',\n",
       "   \"Name: Nucleoside:oxygen 5'-oxidoreductase; inosine:oxygen 5'-oxidoreductase; nucleoside oxidase; nucleoside:oxygen 5'-oxidoreductase\"],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00246[0] <=> (2) cpd00001[0] + (2) cpd01365[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00246;cpd01365',\n",
       "  'definition': \"(1) O2[0] + (2) Inosine[0] => (2) H2O[0] + (2) 5'-Oxoinosine[0]\",\n",
       "  'deltag': -101.47,\n",
       "  'deltagerr': 1.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.28'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00246[0] => (2) cpd00001[0] + (2) cpd01365[0]',\n",
       "  'id': 'rxn00042',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"inosine:oxygen 5'-oxidoreductase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00246:0:0:\"Inosine\";2:cpd00001:0:0:\"H2O\";2:cpd01365:0:0:\"5\\'-Oxoinosine\"'},\n",
       " {'abbreviation': 'R00058',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: CATECHOL-OXIDASE-RXN',\n",
       "   'CornCyc: CATECHOL-OXIDASE-RXN',\n",
       "   'KEGG: R00058',\n",
       "   'MaizeCyc: CATECHOL-OXIDASE-RXN',\n",
       "   'MetaCyc: CATECHOL-OXIDASE-RXN',\n",
       "   'Name: 1,2-benzenediol:oxygen oxidoreductase; Dopa oxidase; catechol oxidase; catechol:oxygen oxidoreductase; catecholase; diphenol oxidase; o-diphenol oxidoreductase; o-diphenol:oxygen oxidoreductase; o-diphenolase; phenolase; polyphenol oxidase; pyrocatechol oxidase; tyrosinase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00077[0] <=> (2) cpd00001[0] + (2) cpd01569[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00077;cpd01569',\n",
       "  'definition': '(1) O2[0] + (2) Catechol[0] => (2) H2O[0] + (2) 1,2-Benzoquinone[0]',\n",
       "  'deltag': -71.27,\n",
       "  'deltagerr': 4.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.10.3.1', '1.14.18.1'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00077[0] => (2) cpd00001[0] + (2) cpd01569[0]',\n",
       "  'id': 'rxn00043',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'catechol:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); PWY-6752 (<i>o</i>-diquinones biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00077:0:0:\"Catechol\";2:cpd00001:0:0:\"H2O\";2:cpd01569:0:0:\"1,2-Benzoquinone\"'},\n",
       " {'abbreviation': 'R00059',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AHXDH',\n",
       "   'KEGG: R00059',\n",
       "   'MetaCyc: RXN-3962',\n",
       "   'iAF692: AHXDH',\n",
       "   'Name: 6-aminohexanoate-dimer hydrolase; 6-aminohexanoic acid oligomer hydrolase; N-(6-Aminohexanoyl)-6-aminohexanoate amidohydrolase; N-(6-aminohexanoyl)-6-aminohexanoate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00921[0] <=> (2) cpd01588[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00921;cpd01588',\n",
       "  'definition': '(1) H2O[0] + (1) N-(6-Aminohexanoyl)-6-aminohexanoate[0] <=> (2) Epsilcapramine[0]',\n",
       "  'deltag': 1.15,\n",
       "  'deltagerr': 0.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.46'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00921[0] <=> (2) cpd01588[0]',\n",
       "  'id': 'rxn00044',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-(6-aminohexanoyl)-6-aminohexanoate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Other-Degradation (Degradation/Utilization/Assimilation-Other); P621-PWY (nylon-6 oligomer degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00921:0:0:\"N-(6-Aminohexanoyl)-6-aminohexanoate\";2:cpd01588:0:0:\"Epsilcapramine\"'},\n",
       " {'abbreviation': 'R00062',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00062',\n",
       "   'MetaCyc: 2.4.1.95-RXN',\n",
       "   'Name: Bilirubin-glucuronoside:bilirubin-glucuronoside; D-glucuronosyltransferase; bilirubin glucuronoside glucuronosyltransferase; bilirubin monoglucuronide transglucuronidase; bilirubin-glucuronoside glucuronosyltransferase; bilirubin-glucuronoside:bilirubin-glucuronoside D-glucuronosyltransferase'],\n",
       "  'code': '(2) cpd02141[0] <=> (1) cpd00376[0] + (1) cpd03428[0]',\n",
       "  'compound_ids': 'cpd00376;cpd02141;cpd03428',\n",
       "  'definition': '(2) Bilirubin-glucuronoside[0] <=> (1) Bilirubin[0] + (1) Bilirubin-bisglucuronoside[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 18.06,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.95'],\n",
       "  'equation': '(2) cpd02141[0] <=> (1) cpd00376[0] + (1) cpd03428[0]',\n",
       "  'id': 'rxn00045',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'bilirubin-glucuronoside:bilirubin-glucuronoside D-glucuronosyltransferase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd02141:0:0:\"Bilirubin-glucuronoside\";1:cpd00376:0:0:\"Bilirubin\";1:cpd03428:0:0:\"Bilirubin-bisglucuronoside\"'},\n",
       " {'abbreviation': 'R00063',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00063',\n",
       "   'MetaCyc: 2.3.1.103-RXN',\n",
       "   'Name: 1-O-(4-Hydroxy-3,5-dimethoxycinnamoyl)-beta-D-glucoside:1-O-(4-; 1-O-(4-hydroxy-3,5-dimethoxycinnamoyl)-beta-D-glucoside:1-O-(4-hydroxy-3,5-dimethoxycinnamoyl-beta-D-glucoside 1-O-sinapoyltransferase; 1-O-sinapoyl-beta-D-glucose:1-O-sinapoyl-beta-D-glucose 1-O-sinapoyltransferase; hydroxy-3,5-dimethoxycinnamoyl)-beta-D-glucoside'],\n",
       "  'code': '(2) cpd00865[0] <=> (1) cpd00027[0] + (1) cpd02622[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00865;cpd02622',\n",
       "  'definition': '(2) 1-O-Sinapoyl-beta-D-glucose[0] <=> (1) D-Glucose[0] + (1) 1,2-Bis-O-sinapoyl-beta-D-glucoside[0]',\n",
       "  'deltag': -1.82,\n",
       "  'deltagerr': 0.96,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.103'],\n",
       "  'equation': '(2) cpd00865[0] <=> (1) cpd00027[0] + (1) cpd02622[0]',\n",
       "  'id': 'rxn00046',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '1-O-sinapoyl-beta-D-glucose:1-O-sinapoyl-beta-D-glucose 1-O-sinapoyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CINNAMATE-SYN (Cinnamate Biosynthesis); PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PWY-3301 (sinapate ester biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00865:0:0:\"1-O-Sinapoyl-beta-D-glucose\";1:cpd00027:0:0:\"D-Glucose\";1:cpd02622:0:0:\"1,2-Bis-O-sinapoyl-beta-D-glucoside\"'},\n",
       " {'abbreviation': 'R_R00064_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00064',\n",
       "   'iRS1563: R_R00064_c',\n",
       "   'Name: 4-(4-Deoxy-beta-D-gluc-4-enuronosyl)-D-galacturonate lyase; R_4_4_Deoxy_beta_D_gluc_4_enuronosyl__D_galacturonate_lyase'],\n",
       "  'code': '(1) cpd02883[0] <=> (2) cpd02508[0]',\n",
       "  'compound_ids': 'cpd02508;cpd02883',\n",
       "  'definition': '(1) 4-(4-Deoxy-beta-D-gluc-4-enuronosyl)-D-galacturonate[0] => (2) 5-Dehydro-4-deoxy-D-glucuronate[0]',\n",
       "  'deltag': -14.55,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.2.6'],\n",
       "  'equation': '(1) cpd02883[0] => (2) cpd02508[0]',\n",
       "  'id': 'rxn00047',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R_4__4_Deoxy_beta_D_gluc_4_enuronosyl__D_galacturonate_lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02883:0:0:\"4-(4-Deoxy-beta-D-gluc-4-enuronosyl)-D-galacturonate\";2:cpd02508:0:0:\"5-Dehydro-4-deoxy-D-glucuronate\"'},\n",
       " {'abbreviation': 'R00066',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RIBOFLAVIN-SYN-RXN',\n",
       "   'BrachyCyc: RIBOFLAVIN-SYN-RXN',\n",
       "   'ChlamyCyc: RIBOFLAVIN-SYN-RXN',\n",
       "   'KEGG: R00066',\n",
       "   'MetaCyc: RIBOFLAVIN-SYN-RXN',\n",
       "   'Name: 6,7-Dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl); 6,7-Dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl)lumazine 2,3-butanediyltransferase; 6,7-dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl)lumazine 2,3-butanediyltransferase; RBFSb; heavy riboflavin synthase; light riboflavin synthase; lumazine 2,3-butanediyltransferase; riboflavin synthase; riboflavin synthase beta chain; riboflavin synthetase; riboflavine synthase; riboflavine synthetase; riboflavinsynthase'],\n",
       "  'code': '(2) cpd02656[0] <=> (1) cpd00220[0] + (1) cpd02882[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00220;cpd02656;cpd02882',\n",
       "  'definition': '(1) H+[0] + (2) 6-7-Dimethyl-8--1-D-ribityllumazine[0] => (1) Riboflavin[0] + (1) 4--1-D-Ribitylamino-5-aminouracil[0]',\n",
       "  'deltag': -22.96,\n",
       "  'deltagerr': 1.98,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.5.1.9'],\n",
       "  'equation': '(1) cpd00067[0] + (2) cpd02656[0] => (1) cpd00220[0] + (1) cpd02882[0]',\n",
       "  'id': 'rxn00048',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32591',\n",
       "  'name': '6,7-Dimethyl-8-(1-D-ribityl)lumazine:6,7-dimethyl-8-(1-D-ribityl)lumazine 2,3-butanediyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Flavin-Biosynthesis (Flavin Biosynthesis); PWY-6167 (flavin biosynthesis II (archaea)); PWY-6168 (flavin biosynthesis III (fungi)); RIBOSYN2-PWY (flavin biosynthesis I (bacteria and plants)); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00740 (Riboflavin metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:H+;-2:cpd02656:0:0:\"6-7-Dimethyl-8--1-D-ribityllumazine\";1:cpd00220:0:0:\"Riboflavin\";1:cpd02882:0:0:\"4--1-D-Ribitylamino-5-aminouracil\"'},\n",
       " {'abbreviation': 'rxn00050',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(2) cpd03299[0] <=> (1) cpd00007[0] + (2) cpd02311[0]',\n",
       "  'compound_ids': 'cpd00007;cpd02311;cpd03299',\n",
       "  'definition': '(2) Hydantoin-propionate[0] <=> (1) O2[0] + (2) 4-Imidazolone-5-propanoate[0]',\n",
       "  'deltag': 134.22,\n",
       "  'deltagerr': 3.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.13.-'],\n",
       "  'equation': '(2) cpd03299[0] <=> (1) cpd00007[0] + (2) cpd02311[0]',\n",
       "  'id': 'rxn00050',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00050',\n",
       "  'notes': ['GCC', 'HB', 'EQU', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd03299:0:0:\"Hydantoin-propionate\";1:cpd00007:0:0:\"O2\";2:cpd02311:0:0:\"4-Imidazolone-5-propanoate\"'},\n",
       " {'abbreviation': 'R00070',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00070',\n",
       "   'MetaCyc: RXN-16676',\n",
       "   'Name: D-urobilinogen:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd03431[0] <=> (2) cpd00001[0] + (2) cpd03434[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd03431;cpd03434',\n",
       "  'definition': '(1) O2[0] + (2) D-Urobilinogen[0] => (2) H2O[0] + (2) D-Urobilin[0]',\n",
       "  'deltag': -118.04,\n",
       "  'deltagerr': 4.77,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd03431[0] => (2) cpd00001[0] + (2) cpd03434[0]',\n",
       "  'id': 'rxn00051',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-urobilinogen:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00860 (Porphyrin and chlorophyll metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd03431:0:0:\"D-Urobilinogen\";2:cpd00001:0:0:\"H2O\";2:cpd03434:0:0:\"D-Urobilin\"'},\n",
       " {'abbreviation': 'R00072',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00072',\n",
       "   'Name: thiamine-aldehyde:oxygen 5-oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd03484[0] <=> (1) cpd00025[0] + (1) cpd01852[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00067;cpd01852;cpd03484',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Thiamine aldehyde[0] => (1) H2O2[0] + (1) H+[0] + (1) Thiamin acetic acid[0]',\n",
       "  'deltag': -40.01,\n",
       "  'deltagerr': 1.05,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.23'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd03484[0] => (1) cpd00025[0] + (1) cpd00067[0] + (1) cpd01852[0]',\n",
       "  'id': 'rxn00052',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'thiamine-aldehyde:oxygen 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd03484:0:0:\"Thiamine aldehyde\";1:cpd00025:0:0:\"H2O2\";1:cpd00067:0:0:\"H+\";1:cpd01852:0:0:\"Thiamin acetic acid\"'},\n",
       " {'abbreviation': 'Prostaglandin H2:oxygen oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Prostaglandin H2:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00336[0] <=> (2) cpd03545[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00336;cpd03545',\n",
       "  'definition': '(1) O2[0] + (2) Prostaglandin H2[0] <=> (2) PGG2[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.99.1'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00336[0] <=> (2) cpd03545[0]',\n",
       "  'id': 'rxn00053',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Prostaglandin H2:oxygen oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00336:0:0:\"Prostaglandin H2\";2:cpd03545:0:0:\"PGG2\"'},\n",
       " {'abbreviation': 'R00074',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00074',\n",
       "   'MetaCyc: O-AMINOPHENOL-OXIDASE-RXN',\n",
       "   'Name: 2-Aminophenol:oxygen oxidoreductase; 2-aminophenol:oxygen oxidoreductase (dimerizing)'],\n",
       "  'code': '(3) cpd00007[0] + (4) cpd01360[0] <=> (6) cpd00001[0] + (2) cpd01462[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd01360;cpd01462',\n",
       "  'definition': '(3) O2[0] + (4) o-Aminophenol[0] => (6) H2O[0] + (2) Isophenoxazine[0]',\n",
       "  'deltag': -286.32,\n",
       "  'deltagerr': 7.81,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.10.3.4'],\n",
       "  'equation': '(3) cpd00007[0] + (4) cpd01360[0] => (6) cpd00001[0] + (2) cpd01462[0]',\n",
       "  'id': 'rxn00054',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-aminophenol:oxygen oxidoreductase (dimerizing)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-3:cpd00007:0:0:\"O2\";-4:cpd01360:0:0:\"o-Aminophenol\";6:cpd00001:0:0:\"H2O\";2:cpd01462:0:0:\"Isophenoxazine\"'},\n",
       " {'abbreviation': 'R00075',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.5.1.43-RXN',\n",
       "   'BrachyCyc: 2.5.1.43-RXN',\n",
       "   'CornCyc: 2.5.1.43-RXN',\n",
       "   'KEGG: R00075',\n",
       "   'MetaCyc: 2.5.1.43-RXN',\n",
       "   'Name: 3-amino-3-carboxypropyltransferase; S-adenosyl-L-methionine:S-adenosyl-L-methionine; S-adenosyl-L-methionine:S-adenosyl-L-methionine 3-amino-3-carboxypropyltransferase; S-adenosyl-L-methionine:S-adenosyl-L-methionine:S-adenosyl-L-methionine 3-amino-3-carboxypropyltransferase; S-adenosyl-L-methionine:S-adenosyl-L-methionine:S-adenosyl-Lmethionine 3-amino-3-carboxypropyltransferase; nicotianamine synthase'],\n",
       "  'code': '(3) cpd00017[0] <=> (3) cpd00147[0] + (1) cpd03158[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00067;cpd00147;cpd03158',\n",
       "  'definition': '(3) S-Adenosyl-L-methionine[0] => (4) H+[0] + (3) 5-Methylthioadenosine[0] + (1) Nicotianamine[0]',\n",
       "  'deltag': -34.08,\n",
       "  'deltagerr': 14.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.5.1.43'],\n",
       "  'equation': '(3) cpd00017[0] => (4) cpd00067[0] + (3) cpd00147[0] + (1) cpd03158[0]',\n",
       "  'id': 'rxn00055',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn37935',\n",
       "  'name': 'S-adenosyl-L-methionine:S-adenosyl-L-methionine 3-amino-3-carboxypropyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': [\"MetaCyc: PWY-5912 (2'-deoxymugineic acid phytosiderophore biosynthesis); PWY-5957 (L-nicotianamine biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Siderophores-Biosynthesis (Siderophore Biosynthesis)\"],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-3:cpd00017:0:0:\"S-Adenosyl-L-methionine\";4:cpd00067:0:0:\"H+\";3:cpd00147:0:0:\"5-Methylthioadenosine\";1:cpd03158:0:0:\"Nicotianamine\"'},\n",
       " {'abbreviation': 'R00078',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN0-1483',\n",
       "   'BiGG: FEROpp',\n",
       "   'ChlamyCyc: RXN0-1483',\n",
       "   'KEGG: R00078',\n",
       "   'MetaCyc: RXN0-1483',\n",
       "   'Name: Fe(II):oxygen oxidoreductase; Fe2+:oxygen oxidoreductase; HEPH; caeruloplasmin; ceruloplasmin; ferro:O2 oxidoreductase; ferroxidase; ferroxidase I; ferroxidase, iron II:oxygen oxidoreductase; hephaestin; iron II:oxygen oxidoreductase; iron oxidase, iron(II):oxygen oxidoreductase; iron(II): oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (4) cpd10515[0] <=> (2) cpd00001[0] + (4) cpd10516[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00067;cpd10515;cpd10516',\n",
       "  'definition': '(1) O2[0] + (4) H+[0] + (4) Fe+2[0] <=> (2) H2O[0] + (4) Fe+3[0]',\n",
       "  'deltag': -11.89,\n",
       "  'deltagerr': 4.12,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.16.3.1'],\n",
       "  'equation': '(1) cpd00007[0] + (4) cpd00067[0] + (4) cpd10515[0] <=> (2) cpd00001[0] + (4) cpd10516[0]',\n",
       "  'id': 'rxn00056',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn25990;rxn31826',\n",
       "  'name': 'Fe(II):oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00860 (Porphyrin and chlorophyll metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-4:cpd00067:0:0:\"H+\";-4:cpd10515:0:0:\"Fe2+\";2:cpd00001:0:0:\"H2O\";4:cpd10516:0:0:\"fe3\"'},\n",
       " {'abbreviation': 'R00080',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00080',\n",
       "   'MetaCyc: CATECHOL-OXIDASE-DIMERIZING-RXN',\n",
       "   'Name: Catechol:oxygen oxidoreductase (dimerizing); catechol oxidase (dimerizing); catechol:oxygen oxidoreductase (dimerizing)'],\n",
       "  'code': '(3) cpd00007[0] + (4) cpd00077[0] <=> (6) cpd00001[0] + (2) cpd02409[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00077;cpd02409',\n",
       "  'definition': '(3) O2[0] + (4) Catechol[0] => (6) H2O[0] + (2) Dibenzo[1,4]dioxin-2,3-dione[0]',\n",
       "  'deltag': -264.94,\n",
       "  'deltagerr': 7.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.14', '1.10.3.1'],\n",
       "  'equation': '(3) cpd00007[0] + (4) cpd00077[0] => (6) cpd00001[0] + (2) cpd02409[0]',\n",
       "  'id': 'rxn00057',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Catechol:oxygen oxidoreductase (dimerizing)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-3:cpd00007:0:0:\"O2\";-4:cpd00077:0:0:\"Catechol\";6:cpd00001:0:0:\"H2O\";2:cpd02409:0:0:\"Dibenzo[1,4]dioxin-2,3-dione\"'},\n",
       " {'abbreviation': 'R00081',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JM_Creinhardtii: JM_Cre_137; JM_Cre_138; JM_Cre_148; JM_Cre_149',\n",
       "   'KEGG: R00081',\n",
       "   'Name: Cytochrome C Oxidase; Ferrocytochrome-c:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (4) cpd00110[0] <=> (2) cpd00001[0] + (4) cpd00109[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00067;cpd00109;cpd00110',\n",
       "  'definition': '(1) O2[0] + (4) H+[0] + (4) Cytochrome c2+[0] => (2) H2O[0] + (4) Cytochrome c3+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.7.2.1', '1.9.3.1'],\n",
       "  'equation': '(1) cpd00007[0] + (4) cpd00067[0] + (4) cpd00110[0] => (2) cpd00001[0] + (4) cpd00109[0]',\n",
       "  'id': 'rxn00058',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33884',\n",
       "  'name': 'Ferrocytochrome-c:oxygen oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-4:cpd00067:0:0:\"H+\";-4:cpd00110:0:0:\"Cytochrome c2+\";2:cpd00001:0:0:\"H2O\";4:cpd00109:0:0:\"Cytochrome c3+\"'},\n",
       " {'abbreviation': 'Benzenediol:oxygen oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Benzenediol:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (4) cpd01232[0] <=> (2) cpd00001[0] + (4) cpd00415[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00067;cpd00415;cpd01232',\n",
       "  'definition': '(1) O2[0] + (4) H+[0] + (4) Benzenediol[0] <=> (2) H2O[0] + (4) Quinol[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.10.3.2'],\n",
       "  'equation': '(1) cpd00007[0] + (4) cpd00067[0] + (4) cpd01232[0] <=> (2) cpd00001[0] + (4) cpd00415[0]',\n",
       "  'id': 'rxn00059',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Benzenediol:oxygen oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:-4',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-4:cpd00067:0:0:\"H+\";-4:cpd01232:0:0:\"Benzenediol\";2:cpd00001:0:0:\"H2O\";4:cpd00415:0:0:\"Quinol\"'},\n",
       " {'abbreviation': 'R00084',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: OHMETHYLBILANESYN-RXN',\n",
       "   'BiGG: HEM3; HMBS',\n",
       "   'BrachyCyc: OHMETHYLBILANESYN-RXN',\n",
       "   'KEGG: R00084',\n",
       "   'MetaCyc: OHMETHYLBILANESYN-RXN',\n",
       "   'Name: (4-[2-carboxyethyl]-3-[carboxymethyl]pyrrol-2-yl)methyltransferase (hydrolysing); HMB-synthase; Porphobilinogen ammonia-lyase (polymerizing); Porphobilinogen deaminase; hydroxymethylbilane synthase; hydroxymethylbilanesynthase; porphobilinogen ammonia-lyase (polymerizing); porphobilinogen deaminase; porphobilinogen:(4-[2-carboxyethyl]-3-[carboxymethyl]pyrrol-2-yl)methyltransferase (hydrolysing); pre-uroporphyrinogen synthase; uroporphyrinogen I synthase; uroporphyrinogen I synthetase; uroporphyrinogen synthase; uroporphyrinogen synthetase'],\n",
       "  'code': '(1) cpd00001[0] + (4) cpd00689[0] <=> (4) cpd00013[0] + (1) cpd00755[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00689;cpd00755',\n",
       "  'definition': '(1) H2O[0] + (4) Porphobilinogen[0] => (4) NH3[0] + (1) Hydroxymethylbilane[0]',\n",
       "  'deltag': -32.71,\n",
       "  'deltagerr': 2.06,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.5.1.61', '4.3.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (4) cpd00689[0] => (4) cpd00013[0] + (1) cpd00755[0]',\n",
       "  'id': 'rxn00060',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20490;rxn30032;rxn30743;rxn35409',\n",
       "  'name': 'porphobilinogen:(4-[2-carboxyethyl]-3-[carboxymethyl]pyrrol-2-yl)methyltransferase (hydrolysing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenosylcobamide-Biosynthesis (Adenosylcobamide Biosynthesis); Chlorophyll-Biosynthesis (Chlorophyll Biosynthesis); Chlorophyll-a-Biosynthesis (Chlorophyll <i>a</i> Biosynthesis); Cobalamin-Biosynthesis (Cobamide Biosynthesis); Cobamide-De-Novo-Biosynthesis (Cobamide <i>de novo</i> Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); De-Novo-Adenosylcobalamin-Biosynthesis (Adenosylcobalamin <i>de novo</i> Biosynthesis); HEME-SYN (Heme Biosynthesis); Heme-b-Biosynthesis (Heme <i>b</i> Biosynthesis); P381-PWY (adenosylcobalamin biosynthesis II (aerobic)); PWY-5188 (tetrapyrrole biosynthesis I (from glutamate)); PWY-5189 (tetrapyrrole biosynthesis II (from glycine)); PWY-5507 (adenosylcobalamin biosynthesis I (anaerobic)); PWY-5529 (superpathway of bacteriochlorophyll <i>a</i> biosynthesis); PWY-5918 (superpathay of heme <i>b</i> biosynthesis from glutamate); PWY-5920 (superpathway of <i>b</i> heme biosynthesis from glycine); Porphyrin-Compounds-Biosynthesis (Porphyrin Compound Biosynthesis); Tetrapyrrole-Biosynthesis (Tetrapyrrole Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00860 (Porphyrin and chlorophyll metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-4:cpd00689:0:0:\"Porphobilinogen\";4:cpd00013:0:0:\"NH3\";1:cpd00755:0:0:\"Hydroxymethylbilane\"'},\n",
       " {'abbreviation': 'R00085',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: APYRASE-RXN',\n",
       "   'BiGG: ATPH1',\n",
       "   'BrachyCyc: APYRASE-RXN',\n",
       "   'KEGG: R00085',\n",
       "   'MetaCyc: APYRASE-RXN',\n",
       "   'Name: ADPase; ATP diphosphohydrolase; ATP diphosphohydrolase (phosphate-forming); ATP diphosphohydrolase [ambiguous]; ATP-diphosphatase; ATP-diphosphohydrolase; Adenosine diphosphatase; Apyrase; adenosine diphosphatase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd00002[0] <=> (2) cpd00009[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00009;cpd00018;cpd00067',\n",
       "  'definition': '(2) H2O[0] + (1) ATP[0] => (2) Phosphate[0] + (1) AMP[0] + (3) H+[0]',\n",
       "  'deltag': -11.9,\n",
       "  'deltagerr': 0.15,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.5'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd00002[0] => (2) cpd00009[0] + (1) cpd00018[0] + (3) cpd00067[0]',\n",
       "  'id': 'rxn00061',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27710',\n",
       "  'name': 'ATP diphosphohydrolase (phosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";2:cpd00009:0:0:\"Phosphate\";1:cpd00018:0:0:\"AMP\";3:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00086',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.4.3-RXN; 3.6.4.4-RXN; 3.6.4.5-RXN; 3.6.4.6-RXN; ADENOSINETRIPHOSPHATASE-RXN; NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN-11109; RXN-11135; RXN0-1061',\n",
       "   'BiGG: ATPM; ATPOBJ; FATP; NTP1; U214',\n",
       "   'BrachyCyc: 3.6.3.1-RXN; ADENOSINETRIPHOSPHATASE-RXN; NUCLEOSIDE-TRIPHOSPHATASE-RXN',\n",
       "   'KEGG: R00086; R10531',\n",
       "   'MetaCyc: 3.6.3.11-RXN; 3.6.3.47-RXN; 3.6.3.50-RXN; 3.6.3.51-RXN; 3.6.3.52-RXN; 3.6.4.1-RXN; 3.6.4.11-RXN; 3.6.4.2-RXN; 3.6.4.3-RXN; 3.6.4.4-RXN; 3.6.4.5-RXN; 3.6.4.6-RXN; 3.6.4.7-RXN; 3.6.4.8-RXN; 3.6.4.9-RXN; ABC-16-RXN; ABC-19-RXN; ABC-64-RXN; ABC-70-RXN; ADENOSINETRIPHOSPHATASE-RXN; ATPASE-RXN; NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN-10799; RXN-10800; RXN-11109; RXN-11135; RXN0-1061; TRANS-RXN-332; TRANS-RXN-333; TRANS-RXN-334',\n",
       "   'Name: ATP maintanence; ATP maintenance requirement; ATP maintenance requirment; ATP phosphohydrolase; ATP phosphohydrolase (protein-secreting); ATP phosphohydrolase (tubulin-dimerizing); ATP phosphohydrolase (tungstate-importing); ATPase; D-galactofuranose ABC transporter; D-galactose ABC transporter; Katanin; Microtubule-severing ATPase; Model-specific reaction, used to affect ATP levels by converting ADT to ATP; microtubule-severing ATPase; nucleoside-triphosphatase (ATP); ribose ABC transporter'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] <=> (1) cpd00008[0] + (1) cpd00009[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00008;cpd00009;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0]',\n",
       "  'deltag': -6.16,\n",
       "  'deltagerr': 0.07,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.15',\n",
       "   '3.6.1.3',\n",
       "   '3.6.1.5',\n",
       "   '3.6.1.8',\n",
       "   '3.6.3.1',\n",
       "   '3.6.3.10',\n",
       "   '3.6.3.11',\n",
       "   '3.6.3.12',\n",
       "   '3.6.3.14',\n",
       "   '3.6.3.15',\n",
       "   '3.6.3.16',\n",
       "   '3.6.3.17',\n",
       "   '3.6.3.18',\n",
       "   '3.6.3.19',\n",
       "   '3.6.3.2',\n",
       "   '3.6.3.20',\n",
       "   '3.6.3.21',\n",
       "   '3.6.3.22',\n",
       "   '3.6.3.23',\n",
       "   '3.6.3.24',\n",
       "   '3.6.3.25',\n",
       "   '3.6.3.26',\n",
       "   '3.6.3.27',\n",
       "   '3.6.3.28',\n",
       "   '3.6.3.29',\n",
       "   '3.6.3.3',\n",
       "   '3.6.3.30',\n",
       "   '3.6.3.31',\n",
       "   '3.6.3.32',\n",
       "   '3.6.3.33',\n",
       "   '3.6.3.34',\n",
       "   '3.6.3.35',\n",
       "   '3.6.3.36',\n",
       "   '3.6.3.37',\n",
       "   '3.6.3.38',\n",
       "   '3.6.3.39',\n",
       "   '3.6.3.4',\n",
       "   '3.6.3.40',\n",
       "   '3.6.3.41',\n",
       "   '3.6.3.42',\n",
       "   '3.6.3.43',\n",
       "   '3.6.3.44',\n",
       "   '3.6.3.46',\n",
       "   '3.6.3.47',\n",
       "   '3.6.3.48',\n",
       "   '3.6.3.49',\n",
       "   '3.6.3.5',\n",
       "   '3.6.3.50',\n",
       "   '3.6.3.51',\n",
       "   '3.6.3.52',\n",
       "   '3.6.3.53',\n",
       "   '3.6.3.54',\n",
       "   '3.6.3.55',\n",
       "   '3.6.3.6',\n",
       "   '3.6.3.7',\n",
       "   '3.6.3.8',\n",
       "   '3.6.3.9',\n",
       "   '3.6.4.1',\n",
       "   '3.6.4.10',\n",
       "   '3.6.4.11',\n",
       "   '3.6.4.12',\n",
       "   '3.6.4.13',\n",
       "   '3.6.4.2',\n",
       "   '3.6.4.3',\n",
       "   '3.6.4.4',\n",
       "   '3.6.4.5',\n",
       "   '3.6.4.6',\n",
       "   '3.6.4.7',\n",
       "   '3.6.4.8',\n",
       "   '3.6.4.9',\n",
       "   '5.6.1.c',\n",
       "   '5.6.1.d',\n",
       "   '5.6.1.f',\n",
       "   '5.6.1.g',\n",
       "   '7.3.2.a',\n",
       "   '7.3.2.f',\n",
       "   '7.5.2.h',\n",
       "   '7.5.2.j',\n",
       "   '7.5.2.l',\n",
       "   '7.6.2.c',\n",
       "   '7.6.2.f'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00062',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn11300;rxn27586;rxn27587;rxn27588;rxn27616;rxn38021;rxn38022;rxn38023;rxn38233',\n",
       "  'name': 'ATP phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00087',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.1.19-RXN',\n",
       "   'BiGG: NTPP6',\n",
       "   'BrachyCyc: 3.6.1.19-RXN; ATP-PYROPHOSPHATASE-RXN',\n",
       "   'KEGG: R00087',\n",
       "   'MetaCyc: 3.6.1.19-RXN; ATP-PYROPHOSPHATASE-RXN',\n",
       "   'Name: ATP Pyrophosphatase; ATP diphosphatase; ATP diphosphohydrolase (diphosphate-forming); ATP diphosphohydrolase [ambiguous]; ATP pyrophosphatase; ATP pyrophosphohydrolase; ATPase (ambiguous); Nucleoside triphosphate pyrophosphorylase (atp); adenosine triphosphate pyrophosphatase; atp pyrophosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] <=> (1) cpd00012[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00012;cpd00018;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0]',\n",
       "  'deltag': -8.43,\n",
       "  'deltagerr': 0.14,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00063',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33891',\n",
       "  'name': 'ATP diphosphohydrolase (diphosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00088',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ATPHs',\n",
       "   'KEGG: R00088',\n",
       "   'MetaCyc: ATP-DEAMINASE-RXN',\n",
       "   'iAF1260: ATPHs',\n",
       "   'iMA945: ATPHs',\n",
       "   'Name: ATP amine hydrolysis (spontaneous); ATP aminohydrolase; ATP deaminase; adenosine triphosphate deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] <=> (1) cpd00013[0] + (1) cpd00068[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00013;cpd00067;cpd00068',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) H+[0] => (1) NH3[0] + (1) ITP[0]',\n",
       "  'deltag': -13.75,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.18'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00067[0] => (1) cpd00013[0] + (1) cpd00068[0]',\n",
       "  'id': 'rxn00064',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29995',\n",
       "  'name': 'ATP aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00067:0:0:\"H+\";1:cpd00013:0:0:\"NH3\";1:cpd00068:0:0:\"ITP\"'},\n",
       " {'abbreviation': 'R00089',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENYLATECYC-RXN',\n",
       "   'BiGG: ADNCYC; CYR1',\n",
       "   'BrachyCyc: ADENYLATECYC-RXN',\n",
       "   'KEGG: R00089',\n",
       "   'MetaCyc: ADENYLATECYC-RXN',\n",
       "   \"Name: 3',5'-cyclic AMP synthetase; ATP diphosphate-lyase (cyclizing); ATP diphosphate-lyase (cyclizing, 3',5'-cyclic-AMP-forming); ATP diphosphate-lyase (cyclizing; 3',5'-cyclic-AMP-forming); ATP pyrophosphate-lyase (cyclizing); Adenylate cyclase; adenyl cyclase; adenylate cyclase; adenylylcyclase\"],\n",
       "  'code': '(1) cpd00002[0] <=> (1) cpd00012[0] + (1) cpd00446[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00067;cpd00446',\n",
       "  'definition': '(1) ATP[0] => (1) PPi[0] + (1) H+[0] + (1) cAMP[0]',\n",
       "  'deltag': 1.72,\n",
       "  'deltagerr': 0.24,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.6.1.1'],\n",
       "  'equation': '(1) cpd00002[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00446[0]',\n",
       "  'id': 'rxn00065',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35429',\n",
       "  'name': \"ATP diphosphate-lyase (cyclizing; 3',5'-cyclic-AMP-forming)\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd00446:0:0:\"cAMP\"'},\n",
       " {'abbreviation': 'R00090',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00090',\n",
       "   'MetaCyc: NADH-PEROXIDASE-RXN',\n",
       "   'Name: DPNH peroxidase; NAD peroxidase; NADH peroxidase; NADH-peroxidase; NADH2 peroxidase; NADH:hydrogen-peroxide oxidoreductase; diphosphopyridine nucleotide peroxidase; nicotinamide adenine dinucleotide peroxidase'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00025[0] <=> (2) cpd00001[0] + (1) cpd00003[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00025;cpd00067',\n",
       "  'definition': '(1) NADH[0] + (1) H2O2[0] + (1) H+[0] => (2) H2O[0] + (1) NAD[0]',\n",
       "  'deltag': -77.79,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.1'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00025[0] + (1) cpd00067[0] => (2) cpd00001[0] + (1) cpd00003[0]',\n",
       "  'id': 'rxn00066',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NADH:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00025:0:0:\"H2O2\";-1:cpd00067:0:0:\"H+\";2:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\"'},\n",
       " {'abbreviation': 'R00091',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00091',\n",
       "   'MetaCyc: COA-DISULFIDE-REDUCTASE-NADH-RXN; RXN-16679',\n",
       "   'Name: CoA-disulfide reductase; CoA-disulfide reductase (NADH2); CoA:NAD(P)+ oxidoreductase; CoA:NAD+ oxidoreductase; CoADR; NADH2:CoA-disulfide oxidoreductase; coenzyme A disulfide reductase'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00010[0] <=> (1) cpd00004[0] + (1) cpd01378[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00067;cpd01378',\n",
       "  'definition': '(1) NAD[0] + (2) CoA[0] <= (1) NADH[0] + (1) H+[0] + (1) CoA-disulfide[0]',\n",
       "  'deltag': 2.91,\n",
       "  'deltagerr': 11.5,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.8.1.14', '1.8.1.M1'],\n",
       "  'equation': '(1) cpd00003[0] + (2) cpd00010[0] <= (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd01378[0]',\n",
       "  'id': 'rxn00067',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CoA:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-2:cpd00010:0:0:\"CoA\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd01378:0:0:\"CoA-disulfide\"'},\n",
       " {'abbreviation': 'R00092',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FERRIC-CHELATE-REDUCTASE-RXN',\n",
       "   'BrachyCyc: FERRIC-CHELATE-REDUCTASE-RXN',\n",
       "   'ChlamyCyc: FERRIC-CHELATE-REDUCTASE-RXN',\n",
       "   'KEGG: R00092',\n",
       "   'MetaCyc: FERRIC-CHELATE-REDUCTASE-RXN',\n",
       "   'Name: Fe(II):NAD+ oxidoreductase; Fe2+:NAD+ oxidoreductase; Ferric-chelate reductase; NADH2:Fe3+ oxidoreductase; NADH:Fe2+ oxidoreductase; NADH:Fe3+ oxidoreductase; NADH:Fe3+-EDTA reductase; ferric chelate reductase; ferric chelate reductase (ambigous); ferric-chelate reductase; ferric-chelate reductase (NADH); iron chelate reductase; iron chelate reductase (ambigous)'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd10515[0] <=> (1) cpd00004[0] + (2) cpd10516[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd10515;cpd10516',\n",
       "  'definition': '(1) NAD[0] + (1) H+[0] + (2) Fe+2[0] <= (1) NADH[0] + (2) Fe+3[0]',\n",
       "  'deltag': 48.81,\n",
       "  'deltagerr': 2.02,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.16.1.7'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00067[0] + (2) cpd10515[0] <= (1) cpd00004[0] + (2) cpd10516[0]',\n",
       "  'id': 'rxn00068',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27853;rxn32326;rxn32328;rxn32703',\n",
       "  'name': 'Fe2+:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Ethylene-Biosynthesis (Ethylene Biosynthesis); HORMONE-SYN (Hormone Biosynthesis); Iron-Metabolism (Iron Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5934 (iron reduction and absorption); PWY-6854 (ethylene biosynthesis III (microbes)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00067:0:0:\"H+\";-2:cpd10515:0:0:\"Fe2+\";1:cpd00004:0:0:\"NADH\";2:cpd10516:0:0:\"fe3\"'},\n",
       " {'abbreviation': 'R00093',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTAMATE-SYNTHASE-NADH-RXN',\n",
       "   'BiGG: GLT1; GLUSx',\n",
       "   'BrachyCyc: GLUTAMATE-SYNTHASE-NADH-RXN',\n",
       "   'KEGG: R00093',\n",
       "   'MetaCyc: GLUTAMATE-SYNTHASE-NADH-RXN',\n",
       "   'Name: Glutamate synthase [NADH]; L-glutamate synthase (NADH); L-glutamate synthetase; L-glutamate:NAD+ oxidoreductase (transaminating); NADH-dependent glutamate synthase; NADH-glutamate synthase; NADH: GOGAT; glutamate (reduced nicotinamide adenine dinucleotide) synthase; glutamate synthase (NADH); glutamate synthase (NADH2)'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00023[0] <=> (1) cpd00004[0] + (1) cpd00024[0] + (1) cpd00053[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00023;cpd00024;cpd00053;cpd00067',\n",
       "  'definition': '(1) NAD[0] + (2) L-Glutamate[0] <=> (1) NADH[0] + (1) 2-Oxoglutarate[0] + (1) L-Glutamine[0] + (1) H+[0]',\n",
       "  'deltag': 11.53,\n",
       "  'deltagerr': 0.22,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.14'],\n",
       "  'equation': '(1) cpd00003[0] + (2) cpd00023[0] <=> (1) cpd00004[0] + (1) cpd00024[0] + (1) cpd00053[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00069',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27888',\n",
       "  'name': 'L-glutamate:NAD+ oxidoreductase (transaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Ammonia-Assimilation (Ammonia Assimilation); Degradation (Degradation/Utilization/Assimilation); GLUGLNSYN-PWY (L-glutamate biosynthesis IV); GLUTAMATE-SYN (L-glutamate Biosynthesis); GLUTAMINE-SYN (L-glutamine Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Metabolic-Clusters (Metabolic Clusters); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-3282 (superpathway of ammonia assimilation (plants)); PWY-5505 (L-glutamate and L-glutamine biosynthesis); PWY-6963 (ammonia assimilation cycle I)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00910 (Nitrogen metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-2:cpd00023:0:0:\"L-Glutamate\";1:cpd00004:0:0:\"NADH\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00053:0:0:\"L-Glutamine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00094',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_GDR; R_GDRh; R_GDRm',\n",
       "   'KEGG: R00094',\n",
       "   'Name: glutathione-disulfide reductase; glutathione:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00042[0] <=> (1) cpd00004[0] + (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00042;cpd00067;cpd00111',\n",
       "  'definition': '(1) NAD[0] + (2) GSH[0] <=> (1) NADH[0] + (1) H+[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': 3.7,\n",
       "  'deltagerr': 0.21,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.1.7'],\n",
       "  'equation': '(1) cpd00003[0] + (2) cpd00042[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00070',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34788;rxn34789',\n",
       "  'name': 'glutathione:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-2:cpd00042:0:0:\"GSH\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'R00095',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.6.5.4-RXN',\n",
       "   'BrachyCyc: 1.6.5.4-RXN',\n",
       "   'ChlamyCyc: 1.6.5.4-RXN',\n",
       "   'KEGG: R00095',\n",
       "   'MetaCyc: 1.6.5.4-RXN',\n",
       "   'Name: AFR; AFR-reductase; MDAsA reductase (NADPH); MDHA; Monodehydroascorbate reductase (NADH); NADH-semidehydroascorbate oxidoreductase; NADH:AFR oxidoreductase; NADH:ascorbate radical oxidoreductase; NADH:monodehydroascorbate oxidoreductase; NADH:semidehydroascorbic acid oxidoreductase; SDA reductase; SOR; ascorbate free radical reductase; ascorbate free-radical reductase; ascorbic free radical reductase; monodehydroascorbate reductase (NADH); monodehydroascorbate reductase (NADH2); semidehydroascorbate reductase'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00059[0] <=> (1) cpd00004[0] + (2) cpd00766[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00059;cpd00067;cpd00766',\n",
       "  'definition': '(1) NAD[0] + (2) L-Ascorbate[0] <= (1) NADH[0] + (3) H+[0] + (2) Ascorbate radical[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.6.5.4'],\n",
       "  'equation': '(1) cpd00003[0] + (2) cpd00059[0] <= (1) cpd00004[0] + (3) cpd00067[0] + (2) cpd00766[0]',\n",
       "  'id': 'rxn00071',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27489;rxn32941;rxn32953',\n",
       "  'name': 'NADH:monodehydroascorbate oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['MetaCyc: COFACTOR-DEGRADATION (Cofactor, Prosthetic Group, Electron Carrier Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-6370 (ascorbate recycling (cytosolic))',\n",
       "   'KEGG: rn00053 (Ascorbate and aldarate metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-2:cpd00059:0:0:\"L-Ascorbate\";1:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";2:cpd00766:0:0:\"Ascorbate radical\"'},\n",
       " {'abbreviation': 'R00097',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00097',\n",
       "   'MetaCyc: AQUACOBALAMIN-REDUCTASE-RXN',\n",
       "   'Name: NADH:cob(III)alamine oxidoreductase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd00003[0] + (2) cpd00423[0] <=> (1) cpd00004[0] + (2) cpd00730[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00423;cpd00730',\n",
       "  'definition': '(2) H2O[0] + (1) NAD[0] + (1) H+[0] + (2) Vitamin B12r[0] <= (1) NADH[0] + (2) Aquacobalamin[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.16.1.3'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd00003[0] + (1) cpd00067[0] + (2) cpd00423[0] <= (1) cpd00004[0] + (2) cpd00730[0]',\n",
       "  'id': 'rxn00072',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NADH:cob(III)alamine oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00860 (Porphyrin and chlorophyll metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00067:0:0:\"H+\";-2:cpd00423:0:0:\"Vitamin B12r\";1:cpd00004:0:0:\"NADH\";2:cpd00730:0:0:\"Aquacobalamin\"'},\n",
       " {'abbreviation': 'rxn00073',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00423[0] <=> (1) cpd00004[0] + (2) cpd03424[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00423;cpd03424',\n",
       "  'definition': '(1) NAD[0] + (1) H+[0] + (2) Vitamin B12r[0] <=> (1) NADH[0] + (2) Vitamin B12[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00067[0] + (2) cpd00423[0] <=> (1) cpd00004[0] + (2) cpd03424[0]',\n",
       "  'id': 'rxn00073',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00073',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:H:6/R:2|CI:4',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00067:0:0:H+;-2:cpd00423:0:0:\"Vitamin B12r\";1:cpd00004:0:0:\"NADH\";2:cpd03424:0:0:\"Vitamin B12\"'},\n",
       " {'abbreviation': 'R00099',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00099',\n",
       "   'MetaCyc: COBIIALAMIN-REDUCTASE-RXN',\n",
       "   'Name: B12r reductase; NADH2:cob(II)alamin oxidoreductase; NADH:cob(II)alamin oxidoreductase; cob(I)alamin:NAD+ oxidoreductase; cob(II)alamin reductase; vitamin B12r reductase'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00635[0] <=> (1) cpd00004[0] + (2) cpd00423[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00423;cpd00635',\n",
       "  'definition': '(1) NAD[0] + (1) H+[0] + (2) Cbl[0] <= (1) NADH[0] + (2) Vitamin B12r[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.16.1.4'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00067[0] + (2) cpd00635[0] <= (1) cpd00004[0] + (2) cpd00423[0]',\n",
       "  'id': 'rxn00074',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'cob(I)alamin:NAD+ oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00860 (Porphyrin and chlorophyll metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00067:0:0:\"H+\";-2:cpd00635:0:0:\"Cbl\";1:cpd00004:0:0:\"NADH\";2:cpd00423:0:0:\"Vitamin B12r\"'},\n",
       " {'abbreviation': 'NADNm',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NADNUCLEOSID-RXN',\n",
       "   'BiGG: NADN; NADNm; U102',\n",
       "   'BrachyCyc: NADNUCLEOSID-RXN',\n",
       "   'KEGG: R00102',\n",
       "   'MetaCyc: NADNUCLEOSID-RXN; RXN-13859',\n",
       "   'Name: DPN hydrolase; DPNase; NAD glycohydrolase; NAD hydrolase; NAD nucleosidase; NAD+ glycohydrolase; NAD+ nucleosidase; NADase; diphosphopyridine nucleosidase; nicotinamide adenine dinucleotide glycohydrolase; nicotinamide adenine dinucleotide nucleosidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] <=> (1) cpd00133[0] + (1) cpd00251[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00067;cpd00133;cpd00251',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] => (1) H+[0] + (1) Nicotinamide[0] + (1) ADPribose[0]',\n",
       "  'deltag': -8.85,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.2.5', '3.2.2.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] => (1) cpd00067[0] + (1) cpd00133[0] + (1) cpd00251[0]',\n",
       "  'id': 'rxn00075',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn14307;rxn20263',\n",
       "  'name': 'NAD nucleosidase, mitochondrial',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";1:cpd00067:0:0:\"H+\";1:cpd00133:0:0:\"Nicotinamide\";1:cpd00251:0:0:\"ADPribose\"'},\n",
       " {'abbreviation': 'R00103',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NADPYROPHOSPHAT-RXN',\n",
       "   'BiGG: NADDP; NADDPe; NADDPp',\n",
       "   'BrachyCyc: NADPYROPHOSPHAT-RXN',\n",
       "   'KEGG: R00103',\n",
       "   'MetaCyc: NADPYROPHOSPHAT-RXN',\n",
       "   'Name: NAD diphosphatase; NAD+ diphosphatase; NAD+ phosphohydrolase; NADH pyrophosphatase; NADP pyrophosphatase; RXNBWI-18; nicotinamide adenine dinucleotide pyrophosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] <=> (1) cpd00018[0] + (1) cpd00355[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00018;cpd00067;cpd00355',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] => (1) AMP[0] + (2) H+[0] + (1) Nicotinamide ribonucleotide[0]',\n",
       "  'deltag': -8.79,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.22', '3.6.1.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] => (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd00355[0]',\n",
       "  'id': 'rxn00076',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20269;rxn30410;rxn33290;rxn38181',\n",
       "  'name': 'NAD+ phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-BIOSYNTHESIS-II (NAD salvage pathway III (to nicotinamide riboside)); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); PWY-5381 (pyridine nucleotide cycling (plants)); PWY-7761 (NAD salvage pathway II (PNC IV cycle)); PYRIDNUCSAL-PWY (NAD salvage pathway I (PNC VI cycle))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd00355:0:0:\"Nicotinamide ribonucleotide\"'},\n",
       " {'abbreviation': 'R00104',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NAD-KIN-RXN',\n",
       "   'BiGG: NADK; NADKm; POS5_1; POS5_2; UTR1_1; UTR1_2; YEL041W_1; YEL041W_2',\n",
       "   'BrachyCyc: NAD-KIN-RXN',\n",
       "   'KEGG: R00104',\n",
       "   'MetaCyc: NAD-KIN-RXN',\n",
       "   \"Name: ATP:NAD+ 2'-phosphotransferase; DPN kinase; NAD kinase; NAD+ kinase; NADH kinase POS5; NADkinase; RXNBWI-16; RXNBWI-19; nad(+) kinase; nicotinamide adenine dinucleotide kinase; nicotinamide adenine dinucleotide kinase (phosphorylating)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00003[0] <=> (1) cpd00006[0] + (1) cpd00008[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00003;cpd00006;cpd00008;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) NAD[0] => (1) NADP[0] + (1) ADP[0] + (1) H+[0]',\n",
       "  'deltag': -3.14,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.23'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00003[0] => (1) cpd00006[0] + (1) cpd00008[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00077',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28003;rxn30190;rxn38696',\n",
       "  'name': \"ATP:NAD+ 2'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); NADPHOS-DEPHOS-PWY (NAD phosphorylation and dephosphorylation); NADPHOS-DEPHOS-PWY-1 (NAD phosphorylation and transhydrogenation); PWY-5083 (NAD/NADH phosphorylation and dephosphorylation); PWY-7245 (superpathway NAD/NADP-NADH/NADPH interconversion (yeast)); PWY-7268 (NAD/NADP-NADH/NADPH cytosolic interconversion (yeast)); PWY-7269 (NAD/NADP-NADH/NADPH mitochondrial interconversion (yeast))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00003:0:0:\"NAD\";1:cpd00006:0:0:\"NADP\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00105',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NADH-KINASE-RXN',\n",
       "   'BrachyCyc: NADH-KINASE-RXN',\n",
       "   'ChlamyCyc: NADH-KINASE-RXN',\n",
       "   'KEGG: R00105',\n",
       "   'MetaCyc: NADH-KINASE-RXN',\n",
       "   \"Name: ATP:NADH 2'-phosphotransferase; DPNH kinase; NADH kinase; NADH2 kinase; reduced diphosphopyridine nucleotide kinase; reduced nicotinamide adenine dinucleotide kinase (phosphorylating)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00004[0] <=> (1) cpd00005[0] + (1) cpd00008[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00004;cpd00005;cpd00008;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) NADH[0] => (1) NADPH[0] + (1) ADP[0] + (1) H+[0]',\n",
       "  'deltag': -3.14,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.86'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00004[0] => (1) cpd00005[0] + (1) cpd00008[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00078',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33597',\n",
       "  'name': \"ATP:NADH 2'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); PWY-5083 (NAD/NADH phosphorylation and dephosphorylation); PWY-7245 (superpathway NAD/NADP-NADH/NADPH interconversion (yeast)); PWY-7268 (NAD/NADP-NADH/NADPH cytosolic interconversion (yeast)); PWY-7269 (NAD/NADP-NADH/NADPH mitochondrial interconversion (yeast))'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00004:0:0:\"NADH\";1:cpd00005:0:0:\"NADPH\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00106',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00106', 'Name: NADPH:ferricytochrome oxidoreductase'],\n",
       "  'code': '(1) cpd00005[0] + (2) cpd00685[0] <=> (1) cpd00006[0] + (2) cpd00686[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00685;cpd00686',\n",
       "  'definition': '(1) NADPH[0] + (2) Ferricytochrome[0] => (1) NADP[0] + (1) H+[0] + (2) Ferrocytochrome[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.6.2.4'],\n",
       "  'equation': '(1) cpd00005[0] + (2) cpd00685[0] => (1) cpd00006[0] + (1) cpd00067[0] + (2) cpd00686[0]',\n",
       "  'id': 'rxn00079',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NADPH:ferricytochrome oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-2:cpd00685:0:0:\"Ferricytochrome\";1:cpd00006:0:0:\"NADP\";1:cpd00067:0:0:\"H+\";2:cpd00686:0:0:\"Ferrocytochrome\"'},\n",
       " {'abbreviation': 'R00107',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00107',\n",
       "   'MetaCyc: AQUACOBALAMIN-REDUCTASE-NADPH-RXN',\n",
       "   'Name: NADPH:aquacob(III)alamin oxidoreductase; cob(II)alamin:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00005[0] + (2) cpd00730[0] <=> (2) cpd00001[0] + (1) cpd00006[0] + (2) cpd00423[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00067;cpd00423;cpd00730',\n",
       "  'definition': '(1) NADPH[0] + (2) Aquacobalamin[0] => (2) H2O[0] + (1) NADP[0] + (1) H+[0] + (2) Vitamin B12r[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.16.1.5'],\n",
       "  'equation': '(1) cpd00005[0] + (2) cpd00730[0] => (2) cpd00001[0] + (1) cpd00006[0] + (1) cpd00067[0] + (2) cpd00423[0]',\n",
       "  'id': 'rxn00080',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'cob(II)alamin:NADP+ oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00860 (Porphyrin and chlorophyll metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-2:cpd00730:0:0:\"Aquacobalamin\";2:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00067:0:0:\"H+\";2:cpd00423:0:0:\"Vitamin B12r\"'},\n",
       " {'abbreviation': 'rxn00081',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00005[0] + (2) cpd03424[0] <=> (1) cpd00006[0] + (2) cpd00423[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00423;cpd03424',\n",
       "  'definition': '(1) NADPH[0] + (2) Vitamin B12[0] <=> (1) NADP[0] + (1) H+[0] + (2) Vitamin B12r[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00005[0] + (2) cpd03424[0] <=> (1) cpd00006[0] + (1) cpd00067[0] + (2) cpd00423[0]',\n",
       "  'id': 'rxn00081',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00081',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:H:-6/R:-2|CI:-4',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-2:cpd03424:0:0:\"Vitamin B12\";1:cpd00006:0:0:\"NADP\";1:cpd00067:0:0:H+;2:cpd00423:0:0:\"Vitamin B12r\"'},\n",
       " {'abbreviation': '(nitric-oxide-forming)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: (nitric-oxide-forming); N-(omega)-Hydroxyarginine,NADPH:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (2) cpd03527[0] <=> (1) cpd00006[0] + (2) cpd00274[0] + (2) cpd00418[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00007;cpd00067;cpd00274;cpd00418;cpd03527',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (2) N-(omega)-Hydroxyarginine[0] <=> (1) NADP[0] + (5) H+[0] + (2) Citrulline[0] + (2) NO[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.13.39'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (2) cpd03527[0] <=> (1) cpd00006[0] + (5) cpd00067[0] + (2) cpd00274[0] + (2) cpd00418[0]',\n",
       "  'id': 'rxn00082',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-(omega)-Hydroxyarginine,NADPH:oxygen oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:4',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-2:cpd03527:0:0:\"N-(omega)-Hydroxyarginine\";1:cpd00006:0:0:\"NADP\";5:cpd00067:0:0:\"H+\";2:cpd00274:0:0:\"Citrulline\";2:cpd00418:0:0:\"NO\"'},\n",
       " {'abbreviation': 'R00112',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRNUTRANSHYDROGEN-RXN',\n",
       "   'BiGG: NADTRHD; TEST_NADTRHD; THD1; THD5',\n",
       "   'BrachyCyc: PYRNUTRANSHYDROGEN-RXN',\n",
       "   'KEGG: R00112',\n",
       "   'MetaCyc: 1.6.1.2-RXN; PYRNUTRANSHYDROGEN-RXN',\n",
       "   'Name: H+-Thase; NAD transhydrogenase; NAD(P) transhydrogenase; NAD(P) transhydrogenase (AB-specific); NAD(P) transhydrogenase (B-specific); NAD(P)(+) transhydrogenase (AB-specific); NAD(P)(+) transhydrogenase (B-specific); NAD(P)+ transhydrogenase; NAD(P)+ transhydrogenase (AB-specific); NADH transhydrogenase; NADH-NADP-transhydrogenase; NADPH-NAD oxidoreductase; NADPH-NAD transhydrogenase; NADPH:NAD+ oxidoreductase; NADPH:NAD+ oxidoreductase (AB-specific); NADPH:NAD+ oxidoreductase (B-specific); NADPH:NAD+ transhydrogenase; Pyridine nucleotide transhydrogenase; Transhydrogenase; energy-linked transhydrogenase; nad(p)(+) transhydrogenase (b-specific); nicotinamide adenine dinucleotide (phosphate) transhydrogenase; nicotinamide nucleotide transhydrogenase; pyridine nucleotide transferase; pyridine nucleotide transhydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00005[0] <=> (1) cpd00004[0] + (1) cpd00006[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00005;cpd00006',\n",
       "  'definition': '(1) NAD[0] + (1) NADPH[0] <=> (1) NADH[0] + (1) NADP[0]',\n",
       "  'deltag': -0.0,\n",
       "  'deltagerr': 0.08,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.6.1.1', '1.6.1.2', '1.6.1.3'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00005[0] <=> (1) cpd00004[0] + (1) cpd00006[0]',\n",
       "  'id': 'rxn00083',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27488;rxn32580',\n",
       "  'name': 'NADPH:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); PWY-5083 (NAD/NADH phosphorylation and dephosphorylation)',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00005:0:0:\"NADPH\";1:cpd00004:0:0:\"NADH\";1:cpd00006:0:0:\"NADP\"'},\n",
       " {'abbreviation': 'R00113',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NADPH-PEROXIDASE-RXN',\n",
       "   'BrachyCyc: NADPH-PEROXIDASE-RXN',\n",
       "   'KEGG: R00113',\n",
       "   'MetaCyc: NADPH-PEROXIDASE-RXN',\n",
       "   'TS_Athaliana: rxn474_c; rxn474_v',\n",
       "   'Name: NADP peroxidase; NADPH peroxidase; NADPH2 peroxidase; NADPH:hydrogen-peroxide oxidoreductase; TPN peroxidase; TPNH peroxidase; nicotinamide adenine dinucleotide phosphate peroxidase; triphosphopyridine nucleotide peroxidase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00025[0] <=> (2) cpd00001[0] + (1) cpd00006[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00025;cpd00067',\n",
       "  'definition': '(1) NADPH[0] + (1) H2O2[0] + (1) H+[0] => (2) H2O[0] + (1) NADP[0]',\n",
       "  'deltag': -77.79,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.2'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00025[0] + (1) cpd00067[0] => (2) cpd00001[0] + (1) cpd00006[0]',\n",
       "  'id': 'rxn00084',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32993',\n",
       "  'name': 'NADPH:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00025:0:0:\"H2O2\";-1:cpd00067:0:0:\"H+\";2:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\"'},\n",
       " {'abbreviation': 'R00114',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTAMATESYN-RXN',\n",
       "   'BiGG: GLUSy',\n",
       "   'ChlamyCyc: GLUTAMATESYN-RXN',\n",
       "   'KEGG: R00114',\n",
       "   'MetaCyc: GLUTAMATESYN-RXN',\n",
       "   'Name: GOGAT; L-Glutamate:NADP+ oxidoreductase (transaminating); L-glutamate synthase; L-glutamate synthetase; L-glutamate:NADP+ oxidoreductase (transaminating); L-glutamine:2-oxoglutarate aminotransferase, NADPH oxidizing; NADPH-dependent glutamate synthase; NADPH-glutamate synthase; NADPH-linked glutamate synthase; glutamate (reduced nicotinamide adenine dinucleotide phosphate) synthase; glutamate synthase (NADPH); glutamate synthase (NADPH) large chain; glutamate synthase (NADPH) small chain; glutamate synthetase (NADP); glutamine amide-2-oxoglutarate aminotransferase (oxidoreductase, NADP); glutamine-ketoglutaric aminotransferase'],\n",
       "  'code': '(1) cpd00006[0] + (2) cpd00023[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (1) cpd00053[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00023;cpd00024;cpd00053;cpd00067',\n",
       "  'definition': '(1) NADP[0] + (2) L-Glutamate[0] <=> (1) NADPH[0] + (1) 2-Oxoglutarate[0] + (1) L-Glutamine[0] + (1) H+[0]',\n",
       "  'deltag': 11.53,\n",
       "  'deltagerr': 0.23,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.13'],\n",
       "  'equation': '(1) cpd00006[0] + (2) cpd00023[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (1) cpd00053[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00085',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27889',\n",
       "  'name': 'L-Glutamate:NADP+ oxidoreductase (transaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMMASSIM-PWY (ammonia assimilation cycle III); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Ammonia-Assimilation (Ammonia Assimilation); Degradation (Degradation/Utilization/Assimilation); GLUTAMATE-SYN (L-glutamate Biosynthesis); GLUTAMINE-DEG (L-glutamine Degradation); GLUTAMINE-SYN (L-glutamine Biosynthesis); GLUTAMINEFUM-PWY (L-glutamine degradation II); GLUTSYN-PWY (L-glutamate biosynthesis I); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Metabolic-Clusters (Metabolic Clusters); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5505 (L-glutamate and L-glutamine biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00910 (Nitrogen metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-2:cpd00023:0:0:\"L-Glutamate\";1:cpd00005:0:0:\"NADPH\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00053:0:0:\"L-Glutamine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00115',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTATHIONE-REDUCT-NADPH-RXN',\n",
       "   'BiGG: GLR1; GTHO; GTHOm; GTHOr; GTHRD',\n",
       "   'BrachyCyc: GLUTATHIONE-REDUCT-NADPH-RXN',\n",
       "   'KEGG: R00115',\n",
       "   'MetaCyc: GLUTATHIONE-REDUCT-NADPH-RXN',\n",
       "   'Name: GSH reductase; GSSG reductase; Glutathione reductase; Glutathione reductase (NADPH); NADPH-GSSG reductase; NADPH-glutathione reductase; NADPH:oxidized-glutathione oxidoreductase; glutathione S-reductase; glutathione oxidoreductase; glutathione reductase; glutathione reductase (NADPH); glutathione reductase (nadph); glutathione-disulfide reductase; glutathione:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (2) cpd00042[0] <=> (1) cpd00005[0] + (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00042;cpd00067;cpd00111',\n",
       "  'definition': '(1) NADP[0] + (2) GSH[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': 3.7,\n",
       "  'deltagerr': 0.19,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.6.4.2', '1.8.1.7'],\n",
       "  'equation': '(1) cpd00006[0] + (2) cpd00042[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00086',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19841;rxn19842;rxn32133',\n",
       "  'name': 'glutathione:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-4081 (glutathione-peroxide redox reactions); Reductants (Reductant Biosynthesis)',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-2:cpd00042:0:0:\"GSH\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'NADPH:CoA-glutathione oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: NADPH:CoA-glutathione oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (2) cpd00084[0] <=> (1) cpd00005[0] + (1) cpd00381[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00084;cpd00381',\n",
       "  'definition': '(1) NADP[0] + (2) L-Cysteine[0] <=> (1) NADPH[0] + (1) H+[0] + (1) L-Cystine[0]',\n",
       "  'deltag': -1.54,\n",
       "  'deltagerr': 1.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.1.10'],\n",
       "  'equation': '(1) cpd00006[0] + (2) cpd00084[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00381[0]',\n",
       "  'id': 'rxn00087',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NADPH:CoA-glutathione oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-2:cpd00084:0:0:\"L-Cysteine\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00381:0:0:\"L-Cystine\"'},\n",
       " {'abbreviation': 'R00118',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NADPPHOSPHAT-RXN; RXN-5822',\n",
       "   'BiGG: NADPPPS; NADPPPSm; U101; U105',\n",
       "   'BrachyCyc: NADPPHOSPHAT-RXN; RXN-5822',\n",
       "   'KEGG: R00118',\n",
       "   'MetaCyc: RXN-5822',\n",
       "   \"Name: NADP phosphatase; NADP+ 2'-phosphohydrolase; phosphatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] <=> (1) cpd00003[0] + (1) cpd00009[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00006;cpd00009',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] <=> (1) NAD[0] + (1) Phosphate[0]',\n",
       "  'deltag': -3.02,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.-', '3.1.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] <=> (1) cpd00003[0] + (1) cpd00009[0]',\n",
       "  'id': 'rxn00088',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30204;rxn30737;rxn32045;rxn38392;rxn38393',\n",
       "  'name': \"NADP+ 2'-phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-BIOSYNTHESIS-II (NAD salvage pathway III (to nicotinamide riboside)); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); NADPHOS-DEPHOS-PWY (NAD phosphorylation and dephosphorylation); PWY-5083 (NAD/NADH phosphorylation and dephosphorylation)',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";1:cpd00003:0:0:\"NAD\";1:cpd00009:0:0:\"Phosphate\"'},\n",
       " {'abbreviation': 'R00119',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00119', 'Name: NADP+ glycohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] <=> (1) cpd00133[0] + (1) cpd02079[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00006;cpd00067;cpd00133;cpd02079',\n",
       "  'definition': \"(1) H2O[0] + (1) NADP[0] => (1) H+[0] + (1) Nicotinamide[0] + (1) ADPribose 2'-phosphate[0]\",\n",
       "  'deltag': -3.67,\n",
       "  'deltagerr': 5.98,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.2.5', '3.2.2.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] => (1) cpd00067[0] + (1) cpd00133[0] + (1) cpd02079[0]',\n",
       "  'id': 'rxn00089',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NADP+ glycohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";1:cpd00067:0:0:H+;1:cpd00133:0:0:\"Nicotinamide\";1:cpd02079:0:0:\"ADPribose 2\\'-phosphate\"'},\n",
       " {'abbreviation': 'R00120',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00120',\n",
       "   'MetaCyc: GLUTATHIONE-OXIDASE-RXN',\n",
       "   'Name: Glutathione:oxygen oxidoreductase; glutathione oxidase; glutathione:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (2) cpd00042[0] <=> (1) cpd00025[0] + (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00042;cpd00111',\n",
       "  'definition': '(1) O2[0] + (2) GSH[0] => (1) H2O2[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': -28.02,\n",
       "  'deltagerr': 1.04,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.8.3.3'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00042[0] => (1) cpd00025[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00090',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glutathione:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00042:0:0:\"GSH\";1:cpd00025:0:0:\"H2O2\";1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'rxn00091',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (2) cpd00074[0] <=> (1) cpd00081[0] + (1) cpd00239[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00074;cpd00081;cpd00239',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (2) S[0] <=> (1) Sulfite[0] + (1) H2S[0]',\n",
       "  'deltag': -70.71,\n",
       "  'deltagerr': 1.25,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.13.99.-'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (2) cpd00074[0] <=> (1) cpd00081[0] + (1) cpd00239[0]',\n",
       "  'id': 'rxn00091',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00091',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:-2',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-2:cpd00074:0:0:\"S\";1:cpd00081:0:0:\"Sulfite\";1:cpd00239:0:0:\"H2S\"'},\n",
       " {'abbreviation': 'R00122',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-10862',\n",
       "   'BiGG: NDP1',\n",
       "   'CornCyc: RXN-10862',\n",
       "   'KEGG: R00122',\n",
       "   'MetaCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-10862',\n",
       "   'Name: ADP phosphohydrolase; nucleoside-diphosphatase (ADP)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00008[0] <=> (1) cpd00009[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00008;cpd00009;cpd00018;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) ADP[0] => (1) Phosphate[0] + (1) AMP[0] + (2) H+[0]',\n",
       "  'deltag': -5.74,\n",
       "  'deltagerr': 0.08,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00008[0] => (1) cpd00009[0] + (1) cpd00018[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00092',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33337',\n",
       "  'name': 'ADP phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00123',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00123',\n",
       "   'MetaCyc: ADP-DEAMINASE-RXN',\n",
       "   'Name: ADP aminohydrolase; ADP deaminase; adenosine diphosphate deaminase; adenosinepyrophosphate deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00008[0] <=> (1) cpd00013[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00008;cpd00013;cpd00067;cpd00090',\n",
       "  'definition': '(1) H2O[0] + (1) ADP[0] + (1) H+[0] => (1) NH3[0] + (1) IDP[0]',\n",
       "  'deltag': -14.62,\n",
       "  'deltagerr': 0.93,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.17', '3.5.4.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00008[0] + (1) cpd00067[0] => (1) cpd00013[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00093',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ADP aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00008:0:0:\"ADP\";-1:cpd00067:0:0:\"H+\";1:cpd00013:0:0:\"NH3\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00125',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AP4AH; AP4AHr',\n",
       "   'EcoCyc: 3.6.1.41-RXN',\n",
       "   'JP_Creinhardtii_MSB: R_BTPS',\n",
       "   'KEGG: R00125',\n",
       "   'MetaCyc: 3.6.1.41-RXN',\n",
       "   \"Name: 1-P,4-P-bis(5'-nucleosyl)-tetraphosphate nucleosidebisphosphohydrolase; Ap4A hydrolase; Ap4A hydrolase, reversible; P1,P4-Bis(5'-nucleosyl)-tetraphosphate nucleotidebisphosphohydrolase; P1,P4-bis(5'-adenosyl)-tetraphosphate nucleotidebisphosphohydrolase; P1,P4-bis(5'-nucleosyl)-tetraphosphate nucleosidebisphosphohydrolase; adenosine tetraphosphate phosphodiesterase; bis(5'-adenosyl) tetraphosphatase; bis(5'-nucleosyl)-tetraphosphatase (symmetrical); diadenosine 5',5'''-P1,P4-tetraphosphatase; diadenosine polyphosphate hydrolase; diadenosine tetraphosphate hydrolase; diadenosinetetraphosphatase (symmetrical); dinucleosidetetraphosphatasee (symmetrical); symmetrical diadenosine tetraphosphate hydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00924[0] <=> (2) cpd00008[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00008;cpd00924',\n",
       "  'definition': '(1) H2O[0] + (1) AppppA[0] => (2) ADP[0]',\n",
       "  'deltag': -7.85,\n",
       "  'deltagerr': 1.99,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.41'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00924[0] => (2) cpd00008[0]',\n",
       "  'id': 'rxn00095',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35438',\n",
       "  'name': \"P1,P4-bis(5'-adenosyl)-tetraphosphate nucleotidebisphosphohydrolase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00924:0:0:\"AppppA\";2:cpd00008:0:0:\"ADP\"'},\n",
       " {'abbreviation': 'R00126',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00126_c',\n",
       "   'BiGG: APA2; ATPATF1',\n",
       "   'ChlamyCyc: ATP-ADENYLYLTRANSFERASE-RXN',\n",
       "   'KEGG: R00126',\n",
       "   'MetaCyc: ATP-ADENYLYLTRANSFERASE-RXN',\n",
       "   \"Name: ADP:ATP adenylyltransferase; ATP adenylyltransferase; adenine triphosphate adenylyltransferase; bis(5'-nucleosyl)-tetraphosphate phosphorylase (NDP-forming); diadenosine 5',5'''-P1,P4-tetraphosphate alphabeta-phosphorylase (ADP-forming); diadenosinetetraphosphate alphabeta-phosphorylase; dinucleoside oligophosphate alphabeta-phosphorylase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00008[0] <=> (1) cpd00009[0] + (1) cpd00924[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00067;cpd00924',\n",
       "  'definition': '(1) ATP[0] + (1) ADP[0] <= (1) Phosphate[0] + (1) H+[0] + (1) AppppA[0]',\n",
       "  'deltag': 1.69,\n",
       "  'deltagerr': 1.99,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.7.7.53'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00008[0] <= (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00924[0]',\n",
       "  'id': 'rxn00096',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35428',\n",
       "  'name': 'ADP:ATP adenylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00924:0:0:\"AppppA\"'},\n",
       " {'abbreviation': 'R00127',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENYL-KIN-RXN; RXN4FS-19',\n",
       "   'BiGG: ADK1; ADK1_1; ADK1m; ADK2_1',\n",
       "   'BrachyCyc: ADENYL-KIN-RXN',\n",
       "   'KEGG: R00127',\n",
       "   'MetaCyc: ADENYL-KIN-RXN',\n",
       "   \"Name: 5'-AMP-kinase; ATP:AMP phosphotransferase; Adenylate kinase; Adenylate kinase 2; Adenylic kinase; Adenylokinase; Myokinase; adenylate kinase; adenylatekinase; adenylic kinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00018[0] <=> (2) cpd00008[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00018;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) AMP[0] + (1) H+[0] <=> (2) ADP[0]',\n",
       "  'deltag': -0.42,\n",
       "  'deltagerr': 0.04,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00018[0] + (1) cpd00067[0] <=> (2) cpd00008[0]',\n",
       "  'id': 'rxn00097',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27684;rxn27685',\n",
       "  'name': 'ATP:AMP phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6126 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis II); PWY-7219 (adenosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7229 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00018:0:0:\"AMP\";-1:cpd00067:0:0:\"H+\";2:cpd00008:0:0:\"ADP\"'},\n",
       " {'abbreviation': 'R00128',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00128',\n",
       "   'MetaCyc: ADENOSINE-TETRAPHOSPHATASE-RXN',\n",
       "   'Name: Adenosine-tetraphosphate phosphohydrolase; adenosine-tetraphosphatase; adenosine-tetraphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02198[0] <=> (1) cpd00002[0] + (1) cpd00009[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00009;cpd00067;cpd02198',\n",
       "  'definition': '(1) H2O[0] + (1) Adenosine tetraphosphate[0] => (1) ATP[0] + (1) Phosphate[0] + (1) H+[0]',\n",
       "  'deltag': -7.49,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.14'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02198[0] => (1) cpd00002[0] + (1) cpd00009[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00098',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Adenosine-tetraphosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02198:0:0:\"Adenosine tetraphosphate\";1:cpd00002:0:0:\"ATP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00129',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00129',\n",
       "   'MetaCyc: 4.2.1.93-RXN',\n",
       "   'Name: (6S)-6-Hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase (ATP-hydrolysing); (6S)-6-beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase (ATP-hydrolysing; (6S)-6beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine dinucleotide hydro-lyase (ATP-hydrolysing, NADH-forming); (6S)-6beta-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase (ATP-hydrolysing, NADH-forming); (6S)-beta-6-Hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-; (6S)-beta-6-hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase(ATP-hydrolysing); ATP-dependent H4NAD(P)+OH dehydratase; ATP-dependent NAD(P)H-hydrate dehydratase; NADH-forming); dinucleotide hydro-lyase (ATP-hydrolysing); reduced nicotinamide adenine dinucleotide hydrate dehydratase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd02951[0] <=> (1) cpd00004[0] + (1) cpd00008[0] + (1) cpd00009[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00004;cpd00008;cpd00009;cpd00067;cpd02951',\n",
       "  'definition': '(1) ATP[0] + (1) (6S)-6-beta-Hydroxy-1,4,5,6-tetrahydronicotinamide-adenine dinucleotide[0] => (1) NADH[0] + (1) ADP[0] + (1) Phosphate[0] + (1) H+[0]',\n",
       "  'deltag': -11.69,\n",
       "  'deltagerr': 0.9,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.93'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd02951[0] => (1) cpd00004[0] + (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00099',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(6S)-6-Hydroxy-1,4,5,6-tetrahydronicotinamide-adenine-dinucleotide hydro-lyase (ATP-hydrolysing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-6938 (NADH repair)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd02951:0:0:\"(6S)-6-beta-Hydroxy-1,4,5,6-tetrahydronicotinamide-adenine dinucleotide\";1:cpd00004:0:0:\"NADH\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00130',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DEPHOSPHOCOAKIN-RXN',\n",
       "   'BiGG: DPCOAK; DPCOAKm; U96; U97',\n",
       "   'BrachyCyc: DEPHOSPHOCOAKIN-RXN',\n",
       "   'KEGG: R00130',\n",
       "   'MetaCyc: DEPHOSPHOCOAKIN-RXN',\n",
       "   \"Name: 3'-dephospho-CoA kinase; ATP:3'-dephospho-CoA 3'-phosphotransferase; ATP:dephospho-CoA 3'-phosphotransferase; Dephospho-CoA kinase; Dephosphocoa kinase; dephospho-CoA kinase; dephospho-CoAkinase; dephosphocoenzyme A kinase; dephosphocoenzyme A kinase (phosphorylating)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00655[0] <=> (1) cpd00008[0] + (1) cpd00010[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00010;cpd00067;cpd00655',\n",
       "  'definition': '(1) ATP[0] + (1) Dephospho-CoA[0] => (1) ADP[0] + (1) CoA[0] + (1) H+[0]',\n",
       "  'deltag': 0.57,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.24'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00655[0] => (1) cpd00008[0] + (1) cpd00010[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00100',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30214;rxn33067',\n",
       "  'name': \"ATP:dephospho-CoA 3'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: COA-PWY (coenzyme A biosynthesis I (prokaryotic)); COA-PWY-1 (superpathway of coenzyme A biosynthesis III (mammals)); CoA-Biosynthesis (Coenzyme A Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PANTOSYN-PWY (superpathway of coenzyme A biosynthesis I (bacteria)); PWY-4221 (superpathway of coenzyme A biosynthesis II (plants)); PWY-7851 (coenzyme A biosynthesis II (eukaryotic)); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00655:0:0:\"Dephospho-CoA\";1:cpd00008:0:0:\"ADP\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00131',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UREASE-RXN',\n",
       "   'BiGG: UREA; UREA2',\n",
       "   'BrachyCyc: UREASE-RXN',\n",
       "   'KEGG: R00131',\n",
       "   'MetaCyc: UREASE-RXN',\n",
       "   'Name: Urea amidohydrolase; urea amidohydrolase; urease'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00073[0] <=> (1) cpd00011[0] + (2) cpd00013[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00067;cpd00073',\n",
       "  'definition': '(1) H2O[0] + (2) H+[0] + (1) Urea[0] => (1) CO2[0] + (2) NH3[0]',\n",
       "  'deltag': -7.11,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00067[0] + (1) cpd00073[0] => (1) cpd00011[0] + (2) cpd00013[0]',\n",
       "  'id': 'rxn00101',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30324',\n",
       "  'name': 'Urea amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Allantoin-degradation (Allantoin Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5704 (urea degradation II); URDEGR-PWY (superpathway of allantoin degradation in plants); Urea-Degradation (Urea Degradation)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn00330 (Arginine and proline metabolism); rn00791 (Atrazine degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00067:0:0:\"H+\";-1:cpd00073:0:0:\"Urea\";1:cpd00011:0:0:\"CO2\";2:cpd00013:0:0:\"NH3\"'},\n",
       " {'abbreviation': 'R00132',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CARBODEHYDRAT-RXN; RXN0-5224',\n",
       "   'BiGG: H2CO3D; HCO3E; HCO3Em; HCO3En',\n",
       "   'BrachyCyc: CARBODEHYDRAT-RXN; RXN0-5224',\n",
       "   'KEGG: R00132; R10092',\n",
       "   'MetaCyc: CARBODEHYDRAT-RXN; RXN0-5224',\n",
       "   'Name: Carbonic acid hydro-lyase; HCO3 equilibration reaction; HCO3 equilibrium reaction; HCO3equilibrationreaction; anhydrase; carbonate anhydrase; carbonate dehydratase; carbonate dehydratase (HCO3 equilibration reaction); carbonate hydro-lyase; carbonate hydro-lyase (carbon-dioxide-forming); carbonic acid anhydrase; carbonic acid hydro-lyase; carbonic anhydrase; carbonic anhydrase A; carbonic dehydratase; carboxyanhydrase; carboxylic acid dissociation; carboxylicaciddissociation; dissolution of carbon dioxide'],\n",
       "  'code': '(1) cpd00242[0] <=> (1) cpd00001[0] + (1) cpd00011[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00067;cpd00242',\n",
       "  'definition': '(1) H+[0] + (1) H2CO3[0] <=> (1) H2O[0] + (1) CO2[0]',\n",
       "  'deltag': 1.26,\n",
       "  'deltagerr': 0.75,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.1'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00242[0] <=> (1) cpd00001[0] + (1) cpd00011[0]',\n",
       "  'id': 'rxn00102',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn26146;rxn27740;rxn27741;rxn30392;rxn37640',\n",
       "  'name': 'carbonate hydro-lyase (carbon-dioxide-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CO2-Fixation (CO<sub>2</sub> Fixation); CYANCAT-PWY (cyanate degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Gluconeogenesis (Gluconeogenesis); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-241 (C4 photosynthetic carbon assimilation cycle, NADP-ME type); PWY-5743 (3-hydroxypropanoate cycle); PWY-5744 (glyoxylate assimilation); PWY-5789 (3-hydroxypropanoate/4-hydroxybutanate cycle); PWY-6142 (gluconeogenesis II (<i>Methanobacterium thermoautotrophicum</i>)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle); PWY-7115 (C4 photosynthetic carbon assimilation cycle, NAD-ME type); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); PWYQT-4429 (CO<sub>2</sub> fixation into oxaloacetate (anaplerotic)); Photosynthesis ()',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00242:0:0:\"H2CO3\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\"'},\n",
       " {'abbreviation': 'R00134',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FORMATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'KEGG: R00134',\n",
       "   'MetaCyc: FORMATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'PlantCyc: FORMATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'RiceCyc: FORMATE-DEHYDROGENASE-(NADP+)-RXN',\n",
       "   'Name: Formate dehydrogenase (NADP+); Formate:NADP+ oxidoreductase; NADP-dependent formate dehydrogenase; formate dehydrogenase (NADP); formate dehydrogenase (NADP+); formate:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00047[0] <=> (1) cpd00005[0] + (1) cpd00011[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00011;cpd00047',\n",
       "  'definition': '(1) NADP[0] + (1) Formate[0] => (1) NADPH[0] + (1) CO2[0]',\n",
       "  'deltag': -3.53,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.17.1.10', '1.2.1.43'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00047[0] => (1) cpd00005[0] + (1) cpd00011[0]',\n",
       "  'id': 'rxn00103',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33750',\n",
       "  'name': 'formate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CHLORINATED-COMPOUNDS-DEG (Chlorinated Compound Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); CODH-PWY (reductive acetyl coenzyme A pathway I (homoacetogenic bacteria)); Carbon-tetrachloride-degradation (Carbon Tetrachloride Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5372 (carbon tetrachloride degradation II); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism)',\n",
       "   'KEGG: rn00680 (Methane metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00047:0:0:\"Formate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\"'},\n",
       " {'abbreviation': 'PPK2r',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PPIK; PPK2; PPK2r',\n",
       "   'iAF1260: PPK2r',\n",
       "   'iAF692: PPK2',\n",
       "   'iJN746: PPK2r',\n",
       "   'iMA945: PPK2r',\n",
       "   'Name: polyphosphate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00012[0] <=> (1) cpd00008[0] + (1) cpd00421[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00012;cpd00421',\n",
       "  'definition': '(1) ATP[0] + (1) PPi[0] <=> (1) ADP[0] + (1) Triphosphate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00012[0] <=> (1) cpd00008[0] + (1) cpd00421[0]',\n",
       "  'id': 'rxn00104',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'polyphosphate kinase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Secondary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00012:0:0:\"PPi\";1:cpd00008:0:0:\"ADP\";1:cpd00421:0:0:\"Triphosphate\"'},\n",
       " {'abbreviation': 'R00137',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.7.1-RXN',\n",
       "   'BiGG: NMNAT; NMNATm; NMNATr; YLR328W_3',\n",
       "   'BrachyCyc: 2.7.7.1-RXN',\n",
       "   'KEGG: R00137',\n",
       "   'MetaCyc: 2.7.7.1-RXN',\n",
       "   'Name: ATP:NMN adenylyltransferase; ATP:nicotinamide-nucleotide adenylyltransferase; NAD+ pyrophosphorylase; NMN adenylyltransferase; Nicotinamide-nucleotide adenylyltransferase; Nicotinamide-nucleotide adenylyltransferase 1; adenosine triphosphate-nicotinamide mononucleotide transadenylase; diphosphopyridine nucleotide pyrophosphorylase; nicotinamide adenine dinucleotide pyrophosphorylase; nicotinamide mononucleotide adenylyltransferase; nicotinamide-nucleotide adenylyltransferase; nicotinamide-nucleotideadenylyltransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00355[0] <=> (1) cpd00003[0] + (1) cpd00012[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00003;cpd00012;cpd00355',\n",
       "  'definition': '(1) ATP[0] + (1) Nicotinamide ribonucleotide[0] <=> (1) NAD[0] + (1) PPi[0]',\n",
       "  'deltag': 0.36,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.1', '2.7.7.18'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00355[0] <=> (1) cpd00003[0] + (1) cpd00012[0]',\n",
       "  'id': 'rxn00105',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30218;rxn31803;rxn31815;rxn31816;rxn35311',\n",
       "  'name': 'ATP:nicotinamide-nucleotide adenylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-BIOSYNTHESIS-III (NAD biosynthesis III (from nicotinamide)); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); PWY-3502 (superpathway of NAD biosynthesis in eukaryotes); PWY3O-4106 (NAD salvage pathway IV (from nicotinamide riboside))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00355:0:0:\"Nicotinamide ribonucleotide\";1:cpd00003:0:0:\"NAD\";1:cpd00012:0:0:\"PPi\"'},\n",
       " {'abbreviation': 'R00138',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PPA2',\n",
       "   'KEGG: R00138',\n",
       "   'MetaCyc: TRIPHOSPHATASE-RXN',\n",
       "   'iAF1260: PPA2',\n",
       "   'iIN800: PPA2',\n",
       "   'Name: Triphosphate phosphohydrolase; inorganic triphosphatase; triphosphatase; triphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00421[0] <=> (1) cpd00009[0] + (1) cpd00012[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00012;cpd00067;cpd00421',\n",
       "  'definition': '(1) H2O[0] + (1) Triphosphate[0] <=> (1) Phosphate[0] + (1) PPi[0] + (1) H+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.6.1.1', '3.6.1.25'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00421[0] <=> (1) cpd00009[0] + (1) cpd00012[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00106',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Triphosphate phosphohydrolase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00421:0:0:\"Triphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00139',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00139',\n",
       "   \"Name: ATP:2'-deoxy-5-hydroxymethylcytidine-5'-diphosphate phosphotransferase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd07923[0] <=> (1) cpd00008[0] + (1) cpd07924[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd07923;cpd07924',\n",
       "  'definition': \"(1) ATP[0] + (1) 2'-Deoxy-5-hydroxymethylcytidine-5'-diphosphate[0] <=> (1) ADP[0] + (1) 2'-Deoxy-5-hydroxymethylcytidine-5'-triphosphate[0]\",\n",
       "  'deltag': 0.62,\n",
       "  'deltagerr': 0.3,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd07923[0] <=> (1) cpd00008[0] + (1) cpd07924[0]',\n",
       "  'id': 'rxn00107',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"ATP:2'-deoxy-5-hydroxymethylcytidine-5'-diphosphate phosphotransferase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd07923:0:0:\"2\\'-Deoxy-5-hydroxymethylcytidine-5\\'-diphosphate\";1:cpd00008:0:0:\"ADP\";1:cpd07924:0:0:\"2\\'-Deoxy-5-hydroxymethylcytidine-5\\'-triphosphate\"'},\n",
       " {'abbreviation': 'R00140',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00140',\n",
       "   'Name: ATP:5-hydroxymethyldeoxycytidylate phosphotransferase'],\n",
       "  'code': '(1) cpd00008[0] + (1) cpd07923[0] <=> (1) cpd00002[0] + (1) cpd02480[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd02480;cpd07923',\n",
       "  'definition': \"(1) ADP[0] + (1) 2'-Deoxy-5-hydroxymethylcytidine-5'-diphosphate[0] <=> (1) ATP[0] + (1) H+[0] + (1) 5-Hydroxymethyldeoxycytidylate[0]\",\n",
       "  'deltag': -0.23,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.12'],\n",
       "  'equation': '(1) cpd00008[0] + (1) cpd07923[0] <=> (1) cpd00002[0] + (1) cpd00067[0] + (1) cpd02480[0]',\n",
       "  'id': 'rxn00108',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:5-hydroxymethyldeoxycytidylate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00008:0:0:\"ADP\";-1:cpd07923:0:0:\"2\\'-Deoxy-5-hydroxymethylcytidine-5\\'-diphosphate\";1:cpd00002:0:0:\"ATP\";1:cpd00067:0:0:\"H+\";1:cpd02480:0:0:\"5-Hydroxymethyldeoxycytidylate\"'},\n",
       " {'abbreviation': 'R00143',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: HYDROXYLAMINE-REDUCTASE-NADH-RXN',\n",
       "   'KEGG: R00143',\n",
       "   'MetaCyc: HYDROXYLAMINE-REDUCTASE-NADH-RXN',\n",
       "   'Name: Ammonia:(acceptor) oxidoreductase; Hydroxylamine reductase (NADH); N-hydroxy amine reductase; NADH-hydroxylamine reductase; NADH2:hydroxylamine oxidoreductase; NADH:hydroxylamine oxidoreductase; ammonia:NAD+ oxidoreductase; ammonium dehydrogenase; ammonium:NAD+ oxidoreductase; hydroxylamine reductase; hydroxylamine reductase (NADH); hydroxylamine reductase (NADH2)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00013[0] <=> (1) cpd00004[0] + (1) cpd00165[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00067;cpd00165',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) NH3[0] <= (1) NADH[0] + (2) H+[0] + (1) Hydroxylamine[0]',\n",
       "  'deltag': 22.97,\n",
       "  'deltagerr': 6.2,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.7.1.10', '1.7.99.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00013[0] <= (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00165[0]',\n",
       "  'id': 'rxn00109',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27936',\n",
       "  'name': 'ammonia:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Macromolecule-Modification (Macromolecule Modification); PWY-7798 (protein <i>S</i>-nitrosylation and denitrosylation); Protein-Modification (Protein Modification)',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00013:0:0:\"NH3\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd00165:0:0:\"Hydroxylamine\"'},\n",
       " {'abbreviation': 'deaminating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Nitrogenous compounds,NADH:oxygen oxidoreductase (3-hydroxylating,; deaminating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd03612[0] <=> (1) cpd00003[0] + (1) cpd00013[0] + (1) cpd00049[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00007;cpd00013;cpd00049;cpd03612',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (1) Nitrogenous compounds[0] <=> (1) NAD[0] + (1) NH3[0] + (1) Carboxylic acid[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.13.35'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd03612[0] <=> (1) cpd00003[0] + (1) cpd00013[0] + (1) cpd00049[0]',\n",
       "  'id': 'rxn00110',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nitrogenous compounds,NADH:oxygen oxidoreductase (3-hydroxylating,',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:C:1/H:1|CI:1',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-1:cpd03612:0:0:\"Nitrogenous compounds\";1:cpd00003:0:0:\"NAD\";1:cpd00013:0:0:\"NH3\";1:cpd00049:0:0:\"Carboxylic acid\"'},\n",
       " {'abbreviation': 'deaminating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Nitrogenous compounds,NADPH:oxygen oxidoreductase (3-hydroxylating,; deaminating)'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd03612[0] <=> (1) cpd00006[0] + (1) cpd00013[0] + (1) cpd00049[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00007;cpd00013;cpd00049;cpd03612',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) Nitrogenous compounds[0] <=> (1) NADP[0] + (1) NH3[0] + (1) Carboxylic acid[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.13.35'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd03612[0] <=> (1) cpd00006[0] + (1) cpd00013[0] + (1) cpd00049[0]',\n",
       "  'id': 'rxn00111',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nitrogenous compounds,NADPH:oxygen oxidoreductase (3-hydroxylating,',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:C:1/H:1|CI:1',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd03612:0:0:\"Nitrogenous compounds\";1:cpd00006:0:0:\"NADP\";1:cpd00013:0:0:\"NH3\";1:cpd00049:0:0:\"Carboxylic acid\"'},\n",
       " {'abbreviation': 'rxn00112',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00013[0] <=> (1) cpd00001[0] + (1) cpd00165[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00067;cpd00165',\n",
       "  'definition': '(1) O2[0] + (1) NH3[0] + (1) H+[0] <=> (1) H2O[0] + (1) Hydroxylamine[0]',\n",
       "  'deltag': -71.01,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.13.12.-'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00013[0] + (1) cpd00067[0] <=> (1) cpd00001[0] + (1) cpd00165[0]',\n",
       "  'id': 'rxn00112',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00112',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:-2',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00013:0:0:\"NH3\";-1:cpd00067:0:0:\"H+\";1:cpd00001:0:0:\"H2O\";1:cpd00165:0:0:\"Hydroxylamine\"'},\n",
       " {'abbreviation': 'R00149',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 6.3.4.16-RXN',\n",
       "   'BrachyCyc: 6.3.4.16-RXN',\n",
       "   'ChlamyCyc: 6.3.4.16-RXN',\n",
       "   'KEGG: R00149',\n",
       "   'MetaCyc: 6.3.4.16-RXN',\n",
       "   'Name: (ADP-forming,carbamate-phosphorylating); Carbon-dioxide:ammonia ligase; Carbon-dioxide:ammonia ligase (ADP-forming,carbamate-phosphorylating); carbamoyl-phosphate synthase (ammonia); carbamoylphosphate synthase; carbamoylphosphate synthase (ammonia); carbamoylphosphate synthetase; carbamylphosphate synthetase; carbamylphosphate synthetase I; carbon-dioxide-ammonia ligase; carbon-dioxide:ammonia ligase (ADP-forming, carbamate-phosphorylating)'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00002[0] + (1) cpd00011[0] + (1) cpd00013[0] <=> (2) cpd00008[0] + (1) cpd00009[0] + (1) cpd00146[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00008;cpd00009;cpd00011;cpd00013;cpd00067;cpd00146',\n",
       "  'definition': '(1) H2O[0] + (2) ATP[0] + (1) CO2[0] + (1) NH3[0] => (2) ADP[0] + (1) Phosphate[0] + (3) H+[0] + (1) Carbamoylphosphate[0]',\n",
       "  'deltag': 0.14,\n",
       "  'deltagerr': 0.96,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.4.16', '6.3.4.16-'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00002[0] + (1) cpd00011[0] + (1) cpd00013[0] => (2) cpd00008[0] + (1) cpd00009[0] + (3) cpd00067[0] + (1) cpd00146[0]',\n",
       "  'id': 'rxn00113',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19010;rxn30592;rxn31319;rxn35314;rxn38000',\n",
       "  'name': 'Carbon-dioxide:ammonia ligase (ADP-forming,carbamate-phosphorylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism); rn00910 (Nitrogen metabolism); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00002:0:0:\"ATP\";-1:cpd00011:0:0:\"CO2\";-1:cpd00013:0:0:\"NH3\";2:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";3:cpd00067:0:0:\"H+\";1:cpd00146:0:0:\"Carbamoylphosphate\"'},\n",
       " {'abbreviation': 'R00150',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CARBAMATE-KINASE-RXN',\n",
       "   'BiGG: CBMK; CBMK2; CBMKr',\n",
       "   'BrachyCyc: CARBAMATE-KINASE-RXN',\n",
       "   'KEGG: R00150',\n",
       "   'MetaCyc: CARBAMATE-KINASE-RXN',\n",
       "   'Name: ATP:carbamate phosphotransferase; CKase; Carbamate kinase; carbamate kinase; carbamoyl phosphokinase; carbamyl phosphokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00011[0] + (1) cpd00013[0] <=> (1) cpd00008[0] + (1) cpd00146[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00011;cpd00013;cpd00067;cpd00146',\n",
       "  'definition': '(1) ATP[0] + (1) CO2[0] + (1) NH3[0] <= (1) ADP[0] + (2) H+[0] + (1) Carbamoylphosphate[0]',\n",
       "  'deltag': 6.3,\n",
       "  'deltagerr': 0.96,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.7.2.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00011[0] + (1) cpd00013[0] <= (1) cpd00008[0] + (2) cpd00067[0] + (1) cpd00146[0]',\n",
       "  'id': 'rxn00114',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29996;rxn37674',\n",
       "  'name': 'ATP:carbamate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism); rn00330 (Arginine and proline metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00011:0:0:\"CO2\";-1:cpd00013:0:0:\"NH3\";1:cpd00008:0:0:\"ADP\";2:cpd00067:0:0:\"H+\";1:cpd00146:0:0:\"Carbamoylphosphate\"'},\n",
       " {'abbreviation': 'R00152',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00152', 'iRS1563: R_R00152_c', 'Name: '],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00013[0] <=> (1) cpd00150[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00013;cpd00150',\n",
       "  'definition': '(1) CO2[0] + (1) NH3[0] <=> (1) HCN[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00013[0] <=> (1) cpd00150[0]',\n",
       "  'id': 'rxn00115',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R00152',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00460 (Cyanoamino acid metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:H:-3/O:-2|CI:-1',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00013:0:0:\"NH3\";1:cpd00150:0:0:\"HCN\"'},\n",
       " {'abbreviation': 'R00155',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-12197',\n",
       "   'BiGG: NDP7',\n",
       "   'CornCyc: RXN-12197',\n",
       "   'KEGG: R00155',\n",
       "   'MetaCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-12197',\n",
       "   'Name: UDP diphosphohydrolase; UDP phosphohydrolase; nucleoside-diphosphatase (UDP); nucleoside-diphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00014[0] <=> (1) cpd00009[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00014;cpd00067;cpd00091',\n",
       "  'definition': '(1) H2O[0] + (1) UDP[0] => (1) Phosphate[0] + (2) H+[0] + (1) UMP[0]',\n",
       "  'deltag': -5.03,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.5', '3.6.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00014[0] => (1) cpd00009[0] + (2) cpd00067[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00116',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00014:0:0:\"UDP\";1:cpd00009:0:0:\"Phosphate\";2:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00156',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-DIP-KIN-RXN; UDPKIN-RXN',\n",
       "   'BiGG: NDPK2; YNK1_1',\n",
       "   'BrachyCyc: NUCLEOSIDE-DIP-KIN-RXN; UDPKIN-RXN',\n",
       "   'KEGG: R00156',\n",
       "   'MetaCyc: NUCLEOSIDE-DIP-KIN-RXN; UDPKIN-RXN',\n",
       "   \"Name: ATP:UDP phosphotransferase; NDK; Nucleoside 5'-diphosphate phosphotransferase; Nucleoside diphosphokinase; Nucleoside-diphosphate kinase; nucleoside-diphosphate kinase (ATP:UDP); nucleoside-diphosphatekinase-ATP-UDP; udp kinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00014[0] <=> (1) cpd00008[0] + (1) cpd00062[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00014;cpd00062',\n",
       "  'definition': '(1) ATP[0] + (1) UDP[0] <=> (1) ADP[0] + (1) UTP[0]',\n",
       "  'deltag': -0.68,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00014[0] <=> (1) cpd00008[0] + (1) cpd00062[0]',\n",
       "  'id': 'rxn00117',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20461;rxn28042;rxn28044;rxn33550',\n",
       "  'name': 'ATP:UDP phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-7176 (UTP and CTP <i>de novo</i> biosynthesis); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PWY-7208 (superpathway of pyrimidine nucleobases salvage); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY0-162 (superpathway of pyrimidine ribonucleotides <i>de novo</i> biosynthesis); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimid-Ribonucleot-De-Novo-Biosyn (Pyrimidine Ribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00014:0:0:\"UDP\";1:cpd00008:0:0:\"ADP\";1:cpd00062:0:0:\"UTP\"'},\n",
       " {'abbreviation': 'R00157',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00157_c',\n",
       "   'AraGEM: R_R00157_c',\n",
       "   'KEGG: R00157',\n",
       "   'iRS1597: R00157_c',\n",
       "   'Name: UTP:AMP phosphotransferase; Uridine triphosphate:AMP phosphotransferase'],\n",
       "  'code': '(1) cpd00018[0] + (1) cpd00062[0] <=> (1) cpd00008[0] + (1) cpd00014[0]',\n",
       "  'compound_ids': 'cpd00008;cpd00014;cpd00018;cpd00062;cpd00067',\n",
       "  'definition': '(1) AMP[0] + (1) UTP[0] + (1) H+[0] <=> (1) ADP[0] + (1) UDP[0]',\n",
       "  'deltag': 0.25,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.10'],\n",
       "  'equation': '(1) cpd00018[0] + (1) cpd00062[0] + (1) cpd00067[0] <=> (1) cpd00008[0] + (1) cpd00014[0]',\n",
       "  'id': 'rxn00118',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Uridine triphosphate:AMP phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00018:0:0:\"AMP\";-1:cpd00062:0:0:\"UTP\";-1:cpd00067:0:0:\"H+\";1:cpd00008:0:0:\"ADP\";1:cpd00014:0:0:\"UDP\"'},\n",
       " {'abbreviation': 'R00158',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.4.22-RXN; RXN-12002; UMPKI-RXN',\n",
       "   'BiGG: UMPK; UMPKURIDK1; UMPKn; URA6_1; URA6_2; URIDK1',\n",
       "   'BrachyCyc: 2.7.4.22-RXN; UMPKI-RXN',\n",
       "   'KEGG: R00158',\n",
       "   'MetaCyc: 2.7.4.22-RXN; RXN-12002',\n",
       "   'Name: ATP:CMP(UMP) phosphotransferase; ATP:UMP phosphotransferase; ATP:UMP-CMP phosphotransferase; CTP:CMP phosphotransferase; UMP kinase; UMP-CMP kinase; UMP-kinase; UMP/CMP kinase; UMPkinase; cytidylate kinase; dCMP kinase; deoxycytidine monophosphokinase; deoxycytidylate kinase; pyrimidine nucleoside monophosphate kinase; uridine monophosphate-cytidine monophosphate phosphotransferase; uridylate kinase (UMP)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00091[0] <=> (1) cpd00008[0] + (1) cpd00014[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00014;cpd00067;cpd00091',\n",
       "  'definition': '(1) ATP[0] + (1) H+[0] + (1) UMP[0] <=> (1) ADP[0] + (1) UDP[0]',\n",
       "  'deltag': -1.13,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.14', '2.7.4.22', '2.7.4.4'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00067[0] + (1) cpd00091[0] <=> (1) cpd00008[0] + (1) cpd00014[0]',\n",
       "  'id': 'rxn00119',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30446;rxn32095',\n",
       "  'name': 'ATP:UMP phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-7176 (UTP and CTP <i>de novo</i> biosynthesis); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PWY-7208 (superpathway of pyrimidine nucleobases salvage); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY0-162 (superpathway of pyrimidine ribonucleotides <i>de novo</i> biosynthesis); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimid-Ribonucleot-De-Novo-Biosyn (Pyrimidine Ribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00067:0:0:\"H+\";-1:cpd00091:0:0:\"UMP\";1:cpd00008:0:0:\"ADP\";1:cpd00014:0:0:\"UDP\"'},\n",
       " {'abbreviation': 'R00159',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN-12196',\n",
       "   'BiGG: NTP7',\n",
       "   'BrachyCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN',\n",
       "   'KEGG: R00159',\n",
       "   'MetaCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN-12196',\n",
       "   'Name: UTP phosphohydrolase; nucleoside-triphosphatase (UTP); unspecific diphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00062[0] <=> (1) cpd00009[0] + (1) cpd00014[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00014;cpd00062;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) UTP[0] => (1) Phosphate[0] + (1) UDP[0] + (1) H+[0]',\n",
       "  'deltag': -5.48,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.15', '3.6.1.39', '3.6.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00062[0] => (1) cpd00009[0] + (1) cpd00014[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00120',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38190;rxn38198;rxn38206;rxn38214;rxn38222;rxn38231;rxn38241',\n",
       "  'name': 'UTP phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00062:0:0:\"UTP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00014:0:0:\"UDP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00160',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FAD-PYROPHOSPHATASE-RXN',\n",
       "   'BrachyCyc: FAD-PYROPHOSPHATASE-RXN',\n",
       "   'CornCyc: FAD-PYROPHOSPHATASE-RXN',\n",
       "   'KEGG: R00160',\n",
       "   'MetaCyc: FAD-PYROPHOSPHATASE-RXN',\n",
       "   'Name: FAD diphosphatase; FAD nucleotidohydrolase; FAD pyrophosphatase; flavin adenine dinucleotide pyrophosphatase; flavine adenine dinucleotide pyrophosphatase; riboflavin adenine dinucleotide pyrophosphatase; riboflavine adenine dinucleotide pyrophosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00015[0] <=> (1) cpd00018[0] + (1) cpd00050[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00015;cpd00018;cpd00050;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) FAD[0] => (1) AMP[0] + (1) FMN[0] + (2) H+[0]',\n",
       "  'deltag': -5.35,\n",
       "  'deltagerr': 0.94,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.18', '3.6.1.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00015[0] => (1) cpd00018[0] + (1) cpd00050[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00121',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33436',\n",
       "  'name': 'FAD nucleotidohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00740 (Riboflavin metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00015:0:0:\"FAD\";1:cpd00018:0:0:\"AMP\";1:cpd00050:0:0:\"FMN\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00161',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FADSYN-RXN',\n",
       "   'BiGG: FAD1; FMNAT; FMNATm; FMNATr; U80',\n",
       "   'BrachyCyc: FADSYN-RXN',\n",
       "   'KEGG: R00161',\n",
       "   'MetaCyc: FADSYN-RXN',\n",
       "   'Name: ATP:FMN adenylyltransferase; FAD pyrophosphorylase; FAD synthetase; FADS; FMN adenylyltransferase; FMNadenylyltransferase; adenosine triphosphate-riboflavin mononucleotide transadenylase; adenosine triphosphate-riboflavine mononucleotide transadenylase; flavin adenine dinucleotide synthetase; riboflavin adenine dinucleotide pyrophosphorylase; riboflavin mononucleotide adenylyltransferase; riboflavine adenine dinucleotide adenylyltransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00050[0] <=> (1) cpd00012[0] + (1) cpd00015[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00015;cpd00050',\n",
       "  'definition': '(1) ATP[0] + (1) FMN[0] <=> (1) PPi[0] + (1) FAD[0]',\n",
       "  'deltag': -3.08,\n",
       "  'deltagerr': 0.94,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00050[0] <=> (1) cpd00012[0] + (1) cpd00015[0]',\n",
       "  'id': 'rxn00122',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30210;rxn33129',\n",
       "  'name': 'ATP:FMN adenylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Flavin-Biosynthesis (Flavin Biosynthesis); PWY-6167 (flavin biosynthesis II (archaea)); PWY-6168 (flavin biosynthesis III (fungi)); PWY66-366 (flavin biosynthesis IV (mammalian)); RIBOSYN2-PWY (flavin biosynthesis I (bacteria and plants)); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00740 (Riboflavin metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00050:0:0:\"FMN\";1:cpd00012:0:0:\"PPi\";1:cpd00015:0:0:\"FAD\"'},\n",
       " {'abbreviation': 'R00173',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.1.3.74-RXN',\n",
       "   'BiGG: PYDXPP',\n",
       "   'CornCyc: 3.1.3.74-RXN',\n",
       "   'KEGG: R00173',\n",
       "   'MetaCyc: 3.1.3.74-RXN',\n",
       "   \"Name: PLP phosphatase; PNP phosphatase; Pyridoxal 5-phosphate phosphatase; pyridoxal phosphatase; pyridoxal-5'-phosphate phosphohydrolase; vitamin B6-phosphate phosphatase; vitamine B6 (pyridoxine) phosphatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00016[0] <=> (1) cpd00009[0] + (1) cpd00215[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00016;cpd00215',\n",
       "  'definition': '(1) H2O[0] + (1) Pyridoxal phosphate[0] <=> (1) Phosphate[0] + (1) Pyridoxal[0]',\n",
       "  'deltag': -2.71,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.74'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00016[0] <=> (1) cpd00009[0] + (1) cpd00215[0]',\n",
       "  'id': 'rxn00123',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"pyridoxal-5'-phosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-7204 (pyridoxal 5'-phosphate salvage II (plants)); Vitamin-B6-Biosynthesis (Vitamin B<sub>6</sub> Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)\",\n",
       "   'KEGG: rn00750 (Vitamin B6 metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00016:0:0:\"Pyridoxal phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00215:0:0:\"Pyridoxal\"'},\n",
       " {'abbreviation': 'R00174',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRIDOXKIN-RXN',\n",
       "   'BiGG: PYDXK; U83',\n",
       "   'BrachyCyc: PYRIDOXKIN-RXN',\n",
       "   'KEGG: R00174',\n",
       "   'MetaCyc: PYRIDOXKIN-RXN',\n",
       "   \"Name: ATP:pyridoxal 5'-phosphotransferase; RXNBWI-11; pyridoxal 5-phosphate-kinase; pyridoxal kinase; pyridoxal kinase (Pyridoxal); pyridoxal kinase (phosphorylating); pyridoxal phosphokinase; pyridoxine kinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00215[0] <=> (1) cpd00008[0] + (1) cpd00016[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00016;cpd00067;cpd00215',\n",
       "  'definition': '(1) ATP[0] + (1) Pyridoxal[0] => (1) ADP[0] + (1) Pyridoxal phosphate[0] + (1) H+[0]',\n",
       "  'deltag': -3.45,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.35'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00215[0] => (1) cpd00008[0] + (1) cpd00016[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00124',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38691',\n",
       "  'name': \"ATP:pyridoxal 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); HMP-PP-Biosynthesis (4-amino-2-methyl-5-diphosphomethylpyrimidine biosynthesis); PLPSAL-PWY (pyridoxal 5'-phosphate salvage I); PWY-7204 (pyridoxal 5'-phosphate salvage II (plants)); PWY-7282 (4-amino-2-methyl-5-diphosphomethylpyrimidine biosynthesis (yeast)); PWY0-845 (superpathway of pyridoxal 5'-phosphate biosynthesis and salvage); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-B6-Biosynthesis (Vitamin B<sub>6</sub> Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)\",\n",
       "   'KEGG: rn00750 (Vitamin B6 metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00215:0:0:\"Pyridoxal\";1:cpd00008:0:0:\"ADP\";1:cpd00016:0:0:\"Pyridoxal phosphate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00175',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00175',\n",
       "   'MetaCyc: ADENOSYLMETHIONINE-HYDROLASE-RXN',\n",
       "   'iRS1563: R_R00175_c',\n",
       "   'Name: S-Adenosyl-L-methionine hydrolase; S-adenosyl-L-methionine hydrolase; S-adenosylmethionine cleaving enzyme; adenosylmethionine hydrolase; adenosylmethionine lyase; methylmethionine-sulfonium-salt hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00017[0] <=> (1) cpd00147[0] + (1) cpd00227[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00017;cpd00067;cpd00147;cpd00227',\n",
       "  'definition': '(1) H2O[0] + (1) S-Adenosyl-L-methionine[0] => (1) H+[0] + (1) 5-Methylthioadenosine[0] + (1) L-Homoserine[0]',\n",
       "  'deltag': -7.47,\n",
       "  'deltagerr': 4.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.3.1.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00017[0] => (1) cpd00067[0] + (1) cpd00147[0] + (1) cpd00227[0]',\n",
       "  'id': 'rxn00125',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine hydrolase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";1:cpd00067:0:0:\"H+\";1:cpd00147:0:0:\"5-Methylthioadenosine\";1:cpd00227:0:0:\"L-Homoserine\"'},\n",
       " {'abbreviation': 'R00177',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: S-ADENMETSYN-RXN',\n",
       "   'BiGG: METAT; SAM1; SAM2',\n",
       "   'BrachyCyc: S-ADENMETSYN-RXN',\n",
       "   'KEGG: R00177',\n",
       "   'MetaCyc: S-ADENMETSYN-RXN',\n",
       "   'Name: ATP-methionine adenosyltransferase; ATP:L-methione S-adenosyltransferase; ATP:L-methionine S-adenosyltransferase; AdoMet synthetase; Adomet synthetase; Methionine adenosyltransferase; S-adenosyl-L-methionine synthetase; S-adenosylmethionine synthase; S-adenosylmethionine synthetase; adenosylmethionine synthetase; methionine S-adenosyltransferase; methionine adenosyltransferase; methionine-activating enzyme; methionineadenosyltransferase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00060[0] <=> (1) cpd00009[0] + (1) cpd00012[0] + (1) cpd00017[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00009;cpd00012;cpd00017;cpd00060;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) L-Methionine[0] <= (1) Phosphate[0] + (1) PPi[0] + (1) S-Adenosyl-L-methionine[0] + (1) H+[0]',\n",
       "  'deltag': -12.61,\n",
       "  'deltagerr': 4.92,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.5.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00060[0] <= (1) cpd00009[0] + (1) cpd00012[0] + (1) cpd00017[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00126',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28979',\n",
       "  'name': 'ATP:L-methionine S-adenosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': [\"MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CYSTEINE-SYN (L-cysteine Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); ETHYL-PWY (ethylene biosynthesis I (plants)); Ethylene-Biosynthesis (Ethylene Biosynthesis); HORMONE-SYN (Hormone Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); MET-SAM-PWY (superpathway of <i>S</i>-adenosyl-L-methionine biosynthesis); METHIONINE-DEG (L-methionine Degradation); METHIONINE-DEG1-PWY (L-methionine degradation I (to L-homocysteine)); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-Salvage (L-methionine Salvage); PWY-5041 (<i>S</i>-adenosyl-L-methionine cycle II); PWY-5328 (superpathway of L-methionine salvage and degradation); PWY-5912 (2'-deoxymugineic acid phytosiderophore biosynthesis); PWY-6151 (<i>S</i>-adenosyl-L-methionine cycle I); PWY-6292 (superpathway of L-cysteine biosynthesis (mammalian)); PWY-7270 (L-methionine salvage cycle II (plants)); PWY-7527 (L-methionine salvage cycle III); PWY-7528 (L-methionine salvage cycle I (bacteria and plants)); PWY-I9 (L-cysteine biosynthesis VI (from L-methionine)); PWY0-781 (aspartate superpathway); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); S-adenosyl-L-methionine-cycle (<i>S</i>-adenosyl-L-methionine cycle); SAM-PWY (S-adenosyl-L-methionine biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Siderophores-Biosynthesis (Siderophore Biosynthesis)\",\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00060:0:0:\"L-Methionine\";1:cpd00009:0:0:\"Phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00178',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SAMDECARB-RXN',\n",
       "   'BiGG: ADMDC; ADMDCr; SPE2',\n",
       "   'BrachyCyc: SAMDECARB-RXN',\n",
       "   'KEGG: R00178',\n",
       "   'MetaCyc: SAMDECARB-RXN',\n",
       "   'Name: Adenosylmethionine decarboxylase; Adenosylmethioninedecarboxylase; S-Adenosyl-L-methionine carboxy-lyase; S-adenosyl-L-methionine carboxy-lyase; S-adenosyl-L-methionine carboxy-lyase [(5-deoxy-5-adenosyl)(3-aminopropyl)methylsulfonium-forming]; S-adenosyl-L-methionine carboxy-lyase [(5-deoxy-5-adenosyl)(3-aminopropyl)methylsulfonium-salt-forming]; S-adenosyl-L-methionine decarboxylase; S-adenosylmethionine decarboxylase; adenosylmethionine decarboxylase'],\n",
       "  'code': '(1) cpd00017[0] <=> (1) cpd00011[0] + (1) cpd00837[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00017;cpd00067;cpd00837',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) H+[0] => (1) CO2[0] + (1) S-Adenosylmethioninamine[0]',\n",
       "  'deltag': -5.47,\n",
       "  'deltagerr': 0.75,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.50'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00837[0]',\n",
       "  'id': 'rxn00127',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32132',\n",
       "  'name': 'S-adenosyl-L-methionine carboxy-lyase [(5-deoxy-5-adenosyl)(3-aminopropyl)methylsulfonium-forming]',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGSPECAT-PWY (spermine biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); BSUBPOLYAMSYN-PWY (spermidine biosynthesis I); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-Salvage (L-methionine Salvage); POLYAMINSYN3-PWY (superpathway of polyamine biosynthesis II); POLYAMSYN-PWY (superpathway of polyamine biosynthesis I); PWY-6834 (spermidine biosynthesis III); PWY-7527 (L-methionine salvage cycle III); PWY-7528 (L-methionine salvage cycle I (bacteria and plants)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Spermidine-Biosynthesis (Spermidine Biosynthesis)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00837:0:0:\"S-Adenosylmethioninamine\"'},\n",
       " {'abbreviation': 'R00179',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 4.4.1.14-RXN',\n",
       "   'BrachyCyc: 4.4.1.14-RXN',\n",
       "   'ChlamyCyc: 4.4.1.14-RXN',\n",
       "   'KEGG: R00179',\n",
       "   'MetaCyc: 4.4.1.14-RXN',\n",
       "   'Name: 1-aminocyclopropane-1-carboxylate synthase; 1-aminocyclopropane-1-carboxylate synthetase; 1-aminocyclopropane-1-carboxylic acid synthase; 1-aminocyclopropanecarboxylate synthase; ACC synthase; S-Adenosyl-L-methionine methylthioadenosine-lyase; S-adenosyl-L-methionine methylthioadenosine-lyase; S-adenosyl-L-methionine methylthioadenosine-lyase (1-aminocyclopropane-1-carboxylate-forming); aminocyclopropanecarboxylate synthase; aminocyclopropanecarboxylic acid synthase'],\n",
       "  'code': '(1) cpd00017[0] <=> (1) cpd00147[0] + (1) cpd00909[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00067;cpd00147;cpd00909',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] <=> (1) H+[0] + (1) 5-Methylthioadenosine[0] + (1) 1-Aminocyclopropane-1-carboxylate[0]',\n",
       "  'deltag': -3.05,\n",
       "  'deltagerr': 5.12,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.4.1.14'],\n",
       "  'equation': '(1) cpd00017[0] <=> (1) cpd00067[0] + (1) cpd00147[0] + (1) cpd00909[0]',\n",
       "  'id': 'rxn00128',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27619;rxn33519;rxn37998;rxn37999',\n",
       "  'name': 'S-adenosyl-L-methionine methylthioadenosine-lyase (1-aminocyclopropane-1-carboxylate-forming)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); ETHYL-PWY (ethylene biosynthesis I (plants)); Ethylene-Biosynthesis (Ethylene Biosynthesis); HORMONE-SYN (Hormone Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-Salvage (L-methionine Salvage); PWY-7270 (L-methionine salvage cycle II (plants)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";1:cpd00067:0:0:\"H+\";1:cpd00147:0:0:\"5-Methylthioadenosine\";1:cpd00909:0:0:\"1-Aminocyclopropane-1-carboxylate\"'},\n",
       " {'abbreviation': 'R00180',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00180',\n",
       "   'MetaCyc: ADENOSYLMETHIONINE-CYCLOTRANSFERASE-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine alkyltransferase (cyclizing); S-adenosyl-L-methionine alkyltransferase (cyclizing); adenosylmethioninase; adenosylmethionine cyclotransferase'],\n",
       "  'code': '(1) cpd00017[0] <=> (1) cpd00147[0] + (1) cpd01873[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00147;cpd01873',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] <=> (1) 5-Methylthioadenosine[0] + (1) 2-Aminobutan-4-olide[0]',\n",
       "  'deltag': 1.85,\n",
       "  'deltagerr': 4.92,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.5.1.4'],\n",
       "  'equation': '(1) cpd00017[0] <=> (1) cpd00147[0] + (1) cpd01873[0]',\n",
       "  'id': 'rxn00129',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine alkyltransferase (cyclizing)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";1:cpd00147:0:0:\"5-Methylthioadenosine\";1:cpd01873:0:0:\"2-Aminobutan-4-olide\"'},\n",
       " {'abbreviation': 'R00181',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: AMP-DEAMINASE-RXN',\n",
       "   'BiGG: AMD1_1; AMPDA',\n",
       "   'BrachyCyc: AMP-DEAMINASE-RXN',\n",
       "   'KEGG: R00181',\n",
       "   'MetaCyc: ADENOSINE-PHOSPHATE-DEAMINASE-RXN; AMP-DEAMINASE-RXN',\n",
       "   'Name: AMP aminohydrolase; AMP deaminase; Adenosine monophosphate deaminase; adenine nucleotide deaminase; adenosine (phosphate) deaminase; adenosine-phosphate aminohydrolase; adenosine-phosphate deaminase; adenylate deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00018[0] <=> (1) cpd00013[0] + (1) cpd00114[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00018;cpd00067;cpd00114',\n",
       "  'definition': '(1) H2O[0] + (1) AMP[0] + (1) H+[0] => (1) NH3[0] + (1) IMP[0]',\n",
       "  'deltag': -7.56,\n",
       "  'deltagerr': 0.47,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.17', '3.5.4.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00018[0] + (1) cpd00067[0] => (1) cpd00013[0] + (1) cpd00114[0]',\n",
       "  'id': 'rxn00130',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30350;rxn32247;rxn32251',\n",
       "  'name': 'AMP aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Adenosine-Nucleotides-Degradation (Adenosine Nucleotide Degradation); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); P165-PWY (superpathway of purines degradation in plants); PWY-5044 (purine nucleotides degradation I (plants)); PWY-6596 (adenosine nucleotides degradation I); Purine-Degradation (Purine Nucleotide Degradation); Urate-Degradation (Urate Degradation)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00018:0:0:\"AMP\";-1:cpd00067:0:0:\"H+\";1:cpd00013:0:0:\"NH3\";1:cpd00114:0:0:\"IMP\"'},\n",
       " {'abbreviation': 'R00182',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: AMP-NUCLEOSID-RXN',\n",
       "   'ChlamyCyc: AMP-NUCLEOSID-RXN',\n",
       "   'EcoCyc: AMP-NUCLEOSID-RXN',\n",
       "   'KEGG: R00182',\n",
       "   'MetaCyc: AMP-NUCLEOSID-RXN',\n",
       "   'Name: AMP nucleosidase; AMP phosphoribohydrolase; adenosine monophosphate nucleosidase; adenylate nucleosidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00018[0] <=> (1) cpd00101[0] + (1) cpd00128[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00018;cpd00101;cpd00128',\n",
       "  'definition': '(1) H2O[0] + (1) AMP[0] <=> (1) ribose-5-phosphate[0] + (1) Adenine[0]',\n",
       "  'deltag': -4.78,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00018[0] <=> (1) cpd00101[0] + (1) cpd00128[0]',\n",
       "  'id': 'rxn00131',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'AMP phosphoribohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenosine-Nucleotides-Degradation (Adenosine Nucleotide Degradation); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-6617 (adenosine nucleotides degradation III); Purine-Degradation (Purine Nucleotide Degradation)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00018:0:0:\"AMP\";1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00128:0:0:\"Adenine\"'},\n",
       " {'abbreviation': 'R00183',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5-NUCLEOTID-RXN; AMP-DEPHOSPHORYLATION-RXN',\n",
       "   'BiGG: NTD7; NTD7pp; U30',\n",
       "   'BrachyCyc: AMP-DEPHOSPHORYLATION-RXN',\n",
       "   'KEGG: R00183',\n",
       "   'MetaCyc: 5-NUCLEOTID-RXN; AMP-DEPHOSPHORYLATION-RXN',\n",
       "   \"Name: 5'-AMP nucleotidase; 5'-AMPase; 5'-adenylic phosphatase; 5'-mononucleotidase; 5'-nucleotidase; 5'-nucleotidase (AMP); 5'-ribonucleotide phosphohydrolase; AMP phosphatase; AMP phosphohydrolase; AMP-5'-nucleotidase; AMPase; Adenosine 5'-monophosphate phosphohydrolase; IMP 5'-nucleotidase; UMPase; adenosine 5'-phosphatase; adenosine monophosphatase; amp nucleotidase; snake venom 5'-nucleotidase; thimidine monophosphate nucleotidase; uridine 5'-nucleotidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00018[0] <=> (1) cpd00009[0] + (1) cpd00182[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00018;cpd00182',\n",
       "  'definition': '(1) H2O[0] + (1) AMP[0] <=> (1) Phosphate[0] + (1) Adenosine[0]',\n",
       "  'deltag': -3.37,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00018[0] <=> (1) cpd00009[0] + (1) cpd00182[0]',\n",
       "  'id': 'rxn00132',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18970;rxn30611;rxn31965;rxn31971;rxn31973',\n",
       "  'name': \"Adenosine 5'-monophosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenosine-Nucleotides-Degradation (Adenosine Nucleotide Degradation); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-6353 (purine nucleotides degradation II (aerobic)); Purine-Degradation (Purine Nucleotide Degradation); SALVADEHYPOX-PWY (adenosine nucleotides degradation II)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00018:0:0:\"AMP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00182:0:0:\"Adenosine\"'},\n",
       " {'abbreviation': 'R00184',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00184_c',\n",
       "   'AraGEM: R_R00184_c',\n",
       "   'KEGG: R00184',\n",
       "   'MetaCyc: RXN-12820',\n",
       "   'iRS1597: R00184_c',\n",
       "   \"Name: P1,P4-Bis(5'-adenosyl) tetraphosphate nucleotidohydrolase; P1,P4-bis(5'-adenosyl)-tetraphosphate adenylohydrolase; diadenosine hexaphosphate hydrolase (ATP-forming)\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00924[0] <=> (1) cpd00002[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00018;cpd00067;cpd00924',\n",
       "  'definition': '(1) H2O[0] + (1) AppppA[0] => (1) ATP[0] + (1) AMP[0] + (1) H+[0]',\n",
       "  'deltag': -7.42,\n",
       "  'deltagerr': 1.99,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.17', '3.6.1.61'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00924[0] => (1) cpd00002[0] + (1) cpd00018[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00133',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"P1,P4-bis(5'-adenosyl)-tetraphosphate adenylohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00924:0:0:\"AppppA\";1:cpd00002:0:0:\"ATP\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00185',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENOSINE-KINASE-RXN',\n",
       "   'BiGG: ADNK1; YJR105W',\n",
       "   'BrachyCyc: ADENOSINE-KINASE-RXN',\n",
       "   'KEGG: R00185',\n",
       "   'MetaCyc: ADENOSINE-KINASE-RXN',\n",
       "   \"Name: ATP:adenosine 5'-phosphotransferase; Adenosine kinase; adenosine kinase; adenosine kinase (phosphorylating)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00182[0] <=> (1) cpd00008[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00018;cpd00067;cpd00182',\n",
       "  'definition': '(1) ATP[0] + (1) Adenosine[0] => (1) ADP[0] + (1) AMP[0] + (1) H+[0]',\n",
       "  'deltag': -2.79,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.20', '2.7.1.74'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00182[0] => (1) cpd00008[0] + (1) cpd00018[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00134',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27678;rxn38020',\n",
       "  'name': \"ATP:adenosine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenine-Adenosine-Salvage (Adenine and Adenosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6619 (adenine and adenosine salvage VI); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00182:0:0:\"Adenosine\";1:cpd00008:0:0:\"ADP\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': '(S)-Lactate adenylohydrolase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: (S)-Lactate adenylohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02644[0] <=> (1) cpd00008[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00008;cpd00018;cpd00067;cpd02644',\n",
       "  'definition': \"(1) H2O[0] + (1) P1,P2-Bis(5'-adenosyl) triphosphate[0] <=> (1) ADP[0] + (1) AMP[0] + (1) H+[0]\",\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.6.1.29'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02644[0] <=> (1) cpd00008[0] + (1) cpd00018[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00135',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-Lactate adenylohydrolase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02644:0:0:\"P1,P2-Bis(5\\'-adenosyl) triphosphate\";1:cpd00008:0:0:\"ADP\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00187',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: BIS5-ADENOSYL-TRIPHOSPHATASE-RXN',\n",
       "   'BrachyCyc: BIS5-ADENOSYL-TRIPHOSPHATASE-RXN',\n",
       "   'ChlamyCyc: BIS5-ADENOSYL-TRIPHOSPHATASE-RXN',\n",
       "   'KEGG: R00187',\n",
       "   'MetaCyc: BIS5-ADENOSYL-TRIPHOSPHATASE-RXN',\n",
       "   \"Name: 1-P,3-P-bis(5'-adenosyl)-triphosphate adenylohydrolase; P1,P3-Bis(5'-adenosyl) triphosphate adenylohydrolase; P1,P3-bis(5'-adenosyl)-triphosphate adenylohydrolase; bis(5'-adenosyl)-triphosphatase; diadenosine 5,5-P1,P3-triphosphatase; dinucleosidetriphosphatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03705[0] <=> (1) cpd00008[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00008;cpd00018;cpd00067;cpd03705',\n",
       "  'definition': '(1) H2O[0] + (1) ApppA[0] => (1) ADP[0] + (1) AMP[0] + (1) H+[0]',\n",
       "  'deltag': -18.84,\n",
       "  'deltagerr': 7.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.29'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03705[0] => (1) cpd00008[0] + (1) cpd00018[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00136',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35437',\n",
       "  'name': \"P1,P3-bis(5'-adenosyl)-triphosphate adenylohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03705:0:0:\"ApppA\";1:cpd00008:0:0:\"ADP\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00188',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 325-BISPHOSPHATE-NUCLEOTIDASE-RXN',\n",
       "   'BiGG: BPNT; MET22',\n",
       "   'BrachyCyc: 325-BISPHOSPHATE-NUCLEOTIDASE-RXN',\n",
       "   'KEGG: R00188',\n",
       "   'MetaCyc: 325-BISPHOSPHATE-NUCLEOTIDASE-RXN',\n",
       "   \"Name: 3'(2'),5'-BISPHOSPHATE-NUCLEOTIDASE-RXN; 3'(2'),5'-bisphosphate nucleotidase; 3'(2'),5'-bisphosphonucleoside 3'(2')-phosphohydrolase; 3',5'-bisphosphate nucleotidase; 3',5'-bisphosphate nucleotidase (EC 3.1.3.7); 3'-phosphoadenylylsulfate 3'-phosphatase; 3-5-bisphosphatenucleotidase; Adenosine 3',5'-bisphosphate 3'(2')-phosphohydrolase; Adenosine 3',5'-bisphosphate 3'-phosphohydrolase; adenosine-3'(2'),5'-bisphosphate 3'(2')-phosphohydrolase; phosphoadenylate 3'-nucleotidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00045[0] <=> (1) cpd00009[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00018;cpd00045',\n",
       "  'definition': '(1) H2O[0] + (1) Adenosine 3-5-bisphosphate[0] <=> (1) Phosphate[0] + (1) AMP[0]',\n",
       "  'deltag': -2.59,\n",
       "  'deltagerr': 0.75,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00045[0] <=> (1) cpd00009[0] + (1) cpd00018[0]',\n",
       "  'id': 'rxn00137',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn11357;rxn32810',\n",
       "  'name': \"Adenosine 3',5'-bisphosphate 3'-phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00045:0:0:\"Adenosine 3-5-bisphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00018:0:0:\"AMP\"'},\n",
       " {'abbreviation': 'R00189',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NAD-SYNTH-NH3-RXN',\n",
       "   'BiGG: NADS1; NADS1m; NADS1n; QNS1_1; QNS1_2',\n",
       "   'BrachyCyc: NAD-SYNTH-NH3-RXN',\n",
       "   'KEGG: R00189',\n",
       "   'MetaCyc: NAD-SYNTH-NH3-RXN',\n",
       "   'Name: Deamino-NAD+:ammonia ligase (AMP-forming); Glutamine-dependent NAD(+) synthetase; NAD synthase; NAD synthase (nh3); NAD synthase (nh4); NAD synthetase; NAD(+) synthetase; NAD+ synthase; NADsynthase-nh3; deamido-NAD+:ammonia ligase (AMP-forming); diphosphopyridine nucleotide synthetase; nicotinamide adenine dinucleotide synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00638[0] <=> (1) cpd00003[0] + (1) cpd00012[0] + (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00003;cpd00012;cpd00013;cpd00018;cpd00067;cpd00638',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) Deamido-NAD[0] => (1) NAD[0] + (1) PPi[0] + (1) AMP[0] + (2) H+[0]',\n",
       "  'deltag': -3.69,\n",
       "  'deltagerr': 0.67,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.1.5', '6.3.5.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00638[0] => (1) cpd00003[0] + (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00138',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30039;rxn30193;rxn30412;rxn32216',\n",
       "  'name': 'deamido-NAD+:ammonia ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); PWY0-781 (aspartate superpathway); PYRIDNUCSYN-PWY (NAD <i>de novo</i> biosynthesis I (from aspartate))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00638:0:0:\"Deamido-NAD\";1:cpd00003:0:0:\"NAD\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00190',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENPRIBOSYLTRAN-RXN',\n",
       "   'BiGG: ADPT; APT1; APT2',\n",
       "   'BrachyCyc: ADENPRIBOSYLTRAN-RXN',\n",
       "   'KEGG: R00190',\n",
       "   'MetaCyc: ADENPRIBOSYLTRAN-RXN',\n",
       "   'Name: AMP pyrophosphorylase; AMP-pyrophosphate phosphoribosyltransferase; AMP:diphosphate phospho-D-ribosyltransferase; AMP:pyrophosphate phosphoribosyltransferase; APRT; adenine phosphoribosylpyrophosphate transferase; adenine phosphoribosyltransferase; adeninephosphoribosyltransferase; adenosine phosphoribosyltransferase; adenylate pyrophosphorylase; adenylic pyrophosphorylase; transphosphoribosidase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00018[0] <=> (1) cpd00103[0] + (1) cpd00128[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00018;cpd00067;cpd00103;cpd00128',\n",
       "  'definition': '(1) PPi[0] + (1) AMP[0] + (1) H+[0] <= (1) PRPP[0] + (1) Adenine[0]',\n",
       "  'deltag': 3.85,\n",
       "  'deltagerr': 0.33,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.7', '2.4.2.8'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] <= (1) cpd00103[0] + (1) cpd00128[0]',\n",
       "  'id': 'rxn00139',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38024',\n",
       "  'name': 'AMP:diphosphate phospho-D-ribosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenine-Adenosine-Salvage (Adenine and Adenosine Salvage); Degradation (Degradation/Utilization/Assimilation); Glyphosate-Degradation (glyphosate degradation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); P121-PWY (adenine and adenosine salvage I); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6605 (adenine and adenosine salvage II); PWY-6610 (adenine salvage); PWY-7805 ((aminomethyl)phosphonate degradation); PWY-7807 (glyphosate degradation III); PWY66-409 (superpathway of purine nucleotide salvage); Phosphorus-Compounds (Phosphorus Compound Metabolism); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00018:0:0:\"AMP\";-1:cpd00067:0:0:\"H+\";1:cpd00103:0:0:\"PRPP\";1:cpd00128:0:0:\"Adenine\"'},\n",
       " {'abbreviation': 'R00191',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PDE1; PDE2_1',\n",
       "   'BrachyCyc: RXN0-5038',\n",
       "   'EcoCyc: RXN0-5038',\n",
       "   'KEGG: R00191',\n",
       "   'MetaCyc: 3.1.4.17-RXN; RXN0-5038',\n",
       "   \"Name: 3',5'-cyclic-AMP 5'-nucleotidohydrolase; 3',5'-cyclic-AMP phosphodiesterase; 3',5'-cyclic-nucleotide phosphodiesterase; 3',5'-cyclic-nucleotide phosphodiesterase 2; Adenosine 3',5'-phosphate 5'-nucleotidohydrolase; PDE2A; PDE2B; PDE4; PDE7; PDE8; PDEB1; PDEB2; adenosine 3',5'-phosphate 5'-nucleotidohydrolase; cAMP-specific PDE; cAMP-specific phosphodiesterase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00446[0] <=> (1) cpd00018[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00018;cpd00067;cpd00446',\n",
       "  'definition': '(1) H2O[0] + (1) cAMP[0] => (1) AMP[0] + (1) H+[0]',\n",
       "  'deltag': -10.15,\n",
       "  'deltagerr': 0.27,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.4.17', '3.1.4.53'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00446[0] => (1) cpd00018[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00140',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35432',\n",
       "  'name': \"adenosine 3',5'-phosphate 5'-nucleotidohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00446:0:0:\"cAMP\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00192',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENOSYLHOMOCYSTEINASE-RXN',\n",
       "   'BiGG: AHC; AHCi; SAH1',\n",
       "   'BrachyCyc: ADENOSYLHOMOCYSTEINASE-RXN',\n",
       "   'KEGG: R00192',\n",
       "   'MetaCyc: ADENOSYLHOMOCYSTEINASE-RXN',\n",
       "   'Name: Adenosylhomocysteinase; AdoHcyase; S-Adenosyl-L-homocysteine hydrolase; S-adenosyl-L-homocysteine hydrolase; S-adenosylhomocysteinase; S-adenosylhomocysteine hydrolase (ambiguous); S-adenosylhomocysteine synthase; SAHase; adenosylhomocysteine hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00019[0] <=> (1) cpd00135[0] + (1) cpd00182[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00019;cpd00135;cpd00182',\n",
       "  'definition': '(1) H2O[0] + (1) S-Adenosyl-homocysteine[0] <=> (1) Homocysteine[0] + (1) Adenosine[0]',\n",
       "  'deltag': 8.79,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.3.1.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00019[0] <=> (1) cpd00135[0] + (1) cpd00182[0]',\n",
       "  'id': 'rxn00141',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27683;rxn30600;rxn33400',\n",
       "  'name': 'S-Adenosyl-L-homocysteine hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CYSTEINE-SYN (L-cysteine Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-DEG (L-methionine Degradation); METHIONINE-DEG1-PWY (L-methionine degradation I (to L-homocysteine)); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-Salvage (L-methionine Salvage); PWY-5041 (<i>S</i>-adenosyl-L-methionine cycle II); PWY-5328 (superpathway of L-methionine salvage and degradation); PWY-6292 (superpathway of L-cysteine biosynthesis (mammalian)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); S-adenosyl-L-methionine-cycle (<i>S</i>-adenosyl-L-methionine cycle)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00135:0:0:\"Homocysteine\";1:cpd00182:0:0:\"Adenosine\"'},\n",
       " {'abbreviation': 'R00193',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00193',\n",
       "   'MetaCyc: S-ADENOSYLHOMOCYSTEINE-DEAMINASE-RXN',\n",
       "   'PoplarCyc: S-ADENOSYLHOMOCYSTEINE-DEAMINASE-RXN',\n",
       "   'SoyCyc: S-ADENOSYLHOMOCYSTEINE-DEAMINASE-RXN',\n",
       "   'Name: S-Adenosyl-L-homocysteine aminohydrolase; S-adenosyl-L-homocysteine aminohydrolase; S-adenosylhomocysteine deaminase; adenosylhomocysteine deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00019[0] <=> (1) cpd00013[0] + (1) cpd02171[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00019;cpd00067;cpd02171',\n",
       "  'definition': '(1) H2O[0] + (1) S-Adenosyl-homocysteine[0] + (1) H+[0] <=> (1) NH3[0] + (1) S-Inosyl-L-homocysteine[0]',\n",
       "  'deltag': -5.45,\n",
       "  'deltagerr': 0.69,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.4.28'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00019[0] + (1) cpd00067[0] <=> (1) cpd00013[0] + (1) cpd02171[0]',\n",
       "  'id': 'rxn00142',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-homocysteine aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";-1:cpd00067:0:0:\"H+\";1:cpd00013:0:0:\"NH3\";1:cpd02171:0:0:\"S-Inosyl-L-homocysteine\"'},\n",
       " {'abbreviation': 'R00194',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENOSYLHOMOCYSTEINE-NUCLEOSIDASE-RXN',\n",
       "   'BiGG: AHCYSNS',\n",
       "   'BrachyCyc: ADENOSYLHOMOCYSTEINE-NUCLEOSIDASE-RXN',\n",
       "   'KEGG: R00194',\n",
       "   'MetaCyc: ADENOSYLHOMOCYSTEINE-NUCLEOSIDASE-RXN',\n",
       "   \"Name: 5'-methyladenosine nucleosidase; AdoHcy/MTA nucleosidase; S-Adenosyl-L-homocysteine homocysteinylribohydrolase; S-adenosyl-L-homocysteine homocysteinylribohydrolase; S-adenosylhomocysteine hydrolase (ambiguous); S-adenosylhomocysteine nucleosidase; S-adenosylhomocysteine/5'-methylthioadenosine nucleosidase; S-adenosylhomocysteinenucleosidase; adenosylhomocysteine nucleosidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00019[0] <=> (1) cpd00128[0] + (1) cpd02227[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00019;cpd00128;cpd02227',\n",
       "  'definition': '(1) H2O[0] + (1) S-Adenosyl-homocysteine[0] <=> (1) Adenine[0] + (1) S-Ribosylhomocysteine[0]',\n",
       "  'deltag': -3.86,\n",
       "  'deltagerr': 0.68,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00019[0] <=> (1) cpd00128[0] + (1) cpd02227[0]',\n",
       "  'id': 'rxn00143',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-homocysteine homocysteinylribohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Autoinducer-Biosynthesis (Autoinducer Biosynthesis); CYSTEINE-SYN (L-cysteine Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-Salvage (L-methionine Salvage); PWY-6151 (<i>S</i>-adenosyl-L-methionine cycle I); PWY-6153 (autoinducer AI-2 biosynthesis I); PWY-6154 (autoinducer AI-2 biosynthesis II (<i>Vibrio</i>)); PWY-I9 (L-cysteine biosynthesis VI (from L-methionine)); S-adenosyl-L-methionine-cycle (<i>S</i>-adenosyl-L-methionine cycle); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00128:0:0:\"Adenine\";1:cpd02227:0:0:\"S-Ribosylhomocysteine\"'},\n",
       " {'abbreviation': 'R00195',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: DAPAL',\n",
       "   'KEGG: R00195',\n",
       "   'iAF1260: DAPAL',\n",
       "   'Name: 2,3-Diaminopropionate ammonia-lyase; 2,3-diaminopropanoate ammonia-lyase (adding water; pyruvate-forming); 2,3-diaminopropionate amonnia lyase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03828[0] <=> (2) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00067;cpd03828',\n",
       "  'definition': '(1) H2O[0] + (1) H+[0] + (1) 2,3-Diaminopropionate[0] => (2) NH3[0] + (1) Pyruvate[0]',\n",
       "  'deltag': 0.61,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.15'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00067[0] + (1) cpd03828[0] => (2) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00144',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30005',\n",
       "  'name': '2,3-diaminopropanoate ammonia-lyase (adding water; pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00067:0:0:\"H+\";-1:cpd03828:0:0:\"2,3-Diaminopropionate\";2:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00196',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CYB2; L-LACD; L-LACDm; L_LACD',\n",
       "   'JP_Creinhardtii_MSB: R_LLDH_LPAREN_ferr_RPAREN_m; R_LLDH_LPAREN_ferr_RPAREN_x',\n",
       "   'JP_Creinhardtii_NMeth: R_LLDH(ferr)g; R_LLDH(ferr)m',\n",
       "   'KEGG: R00196',\n",
       "   'iIN800: CYB2',\n",
       "   'Name: (S)-Lactate:ferricytochrome-c 2-oxidoreductase; Cytochrome b2; L-Lactate dehydrogenase (ferricytochrome); L-Lactate dehydrogenase irreversible'],\n",
       "  'code': '(2) cpd00109[0] + (1) cpd00159[0] <=> (1) cpd00020[0] + (2) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00067;cpd00109;cpd00110;cpd00159',\n",
       "  'definition': '(2) Cytochrome c3+[0] + (1) L-Lactate[0] <=> (1) Pyruvate[0] + (2) H+[0] + (2) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.2.3'],\n",
       "  'equation': '(2) cpd00109[0] + (1) cpd00159[0] <=> (1) cpd00020[0] + (2) cpd00067[0] + (2) cpd00110[0]',\n",
       "  'id': 'rxn00145',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30103;rxn34237;rxn35452',\n",
       "  'name': '(S)-Lactate:ferricytochrome-c 2-oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00109:0:0:\"Cytochrome c3+\";-1:cpd00159:0:0:\"L-Lactate\";1:cpd00020:0:0:\"Pyruvate\";2:cpd00067:0:0:\"H+\";2:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'R00197',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: D-LACDm',\n",
       "   'JP_Creinhardtii_MSB: R_D_DASH_LACDHm',\n",
       "   'JP_Creinhardtii_NMeth: R_D-LACDHm',\n",
       "   'KEGG: R00197',\n",
       "   'iMM904: D-LACDm',\n",
       "   'Name: (R)-Lactate:ferricytochrome-c 2-oxidoreductase; D-lactate dehydrogenase'],\n",
       "  'code': '(2) cpd00109[0] + (1) cpd00221[0] <=> (1) cpd00020[0] + (2) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00067;cpd00109;cpd00110;cpd00221',\n",
       "  'definition': '(2) Cytochrome c3+[0] + (1) D-Lactate[0] <=> (1) Pyruvate[0] + (2) H+[0] + (2) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.2.4'],\n",
       "  'equation': '(2) cpd00109[0] + (1) cpd00221[0] <=> (1) cpd00020[0] + (2) cpd00067[0] + (2) cpd00110[0]',\n",
       "  'id': 'rxn00146',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30364',\n",
       "  'name': '(R)-Lactate:ferricytochrome-c 2-oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00109:0:0:\"Cytochrome c3+\";-1:cpd00221:0:0:\"D-Lactate\";1:cpd00020:0:0:\"Pyruvate\";2:cpd00067:0:0:\"H+\";2:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'R00199',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PEPSYNTH-RXN',\n",
       "   'BiGG: PPS',\n",
       "   'BrachyCyc: PEPSYNTH-RXN',\n",
       "   'KEGG: R00199',\n",
       "   'MetaCyc: PEPSYNTH-RXN',\n",
       "   'Name: ATP:pyruvate, water phosphotransferase; ATP:pyruvate,water phosphotransferase; PEP synthetase; phoephoenolpyruvate synthetase; phosphoenolpyruvate synthase; phosphoenolpyruvatesynthase; phosphoenolpyruvic synthase; phosphopyruvate synthetase; pyruvate, water dikinase; pyruvate,water dikinase; pyruvate-water dikinase (phosphorylating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00020[0] <=> (1) cpd00009[0] + (1) cpd00018[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00009;cpd00018;cpd00020;cpd00061;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) Pyruvate[0] => (1) Phosphate[0] + (1) AMP[0] + (1) Phosphoenolpyruvate[0] + (3) H+[0]',\n",
       "  'deltag': 0.79,\n",
       "  'deltagerr': 0.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.9.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00020[0] => (1) cpd00009[0] + (1) cpd00018[0] + (1) cpd00061[0] + (3) cpd00067[0]',\n",
       "  'id': 'rxn00147',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33266',\n",
       "  'name': 'ATP:pyruvate,water phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); GLUCONEO-PWY (gluconeogenesis I); GLYCOLYSIS (glycolysis I (from glucose 6-phosphate)); GLYCOLYSIS-E-D (superpathway of glycolysis and the Entner-Doudoroff pathway); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYCOLYSIS-VARIANTS (Glycolysis); Gluconeogenesis (Gluconeogenesis); HEXITOLDEGSUPER-PWY (superpathway of hexitol degradation (bacteria)); P23-PWY (reductive TCA cycle I); P341-PWY (glycolysis V (Pyrococcus)); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-5484 (glycolysis II (from fructose 6-phosphate)); PWY-6142 (gluconeogenesis II (<i>Methanobacterium thermoautotrophicum</i>)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-7345 (superpathway of anaerobic sucrose degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Reductive-TCA-Cycles (Reductive TCA Cycles); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00680 (Methane metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00020:0:0:\"Pyruvate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00018:0:0:\"AMP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";3:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00200',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PEPDEPHOS-RXN',\n",
       "   'BiGG: CDC19; PYK; PYK2',\n",
       "   'BrachyCyc: PEPDEPHOS-RXN',\n",
       "   'KEGG: R00200',\n",
       "   'MetaCyc: PEPDEPHOS-RXN',\n",
       "   'Name: ATP:pyruvate 2-O-phosphotransferase; ATP:pyruvate O2-phosphotransferase; phosphoenol transphosphorylase; phosphoenolpyruvate kinase; pyruvate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00020[0] <=> (1) cpd00008[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00020;cpd00061;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) Pyruvate[0] <=> (1) ADP[0] + (1) Phosphoenolpyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 6.53,\n",
       "  'deltagerr': 0.1,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.40'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00020[0] <=> (1) cpd00008[0] + (1) cpd00061[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00148',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20509;rxn35242',\n",
       "  'name': 'ATP:pyruvate 2-O-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROFRUCAT-PWY (homolactic fermentation); ANAGLYCOLYSIS-PWY (glycolysis III (from glucose)); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Entner-Duodoroff-Pathways (Entner-Duodoroff Pathways); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); GLYCOLYSIS (glycolysis I (from glucose 6-phosphate)); GLYCOLYSIS-E-D (superpathway of glycolysis and the Entner-Doudoroff pathway); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYCOLYSIS-VARIANTS (Glycolysis); Gluconeogenesis (Gluconeogenesis); HEXITOLDEGSUPER-PWY (superpathway of hexitol degradation (bacteria)); NPGLUCAT-PWY (Entner-Doudoroff pathway II (non-phosphorylative)); P122-PWY (heterolactic fermentation); P124-PWY (Bifidobacterium shunt); P341-PWY (glycolysis V (Pyrococcus)); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-1042 (glycolysis IV (plant cytosol)); PWY-2221 (Entner-Doudoroff pathway III (semi-phosphorylative)); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-5484 (glycolysis II (from fructose 6-phosphate)); PWY-5723 (Rubisco shunt); PWY-6142 (gluconeogenesis II (<i>Methanobacterium thermoautotrophicum</i>)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-6901 (superpathway of glucose and xylose degradation); PWY-7003 (glycerol degradation to butanol); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY-8004 (Entner-Doudoroff pathway I); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00230 (Purine metabolism); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00020:0:0:\"Pyruvate\";1:cpd00008:0:0:\"ADP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00203',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN',\n",
       "   'ChlamyCyc: ALDHDEHYDROG-RXN',\n",
       "   'KEGG: R00203',\n",
       "   'MaizeCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN; METHYL-GLYOXAL-DEHYDROG-RXN',\n",
       "   'MetaCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN; METHYL-GLYOXAL-DEHYDROG-RXN',\n",
       "   'Name: 2-oxopropanal:NAD+ oxidoreductase; ALDEHYDE DEHYDROGENASE; Aldehyde dehydrogenase (NAD+); Pyruvaldehyde:NAD+ 2-oxidoreductase; Pyruvaldehyde:NAD+ oxidoreductase; pyruvaldehyde:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00428[0] <=> (1) cpd00004[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00020;cpd00067;cpd00428',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) 2-Oxopropanal[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (2) H+[0]',\n",
       "  'deltag': -6.69,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.22', '1.2.1.23', '1.2.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00428[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00149',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38862',\n",
       "  'name': 'pyruvaldehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Aldehyde-Degradation (Aldehyde Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); Methylglyoxal-Detoxification (Methylglyoxal Detoxification); PWY-5456 (methylglyoxal degradation VII)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00428:0:0:\"2-Oxopropanal\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00205',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN',\n",
       "   'KEGG: R00205',\n",
       "   'MaizeCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN',\n",
       "   'MetaCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN; RXN-8639',\n",
       "   'SorghumCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN',\n",
       "   'Name: 2-Oxopropanal:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00428[0] <=> (1) cpd00005[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00020;cpd00067;cpd00428',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) 2-Oxopropanal[0] <=> (1) NADPH[0] + (1) Pyruvate[0] + (2) H+[0]',\n",
       "  'deltag': -6.69,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.49'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00428[0] <=> (1) cpd00005[0] + (1) cpd00020[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00150',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Oxopropanal:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Aldehyde-Degradation (Aldehyde Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); Methylglyoxal-Detoxification (Methylglyoxal Detoxification); PWY-5462 (methylglyoxal degradation II)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00428:0:0:\"2-Oxopropanal\";1:cpd00005:0:0:\"NADPH\";1:cpd00020:0:0:\"Pyruvate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00206',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRUVATEORTHOPHOSPHATE-DIKINASE-RXN',\n",
       "   'BiGG: PPDK',\n",
       "   'BrachyCyc: PYRUVATEORTHOPHOSPHATE-DIKINASE-RXN',\n",
       "   'KEGG: R00206',\n",
       "   'MetaCyc: PYRUVATEORTHOPHOSPHATE-DIKINASE-RXN',\n",
       "   'Name: ATP:pyruvate, phosphate phosphotransferase; ATP:pyruvate,orthophosphate phosphotransferase; ATP:pyruvate,phosphate phosphotransferase; Pyruvate,phosphate dikinase.; pyruvate, Pi dikinase; pyruvate, orthophosphate dikinase; pyruvate, phosphate dikinase; pyruvate-inorganic phosphate dikinase; pyruvate-phosphate dikinase; pyruvate-phosphate dikinase (phosphorylating); pyruvate-phosphate ligase; pyruvic-phosphate dikinase; pyruvic-phosphate ligase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00009[0] + (1) cpd00020[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00009;cpd00012;cpd00018;cpd00020;cpd00061;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) Phosphate[0] + (1) Pyruvate[0] => (1) PPi[0] + (1) AMP[0] + (1) Phosphoenolpyruvate[0] + (2) H+[0]',\n",
       "  'deltag': 4.26,\n",
       "  'deltagerr': 0.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.9.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00009[0] + (1) cpd00020[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00061[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00151',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20655',\n",
       "  'name': 'ATP:pyruvate,phosphate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); GLUTAMINE-SYN (L-glutamine Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-241 (C4 photosynthetic carbon assimilation cycle, NADP-ME type); PWY-6549 (L-glutamine biosynthesis III); PWY-7115 (C4 photosynthetic carbon assimilation cycle, NAD-ME type); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); Photosynthesis ()',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00009:0:0:\"Phosphate\";-1:cpd00020:0:0:\"Pyruvate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00207',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00207',\n",
       "   'MetaCyc: 1.2.3.3-RXN',\n",
       "   'iRS1563: R_R00207_c',\n",
       "   'Name: Pyruvate:oxygen 2-oxidoreductase (phosphorylating); phosphate-dependent pyruvate oxidase; pyruvate oxidase; pyruvate:oxygen 2-oxidoreductase (phosphorylating); pyruvic oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00009[0] + (1) cpd00020[0] <=> (1) cpd00011[0] + (1) cpd00025[0] + (1) cpd00196[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00009;cpd00011;cpd00020;cpd00025;cpd00067;cpd00196',\n",
       "  'definition': '(1) O2[0] + (1) Phosphate[0] + (1) Pyruvate[0] + (1) H+[0] => (1) CO2[0] + (1) H2O2[0] + (1) Acetylphosphate[0]',\n",
       "  'deltag': -37.93,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.3.3'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00009[0] + (1) cpd00020[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00025[0] + (1) cpd00196[0]',\n",
       "  'id': 'rxn00152',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Pyruvate:oxygen 2-oxidoreductase (phosphorylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00009:0:0:\"Phosphate\";-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00025:0:0:\"H2O2\";1:cpd00196:0:0:\"Acetylphosphate\"'},\n",
       " {'abbreviation': 'R00208',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHOSPHOENOLPYRUVATE-PHOSPHATASE-RXN',\n",
       "   'KEGG: R00208',\n",
       "   'MetaCyc: PHOSPHOENOLPYRUVATE-PHOSPHATASE-RXN',\n",
       "   'PlantCyc: PHOSPHOENOLPYRUVATE-PHOSPHATASE-RXN',\n",
       "   'Name: PEP phosphatase; Phosphoenolpyruvate phosphohydrolase; phosphoenolpyruvate phosphatase; phosphoenolpyruvate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00061[0] <=> (1) cpd00009[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00020;cpd00061',\n",
       "  'definition': '(1) H2O[0] + (1) Phosphoenolpyruvate[0] => (1) Phosphate[0] + (1) Pyruvate[0]',\n",
       "  'deltag': -12.69,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.3.60'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00061[0] => (1) cpd00009[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00153',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phosphoenolpyruvate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00209',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRUVDEH-RXN',\n",
       "   'BiGG: PDH; PDHm',\n",
       "   'BrachyCyc: PYRUVDEH-RXN',\n",
       "   'KEGG: R00209',\n",
       "   'MetaCyc: PYRUVDEH-RXN',\n",
       "   'Name: pyruvate dehydrogenase; pyruvate dehydrogenase complex; pyruvate:NAD+ 2-oxidoreductase (CoA-acetylating)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00020[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00011;cpd00020;cpd00022',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) Pyruvate[0] => (1) NADH[0] + (1) CO2[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': -8.47,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.-', '1.2.1.M10', '1.2.4.1', '1.8.1.4', '2.3.1.12'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00020[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00154',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20656;rxn30418',\n",
       "  'name': 'pyruvate:NAD+ 2-oxidoreductase (CoA-acetylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); P41-PWY (pyruvate fermentation to acetate and lactate I); PWY-5482 (pyruvate fermentation to acetate II); PWY-5537 (pyruvate fermentation to acetate V); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); PWY-7384 (anaerobic energy metabolism (invertebrates, mitochondrial)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Lactate-Fermentation (pyruvate fermentation to lactate); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis)',\n",
       "   'KEGG: rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd00020:0:0:\"Pyruvate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00210',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JM_Creinhardtii: JM_Cre_38; JM_Cre_39; JM_Cre_40',\n",
       "   'KEGG: R00210',\n",
       "   'MetaCyc: PYRUVATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'iRS1563: R_R00210_c',\n",
       "   'Name: Pyruvate Decarboxylase; pyruvate dehydrogenase (NADP); pyruvate dehydrogenase (NADP+); pyruvate:NADP+ 2-oxidoreductase (CoA-acetylating)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00020[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00010;cpd00011;cpd00020;cpd00022',\n",
       "  'definition': '(1) NADP[0] + (1) CoA[0] + (1) Pyruvate[0] => (1) NADPH[0] + (1) CO2[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': -8.47,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.51'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00020[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00155',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33851;rxn33852',\n",
       "  'name': 'pyruvate:NADP+ 2-oxidoreductase (CoA-acetylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetyl-CoA-Biosynthesis (Acetyl-CoA Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-6970 (acetyl-CoA biosynthesis II (NADP-dependent pyruvate dehydrogenase))'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00020:0:0:\"Pyruvate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00211',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00211',\n",
       "   'MetaCyc: PYRUVATE-OXIDASE-COA-ACETYLATING-RXN',\n",
       "   'iRS1563: R_R00211_c',\n",
       "   'Name: pyruvate oxidase (CoA-acetylating); pyruvate:oxygen 2-oxidoreductase (CoA-acetylating)'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00010[0] + (1) cpd00020[0] <=> (1) cpd00011[0] + (1) cpd00022[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00010;cpd00011;cpd00020;cpd00022;cpd00025;cpd00067',\n",
       "  'definition': '(1) O2[0] + (1) CoA[0] + (1) Pyruvate[0] + (1) H+[0] => (1) CO2[0] + (1) Acetyl-CoA[0] + (1) H2O2[0]',\n",
       "  'deltag': -40.2,\n",
       "  'deltagerr': 1.27,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.3.6'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00010[0] + (1) cpd00020[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00022[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00156',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'pyruvate:oxygen 2-oxidoreductase (CoA-acetylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00010:0:0:\"CoA\";-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00212',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PFL; PFLi; PFLr',\n",
       "   'ChlamyCyc: PYRUVFORMLY-RXN',\n",
       "   'EcoCyc: PYRUVFORMLY-RXN',\n",
       "   'KEGG: R00212',\n",
       "   'MetaCyc: PYRUVFORMLY-RXN',\n",
       "   'Name: Acetyl-CoA:formate C-acetyltransferase; Formate C-acetyltransferase; Pyruvate formate-lyase; acetyl-CoA:formate C-acetyltransferase; formate C-acetyltransferase; formate acetyltransferase; pyruvate formate lyase; pyruvate formate-lyase; pyruvate-formate lyase; pyruvic formate-lyase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00047[0] <=> (1) cpd00010[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00020;cpd00022;cpd00047',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Formate[0] <=> (1) CoA[0] + (1) Pyruvate[0]',\n",
       "  'deltag': 4.94,\n",
       "  'deltagerr': 0.36,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.54'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00047[0] <=> (1) cpd00010[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00157',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28121;rxn34258',\n",
       "  'name': 'Acetyl-CoA:formate C-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-5480 (pyruvate fermentation to ethanol I); PWY-5485 (pyruvate fermentation to acetate IV); PWY-5493 (reductive monocarboxylic acid cycle); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00047:0:0:\"Formate\";1:cpd00010:0:0:\"CoA\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00213',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00213',\n",
       "   'MetaCyc: CARBAMOYL-SERINE-AMMONIA-LYASE-RXN',\n",
       "   'Name: O-Carbamoyl-L-serine ammonia-lyase (pyruvate-forming); O-carbamoyl-L-serine ammonia-lyase (decarboxylating, pyruvate-forming); O-carbamoyl-L-serine ammonia-lyase (decarboxylating; pyruvate-forming); O-carbamoyl-L-serine ammonia-lyase (pyruvate-forming); O-carbamoyl-L-serine deaminase; carbamoyl-serine ammonia-lyase; carbamoylserine deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01933[0] <=> (1) cpd00011[0] + (2) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00020;cpd00067;cpd01933',\n",
       "  'definition': '(1) H2O[0] + (1) H+[0] + (1) O-Carbamoyl-L-serine[0] => (1) CO2[0] + (2) NH3[0] + (1) Pyruvate[0]',\n",
       "  'deltag': -6.46,\n",
       "  'deltagerr': 1.44,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.13'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00067[0] + (1) cpd01933[0] => (1) cpd00011[0] + (2) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00158',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'O-carbamoyl-L-serine ammonia-lyase (decarboxylating; pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00067:0:0:\"H+\";-1:cpd01933:0:0:\"O-Carbamoyl-L-serine\";1:cpd00011:0:0:\"CO2\";2:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00214',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.1.1.39-RXN; MALIC-NAD-RXN',\n",
       "   'BiGG: ME1; ME1_rev; ME1m; ME1x',\n",
       "   'BrachyCyc: 1.1.1.39-RXN; MALIC-NAD-RXN',\n",
       "   'KEGG: R00214',\n",
       "   'MetaCyc: 1.1.1.39-RXN; MALIC-NAD-RXN; RXN-19748',\n",
       "   \"Name: 'malic' enzyme; (S)-malate:NAD+ oxidoreductase (decarboxylating); (S)-malate:NAD+ oxidoreductase (oxaloacetate-decarboxylating); Malic enzyme; NAD-linked malic enzyme; NAD-malic enzyme; NAD-specific malic enzyme; Pyruvic-malic carboxylase; R_S__malate_NAD__oxidoreductase__decarboxylating_; malate dehydrogenase (oxaloacetate decarboxylating); malate dehydrogenase (oxaloacetate-decarboxylating); malate oxidoreductase; malic enzyme; malic enzyme (NAD); malicenzyme; pyruvic-malic carboxylase\"],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00130[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00020;cpd00130',\n",
       "  'definition': '(1) NAD[0] + (1) L-Malate[0] => (1) NADH[0] + (1) CO2[0] + (1) Pyruvate[0]',\n",
       "  'deltag': 3.13,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.38', '1.1.1.39'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00130[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00159',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27459;rxn31389;rxn37872',\n",
       "  'name': '(S)-malate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); CARN-DEG (Carnitine Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GLUCONEO-PWY (gluconeogenesis I); Gluconeogenesis (Gluconeogenesis); L-Malate-Degradation ((<i>S</i>)-Malate Degradation); PWY-3641 (L-carnitine degradation III); PWY-7115 (C4 photosynthetic carbon assimilation cycle, NAD-ME type); PWY-7118 (chitin degradation to ethanol); PWY-7384 (anaerobic energy metabolism (invertebrates, mitochondrial)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY-7686 (L-malate degradation II); Photosynthesis ()',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00215',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MALDDH; ME1x_D',\n",
       "   'EcoCyc: 1.1.1.83-RXN',\n",
       "   'KEGG: R00215',\n",
       "   'MetaCyc: 1.1.1.83-RXN',\n",
       "   'iAF1260: MALDDH',\n",
       "   'Name: (R)-Malate:NAD+ oxidoreductase (decarboxylating); (R)-malate:NAD+ oxidoreductase (decarboxylating); D-malate dehydrogenase; D-malate dehydrogenase (NAD); D-malate dehydrogenase (decarboxylating); D-malic enzyme; bifunctional L(+)-tartrate dehydrogenase-D(+)-malate (decarboxylating); malate decarboxylating oxidoreductase (decarboxylating)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00386[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00020;cpd00386',\n",
       "  'definition': '(1) NAD[0] + (1) D-Malate[0] => (1) NADH[0] + (1) CO2[0] + (1) Pyruvate[0]',\n",
       "  'deltag': 5.29,\n",
       "  'deltagerr': 1.26,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.83'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00386[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00160',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(R)-Malate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY0-1465 (D-malate degradation)',\n",
       "   'KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00386:0:0:\"D-Malate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00216',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MALIC-NADP-RXN',\n",
       "   'BiGG: MAE1; ME2; ME2m',\n",
       "   'BrachyCyc: MALIC-NADP-RXN',\n",
       "   'KEGG: R00216',\n",
       "   'MetaCyc: MALIC-NADP-RXN',\n",
       "   \"Name: 'malic' enzyme; (S)-Malate:NADP+ oxidoreductase(oxaloacetate-decarboxylating); (S)-malate:NADP+ oxidoreductase (oxaloacetate-decarboxylating); L-malate:NADP oxidoreductase; Malate dehydrogenase (oxaloacetate decarboxylating) (NADP+); Malic enzyme; NAD-dependent malic enzyme; NADP-linked decarboxylating malic enzyme; NADP-malic enzyme; NADP-specific malate dehydrogenase; NADP-specific malic enzyme; Pyruvic-malic carboxylase; malate dehydrogenase (NADP, decarboxylating); malate dehydrogenase (decarboxylating, NADP); malate dehydrogenase (oxaloacetate decarboxylating) (nadp+); malate dehydrogenase (oxaloacetate-decarboxylating) (NADP); malate dehydrogenase (oxaloacetate-decarboxylating) (NADP+); malic enzyme (NADP); pyruvic-malic carboxylase\"],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00130[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00011;cpd00020;cpd00130',\n",
       "  'definition': '(1) NADP[0] + (1) L-Malate[0] => (1) NADPH[0] + (1) CO2[0] + (1) Pyruvate[0]',\n",
       "  'deltag': 3.13,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.40'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00130[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00161',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27982;rxn27983',\n",
       "  'name': '(S)-Malate:NADP+ oxidoreductase(oxaloacetate-decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); GLUCONEO-PWY (gluconeogenesis I); Gluconeogenesis (Gluconeogenesis); PWY-241 (C4 photosynthetic carbon assimilation cycle, NADP-ME type); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); Photosynthesis ()',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00217',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: OXALODECARB-RXN',\n",
       "   'BiGG: OAADC; OXDE',\n",
       "   'EcoCyc: OXALODECARB-RXN',\n",
       "   'KEGG: R00217',\n",
       "   'MetaCyc: OXALODECARB-RXN',\n",
       "   'Name: Oxaloacetate carboxy-lyase; oxalacetic acid decarboxylase; oxalate beta-decarboxylase; oxaloacetate beta-decarboxylase; oxaloacetate carboxy-lyase; oxaloacetate carboxy-lyase (pyruvate-forming); oxaloacetate decarboxylase; oxaloacetate decarboxylate'],\n",
       "  'code': '(1) cpd00032[0] <=> (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00020;cpd00032;cpd00067',\n",
       "  'definition': '(1) Oxaloacetate[0] + (1) H+[0] => (1) CO2[0] + (1) Pyruvate[0]',\n",
       "  'deltag': -3.96,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.38', '1.1.1.40', '4.1.1.112', '4.1.1.3'],\n",
       "  'equation': '(1) cpd00032[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00162',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'oxaloacetate carboxy-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); METHYLGALLATE-DEGRADATION-PWY (methylgallate degradation); P184-PWY (protocatechuate degradation I (<I>meta</I>-cleavage pathway)); PWY-6338 (superpathway of vanillin and vanillate degradation); PWY-6339 (syringate degradation); Protocatechuate-Degradation (Protocatechuate Degradation); Vanillin-Degradation (Vanillin Degradation)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00032:0:0:\"Oxaloacetate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00218',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00218',\n",
       "   'MetaCyc: 4.1.1.78-RXN',\n",
       "   'Name: acetylenedicarboxylate carboxy-lyase; acetylenedicarboxylate carboxy-lyase (pyruvate-forming); acetylenedicarboxylate decarboxylase; acetylenedicarboxylate hydrase; acetylenedicarboxylate hydratase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02080[0] <=> (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00020;cpd00067;cpd02080',\n",
       "  'definition': '(1) H2O[0] + (1) H+[0] + (1) 2-Butynedioic acid[0] => (1) CO2[0] + (1) Pyruvate[0]',\n",
       "  'deltag': -55.73,\n",
       "  'deltagerr': 4.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.78'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00067[0] + (1) cpd02080[0] => (1) cpd00011[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00163',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'acetylenedicarboxylate carboxy-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00067:0:0:\"H+\";-1:cpd02080:0:0:\"2-Butynedioic acid\";1:cpd00011:0:0:\"CO2\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00219',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00219_c',\n",
       "   'AraGEM: R_R00219_c',\n",
       "   'KEGG: R00219',\n",
       "   'Maize_C4GEM: R_R00219_c',\n",
       "   'iRS1597: R00219_c',\n",
       "   'Name: 2-Hydroxyethylenedicarboxylate carboxy-lyase (pyruvate-forming)'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00020[0] <=> (1) cpd02469[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00020;cpd00067;cpd02469',\n",
       "  'definition': '(1) CO2[0] + (1) Pyruvate[0] <= (1) H+[0] + (1) enol-Oxaloacetate[0]',\n",
       "  'deltag': 5.33,\n",
       "  'deltagerr': 1.04,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['4.1.1.-'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00020[0] <= (1) cpd00067[0] + (1) cpd02469[0]',\n",
       "  'id': 'rxn00164',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Hydroxyethylenedicarboxylate carboxy-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\";1:cpd02469:0:0:\"enol-Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00220',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 4.3.1.17-RXN',\n",
       "   'BiGG: CHA1_2; SERD_L; SERD_Lr; SER_AL; YIL167W',\n",
       "   'BrachyCyc: 4.3.1.17-RXN',\n",
       "   'KEGG: R00220',\n",
       "   'MetaCyc: 4.3.1.17-RXN',\n",
       "   'Name: L-hydroxyaminoacid dehydratase; L-serine ammonia-lyase; L-serine ammonia-lyase (pyruvate-forming); L-serine deaminase; L-serine deaminase (rev); L-serine dehydratase; L-serine dehydratase alpha subunit; L-serine hydro-lyase (deaminating); L-serinedeaminase; Serine ammonia-lyase; serine deaminase; threonine ammonia-lyase (serine)'],\n",
       "  'code': '(1) cpd00054[0] <=> (1) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00020;cpd00054',\n",
       "  'definition': '(1) L-Serine[0] => (1) NH3[0] + (1) Pyruvate[0]',\n",
       "  'deltag': -6.93,\n",
       "  'deltagerr': 0.23,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.13', '4.3.1.17', '4.3.1.19'],\n",
       "  'equation': '(1) cpd00054[0] => (1) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00165',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30061;rxn32961;rxn35215',\n",
       "  'name': 'L-serine ammonia-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00260 (Glycine, serine and threonine metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00054:0:0:\"L-Serine\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00221',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DSERDEAM-RXN',\n",
       "   'BiGG: SERD_D',\n",
       "   'BrachyCyc: DSERDEAM-RXN',\n",
       "   'KEGG: R00221',\n",
       "   'MetaCyc: DSERDEAM-RXN',\n",
       "   'Name: D-hydroxy amino acid dehydratase; D-hydroxyaminoacid dehydratase; D-serine ammonia-lyase; D-serine ammonia-lyase (pyruvate-forming); D-serine deaminase; D-serine dehydrase; D-serine dehydratase (deaminating); D-serine hydro-lyase (deaminating); D-serine hydrolase; d-serine dehydratase'],\n",
       "  'code': '(1) cpd00550[0] <=> (1) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00020;cpd00550',\n",
       "  'definition': '(1) D-Serine[0] => (1) NH3[0] + (1) Pyruvate[0]',\n",
       "  'deltag': -6.8,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.18'],\n",
       "  'equation': '(1) cpd00550[0] => (1) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'id': 'rxn00166',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30060;rxn32727',\n",
       "  'name': 'D-serine ammonia-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00550:0:0:\"D-Serine\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\"'},\n",
       " {'abbreviation': 'R00224',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-6161',\n",
       "   'BiGG: PDC1; PDC5; PDC6; PYRDC',\n",
       "   'BrachyCyc: RXN-6161',\n",
       "   'KEGG: R00224',\n",
       "   'MetaCyc: RXN-6161',\n",
       "   'Name: Pyruvate carboxy-lyase; pyruvate carboxy-lyase (acetaldehyde-forming); pyruvate decarboxylase; pyruvatedecarboxylase'],\n",
       "  'code': '(1) cpd00020[0] <=> (1) cpd00011[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00020;cpd00067;cpd00071',\n",
       "  'definition': '(1) Pyruvate[0] + (1) H+[0] => (1) CO2[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': -4.29,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.1'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00168',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn24132;rxn28118',\n",
       "  'name': 'pyruvate carboxy-lyase (acetaldehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetaldehyde-Biosynthesis (Acetaldehyde Biosynthesis); Acetate-Formation (Fermentation to Acetate); Acetoin-Biosynthesis (Pyruvate Fermentation to Acetoin); Alcohol-Biosynthesis (Fermentation to Alcohols); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); PWY-5486 (pyruvate fermentation to ethanol II); PWY-5768 (pyruvate fermentation to acetate VIII); PWY-6330 (acetaldehyde biosynthesis II); PWY-6873 (long chain fatty acid ester synthesis (engineered)); PWY-7118 (chitin degradation to ethanol); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY3O-440 (pyruvate fermentation to acetoin III); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SUCROSE-DEG (Sucrose Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00227',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETYL-COA-HYDROLASE-RXN',\n",
       "   'BiGG: ACH1; ACOAH; ACOAHim',\n",
       "   'KEGG: R00227',\n",
       "   'MetaCyc: ACETYL-COA-HYDROLASE-RXN',\n",
       "   'PlantCyc: ACETYL-COA-HYDROLASE-RXN',\n",
       "   'Name: Acetyl-CoA hydrolase; acetyl coenzyme A acylase; acetyl coenzyme A deacylase; acetyl coenzyme A hydrolase; acetyl-CoA acylase; acetyl-CoA deacylase; acetyl-CoA hydrolase; acetyl-CoA hydrolase, (irreversible); acetyl-CoA thiol esterase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00022[0] <=> (1) cpd00010[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00022;cpd00029;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) Acetyl-CoA[0] => (1) CoA[0] + (1) Acetate[0] + (1) H+[0]',\n",
       "  'deltag': -7.3,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00022[0] => (1) cpd00010[0] + (1) cpd00029[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00170',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30335',\n",
       "  'name': 'Acetyl-CoA hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00228',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETALD-DEHYDROG-RXN',\n",
       "   'BiGG: ACALD; ACALDi',\n",
       "   'ChlamyCyc: ACETALD-DEHYDROG-RXN',\n",
       "   'KEGG: R00228',\n",
       "   'MetaCyc: ACETALD-DEHYDROG-RXN',\n",
       "   'Name: ADA; Acetaldehyde dehydrogenase (acetylating); DmpF; acetaldehyde dehydrogenase (acetylating); acetaldehyde:NAD+ oxidoreductase (CoA-acetylating); acylating acetaldehyde dehyrogenase; alcohol dehydrogenase / acetaldehyde dehydrogenase; aldehyde dehydrogenase (acylating)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00071[0] <=> (1) cpd00004[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00022;cpd00067;cpd00071',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) Acetaldehyde[0] <=> (1) NADH[0] + (1) Acetyl-CoA[0] + (1) H+[0]',\n",
       "  'deltag': -4.18,\n",
       "  'deltagerr': 0.29,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.10'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00071[0] <=> (1) cpd00004[0] + (1) cpd00022[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00171',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27656;rxn32710;rxn32711',\n",
       "  'name': 'acetaldehyde:NAD+ oxidoreductase (CoA-acetylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: 2-Aminoethylphosphonate-Degradation (); AMINE-DEG (Amine and Polyamine Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); Alcohol-Degradation (Alcohol Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Benzoate-Degradation (Benzoate Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); Catechol-Degradation (Catechol Degradation); Degradation (Degradation/Utilization/Assimilation); ETOH-ACETYLCOA-ANA-PWY (ethanol degradation I); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethanol-Degradation (Ethanol Degradation); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); MANDELATE-DEG (Mandelate Degradation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); P122-PWY (heterolactic fermentation); P161-PWY (acetylene degradation); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PHOSPHONOTASE-PWY (2-aminoethylphosphonate degradation I); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5162 (2-hydroxypenta-2,4-dienoate degradation); PWY-5178 (toluene degradation IV (aerobic) (via catechol)); PWY-5179 (toluene degradation V (aerobic) (<I>via</I> toluene-<I>cis</I>-diol)); PWY-5180 (toluene degradation I (aerobic) (<I>via</I> <i>o</I>-cresol)); PWY-5182 (toluene degradation II (aerobic) (via 4-methylcatechol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-5266 (<I>p</I>-cymene degradation); PWY-5273 (<i>p</i>-cumate degradation); PWY-5415 (catechol degradation I (<I>meta</I>-cleavage pathway)); PWY-5420 (catechol degradation II (<I>meta</I>-cleavage pathway)); PWY-5430 (<i>meta</i> cleavage pathway of aromatic compounds); PWY-5436 (L-threonine degradation IV); PWY-5480 (pyruvate fermentation to ethanol I); PWY-5655 (L-tryptophan degradation IX); PWY-6505 (L-tryptophan degradation XII (Geobacillus)); PWY-6587 (pyruvate fermentation to ethanol III); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-6956 (naphthalene degradation to acetyl-CoA); PWY-6957 (mandelate degradation to acetyl-CoA); PWY-7085 (triethylamine degradation); PWY-7180 (2'-deoxy-&alpha;-D-ribose 1-phosphate degradation); PWY0-1277 (3-phenylpropanoate and 3-(3-hydroxyphenyl)propanoate degradation); PWY0-1297 (superpathway of purine deoxyribonucleosides degradation); PWY0-1298 (superpathway of pyrimidine deoxyribonucleosides degradation); PWY0-1477 (ethanolamine utilization); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); Phosphorus-Compounds (Phosphorus Compound Metabolism); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); THREOCAT-PWY (superpathway of L-threonine metabolism); THREONINE-DEG (L-threonine Degradation); TOLUENE-DEG (Toluene Degradation); TRYPTOPHAN-DEG (L-tryptophan Degradation)\",\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism); rn00362 (Benzoate degradation); rn00620 (Pyruvate metabolism); rn00621 (Dioxin degradation); rn00622 (Xylene degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd00071:0:0:\"Acetaldehyde\";1:cpd00004:0:0:\"NADH\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00229',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETATE--COA-LIGASE-ADP-FORMING-RXN',\n",
       "   'BrachyCyc: ACETATE--COA-LIGASE-ADP-FORMING-RXN',\n",
       "   'ChlamyCyc: ACETATE--COA-LIGASE-ADP-FORMING-RXN',\n",
       "   'KEGG: R00229',\n",
       "   'MetaCyc: ACETATE--COA-LIGASE-ADP-FORMING-RXN',\n",
       "   'Name: Acetate thiokinase; Acetate--CoA ligase (ADP-forming); Acetate:CoA ligase (ADP-forming); Acetyl-CoA synthetase (ADP-forming); acetate thiokinase; acetate-CoA ligase (ADP-forming); acetate:CoA ligase (ADP-forming); acetyl coenzyme A synthetase (adenosine diphosphate-forming); acetyl-CoA synthetase (ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00029[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00010;cpd00022;cpd00029',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Acetate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': 1.14,\n",
       "  'deltagerr': 0.09,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.13'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00029[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00172',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27657;rxn27658;rxn31758',\n",
       "  'name': 'Acetate:CoA ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GLUTAMATE-DEG (L-glutamate Degradation); P162-PWY (L-glutamate degradation V (via hydroxyglutarate)); PWY-5096 (pyruvate fermentation to acetate and alanine); PWY-5483 (pyruvate fermentation to acetate III); PWY-5535 (acetate and ATP formation from acetyl-CoA II); PWY-7401 (crotonate fermentation (to acetate and cyclohexane carboxylate)); PWY-7402 (benzoate fermentation (to acetate and cyclohexane carboxylate)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00029:0:0:\"Acetate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00230',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PTA; PTAr',\n",
       "   'BrachyCyc: PHOSACETYLTRANS-RXN',\n",
       "   'ChlamyCyc: PHOSACETYLTRANS-RXN',\n",
       "   'KEGG: R00230',\n",
       "   'MetaCyc: PHOSACETYLTRANS-RXN',\n",
       "   'Name: Acetyl-CoA:orthophosphate acetyltransferase; Phosphate acetyltransferase; Phosphoacylase; Phosphotransacetylase; acetyl-CoA:phosphate acetyltransferase; phosphate acetyltransferase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00022[0] <=> (1) cpd00010[0] + (1) cpd00196[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00010;cpd00022;cpd00196',\n",
       "  'definition': '(1) Phosphate[0] + (1) Acetyl-CoA[0] <=> (1) CoA[0] + (1) Acetylphosphate[0]',\n",
       "  'deltag': 2.26,\n",
       "  'deltagerr': 0.15,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.8'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00022[0] <=> (1) cpd00010[0] + (1) cpd00196[0]',\n",
       "  'id': 'rxn00173',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20541;rxn28067;rxn28068;rxn34298',\n",
       "  'name': 'acetyl-CoA:phosphate acetyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ACETATEUTIL-PWY (superpathway of acetate utilization and formation); AMINE-DEG (Amine and Polyamine Degradation); ANAEROBIC-RESPIRATION (Anaerobic Respiration); ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Fermentation-to-propanoate (Fermentation to Propanoate); GALLATE-DEG (Gallate Degradation); LYSINE-DEG (L-lysine Degradation); METH-ACETATE-PWY (methanogenesis from acetate); METHANOGENESIS (Methanogenesis); NITROGEN-DEG (Nitrogen Compound Metabolism); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); P122-PWY (heterolactic fermentation); P142-PWY (pyruvate fermentation to acetate I); P161-PWY (acetylene degradation); P163-PWY (L-lysine fermentation to acetate and butanoate); P3-PWY (gallate degradation III (anaerobic)); P41-PWY (pyruvate fermentation to acetate and lactate I); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PROPFERM-PWY (L-alanine fermentation to propanoate and acetate); PWY-1281 (sulfoacetaldehyde degradation I); PWY-1541 (superpathway of taurine degradation); PWY-5100 (pyruvate fermentation to acetate and lactate II); PWY-5327 (superpathway of L-lysine degradation); PWY-5482 (pyruvate fermentation to acetate II); PWY-5485 (pyruvate fermentation to acetate IV); PWY-5497 (purine nucleobases degradation II (anaerobic)); PWY-5600 (pyruvate fermentation to acetate VII); PWY-5676 (acetyl-CoA fermentation to butanoate II); PWY-6590 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6637 (sulfolactate degradation II); PWY-6641 (superpathway of sulfolactate degradation); PWY-6830 (superpathway of methanogenesis); PWY0-1312 (acetate and ATP formation from acetyl-CoA I); PWY0-1477 (ethanolamine utilization); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Purine-Degradation (Purine Nucleotide Degradation); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Pyruvate-Lactate-Fermentation (pyruvate fermentation to lactate); Respiration (); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sulfoacetaldehyde-Degradation (Sulfoacetaldehyde Degradation); Sulfolactate-Degradation (Sulfolactate Degradation); Sulfur-Metabolism (Sulfur Compound Metabolism); Taurine-degradation (Taurine Degradation)',\n",
       "   'KEGG: rn00430 (Taurine and hypotaurine metabolism); rn00620 (Pyruvate metabolism); rn00680 (Methane metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00196:0:0:\"Acetylphosphate\"'},\n",
       " {'abbreviation': 'R00233',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MALONYL-COA-DECARBOXYLASE-RXN',\n",
       "   'BiGG: MCD',\n",
       "   'BrachyCyc: MALONYL-COA-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R00233',\n",
       "   'MetaCyc: MALONYL-COA-DECARBOXYLASE-RXN',\n",
       "   'Name: Malonyl-CoA Decarboxylase; Malonyl-CoA carboxy-lyase; Malonyl-CoA decarboxylase; malonyl coenzyme A decarboxylase; malonyl-CoA carboxy-lyase; malonyl-CoA carboxy-lyase (acetyl-CoA-forming); malonyl-CoA decarboxylase'],\n",
       "  'code': '(1) cpd00070[0] <=> (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00022;cpd00067;cpd00070',\n",
       "  'definition': '(1) H+[0] + (1) Malonyl-CoA[0] => (1) CO2[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': -3.95,\n",
       "  'deltagerr': 0.77,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.9'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00070[0] => (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00174',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27986;rxn32740',\n",
       "  'name': 'malonyl-CoA carboxy-lyase (acetyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism); rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00070:0:0:\"Malonyl-CoA\";1:cpd00011:0:0:\"CO2\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00235',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETATE--COA-LIGASE-RXN',\n",
       "   'BiGG: ACS; ACS1; ACSm; ACSp; ACSr',\n",
       "   'BrachyCyc: ACETATE--COA-LIGASE-RXN',\n",
       "   'KEGG: R00235',\n",
       "   'MetaCyc: ACETATE--COA-LIGASE-RXN',\n",
       "   'Name: Acetate thiokinase; Acetate:CoA ligase (AMP-forming); Acetyl-CoA synthase; Acetyl-CoA synthetase; Acetyl-activating enzyme; Acyl-activating enzyme; acetate thiokinase; acetate--CoA ligase; acetate--coa ligase; acetate-CoA ligase; acetate:CoA ligase (AMP-forming); acetic thiokinase; acetyl CoA ligase; acetyl CoA synthase; acetyl activating enzyme; acetyl coenzyme A synthetase; acetyl-CoA synthetase; acetyl-CoA synthetase reversible; acetyl-CoAsynthetase; acetyl-coenzyme A synthase; acyl-activating enzyme; short chain fatty acyl-CoA synthetase; short-chain acyl-coenzyme A synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00029[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00022;cpd00029;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Acetate[0] => (1) PPi[0] + (1) AMP[0] + (1) Acetyl-CoA[0] + (1) H+[0]',\n",
       "  'deltag': -1.13,\n",
       "  'deltagerr': 0.1,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00029[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00022[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00175',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27659;rxn27660;rxn30338',\n",
       "  'name': 'Acetate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ACETATEUTIL-PWY (superpathway of acetate utilization and formation); Alcohol-Degradation (Alcohol Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethanol-Degradation (Ethanol Degradation); PWY-6672 (<i>cis</i>-genanyl-CoA degradation); PWY-7118 (chitin degradation to ethanol); PWY0-1313 (acetate conversion to acetyl-CoA); PWY66-161 (ethanol degradation III); PWY66-162 (ethanol degradation IV); PWY66-21 (ethanol degradation II); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); TERPENOID-DEG (Terpenoid Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00680 (Methane metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00029:0:0:\"Acetate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00236',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00236_c; R_R00236_m',\n",
       "   'AraGEM: R_R00236_c; R_R00236_m',\n",
       "   'KEGG: R00236',\n",
       "   'Maize_C4GEM: R_R00236_c; R_R00236_m',\n",
       "   'TS_Athaliana: rxn563_m; rxn563_p',\n",
       "   'Name: Acetyl adenylate:CoA ligase (AMP-forming); acetyl adenylate:CoA acetyltransferase; acetyl-CoA synthetase (acetate-CoA ligase), putative'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd03566[0] <=> (1) cpd00018[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00018;cpd00022;cpd00067;cpd03566',\n",
       "  'definition': '(1) CoA[0] + (1) Acetyl adenylate[0] => (1) AMP[0] + (1) Acetyl-CoA[0] + (1) H+[0]',\n",
       "  'deltag': -10.42,\n",
       "  'deltagerr': 6.55,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.1'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd03566[0] => (1) cpd00018[0] + (1) cpd00022[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00176',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31208;rxn33484',\n",
       "  'name': 'acetyl adenylate:CoA acetyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd03566:0:0:\"Acetyl adenylate\";1:cpd00018:0:0:\"AMP\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00237',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: CITRAMALYL-COA-LYASE-RXN',\n",
       "   'KEGG: R00237',\n",
       "   'MetaCyc: CITRAMALYL-COA-LYASE-RXN',\n",
       "   'Name: (+)-CMA-CoA lyase; (3S)-Citramalyl-CoA pyruvate-lyase; (3S)-citramalyl-CoA pyruvate-lyase; (3S)-citramalyl-CoA pyruvate-lyase (acetyl-CoA-forming); citramalyl coenzyme A lyase; citramalyl-CoA lyase'],\n",
       "  'code': '(1) cpd00743[0] <=> (1) cpd00020[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00022;cpd00743',\n",
       "  'definition': '(1) L-Citramalyl-CoA[0] <=> (1) Pyruvate[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': 2.81,\n",
       "  'deltagerr': 0.38,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.24', '4.1.3.25'],\n",
       "  'equation': '(1) cpd00743[0] <=> (1) cpd00020[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00177',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(3S)-citramalyl-CoA pyruvate-lyase (acetyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5744 (glyoxylate assimilation); PWY-5749 (itaconate degradation); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle)',\n",
       "   'KEGG: rn00660 (C5-Branched dibasic acid metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00743:0:0:\"L-Citramalyl-CoA\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00238',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETYL-COA-ACETYLTRANSFER-RXN',\n",
       "   'BiGG: AACOAT; ACACT1; ACACT1m; ACACT1r; ERG10; KAT1',\n",
       "   'BrachyCyc: ACETYL-COA-ACETYLTRANSFER-RXN',\n",
       "   'KEGG: R00238',\n",
       "   'MetaCyc: ACETYL-COA-ACETYLTRANSFER-RXN',\n",
       "   'Name: 2-methylacetoacetyl-CoA thiolase [misleading]; 3-oxothiolase; Acetoacetyl_CoA_C_acety_transferase; Acetyl-CoA:acetyl-CoA C-acetyltransferase; acetoacetyl-CoA thiolase; acetyl coenzyme A acetyltransferase; acetyl coenzyme A thiolase; acetyl-CoA C-acetyltransferase; acetyl-CoA C-acyltransferase, putative / 3-ketoacyl-CoA thiolase, putative; acetyl-CoA acetyltransferase; acetyl-CoA:N-acetyltransferase; acetyl-CoA:acetyl-CoA C-acetyltransferase; acetyl-CoAC-acetyltransferase; beta-acetoacetyl coenzyme A thiolase; thiolase II'],\n",
       "  'code': '(2) cpd00022[0] <=> (1) cpd00010[0] + (1) cpd00279[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00279',\n",
       "  'definition': '(2) Acetyl-CoA[0] <=> (1) CoA[0] + (1) Acetoacetyl-CoA[0]',\n",
       "  'deltag': 6.02,\n",
       "  'deltagerr': 0.21,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.9'],\n",
       "  'equation': '(2) cpd00022[0] <=> (1) cpd00010[0] + (1) cpd00279[0]',\n",
       "  'id': 'rxn00178',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19031;rxn31209;rxn31210',\n",
       "  'name': 'Acetyl-CoA:acetyl-CoA C-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ACETOACETATE-DEG-PWY (acetoacetate degradation (to acetyl CoA)); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Alcohol-Biosynthesis (Fermentation to Alcohols); Alkene-Degradation (Alkene Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CENTFERM-PWY (pyruvate fermentation to butanoate); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Degradation (Carbohydrate Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); DITERPENOID-SYN (Diterpenoid Biosynthesis); Degradation (Degradation/Utilization/Assimilation); ERGOSTEROL-SYN-PWY (superpathway of ergosterol biosynthesis I); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ergosterol-Biosynthesis (Ergosterol Biosynthesis); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GGPP-Biosynthesis (Geranylgeranyl Diphosphate Biosynthesis); GLUDEG-II-PWY (L-glutamate degradation VII (to butanoate)); GLUTAMATE-DEG (L-glutamate Degradation); ISOPRENOIDS (Hemiterpene Biosynthesis); Isopentenyl-Diphosphate-Biosynthesis (Isopentenyl Diphosphate Biosynthesis); LYSINE-DEG (L-lysine Degradation); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); Mevalonate-Pathways (Mevalonate Pathways); OTHER-ENERGY (Other); P162-PWY (L-glutamate degradation V (via hydroxyglutarate)); P163-PWY (L-lysine fermentation to acetate and butanoate); PWY-5177 (glutaryl-CoA degradation); PWY-5184 (toluene degradation VI (anaerobic)); PWY-5327 (superpathway of L-lysine degradation); PWY-5676 (acetyl-CoA fermentation to butanoate II); PWY-5741 (ethylmalonyl-CoA pathway); PWY-5789 (3-hydroxypropanoate/4-hydroxybutanate cycle); PWY-5910 (superpathway of geranylgeranyldiphosphate biosynthesis I (via mevalonate)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6174 (mevalonate pathway II (archaea)); PWY-6583 (pyruvate fermentation to butanol I); PWY-6588 (pyruvate fermentation to acetone); PWY-6590 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic fermentation); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6863 (pyruvate fermentation to hexanol (engineered)); PWY-6876 (isopropanol biosynthesis (engineered)); PWY-6883 (pyruvate fermentation to butanol II (engineered)); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7003 (glycerol degradation to butanol); PWY-7216 ((R)- and (S)-3-hydroxybutanoate biosynthesis (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); PWY-7391 (isoprene biosynthesis II (engineered)); PWY-7401 (crotonate fermentation (to acetate and cyclohexane carboxylate)); PWY-7402 (benzoate fermentation (to acetate and cyclohexane carboxylate)); PWY-7524 (mevalonate pathway III (archaea)); PWY-7778 (2-methylpropene degradation); PWY-7779 (methyl <i>tert</i>-butyl ether degradation); PWY-922 (mevalonate pathway I); PWY1-3 (polyhydroxybutanoate biosynthesis); PWY66-367 (ketogenesis); PWY66-368 (ketolysis); PWY66-5 (superpathway of cholesterol biosynthesis); Polyprenyl-Biosynthesis (Polyprenyl Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Butanol-Fermentation (Pyruvate Fermentation to Butanol); Pyruvate-Degradation (Fermentation of Pyruvate); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Sterol-Biosynthesis (Sterol Biosynthesis); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis); TOLUENE-DEG (Toluene Degradation); TRYPTOPHAN-DEG (L-tryptophan Degradation); TRYPTOPHAN-DEGRADATION-1 (L-tryptophan degradation III (eukaryotic)); Terpenoid-Biosynthesis (Terpenoid Biosynthesis)',\n",
       "   'KEGG: rn00071 (Fatty acid degradation); rn00072 (Synthesis and degradation of ketone bodies); rn00280 (Valine, leucine and isoleucine degradation); rn00310 (Lysine degradation); rn00362 (Benzoate degradation); rn00380 (Tryptophan metabolism); rn00620 (Pyruvate metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00640 (Propanoate metabolism); rn00650 (Butanoate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn00900 (Terpenoid backbone biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00279:0:0:\"Acetoacetyl-CoA\"'},\n",
       " {'abbreviation': 'R00239',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTKIN-RXN',\n",
       "   'BiGG: GLU5K',\n",
       "   'BrachyCyc: GLUTKIN-RXN',\n",
       "   'KEGG: R00239',\n",
       "   'MetaCyc: GLUTKIN-RXN',\n",
       "   'Name: ATP-L-glutamate 5-phosphotransferase; ATP:L-glutamate 5-phosphotransferase; ATP:gamma-L-glutamate phosphotransferase; gamma-glutamate kinase; gamma-glutamyl kinase; glutamate 5-kinase; glutamate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd02097[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00023;cpd02097',\n",
       "  'definition': '(1) ATP[0] + (1) L-Glutamate[0] <=> (1) ADP[0] + (1) L-Glutamyl 5-phosphate[0]',\n",
       "  'deltag': 6.51,\n",
       "  'deltagerr': 0.55,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.11'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd02097[0]',\n",
       "  'id': 'rxn00179',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31391;rxn38122;rxn38123',\n",
       "  'name': 'ATP:L-glutamate 5-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGININE-SYN4-PWY (L-ornithine biosynthesis II); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); CITRULBIO-PWY (L-citrulline biosynthesis); Citrulline-Biosynthesis (L-citrulline Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PROLINE-SYN (L-proline Biosynthesis); PROSYN-PWY (L-proline biosynthesis I); PWY-3341 (L-proline biosynthesis III); PWY-5004 (superpathway of L-citrulline metabolism); PWY-6922 (L-<i>N<sup>&delta;</sup></i>-acetylornithine biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00008:0:0:\"ADP\";1:cpd02097:0:0:\"L-Glutamyl 5-phosphate\"'},\n",
       " {'abbreviation': 'PRO1',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PRO1', 'iIN800: PRO1', 'Name: Glutamate 5-kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd02339[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00023;cpd02339',\n",
       "  'definition': '(1) ATP[0] + (1) L-Glutamate[0] <=> (1) ADP[0] + (1) alpha-D-Glutamyl phosphate[0]',\n",
       "  'deltag': 5.82,\n",
       "  'deltagerr': 0.55,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd02339[0]',\n",
       "  'id': 'rxn00180',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glutamate 5-kinase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Secondary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00008:0:0:\"ADP\";1:cpd02339:0:0:\"alpha-D-Glutamyl phosphate\"'},\n",
       " {'abbreviation': 'R00241',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00241',\n",
       "   'MetaCyc: GLUTAMATE-1-KINASE-RXN',\n",
       "   'Name: ATP:L-glutamate 1-phosphotransferase; glutamate 1-kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd03413[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00023;cpd03413',\n",
       "  'definition': '(1) ATP[0] + (1) L-Glutamate[0] <=> (1) ADP[0] + (1) L-Glutamyl 1-phosphate[0]',\n",
       "  'deltag': 5.82,\n",
       "  'deltagerr': 0.55,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.13'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd03413[0]',\n",
       "  'id': 'rxn00181',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:L-glutamate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00008:0:0:\"ADP\";1:cpd03413:0:0:\"L-Glutamyl 1-phosphate\"'},\n",
       " {'abbreviation': 'R00243',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTAMATE-DEHYDROGENASE-NADP+-RXN; GLUTAMATE-DEHYDROGENASE-RXN',\n",
       "   'BiGG: GDH2; GLUDx; GLUDxi',\n",
       "   'BrachyCyc: GLUTAMATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'KEGG: R00243',\n",
       "   'MetaCyc: GLUTAMATE-DEHYDROGENASE-NADP+-RXN; GLUTAMATE-DEHYDROGENASE-RXN',\n",
       "   'Name: L-Glutamate:NAD+ oxidoreductase (deaminating); L-glutamate dehydrogenase; L-glutamate:NAD+ oxidoreductase (deaminating); NAD-dependent glutamate dehydrogenase; NAD-dependent glutamic dehydrogenase; NAD-glutamate dehydrogenase; NAD-linked glutamate dehydrogenase; NAD-linked glutamic dehydrogenase; NAD-specific glutamate dehydrogenase; NAD-specific glutamic dehydrogenase; NAD:glutamate oxidoreductase; NADH-linked glutamate dehydrogenase; glutamate dehydrogenase; glutamate dehydrogenase (NAD); glutamate oxidoreductase; glutamic acid dehydrogenase; glutamic dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00023[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00023;cpd00024;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Glutamate[0] <=> (1) NADH[0] + (1) NH3[0] + (1) 2-Oxoglutarate[0] + (1) H+[0]',\n",
       "  'deltag': 8.71,\n",
       "  'deltagerr': 0.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.2', '1.4.1.3', '1.4.1.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00023[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00182',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19827;rxn30075;rxn34790;rxn34793',\n",
       "  'name': 'L-glutamate:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Aminobutyraye-Degradation (4-Aminobutanoate Degradation); ALACAT2-PWY (L-alanine degradation II (to D-lactate)); ALANINE-DEG (L-alanine Degradation); AMINE-DEG (Amine and Polyamine Degradation); ARGININE-SYN4-PWY (L-ornithine biosynthesis II); Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); GLUDEG-I-PWY (GABA shunt); GLUTAMATE-DEG (L-glutamate Degradation); GLUTAMATE-DEG1-PWY (L-glutamate degradation I); GLUTAMATE-SYN (L-glutamate Biosynthesis); GLUTAMATE-SYN2-PWY (L-glutamate biosynthesis II); GLUTAMINE-SYN (L-glutamine Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); P162-PWY (L-glutamate degradation V (via hydroxyglutarate)); PROPFERM-PWY (L-alanine fermentation to propanoate and acetate); PWY-5022 (4-aminobutanoate degradation V); PWY-5505 (L-glutamate and L-glutamine biosynthesis); PWY-5766 (L-glutamate degradation X); PWY-6728 (methylaspartate cycle); PWY-7126 (ethylene biosynthesis IV (engineered)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism); rn00430 (Taurine and hypotaurine metabolism); rn00471 (D-Glutamine and D-glutamate metabolism); rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00245',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00245_c',\n",
       "   'AraGEM: R_R00245_c',\n",
       "   'BiGG: GSADH; PUTA3',\n",
       "   'KEGG: R00245',\n",
       "   'MetaCyc: RXN-14116',\n",
       "   'Name: L-Glutamate 5-semialdehyde:NAD+ oxidoreductase; L-glutamate gamma-semialdehyde:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00858[0] <=> (1) cpd00004[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00023;cpd00067;cpd00858',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Glutamate5-semialdehyde[0] <=> (1) NADH[0] + (1) L-Glutamate[0] + (2) H+[0]',\n",
       "  'deltag': -10.34,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.88', '1.5.1.12'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00858[0] <=> (1) cpd00004[0] + (1) cpd00023[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00183',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33819;rxn33820',\n",
       "  'name': 'L-Glutamate 5-semialdehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGASEDEG-PWY (L-arginine degradation I (arginase pathway)); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Ethylene-Biosynthesis (Ethylene Biosynthesis); GLUTAMATE-SYN (L-glutamate Biosynthesis); HORMONE-SYN (Hormone Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PROLINE-DEG (L-proline Degradation); PROUT-PWY (L-proline degradation); PWY-6853 (ethylene biosynthesis II (microbes)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00858:0:0:\"L-Glutamate5-semialdehyde\";1:cpd00004:0:0:\"NADH\";1:cpd00023:0:0:\"L-Glutamate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00248',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTAMATE-DEHYDROGENASE-NADP+-RXN; GLUTDEHYD-RXN',\n",
       "   'BiGG: GDH1; GDH3; GLUDy; GLUDyi',\n",
       "   'BrachyCyc: GLUTAMATE-DEHYDROGENASE-NADP+-RXN; GLUTDEHYD-RXN',\n",
       "   'KEGG: R00248',\n",
       "   'MetaCyc: GLUTAMATE-DEHYDROGENASE-NADP+-RXN; GLUTDEHYD-RXN',\n",
       "   'Name: GLUTAMATE-DEHYDROGENASE-(NAD(P)+)-RXN-(NADP); Glutamate Dehyrdogenase (NADP); Glutamate dehydrogenase (NAD(P)+); Glutamic dehydrogenase; L-Glutamate:NADP+ oxidoreductase (deaminating); L-glutamate dehydrogenase; L-glutamate:NADP+ oxidoreductase (deaminating); L-glutamic acid dehydrogenase; NAD(P)-glutamate dehydrogenase; NAD(P)H-dependent glutamate dehydrogenase; NADP-specific glutamate dehydrogenase 1; dehydrogenase, glutamate (nicotinamide adenine dinucleotide (phosphate)); glutamate dehydrogenase (NADP); glutamate dehydrogenase (NADP+); glutamate dehydrogenase (nadp+); glutamate dehydrogenase 2; glutamate dehydrogenase [NAD(P)+]; glutamatedehydrogenase-NADP; glutamic acid dehydrogenase; glutamic dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00023[0] <=> (1) cpd00005[0] + (1) cpd00013[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00013;cpd00023;cpd00024;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) L-Glutamate[0] <=> (1) NADPH[0] + (1) NH3[0] + (1) 2-Oxoglutarate[0] + (1) H+[0]',\n",
       "  'deltag': 8.72,\n",
       "  'deltagerr': 0.09,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.3', '1.4.1.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00023[0] <=> (1) cpd00005[0] + (1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00184',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27887;rxn30023;rxn33943;rxn34794',\n",
       "  'name': 'L-Glutamate:NADP+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Aminobutyraye-Degradation (4-Aminobutanoate Degradation); AMINE-DEG (Amine and Polyamine Degradation); ARGININE-SYN4-PWY (L-ornithine biosynthesis II); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); GLUDEG-I-PWY (GABA shunt); GLUTAMATE-DEG (L-glutamate Degradation); GLUTAMATE-SYN (L-glutamate Biosynthesis); GLUTAMATE-SYN2-PWY (L-glutamate biosynthesis II); GLUTAMINE-SYN (L-glutamine Biosynthesis); GLUTSYNIII-PWY (L-glutamate biosynthesis III); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); Metabolic-Clusters (Metabolic Clusters); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitrate-Reduction (Nitrate Reduction); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-5505 (L-glutamate and L-glutamine biosynthesis); PWY-5675 (nitrate reduction V (assimilatory)); PWY-5766 (L-glutamate degradation X); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY490-3 (nitrate reduction VI (assimilatory)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism); rn00471 (D-Glutamine and D-glutamate metabolism); rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00005:0:0:\"NADPH\";1:cpd00013:0:0:\"NH3\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00250',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00250',\n",
       "   'MetaCyc: L-GLUTAMATE-OXIDASE-RXN',\n",
       "   'Name: L-Glutamate:oxygen oxidoreductase (deaminating); L-glutamate oxidase; L-glutamate:oxygen oxidoreductase (deaminating); L-glutamic acid oxidase; glutamate (acceptor) dehydrogenase; glutamate oxidase; glutamic acid oxidase; glutamic dehydrogenase (acceptor)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00023[0] <=> (1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00023;cpd00024;cpd00025',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Glutamate[0] => (1) NH3[0] + (1) 2-Oxoglutarate[0] + (1) H2O2[0]',\n",
       "  'deltag': -23.01,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00023[0] => (1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00185',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Glutamate:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00013:0:0:\"NH3\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00251',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5-OXOPROLINASE-ATP-HYDROLYSING-RXN',\n",
       "   'BiGG: OPAH',\n",
       "   'BrachyCyc: 5-OXOPROLINASE-ATP-HYDROLYSING-RXN',\n",
       "   'KEGG: R00251',\n",
       "   'MetaCyc: 5-OXOPROLINASE-ATP-HYDROLYSING-RXN',\n",
       "   'Name: 5-OXOPROLINASE-(ATP-HYDROLYSING)-RXN; 5-Oxoproline amidohydrolase (ATP-hydrolysing); 5-oxo-L-prolinase; 5-oxo-L-proline amidohydrolase (ATP-hydrolysing); 5-oxoprolinase; 5-oxoprolinase (ATP-hydrolysing); L-pyroglutamate hydrolase; oxoprolinase; pyroglutamase; pyroglutamase (ATP-hydrolysing); pyroglutamate hydrolase; pyroglutamic hydrolase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd00002[0] + (1) cpd01293[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00008;cpd00009;cpd00023;cpd00067;cpd01293',\n",
       "  'definition': '(2) H2O[0] + (1) ATP[0] + (1) 5-Oxoproline[0] => (1) ADP[0] + (1) Phosphate[0] + (1) L-Glutamate[0] + (1) H+[0]',\n",
       "  'deltag': -0.72,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.2.9'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd00002[0] + (1) cpd01293[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00023[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00186',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30590',\n",
       "  'name': '5-oxo-L-proline amidohydrolase (ATP-hydrolysing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Macromolecule-Modification (Macromolecule Modification); PWY-4041 (&gamma;-glutamyl cycle); PWY-7942 (5-oxo-L-proline metabolism); Protein-Modification (Protein Modification); Reductants (Reductant Biosynthesis)',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd01293:0:0:\"5-Oxoproline\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00253',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTAMINESYN-RXN',\n",
       "   'BiGG: GLN1; GLNS; GLNS_i',\n",
       "   'BrachyCyc: GLUTAMINESYN-RXN',\n",
       "   'KEGG: R00253',\n",
       "   'MetaCyc: GLUTAMINESYN-RXN',\n",
       "   'Name: Glutamine synthetase; L-Glutamate:ammonia ligase (ADP-forming); L-glutamate:ammonia ligase (ADP-forming); L-glutamine synthetase; glutamate---ammonia ligase; glutamate--ammonia ligase; glutamate-ammonia ligase; glutamine synthetase; glutaminesynthetase; glutamylhydroxamic synthetase; type I glutamate--ammonia ligase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00023[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00053[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00013;cpd00023;cpd00053;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) L-Glutamate[0] => (1) ADP[0] + (1) Phosphate[0] + (1) L-Glutamine[0] + (1) H+[0]',\n",
       "  'deltag': -3.34,\n",
       "  'deltagerr': 0.2,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.1.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00023[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00053[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00187',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19835;rxn19836;rxn30022;rxn34791;rxn34792',\n",
       "  'name': 'L-Glutamate:ammonia ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMMASSIM-PWY (ammonia assimilation cycle III); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Ammonia-Assimilation (Ammonia Assimilation); Degradation (Degradation/Utilization/Assimilation); GLNSYN-PWY (L-glutamine biosynthesis I); GLUTAMATE-SYN (L-glutamate Biosynthesis); GLUTAMINE-SYN (L-glutamine Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Metabolic-Clusters (Metabolic Clusters); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitrate-Reduction (Nitrate Reduction); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-3282 (superpathway of ammonia assimilation (plants)); PWY-381 (nitrate reduction II (assimilatory)); PWY-5505 (L-glutamate and L-glutamine biosynthesis); PWY-5675 (nitrate reduction V (assimilatory)); PWY-6549 (L-glutamine biosynthesis III); PWY-6963 (ammonia assimilation cycle I); PWY-6964 (ammonia assimilation cycle II); PWY490-3 (nitrate reduction VI (assimilatory))',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00910 (Nitrogen metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00053:0:0:\"L-Glutamine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00254',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00254', 'Name: N-carbamyl-L-glutamate amidohydrolase'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00023[0] <=> (1) cpd00001[0] + (1) cpd03460[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00023;cpd00067;cpd03460',\n",
       "  'definition': '(1) CO2[0] + (1) NH3[0] + (1) L-Glutamate[0] <= (1) H2O[0] + (2) H+[0] + (1) N-Carbamyl-L-glutamate[0]',\n",
       "  'deltag': 13.21,\n",
       "  'deltagerr': 1.19,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['3.5.1.87'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00023[0] <= (1) cpd00001[0] + (2) cpd00067[0] + (1) cpd03460[0]',\n",
       "  'id': 'rxn00188',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-carbamyl-L-glutamate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00340 (Histidine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00013:0:0:\"NH3\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00001:0:0:\"H2O\";2:cpd00067:0:0:\"H+\";1:cpd03460:0:0:\"N-Carbamyl-L-glutamate\"'},\n",
       " {'abbreviation': 'R00256',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTAMIN-RXN',\n",
       "   'BiGG: GLUN; GLUNpp; U42; U43',\n",
       "   'BrachyCyc: GLUTAMIN-RXN',\n",
       "   'KEGG: R00256',\n",
       "   'MetaCyc: GLUTAMIN-RXN; GLUTAMINASE-ASPARAGIN-ASE-RXN',\n",
       "   'Name: Glutaminase A; L-Glutamine amidohydrolase; L-glutaminase; L-glutamine amidohydrolase; glutaminase; glutaminase I; glutamine aminohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00053[0] <=> (1) cpd00013[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00023;cpd00053',\n",
       "  'definition': '(1) H2O[0] + (1) L-Glutamine[0] => (1) NH3[0] + (1) L-Glutamate[0]',\n",
       "  'deltag': -2.82,\n",
       "  'deltagerr': 0.21,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.2', '3.5.1.38', '6.3.5.4', '6.3.5.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00053[0] => (1) cpd00013[0] + (1) cpd00023[0]',\n",
       "  'id': 'rxn00189',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19831;rxn29962;rxn30024;rxn30680;rxn34787',\n",
       "  'name': 'L-Glutamine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CITRULBIO-PWY (L-citrulline biosynthesis); Citrulline-Biosynthesis (L-citrulline Biosynthesis); Degradation (Degradation/Utilization/Assimilation); GLUTAMINDEG-PWY (L-glutamine degradation I); GLUTAMINE-DEG (L-glutamine Degradation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-5004 (superpathway of L-citrulline metabolism); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism); rn00471 (D-Glutamine and D-glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00053:0:0:\"L-Glutamine\";1:cpd00013:0:0:\"NH3\";1:cpd00023:0:0:\"L-Glutamate\"'},\n",
       " {'abbreviation': 'R00257',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NAD-SYNTH-GLN-RXN',\n",
       "   'BiGG: NADS2; NADS2i',\n",
       "   'BrachyCyc: NAD-SYNTH-GLN-RXN',\n",
       "   'KEGG: R00257',\n",
       "   'MetaCyc: NAD-SYNTH-GLN-RXN',\n",
       "   'Name: DPN synthetase; Deamido-NAD+:L-glutamine amido-ligase (AMP-forming); NAD synthase (glutamine-hydrolysing); NAD synthetase (glutamine-hydrolysing); NAD(+) synthase (glutamine-hydrolysing); NAD+ synthase (glutamine-hydrolysing); NAD-synthetase; deamido-NAD+:L-glutamine amido-ligase (AMP-forming); desamidonicotinamide adenine dinucleotide amidotransferase; nad(+) synthase (glutamine-hydrolysing); nicotinamide adenine dinucleotide synthetase (glutamine)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00053[0] + (1) cpd00638[0] <=> (1) cpd00003[0] + (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00003;cpd00012;cpd00018;cpd00023;cpd00053;cpd00067;cpd00638',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) L-Glutamine[0] + (1) Deamido-NAD[0] => (1) NAD[0] + (1) PPi[0] + (1) AMP[0] + (1) L-Glutamate[0] + (2) H+[0]',\n",
       "  'deltag': -6.51,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.5.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00053[0] + (1) cpd00638[0] => (1) cpd00003[0] + (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00023[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00190',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33083',\n",
       "  'name': 'Deamido-NAD+:L-glutamine amido-ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); NADSYN-PWY (NAD <i>de novo</i> biosynthesis II (from tryptophan)); PWY-3502 (superpathway of NAD biosynthesis in eukaryotes); PWY-5381 (pyridine nucleotide cycling (plants)); PWY-5653 (NAD biosynthesis from 2-amino-3-carboxymuconate semialdehyde); PWY-7761 (NAD salvage pathway II (PNC IV cycle)); PWY0-781 (aspartate superpathway); PWY3O-4107 (NAD salvage pathway V (PNC V cycle)); PYRIDNUCSAL-PWY (NAD salvage pathway I (PNC VI cycle)); PYRIDNUCSYN-PWY (NAD <i>de novo</i> biosynthesis I (from aspartate))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00638:0:0:\"Deamido-NAD\";1:cpd00003:0:0:\"NAD\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00023:0:0:\"L-Glutamate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00258',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ALANINE-AMINOTRANSFERASE-RXN; RXN-13698',\n",
       "   'BiGG: ALATA_L; ALATA_Lm; YDR111C; YLR089C',\n",
       "   'BrachyCyc: ALANINE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00258',\n",
       "   'MetaCyc: ALANINE-AMINOTRANSFERASE-RXN; RXN-13698',\n",
       "   'Name: ALT; Alanine aminotransferase; Alanine transaminase; GPT; GPT (ambiguous); Glutamic--alanine transaminase; Glutamic--pyruvic transaminase; L-Alanine:2-oxoglutarate aminotransferase; L-alanine aminotransferase; L-alanine transaminase; L-alanine-alpha-ketoglutarate aminotransferase; L-alanine:2-oxoglutarate aminotransferase; Probable alanine aminotransferase; alanine aminotransferase; alanine aminotransferase, putative; alanine transaminase; alanine-alpha-ketoglutarate aminotransferase; alanine-pyruvate aminotransferase; beta-alanine aminotransferase; glutamic acid-pyruvic acid transaminase; glutamic-alanine transaminase; glutamic-pyruvic aminotransferase; glutamic-pyruvic transaminase; pyruvate transaminase; pyruvate-alanine aminotransferase; pyruvate-glutamate transaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00035[0] <=> (1) cpd00020[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00023;cpd00024;cpd00035',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Alanine[0] <=> (1) Pyruvate[0] + (1) L-Glutamate[0]',\n",
       "  'deltag': 0.07,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.2'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00035[0] <=> (1) cpd00020[0] + (1) cpd00023[0]',\n",
       "  'id': 'rxn00191',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27697;rxn27698;rxn31211;rxn38028;rxn38029',\n",
       "  'name': 'L-Alanine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALACAT2-PWY (L-alanine degradation II (to D-lactate)); ALANINE-DEG (L-alanine Degradation); ALANINE-DEG3-PWY (L-alanine degradation III); ALANINE-SYN (L-alanine Biosynthesis); ALANINE-SYN2-PWY (L-alanine biosynthesis II); Acetate-Formation (Fermentation to Acetate); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PROPFERM-PWY (L-alanine fermentation to propanoate and acetate); PWY-5096 (pyruvate fermentation to acetate and alanine); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-7115 (C4 photosynthetic carbon assimilation cycle, NAD-ME type); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY0-1061 (superpathway of L-alanine biosynthesis); Photosynthesis (); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00035:0:0:\"L-Alanine\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00023:0:0:\"L-Glutamate\"'},\n",
       " {'abbreviation': 'R00259',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: N-ACETYLTRANSFER-RXN',\n",
       "   'BiGG: ACGS; ACGSm; ECM40_1',\n",
       "   'BrachyCyc: N-ACETYLTRANSFER-RXN',\n",
       "   'KEGG: R00259',\n",
       "   'MetaCyc: N-ACETYLTRANSFER-RXN',\n",
       "   'Name: AGAS; Acetyl-CoA:L-glutamate N-acetyltransferase; Amino-acid N-acetyltransferase; Amino-acid acetyltransferase; N-acetyl-L-glutamate synthetase; N-acetylglutamate synthase; N-acetylglutamate synthetase; N-acteylglutamate synthase; acetyl-CoA:L-glutamate N-acetyltransferase; acetylglutamate acetylglutamate synthetase; acetylglutamic synthetase; amino acid acetyltransferase; amino-acid N-acetyltransferase; amino-acid n-acetyltransferase; glutamate N-acetyltransferase / amino-acid N-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00023[0] <=> (1) cpd00010[0] + (1) cpd00477[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00023;cpd00067;cpd00477',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Glutamate[0] => (1) CoA[0] + (1) H+[0] + (1) N-Acetyl-L-glutamate[0]',\n",
       "  'deltag': -5.12,\n",
       "  'deltagerr': 0.81,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.1'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00023[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00477[0]',\n",
       "  'id': 'rxn00192',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30107;rxn33610',\n",
       "  'name': 'acetyl-CoA:L-glutamate N-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGININE-SYN (L-arginine Biosynthesis); ARGSYN-PWY (L-arginine biosynthesis I (via L-ornithine)); ARGSYNBSUB-PWY (L-arginine biosynthesis II (acetyl cycle)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); GLUTORN-PWY (L-ornithine biosynthesis I); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-5154 (L-arginine biosynthesis III (via <i>N</i>-acetyl-L-citrulline)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd00477:0:0:\"N-Acetyl-L-glutamate\"'},\n",
       " {'abbreviation': 'R00260',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GLUR',\n",
       "   'BrachyCyc: GLUTRACE-RXN',\n",
       "   'EcoCyc: GLUTRACE-RXN',\n",
       "   'KEGG: R00260',\n",
       "   'MetaCyc: GLUTRACE-RXN',\n",
       "   'Name: L-Glutamate racemase; glutamate racemase; glutamateracemase'],\n",
       "  'code': '(1) cpd00023[0] <=> (1) cpd00186[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00186',\n",
       "  'definition': '(1) L-Glutamate[0] <=> (1) D-Glutamate[0]',\n",
       "  'deltag': 0.13,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.3'],\n",
       "  'equation': '(1) cpd00023[0] <=> (1) cpd00186[0]',\n",
       "  'id': 'rxn00193',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glutamate racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetylmuramoyl-Pentapeptide-Biosynthesis (UDP-<i>N</i>-Acetylmuramoyl-Pentapeptide Biosynthesis); Antibiotic-Resistance (Antibiotic Resistance); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Detoxification (Detoxification); PEPTIDOGLYCANSYN-PWY (peptidoglycan biosynthesis I (<I>meso</I>-diaminopimelate containing)); PWY-5265 (peptidoglycan biosynthesis II (staphylococci)); PWY-6385 (peptidoglycan biosynthesis III (mycobacteria)); PWY-6386 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis II (lysine-containing)); PWY-6387 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis I (<i>meso</i>-diaminopimelate containing)); PWY-6470 (peptidoglycan biosynthesis V (&beta;-lactam resistance)); PWY-6471 (peptidoglycan biosynthesis IV (Enterococcus faecium)); Peptidoglycan-Biosynthesis (Peptidoglycan Biosynthesis)',\n",
       "   'KEGG: rn00471 (D-Glutamine and D-glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00023:0:0:\"L-Glutamate\";1:cpd00186:0:0:\"D-Glutamate\"'},\n",
       " {'abbreviation': 'R00261',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTDECARBOX-RXN',\n",
       "   'BiGG: GAD1; GLUDC',\n",
       "   'BrachyCyc: GLUTDECARBOX-RXN',\n",
       "   'KEGG: R00261',\n",
       "   'MetaCyc: GLUTDECARBOX-RXN',\n",
       "   'Name: Glutamate Decarboxylase; GlutamateDecarboxylase; L-Glutamate 1-carboxy-lyase; L-aspartate-alpha-decarboxylase; L-glutamate 1-carboxy-lyase; L-glutamate 1-carboxy-lyase (4-aminobutanoate-forming); L-glutamate alpha-decarboxylase; L-glutamic acid decarboxylase; L-glutamic decarboxylase; arginine decarboxylase; aspartate 1-decarboxylase; aspartic alpha-decarboxylase; cysteic acid decarboxylase; gamma-glutamate decarboxylase; glutamate decarboxylase'],\n",
       "  'code': '(1) cpd00023[0] <=> (1) cpd00011[0] + (1) cpd00281[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00023;cpd00067;cpd00281',\n",
       "  'definition': '(1) L-Glutamate[0] + (1) H+[0] => (1) CO2[0] + (1) GABA[0]',\n",
       "  'deltag': -5.69,\n",
       "  'deltagerr': 0.85,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.15'],\n",
       "  'equation': '(1) cpd00023[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00281[0]',\n",
       "  'id': 'rxn00194',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34786',\n",
       "  'name': 'L-glutamate 1-carboxy-lyase (4-aminobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Aminobutyraye-Degradation (4-Aminobutanoate Degradation); AMINE-DEG (Amine and Polyamine Degradation); Acid-Resistance (Acid Resistance); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); GLUDEG-I-PWY (GABA shunt); GLUTAMATE-DEG (L-glutamate Degradation); PWY-4321 (L-glutamate degradation IV); PWY0-1305 (L-glutamate degradation IX (via 4-aminobutanoate)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00023:0:0:\"L-Glutamate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00281:0:0:\"GABA\"'},\n",
       " {'abbreviation': 'R00262',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: METHYLASPARTATE-MUTASE-RXN',\n",
       "   'KEGG: R00262',\n",
       "   'MetaCyc: METHYLASPARTATE-MUTASE-RXN',\n",
       "   'Name: L-threo-3-Methylaspartate carboxy-aminomethylmutase; L-threo-3-methylaspartate carboxy-aminomethylmutase; beta-methylaspartate-glutamate mutase; glutamate isomerase; glutamate mutase; glutamic acid isomerase; glutamic acid mutase; glutamic isomerase; glutamic mutase; methylaspartate mutase; methylaspartic acid mutase'],\n",
       "  'code': '(1) cpd02273[0] <=> (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00023;cpd02273',\n",
       "  'definition': '(1) L-threo-3-Methylaspartate[0] <=> (1) L-Glutamate[0]',\n",
       "  'deltag': -1.77,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.99.1'],\n",
       "  'equation': '(1) cpd02273[0] <=> (1) cpd00023[0]',\n",
       "  'id': 'rxn00195',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-threo-3-Methylaspartate carboxy-aminomethylmutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); GLUDEG-II-PWY (L-glutamate degradation VII (to butanoate)); GLUTAMATE-DEG (L-glutamate Degradation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); ISOLEUCINE-SYN (L-isoleucine Biosynthesis); PWY-5087 (L-glutamate degradation VI (to pyruvate)); PWY-5088 (L-glutamate degradation VIII (to propanoate)); PWY-5103 (L-isoleucine biosynthesis III); PWY-6728 (methylaspartate cycle); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00660 (C5-Branched dibasic acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02273:0:0:\"L-threo-3-Methylaspartate\";1:cpd00023:0:0:\"L-Glutamate\"'},\n",
       " {'abbreviation': 'R00264',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00264',\n",
       "   'MetaCyc: 25-DIOXOVALERATE-DEHYDROGENASE-RXN; RXN-8775',\n",
       "   'iAbaylyiv4: 2,5-DIOXOVALERATE-DEHYDROGENASE-RXN',\n",
       "   'iRS1563: R_R00264_c',\n",
       "   'Name: 2,5-dioxopentanoate:NADP+ 5-oxidoreductase; 2,5-dioxovalerate dehydrogenase; 2-oxoglutarate semialdehyde dehydrogenase; alpha-ketoglutaric semialdehyde dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00340[0] <=> (1) cpd00005[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00024;cpd00067;cpd00340',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) 2,5-Dioxopentanoate[0] <=> (1) NADPH[0] + (1) 2-Oxoglutarate[0] + (2) H+[0]',\n",
       "  'deltag': -8.47,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.26', '1.2.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00340[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00196',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2,5-dioxopentanoate:NADP+ 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); D-Arabinose-Degradation (D-arabinose Degradation); D-Galactarate-Degradation (D-Galactarate Degradation); D-Galacturonate-Degradation (D-Galacturonate Degradation); D-Glucarate-Degradation (D-Glucarate Degradation); D-Glucuronate-Degradation (D-Glucuronate Degradation); Degradation (Degradation/Utilization/Assimilation); Hydroxyprolines-Degradation (hydroxy-L-proline Degradation); L-Arabinose-Degradation (L-arabinose Degradation); MISCELLANEOUS-DEG (Other Amino Acid Degradation); PWY-5159 (<i>trans</i>-4-hydroxy-L-proline degradation II); PWY-5257 (superpathway of pentose and pentitol degradation); PWY-5517 (L-arabinose degradation III); PWY-5519 (D-arabinose degradation III); PWY-6486 (D-galacturonate degradation II); PWY-6497 (D-galactarate degradation II); PWY-6499 (D-glucarate degradation II); PWY-6501 (D-glucuronate degradation II); PWY-6516 (superpathway of microbial D-galacturonate and D-glucuronate degradation); PWY-6760 (D-xylose degradation III); PWY-7516 (L-lyxonate degradation); PWY-8020 (D-xylose degradation V); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ACIDS-DEG (Sugar Acid Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); Xylose-Degradation (Xylose Degradation)',\n",
       "   'KEGG: rn00040 (Pentose and glucuronate interconversions); rn00053 (Ascorbate and aldarate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00340:0:0:\"2,5-Dioxopentanoate\";1:cpd00005:0:0:\"NADPH\";1:cpd00024:0:0:\"2-Oxoglutarate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00265',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00265',\n",
       "   'MetaCyc: OXOGLUTARATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'Name: 2-Oxoglutarate:NADP+ 2-oxidoreductase (CoA-succinylating); 2-oxoglutarate:NADP+ 2-oxidoreductase (CoA-succinylating); oxoglutarate dehydrogenase (NADP); oxoglutarate dehydrogenase (NADP+)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00024[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00078[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00010;cpd00011;cpd00024;cpd00078',\n",
       "  'definition': '(1) NADP[0] + (1) CoA[0] + (1) 2-Oxoglutarate[0] => (1) NADPH[0] + (1) CO2[0] + (1) Succinyl-CoA[0]',\n",
       "  'deltag': -6.93,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.52'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00024[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00078[0]',\n",
       "  'id': 'rxn00197',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-oxoglutarate:NADP+ 2-oxidoreductase (CoA-succinylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00078:0:0:\"Succinyl-CoA\"'},\n",
       " {'abbreviation': 'R00267',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00267_m',\n",
       "   'BiGG: ICDHy; ICDHym; ICDHyp; ICDHyr',\n",
       "   'JM_Creinhardtii: JM_Cre_49; JM_Cre_50',\n",
       "   'JP_Creinhardtii_MSB: R_ICDH; R_ICDHhr',\n",
       "   'KEGG: R00267',\n",
       "   'Name: Isocitrate dehydrogenase (NADP+); Isocitrate:NADP+ oxidoreductase (decarboxylating); isocitrate dehydrogenase (NADP); isocitrate dehydrogenase (nadp+); isocitratedehydrogenase-NADP'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00260[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00011;cpd00024;cpd00260',\n",
       "  'definition': '(1) NADP[0] + (1) Isocitrate[0] => (1) NADPH[0] + (1) CO2[0] + (1) 2-Oxoglutarate[0]',\n",
       "  'deltag': 1.41,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.42'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00260[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00024[0]',\n",
       "  'id': 'rxn00198',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30399;rxn30400;rxn33806',\n",
       "  'name': 'Isocitrate:NADP+ oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00480 (Glutathione metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00260:0:0:\"Isocitrate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00024:0:0:\"2-Oxoglutarate\"'},\n",
       " {'abbreviation': 'R00268',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-8642',\n",
       "   'BiGG: IDP1_2; IDP2_2; IDP3_2',\n",
       "   'BrachyCyc: RXN-8642',\n",
       "   'KEGG: R00268',\n",
       "   'MetaCyc: RXN-8642',\n",
       "   'Name: IDH (ambiguous); IDP; IDP1; IDP2; IDP3; Isocitrate dehydrogenase [NADP]; Isocitrate dehydrogenase [NADP],; NADP isocitric dehydrogenase; NADP+-ICDH; NADP+-IDH; NADP+-linked isocitrate dehydrogenase; NADP-dependent isocitrate dehydrogenase; NADP-dependent isocitric dehydrogenase; NADP-linked isocitrate dehydrogenase; NADP-specific isocitrate dehydrogenase; dual-cofactor-specific isocitrate dehydrogenase; isocitrate (NADP) dehydrogenase; isocitrate (nicotinamide adenine dinucleotide phosphate) dehydrogenase; isocitrate dehydrogenase (NADP-dependent); isocitrate:NADP+ oxidoreductase (decarboxylating); oxalosuccinate carboxy-lyase (2-oxoglutarate-forming); oxalosuccinate decarboxylase; oxalsuccinic decarboxylase; triphosphopyridine nucleotide-linked isocitrate dehydrogenase-oxalosuccinate carboxylase'],\n",
       "  'code': '(1) cpd03187[0] <=> (1) cpd00011[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00024;cpd00067;cpd03187',\n",
       "  'definition': '(1) H+[0] + (1) Oxalosuccinate[0] => (1) CO2[0] + (1) 2-Oxoglutarate[0]',\n",
       "  'deltag': -3.33,\n",
       "  'deltagerr': 0.96,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.42', '4.1.1.-'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd03187[0] => (1) cpd00011[0] + (1) cpd00024[0]',\n",
       "  'id': 'rxn00199',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn25050;rxn25051',\n",
       "  'name': 'oxalosuccinate carboxy-lyase (2-oxoglutarate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00020 (Citrate cycle (TCA cycle)); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd03187:0:0:\"Oxalosuccinate\";1:cpd00011:0:0:\"CO2\";1:cpd00024:0:0:\"2-Oxoglutarate\"'},\n",
       " {'abbreviation': 'R00269',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00269',\n",
       "   'MetaCyc: RXN-13072',\n",
       "   'TS_Athaliana: rxn565_p',\n",
       "   'Name: 2-Oxoglutaramate amidohydrolase; 2-ketoglutaramate amidase; 2-oxoglutaramate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00695[0] <=> (1) cpd00013[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00024;cpd00695',\n",
       "  'definition': '(1) H2O[0] + (1) 2-Oxoglutaramate[0] => (1) NH3[0] + (1) 2-Oxoglutarate[0]',\n",
       "  'deltag': -0.14,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.-', '3.5.1.111', '3.5.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00695[0] => (1) cpd00013[0] + (1) cpd00024[0]',\n",
       "  'id': 'rxn00200',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn23361;rxn31885',\n",
       "  'name': '2-oxoglutaramate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00695:0:0:\"2-Oxoglutaramate\";1:cpd00013:0:0:\"NH3\";1:cpd00024:0:0:\"2-Oxoglutarate\"'},\n",
       " {'abbreviation': 'R00270',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_HYNE',\n",
       "   'KEGG: R00270',\n",
       "   'MetaCyc: RXN-7594',\n",
       "   'iRS1563: R_R00270_c',\n",
       "   'Name: Hydrolysis (non-enzymatic)'],\n",
       "  'code': '(1) cpd00013[0] + (1) cpd00024[0] <=> (1) cpd00001[0] + (1) cpd03303[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00024;cpd03303',\n",
       "  'definition': '(1) NH3[0] + (1) 2-Oxoglutarate[0] <= (1) H2O[0] + (1) 4-Oxoglutaramate[0]',\n",
       "  'deltag': 10.04,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00013[0] + (1) cpd00024[0] <= (1) cpd00001[0] + (1) cpd03303[0]',\n",
       "  'id': 'rxn00201',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35319',\n",
       "  'name': 'R00270',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); HISHP-PWY (L-histidine degradation VI); HISTIDINE-DEG (L-histidine Degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00340 (Histidine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00013:0:0:\"NH3\";-1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00001:0:0:\"H2O\";1:cpd03303:0:0:\"4-Oxoglutaramate\"'},\n",
       " {'abbreviation': 'R00271',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: HCITS; HCITSm; HCITSn; LYS20_1; LYS20_2; LYS21',\n",
       "   'KEGG: R00271',\n",
       "   'MetaCyc: HOMOCITRATE-SYNTHASE-RXN',\n",
       "   'RiceCyc: HOMOCITRATE-SYNTHASE-RXN',\n",
       "   'iAF692: HCITS',\n",
       "   'Name: (CoA-acetylating); (thioester-hydrolysing, carboxymethyl forming); 2-Hydroxybutane-1,2,4-tricarboxylate 2-oxoglutarate-lyase; 2-hydroxybutane-1,2,4-tricarboxylate 2-oxoglutarate-lyase (CoA-acetylating); HCS; Homocitrate synthase, isozyme; acetyl-CoA:2-oxoglutarate C-acetyltransferase; acetyl-CoA:2-oxoglutarate C-acetyltransferase (thioester-hydrolysing, carboxymethyl forming); acetyl-coenzyme A:2-ketoglutarate C-acetyl transferase; homocitrate synthase; homocitrate synthetase'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00919[0] <=> (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00022;cpd00024;cpd00067;cpd00919',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) Homocitrate[0] => (1) H2O[0] + (1) Acetyl-CoA[0] + (1) 2-Oxoglutarate[0]',\n",
       "  'deltag': 6.93,\n",
       "  'deltagerr': 0.67,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.3.14', '4.1.3.21'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00919[0] => (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00024[0]',\n",
       "  'id': 'rxn00202',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30154;rxn30391',\n",
       "  'name': 'acetyl-CoA:2-oxoglutarate C-acetyltransferase (thioester-hydrolysing, carboxymethyl forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); LYSINE-AMINOAD-PWY (L-lysine biosynthesis IV); LYSINE-SYN (L-lysine Biosynthesis); P241-PWY (coenzyme B biosynthesis); PWY-3081 (L-lysine biosynthesis V); PWY-7710 (FeMo cofactor biosynthesis)',\n",
       "   'KEGG: rn00300 (Lysine biosynthesis); rn00620 (Pyruvate metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd00919:0:0:\"Homocitrate\";1:cpd00001:0:0:\"H2O\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00024:0:0:\"2-Oxoglutarate\"'},\n",
       " {'abbreviation': 'R00272',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-7774',\n",
       "   'BiGG: OXGDC',\n",
       "   'CornCyc: RXN-7774',\n",
       "   'KEGG: R00272',\n",
       "   'MetaCyc: RXN-7774',\n",
       "   'Name: 2-Oxoglutarate carboxy-lyase; 2-oxoglutarate carboxy-lyase; 2-oxoglutarate carboxy-lyase (succinate-semialdehyde-forming); 2-oxoglutarate decarboxylase; alpha-ketoglutarate decarboxylase; alpha-ketoglutaric decarboxylase; oxoglutarate decarboxylase; pre-2-oxoglutarate decarboxylase'],\n",
       "  'code': '(1) cpd00024[0] <=> (1) cpd00011[0] + (1) cpd00199[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00024;cpd00067;cpd00199',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) H+[0] => (1) CO2[0] + (1) 4-Oxobutanoate[0]',\n",
       "  'deltag': -4.14,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.71'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00199[0]',\n",
       "  'id': 'rxn00203',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn24434',\n",
       "  'name': '2-Oxoglutarate carboxy-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); TCA-VARIANTS (TCA cycle)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00199:0:0:\"4-Oxobutanoate\"'},\n",
       " {'abbreviation': 'R00273',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: OXALATE-OXIDASE-RXN',\n",
       "   'CornCyc: OXALATE-OXIDASE-RXN',\n",
       "   'KEGG: R00273',\n",
       "   'MetaCyc: OXALATE-OXIDASE-RXN',\n",
       "   'PlantCyc: OXALATE-OXIDASE-RXN',\n",
       "   'Name: aero-oxalo dehydrogenase; oxalate oxidase; oxalate:oxygen oxidoreductase; oxalic acid oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00180[0] <=> (2) cpd00011[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00025;cpd00067;cpd00180',\n",
       "  'definition': '(1) O2[0] + (2) H+[0] + (1) Oxalate[0] => (2) CO2[0] + (1) H2O2[0]',\n",
       "  'deltag': -39.15,\n",
       "  'deltagerr': 1.89,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.3.4'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00067[0] + (1) cpd00180[0] => (2) cpd00011[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00204',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'oxalate:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Oxalate-Degradation (Oxalate Degradation); PWY-6697 (oxalate degradation IV)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-1:cpd00180:0:0:\"Oxalate\";2:cpd00011:0:0:\"CO2\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00274',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTATHIONE-PEROXIDASE-RXN',\n",
       "   'BiGG: GPX1; GPX2; GSHPO; GTHP; GTHPe; GTHPm; HYR1',\n",
       "   'BrachyCyc: GLUTATHIONE-PEROXIDASE-RXN',\n",
       "   'KEGG: R00274',\n",
       "   'MetaCyc: GLUTATHIONE-PEROXIDASE-RXN',\n",
       "   'Name: GLUTATHIONE PEROXIDASE 6; GSH peroxidase; Glutathione peroxidase; Glutathione peroxidase 1; Glutathione:hydrogen-peroxide oxidoreductase; glutathione peridoxase; glutathione peroxidase; glutathione:hydrogen-peroxide oxidoreductase; reduced glutathione peroxidase; selenium-glutathione peroxidase'],\n",
       "  'code': '(1) cpd00025[0] + (2) cpd00042[0] <=> (2) cpd00001[0] + (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00025;cpd00042;cpd00111',\n",
       "  'definition': '(1) H2O2[0] + (2) GSH[0] => (2) H2O[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': -74.09,\n",
       "  'deltagerr': 0.81,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.9'],\n",
       "  'equation': '(1) cpd00025[0] + (2) cpd00042[0] => (2) cpd00001[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00205',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27891;rxn33345;rxn33349',\n",
       "  'name': 'glutathione:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); DETOX1-PWY-1 (reactive oxygen species degradation); Detoxification (Detoxification); PWY-4081 (glutathione-peroxide redox reactions); REACTIVE-OXYGEN-SPECIES-DEGRADATION (Reactive Oxygen Specie Degradation); Reductants (Reductant Biosynthesis)',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00025:0:0:\"H2O2\";-2:cpd00042:0:0:\"GSH\";2:cpd00001:0:0:\"H2O\";1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'R00275',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUPEROX-DISMUT-RXN',\n",
       "   'BiGG: SOD; SODe; SODr; SPODM; SPODMpp',\n",
       "   'BrachyCyc: SUPEROX-DISMUT-RXN',\n",
       "   'KEGG: R00275',\n",
       "   'MetaCyc: SUPEROX-DISMUT-RXN',\n",
       "   'Name: Cu,Zn-SOD; Cu-Zn superoxide dismutase; Fe-SOD; Mn-SOD; SOD-1; SOD-2; SOD-3; SOD-4; SODF; SODS; Superoxide dismutase; Superoxide:superoxide oxidoreductase; copper-zinc superoxide dismutase; cuprein; cytocuprein; erythrocuprein; ferrisuperoxide dismutase; hemocuprein; hepatocuprein; superoxidase dismutase; superoxide dismutase; superoxide dismutase I; superoxide dismutase II; superoxide:superoxide oxidoreductase; superoxidedismutase'],\n",
       "  'code': '(2) cpd00532[0] <=> (1) cpd00007[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00067;cpd00532',\n",
       "  'definition': '(2) H+[0] + (2) O2-[0] => (1) O2[0] + (1) H2O2[0]',\n",
       "  'deltag': -22.87,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.15.1.1'],\n",
       "  'equation': '(2) cpd00067[0] + (2) cpd00532[0] => (1) cpd00007[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00206',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27269;rxn27270;rxn27271;rxn27272;rxn30523;rxn32404',\n",
       "  'name': 'superoxide:superoxide oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: DETOX1-PWY (superoxide radicals degradation); DETOX1-PWY-1 (reactive oxygen species degradation); Detoxification (Detoxification); Ethylene-Biosynthesis (Ethylene Biosynthesis); HORMONE-SYN (Hormone Biosynthesis); PWY-6854 (ethylene biosynthesis III (microbes)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); REACTIVE-OXYGEN-SPECIES-DEGRADATION (Reactive Oxygen Specie Degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00067:0:0:\"H+\";-2:cpd00532:0:0:\"O2-\";1:cpd00007:0:0:\"O2\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00276',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00276',\n",
       "   'iRS1563: R_R00276_c',\n",
       "   'Name: Carbon-monoxide:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00025[0] <=> (1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00204[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00011;cpd00025;cpd00067;cpd00204',\n",
       "  'definition': '(1) CO2[0] + (1) H2O2[0] + (1) H+[0] <=> (1) H2O[0] + (1) O2[0] + (1) CO[0]',\n",
       "  'deltag': 52.41,\n",
       "  'deltagerr': 1.12,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.2.4'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00025[0] + (1) cpd00067[0] <=> (1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00204[0]',\n",
       "  'id': 'rxn00207',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Carbon-monoxide:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00025:0:0:\"H2O2\";-1:cpd00067:0:0:\"H+\";1:cpd00001:0:0:\"H2O\";1:cpd00007:0:0:\"O2\";1:cpd00204:0:0:\"CO\"'},\n",
       " {'abbreviation': 'R00277',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PMPOXI-RXN',\n",
       "   'BiGG: PDX3_1; PDX3_5; PYAM5PO',\n",
       "   'BrachyCyc: PMPOXI-RXN',\n",
       "   'KEGG: R00277',\n",
       "   'MetaCyc: PMPOXI-RXN',\n",
       "   \"Name: PMP oxidase; PdxH; Pyridoxamine 5'-phosphate oxidase; Pyridoxamine-5'-phosphate:oxygen oxidoreductase (deaminating); pyridoxal 5'-phosphate synthase; pyridoxamine 5'-phosphate oxidase; pyridoxamine phosphate oxidase; pyridoxamine-5'-phosphate:oxygen oxidoreductase (deaminating); pyridoxamine-phosphate oxidase; pyridoxaminephosphate oxidase (EC 1.4.3.5: deaminating); pyridoxine (pyridoxamine) 5'-phosphate oxidase; pyridoxine (pyridoxamine)phosphate oxidase; pyridoxol-5'-phosphate:oxygen oxidoreductase (deaminating) (incorrect)\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00493[0] <=> (1) cpd00013[0] + (1) cpd00016[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00016;cpd00025;cpd00493',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Pyridoxamine phosphate[0] => (1) NH3[0] + (1) Pyridoxal phosphate[0] + (1) H2O2[0]',\n",
       "  'deltag': -23.36,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00493[0] => (1) cpd00013[0] + (1) cpd00016[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00208',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30055;rxn30747;rxn31960',\n",
       "  'name': \"Pyridoxamine-5'-phosphate:oxygen oxidoreductase (deaminating)\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PLPSAL-PWY (pyridoxal 5'-phosphate salvage I); PWY-7204 (pyridoxal 5'-phosphate salvage II (plants)); PWY0-845 (superpathway of pyridoxal 5'-phosphate biosynthesis and salvage); Vitamin-B6-Biosynthesis (Vitamin B<sub>6</sub> Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)\",\n",
       "   'KEGG: rn00750 (Vitamin B6 metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00493:0:0:\"Pyridoxamine phosphate\";1:cpd00013:0:0:\"NH3\";1:cpd00016:0:0:\"Pyridoxal phosphate\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00278',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PNPOXI-RXN',\n",
       "   'BiGG: PDX3_2; PDX5PO; PDX5POi',\n",
       "   'BrachyCyc: PNPOXI-RXN',\n",
       "   'KEGG: R00278',\n",
       "   'MetaCyc: PNPOXI-RXN',\n",
       "   \"Name: PMP oxidase; PdxH; Pyridoxine 5-phosphate:oxygen oxidoreductase; Pyridoxine 5-phosphate:oxygen oxidoreductase (deaminating); pyridoxal 5'-phosphate synthase; pyridoxamine 5'-phosphate oxidase; pyridoxamine phosphate oxidase; pyridoxamine-5'-phosphate:oxygen oxidoreductase (deaminating); pyridoxamine-phosphate oxidase; pyridoxaminephosphate oxidase (EC 1.4.3.5: deaminating); pyridoxine (pyridoxamine) 5'-phosphate oxidase; pyridoxine (pyridoxamine)phosphate oxidase; pyridoxine 5'-phosphate oxidase; pyridoxol-5'-phosphate:oxygen oxidoreductase (deaminating) (incorrect)\"],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00478[0] <=> (1) cpd00016[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00016;cpd00025;cpd00478',\n",
       "  'definition': '(1) O2[0] + (1) Pyridoxine phosphate[0] => (1) Pyridoxal phosphate[0] + (1) H2O2[0]',\n",
       "  'deltag': -27.09,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.5'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00478[0] => (1) cpd00016[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00209',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38693',\n",
       "  'name': 'Pyridoxine 5-phosphate:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); HMP-PP-Biosynthesis (4-amino-2-methyl-5-diphosphomethylpyrimidine biosynthesis); PLPSAL-PWY (pyridoxal 5'-phosphate salvage I); PWY-7204 (pyridoxal 5'-phosphate salvage II (plants)); PWY-7282 (4-amino-2-methyl-5-diphosphomethylpyrimidine biosynthesis (yeast)); PWY0-845 (superpathway of pyridoxal 5'-phosphate biosynthesis and salvage); PYRIDOXSYN-PWY (pyridoxal 5'-phosphate biosynthesis I); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-B6-Biosynthesis (Vitamin B<sub>6</sub> Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)\",\n",
       "   'KEGG: rn00750 (Vitamin B6 metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00478:0:0:\"Pyridoxine phosphate\";1:cpd00016:0:0:\"Pyridoxal phosphate\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00279',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00279',\n",
       "   'MetaCyc: D-GLUTAMATE-OXIDASE-RXN; D-GLUTAMATED-ASPARTATE-OXIDASE-RXN',\n",
       "   'Name: D-Glutamate(D-aspartate):oxygen oxidoreductase (deaminating); D-Glutamate:oxygen oxidoreductase (deaminating); D-glutamate(D-aspartate) oxidase; D-glutamate(D-aspartate):oxygen oxidoreductase (deaminating); D-glutamic-aspartic oxidase; D-monoaminodicarboxylic acid oxidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00186[0] <=> (1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00025[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00024;cpd00025;cpd00186',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) D-Glutamate[0] => (1) NH3[0] + (1) 2-Oxoglutarate[0] + (1) H2O2[0]',\n",
       "  'deltag': -23.14,\n",
       "  'deltagerr': 1.15,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.15', '1.4.3.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00186[0] => (1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00025[0]',\n",
       "  'id': 'rxn00210',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glutamate:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00471 (D-Glutamine and D-glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00186:0:0:\"D-Glutamate\";1:cpd00013:0:0:\"NH3\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00025:0:0:\"H2O2\"'},\n",
       " {'abbreviation': 'R00286',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UGD-RXN',\n",
       "   'BiGG: UDPGD; UDPGDi; UDPGDr',\n",
       "   'CornCyc: UGD-RXN',\n",
       "   'KEGG: R00286',\n",
       "   'MetaCyc: UGD-RXN',\n",
       "   'Name: UDP-D-glucose dehydrogenase; UDP-alpha-D-glucose:NAD oxidoreductase; UDP-glucose 6-dehydrogenase; UDP-glucose dehydrogenase; UDP-glucose:NAD+ 6-oxidoreductase; UDP-glucose:NAD+ oxidoreductase; UDPG dehydrogenase; UDPG:NAD oxidoreductase; UDPglucose 6-dehydrogenase; UDPglucose:NAD+ oxidoreductase; udp-glucose 6-dehydrogenase; uridine diphosphate D-glucose dehydrogenase; uridine diphosphate glucose dehydrogenase; uridine diphosphoglucose dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00026[0] <=> (2) cpd00004[0] + (1) cpd00144[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00026;cpd00067;cpd00144',\n",
       "  'definition': '(1) H2O[0] + (2) NAD[0] + (1) UDP-glucose[0] <=> (2) NADH[0] + (3) H+[0] + (1) UDPglucuronate[0]',\n",
       "  'deltag': -4.28,\n",
       "  'deltagerr': 0.56,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00026[0] <=> (2) cpd00004[0] + (3) cpd00067[0] + (1) cpd00144[0]',\n",
       "  'id': 'rxn00211',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-glucose:NAD+ 6-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); Ascorbate-Biosynthesis (Ascorbate Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); COLANSYN-PWY (colanic acid building blocks biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Interconversion (Interconversions); PWY-4821 (UDP-&alpha;-D-xylose biosynthesis); PWY-5114 (UDP-sugars interconversion); PWY-6415 (L-ascorbate biosynthesis V); PWY-7328 (superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis); PWY-7346 (UDP-&alpha;-D-glucuronate biosynthesis (from UDP-glucose)); PWY-7820 (teichuronic acid biosynthesis (<i>B. subtilis</i> 168)); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00040 (Pentose and glucuronate interconversions); rn00053 (Ascorbate and aldarate metabolism); rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00003:0:0:\"NAD\";-1:cpd00026:0:0:\"UDP-glucose\";2:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";1:cpd00144:0:0:\"UDPglucuronate\"'},\n",
       " {'abbreviation': 'R00287',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: UDPGPpp',\n",
       "   'JP_Creinhardtii_MSB: R_UDPGP',\n",
       "   'KEGG: R00287',\n",
       "   'iAF1260: UDPGPpp',\n",
       "   'Name: UDP-glucose glucophosphohydrolase; UDPglucose pyrophosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00026[0] <=> (1) cpd00089[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00026;cpd00067;cpd00089;cpd00091',\n",
       "  'definition': '(1) H2O[0] + (1) UDP-glucose[0] => (2) H+[0] + (1) Glucose-1-phosphate[0] + (1) UMP[0]',\n",
       "  'deltag': -7.08,\n",
       "  'deltagerr': 0.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.45', '3.6.1.8', '3.6.1.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00026[0] => (2) cpd00067[0] + (1) cpd00089[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00212',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29978',\n",
       "  'name': 'UDP-glucose glucophosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism); rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00026:0:0:\"UDP-glucose\";2:cpd00067:0:0:\"H+\";1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00289',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUC1PURIDYLTRANS-RXN; RXN-11505',\n",
       "   'BiGG: GALU; GALUi; UGP1; YHL012W',\n",
       "   'CornCyc: GLUC1PURIDYLTRANS-RXN; RXN-11505',\n",
       "   'KEGG: R00289',\n",
       "   'MetaCyc: GLUC1PURIDYLTRANS-RXN; RXN-11505; RXN-11988',\n",
       "   \"Name: UDP glucose pyrophosphorylase; UDPG phosphorylase; UDPG pyrophosphorylase; UTP-glucose-1-phosphate uridylyltransferase; UTP-glucose-1-phosphate uridylyltransferase (irreversible); UTP-glucose-1-phosphateuridylyltransferase; UTP:alpha-D-glucose-1-phosphate uridylyltransferase; glucose-1-phosphate uridylyltransferase; uridine 5'-diphosphoglucose pyrophosphorylase; uridine diphosphate-D-glucose pyrophosphorylase; uridine diphosphoglucose pyrophosphorylase; uridine-diphosphate glucose pyrophosphorylase; utp--glucose-1-phosphate uridylyltransferase\"],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00026[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00026;cpd00062;cpd00089',\n",
       "  'definition': '(1) UTP[0] + (1) Glucose-1-phosphate[0] <=> (1) PPi[0] + (1) UDP-glucose[0]',\n",
       "  'deltag': 0.03,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.9'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00026[0]',\n",
       "  'id': 'rxn00213',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38117',\n",
       "  'name': 'UTP:alpha-D-glucose-1-phosphate uridylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Ascorbate-Biosynthesis (Ascorbate Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); COLANSYN-PWY (colanic acid building blocks biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); PWY-3801 (sucrose degradation II (sucrose synthase)); PWY-6415 (L-ascorbate biosynthesis V); PWY-6527 (stachyose degradation); PWY-7238 (sucrose biosynthesis II); PWY-7328 (superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis); PWY-7343 (UDP-&alpha;-D-glucose biosynthesis I); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY-7817 (type I lipoteichoic acid biosynthesis (<i>S. aureus</i>)); SUCROSE-DEG (Sucrose Degradation); SUCSYN-PWY (sucrose biosynthesis I (from photosynthesis)); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sucrose-Biosynthesis (Sucrose Biosynthesis); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); Teichoic-Acids-Biosynthesis (Teichoic Acid Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00040 (Pentose and glucuronate interconversions); rn00052 (Galactose metabolism); rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00026:0:0:\"UDP-glucose\"'},\n",
       " {'abbreviation': 'R00291',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UDPGLUCEPIM-RXN',\n",
       "   'BiGG: GAL10; UDPG4E',\n",
       "   'CornCyc: UDPGLUCEPIM-RXN',\n",
       "   'KEGG: R00291',\n",
       "   'MetaCyc: UDPGLUCEPIM-RXN',\n",
       "   'Name: 4-epimerase; Galactowaldenase; UDP-D-galactose 4-epimerase; UDP-alpha-D-glucose 4-epimerase; UDP-galactose 4-epimerase; UDP-glucose 4-epimerase; UDP-glucose epimerase; UDPG-4-epimerase; UDPglucose 4-epimerase; UDPglucose4-epimerase; Uridine diphosphate galactose 4-epimerase; Uridine diphospho-galactose-4-epimerase; uridine diphosphate galactose 4-epimerase; uridine diphosphate glucose 4-epimerase; uridine diphospho-galactose-4-epimerase; uridine diphosphoglucose 4-epimerase; uridine diphosphoglucose epimerase'],\n",
       "  'code': '(1) cpd00026[0] <=> (1) cpd00043[0]',\n",
       "  'compound_ids': 'cpd00026;cpd00043',\n",
       "  'definition': '(1) UDP-glucose[0] <=> (1) UDP-galactose[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 4.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.3.2'],\n",
       "  'equation': '(1) cpd00026[0] <=> (1) cpd00043[0]',\n",
       "  'id': 'rxn00214',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33514;rxn33931;rxn37863;rxn38827;rxn38829',\n",
       "  'name': 'UDP-glucose 4-epimerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); COLANSYN-PWY (colanic acid building blocks biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); GALACTOSE-DEGRADATION (Galactose Degradation); Interconversion (Interconversions); PWY-3821 (D-galactose detoxification); PWY-5114 (UDP-sugars interconversion); PWY-6317 (D-galactose degradation I (Leloir pathway)); PWY-6397 (mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY-6404 (superpathway of mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY-6527 (stachyose degradation); PWY-7328 (superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis); PWY-7344 (UDP-&alpha;-D-galactose biosynthesis); PWY-7622 (UDP-&alpha;-D-galactofuranose biosynthesis); PWY66-422 (D-galactose degradation V (Leloir pathway)); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00052 (Galactose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";1:cpd00043:0:0:\"UDP-galactose\"'},\n",
       " {'abbreviation': 'R00293',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UDP-GLUCOSE-46-DEHYDRATASE-RXN',\n",
       "   'CornCyc: UDP-GLUCOSE-46-DEHYDRATASE-RXN',\n",
       "   'KEGG: R00293',\n",
       "   'MaizeCyc: UDP-GLUCOSE-46-DEHYDRATASE-RXN',\n",
       "   'MetaCyc: UDP-GLUCOSE-46-DEHYDRATASE-RXN',\n",
       "   'Name: UDP-D-glucose oxidoreductase; UDP-D-glucose-4,6-hydrolyase; UDP-GLUCOSE-4,6-DEHYDRATASE-RXN; UDP-glucose 4,6-dehydratase; UDP-glucose 4,6-hydro-lyase; UDP-glucose 4,6-hydro-lyase (UDP-4-dehydro-6-deoxy-D-glucose-forming); UDPglucose 4,6-hydro-lyase'],\n",
       "  'code': '(1) cpd00026[0] <=> (1) cpd00001[0] + (1) cpd02528[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00026;cpd02528',\n",
       "  'definition': '(1) UDP-glucose[0] => (1) H2O[0] + (1) UDP-4-oxo-6-deoxy-D-glucose[0]',\n",
       "  'deltag': -15.71,\n",
       "  'deltagerr': 0.62,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.76'],\n",
       "  'equation': '(1) cpd00026[0] => (1) cpd00001[0] + (1) cpd02528[0]',\n",
       "  'id': 'rxn00215',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31097',\n",
       "  'name': 'UDP-glucose 4,6-hydro-lyase (UDP-4-dehydro-6-deoxy-D-glucose-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Interconversion (Interconversions); PWY-3261 (UDP-&beta;-L-rhamnose biosynthesis); PWY-5114 (UDP-sugars interconversion); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";1:cpd00001:0:0:\"H2O\";1:cpd02528:0:0:\"UDP-4-oxo-6-deoxy-D-glucose\"'},\n",
       " {'abbreviation': 'R00299',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00299_c',\n",
       "   'BiGG: GLK1_1; HEX1; HXK1_2; HXK2_2',\n",
       "   'JM_Creinhardtii: JM_Cre_1',\n",
       "   'KEGG: R00299',\n",
       "   'MetaCyc: GLUCOKIN-RXN',\n",
       "   'Name: ATP:D-glucose 6-phosphotransferase; glucose kinase; hexokinase; hexokinase (D-glucose:ATP); hexokinase-D-glucose-ATP'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00027[0] <=> (1) cpd00008[0] + (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00027;cpd00067;cpd00079',\n",
       "  'definition': '(1) ATP[0] + (1) D-Glucose[0] => (1) ADP[0] + (1) H+[0] + (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': -4.01,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1', '2.7.1.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00027[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00079[0]',\n",
       "  'id': 'rxn00216',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34232',\n",
       "  'name': 'ATP:D-glucose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROFRUCAT-PWY (homolactic fermentation); ANAGLYCOLYSIS-PWY (glycolysis III (from glucose)); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); GDP-Sugar-Biosynthesis (GDP-sugar Biosynthesis); GLUCOSE1PMETAB-PWY (glucose and glucose-1-phosphate degradation); GLYCOCAT-PWY (glycogen degradation I); GLYCOLYSIS-VARIANTS (Glycolysis); Glycan-Biosynthesis (Glycan Biosynthesis); Glycan-Pathways (Glycan Pathways); Glycogen-Degradation (Glycogen Degradation); P122-PWY (heterolactic fermentation); P124-PWY (Bifidobacterium shunt); POLYSACCHARIDES-DEG (Polysaccharide Degradation); PWY-2722 (trehalose degradation IV); PWY-2723 (trehalose degradation V); PWY-5514 (UDP-<i>N</i>-acetyl-D-galactosamine biosynthesis II); PWY-5661 (GDP-glucose biosynthesis); PWY-621 (sucrose degradation III (sucrose invertase)); PWY-6981 (chitin biosynthesis); PWY-7238 (sucrose biosynthesis II); PWY-7385 (1,3-propanediol biosynthesis (engineered)); PWY0-1182 (trehalose degradation II (cytosolic)); Polymer-Degradation (Polymeric Compound Degradation); Polysaccharides-Biosynthesis (Polysaccharide Biosynthesis); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SUCROSE-DEG (Sucrose Degradation); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sucrose-Biosynthesis (Sucrose Biosynthesis); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); TREDEGLOW-PWY (trehalose degradation I (low osmolarity)); Trehalose-Degradation (Trehalose Degradation); UDP-NAc-Glucosamine-Biosynthesis (UDP-<i>N</i>-acetyl-D-glucosamine Biosynthesis); UDP-Nac-Galactosamine-Biosynthesis (UDP-<i>N</i>-acetyl-D-galactosamine Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis); UDPNACETYLGALSYN-PWY (UDP-<i>N</i>-acetyl-D-glucosamine biosynthesis II)',\n",
       "   'KEGG: rn00521 (Streptomycin biosynthesis); rn00524 (Butirosin and neomycin biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00027:0:0:\"D-Glucose\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00300',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G1Dx',\n",
       "   'KEGG: R00300',\n",
       "   'MetaCyc: GLUCOSE-1-DEHYDROGENASE-NAD+-RXN',\n",
       "   'iYO844: G1Dx',\n",
       "   'Name: D-Glucose:NAD+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00027[0] <=> (1) cpd00004[0] + (1) cpd00170[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00027;cpd00067;cpd00170',\n",
       "  'definition': '(1) NAD[0] + (1) D-Glucose[0] <=> (1) NADH[0] + (1) H+[0] + (1) Gluconolactone[0]',\n",
       "  'deltag': -1.02,\n",
       "  'deltagerr': 0.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.118', '1.1.1.121'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00027[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00170[0]',\n",
       "  'id': 'rxn00217',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose:NAD+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00027:0:0:\"D-Glucose\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00170:0:0:\"Gluconolactone\"'},\n",
       " {'abbreviation': 'R00301',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00301', 'Name: D-Glucose:NADP+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00027[0] <=> (1) cpd00005[0] + (1) cpd01992[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00027;cpd00067;cpd01992',\n",
       "  'definition': '(1) NADP[0] + (1) D-Glucose[0] <=> (1) NADPH[0] + (1) H+[0] + (1) D-Glucono-1,4-lactone[0]',\n",
       "  'deltag': -2.59,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.119'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00027[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd01992[0]',\n",
       "  'id': 'rxn00218',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00027:0:0:\"D-Glucose\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd01992:0:0:\"D-Glucono-1,4-lactone\"'},\n",
       " {'abbreviation': 'R00302',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00302',\n",
       "   'MetaCyc: PYRANOSE-OXIDASE-RXN',\n",
       "   'Name: D-glucose:oxygen 2-oxidoreductase; Pyranose:oxygen 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00027[0] <=> (1) cpd00025[0] + (1) cpd01792[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00027;cpd01792',\n",
       "  'definition': '(1) O2[0] + (1) D-Glucose[0] => (1) H2O2[0] + (1) 2-Dehydro-D-glucose[0]',\n",
       "  'deltag': -31.6,\n",
       "  'deltagerr': 1.1,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.10'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00027[0] => (1) cpd00025[0] + (1) cpd01792[0]',\n",
       "  'id': 'rxn00219',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-glucose:oxygen 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00027:0:0:\"D-Glucose\";1:cpd00025:0:0:\"H2O2\";1:cpd01792:0:0:\"2-Dehydro-D-glucose\"'},\n",
       " {'abbreviation': 'R00303',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G6PP',\n",
       "   'JM_Creinhardtii: JM_Cre_2',\n",
       "   'KEGG: R00303',\n",
       "   'MetaCyc: RXN66-526',\n",
       "   'iAF1260: G6PP',\n",
       "   'Name: D-Glucose-6-phosphate phosphohydrolase; Polyphosphate glucokinase reversible; glucose-6-phosphate phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00079[0] <=> (1) cpd00009[0] + (1) cpd00027[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00027;cpd00079',\n",
       "  'definition': '(1) H2O[0] + (1) D-glucose-6-phosphate[0] <=> (1) Phosphate[0] + (1) D-Glucose[0]',\n",
       "  'deltag': -2.15,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.58', '3.1.3.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00079[0] <=> (1) cpd00009[0] + (1) cpd00027[0]',\n",
       "  'id': 'rxn00220',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose-6-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Gluconeogenesis (Gluconeogenesis); PWY66-399 (gluconeogenesis III)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00027:0:0:\"D-Glucose\"'},\n",
       " {'abbreviation': 'R00304',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G1PP; G1PPpp',\n",
       "   'KEGG: R00304',\n",
       "   'MetaCyc: GLUCOSE-1-PHOSPHAT-RXN',\n",
       "   'iAF1260: G1PPpp',\n",
       "   'iJR904: G1PP',\n",
       "   'Name: D-Glucose-1-phosphate phosphohydrolase; Glucose-1-phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00089[0] <=> (1) cpd00009[0] + (1) cpd00027[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00027;cpd00089',\n",
       "  'definition': '(1) H2O[0] + (1) Glucose-1-phosphate[0] <=> (1) Phosphate[0] + (1) D-Glucose[0]',\n",
       "  'deltag': -4.86,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00089[0] <=> (1) cpd00009[0] + (1) cpd00027[0]',\n",
       "  'id': 'rxn00221',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29957',\n",
       "  'name': 'D-Glucose-1-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); GLUCOSE1PMETAB-PWY (glucose and glucose-1-phosphate degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00027:0:0:\"D-Glucose\"'},\n",
       " {'abbreviation': 'R00306',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: BG(CELLB)',\n",
       "   'KEGG: R00306; R06104',\n",
       "   'TS_Athaliana: rxn566_r',\n",
       "   'iMA945: BG(CELLB)',\n",
       "   'iYO844: BG(CELLB)',\n",
       "   'Name: beta-glucosidase (cellobiose); cellobiose glucohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00158[0] <=> (2) cpd00027[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00158',\n",
       "  'definition': '(1) H2O[0] + (1) CELB[0] => (2) D-Glucose[0]',\n",
       "  'deltag': -3.44,\n",
       "  'deltagerr': 0.61,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.21', '3.2.1.74'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00158[0] => (2) cpd00027[0]',\n",
       "  'id': 'rxn00222',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30251;rxn32119',\n",
       "  'name': 'cellobiose glucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00158:0:0:\"CELB\";2:cpd00027:0:0:\"D-Glucose\"'},\n",
       " {'abbreviation': 'R00307',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: XYLI2; XYLI2i',\n",
       "   'KEGG: R00307',\n",
       "   'iAF1260: XYLI2',\n",
       "   'iJR904: XYLI2i',\n",
       "   'iMA945: XYLI2',\n",
       "   'Name: D-glucose aldose-ketose-isomerase; alpha-D-Glucose ketol-isomerase; xylose isomerase'],\n",
       "  'code': '(1) cpd00027[0] <=> (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00082',\n",
       "  'definition': '(1) D-Glucose[0] <=> (1) D-Fructose[0]',\n",
       "  'deltag': 0.19,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.5'],\n",
       "  'equation': '(1) cpd00027[0] <=> (1) cpd00082[0]',\n",
       "  'id': 'rxn00223',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-glucose aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00027:0:0:\"D-Glucose\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00310',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PROTOHEMEFERROCHELAT-RXN',\n",
       "   'BiGG: FCLT; FCLTm',\n",
       "   'BrachyCyc: PROTOHEMEFERROCHELAT-RXN',\n",
       "   'KEGG: R00310',\n",
       "   'MetaCyc: PROTOHEMEFERROCHELAT-RXN; RXN0-6258',\n",
       "   'Name: Ferrochelatase; Heme B synthesis reaction; Protoporphyrin ferro-lyase; ferro-protoporphyrin chelatase; heme synthase; heme synthetase; iron chelatase; protoheme ferro-lyase; protoheme ferro-lyase (protoporphyrin-forming)'],\n",
       "  'code': '(1) cpd01476[0] + (1) cpd10515[0] <=> (1) cpd00028[0]',\n",
       "  'compound_ids': 'cpd00028;cpd00067;cpd01476;cpd10515',\n",
       "  'definition': '(1) Protoporphyrin[0] + (1) Fe+2[0] <=> (1) Heme[0] + (2) H+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.99.1.1'],\n",
       "  'equation': '(1) cpd01476[0] + (1) cpd10515[0] <=> (1) cpd00028[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00224',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20626;rxn26363;rxn28098;rxn30378',\n",
       "  'name': 'protoheme ferro-lyase (protoporphyrin-forming)',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); HEME-BIOSYNTHESIS-II (heme <i>b</i> biosynthesis I (aerobic)); HEME-SYN (Heme Biosynthesis); HEMESYN2-PWY (heme <i>b</i> biosynthesis II (anaerobic)); Heme-b-Biosynthesis (Heme <i>b</i> Biosynthesis); PWY-5918 (superpathay of heme <i>b</i> biosynthesis from glutamate); PWY-5920 (superpathway of <i>b</i> heme biosynthesis from glycine); PWY0-1415 (superpathway of heme <i>b</i> biosynthesis from uroporphyrinogen-III); Porphyrin-Compounds-Biosynthesis (Porphyrin Compound Biosynthesis)',\n",
       "   'KEGG: rn00860 (Porphyrin and chlorophyll metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd01476:0:0:\"Protoporphyrin\";-1:cpd10515:0:0:\"Fe2+\";1:cpd00028:0:0:\"Heme\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00315',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACKr',\n",
       "   'BrachyCyc: ACETATEKIN-RXN',\n",
       "   'ChlamyCyc: ACETATEKIN-RXN',\n",
       "   'KEGG: R00315',\n",
       "   'MetaCyc: ACETATEKIN-RXN; RXN-7957',\n",
       "   'Name: AK; ATP:acetate phosphotransferase; Acetate kinase; Acetokinase; AckA; acetate kinase; acetate kinase (EC:2.7.2.1); acetate kinase (phosphorylating); acetatekinase; acetic kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00029[0] <=> (1) cpd00008[0] + (1) cpd00196[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00029;cpd00196',\n",
       "  'definition': '(1) ATP[0] + (1) Acetate[0] <=> (1) ADP[0] + (1) Acetylphosphate[0]',\n",
       "  'deltag': 3.41,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.1', '2.7.2.15'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00029[0] <=> (1) cpd00008[0] + (1) cpd00196[0]',\n",
       "  'id': 'rxn00225',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27661;rxn27662;rxn34299',\n",
       "  'name': 'ATP:acetate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ACETATEUTIL-PWY (superpathway of acetate utilization and formation); AMINE-DEG (Amine and Polyamine Degradation); ANAEROBIC-RESPIRATION (Anaerobic Respiration); ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Fermentation-to-propanoate (Fermentation to Propanoate); GALLATE-DEG (Gallate Degradation); GLYCINE-DEG (Glycine Degradation); LYSINE-DEG (L-lysine Degradation); METH-ACETATE-PWY (methanogenesis from acetate); METHANOGENESIS (Methanogenesis); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); P124-PWY (Bifidobacterium shunt); P142-PWY (pyruvate fermentation to acetate I); P161-PWY (acetylene degradation); P163-PWY (L-lysine fermentation to acetate and butanoate); P164-PWY (purine nucleobases degradation I (anaerobic)); P3-PWY (gallate degradation III (anaerobic)); P41-PWY (pyruvate fermentation to acetate and lactate I); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PROPFERM-PWY (L-alanine fermentation to propanoate and acetate); PWY-5100 (pyruvate fermentation to acetate and lactate II); PWY-5327 (superpathway of L-lysine degradation); PWY-5482 (pyruvate fermentation to acetate II); PWY-5485 (pyruvate fermentation to acetate IV); PWY-5497 (purine nucleobases degradation II (anaerobic)); PWY-5600 (pyruvate fermentation to acetate VII); PWY-5676 (acetyl-CoA fermentation to butanoate II); PWY-6590 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6830 (superpathway of methanogenesis); PWY-8015 (glycine degradation (Stickland reaction)); PWY0-1312 (acetate and ATP formation from acetyl-CoA I); PWY0-1477 (ethanolamine utilization); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Purine-Degradation (Purine Nucleotide Degradation); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Pyruvate-Lactate-Fermentation (pyruvate fermentation to lactate); Respiration (); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00430 (Taurine and hypotaurine metabolism); rn00620 (Pyruvate metabolism); rn00680 (Methane metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00029:0:0:\"Acetate\";1:cpd00008:0:0:\"ADP\";1:cpd00196:0:0:\"Acetylphosphate\"'},\n",
       " {'abbreviation': 'R00316',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00316_m',\n",
       "   'AraGEM: R_R00316_m',\n",
       "   'KEGG: R00316',\n",
       "   'Maize_C4GEM: R_R00316_m',\n",
       "   'TS_Athaliana: rxn567_m; rxn567_p',\n",
       "   'Name: ATP:acetate adenylyltransferase; acetyl-CoA synthetase (acetate-CoA ligase), putative'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00029[0] <=> (1) cpd00012[0] + (1) cpd03566[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00029;cpd03566',\n",
       "  'definition': '(1) ATP[0] + (1) Acetate[0] => (1) PPi[0] + (1) Acetyl adenylate[0]',\n",
       "  'deltag': -9.59,\n",
       "  'deltagerr': 4.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00029[0] => (1) cpd00012[0] + (1) cpd03566[0]',\n",
       "  'id': 'rxn00226',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31212;rxn32413',\n",
       "  'name': 'ATP:acetate adenylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00029:0:0:\"Acetate\";1:cpd00012:0:0:\"PPi\";1:cpd03566:0:0:\"Acetyl adenylate\"'},\n",
       " {'abbreviation': 'R00317',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00317_c',\n",
       "   'BiGG: ACYP_2',\n",
       "   'JP_Creinhardtii_MSB: R_APhi',\n",
       "   'JP_Creinhardtii_NMeth: R_APhi',\n",
       "   'KEGG: R00317',\n",
       "   'Name: Acetyl phosphate phosphohydrolase; Acylphosphatase; acylphosphatase (2)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00196[0] <=> (1) cpd00009[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00029;cpd00067;cpd00196',\n",
       "  'definition': '(1) H2O[0] + (1) Acetylphosphate[0] => (1) Phosphate[0] + (1) Acetate[0] + (1) H+[0]',\n",
       "  'deltag': -9.57,\n",
       "  'deltagerr': 0.15,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00196[0] => (1) cpd00009[0] + (1) cpd00029[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00227',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32635;rxn34196',\n",
       "  'name': 'Acetyl phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00196:0:0:\"Acetylphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00318',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00318',\n",
       "   'MetaCyc: 3.11.1.2-RXN',\n",
       "   'Name: Phosphonoacetate phosphonohydrolase; phosphonoacetate hydrolase; phosphonoacetate phosphonohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03385[0] <=> (1) cpd00009[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00029;cpd00067;cpd03385',\n",
       "  'definition': '(1) H2O[0] + (1) Fosfonet[0] => (1) Phosphate[0] + (1) Acetate[0] + (1) H+[0]',\n",
       "  'deltag': -20.13,\n",
       "  'deltagerr': 1.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.11.1.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03385[0] => (1) cpd00009[0] + (1) cpd00029[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00228',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phosphonoacetate phosphonohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); P483-PWY (phosphonoacetate degradation); Phosphorus-Compounds (Phosphorus Compound Metabolism)',\n",
       "   'KEGG: rn00440 (Phosphonate and phosphinate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03385:0:0:\"Fosfonet\";1:cpd00009:0:0:\"Phosphate\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:H+'},\n",
       " {'abbreviation': 'R00319',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: LO',\n",
       "   'KEGG: R00319',\n",
       "   'MetaCyc: LACTATE-2-MONOOXYGENASE-RXN; RXN-19631',\n",
       "   'iMEO21: LO',\n",
       "   'iNJ661: LO',\n",
       "   'Name: (S)-Lactate:oxygen 2-oxidoreductase(decarboxylating); (S)-lactate:oxygen 2-oxidoreductase (decarboxylating); L-lactate monooxygenase; L-lactate oxidase; L-lactate-2-monooxygenase; lactate 2-monooxygenase; lactate monooxygenase; lactate oxidase; lactate oxidative decarboxylase; lactate oxygenase; lactic oxidase; lactic oxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00159[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00011;cpd00029;cpd00159',\n",
       "  'definition': '(1) O2[0] + (1) L-Lactate[0] => (1) H2O[0] + (1) CO2[0] + (1) Acetate[0]',\n",
       "  'deltag': -118.86,\n",
       "  'deltagerr': 1.08,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.12.4'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00159[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00029[0]',\n",
       "  'id': 'rxn00229',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-lactate:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00159:0:0:\"L-Lactate\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd00029:0:0:\"Acetate\"'},\n",
       " {'abbreviation': 'R00320',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00320',\n",
       "   'MetaCyc: ACETATE-KINASE-PYROPHOSPHATE-RXN',\n",
       "   'iRS1563: R_R00320_c',\n",
       "   'Name: Pyrophosphate:acetate phosphotransferase; acetate kinase (diphosphate); diphosphate:acetate phosphotransferase; pyrophosphate-acetate phosphotransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00029[0] <=> (1) cpd00009[0] + (1) cpd00196[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00012;cpd00029;cpd00196',\n",
       "  'definition': '(1) PPi[0] + (1) Acetate[0] <=> (1) Phosphate[0] + (1) Acetylphosphate[0]',\n",
       "  'deltag': 6.1,\n",
       "  'deltagerr': 0.16,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.12'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00029[0] <=> (1) cpd00009[0] + (1) cpd00196[0]',\n",
       "  'id': 'rxn00230',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'diphosphate:acetate phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00029:0:0:\"Acetate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00196:0:0:\"Acetylphosphate\"'},\n",
       " {'abbreviation': 'R00321',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AMID4',\n",
       "   'JP_Creinhardtii_MSB: R_AMID',\n",
       "   'KEGG: R00321',\n",
       "   'MetaCyc: RXN-14728',\n",
       "   'iIT341: AMID4',\n",
       "   'Name: Acetamide amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03734[0] <=> (1) cpd00013[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00029;cpd03734',\n",
       "  'definition': '(1) H2O[0] + (1) Acetamide[0] => (1) NH3[0] + (1) Acetate[0]',\n",
       "  'deltag': -5.66,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03734[0] => (1) cpd00013[0] + (1) cpd00029[0]',\n",
       "  'id': 'rxn00231',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30303',\n",
       "  'name': 'Acetamide amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03734:0:0:\"Acetamide\";1:cpd00013:0:0:\"NH3\";1:cpd00029:0:0:\"Acetate\"'},\n",
       " {'abbreviation': 'R00322',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00322',\n",
       "   \"Name: 5'-Acetylphosphoadenosine acylhydrolase; 5'-acetylphosphoadenosine acetylhydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03566[0] <=> (1) cpd00018[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00018;cpd00029;cpd00067;cpd03566',\n",
       "  'definition': '(1) H2O[0] + (1) Acetyl adenylate[0] => (1) AMP[0] + (1) Acetate[0] + (2) H+[0]',\n",
       "  'deltag': -17.71,\n",
       "  'deltagerr': 4.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.20'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03566[0] => (1) cpd00018[0] + (1) cpd00029[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00232',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"5'-acetylphosphoadenosine acetylhydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03566:0:0:\"Acetyl adenylate\";1:cpd00018:0:0:\"AMP\";1:cpd00029:0:0:\"Acetate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00324',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00324',\n",
       "   'MetaCyc: ACETYLPYRUVATE-HYDROLASE-RXN',\n",
       "   'Name: 2,4-dioxopentanoate acetylhydrolase; Acetylpyruvate acetylhydrolase; acetylpyruvate hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01443[0] <=> (1) cpd00020[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00020;cpd00029;cpd00067;cpd01443',\n",
       "  'definition': '(1) H2O[0] + (1) Acetylpyruvate[0] => (1) Pyruvate[0] + (1) Acetate[0] + (1) H+[0]',\n",
       "  'deltag': -10.35,\n",
       "  'deltagerr': 1.06,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.7.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01443[0] => (1) cpd00020[0] + (1) cpd00029[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00233',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2,4-dioxopentanoate acetylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); P342-PWY (orcinol degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01443:0:0:\"Acetylpyruvate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00325',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: CITRAMALATE-LYASE-RXN',\n",
       "   'KEGG: R00325',\n",
       "   'MetaCyc: CITRAMALATE-LYASE-RXN',\n",
       "   'Name: (+)-citramalate pyruvate-lyase; (2S)-2-hydroxy-2-methylbutanedioate pyruvate-lyase; (2S)-2-hydroxy-2-methylbutanedioate pyruvate-lyase (acetate-forming); (3S)-citramalate pyruvate-lyase; (S)-citramalate lyase; citramalate lyase; citramalate pyruvate lyase; citramalate pyruvate-lyase; citramalate synthase; citramalate synthetase; citramalic synthase; citramalic-condensing enzyme'],\n",
       "  'code': '(1) cpd01701[0] <=> (1) cpd00020[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00029;cpd01701',\n",
       "  'definition': '(1) S-Citramalate[0] <=> (1) Pyruvate[0] + (1) Acetate[0]',\n",
       "  'deltag': 0.62,\n",
       "  'deltagerr': 0.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.22'],\n",
       "  'equation': '(1) cpd01701[0] <=> (1) cpd00020[0] + (1) cpd00029[0]',\n",
       "  'id': 'rxn00234',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(2S)-2-hydroxy-2-methylbutanedioate pyruvate-lyase (acetate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); GLUDEG-II-PWY (L-glutamate degradation VII (to butanoate)); GLUTAMATE-DEG (L-glutamate Degradation); PWY-5087 (L-glutamate degradation VI (to pyruvate)); PWY-5088 (L-glutamate degradation VIII (to propanoate)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00660 (C5-Branched dibasic acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd01701:0:0:\"S-Citramalate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00029:0:0:\"Acetate\"'},\n",
       " {'abbreviation': 'R00327',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00327',\n",
       "   'MetaCyc: 6-ACETYLGLUCOSE-DEACETYLASE-RXN',\n",
       "   'Name: 6-Acetyl-D-glucose acetylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01726[0] <=> (1) cpd00027[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00029;cpd00067;cpd01726',\n",
       "  'definition': '(1) H2O[0] + (1) 6-Acetyl-D-glucose[0] => (1) D-Glucose[0] + (1) Acetate[0] + (1) H+[0]',\n",
       "  'deltag': -6.8,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.1.33'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01726[0] => (1) cpd00027[0] + (1) cpd00029[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00235',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '6-Acetyl-D-glucose acetylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01726:0:0:\"6-Acetyl-D-glucose\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00328',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN',\n",
       "   'BiGG: NDP3; NDP3g',\n",
       "   'EcoCyc: GUANOSINE-DIPHOSPHATASE-RXN; NUCLEOSIDE-DIPHOSPHATASE-RXN',\n",
       "   'KEGG: R00328',\n",
       "   'MetaCyc: GUANOSINE-DIPHOSPHATASE-RXN; NUCLEOSIDE-DIPHOSPHATASE-RXN',\n",
       "   'Name: GDP phosphohydrolase; GDPase; guanosine-diphosphatase; nucleoside-diphosphatase (GDP)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00031[0] <=> (1) cpd00009[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00031;cpd00067;cpd00126',\n",
       "  'definition': '(1) H2O[0] + (1) GDP[0] => (1) Phosphate[0] + (2) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -5.52,\n",
       "  'deltagerr': 0.3,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.42', '3.6.1.5', '3.6.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00031[0] => (1) cpd00009[0] + (2) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00236',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30413',\n",
       "  'name': 'GDP phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00031:0:0:\"GDP\";1:cpd00009:0:0:\"Phosphate\";2:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R00330',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GDPKIN-RXN; NUCLEOSIDE-DIP-KIN-RXN',\n",
       "   'BiGG: NDPK1; YNK1_8',\n",
       "   'BrachyCyc: GDPKIN-RXN; NUCLEOSIDE-DIP-KIN-RXN',\n",
       "   'KEGG: R00330',\n",
       "   'MetaCyc: GDPKIN-RXN; NUCLEOSIDE-DIP-KIN-RXN',\n",
       "   'Name: ATP:GDP phosphotransferase; GDP phosphotransferase; gdp kinase; nucleoside-diphosphate kinase (ATP:GDP); nucleoside-diphosphatekinase-ATP-GDP'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00031[0] <=> (1) cpd00008[0] + (1) cpd00038[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00031;cpd00038',\n",
       "  'definition': '(1) ATP[0] + (1) GDP[0] <=> (1) ADP[0] + (1) GTP[0]',\n",
       "  'deltag': -0.6,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00031[0] <=> (1) cpd00008[0] + (1) cpd00038[0]',\n",
       "  'id': 'rxn00237',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19769;rxn27873;rxn27874;rxn33024',\n",
       "  'name': 'ATP:GDP phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Metabolic-Regulators (Metabolic Regulator Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PPGPPMET-PWY (ppGpp biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6125 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis II); PWY-7221 (guanosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7228 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00031:0:0:\"GDP\";1:cpd00008:0:0:\"ADP\";1:cpd00038:0:0:\"GTP\"'},\n",
       " {'abbreviation': 'R00331',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00331',\n",
       "   'Name: ATP:nucleoside-diphosphate phosphotransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00354[0] <=> (1) cpd00008[0] + (1) cpd00173[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00173;cpd00354',\n",
       "  'definition': '(1) ATP[0] + (1) NDP[0] <=> (1) ADP[0] + (1) NTP[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00354[0] <=> (1) cpd00008[0] + (1) cpd00173[0]',\n",
       "  'id': 'rxn00238',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:nucleoside-diphosphate phosphotransferase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00354:0:0:\"NDP\";1:cpd00008:0:0:\"ADP\";1:cpd00173:0:0:\"NTP\"'},\n",
       " {'abbreviation': 'R00332',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GUANYL-KIN-RXN',\n",
       "   'BiGG: GK1; GUK1_1',\n",
       "   'BrachyCyc: GUANYL-KIN-RXN',\n",
       "   'KEGG: R00332',\n",
       "   'MetaCyc: GUANYL-KIN-RXN',\n",
       "   \"Name: 5'-GMP kinase; ATP:(d)GMP phosphotransferase; ATP:GMP phosphotransferase; Deoxyguanylate kinase; GMP kinase; Guanosine monophosphate kinase; Guanylate kinase; deoxyguanylate kinase; gmp kinase; guanosine monophosphate kinase; guanylate kinase; guanylate kinase (GMP:ATP); guanylatekinase-GMP-ATP\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00126[0] <=> (1) cpd00008[0] + (1) cpd00031[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00031;cpd00067;cpd00126',\n",
       "  'definition': '(1) ATP[0] + (1) H+[0] + (1) GMP[0] <=> (1) ADP[0] + (1) GDP[0]',\n",
       "  'deltag': -0.64,\n",
       "  'deltagerr': 0.29,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.8'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00067[0] + (1) cpd00126[0] <=> (1) cpd00008[0] + (1) cpd00031[0]',\n",
       "  'id': 'rxn00239',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27924;rxn32165;rxn32168;rxn32175;rxn32176',\n",
       "  'name': 'ATP:GMP phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6125 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis II); PWY-7221 (guanosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7228 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00067:0:0:\"H+\";-1:cpd00126:0:0:\"GMP\";1:cpd00008:0:0:\"ADP\";1:cpd00031:0:0:\"GDP\"'},\n",
       " {'abbreviation': 'R00333',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00333',\n",
       "   'Name: nucleoside-triphosphate:AMP phosphotransferase'],\n",
       "  'code': '(1) cpd00018[0] + (1) cpd00173[0] <=> (1) cpd00008[0] + (1) cpd00354[0]',\n",
       "  'compound_ids': 'cpd00008;cpd00018;cpd00067;cpd00173;cpd00354',\n",
       "  'definition': '(1) AMP[0] + (1) H+[0] + (1) NTP[0] <=> (1) ADP[0] + (1) NDP[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.10'],\n",
       "  'equation': '(1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00173[0] <=> (1) cpd00008[0] + (1) cpd00354[0]',\n",
       "  'id': 'rxn00240',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'nucleoside-triphosphate:AMP phosphotransferase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00018:0:0:\"AMP\";-1:cpd00067:0:0:\"H+\";-1:cpd00173:0:0:\"NTP\";1:cpd00008:0:0:\"ADP\";1:cpd00354:0:0:\"NDP\"'},\n",
       " {'abbreviation': 'R00335',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.5.5-RXN; NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN0-1021; RXN0-5462',\n",
       "   'BiGG: NTP3; NTP3pp',\n",
       "   'BrachyCyc: 3.6.5.1-RXN; 3.6.5.2-RXN; 3.6.5.3-RXN; 3.6.5.4-RXN; NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN0-5462',\n",
       "   'KEGG: R00335',\n",
       "   'MetaCyc: 3.6.5.1-RXN; 3.6.5.2-RXN; 3.6.5.3-RXN; 3.6.5.4-RXN; 3.6.5.5-RXN; 3.6.5.6-RXN; NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN0-5462',\n",
       "   'Name: GTP phosphohydrolase; GTP phosphohydrolase (cell-regulating); GTP phosphohydrolase (signalling); GTP phosphohydrolase (vesicle-releasing); heterotrimeric G-protein GTPase; nucleoside-triphosphatase (GTP); small monomeric GTPase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00038[0] <=> (1) cpd00009[0] + (1) cpd00031[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00031;cpd00038;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) GTP[0] => (1) Phosphate[0] + (1) GDP[0] + (1) H+[0]',\n",
       "  'deltag': -5.56,\n",
       "  'deltagerr': 0.33,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.15',\n",
       "   '3.6.1.5',\n",
       "   '3.6.5.1',\n",
       "   '3.6.5.2',\n",
       "   '3.6.5.3',\n",
       "   '3.6.5.4',\n",
       "   '3.6.5.5',\n",
       "   '3.6.5.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00038[0] => (1) cpd00009[0] + (1) cpd00031[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00241',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18911;rxn27617;rxn32003;rxn32004;rxn32006;rxn37993;rxn37994;rxn38229;rxn38239',\n",
       "  'name': 'GTP phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00038:0:0:\"GTP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00031:0:0:\"GDP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00336',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PPGPPSYN-RXN',\n",
       "   'BiGG: G35DP; GDPDP3PH; PPGPP; PPGPPDP',\n",
       "   'BrachyCyc: PPGPPSYN-RXN',\n",
       "   'KEGG: R00336',\n",
       "   'MetaCyc: PPGPPSYN-RXN',\n",
       "   \"Name: Guanosine-3',5'-bis(diphosphate) 3'-pyrophosphohydrolase; PpGpp phosphohydrolase; PpGpp-3'-pyrophosphohydrolase; guanosine-3',5'-bis(diphosphate) 3'-diphosphatase; guanosine-3',5'-bis(diphosphate) 3'-diphosphohydrolase; guanosine-3',5'-bis(diphosphate) 3'-pyrophosphatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00905[0] <=> (1) cpd00012[0] + (1) cpd00031[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00031;cpd00905',\n",
       "  'definition': \"(1) H2O[0] + (1) Guanosine 3',5'-bis(diphosphate)[0] => (1) PPi[0] + (1) GDP[0]\",\n",
       "  'deltag': -5.6,\n",
       "  'deltagerr': 1.51,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.7.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00905[0] => (1) cpd00012[0] + (1) cpd00031[0]',\n",
       "  'id': 'rxn00242',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33062',\n",
       "  'name': \"guanosine-3',5'-bis(diphosphate) 3'-diphosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Metabolic-Regulators (Metabolic Regulator Biosynthesis); PPGPPMET-PWY (ppGpp biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00905:0:0:\"Guanosine 3\\',5\\'-bis(diphosphate)\";1:cpd00012:0:0:\"PPi\";1:cpd00031:0:0:\"GDP\"'},\n",
       " {'abbreviation': 'R00337',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00337',\n",
       "   'MetaCyc: GDP-GLUCOSIDASE-RXN',\n",
       "   'Name: GDP-glucose glucohydrolase; GDPglucose glucohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00315[0] <=> (1) cpd00027[0] + (1) cpd00031[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00031;cpd00315',\n",
       "  'definition': '(1) H2O[0] + (1) GDP-glucose[0] => (1) D-Glucose[0] + (1) GDP[0]',\n",
       "  'deltag': -6.56,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.42'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00315[0] => (1) cpd00027[0] + (1) cpd00031[0]',\n",
       "  'id': 'rxn00243',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GDP-glucose glucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00315:0:0:\"GDP-glucose\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00031:0:0:\"GDP\"'},\n",
       " {'abbreviation': 'R00338',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00338',\n",
       "   'MetaCyc: OXALOACETASE-RXN',\n",
       "   'PlantCyc: OXALOACETASE-RXN',\n",
       "   'iRS1563: R_R00338_c',\n",
       "   'Name: Oxaloacetate acetylhydrolase; oxalacetic hydrolase; oxaloacetase; oxaloacetate acetylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00032[0] <=> (1) cpd00029[0] + (1) cpd00180[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00032;cpd00067;cpd00180',\n",
       "  'definition': '(1) H2O[0] + (1) Oxaloacetate[0] => (1) Acetate[0] + (1) H+[0] + (1) Oxalate[0]',\n",
       "  'deltag': -12.31,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.7.1.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00032[0] => (1) cpd00029[0] + (1) cpd00067[0] + (1) cpd00180[0]',\n",
       "  'id': 'rxn00244',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Oxaloacetate acetylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: PWY-6699 (oxalate biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\";1:cpd00180:0:0:\"Oxalate\"'},\n",
       " {'abbreviation': 'R00339',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: TARTD',\n",
       "   'EcoCyc: LTARTDEHYDRA-RXN',\n",
       "   'KEGG: R00339',\n",
       "   'MetaCyc: LTARTDEHYDRA-RXN',\n",
       "   'iAF1260: TARTD',\n",
       "   'Name: (R,R)-Tartrate hydro-lyase; (R,R)-tartrate hydro-lyase; (R,R)-tartrate hydro-lyase (oxaloacetate-forming); (S,S)-Tartrate hydro-lyase; L(+)-tartrate dehydratase; L-(+)-tartaric acid dehydratase; L-tartrate dehydratase; tartaric acid dehydrase; tartrate dehydratase'],\n",
       "  'code': '(1) cpd00666[0] <=> (1) cpd00001[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00032;cpd00666',\n",
       "  'definition': '(1) Tartrate[0] => (1) H2O[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': -8.31,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.32'],\n",
       "  'equation': '(1) cpd00666[0] => (1) cpd00001[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00245',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(R,R)-tartrate hydro-lyase (oxaloacetate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00666:0:0:\"Tartrate\";1:cpd00001:0:0:\"H2O\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00341',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PEPCARBOXYKIN-RXN',\n",
       "   'BiGG: PCK1; PPCK; PPCKr',\n",
       "   'BrachyCyc: PEPCARBOXYKIN-RXN',\n",
       "   'KEGG: R00341',\n",
       "   'MetaCyc: PEPCARBOXYKIN-RXN',\n",
       "   'Name: ATP:oxaloacetate carboxy-lyase (transphosphorylating); ATP:oxaloacetate carboxy-lyase (transphosphorylating, phosphoenolpyruvate-forming); ATP:oxaloacetate carboxy-lyase (transphosphorylating;phosphoenolpyruvate-forming); PEP carboxykinase; PEP carboxylase; PEPCK; PEPCK (ATP); PEPK; phosphoenolpyruvate carboxykinase; phosphoenolpyruvate carboxykinase (ATP); phosphoenolpyruvate carboxylase; phosphoenolpyruvate carboxylase (ATP); phosphoenolpyruvic carboxykinase; phosphoenolpyruvic carboxylase; phosphopyruvate carboxykinase; phosphopyruvate carboxykinase (adenosine triphosphate); phosphopyruvate carboxylase (ATP)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00032[0] <=> (1) cpd00008[0] + (1) cpd00011[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00011;cpd00032;cpd00061',\n",
       "  'definition': '(1) ATP[0] + (1) Oxaloacetate[0] => (1) ADP[0] + (1) CO2[0] + (1) Phosphoenolpyruvate[0]',\n",
       "  'deltag': 2.57,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.49'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00032[0] => (1) cpd00008[0] + (1) cpd00011[0] + (1) cpd00061[0]',\n",
       "  'id': 'rxn00247',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35519',\n",
       "  'name': 'ATP:oxaloacetate carboxy-lyase (transphosphorylating;phosphoenolpyruvate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); GLUCONEO-PWY (gluconeogenesis I); Gluconeogenesis (Gluconeogenesis); PWY-561 (superpathway of glyoxylate cycle and fatty acid degradation); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); Photosynthesis ()',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00008:0:0:\"ADP\";1:cpd00011:0:0:\"CO2\";1:cpd00061:0:0:\"Phosphoenolpyruvate\"'},\n",
       " {'abbreviation': 'R00342',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MAL-DEH-GLYOX-RXN; MALATE-DEH-RXN; RXN-13408',\n",
       "   'BiGG: MDH; MDH1; MDHi2; MDHm; MDHp',\n",
       "   'BrachyCyc: MALATE-DEH-RXN',\n",
       "   'KEGG: R00342',\n",
       "   'MetaCyc: MALATE-DEH-RXN; RXN-10813; RXN-13408',\n",
       "   'Name: (S)-malate:NAD+ oxidoreductase; L-malate dehydrogenase; L-malate-NAD+ oxidoreductase; Malate dehydrogenase; NAD-L-malate dehydrogenase; NAD-dependent malate dehydrogenase; NAD-dependent malic dehydrogenase; NAD-linked malate dehydrogenase; NAD-malate dehydrogenase; NAD-malic dehydrogenase; NAD-specific malate dehydrogenase; malate (NAD) dehydrogenase; malate dehydrogenase; malate dehydrogenase (NAD); malate dehydrogenase (other direction); malate dehydrogenation; malic acid dehydrogenase; malic dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00130[0] <=> (1) cpd00004[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00032;cpd00067;cpd00130',\n",
       "  'definition': '(1) NAD[0] + (1) L-Malate[0] <=> (1) NADH[0] + (1) Oxaloacetate[0] + (1) H+[0]',\n",
       "  'deltag': 7.09,\n",
       "  'deltagerr': 0.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.299', '1.1.1.37'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00130[0] <=> (1) cpd00004[0] + (1) cpd00032[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00248',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20068;rxn23652;rxn27976;rxn34245',\n",
       "  'name': '(S)-malate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARTATE-DEG (L-aspartate Degradation); Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Degradation (Amino Acid Degradation); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); Formaldehyde-Assimilation (Formaldehyde Assimilation); GLUCONEO-PWY (gluconeogenesis I); GLUTAMATE-DEG (L-glutamate Degradation); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYOXYLATE-BYPASS (glyoxylate cycle); Gluconeogenesis (Gluconeogenesis); MALATE-ASPARTATE-SHUTTLE-PWY (L-aspartate degradation II); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); P108-PWY (pyruvate fermentation to propanoate I); P23-PWY (reductive TCA cycle I); P42-PWY (incomplete reductive TCA cycle); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-5088 (L-glutamate degradation VIII (to propanoate)); PWY-5392 (reductive TCA cycle II); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-561 (superpathway of glyoxylate cycle and fatty acid degradation); PWY-5690 (TCA cycle II (plants and fungi)); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6728 (methylaspartate cycle); PWY-6969 (TCA cycle V (2-oxoglutarate:ferredoxin oxidoreductase)); PWY-7115 (C4 photosynthetic carbon assimilation cycle, NAD-ME type); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY66-398 (TCA cycle III (animals)); PWY66-399 (gluconeogenesis III); Photosynthesis (); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Pyruvate-Propanoate-Fermentation (Pyruvate Fermentation to Propanoate); Reductive-TCA-Cycles (Reductive TCA Cycles); TCA (TCA cycle I (prokaryotic)); TCA-GLYOX-BYPASS (superpathway of glyoxylate bypass and TCA); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00680 (Methane metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00004:0:0:\"NADH\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00343',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MALATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'BiGG: MDHy',\n",
       "   'BrachyCyc: MALATE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'KEGG: R00343',\n",
       "   'MetaCyc: MALATE-DEHYDROGENASE-NADP+-RXN; RXN-10813',\n",
       "   'Name: (S)-Malate:NADP+ oxidoreductase; (S)-malate:NADP+ oxidoreductase; NADP malate dehydrogenase; NADP-linked malate dehydrogenase; NADP-malate dehydrogenase; NADP-malic enzyme; malate NADP dehydrogenase; malate dehydrogenase (NADP); malate dehydrogenase (NADP+); malate dehydrogenase [NADP], putative; malic dehydrogenase (nicotinamide adenine dinucleotide phosphate)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00130[0] <=> (1) cpd00005[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00032;cpd00067;cpd00130',\n",
       "  'definition': '(1) NADP[0] + (1) L-Malate[0] <=> (1) NADPH[0] + (1) Oxaloacetate[0] + (1) H+[0]',\n",
       "  'deltag': 7.09,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.299', '1.1.1.82'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00130[0] <=> (1) cpd00005[0] + (1) cpd00032[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00249',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20158;rxn31213',\n",
       "  'name': '(S)-malate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-241 (C4 photosynthetic carbon assimilation cycle, NADP-ME type); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); Photosynthesis ()',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00005:0:0:\"NADPH\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00344',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRUVATE-CARBOXYLASE-RXN',\n",
       "   'BiGG: PC; PYC1; PYC2',\n",
       "   'ChlamyCyc: PYRUVATE-CARBOXYLASE-RXN',\n",
       "   'KEGG: R00344',\n",
       "   'MetaCyc: PYRUVATE-CARBOXYLASE-RXN',\n",
       "   'Name: Pyruvate carboxylase; Pyruvate:carbon-dioxide ligase (ADP-forming); Pyruvic carboxylase; pyruvate carboxylase; pyruvate:carbon-dioxide ligase (ADP-forming); pyruvic carboxylase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00020[0] + (1) cpd00242[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00020;cpd00032;cpd00067;cpd00242',\n",
       "  'definition': '(1) ATP[0] + (1) Pyruvate[0] + (1) H2CO3[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) Oxaloacetate[0] + (1) H+[0]',\n",
       "  'deltag': -0.94,\n",
       "  'deltagerr': 0.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.4.1.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00020[0] + (1) cpd00242[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00032[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00250',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28116',\n",
       "  'name': 'Pyruvate:carbon-dioxide ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Gluconeogenesis (Gluconeogenesis); P42-PWY (incomplete reductive TCA cycle); PWY-6142 (gluconeogenesis II (<i>Methanobacterium thermoautotrophicum</i>)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY66-399 (gluconeogenesis III); Reductive-TCA-Cycles (Reductive TCA Cycles)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00242:0:0:\"H2CO3\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00345',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00345_c',\n",
       "   'AraGEM: R_R00345_c',\n",
       "   'BiGG: PPC',\n",
       "   'JM_Creinhardtii: JM_Cre_31',\n",
       "   'KEGG: R00345',\n",
       "   'Name: Orthophosphate:oxaloacetate carboxyl-lyase (phosphorylating); Phosphoenolpyruvate carboxylase; phosphate:oxaloacetate carboxy-lyase (adding phosphate;phosphoenolpyruvate-forming); phosphoenolpyruvate carboxylase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00032[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00011;cpd00032;cpd00061;cpd00067',\n",
       "  'definition': '(1) Phosphate[0] + (1) Oxaloacetate[0] + (1) H+[0] => (1) H2O[0] + (1) CO2[0] + (1) Phosphoenolpyruvate[0]',\n",
       "  'deltag': 8.73,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.31'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00032[0] + (1) cpd00067[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00061[0]',\n",
       "  'id': 'rxn00251',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32421',\n",
       "  'name': 'phosphate:oxaloacetate carboxy-lyase (adding phosphate;phosphoenolpyruvate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism); rn00680 (Methane metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00032:0:0:\"Oxaloacetate\";-1:cpd00067:0:0:\"H+\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd00061:0:0:\"Phosphoenolpyruvate\"'},\n",
       " {'abbreviation': 'R00346',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 4.1.1.38-RXN',\n",
       "   'DF_Athaliana: reac_117',\n",
       "   'KEGG: R00346',\n",
       "   'MetaCyc: 4.1.1.38-RXN',\n",
       "   'TS_Athaliana: rxn111_r; rxn111_v; rxn111_x',\n",
       "   'Name: PEP carboxykinase; PEP carboxylase; PEP carboxyphosphotransferase; PEPCTrP; Phosphoenolpyruvate carboxykinase (pyrophosphate); Pyrophosphate:oxaloacetate carboxy-lyase (transphosphorylating); diphosphate:oxaloacetate carboxy-lyase (transphosphorylating); diphosphate:oxaloacetate carboxy-lyase (transphosphorylating, phosphoenolpyruvate-forming); diphosphate:oxaloacetate carboxy-lyase (transphosphorylating;phosphoenolpyruvate-forming); phosphoenolpyruvate carboxykinase; phosphoenolpyruvate carboxykinase (diphosphate); phosphoenolpyruvate carboxylase; phosphoenolpyruvate carboxylase (pyrophosphate); phosphoenolpyruvate carboxyphosphotransferase; phosphoenolpyruvate carboxytransphosphorylase; phosphoenolpyruvic carboxykinase; phosphoenolpyruvic carboxykinase (pyrophosphate); phosphoenolpyruvic carboxylase; phosphoenolpyruvic carboxylase (pyrophosphate); phosphoenolpyruvic carboxytransphosphorylase; phosphopyruvate carboxykinase; phosphopyruvate carboxykinase (pyrophosphate); phosphopyruvate carboxylase; phosphopyruvate carboxylase (pyrophosphate)'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00032[0] <=> (1) cpd00009[0] + (1) cpd00011[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00011;cpd00012;cpd00032;cpd00061',\n",
       "  'definition': '(1) PPi[0] + (1) Oxaloacetate[0] => (1) Phosphate[0] + (1) CO2[0] + (1) Phosphoenolpyruvate[0]',\n",
       "  'deltag': 5.26,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.38'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00032[0] => (1) cpd00009[0] + (1) cpd00011[0] + (1) cpd00061[0]',\n",
       "  'id': 'rxn00252',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32297;rxn32299;rxn32303',\n",
       "  'name': 'diphosphate:oxaloacetate carboxy-lyase (transphosphorylating;phosphoenolpyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00011:0:0:\"CO2\";1:cpd00061:0:0:\"Phosphoenolpyruvate\"'},\n",
       " {'abbreviation': 'R00347',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00347',\n",
       "   'MetaCyc: 4.3.1.20-RXN',\n",
       "   'Name: erythro-3-hydroxy-L-aspartate ammonia-lyase; erythro-3-hydroxy-L-aspartate ammonia-lyase (oxaloacetate-forming); erythro-3-hydroxy-Ls-aspartate ammonia-lyase; erythro-3-hydroxy-Ls-aspartate ammonia-lyase (oxaloacetate-forming); erythro-3-hydroxy-Ls-aspartate hydro-lyase (deaminating); erythro-3-hydroxyaspartate dehydratase; erythro-beta-hydroxyaspartate dehydratase'],\n",
       "  'code': '(1) cpd02458[0] <=> (1) cpd00013[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00032;cpd02458',\n",
       "  'definition': '(1) erythro-3-Hydroxy-L-aspartate[0] => (1) NH3[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': -6.28,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.20'],\n",
       "  'equation': '(1) cpd02458[0] => (1) cpd00013[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00253',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'erythro-3-hydroxy-Ls-aspartate ammonia-lyase (oxaloacetate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Glycolate-Degradation (Glycolate Degradation); PWY-6649 (glycolate and glyoxylate degradation III)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02458:0:0:\"erythro-3-Hydroxy-L-aspartate\";1:cpd00013:0:0:\"NH3\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00348',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00348',\n",
       "   'MetaCyc: L-OXOACET-RXN; RXN-6562',\n",
       "   'PlantCyc: RXN-6562',\n",
       "   'SoyCyc: RXN-6562',\n",
       "   'Name: 2-Oxosuccinamate amidohydrolase; 2-oxosuccinamate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01577[0] <=> (1) cpd00013[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00032;cpd01577',\n",
       "  'definition': '(1) H2O[0] + (1) 2-Oxosuccinamate[0] => (1) NH3[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': -1.53,\n",
       "  'deltagerr': 0.62,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.-', '3.5.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01577[0] => (1) cpd00013[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00254',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20013;rxn24177',\n",
       "  'name': '2-oxosuccinamate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-DEG (L-asparagine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-4002 (L-asparagine degradation II); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01577:0:0:\"2-Oxosuccinamate\";1:cpd00013:0:0:\"NH3\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00350',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00350',\n",
       "   'MetaCyc: RXN-2464',\n",
       "   'Name: 4-Hydroxy-4-carboxymethyl-2-oxoglutarate pyruvate-lyase; 4-carboxy-4-hydroxy-2-oxoadipate aldolase; 4-carboxy-4-hydroxy-2-oxoadipate oxaloacetate-lyase (pyruvate-forming)'],\n",
       "  'code': '(1) cpd02544[0] <=> (1) cpd00020[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00032;cpd02544',\n",
       "  'definition': '(1) 4-Carboxy-4-hydroxy-2-oxoadipate[0] <=> (1) Pyruvate[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 5.06,\n",
       "  'deltagerr': 0.67,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.17'],\n",
       "  'equation': '(1) cpd02544[0] <=> (1) cpd00020[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00255',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-carboxy-4-hydroxy-2-oxoadipate oxaloacetate-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); GALLATE-DEG (Gallate Degradation); GALLATE-DEGRADATION-I-PWY (gallate degradation II); GALLATE-DEGRADATION-II-PWY (gallate degradation I); METHYLGALLATE-DEGRADATION-PWY (methylgallate degradation); P184-PWY (protocatechuate degradation I (<I>meta</I>-cleavage pathway)); PWY-6338 (superpathway of vanillin and vanillate degradation); PWY-6339 (syringate degradation); Protocatechuate-Degradation (Protocatechuate Degradation); Vanillin-Degradation (Vanillin Degradation)',\n",
       "   'KEGG: rn00362 (Benzoate degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02544:0:0:\"4-Carboxy-4-hydroxy-2-oxoadipate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00351',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CITSYN-RXN',\n",
       "   'BiGG: CIT1; CIT2; CIT3; CS; CSm; CSp',\n",
       "   'BrachyCyc: CITSYN-RXN',\n",
       "   'KEGG: R00351',\n",
       "   'MetaCyc: CITRATE-RE-SYNTHASE-RXN; CITSYN-RXN; RXN-14905',\n",
       "   'Name: (R)-citric synthase; Citrate (si)-synthase; Citrate condensing enzyme; Citrate oxaloacetate-lyase ((pro-3S)-CH2COO- -> acetyl-CoA); Citrate synthase; Citrogenase; Condensing enzyme; Oxaloacetate transacetase; acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolysing); acetyl-CoA:oxaloacetate C-acetyltransferase [thioester-hydrolysing, (pro-S)-carboxymethyl forming]; citrate (Si)-synthase; citrate (si)-synthase; citrate condensing enzyme; citrate oxaloacetate-lyase [(pro-3S)-CH2COO-->acetyl-CoA]; citrate oxaloacetate-lyase, CoA-acetylating; citrate synthase; citrate synthesis; citrate synthetase; citratesynthase; citric synthase; citric-condensing enzyme; condensing enzyme; oxalacetic transacetase; oxaloacetate transacetase'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00137[0] <=> (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00022;cpd00032;cpd00067;cpd00137',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) Citrate[0] <= (1) H2O[0] + (1) Acetyl-CoA[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 8.6,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.1', '2.3.3.16', '2.3.3.3', '4.1.3.7'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00137[0] <= (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00256',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19316;rxn27765;rxn32388;rxn34204',\n",
       "  'name': 'acetyl-CoA:oxaloacetate C-acetyltransferase (thioester-hydrolysing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethylene-Biosynthesis (Ethylene Biosynthesis); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GLUTAMINE-SYN (L-glutamine Biosynthesis); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYOXYLATE-BYPASS (glyoxylate cycle); HORMONE-SYN (Hormone Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-561 (superpathway of glyoxylate cycle and fatty acid degradation); PWY-5690 (TCA cycle II (plants and fungi)); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY-6549 (L-glutamine biosynthesis III); PWY-6728 (methylaspartate cycle); PWY-6969 (TCA cycle V (2-oxoglutarate:ferredoxin oxidoreductase)); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7254 (TCA cycle VII (acetate-producers)); PWY66-398 (TCA cycle III (animals)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); REDCITCYC (TCA cycle VIII (helicobacter)); TCA (TCA cycle I (prokaryotic)); TCA-GLYOX-BYPASS (superpathway of glyoxylate bypass and TCA); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00630 (Glyoxylate and dicarboxylate metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd00137:0:0:\"Citrate\";1:cpd00001:0:0:\"H2O\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00352',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ATP-CITRATE-PRO-S--LYASE-RXN',\n",
       "   'BrachyCyc: ATP-CITRATE-PRO-S--LYASE-RXN',\n",
       "   'ChlamyCyc: ATP-CITRATE-PRO-S--LYASE-RXN',\n",
       "   'KEGG: R00352',\n",
       "   'MetaCyc: ATP-CITRATE-PRO-S--LYASE-RXN',\n",
       "   'Name: (ATP-dephosphorylating); ADP-phosphorylating); ATP citrate (pro-S)-lyase; ATP citrate synthase; ATP-citrate (pro-S-)-lyase; ATP-citric lyase; ATP:citrate oxaloacetate-lyase ((pro-3S)-CH2COO- ->acetyl-CoA); ATP:citrate oxaloacetate-lyase [(pro-S)-CH2COO-->acetyl-CoA] (ATP-dephosphorylating); acetyl-CoA:oxaloacetate C-acetyltransferase [(pro-S)-carboxymethyl-forming, ADP-phosphorylating]; acetyl-CoA:oxaloacetate acetyltransferase (isomerizing; adenosine triphosphate citrate lyase; citrate cleavage enzyme; citrate-ATP lyase; citric cleavage enzyme'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00137[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00010;cpd00022;cpd00032;cpd00137',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Citrate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) Acetyl-CoA[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 2.44,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.3.8'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00137[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00257',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33091;rxn33092',\n",
       "  'name': 'acetyl-CoA:oxaloacetate C-acetyltransferase [(pro-S)-carboxymethyl-forming, ADP-phosphorylating]',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetyl-CoA-Biosynthesis (Acetyl-CoA Biosynthesis); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); P23-PWY (reductive TCA cycle I); PWY-5172 (acetyl-CoA biosynthesis III (from citrate)); PWY-5173 (superpathway of acetyl-CoA biosynthesis); Reductive-TCA-Cycles (Reductive TCA Cycles)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00137:0:0:\"Citrate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00353',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00353',\n",
       "   'iRS1563: R_R00353_c',\n",
       "   'Name: Malonyl-CoA:pyruvate carboxyltransferase; Malonyl-CoA:pyruvate carboxytransferase; malonyl-CoA:pyruvate carboxytransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00070[0] <=> (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00022;cpd00032;cpd00070',\n",
       "  'definition': '(1) Pyruvate[0] + (1) Malonyl-CoA[0] <=> (1) Acetyl-CoA[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 0.02,\n",
       "  'deltagerr': 0.36,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.3.1'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00070[0] <=> (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00258',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Malonyl-CoA:pyruvate carboxytransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00070:0:0:\"Malonyl-CoA\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00354',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00354',\n",
       "   'MetaCyc: RXN-7929',\n",
       "   'Name: (3S)-Citryl-CoA oxaloacetate-lyase; (3S)-citryl-CoA oxaloacetate-lyase; (3S)-citryl-CoA oxaloacetate-lyase (acetyl-CoA-forming); citryl-CoA lyase'],\n",
       "  'code': '(1) cpd00442[0] <=> (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00032;cpd00442',\n",
       "  'definition': '(1) (3S)-Citryl-CoA[0] <=> (1) Acetyl-CoA[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 3.17,\n",
       "  'deltagerr': 5.06,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.34'],\n",
       "  'equation': '(1) cpd00442[0] <=> (1) cpd00022[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00259',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(3S)-citryl-CoA oxaloacetate-lyase (acetyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); PWY-5392 (reductive TCA cycle II); Reductive-TCA-Cycles (Reductive TCA Cycles)',\n",
       "   'KEGG: rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00442:0:0:\"(3S)-Citryl-CoA\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00355',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASPAMINOTRANS-RXN; RXN-13697',\n",
       "   'BiGG: AAT1_1; AAT2_1; ASPTA; ASPTA1; ASPTA1r; ASPTAm; ASPTAp',\n",
       "   'BrachyCyc: ASPAMINOTRANS-RXN',\n",
       "   'KEGG: R00355',\n",
       "   'MetaCyc: ASPAMINOTRANS-RXN; RXN-13697',\n",
       "   'Name: 2-oxoglutarate-glutamate aminotransferase; AAT; AST; AspT; Aspartate aminotransferase; GOT (enzyme); Glutamic--aspartic transaminase; Glutamic--oxaloacetic transaminase; L-Aspartate:2-oxoglutarate aminotransferase; L-aspartate transaminase; L-aspartate-2-ketoglutarate aminotransferase; L-aspartate-2-oxoglutarate aminotransferase; L-aspartate-2-oxoglutarate-transaminase; L-aspartate-alpha-ketoglutarate transaminase; L-aspartate:2-oxoglutarate aminotransferase; L-aspartic aminotransferase; Transaminase A; aspartate alpha-ketoglutarate transaminase; aspartate aminotransferase; aspartate transaminase; aspartate-2-oxoglutarate transaminase; aspartate:2-oxoglutarate aminotransferase; aspartatetransaminase; aspartic acid aminotransferase; aspartic aminotransferase; aspartyl aminotransferase; glutamate oxaloacetate transaminase; glutamate-oxalacetate aminotransferase; glutamate-oxalate transaminase; glutamic oxalic transaminase; glutamic-aspartic aminotransferase; glutamic-aspartic transaminase; glutamic-oxalacetic transaminase; glutamic-oxaloacetic transaminase; oxaloacetate transferase; oxaloacetate-aspartate aminotransferase; transaminase A'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00041[0] <=> (1) cpd00023[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00032;cpd00041',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Aspartate[0] <=> (1) L-Glutamate[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 0.69,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.1'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00041[0] <=> (1) cpd00023[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00260',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27719;rxn27720;rxn30355',\n",
       "  'name': 'L-Aspartate:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-DEG (L-asparagine Degradation); ASPARAGINE-DEG1-PWY-1 (L-asparagine degradation III (mammalian)); ASPARTATE-DEG (L-aspartate Degradation); ASPARTATE-DEG1-PWY (L-aspartate degradation I); ASPARTATE-SYN (L-aspartate Biosynthesis); ASPARTATESYN-PWY (L-aspartate biosynthesis); ASPASN-PWY (superpathway of L-aspartate and L-asparagine biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); GLUTAMATE-DEG (L-glutamate Degradation); GLUTDEG-PWY (L-glutamate degradation II); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); ISOLEUCINE-SYN (L-isoleucine Biosynthesis); MALATE-ASPARTATE-SHUTTLE-PWY (L-aspartate degradation II); METHIONINE-SYN (L-methionine Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); P4-PWY (superpathway of L-lysine, L-threonine and L-methionine biosynthesis I); PWY-3001 (superpathway of L-isoleucine biosynthesis I); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-5347 (superpathway of L-methionine biosynthesis (transsulfuration)); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-7115 (C4 photosynthetic carbon assimilation cycle, NAD-ME type); PWY-7117 (C4 photosynthetic carbon assimilation cycle, PEPCK type); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY0-781 (aspartate superpathway); Photosynthesis (); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TCA-VARIANTS (TCA cycle); THREONINE-BIOSYNTHESIS (L-threonine Biosynthesis); THRESYN-PWY (superpathway of L-threonine biosynthesis)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'rxn00261',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00032[0] <=> (2) cpd00007[0] + (1) cpd03366[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00013;cpd00024;cpd00032;cpd00067;cpd03366',\n",
       "  'definition': \"(1) NH3[0] + (1) 2-Oxoglutarate[0] + (1) Oxaloacetate[0] <=> (2) O2[0] + (3) H+[0] + (1) 5-(2'-Carboxyethyl)-4,6-dihydroxypicolinate[0]\",\n",
       "  'deltag': 159.78,\n",
       "  'deltagerr': 1.77,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.13.11.-'],\n",
       "  'equation': '(1) cpd00013[0] + (1) cpd00024[0] + (1) cpd00032[0] <=> (2) cpd00007[0] + (3) cpd00067[0] + (1) cpd03366[0]',\n",
       "  'id': 'rxn00261',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00261',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:4',\n",
       "  'stoichiometry': '-1:cpd00013:0:0:\"NH3\";-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00032:0:0:\"Oxaloacetate\";2:cpd00007:0:0:\"O2\";3:cpd00067:0:0:\"H+\";1:cpd03366:0:0:\"5-(2\\'-Carboxyethyl)-4,6-dihydroxypicolinate\"'},\n",
       " {'abbreviation': 'R00357',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: L-ASPARTATE-OXIDASE-RXN',\n",
       "   'BiGG: ASPO1; ASPO7',\n",
       "   'ChlamyCyc: L-ASPARTATE-OXIDASE-RXN',\n",
       "   'KEGG: R00357',\n",
       "   'RiceCyc: L-ASPARTATE-OXIDASE-RXN',\n",
       "   'Name: L-Aspartic acid:oxygen oxidoreductase (deaminating); L-aspartate oxidase; l-aspartate oxidase (producing oxaloacetate)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00041[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00032;cpd00041',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Aspartate[0] => (1) NH3[0] + (1) H2O2[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': -22.32,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.16', '1.4.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00041[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00262',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30304;rxn30722;rxn32053;rxn34557',\n",
       "  'name': 'L-Aspartic acid:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00359',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00359_c',\n",
       "   'ChlamyCyc: D-ASPARTATE-OXIDASE-RXN',\n",
       "   'KEGG: R00359',\n",
       "   'MetaCyc: D-ASPARTATE-OXIDASE-RXN',\n",
       "   'Name: D-Aspartate:oxygen oxidoreductase (deaminating); D-aspartate oxidase; D-aspartate:oxygen oxidoreductase (deaminating); D-aspartic oxidase; aspartic oxidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00320[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00032;cpd00320',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) D-Aspartate[0] => (1) NH3[0] + (1) H2O2[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': -23.57,\n",
       "  'deltagerr': 1.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.1', '1.4.3.15'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00320[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00263',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19368',\n",
       "  'name': 'D-Aspartate:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00320:0:0:\"D-Aspartate\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00360',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['EcoCyc: MALOX-RXN',\n",
       "   'KEGG: R00360',\n",
       "   'MetaCyc: MALOX-RXN',\n",
       "   'Name: (S)-Malate:oxygen oxidoreductase; (S)-malate:oxygen oxidoreductase; FAD-dependent malate oxidase; malate oxidase; malic dehydrogenase II; malic oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00130[0] <=> (1) cpd00025[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00032;cpd00130',\n",
       "  'definition': '(1) O2[0] + (1) L-Malate[0] => (1) H2O2[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': -24.63,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.3'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00130[0] => (1) cpd00025[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00264',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20161',\n",
       "  'name': '(S)-Malate:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00025:0:0:\"H2O2\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00362',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CITL; CITLr',\n",
       "   'EcoCyc: CITLY-RXN',\n",
       "   'KEGG: R00362',\n",
       "   'MetaCyc: CITLY-RXN',\n",
       "   'RiceCyc: CITLY-RXN',\n",
       "   'Name: Citrate lyase; Citrate lyase (reversible); Citrate oxaloacetate-lyase ((pro-3S)-CH2COO- -->acetate); citrase; citratase; citrate (pro-3S)-lyase; citrate aldolase; citrate lyase; citrate lyase beta chain; citrate oxaloacetate-lyase; citrate oxaloacetate-lyase (forming acetate from the pro-S carboxymethyl group of citrate); citrate oxaloacetate-lyase [(pro-3S)-CH2COO-->acetate]; citric aldolase; citridesmolase; citritase'],\n",
       "  'code': '(1) cpd00137[0] <=> (1) cpd00029[0] + (1) cpd00032[0]',\n",
       "  'compound_ids': 'cpd00029;cpd00032;cpd00137',\n",
       "  'definition': '(1) Citrate[0] <=> (1) Acetate[0] + (1) Oxaloacetate[0]',\n",
       "  'deltag': 1.3,\n",
       "  'deltagerr': 0.12,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.6'],\n",
       "  'equation': '(1) cpd00137[0] <=> (1) cpd00029[0] + (1) cpd00032[0]',\n",
       "  'id': 'rxn00265',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'citrate oxaloacetate-lyase (forming acetate from the pro-S carboxymethyl group of citrate)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00137:0:0:\"Citrate\";1:cpd00029:0:0:\"Acetate\";1:cpd00032:0:0:\"Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00363',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['EcoCyc: OXALOACETATE-TAUTOMERASE-RXN',\n",
       "   'KEGG: R00363',\n",
       "   'MetaCyc: OXALOACETATE-TAUTOMERASE-RXN',\n",
       "   'iRS1563: R_R00363_c',\n",
       "   'Name: Oxaloacetate keto-enol-isomerase; oxalacetic keto-enol isomerase; oxaloacetate keto-enol-isomerase; oxaloacetate tautomerase'],\n",
       "  'code': '(1) cpd00032[0] <=> (1) cpd02469[0]',\n",
       "  'compound_ids': 'cpd00032;cpd02469',\n",
       "  'definition': '(1) Oxaloacetate[0] <= (1) enol-Oxaloacetate[0]',\n",
       "  'deltag': 1.36,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['5.3.2.2'],\n",
       "  'equation': '(1) cpd00032[0] <= (1) cpd02469[0]',\n",
       "  'id': 'rxn00266',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Oxaloacetate keto-enol-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00032:0:0:\"Oxaloacetate\";1:cpd02469:0:0:\"enol-Oxaloacetate\"'},\n",
       " {'abbreviation': 'R00364',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00364',\n",
       "   'Name: Glycine:ferricytochrome-c oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00033[0] + (2) cpd00109[0] <=> (1) cpd00013[0] + (1) cpd00040[0] + (2) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00033;cpd00040;cpd00067;cpd00109;cpd00110',\n",
       "  'definition': '(1) H2O[0] + (1) Glycine[0] + (2) Cytochrome c3+[0] => (1) NH3[0] + (1) Glyoxalate[0] + (2) H+[0] + (2) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00033[0] + (2) cpd00109[0] => (1) cpd00013[0] + (1) cpd00040[0] + (2) cpd00067[0] + (2) cpd00110[0]',\n",
       "  'id': 'rxn00267',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glycine:ferricytochrome-c oxidoreductase (deaminating)',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00033:0:0:\"Glycine\";-2:cpd00109:0:0:\"Cytochrome c3+\";1:cpd00013:0:0:\"NH3\";1:cpd00040:0:0:\"Glyoxalate\";2:cpd00067:0:0:\"H+\";2:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'R00365',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GLYD',\n",
       "   'KEGG: R00365',\n",
       "   'MetaCyc: GLYCINE-DEHYDROGENASE-RXN',\n",
       "   'SorghumCyc: GLYCINE-DEHYDROGENASE-RXN',\n",
       "   'iJN746: GLYD',\n",
       "   'Name: glycine dehydrogenase; glycine:NAD+ oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00033[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00033;cpd00040;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Glycine[0] <=> (1) NADH[0] + (1) NH3[0] + (1) Glyoxalate[0] + (1) H+[0]',\n",
       "  'deltag': 10.75,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.1', '1.4.1.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00033[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00040[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00268',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30314',\n",
       "  'name': 'glycine:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00033:0:0:\"Glycine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00366',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GLYO1',\n",
       "   'ChlamyCyc: L-AMINO-ACID-OXIDASE-RXN',\n",
       "   'KEGG: R00366',\n",
       "   'MetaCyc: 1.4.3.19-RXN',\n",
       "   'iMO1053-PAO1: GLYO1',\n",
       "   'Name: Aminoacetic acid:oxygen oxidoreductase (deaminating); L-amino acid oxidase; Ophio-amino-acid oxidase; glycine oxidase; glycine:oxygen oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00033[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00033;cpd00040',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Glycine[0] => (1) NH3[0] + (1) H2O2[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': -20.98,\n",
       "  'deltagerr': 1.09,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.19', '1.4.3.2', '1.4.3.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00033[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00269',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38888',\n",
       "  'name': 'glycine:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7396 (butanol and isobutanol biosynthesis (engineered))',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00033:0:0:\"Glycine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00367',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00367',\n",
       "   'MetaCyc: 2.1.1.20-RXN; GLYCINE-N-METHYLTRANSFERASE-RXN',\n",
       "   'iRS1563: R_R00367_c',\n",
       "   'Name: GNMT; S-Adenosyl-L-methionine:glycine N-methyltransferase; S-Adenosyl-L-methionine:glycine methyltransferase; S-adenosyl-L-methionine:glycine N-methyltransferase; S-adenosyl-L-methionine:glycine methyltransferase; glycine N-methyltransferase; glycine methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00033[0] <=> (1) cpd00019[0] + (1) cpd00183[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00033;cpd00067;cpd00183',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Glycine[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) Sarcosine[0]',\n",
       "  'deltag': -10.46,\n",
       "  'deltagerr': 4.93,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.156', '2.1.1.162', '2.1.1.20'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00033[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd00183[0]',\n",
       "  'id': 'rxn00270',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:glycine N-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: Betaine-Biosynthesis (Betaine Biosynthesis); P541-PWY (glycine betaine biosynthesis IV (from glycine)); PWY-6004 (glycine betaine biosynthesis V (from glycine)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00033:0:0:\"Glycine\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd00183:0:0:\"Sarcosine\"'},\n",
       " {'abbreviation': 'R00368',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00368',\n",
       "   'MetaCyc: STROMBINE-DEHYDROGENASE-RXN',\n",
       "   'Name: N-(carboxymethyl)-D-alanine: NAD+ oxidoreductase; N-(carboxymethyl)-D-alanine:NAD+ oxidoreductase (glycine-forming); strombine dehydrogenase; strombine[N-(carboxymethyl)-D-alanine]dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd02373[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00033[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00020;cpd00033;cpd00067;cpd02373',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) N-(Carboxymethyl)-D-alanine[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (1) Glycine[0] + (1) H+[0]',\n",
       "  'deltag': 8.39,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd02373[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00033[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00271',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-(carboxymethyl)-D-alanine:NAD+ oxidoreductase (glycine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7351 (pyruvate fermentation to opines); Pyruvate-Degradation (Fermentation of Pyruvate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd02373:0:0:\"N-(Carboxymethyl)-D-alanine\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00033:0:0:\"Glycine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00369',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ALANINE--GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'BiGG: AGT; AGTi; YFL030W',\n",
       "   'BrachyCyc: ALANINE--GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00369',\n",
       "   'MetaCyc: ALANINE--GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: Alanine--glyoxylate aminotransferase; Alanine--glyoxylate aminotransferase 1; L-Alanine:glyoxylate aminotransferase; L-alanine-glycine transaminase; L-alanine:glyoxylate aminotransferase; alanine-glyoxylate aminotransferase; alanine-glyoxylate transaminase; alanine-glyoxylic aminotransferase'],\n",
       "  'code': '(1) cpd00035[0] + (1) cpd00040[0] <=> (1) cpd00020[0] + (1) cpd00033[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00033;cpd00035;cpd00040',\n",
       "  'definition': '(1) L-Alanine[0] + (1) Glyoxalate[0] <=> (1) Pyruvate[0] + (1) Glycine[0]',\n",
       "  'deltag': -1.96,\n",
       "  'deltagerr': 0.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.44'],\n",
       "  'equation': '(1) cpd00035[0] + (1) cpd00040[0] <=> (1) cpd00020[0] + (1) cpd00033[0]',\n",
       "  'id': 'rxn00272',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27694;rxn27695;rxn32557',\n",
       "  'name': 'L-Alanine:glyoxylate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); GLYCINE-SYN (Glycine Biosynthesis); GLYSYN-ALA-PWY (glycine biosynthesis III); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00035:0:0:\"L-Alanine\";-1:cpd00040:0:0:\"Glyoxalate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00033:0:0:\"Glycine\"'},\n",
       " {'abbreviation': 'rbs_30',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['iAG612: rbs_30',\n",
       "   'Name: Acetyl-CoA:glycine C-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00033[0] <=> (1) cpd00010[0] + (1) cpd02053[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00033;cpd02053',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Glycine[0] <=> (1) CoA[0] + (1) 2-Amino-acetoacetate[0]',\n",
       "  'deltag': 8.44,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.29'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00033[0] <=> (1) cpd00010[0] + (1) cpd02053[0]',\n",
       "  'id': 'rxn00273',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:glycine C-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00033:0:0:\"Glycine\";1:cpd00010:0:0:\"CoA\";1:cpd02053:0:0:\"2-Amino-acetoacetate\"'},\n",
       " {'abbreviation': 'R00371',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GLYAT; GLYATRi; GLYATi',\n",
       "   'ChlamyCyc: AKBLIG-RXN',\n",
       "   'EcoCyc: AKBLIG-RXN',\n",
       "   'KEGG: R00371',\n",
       "   'MetaCyc: AKBLIG-RXN',\n",
       "   'Name: 2-amino-3-ketobutyrate CoA ligase; 2-amino-3-ketobutyrate coenzyme A ligase; 2-amino-3-ketobutyrate-CoA ligase; AKB ligase; Acetyl-CoA:glycine C-acetyltransferase; KBL; acetyl-CoA:glycine C-acetyltransferase; aminoacetone synthase; aminoacetone synthetase; glycine C-acetyltransferase; glycine C-acetyltransferase (irreversible); glycine acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00033[0] <=> (1) cpd00010[0] + (1) cpd02211[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00033;cpd02211',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Glycine[0] <=> (1) CoA[0] + (1) L-2-Amino-acetoacetate[0]',\n",
       "  'deltag': 8.44,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.29'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00033[0] <=> (1) cpd00010[0] + (1) cpd02211[0]',\n",
       "  'id': 'rxn00274',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35211',\n",
       "  'name': 'Acetyl-CoA:glycine C-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); THREOCAT-PWY (superpathway of L-threonine metabolism); THREONINE-DEG (L-threonine Degradation); THREONINE-DEG2-PWY (L-threonine degradation II)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00033:0:0:\"Glycine\";1:cpd00010:0:0:\"CoA\";1:cpd02211:0:0:\"L-2-Amino-acetoacetate\"'},\n",
       " {'abbreviation': 'R00372',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCINE-AMINOTRANSFERASE-RXN',\n",
       "   'BrachyCyc: GLYCINE-AMINOTRANSFERASE-RXN',\n",
       "   'CornCyc: GLYCINE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00372',\n",
       "   'MetaCyc: GLYCINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: Glycine aminotransferase; Glycine:2-oxoglutarate aminotransferase; L-glutamate:glyoxylate aminotransferase; glutamic-glyoxylic transaminase; glycine aminotransferase; glycine transaminase; glycine:2-oxoglutarate aminotransferase; glyoxylate-glutamate aminotransferase; glyoxylate-glutamic transaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00033[0] <=> (1) cpd00023[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00033;cpd00040',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) Glycine[0] <=> (1) L-Glutamate[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 2.03,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.4', '2.6.1.44'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00033[0] <=> (1) cpd00023[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00275',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27909;rxn32307;rxn35212',\n",
       "  'name': 'Glycine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-181 (photorespiration); Photosynthesis ()',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00033:0:0:\"Glycine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00373',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00373',\n",
       "   'MetaCyc: 2.6.1.35-RXN',\n",
       "   'Name: Glycine:oxaloacetate aminotransferase; glycine-oxalacetate aminotransferase; glycine-oxaloacetate transaminase; glycine:oxaloacetate aminotransferase'],\n",
       "  'code': '(1) cpd00032[0] + (1) cpd00033[0] <=> (1) cpd00040[0] + (1) cpd00041[0]',\n",
       "  'compound_ids': 'cpd00032;cpd00033;cpd00040;cpd00041',\n",
       "  'definition': '(1) Oxaloacetate[0] + (1) Glycine[0] <=> (1) Glyoxalate[0] + (1) L-Aspartate[0]',\n",
       "  'deltag': 1.34,\n",
       "  'deltagerr': 0.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.35'],\n",
       "  'equation': '(1) cpd00032[0] + (1) cpd00033[0] <=> (1) cpd00040[0] + (1) cpd00041[0]',\n",
       "  'id': 'rxn00276',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glycine:oxaloacetate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00032:0:0:\"Oxaloacetate\";-1:cpd00033:0:0:\"Glycine\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00041:0:0:\"L-Aspartate\"'},\n",
       " {'abbreviation': 'rxn00277',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00033[0] <=> (1) cpd00150[0]',\n",
       "  'compound_ids': 'cpd00033;cpd00150',\n",
       "  'definition': '(1) Glycine[0] <=> (1) HCN[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00033[0] <=> (1) cpd00150[0]',\n",
       "  'id': 'rxn00277',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00277',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:C:-1/H:-4/O:-2',\n",
       "  'stoichiometry': '-1:cpd00033:0:0:\"Glycine\";1:cpd00150:0:0:\"HCN\"'},\n",
       " {'abbreviation': 'R00396',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ALAD_L',\n",
       "   'KEGG: R00396',\n",
       "   'MetaCyc: ALANINE-DEHYDROGENASE-RXN',\n",
       "   'iAG612: rbs_364',\n",
       "   'iIT341: ALAD_L',\n",
       "   'Name: AlaDH; L-Alanine:NAD+ oxidoreductase (deaminating); L-alanine dehydrogenase; L-alanine:NAD+ oxidoreductase (deaminating); L-alaninedehydrogenase; NAD-dependent alanine dehydrogenase; NAD-linked alanine dehydrogenase; NADH-dependent alanine dehydrogenase; alanine dehydrogenase; alanine oxidoreductase; alpha-alanine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00035[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00020;cpd00035;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Alanine[0] <=> (1) NADH[0] + (1) NH3[0] + (1) Pyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 8.78,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00035[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00278',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30302',\n",
       "  'name': 'L-Alanine:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALANINE-DEG (L-alanine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PWY1-2 (L-alanine degradation IV); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00430 (Taurine and hypotaurine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00035:0:0:\"L-Alanine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00397',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00397',\n",
       "   'MetaCyc: ASPARTATE-4-DECARBOXYLASE-RXN',\n",
       "   'Name: L-Aspartate 4-carboxy-lyase; L-aspartate 4-carboxy-lyase; L-aspartate 4-carboxy-lyase (L-alanine-forming); L-aspartate beta-decarboxylase; L-cysteine sulfinate acid desulfinase; aminomalonic decarboxylase; aspartate 4-decarboxylase; aspartate beta-decarboxylase; aspartate omega-decarboxylase; aspartic beta-decarboxylase; aspartic omega-decarboxylase; cysteine sulfinic desulfinase; desulfinase'],\n",
       "  'code': '(1) cpd00041[0] <=> (1) cpd00011[0] + (1) cpd00035[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00035;cpd00041;cpd00067',\n",
       "  'definition': '(1) L-Aspartate[0] + (1) H+[0] => (1) CO2[0] + (1) L-Alanine[0]',\n",
       "  'deltag': -3.34,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.12'],\n",
       "  'equation': '(1) cpd00041[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00035[0]',\n",
       "  'id': 'rxn00279',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-aspartate 4-carboxy-lyase (L-alanine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00035:0:0:\"L-Alanine\"'},\n",
       " {'abbreviation': 'R00398',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00398',\n",
       "   'MetaCyc: ALANOPINE-DEHYDROGENASE-RXN',\n",
       "   \"Name: 2,2'-iminodipropanoate:NAD+ oxidoreductase (L-alanine-forming); ADH; ALPDH; alanopine dehydrogenase; alanopine: NAD oxidoreductase; alanopine:NAD oxidoreductase; alanopine[meso-N-(1-carboxyethyl)-alanine]dehydrogenase; meso-N-(1-carboxyethyl)-alanine:NAD+ oxidoreductase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd02051[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00035[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00020;cpd00035;cpd00067;cpd02051',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Alanopine[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (1) L-Alanine[0] + (1) H+[0]',\n",
       "  'deltag': 8.18,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.17'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd02051[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00035[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00280',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"2,2'-iminodipropanoate:NAD+ oxidoreductase (L-alanine-forming)\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7351 (pyruvate fermentation to opines); Pyruvate-Degradation (Fermentation of Pyruvate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd02051:0:0:\"Alanopine\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00035:0:0:\"L-Alanine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'rxn00281',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00035[0] <=> (1) cpd00010[0] + (1) cpd02121[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00035;cpd02121',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Alanine[0] <=> (1) CoA[0] + (1) 2-Amino-4-oxopentanoate[0]',\n",
       "  'deltag': 7.76,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.-'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00035[0] <=> (1) cpd00010[0] + (1) cpd02121[0]',\n",
       "  'id': 'rxn00281',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00281',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00035:0:0:\"L-Alanine\";1:cpd00010:0:0:\"CoA\";1:cpd02121:0:0:\"2-Amino-4-oxopentanoate\"'},\n",
       " {'abbreviation': 'R00400',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00400', 'Name: L-alanine:oxaloacetate aminotransferase'],\n",
       "  'code': '(1) cpd00032[0] + (1) cpd00035[0] <=> (1) cpd00020[0] + (1) cpd00041[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00032;cpd00035;cpd00041',\n",
       "  'definition': '(1) Oxaloacetate[0] + (1) L-Alanine[0] <=> (1) Pyruvate[0] + (1) L-Aspartate[0]',\n",
       "  'deltag': -0.63,\n",
       "  'deltagerr': 0.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.12'],\n",
       "  'equation': '(1) cpd00032[0] + (1) cpd00035[0] <=> (1) cpd00020[0] + (1) cpd00041[0]',\n",
       "  'id': 'rxn00282',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-alanine:oxaloacetate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00032:0:0:\"Oxaloacetate\";-1:cpd00035:0:0:\"L-Alanine\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00041:0:0:\"L-Aspartate\"'},\n",
       " {'abbreviation': 'R00401',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ALAR; ALARi',\n",
       "   'ChlamyCyc: ALARACECAT-RXN',\n",
       "   'EcoCyc: ALARACECAT-RXN',\n",
       "   'KEGG: R00401',\n",
       "   'MetaCyc: ALARACECAT-RXN',\n",
       "   'Name: L-Alanine racemase; L-alanine racemase; alanine racemase; alanine racemase (irreversible); alanineracemase'],\n",
       "  'code': '(1) cpd00035[0] <=> (1) cpd00117[0]',\n",
       "  'compound_ids': 'cpd00035;cpd00117',\n",
       "  'definition': '(1) L-Alanine[0] <=> (1) D-Alanine[0]',\n",
       "  'deltag': -0.01,\n",
       "  'deltagerr': 0.25,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.1'],\n",
       "  'equation': '(1) cpd00035[0] <=> (1) cpd00117[0]',\n",
       "  'id': 'rxn00283',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19085;rxn38030',\n",
       "  'name': 'alanine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALADEG-PWY (L-alanine degradation I); ALANINE-DEG (L-alanine Degradation); ALANINE-SYN (L-alanine Biosynthesis); ALANINE-VALINESYN-PWY (L-alanine biosynthesis I); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY0-1061 (superpathway of L-alanine biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00473 (D-Alanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00035:0:0:\"L-Alanine\";1:cpd00117:0:0:\"D-Alanine\"'},\n",
       " {'abbreviation': 'R00402',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: FUMARATE-REDUCTASE-NADH-RXN',\n",
       "   'JM_Creinhardtii: JM_Cre_150; JM_Cre_151',\n",
       "   'KEGG: R00402',\n",
       "   'MetaCyc: FUMARATE-REDUCTASE-NADH-RXN',\n",
       "   'Name: Fumarate reductase (NADH); NADH-dependent fumarate reductase; NADH-fumarate reductase; fumarate reductase (NADH); fumarate reductase (NADH2); succinate:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00036[0] <=> (1) cpd00004[0] + (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00036;cpd00067;cpd00106',\n",
       "  'definition': '(1) NAD[0] + (1) Succinate[0] <= (1) NADH[0] + (1) H+[0] + (1) Fumarate[0]',\n",
       "  'deltag': 14.88,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.3.1.6'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00036[0] <= (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00106[0]',\n",
       "  'id': 'rxn00284',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27866',\n",
       "  'name': 'succinate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00036:0:0:\"Succinate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R00405',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUCCCOASYN-RXN',\n",
       "   'BiGG: LSC2; SUCOAS; SUCOAS1m; SUCOASm',\n",
       "   'BrachyCyc: SUCCCOASYN-RXN',\n",
       "   'KEGG: R00405',\n",
       "   'MetaCyc: SUCCCOASYN-RXN',\n",
       "   'Name: A-SCS; A-STK (adenin nucleotide-linked succinate thiokinase); STK; Succinate thiokinase; Succinate--CoA ligase (ADP-forming); Succinate:CoA ligase (ADP-forming); Succinyl-CoA ligase [ADP-forming] subunit beta; Succinyl-CoA synthetase (ADP-forming); succinate thiokinase; succinate--CoA ligase (ADP-forming); succinate-CoA ligase (ADP-forming); succinate:CoA ligase (ADP-forming); succinic thiokinase; succinyl CoA synthesis; succinyl coenzyme A synthetase; succinyl coenzyme A synthetase (adenosine diphosphate-forming); succinyl-CoA synthetase; succinyl-CoA synthetase (ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00036[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00078[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00010;cpd00036;cpd00078',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Succinate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) Succinyl-CoA[0]',\n",
       "  'deltag': -0.49,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.5'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00036[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00078[0]',\n",
       "  'id': 'rxn00285',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28991',\n",
       "  'name': 'Succinate:CoA ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); P23-PWY (reductive TCA cycle I); P42-PWY (incomplete reductive TCA cycle); PWY-5392 (reductive TCA cycle II); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-5537 (pyruvate fermentation to acetate V); PWY-5538 (pyruvate fermentation to acetate VI); PWY-5690 (TCA cycle II (plants and fungi)); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6728 (methylaspartate cycle); PWY-6969 (TCA cycle V (2-oxoglutarate:ferredoxin oxidoreductase)); PWY-7384 (anaerobic energy metabolism (invertebrates, mitochondrial)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY66-398 (TCA cycle III (animals)); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Reductive-TCA-Cycles (Reductive TCA Cycles); TCA (TCA cycle I (prokaryotic)); TCA-GLYOX-BYPASS (superpathway of glyoxylate bypass and TCA); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00640 (Propanoate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00036:0:0:\"Succinate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00078:0:0:\"Succinyl-CoA\"'},\n",
       " {'abbreviation': 'R00406',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00406',\n",
       "   'MetaCyc: RXN-8964',\n",
       "   'Name: succinyl-CoA:(S)-malate CoA-transferase; succinyl-CoA:malate CoA-transferase'],\n",
       "  'code': '(1) cpd00078[0] + (1) cpd00130[0] <=> (1) cpd00036[0] + (1) cpd02662[0]',\n",
       "  'compound_ids': 'cpd00036;cpd00078;cpd00130;cpd02662',\n",
       "  'definition': '(1) Succinyl-CoA[0] + (1) L-Malate[0] <=> (1) Succinate[0] + (1) Malyl-CoA[0]',\n",
       "  'deltag': -1.04,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.-', '2.8.3.22'],\n",
       "  'equation': '(1) cpd00078[0] + (1) cpd00130[0] <=> (1) cpd00036[0] + (1) cpd02662[0]',\n",
       "  'id': 'rxn00286',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'succinyl-CoA:malate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); PWY-5743 (3-hydroxypropanoate cycle); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle)',\n",
       "   'KEGG: rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00078:0:0:\"Succinyl-CoA\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00036:0:0:\"Succinate\";1:cpd02662:0:0:\"Malyl-CoA\"'},\n",
       " {'abbreviation': 'R00407',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUCCINYL-COA-HYDROLASE-RXN',\n",
       "   'BrachyCyc: SUCCINYL-COA-HYDROLASE-RXN',\n",
       "   'KEGG: R00407',\n",
       "   'MetaCyc: SUCCINYL-COA-HYDROLASE-RXN',\n",
       "   'TS_Athaliana: rxn1549_c; rxn1549_g; rxn1549_v; rxn1549_x',\n",
       "   'Name: Succinyl-CoA hydrolase; succinyl coenzyme A deacylase; succinyl coenzyme A hydrolase; succinyl-CoA acylase; succinyl-CoA hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00078[0] <=> (1) cpd00010[0] + (1) cpd00036[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00036;cpd00067;cpd00078',\n",
       "  'definition': '(1) H2O[0] + (1) Succinyl-CoA[0] => (1) CoA[0] + (1) Succinate[0] + (1) H+[0]',\n",
       "  'deltag': -5.67,\n",
       "  'deltagerr': 0.33,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00078[0] => (1) cpd00010[0] + (1) cpd00036[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00287',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27252;rxn32107;rxn32154',\n",
       "  'name': 'Succinyl-CoA hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00078:0:0:\"Succinyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00036:0:0:\"Succinate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00408',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00408_m',\n",
       "   'AraGEM: R_R00408_m',\n",
       "   'BiGG: FRD; FRDSUCD1i; FRDm; OSM1; SDH3_1; SUCD1; SUCD1i; SUCD1m',\n",
       "   'JP_Creinhardtii_MSB: R_SUCDHh; R_SUCDHm',\n",
       "   'KEGG: R00408',\n",
       "   'Name: Fumarate reductase; Succinate:(acceptor) oxidoreductase; fumarate reductase flavoprotein subunit; succinate dehydrogenase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00036[0] <=> (1) cpd00106[0] + (1) cpd00982[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00036;cpd00106;cpd00982',\n",
       "  'definition': '(1) FAD[0] + (1) Succinate[0] <=> (1) Fumarate[0] + (1) FADH2[0]',\n",
       "  'deltag': 11.27,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.99.1'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00036[0] <=> (1) cpd00106[0] + (1) cpd00982[0]',\n",
       "  'id': 'rxn00288',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30176;rxn34271',\n",
       "  'name': 'Succinate:(acceptor) oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00036:0:0:\"Succinate\";1:cpd00106:0:0:\"Fumarate\";1:cpd00982:0:0:\"FADH2\"'},\n",
       " {'abbreviation': 'R00409',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MCITL2; MCITL2m; MCL; MICITL',\n",
       "   'EcoCyc: METHYLISOCITRATE-LYASE-RXN',\n",
       "   'KEGG: R00409',\n",
       "   'MetaCyc: METHYLISOCITRATE-LYASE-RXN',\n",
       "   'TS_Athaliana: rxn570_m; rxn570_p; rxn570_v',\n",
       "   'Name: (2S,3R)-3-Hydroxybutane-1,2,3-tricarboxylate pyruvate-lyase; (2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate pyruvate-lyase; (2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate pyruvate-lyase (succinate-forming); 2-methylisocitrate lyase; MICL; methylisocitrate lyase'],\n",
       "  'code': '(1) cpd02799[0] <=> (1) cpd00020[0] + (1) cpd00036[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00036;cpd02799',\n",
       "  'definition': '(1) Methylisocitrate[0] <=> (1) Pyruvate[0] + (1) Succinate[0]',\n",
       "  'deltag': 2.27,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.30'],\n",
       "  'equation': '(1) cpd02799[0] <=> (1) cpd00020[0] + (1) cpd00036[0]',\n",
       "  'id': 'rxn00289',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30406;rxn31857;rxn31866',\n",
       "  'name': '(2S,3R)-3-hydroxybutane-1,2,3-tricarboxylate pyruvate-lyase (succinate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Methyl-Citrate-Cycle (2-Methylcitrate Cycle); PWY-5747 (2-methylcitrate cycle II); PWY0-42 (2-methylcitrate cycle I); Propionate-Degradation (Propanoate Degradation)',\n",
       "   'KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02799:0:0:\"Methylisocitrate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00036:0:0:\"Succinate\"'},\n",
       " {'abbreviation': 'R00410',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: OCOAT1; OCOAT1r',\n",
       "   'KEGG: R00410',\n",
       "   'MetaCyc: RXNI-2',\n",
       "   'iIT341: OCOAT1',\n",
       "   'iJN746: OCOAT1',\n",
       "   'Name: 3-oxoacid CoA-transferase (Succinyl-CoA: acetoacetate); 3-oxoacidCoA-transferase-Succinyl-CoA-acetoacetate; Succinyl-CoA:3-oxo-acid CoA-transferase; acetoacetate CoA-transferase; succinyl-CoA:acetoacetate CoA-transferase'],\n",
       "  'code': '(1) cpd00078[0] + (1) cpd00142[0] <=> (1) cpd00036[0] + (1) cpd00279[0]',\n",
       "  'compound_ids': 'cpd00036;cpd00078;cpd00142;cpd00279',\n",
       "  'definition': '(1) Succinyl-CoA[0] + (1) Acetoacetate[0] <=> (1) Succinate[0] + (1) Acetoacetyl-CoA[0]',\n",
       "  'deltag': 2.95,\n",
       "  'deltagerr': 0.16,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.5'],\n",
       "  'equation': '(1) cpd00078[0] + (1) cpd00142[0] <=> (1) cpd00036[0] + (1) cpd00279[0]',\n",
       "  'id': 'rxn00290',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'succinyl-CoA:acetoacetate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); OTHER-ENERGY (Other); PWY66-368 (ketolysis); REDCITCYC (TCA cycle VIII (helicobacter)); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00072 (Synthesis and degradation of ketone bodies); rn00280 (Valine, leucine and isoleucine degradation); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00078:0:0:\"Succinyl-CoA\";-1:cpd00142:0:0:\"Acetoacetate\";1:cpd00036:0:0:\"Succinate\";1:cpd00279:0:0:\"Acetoacetyl-CoA\"'},\n",
       " {'abbreviation': 'R00411',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: SGDS',\n",
       "   'EcoCyc: SUCCGLUDESUCC-RXN',\n",
       "   'KEGG: R00411',\n",
       "   'MetaCyc: SUCCGLUDESUCC-RXN',\n",
       "   'RiceCyc: SUCCGLUDESUCC-RXN',\n",
       "   'Name: AstE; N-succinyl-L-glutamate amidohydrolase; N2-succinylglutamate desuccinylase; Succinylglutamate desuccinylase; succinylglutamate desuccinylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03525[0] <=> (1) cpd00023[0] + (1) cpd00036[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00023;cpd00036;cpd03525',\n",
       "  'definition': '(1) H2O[0] + (1) N-Succinyl-L-glutamate[0] <=> (1) L-Glutamate[0] + (1) Succinate[0]',\n",
       "  'deltag': -0.67,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.96'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03525[0] <=> (1) cpd00023[0] + (1) cpd00036[0]',\n",
       "  'id': 'rxn00291',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32708',\n",
       "  'name': 'N-succinyl-L-glutamate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGININE-DEG (L-arginine Degradation); AST-PWY (L-arginine degradation II (AST pathway)); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03525:0:0:\"N-Succinyl-L-glutamate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00036:0:0:\"Succinate\"'},\n",
       " {'abbreviation': 'R00414',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: UAG2EMA',\n",
       "   'KEGG: R00414',\n",
       "   'iAF692: UAG2EMA',\n",
       "   'iSB619: UAG2EMA',\n",
       "   'iYO844: UAG2EMA',\n",
       "   'Name: UDP-N-acetyl-D-glucosamine 2-epimerase; UDP-N-acetyl-D-glucosamine 2-epimerase (Hydrolysis)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00037[0] <=> (1) cpd00014[0] + (1) cpd00492[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00014;cpd00037;cpd00492',\n",
       "  'definition': '(1) H2O[0] + (1) UDP-N-acetylglucosamine[0] => (1) UDP[0] + (1) N-Acetyl-D-mannosamine[0]',\n",
       "  'deltag': -7.01,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.183', '5.1.3.14'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00037[0] => (1) cpd00014[0] + (1) cpd00492[0]',\n",
       "  'id': 'rxn00292',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-N-acetyl-D-glucosamine 2-epimerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";1:cpd00014:0:0:\"UDP\";1:cpd00492:0:0:\"N-Acetyl-D-mannosamine\"'},\n",
       " {'abbreviation': 'R00416',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.7.64-RXN; NAG1P-URIDYLTRANS-RXN',\n",
       "   'BiGG: UAGDP; UAGDPr',\n",
       "   'BrachyCyc: NAG1P-URIDYLTRANS-RXN',\n",
       "   'KEGG: R00416',\n",
       "   'MetaCyc: 2.7.7.64-RXN; NAG1P-URIDYLTRANS-RXN',\n",
       "   'Name: Acetylglucosamine 1-phosphate uridylyltransferase; GlmU uridylyltransferase; UDP-GlcNAc pyrophosphorylase; UDP-N-acetylglucosamine diphosphorylase; UDP-N-acetylglucosamine pyrophosphorylase; UDP-N-acetylglucosaminediphosphorylase; UDP-acetylglucosamine pyrophosphorylase; UTP:2-acetamido-2-deoxy-alpha-D-glucose-1-phosphate uridylyltransferase; UTP:N-acetyl-alpha-D-glucosamine-1-phosphate uridylyltransferase; acetylglucosamine 1-phosphate uridylyltransferase; uridine diphosphate-N-acetylglucosamine pyrophosphorylase; uridine diphosphoacetylglucosamine phosphorylase; uridine diphosphoacetylglucosamine pyrophosphorylase'],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd02611[0] <=> (1) cpd00012[0] + (1) cpd00037[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00037;cpd00062;cpd02611',\n",
       "  'definition': '(1) UTP[0] + (1) N-Acetyl-D-glucosamine1-phosphate[0] <=> (1) PPi[0] + (1) UDP-N-acetylglucosamine[0]',\n",
       "  'deltag': -0.81,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.23'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd02611[0] <=> (1) cpd00012[0] + (1) cpd00037[0]',\n",
       "  'id': 'rxn00293',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30068',\n",
       "  'name': 'UTP:N-acetyl-alpha-D-glucosamine-1-phosphate uridylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Anhydromuropeptides-Recycling (Anhydromuropeptide Recycling); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Glycan-Biosynthesis (Glycan Biosynthesis); Glycan-Pathways (Glycan Pathways); Metabolic-Clusters (Metabolic Clusters); OANTIGEN-PWY (<i>O</i>-antigen building blocks biosynthesis (<i>E. coli</i>)); PWY-5514 (UDP-<i>N</i>-acetyl-D-galactosamine biosynthesis II); PWY-6404 (superpathway of mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY-6981 (chitin biosynthesis); PWY-7332 (superpathway of UDP-<I>N</i>-acetylglucosamine-derived O-antigen building blocks biosynthesis); PWY0-1261 (anhydromuropeptides recycling I); Polysaccharides-Biosynthesis (Polysaccharide Biosynthesis); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); UDP-NAc-Glucosamine-Biosynthesis (UDP-<i>N</i>-acetyl-D-glucosamine Biosynthesis); UDP-Nac-Galactosamine-Biosynthesis (UDP-<i>N</i>-acetyl-D-galactosamine Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis); UDPNACETYLGALSYN-PWY (UDP-<i>N</i>-acetyl-D-glucosamine biosynthesis II); UDPNAGSYN-PWY (UDP-<i>N</i>-acetyl-D-glucosamine biosynthesis I)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd02611:0:0:\"N-Acetyl-D-glucosamine1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00037:0:0:\"UDP-N-acetylglucosamine\"'},\n",
       " {'abbreviation': 'R00418',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UDP-N-ACETYLGLUCOSAMINE-4-EPIMERASE-RXN',\n",
       "   'BiGG: UAG4E',\n",
       "   'BrachyCyc: UDP-N-ACETYLGLUCOSAMINE-4-EPIMERASE-RXN',\n",
       "   'KEGG: R00418',\n",
       "   'MetaCyc: UDP-N-ACETYLGLUCOSAMINE-4-EPIMERASE-RXN',\n",
       "   \"Name: UDP acetylglucosamine epimerase; UDP-N-acetyl-D-glucosamine 4-epimerase; UDP-N-acetylglucosamine 4-epimerase; uridine 5'-diphospho-N-acetylglucosamine-4-epimerase; uridine diphosphate N-acetylglucosamine-4-epimerase; uridine diphosphoacetylglucosamine epimerase\"],\n",
       "  'code': '(1) cpd00037[0] <=> (1) cpd00175[0]',\n",
       "  'compound_ids': 'cpd00037;cpd00175',\n",
       "  'definition': '(1) UDP-N-acetylglucosamine[0] <=> (1) UDP-N-acetyl-D-galactosamine[0]',\n",
       "  'deltag': -0.24,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.3.2', '5.1.3.7'],\n",
       "  'equation': '(1) cpd00037[0] <=> (1) cpd00175[0]',\n",
       "  'id': 'rxn00295',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-N-acetyl-D-glucosamine 4-epimerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); PWY-5512 (UDP-<i>N</i>-acetyl-D-galactosamine biosynthesis I); PWY-5514 (UDP-<i>N</i>-acetyl-D-galactosamine biosynthesis II); PWY-7332 (superpathway of UDP-<I>N</i>-acetylglucosamine-derived O-antigen building blocks biosynthesis); PWY-7820 (teichuronic acid biosynthesis (<i>B. subtilis</i> 168)); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Nac-Galactosamine-Biosynthesis (UDP-<i>N</i>-acetyl-D-galactosamine Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";1:cpd00175:0:0:\"UDP-N-acetyl-D-galactosamine\"'},\n",
       " {'abbreviation': 'rxn00296',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00419'],\n",
       "  'code': '(1) cpd00037[0] <=> (1) cpd00273[0]',\n",
       "  'compound_ids': 'cpd00037;cpd00273',\n",
       "  'definition': '(1) UDP-N-acetylglucosamine[0] <=> (1) Glycoprotein[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00037[0] <=> (1) cpd00273[0]',\n",
       "  'id': 'rxn00296',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00296',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:C:11/H:22/N:2/O:1/P:-2|CI:2',\n",
       "  'stoichiometry': '-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";1:cpd00273:0:0:\"Glycoprotein\"'},\n",
       " {'abbreviation': 'R00420',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: UAG2E; UAG2Ei',\n",
       "   'EcoCyc: UDPGLCNACEPIM-RXN',\n",
       "   'KEGG: R00420',\n",
       "   'MetaCyc: UDPGLCNACEPIM-RXN',\n",
       "   'iAF1260: UAG2E',\n",
       "   \"Name: UDP-N-acetyl-D-glucosamine 2-epimerase; UDP-N-acetylglucosamine 2'-epimerase; UDP-N-acetylglucosamine 2-epimerase; uridine diphosphate-N-acetylglucosamine-2'-epimerase; uridine diphospho-N-acetylglucosamine 2'-epimerase; uridine diphosphoacetylglucosamine 2'-epimerase\"],\n",
       "  'code': '(1) cpd00037[0] <=> (1) cpd00861[0]',\n",
       "  'compound_ids': 'cpd00037;cpd00861',\n",
       "  'definition': '(1) UDP-N-acetylglucosamine[0] <=> (1) UDP-N-acetyl-D-mannosamine[0]',\n",
       "  'deltag': -0.24,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.3.14'],\n",
       "  'equation': '(1) cpd00037[0] <=> (1) cpd00861[0]',\n",
       "  'id': 'rxn00297',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-N-acetyl-D-glucosamine 2-epimerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); ECASYN-PWY (enterobacterial common antigen biosynthesis); Lipopolysaccharide-Biosynthesis (Lipopolysaccharide Biosynthesis); PWY-7332 (superpathway of UDP-<I>N</i>-acetylglucosamine-derived O-antigen building blocks biosynthesis); PWY-7335 (UDP-<i>N</i>-acetyl-&alpha;-D-mannosaminouronate biosynthesis); PWY-7815 (poly(ribitol phosphate) wall teichoic acid biosynthesis I (<i>B. subtilis</i>)); PWY-7816 (poly(ribitol phosphate) wall teichoic acid biosynthesis II (<i>S. aureus</i>)); PWY-7819 (poly(3-O-&beta;-D-glucopyranosyl-<i>N</i>-acetylgalactosamine 1-phosphate) wall teichoic acid biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); TEICHOICACID-PWY (poly(glycerol phosphate) wall teichoic acid biosynthesis); Teichoic-Acids-Biosynthesis (Teichoic Acid Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";1:cpd00861:0:0:\"UDP-N-acetyl-D-mannosamine\"'},\n",
       " {'abbreviation': 'R00421',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00421',\n",
       "   'MetaCyc: 1.1.1.136-RXN',\n",
       "   'TS_Athaliana: rxn573_c',\n",
       "   'Name: UDP-2-acetamido-2-deoxy-D-glucose:NAD oxidoreductase; UDP-GlcNAc dehydrogenase; UDP-N-acetyl-D-glucosamine:NAD+ 6-oxidoreductase; UDP-N-acetylglucosamine 6-dehydrogenase; UDP-acetylglucosamine dehydrogenase; uridine diphosphoacetylglucosamine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00037[0] <=> (2) cpd00004[0] + (1) cpd02782[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00037;cpd00067;cpd02782',\n",
       "  'definition': '(1) H2O[0] + (2) NAD[0] + (1) UDP-N-acetylglucosamine[0] <=> (2) NADH[0] + (3) H+[0] + (1) UDP-N-acetyl-D-glucosaminouronate[0]',\n",
       "  'deltag': -5.07,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.136'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00037[0] <=> (2) cpd00004[0] + (3) cpd00067[0] + (1) cpd02782[0]',\n",
       "  'id': 'rxn00298',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-N-acetyl-D-glucosamine:NAD+ 6-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); PWY-7090 (UDP-2,3-diacetamido-2,3-dideoxy-&alpha;-D-mannuronate biosynthesis); PWY-7332 (superpathway of UDP-<I>N</i>-acetylglucosamine-derived O-antigen building blocks biosynthesis); PWY-7336 (UDP-<i>N</i>-acetyl-&alpha;-D-galactosaminuronate biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00003:0:0:\"NAD\";-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";2:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";1:cpd02782:0:0:\"UDP-N-acetyl-D-glucosaminouronate\"'},\n",
       " {'abbreviation': 'R00424',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GTP-CYCLOHYDRO-I-RXN',\n",
       "   'BiGG: FOL2; GTPCI',\n",
       "   'BrachyCyc: GTP-CYCLOHYDRO-I-RXN',\n",
       "   'KEGG: R00424',\n",
       "   'MetaCyc: GTP-CYCLOHYDRO-I-RXN',\n",
       "   'Name: GTP 7,8-8,9-dihydrolase; GTP 8-formylhydrolase; GTP cyclohydrolase; GTP cyclohydrolase I; GTPcyclohydrolaseI; dihydroneopterin triphosphate synthase; guanosine triphosphate 8-deformylase; guanosine triphosphate cyclohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00038[0] <=> (1) cpd00047[0] + (1) cpd02978[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00038;cpd00047;cpd00067;cpd02978',\n",
       "  'definition': \"(1) H2O[0] + (1) GTP[0] => (1) Formate[0] + (2) H+[0] + (1) 7,8-Dihydroneopterin 3'-triphosphate[0]\",\n",
       "  'deltag': -18.54,\n",
       "  'deltagerr': 1.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.16'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00038[0] => (1) cpd00047[0] + (2) cpd00067[0] + (1) cpd02978[0]',\n",
       "  'id': 'rxn00299',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GTP 7,8-8,9-dihydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 6-HM-Dihydropterin-PP-Biosynthesis (6-Hydroxymethyl-Dihydropterin Diphosphate Biosynthesis); ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); FOLSYN-PWY (superpathway of tetrahydrofolate biosynthesis and salvage); Folate-Biosynthesis (Folate Biosynthesis); Other-biosynthesis (Other Biosynthesis); PWY-5663 (erythro-tetrahydrobiopterin biosynthesis I); PWY-5664 (<i>erythro</i>-tetrahydrobiopterin biosynthesis II); PWY-6147 (6-hydroxymethyl-dihydropterin diphosphate biosynthesis I); PWY-6612 (superpathway of tetrahydrofolate biosynthesis); PWY-6703 (preQ<sub>0</sub> biosynthesis); PWY-6983 (<i>threo</i>-tetrahydrobiopterin biosynthesis); PWY-7442 (drosopterin and aurodrosopterin biosynthesis); PWY-7852 (6-hydroxymethyl-dihydropterin diphosphate biosynthesis IV (Plasmodium)); PWY0-1433 (tetrahydromonapterin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Tetrahydrobiopterin-Biosynthesis (Tetrahydrobiopterin Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00038:0:0:\"GTP\";1:cpd00047:0:0:\"Formate\";2:cpd00067:0:0:\"H+\";1:cpd02978:0:0:\"7,8-Dihydroneopterin 3\\'-triphosphate\"'},\n",
       " {'abbreviation': 'R00425',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GTP-CYCLOHYDRO-II-RXN',\n",
       "   'BiGG: GTPCII2; RIB1',\n",
       "   'BrachyCyc: GTP-CYCLOHYDRO-II-RXN',\n",
       "   'KEGG: R00425',\n",
       "   'MetaCyc: GTP-CYCLOHYDRO-II-RXN',\n",
       "   'Name: GTP 7,8-8,9-dihydrolase (diphosphate-forming); GTP cyclohydrolase II; GTP cyclohydrolase II (25drapp); GTP cyclohydrolase-2; GTP-8-formylhydrolase; GTPCII; GTPcyclohydrolaseII; guanosine triphosphate cyclohydrolase II'],\n",
       "  'code': '(3) cpd00001[0] + (1) cpd00038[0] <=> (1) cpd00012[0] + (1) cpd00047[0] + (1) cpd00957[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00038;cpd00047;cpd00067;cpd00957',\n",
       "  'definition': \"(3) H2O[0] + (1) GTP[0] => (1) PPi[0] + (1) Formate[0] + (3) H+[0] + (1) 2,5-Diamino-6-(5'-phosphoribosylamino)-4-pyrimidineone[0]\",\n",
       "  'deltag': -16.79,\n",
       "  'deltagerr': 1.43,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.25'],\n",
       "  'equation': '(3) cpd00001[0] + (1) cpd00038[0] => (1) cpd00012[0] + (1) cpd00047[0] + (3) cpd00067[0] + (1) cpd00957[0]',\n",
       "  'id': 'rxn00300',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32287',\n",
       "  'name': 'GTP 7,8-8,9-dihydrolase (diphosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 6-HM-Dihydropterin-PP-Biosynthesis (6-Hydroxymethyl-Dihydropterin Diphosphate Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Flavin-Biosynthesis (Flavin Biosynthesis); Folate-Biosynthesis (Folate Biosynthesis); PWY-6168 (flavin biosynthesis III (fungi)); PWY-7539 (6-hydroxymethyl-dihydropterin diphosphate biosynthesis III (Chlamydia)); PWY-7991 (toxoflavin biosynthesis); RIBOSYN2-PWY (flavin biosynthesis I (bacteria and plants)); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00740 (Riboflavin metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-3:cpd00001:0:0:\"H2O\";-1:cpd00038:0:0:\"GTP\";1:cpd00012:0:0:\"PPi\";1:cpd00047:0:0:\"Formate\";3:cpd00067:0:0:\"H+\";1:cpd00957:0:0:\"2,5-Diamino-6-(5\\'-phosphoribosylamino)-4-pyrimidineone\"'},\n",
       " {'abbreviation': 'R00426',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.1.19-RXN',\n",
       "   'BiGG: NTPP2',\n",
       "   'BrachyCyc: 3.6.1.19-RXN',\n",
       "   'KEGG: R00426',\n",
       "   'MetaCyc: 3.6.1.19-RXN; RXN-14140',\n",
       "   \"Name: GTP diphosphohydrolase (diphosphate-forming); GTP diphosphohydrolase (diphosphate-forming);; Guanosine 5'-triphosphate pyrophosphohydrolase; Nucleoside triphosphate pyrophosphorylase (gtp); gtp pyrophosphorylase; nucleoside-triphosphate diphosphatase; nucleoside-triphosphate diphosphohydrolase; nucleoside-triphosphate pyrophosphatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00038[0] <=> (1) cpd00012[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00038;cpd00067;cpd00126',\n",
       "  'definition': '(1) H2O[0] + (1) GTP[0] => (1) PPi[0] + (2) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -7.62,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.19', '3.6.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00038[0] => (1) cpd00012[0] + (2) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00301',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33568',\n",
       "  'name': 'GTP diphosphohydrolase (diphosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00038:0:0:\"GTP\";1:cpd00012:0:0:\"PPi\";2:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R00428',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00428_c',\n",
       "   'AraGEM: R_R00428_c',\n",
       "   'KEGG: R00428',\n",
       "   'TS_Athaliana: rxn575_c',\n",
       "   'iRS1563: R_R00428_c',\n",
       "   'Name: GTP 8,9-hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00038[0] <=> (1) cpd03518[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00038;cpd00067;cpd03518',\n",
       "  'definition': '(1) H2O[0] + (1) GTP[0] => (1) H+[0] + (1) Formamidopyrimidine nucleoside triphosphate[0]',\n",
       "  'deltag': -19.32,\n",
       "  'deltagerr': 3.63,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.16'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00038[0] => (1) cpd00067[0] + (1) cpd03518[0]',\n",
       "  'id': 'rxn00302',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GTP 8,9-hydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00790 (Folate biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00038:0:0:\"GTP\";1:cpd00067:0:0:H+;1:cpd03518:0:0:\"Formamidopyrimidine nucleoside triphosphate\"'},\n",
       " {'abbreviation': 'R00429',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GTPPYPHOSKIN-RXN',\n",
       "   'BiGG: GTPDPK',\n",
       "   'BrachyCyc: GTPPYPHOSKIN-RXN',\n",
       "   'KEGG: R00429',\n",
       "   'MetaCyc: GTPPYPHOSKIN-RXN',\n",
       "   \"Name: (p)ppGpp synthetase I; (p)ppGpp synthetase II; ATP-GTP 3'-diphosphotransferase; ATP:GTP 3'-diphosphotransferase; ATP:GTP 3'-pyrophosphotransferase; GPSI; GPSII; GTP diphosphokinase; GTP pyrophosphokinase; guanosine 3',5'-polyphosphate synthase; guanosine 5',3'-polyphosphate synthetase; guanosine pentaphosphate synthetase; stringent factor\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00038[0] <=> (1) cpd00018[0] + (1) cpd02740[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00018;cpd00038;cpd00067;cpd02740',\n",
       "  'definition': \"(1) ATP[0] + (1) GTP[0] => (1) AMP[0] + (2) H+[0] + (1) Guanosine 5'-triphosphate,3'-diphosphate[0]\",\n",
       "  'deltag': -3.17,\n",
       "  'deltagerr': 1.65,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.6.5'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00038[0] => (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd02740[0]',\n",
       "  'id': 'rxn00303',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33193',\n",
       "  'name': \"ATP:GTP 3'-diphosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Metabolic-Regulators (Metabolic Regulator Biosynthesis); PPGPPMET-PWY (ppGpp biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00038:0:0:\"GTP\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd02740:0:0:\"Guanosine 5\\'-triphosphate,3\\'-diphosphate\"'},\n",
       " {'abbreviation': 'R00430',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_GTPOP; R_GTPOPm',\n",
       "   'KEGG: R00430',\n",
       "   'TS_Athaliana: rxn576_c; rxn576_p',\n",
       "   'iRS1563: R_R00430_c',\n",
       "   'iRS1597: R00430_c',\n",
       "   'Name: GTP:pyruvate 2-O-phosphotransferase; GTP:pyruvate O2-phosphotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00038[0] <=> (1) cpd00031[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00031;cpd00038;cpd00061;cpd00067',\n",
       "  'definition': '(1) Pyruvate[0] + (1) GTP[0] <=> (1) GDP[0] + (1) Phosphoenolpyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 7.13,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.40'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00038[0] <=> (1) cpd00031[0] + (1) cpd00061[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00304',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32979;rxn35442',\n",
       "  'name': 'GTP:pyruvate 2-O-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00038:0:0:\"GTP\";1:cpd00031:0:0:\"GDP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00431',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PEPCK_re',\n",
       "   'BrachyCyc: 4.1.1.32-RXN',\n",
       "   'KEGG: R00431',\n",
       "   'MetaCyc: 4.1.1.32-RXN',\n",
       "   'RiceCyc: 4.1.1.32-RXN',\n",
       "   'Name: GTP:oxaloacetate carboxy-lyase (adding GTP, phosphoenolpyruvate-forming); GTP:oxaloacetate carboxy-lyase (adding GTP;phosphoenolpyruvate-forming); GTP:oxaloacetate carboxy-lyase (transphosphorylating); PEP carboxylase; phosphoenolpyruvate carboxykinase; phosphoenolpyruvate carboxykinase (GTP); phosphoenolpyruvate carboxylase; phosphoenolpyruvic carboxykinase; phosphoenolpyruvic carboxykinase (GTP); phosphoenolpyruvic carboxylase (GTP); phosphopyruvate (guanosine triphosphate) carboxykinase; phosphopyruvate carboxylase; phosphopyruvate carboxylase (GTP)'],\n",
       "  'code': '(1) cpd00032[0] + (1) cpd00038[0] <=> (1) cpd00011[0] + (1) cpd00031[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00031;cpd00032;cpd00038;cpd00061',\n",
       "  'definition': '(1) Oxaloacetate[0] + (1) GTP[0] => (1) CO2[0] + (1) GDP[0] + (1) Phosphoenolpyruvate[0]',\n",
       "  'deltag': 3.17,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.32'],\n",
       "  'equation': '(1) cpd00032[0] + (1) cpd00038[0] => (1) cpd00011[0] + (1) cpd00031[0] + (1) cpd00061[0]',\n",
       "  'id': 'rxn00305',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GTP:oxaloacetate carboxy-lyase (adding GTP;phosphoenolpyruvate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Gluconeogenesis (Gluconeogenesis); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY66-399 (gluconeogenesis III)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00032:0:0:\"Oxaloacetate\";-1:cpd00038:0:0:\"GTP\";1:cpd00011:0:0:\"CO2\";1:cpd00031:0:0:\"GDP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\"'},\n",
       " {'abbreviation': 'R00432',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUCCINATE--COA-LIGASE-GDP-FORMING-RXN',\n",
       "   'BrachyCyc: SUCCINATE--COA-LIGASE-GDP-FORMING-RXN',\n",
       "   'ChlamyCyc: SUCCINATE--COA-LIGASE-GDP-FORMING-RXN',\n",
       "   'KEGG: R00432',\n",
       "   'MetaCyc: SUCCINATE--COA-LIGASE-GDP-FORMING-RXN',\n",
       "   'Name: G-STK; P-enzyme; SCS; Succinate:CoA ligase (GDP-forming); succinate thiokinase; succinate-CoA ligase (GDP-forming); succinate-phosphorylating enzyme; succinate:CoA ligase (GDP-forming); succinic thiokinase; succinyl CoA synthetase; succinyl coenzyme A synthetase; succinyl coenzyme A synthetase (GDP-forming); succinyl coenzyme A synthetase (guanosine diphosphate-forming); succinyl-CoA synthetase (GDP-forming)'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00036[0] + (1) cpd00038[0] <=> (1) cpd00009[0] + (1) cpd00031[0] + (1) cpd00078[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00010;cpd00031;cpd00036;cpd00038;cpd00078',\n",
       "  'definition': '(1) CoA[0] + (1) Succinate[0] + (1) GTP[0] <=> (1) Phosphate[0] + (1) GDP[0] + (1) Succinyl-CoA[0]',\n",
       "  'deltag': 0.11,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.4'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00036[0] + (1) cpd00038[0] <=> (1) cpd00009[0] + (1) cpd00031[0] + (1) cpd00078[0]',\n",
       "  'id': 'rxn00306',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27228',\n",
       "  'name': 'Succinate:CoA ligase (GDP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY66-398 (TCA cycle III (animals)); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00640 (Propanoate metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00036:0:0:\"Succinate\";-1:cpd00038:0:0:\"GTP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00031:0:0:\"GDP\";1:cpd00078:0:0:\"Succinyl-CoA\"'},\n",
       " {'abbreviation': 'R00434',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GUANYLCYC-RXN',\n",
       "   'BiGG: GUACYC',\n",
       "   'ChlamyCyc: GUANYLCYC-RXN',\n",
       "   'KEGG: R00434',\n",
       "   'MetaCyc: GUANYLCYC-RXN',\n",
       "   \"Name: GTP diphosphate-lyase (cyclizing); GTP diphosphate-lyase (cyclizing, 3',5'-cyclic-GMP-forming); GTP diphosphate-lyase (cyclizing; 3',5'-cyclic-GMP-forming); GTP pyrophosphate-lyase (cyclizing); guanyl cyclase; guanylate cyclase; guanylyl cyclase\"],\n",
       "  'code': '(1) cpd00038[0] <=> (1) cpd00012[0] + (1) cpd00697[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00038;cpd00067;cpd00697',\n",
       "  'definition': '(1) GTP[0] => (1) PPi[0] + (1) H+[0] + (1) cGMP[0]',\n",
       "  'deltag': 1.22,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.6.1.1', '4.6.1.2'],\n",
       "  'equation': '(1) cpd00038[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00697[0]',\n",
       "  'id': 'rxn00307',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35443',\n",
       "  'name': \"GTP diphosphate-lyase (cyclizing; 3',5'-cyclic-GMP-forming)\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00038:0:0:\"GTP\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd00697:0:0:\"cGMP\"'},\n",
       " {'abbreviation': 'R00445',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00445',\n",
       "   'MetaCyc: LYSINE-DEHYDROGENASE-RXN',\n",
       "   'iRS1563: R_R00445_c',\n",
       "   'Name: L-lysine:NAD+ oxidoreductase (deaminating, cyclizing); lysine dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00039[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00922[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00013;cpd00039;cpd00067;cpd00922',\n",
       "  'definition': '(1) NAD[0] + (1) L-Lysine[0] => (1) NADH[0] + (1) NH3[0] + (1) H+[0] + (1) delta1-Piperideine-2-carboxylate[0]',\n",
       "  'deltag': -4.58,\n",
       "  'deltagerr': 2.16,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.1.15'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00039[0] => (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00922[0]',\n",
       "  'id': 'rxn00308',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-lysine:NAD+ oxidoreductase (deaminating, cyclizing)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00922:0:0:\"delta1-Piperideine-2-carboxylate\"'},\n",
       " {'abbreviation': 'R00446',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00446',\n",
       "   'MetaCyc: RXN-8808',\n",
       "   'Name: L-Lysine:NAD+ 6-oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00039[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd01046[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00039;cpd00067;cpd01046',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Lysine[0] <=> (1) NADH[0] + (1) NH3[0] + (1) H+[0] + (1) Allysine[0]',\n",
       "  'deltag': 8.65,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.18'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00039[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd01046[0]',\n",
       "  'id': 'rxn00309',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Lysine:NAD+ 6-oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); PWY-5314 (L-lysine degradation VIII); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd01046:0:0:\"Allysine\"'},\n",
       " {'abbreviation': 'R00447',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00447',\n",
       "   'MetaCyc: L-LYSINE-OXIDASE-RXN',\n",
       "   'Name: L-Lysine:oxygen oxidoreductase(deaminating); L-lysine alpha-oxidase; L-lysine oxidase; L-lysine:oxygen 2-oxidoreductase (deaminating); L-lysyl-alpha-oxidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00039[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd02074[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00039;cpd02074',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Lysine[0] => (1) NH3[0] + (1) H2O2[0] + (1) 2-Oxo-6-aminocaproate[0]',\n",
       "  'deltag': -24.99,\n",
       "  'deltagerr': 1.15,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.14'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00039[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd02074[0]',\n",
       "  'id': 'rxn00310',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-lysine:oxygen 2-oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); PWY-5311 (L-lysine degradation VII); PWY-5327 (superpathway of L-lysine degradation); PWY66-425 (L-lysine degradation II (L-pipecolate pathway)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00310 (Lysine degradation); rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd02074:0:0:\"2-Oxo-6-aminocaproate\"'},\n",
       " {'abbreviation': 'R00448',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00448',\n",
       "   'MetaCyc: 1.14.13.59-RXN',\n",
       "   'Name: L-lysine 6-monooxygenase (NADPH); L-lysine,NADPH:oxygen oxidoreductase (6-hydroxylating); lysine N6-hydroxylase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00039[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00757[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00039;cpd00757',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) L-Lysine[0] => (1) H2O[0] + (1) NADP[0] + (1) N6-Hydroxy-L-lysine[0]',\n",
       "  'deltag': -90.63,\n",
       "  'deltagerr': 6.19,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.59'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00039[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00757[0]',\n",
       "  'id': 'rxn00311',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17089',\n",
       "  'name': 'L-lysine,NADPH:oxygen oxidoreductase (6-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: AEROBACTINSYN-PWY (aerobactin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Siderophores-Biosynthesis (Siderophore Biosynthesis)',\n",
       "   'KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00757:0:0:\"N6-Hydroxy-L-lysine\"'},\n",
       " {'abbreviation': 'R00449',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: LYSMO',\n",
       "   'KEGG: R00449',\n",
       "   'MetaCyc: LYSINE-2-MONOOXYGENASE-RXN',\n",
       "   'iJN746: LYSMO',\n",
       "   'Name: L-Lysine:oxygen 2-oxidoreductase (decarboxylating); L-lysine-2-monooxygenase; L-lysine:oxygen 2-oxidoreductase (decarboxylating); lysine 2-monooxygenase; lysine monooxygenase; lysine oxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00039[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00729[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00011;cpd00039;cpd00729',\n",
       "  'definition': '(1) O2[0] + (1) L-Lysine[0] => (1) H2O[0] + (1) CO2[0] + (1) 5-Aminopentanamide[0]',\n",
       "  'deltag': -111.77,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.12.2'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00039[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00729[0]',\n",
       "  'id': 'rxn00312',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Lysine:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); PWY-5280 (L-lysine degradation IV); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd00729:0:0:\"5-Aminopentanamide\"'},\n",
       " {'abbreviation': 'R00451',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DIAMINOPIMDECARB-RXN',\n",
       "   'BiGG: DAPDC',\n",
       "   'BrachyCyc: DIAMINOPIMDECARB-RXN',\n",
       "   'KEGG: R00451',\n",
       "   'MetaCyc: DIAMINOPIMDECARB-RXN',\n",
       "   'Name: DAP-decarboxylase; diaminopimelate decarboxylase; diaminopimelatedecarboxylase; diaminopimelic acid decarboxylase; meso-2,6-Diaminoheptanedioate carboxy-lyase; meso-2,6-diaminoheptanedioate carboxy-lyase; meso-2,6-diaminoheptanedioate carboxy-lyase (L-lysine-forming); meso-diaminopimelate decarboxylase'],\n",
       "  'code': '(1) cpd00516[0] <=> (1) cpd00011[0] + (1) cpd00039[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00039;cpd00067;cpd00516',\n",
       "  'definition': '(1) H+[0] + (1) meso-2,6-Diaminopimelate[0] => (1) CO2[0] + (1) L-Lysine[0]',\n",
       "  'deltag': -4.39,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.20'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00516[0] => (1) cpd00011[0] + (1) cpd00039[0]',\n",
       "  'id': 'rxn00313',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27788',\n",
       "  'name': 'meso-2,6-diaminoheptanedioate carboxy-lyase (L-lysine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); DAPLYSINESYN-PWY (L-lysine biosynthesis I); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); LYSINE-SYN (L-lysine Biosynthesis); P4-PWY (superpathway of L-lysine, L-threonine and L-methionine biosynthesis I); PWY-2941 (L-lysine biosynthesis II); PWY-2942 (L-lysine biosynthesis III); PWY-5097 (L-lysine biosynthesis VI); PWY-724 (superpathway of L-lysine, L-threonine and L-methionine biosynthesis II); PWY0-781 (aspartate superpathway)',\n",
       "   'KEGG: rn00300 (Lysine biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00516:0:0:\"meso-2,6-Diaminopimelate\";1:cpd00011:0:0:\"CO2\";1:cpd00039:0:0:\"L-Lysine\"'},\n",
       " {'abbreviation': 'R00452',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00452',\n",
       "   'MetaCyc: D-LYSOPINE-DEHYDROGENASE-RXN',\n",
       "   'Name: (L-lysine-forming); 2-N-(D-1-carboxyethyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming); D(+)-lysopine dehydrogenase; D-lysopine dehydrogenase; D-lysopine synthase; N2-(D-1-carboxyethyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming); N2-(D-1-carboxylethyl)-L-lysine:NADP+ oxidoreductase; N2-(D-1-carboxylethyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming); lysopine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd02490[0] <=> (1) cpd00005[0] + (1) cpd00020[0] + (1) cpd00039[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00020;cpd00039;cpd00067;cpd02490',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) Lysopine[0] <=> (1) NADPH[0] + (1) Pyruvate[0] + (1) L-Lysine[0] + (1) H+[0]',\n",
       "  'deltag': 9.66,\n",
       "  'deltagerr': 0.84,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.16'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd02490[0] <=> (1) cpd00005[0] + (1) cpd00020[0] + (1) cpd00039[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00314',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N2-(D-1-carboxylethyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd02490:0:0:\"Lysopine\";1:cpd00005:0:0:\"NADPH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00453',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00453',\n",
       "   'MetaCyc: LYSINE--PYRUVATE-6-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-Lysine:pyruvate aminotransferase; L-lysine:pyruvate aminotransferase; Lys-AT; lysine-pyruvate 6-transaminase; lysine-pyruvate aminotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00039[0] <=> (1) cpd00035[0] + (1) cpd02522[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00035;cpd00039;cpd02522',\n",
       "  'definition': '(1) Pyruvate[0] + (1) L-Lysine[0] <=> (1) L-Alanine[0] + (1) 2-Aminoadipate 6-semialdehyde[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.71'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00039[0] <=> (1) cpd00035[0] + (1) cpd02522[0]',\n",
       "  'id': 'rxn00315',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Lysine:pyruvate aminotransferase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); PWY-5324 (L-lysine degradation IX); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd02522:0:0:\"2-Aminoadipate 6-semialdehyde\"'},\n",
       " {'abbreviation': 'R00454',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00454',\n",
       "   'Name: acetyl-CoA:L-lysine N6-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00039[0] <=> (1) cpd00010[0] + (1) cpd01770[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00039;cpd00067;cpd01770',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Lysine[0] => (1) CoA[0] + (1) H+[0] + (1) N6-Acetyl-L-lysine[0]',\n",
       "  'deltag': -5.7,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.-'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00039[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01770[0]',\n",
       "  'id': 'rxn00316',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'acetyl-CoA:L-lysine N6-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd01770:0:0:\"N6-Acetyl-L-lysine\"'},\n",
       " {'abbreviation': 'R00457',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00457',\n",
       "   'MetaCyc: L-LYSINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-Lysine:2-oxoglutarate 6-aminotransferase; L-lysine 6-transaminase; L-lysine-alpha-ketoglutarate 6-aminotransferase; L-lysine-alpha-ketoglutarate aminotransferase; L-lysine:2-oxoglutarate 6-aminotransferase; lysine 6-aminotransferase; lysine epsilon-aminotransferase; lysine epsilon-transaminase; lysine:2-ketoglutarate 6-aminotransferase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00039[0] <=> (1) cpd00023[0] + (1) cpd02522[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00039;cpd02522',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Lysine[0] <=> (1) L-Glutamate[0] + (1) 2-Aminoadipate 6-semialdehyde[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.36'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00039[0] <=> (1) cpd00023[0] + (1) cpd02522[0]',\n",
       "  'id': 'rxn00317',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Lysine:2-oxoglutarate 6-aminotransferase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); PWY-5298 (L-lysine degradation VI); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00039:0:0:\"L-Lysine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd02522:0:0:\"2-Aminoadipate 6-semialdehyde\"'},\n",
       " {'abbreviation': 'R00458',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00458', 'Name: N6-acetyl-L-lysine amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01770[0] <=> (1) cpd00029[0] + (1) cpd00039[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00039;cpd01770',\n",
       "  'definition': '(1) H2O[0] + (1) N6-Acetyl-L-lysine[0] <=> (1) Acetate[0] + (1) L-Lysine[0]',\n",
       "  'deltag': -1.6,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.17'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01770[0] <=> (1) cpd00029[0] + (1) cpd00039[0]',\n",
       "  'id': 'rxn00318',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N6-acetyl-L-lysine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01770:0:0:\"N6-Acetyl-L-lysine\";1:cpd00029:0:0:\"Acetate\";1:cpd00039:0:0:\"L-Lysine\"'},\n",
       " {'abbreviation': 'rxn00319',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R10277',\n",
       "   'MetaCyc: RXN-13654',\n",
       "   'Name: L-lysine ammonia-lyase (cyclizing; ammonia-forming)'],\n",
       "  'code': '(1) cpd00039[0] <=> (1) cpd00013[0] + (1) cpd00323[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00039;cpd00323',\n",
       "  'definition': '(1) L-Lysine[0] <=> (1) NH3[0] + (1) L-Pipecolate[0]',\n",
       "  'deltag': -9.82,\n",
       "  'deltagerr': 2.16,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.3.1.28'],\n",
       "  'equation': '(1) cpd00039[0] <=> (1) cpd00013[0] + (1) cpd00323[0]',\n",
       "  'id': 'rxn00319',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00319',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00039:0:0:\"L-Lysine\";1:cpd00013:0:0:\"NH3\";1:cpd00323:0:0:\"L-Pipecolate\"'},\n",
       " {'abbreviation': 'R00460',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: LYSRC',\n",
       "   'KEGG: R00460',\n",
       "   'MetaCyc: LYSINE-RACEMASE-RXN',\n",
       "   'iJN746: LYSRC',\n",
       "   'Name: L-Lysine racemase; lysine racemase'],\n",
       "  'code': '(1) cpd00039[0] <=> (1) cpd00549[0]',\n",
       "  'compound_ids': 'cpd00039;cpd00549',\n",
       "  'definition': '(1) L-Lysine[0] <=> (1) D-Lysine[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.5', '5.1.1.9'],\n",
       "  'equation': '(1) cpd00039[0] <=> (1) cpd00549[0]',\n",
       "  'id': 'rxn00320',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'lysine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); PWY-5283 (L-lysine degradation V); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00039:0:0:\"L-Lysine\";1:cpd00549:0:0:\"D-Lysine\"'},\n",
       " {'abbreviation': 'R00461',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: LYSAM',\n",
       "   'KEGG: R00461',\n",
       "   'MetaCyc: LYSINE-23-AMINOMUTASE-RXN',\n",
       "   'iAF692: LYSAM',\n",
       "   'iRR1083: LYSAM',\n",
       "   'Name: L-Lysine 2,3-aminomutase; L-lysine 2,3-aminomutase; lysine 2,3-aminomutase'],\n",
       "  'code': '(1) cpd00039[0] <=> (1) cpd00840[0]',\n",
       "  'compound_ids': 'cpd00039;cpd00840',\n",
       "  'definition': '(1) L-Lysine[0] <=> (1) L-beta-Lysine[0]',\n",
       "  'deltag': -1.1,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.3.2'],\n",
       "  'equation': '(1) cpd00039[0] <=> (1) cpd00840[0]',\n",
       "  'id': 'rxn00321',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Lysine 2,3-aminomutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); LYSINE-DEG (L-lysine Degradation); P163-PWY (L-lysine fermentation to acetate and butanoate); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00039:0:0:\"L-Lysine\";1:cpd00840:0:0:\"L-beta-Lysine\"'},\n",
       " {'abbreviation': 'R00462',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: LYSDECARBOX-RXN',\n",
       "   'BiGG: LYSDC',\n",
       "   'ChlamyCyc: LYSDECARBOX-RXN',\n",
       "   'KEGG: R00462',\n",
       "   'MetaCyc: LYSDECARBOX-RXN',\n",
       "   'Name: L-Lysine carboxy-lyase; L-lysine carboxy-lyase; L-lysine carboxy-lyase (cadaverine-forming); lysine decarboxylase'],\n",
       "  'code': '(1) cpd00039[0] <=> (1) cpd00011[0] + (1) cpd01155[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00039;cpd00067;cpd01155',\n",
       "  'definition': '(1) L-Lysine[0] + (1) H+[0] => (1) CO2[0] + (1) Cadaverine[0]',\n",
       "  'deltag': -5.96,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.18'],\n",
       "  'equation': '(1) cpd00039[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd01155[0]',\n",
       "  'id': 'rxn00322',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20057',\n",
       "  'name': 'L-lysine carboxy-lyase (cadaverine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); POLYAMSYN-PWY (superpathway of polyamine biosynthesis I); PWY-5327 (superpathway of L-lysine degradation); PWY-5468 (lupanine biosynthesis); PWY-6328 (L-lysine degradation X); PWY-6375 (desferrioxamine E biosynthesis); PWY-6376 (desferrioxamine B biosynthesis); PWY-6381 (bisucaberin biosynthesis); PWY0-1303 (aminopropylcadaverine biosynthesis); PWY0-461 (L-lysine degradation I); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); QUINOLIZIDINE-ALKALOIDS (Quinolizidine Alkaloid Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Siderophores-Biosynthesis (Siderophore Biosynthesis)',\n",
       "   'KEGG: rn00310 (Lysine degradation); rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00039:0:0:\"L-Lysine\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd01155:0:0:\"Cadaverine\"'},\n",
       " {'abbreviation': 'R00463',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00463',\n",
       "   'MetaCyc: L-LYSINE-LACTAMASE-RXN',\n",
       "   'Name: L-Lysine-1,6-lactam lactamhydrolase; L-alpha-aminocaprolactam hydrolase; L-lysinamidase; L-lysine-1,6-lactam lactamhydrolase; L-lysine-lactamase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01832[0] <=> (1) cpd00039[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00039;cpd01832',\n",
       "  'definition': '(1) H2O[0] + (1) L-Lysine 1,6-lactam[0] <=> (1) L-Lysine[0]',\n",
       "  'deltag': 4.61,\n",
       "  'deltagerr': 1.01,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.2.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01832[0] <=> (1) cpd00039[0]',\n",
       "  'id': 'rxn00323',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Lysine-1,6-lactam lactamhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01832:0:0:\"L-Lysine 1,6-lactam\";1:cpd00039:0:0:\"L-Lysine\"'},\n",
       " {'abbreviation': 'R00465',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYOXYLATE-REDUCTASE-NADP+-RXN',\n",
       "   'BiGG: GLYCLTDy; GLYCLTDyr',\n",
       "   'BrachyCyc: GLYOXYLATE-REDUCTASE-NADP+-RXN',\n",
       "   'KEGG: R00465',\n",
       "   'MetaCyc: GLYOXYLATE-REDUCTASE-NADP+-RXN',\n",
       "   'Name: Glycolate dehydrogenase (NADP); Glycolate:NADP+ oxidoreductase; Glyoxylate reductase (NADP+); NADPH-glyoxylate reductase; glycolate:NADP+ oxidoreductase; glyoxylate reductase (NADP); glyoxylate reductase (NADP+)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00139[0] <=> (1) cpd00005[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00040;cpd00067;cpd00139',\n",
       "  'definition': '(1) NADP[0] + (1) Glycolate[0] <=> (1) NADPH[0] + (1) Glyoxalate[0] + (1) H+[0]',\n",
       "  'deltag': 10.54,\n",
       "  'deltagerr': 0.22,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.26', '1.1.1.79'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00139[0] <=> (1) cpd00005[0] + (1) cpd00040[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00324',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27915;rxn32871',\n",
       "  'name': 'glycolate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00139:0:0:\"Glycolate\";1:cpd00005:0:0:\"NADPH\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00466',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00466',\n",
       "   'MetaCyc: GLYOXYLATE-OXIDASE-RXN',\n",
       "   'iRS1563: R_R00466_c',\n",
       "   'Name: Glyoxylate:oxygen oxidoreductase; glyoxylate oxidase; glyoxylate:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00040[0] <=> (1) cpd00025[0] + (1) cpd00180[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00040;cpd00067;cpd00180',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Glyoxalate[0] => (1) H2O2[0] + (1) H+[0] + (1) Oxalate[0]',\n",
       "  'deltag': -37.75,\n",
       "  'deltagerr': 1.3,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.3.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00040[0] => (1) cpd00025[0] + (1) cpd00067[0] + (1) cpd00180[0]',\n",
       "  'id': 'rxn00325',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glyoxylate:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00040:0:0:\"Glyoxalate\";1:cpd00025:0:0:\"H2O2\";1:cpd00067:0:0:\"H+\";1:cpd00180:0:0:\"Oxalate\"'},\n",
       " {'abbreviation': 'R00468',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00468',\n",
       "   'MetaCyc: GLYOXYLATE-DEHYDROGENASE-ACYLATING-RXN',\n",
       "   'Name: Glyoxylate:NADP+ oxidoreductase (CoA-oxalylating); glyoxylate dehydrogenase (acylating); glyoxylate:NADP+ oxidoreductase (CoA-oxalylating)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00040[0] <=> (1) cpd00005[0] + (1) cpd00262[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00010;cpd00040;cpd00067;cpd00262',\n",
       "  'definition': '(1) NADP[0] + (1) CoA[0] + (1) Glyoxalate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Oxalyl-CoA[0]',\n",
       "  'deltag': -2.55,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.17'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00040[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00262[0]',\n",
       "  'id': 'rxn00326',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glyoxylate:NADP+ oxidoreductase (CoA-oxalylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Oxalate-Degradation (Oxalate Degradation); PWY-6696 (oxalate degradation III)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00040:0:0:\"Glyoxalate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:H+;1:cpd00262:0:0:\"Oxalyl-CoA\"'},\n",
       " {'abbreviation': 'R00469',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UREIDOGLYCOLATE-HYDROLASE-RXN',\n",
       "   'BiGG: DAL3; UGLYCH; UGLYCHr',\n",
       "   'CornCyc: UREIDOGLYCOLATE-HYDROLASE-RXN',\n",
       "   'KEGG: R00469',\n",
       "   'MetaCyc: UREIDOGLYCOLATE-HYDROLASE-RXN',\n",
       "   'Name: (S)-ureidoglycolate amidohydrolase (decarboxylating); Ureidoglycolate amidohydrolase (decarboxylating); Ureidoglycolate hydrolase; ureidoglycolate hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00465[0] <=> (1) cpd00011[0] + (2) cpd00013[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00040;cpd00067;cpd00465',\n",
       "  'definition': '(1) H2O[0] + (2) H+[0] + (1) Ureidoglycolate[0] => (1) CO2[0] + (2) NH3[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': -5.9,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.116', '3.5.3.19'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00067[0] + (1) cpd00465[0] => (1) cpd00011[0] + (2) cpd00013[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00327',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27427;rxn27428;rxn30069',\n",
       "  'name': '(S)-ureidoglycolate amidohydrolase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Allantoin-degradation (Allantoin Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5692 (allantoin degradation to glyoxylate II); URDEGR-PWY (superpathway of allantoin degradation in plants)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00067:0:0:\"H+\";-1:cpd00465:0:0:\"Ureidoglycolate\";1:cpd00011:0:0:\"CO2\";2:cpd00013:0:0:\"NH3\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00470',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: DDPGA',\n",
       "   'EcoCyc: 4OH2OXOGLUTARALDOL-RXN',\n",
       "   'KEGG: R00470',\n",
       "   'MetaCyc: 4OH2OXOGLUTARALDOL-RXN',\n",
       "   'RiceCyc: 4OH2OXOGLUTARALDOL-RXN',\n",
       "   'Name: 2-dehydro-3-deoxy-phosphogluconatealdolase; 2-keto-4-hydroxybutyrate aldolase; 2-keto-4-hydroxyglutarate aldolase; 2-keto-4-hydroxyglutaric aldolase; 2-oxo-4-hydroxyglutarate aldolase; 2-oxo-4-hydroxyglutaric aldolase; 4-Hydroxy-2-oxoglutarate glyoxylate-lyase; 4-hydroxy-2-ketoglutarate aldolase; 4-hydroxy-2-ketoglutaric aldolase; 4-hydroxy-2-oxoglutarate aldolase; 4-hydroxy-2-oxoglutarate glyoxylate-lyase; 4-hydroxy-2-oxoglutarate glyoxylate-lyase (pyruvate-forming); DL-4-hydroxy-2-ketoglutarate aldolase; KHGA; hydroxyketoglutarate aldolase; hydroxyketoglutaric aldolase'],\n",
       "  'code': '(1) cpd00830[0] <=> (1) cpd00020[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00040;cpd00830',\n",
       "  'definition': '(1) 4-Hydroxy-2-oxoglutarate[0] <=> (1) Pyruvate[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 4.36,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.16', '4.1.3.42'],\n",
       "  'equation': '(1) cpd00830[0] <=> (1) cpd00020[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00328',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18964',\n",
       "  'name': '4-hydroxy-2-oxoglutarate glyoxylate-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); HYDROXYPRODEG-PWY (<i>trans</i>-4-hydroxy-L-proline degradation I); Hydroxyprolines-Degradation (hydroxy-L-proline Degradation); MISCELLANEOUS-DEG (Other Amino Acid Degradation)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00830:0:0:\"4-Hydroxy-2-oxoglutarate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00472',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MALSYN-RXN',\n",
       "   'BiGG: DAL7; MALS; MALSp; MLS1',\n",
       "   'BrachyCyc: MALSYN-RXN',\n",
       "   'KEGG: R00472',\n",
       "   'MetaCyc: MALSYN-RXN',\n",
       "   'Name: L-Malate glyoxylate-lyase (CoA-acetylating); L-malate glyoxylate-lyase (CoA-acetylating); Malate Synthase; acetyl-CoA:glyoxylate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming); glyoxylate transacetase; glyoxylate transacetylase; glyoxylic transacetase; malate condensing enzyme; malate synthase; malate synthetase; malatesynthase; malic synthetase; malic-condensing enzyme'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00130[0] <=> (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00022;cpd00040;cpd00067;cpd00130',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) L-Malate[0] <= (1) H2O[0] + (1) Acetyl-CoA[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 8.1,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.9', '4.1.3.2'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00130[0] <= (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00330',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20162;rxn33881;rxn34243;rxn34244',\n",
       "  'name': 'acetyl-CoA:glyoxylate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); GLYCOL-GLYOXDEG-PWY (superpathway of glycol metabolism and degradation); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYOXDEG-PWY (glycolate and glyoxylate degradation II); GLYOXYLATE-BYPASS (glyoxylate cycle); Glycolate-Degradation (Glycolate Degradation); L-Arabinose-Degradation (L-arabinose Degradation); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); PWY-5257 (superpathway of pentose and pentitol degradation); PWY-561 (superpathway of glyoxylate cycle and fatty acid degradation); PWY-6969 (TCA cycle V (2-oxoglutarate:ferredoxin oxidoreductase)); PWY-7118 (chitin degradation to ethanol); PWY-7294 (D-xylose degradation IV); PWY-7295 (L-arabinose degradation IV); PWY-7854 (crotonyl-CoA/ethylmalonyl-CoA/hydroxybutyryl-CoA cycle (engineered)); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); TCA-GLYOX-BYPASS (superpathway of glyoxylate bypass and TCA); TCA-VARIANTS (TCA cycle); Xylose-Degradation (Xylose Degradation)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00001:0:0:\"H2O\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00473',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MCAL',\n",
       "   'KEGG: R00473',\n",
       "   'MetaCyc: MALYL-COA-LYASE-RXN',\n",
       "   'RiceCyc: MALYL-COA-LYASE-RXN',\n",
       "   'iSO783: MCAL',\n",
       "   'Name: (3S)-3-Carboxy-3-hydroxypropanoyl-CoA glyoxylate-lyase; (3S)-3-carboxy-3-hydroxypropanoyl-CoA glyoxylate-lyase; (3S)-3-carboxy-3-hydroxypropanoyl-CoA glyoxylate-lyase (acetyl-CoA-forming); L-malyl-CoA glyoxylate-lyase (acetyl-CoA-forming); Malyl-CoA lyase; malyl-CoA lyase; malyl-coenzyme A lyase'],\n",
       "  'code': '(1) cpd02662[0] <=> (1) cpd00022[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00040;cpd02662',\n",
       "  'definition': '(1) Malyl-CoA[0] <=> (1) Acetyl-CoA[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 3.48,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.24'],\n",
       "  'equation': '(1) cpd02662[0] <=> (1) cpd00022[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00331',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(3S)-3-carboxy-3-hydroxypropanoyl-CoA glyoxylate-lyase (acetyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Formaldehyde-Assimilation (Formaldehyde Assimilation); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-5741 (ethylmalonyl-CoA pathway); PWY-5743 (3-hydroxypropanoate cycle); PWY-6728 (methylaspartate cycle); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn00680 (Methane metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02662:0:0:\"Malyl-CoA\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00474',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00474',\n",
       "   'MetaCyc: 2-HYDROXY-3-OXOADIPATE-SYNTHASE-RXN',\n",
       "   'Name: 2-Hydroxy-3-oxoadipate glyoxylate-lyase (carboxylating); 2-hydroxy-3-oxoadipate glyoxylate-lyase (carboxylating); 2-hydroxy-3-oxoadipate synthase; 2-oxoglutarate:glyoxylate succinaldehydetransferase (decarboxylating); alpha-ketoglutaric-glyoxylic carboligase; oxoglutarate: glyoxylate carboligase'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd02056[0] <=> (1) cpd00024[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00024;cpd00040;cpd00067;cpd02056',\n",
       "  'definition': '(1) CO2[0] + (1) 2-Hydroxy-3-oxoadipate[0] <= (1) 2-Oxoglutarate[0] + (1) Glyoxalate[0] + (1) H+[0]',\n",
       "  'deltag': 7.02,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.2.1.5'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd02056[0] <= (1) cpd00024[0] + (1) cpd00040[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00332',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-oxoglutarate:glyoxylate succinaldehydetransferase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd02056:0:0:\"2-Hydroxy-3-oxoadipate\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00475',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-969',\n",
       "   'BiGG: GLYCTO1; GLYCTO10',\n",
       "   'BrachyCyc: RXN-969',\n",
       "   'KEGG: R00475',\n",
       "   'MetaCyc: RXN-969',\n",
       "   'Name: (S)-2-Hydroxy-acid:oxygen 2-oxidoreductase; (S)-2-hydroxy-acid oxidase; (S)-2-hydroxy-acid oxidase, putative / glycolate oxidase, putative / short chain alpha-hydroxy acid oxidase, putative; Glycolate:oxygen 2-oxidoreductase; Glycolateoxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00139[0] <=> (1) cpd00025[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00040;cpd00139',\n",
       "  'definition': '(1) O2[0] + (1) Glycolate[0] => (1) H2O2[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': -21.19,\n",
       "  'deltagerr': 1.04,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.15'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00139[0] => (1) cpd00025[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00333',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn25769;rxn31214;rxn32255;rxn34231',\n",
       "  'name': 'Glycolate:oxygen 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-181 (photorespiration); Photosynthesis ()',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00139:0:0:\"Glycolate\";1:cpd00025:0:0:\"H2O2\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00477',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00477',\n",
       "   'MetaCyc: OXALOMALATE-LYASE-RXN',\n",
       "   'Name: 3-Oxalomalate glyoxylate-lyase; 3-oxalomalate glyoxylate-lyase; 3-oxalomalate glyoxylate-lyase (oxaloacetate-forming); oxalomalate lyase'],\n",
       "  'code': '(1) cpd01363[0] <=> (1) cpd00032[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00032;cpd00040;cpd01363',\n",
       "  'definition': '(1) 3-Oxalomalate[0] <=> (1) Oxaloacetate[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 4.21,\n",
       "  'deltagerr': 0.74,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.13'],\n",
       "  'equation': '(1) cpd01363[0] <=> (1) cpd00032[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00334',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-oxalomalate glyoxylate-lyase (oxaloacetate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd01363:0:0:\"3-Oxalomalate\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00478',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00478',\n",
       "   'MetaCyc: RXN-11769',\n",
       "   'Name: L-erythro-3-hydroxy-aspartate glyoxylate-lyase (glycine-forming); L-erythro-3-hydroxyaspartate aldolase; L-erythro-beta-hydroxyaspartate aldolase; L-erythro-beta-hydroxyaspartate glycine-lyase; erythro-3-Hydroxy-L-aspartate glyoxylate-lyase; erythro-3-hydroxy-Ls-aspartate glyoxylate-lyase; erythro-3-hydroxy-Ls-aspartate glyoxylate-lyase (glycine-forming)'],\n",
       "  'code': '(1) cpd02458[0] <=> (1) cpd00033[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00033;cpd00040;cpd02458',\n",
       "  'definition': '(1) erythro-3-Hydroxy-L-aspartate[0] <=> (1) Glycine[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 0.6,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.14'],\n",
       "  'equation': '(1) cpd02458[0] <=> (1) cpd00033[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00335',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'erythro-3-hydroxy-Ls-aspartate glyoxylate-lyase (glycine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Glycolate-Degradation (Glycolate Degradation); PWY-6649 (glycolate and glyoxylate degradation III)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02458:0:0:\"erythro-3-Hydroxy-L-aspartate\";1:cpd00033:0:0:\"Glycine\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00479',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00479_x',\n",
       "   'BiGG: ICL; ICL1; ICL2',\n",
       "   'JM_Creinhardtii: JM_Cre_142',\n",
       "   'JP_Creinhardtii_MSB: R_ICL',\n",
       "   'KEGG: R00479',\n",
       "   'Name: Isocitrate Lyase; Isocitrate glyoxylate-lyase; Isocitrate lyase; isocitrate glyoxylate-lyase (succinate-forming)'],\n",
       "  'code': '(1) cpd00260[0] <=> (1) cpd00036[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00036;cpd00040;cpd00260',\n",
       "  'definition': '(1) Isocitrate[0] <=> (1) Succinate[0] + (1) Glyoxalate[0]',\n",
       "  'deltag': 2.25,\n",
       "  'deltagerr': 0.33,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.1'],\n",
       "  'equation': '(1) cpd00260[0] <=> (1) cpd00036[0] + (1) cpd00040[0]',\n",
       "  'id': 'rxn00336',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33532;rxn33534;rxn33880',\n",
       "  'name': 'isocitrate glyoxylate-lyase (succinate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00260:0:0:\"Isocitrate\";1:cpd00036:0:0:\"Succinate\";1:cpd00040:0:0:\"Glyoxalate\"'},\n",
       " {'abbreviation': 'R00480',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASPARTATEKIN-RXN',\n",
       "   'BiGG: ASPK; ASPKi; HOM3',\n",
       "   'BrachyCyc: ASPARTATEKIN-RXN',\n",
       "   'KEGG: R00480',\n",
       "   'MetaCyc: ASPARTATEKIN-RXN',\n",
       "   'Name: AK; ATP:L-aspartate 4-phosphotransferase; Aspartate kinase; Aspartokinase; aspartate kinase; aspartate kinase, irreversible; aspartatekinase; aspartic kinase; beta-aspartokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00041[0] <=> (1) cpd00008[0] + (1) cpd01977[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00041;cpd01977',\n",
       "  'definition': '(1) ATP[0] + (1) L-Aspartate[0] <=> (1) ADP[0] + (1) 4-Phospho-L-aspartate[0]',\n",
       "  'deltag': 4.58,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.4'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00041[0] <=> (1) cpd00008[0] + (1) cpd01977[0]',\n",
       "  'id': 'rxn00337',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27724',\n",
       "  'name': 'ATP:L-aspartate 4-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 3-Dehydroquinate-Biosynthesis (3-Dehydroquinate Biosynthesis); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Chorismate-Biosynthesis (Chorismate Biosynthesis); DAPLYSINESYN-PWY (L-lysine biosynthesis I); HOMOSERSYN-PWY (L-homoserine biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); ISOLEUCINE-SYN (L-isoleucine Biosynthesis); LYSINE-SYN (L-lysine Biosynthesis); MET-SAM-PWY (superpathway of <i>S</i>-adenosyl-L-methionine biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); METSYN-PWY (superpathway of L-homoserine and L-methionine biosynthesis); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); P101-PWY (ectoine biosynthesis); P4-PWY (superpathway of L-lysine, L-threonine and L-methionine biosynthesis I); PWY-2941 (L-lysine biosynthesis II); PWY-2942 (L-lysine biosynthesis III); PWY-3001 (superpathway of L-isoleucine biosynthesis I); PWY-5097 (L-lysine biosynthesis VI); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-5347 (superpathway of L-methionine biosynthesis (transsulfuration)); PWY-6160 (3-dehydroquinate biosynthesis II (archaea)); PWY-6165 (chorismate biosynthesis II (archaea)); PWY-6559 (spermidine biosynthesis II); PWY-6562 (norspermidine biosynthesis); PWY-6565 (superpathway of polyamine biosynthesis III); PWY-7153 (grixazone biosynthesis); PWY-724 (superpathway of L-lysine, L-threonine and L-methionine biosynthesis II); PWY-7977 (L-methionine biosynthesis IV (archaea)); PWY0-781 (aspartate superpathway); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Spermidine-Biosynthesis (Spermidine Biosynthesis); THREONINE-BIOSYNTHESIS (L-threonine Biosynthesis); THRESYN-PWY (superpathway of L-threonine biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00270 (Cysteine and methionine metabolism); rn00300 (Lysine biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00008:0:0:\"ADP\";1:cpd01977:0:0:\"4-Phospho-L-aspartate\"'},\n",
       " {'abbreviation': 'R00481',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: L-ASPARTATE-OXID-RXN',\n",
       "   'BiGG: ASPO6',\n",
       "   'BrachyCyc: L-ASPARTATE-OXID-RXN',\n",
       "   'KEGG: R00481',\n",
       "   'MetaCyc: L-ASPARTATE-OXID-RXN',\n",
       "   'Name: AO; L-aspartate oxidase; L-aspartate:oxygen oxidoreductase; Laspo; NadB; l-aspartate oxidase (producing iminoaspartate)'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00041[0] <=> (1) cpd00025[0] + (1) cpd03470[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00041;cpd03470',\n",
       "  'definition': '(1) O2[0] + (1) L-Aspartate[0] => (1) H2O2[0] + (1) Iminoaspartate[0]',\n",
       "  'deltag': -17.2,\n",
       "  'deltagerr': 1.16,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.16'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00041[0] => (1) cpd00025[0] + (1) cpd03470[0]',\n",
       "  'id': 'rxn00338',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20003',\n",
       "  'name': 'L-aspartate:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); PWY-5316 (nicotine biosynthesis); PWY-7342 (superpathway of nicotine biosynthesis); PWY0-781 (aspartate superpathway); PYRIDNUCSYN-PWY (NAD <i>de novo</i> biosynthesis I (from aspartate)); PYRROLIDINE-ALKALOIDS (Pyrrolidine, Piperidine and Pyridine Alkaloid Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00025:0:0:\"H2O2\";1:cpd03470:0:0:\"Iminoaspartate\"'},\n",
       " {'abbreviation': 'R00482',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00482',\n",
       "   'MetaCyc: 6.3.1.4-RXN',\n",
       "   'Name: L-Aspartate:ammonia ligase (ADP-forming); L-aspartate:ammonia ligase (ADP-forming); asparagine synthetase (ADP-forming); asparagine synthetase (adenosine diphosphate-forming); aspartate-ammonia ligase (ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00041[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00132[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00013;cpd00041;cpd00067;cpd00132',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) L-Aspartate[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) L-Asparagine[0]',\n",
       "  'deltag': -2.15,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.1.4'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00041[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00132[0]',\n",
       "  'id': 'rxn00339',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Aspartate:ammonia ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00132:0:0:\"L-Asparagine\"'},\n",
       " {'abbreviation': 'R00483',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASNSYNA-RXN',\n",
       "   'BiGG: ASNS2',\n",
       "   'CornCyc: ASNSYNA-RXN',\n",
       "   'KEGG: R00483',\n",
       "   'MetaCyc: ASNSYNA-RXN',\n",
       "   'Name: L-Aspartate:ammonia ligase (AMP-forming); L-asparagine synthetase; L-aspartate:ammonia ligase (AMP-forming); asparagine synthetase; asparaginesynthetase; aspartate-ammonia ligase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00041[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00132[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00013;cpd00018;cpd00041;cpd00067;cpd00132',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) L-Aspartate[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0] + (1) L-Asparagine[0]',\n",
       "  'deltag': -4.43,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.1.1', '6.3.5.4'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00041[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd00132[0]',\n",
       "  'id': 'rxn00340',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29993',\n",
       "  'name': 'L-aspartate:ammonia ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-SYN (L-asparagine Biosynthesis); ASPARAGINESYN-PWY (L-asparagine biosynthesis II); ASPASN-PWY (superpathway of L-aspartate and L-asparagine biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Metabolic-Clusters (Metabolic Clusters); PWY0-1325 (superpathway of L-asparagine biosynthesis)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00460 (Cyanoamino acid metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd00132:0:0:\"L-Asparagine\"'},\n",
       " {'abbreviation': 'R00484',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00484',\n",
       "   'MetaCyc: UREIDOSUCCINASE-RXN',\n",
       "   'Name: N-Carbamoyl-L-aspartate amidohydrolase; N-carbamoyl-L-aspartate amidohydrolase; ureidosuccinase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00343[0] <=> (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00041[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00041;cpd00067;cpd00343',\n",
       "  'definition': '(1) H2O[0] + (2) H+[0] + (1) N-Carbamoyl-L-aspartate[0] => (1) CO2[0] + (1) NH3[0] + (1) L-Aspartate[0]',\n",
       "  'deltag': -7.39,\n",
       "  'deltagerr': 1.2,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.7'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00067[0] + (1) cpd00343[0] => (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00041[0]',\n",
       "  'id': 'rxn00341',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Carbamoyl-L-aspartate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00067:0:0:\"H+\";-1:cpd00343:0:0:\"N-Carbamoyl-L-aspartate\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00041:0:0:\"L-Aspartate\"'},\n",
       " {'abbreviation': 'R00485',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASPARAGHYD-RXN',\n",
       "   'BiGG: ASNN; ASNNe; ASNNpp; ASP1; ASP31; ASP32; ASP33; ASP34',\n",
       "   'BrachyCyc: ASPARAGHYD-RXN',\n",
       "   'KEGG: R00485',\n",
       "   'MetaCyc: ASPARAGHYD-RXN',\n",
       "   'Name: L-Asparagine amidohydrolase; L-asparaginase; L-asparaginase 1; L-asparagine amidohydrolase; alpha-asparaginase; asparaginase; asparaginase II; colaspase; crasnitin; elspar; leunase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00132[0] <=> (1) cpd00013[0] + (1) cpd00041[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00041;cpd00132',\n",
       "  'definition': '(1) H2O[0] + (1) L-Asparagine[0] => (1) NH3[0] + (1) L-Aspartate[0]',\n",
       "  'deltag': -4.01,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.1', '3.5.1.38'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00132[0] => (1) cpd00013[0] + (1) cpd00041[0]',\n",
       "  'id': 'rxn00342',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19208;rxn29950;rxn29992;rxn30353;rxn30618;rxn34558',\n",
       "  'name': 'L-Asparagine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-DEG (L-asparagine Degradation); ASPARAGINE-DEG1-PWY (L-asparagine degradation I); ASPARAGINE-DEG1-PWY-1 (L-asparagine degradation III (mammalian)); ASPASN-PWY (superpathway of L-aspartate and L-asparagine biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Metabolic-Clusters (Metabolic Clusters); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00460 (Cyanoamino acid metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00132:0:0:\"L-Asparagine\";1:cpd00013:0:0:\"NH3\";1:cpd00041:0:0:\"L-Aspartate\"'},\n",
       " {'abbreviation': 'R00486',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.5.5.4-RXN',\n",
       "   'BrachyCyc: 3.5.5.4-RXN',\n",
       "   'CornCyc: 3.5.5.4-RXN',\n",
       "   'KEGG: R00486',\n",
       "   'MetaCyc: 3.5.5.4-RXN',\n",
       "   'Name: 3-Cyano-L-alanine aminohydrolase; 3-cyano-L-alanine aminohydrolase; beta-cyanoalanine nitrilase; cyanoalanine nitrilase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd01651[0] <=> (1) cpd00013[0] + (1) cpd00041[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00041;cpd01651',\n",
       "  'definition': '(2) H2O[0] + (1) L-3-Cyanoalanine[0] => (1) NH3[0] + (1) L-Aspartate[0]',\n",
       "  'deltag': 1.81,\n",
       "  'deltagerr': 5.07,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.5.4'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd01651[0] => (1) cpd00013[0] + (1) cpd00041[0]',\n",
       "  'id': 'rxn00343',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-Cyano-L-alanine aminohydrolase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-SYN (L-asparagine Biosynthesis); ASPSYNII-PWY (cyanide detoxification I); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Cyanide-Detoxification (Cyanide Detoxification); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Other-Degradation (Degradation/Utilization/Assimilation-Other)',\n",
       "   'KEGG: rn00460 (Cyanoamino acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd01651:0:0:\"L-3-Cyanoalanine\";1:cpd00013:0:0:\"NH3\";1:cpd00041:0:0:\"L-Aspartate\"'},\n",
       " {'abbreviation': 'R00487',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00487',\n",
       "   'MetaCyc: ASPARTATE-N-ACETYLTRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:L-aspartate N-acetyltransferase; L-aspartate N-acetyltransferase; acetyl-CoA:L-aspartate N-acetyltransferase; aspartate N-acetyltransferase; aspartate acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00041[0] <=> (1) cpd00010[0] + (1) cpd00767[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00041;cpd00067;cpd00767',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Aspartate[0] => (1) CoA[0] + (1) H+[0] + (1) N-Acetyl-L-aspartate[0]',\n",
       "  'deltag': -4.42,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.17'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00041[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00767[0]',\n",
       "  'id': 'rxn00344',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:L-aspartate N-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd00767:0:0:\"N-Acetyl-L-aspartate\"'},\n",
       " {'abbreviation': 'R00488',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00488',\n",
       "   'MetaCyc: RXN-13680',\n",
       "   'Name: N-Acetyl-L-aspartate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00767[0] <=> (1) cpd00029[0] + (1) cpd00041[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00041;cpd00767',\n",
       "  'definition': '(1) H2O[0] + (1) N-Acetyl-L-aspartate[0] <=> (1) Acetate[0] + (1) L-Aspartate[0]',\n",
       "  'deltag': -2.88,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.15'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00767[0] <=> (1) cpd00029[0] + (1) cpd00041[0]',\n",
       "  'id': 'rxn00345',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Acetyl-L-aspartate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00767:0:0:\"N-Acetyl-L-aspartate\";1:cpd00029:0:0:\"Acetate\";1:cpd00041:0:0:\"L-Aspartate\"'},\n",
       " {'abbreviation': 'R00489',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00489_c',\n",
       "   'BiGG: ASP1DC; U98',\n",
       "   'EcoCyc: ASPDECARBOX-RXN',\n",
       "   'KEGG: R00489',\n",
       "   'MetaCyc: ASPDECARBOX-RXN',\n",
       "   'Name: L-Aspartate 1-carboxy-lyase; L-aspartate 1-carboxy-lyase; L-aspartate 1-carboxy-lyase (beta-alanine-forming); L-aspartate alpha-decarboxylase; aspartate 1-decarboxylase; aspartate alpha-decarboxylase; aspartate1-decarboxylase; aspartic alpha-decarboxylase'],\n",
       "  'code': '(1) cpd00041[0] <=> (1) cpd00011[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00041;cpd00067;cpd00085',\n",
       "  'definition': '(1) L-Aspartate[0] + (1) H+[0] => (1) CO2[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': -4.89,\n",
       "  'deltagerr': 1.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.11', '4.1.1.15'],\n",
       "  'equation': '(1) cpd00041[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00346',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-aspartate 1-carboxy-lyase (beta-alanine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Beta-Alanine-Biosynthesis (&beta; Alanine Biosynthesis); CoA-Biosynthesis (Coenzyme A Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PANTOSYN-PWY (superpathway of coenzyme A biosynthesis I (bacteria)); PWY-5155 (&beta;-alanine biosynthesis III); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00410 (beta-Alanine metabolism); rn00770 (Pantothenate and CoA biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'R00490',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ASPT',\n",
       "   'EcoCyc: ASPARTASE-RXN',\n",
       "   'KEGG: R00490',\n",
       "   'MetaCyc: ASPARTASE-RXN',\n",
       "   'PlantCyc: ASPARTASE-RXN',\n",
       "   'Name: L-Aspartate ammonia-lyase; L-aspartase; L-aspartate ammonia-lyase; L-aspartate ammonia-lyase (fumarate-forming); aspartase; aspartate ammonia-lyase; fumaric aminase'],\n",
       "  'code': '(1) cpd00041[0] <=> (1) cpd00013[0] + (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00041;cpd00106',\n",
       "  'definition': '(1) L-Aspartate[0] <=> (1) NH3[0] + (1) Fumarate[0]',\n",
       "  'deltag': 3.14,\n",
       "  'deltagerr': 0.12,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.38', '4.3.1.1'],\n",
       "  'equation': '(1) cpd00041[0] <=> (1) cpd00013[0] + (1) cpd00106[0]',\n",
       "  'id': 'rxn00347',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29994',\n",
       "  'name': 'L-Aspartate ammonia-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARTATE-SYN (L-aspartate Biosynthesis); ASPASN-PWY (superpathway of L-aspartate and L-asparagine biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); GLUTAMATE-DEG (L-glutamate Degradation); GLUTDEG-PWY (L-glutamate degradation II); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00013:0:0:\"NH3\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R00491',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ASPR; ASPRM',\n",
       "   'KEGG: R00491',\n",
       "   'MetaCyc: ASPARTATE-RACEMASE-RXN',\n",
       "   'iAO358: rll_389',\n",
       "   'iAbaylyiv4: ASPARTATE-RACEMASE-RXN',\n",
       "   'Name: D-aspartate racemase; L-Aspartate racemase; McyF; aspartate racemase'],\n",
       "  'code': '(1) cpd00041[0] <=> (1) cpd00320[0]',\n",
       "  'compound_ids': 'cpd00041;cpd00320',\n",
       "  'definition': '(1) L-Aspartate[0] <=> (1) D-Aspartate[0]',\n",
       "  'deltag': 1.25,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.13'],\n",
       "  'equation': '(1) cpd00041[0] <=> (1) cpd00320[0]',\n",
       "  'id': 'rxn00348',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'aspartate racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";1:cpd00320:0:0:\"D-Aspartate\"'},\n",
       " {'abbreviation': 'rxn00349',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00041[0] <=> (1) cpd03077[0]',\n",
       "  'compound_ids': 'cpd00041;cpd03077',\n",
       "  'definition': '(1) L-Aspartate[0] <=> (1) alpha-Amino acid[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00041[0] <=> (1) cpd03077[0]',\n",
       "  'id': 'rxn00349',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00349',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:C:-2/H:-2/O:-2/R:1|CI:1',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";1:cpd03077:0:0:\"alpha-Amino acid\"'},\n",
       " {'abbreviation': 'R00494',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXNQT-4143',\n",
       "   'BiGG: GGTAe2; GTHRDHpp',\n",
       "   'KEGG: R00494',\n",
       "   'MaizeCyc: RXNQT-4143',\n",
       "   'MetaCyc: RXN-12618',\n",
       "   'Name: glutathione gamma-glutamylaminopeptidase; glutathione hydralase; glutathione hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00042[0] <=> (1) cpd00023[0] + (1) cpd01017[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00023;cpd00042;cpd01017',\n",
       "  'definition': '(1) H2O[0] + (1) GSH[0] <=> (1) L-Glutamate[0] + (1) Cys-Gly[0]',\n",
       "  'deltag': -0.44,\n",
       "  'deltagerr': 0.64,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.2.2', '3.4.19.13'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00042[0] <=> (1) cpd00023[0] + (1) cpd01017[0]',\n",
       "  'id': 'rxn00350',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn23048;rxn28902;rxn29964;rxn31054;rxn33431',\n",
       "  'name': 'glutathione gamma-glutamylaminopeptidase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: COFACTOR-DEGRADATION (Cofactor, Prosthetic Group, Electron Carrier Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-7559 (glutathione degradation (DUG pathway-yeast))',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00042:0:0:\"GSH\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd01017:0:0:\"Cys-Gly\"'},\n",
       " {'abbreviation': 'R00497',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTATHIONE-SYN-RXN',\n",
       "   'BiGG: GSH2; GTHS',\n",
       "   'BrachyCyc: GLUTATHIONE-SYN-RXN',\n",
       "   'KEGG: R00497',\n",
       "   'MetaCyc: GLUTATHIONE-SYN-RXN',\n",
       "   'Name: GSH synthetase; Glutathione synthetase; gamma-L-Glutamyl-L-cysteine:glycine ligase (ADP-forming); gamma-L-glutamyl-L-cysteine:glycine ligase (ADP-forming); glutathione synthase; glutathione synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00033[0] + (1) cpd00506[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00042[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00033;cpd00042;cpd00067;cpd00506',\n",
       "  'definition': '(1) ATP[0] + (1) Glycine[0] + (1) gamma-Glutamylcysteine[0] => (1) ADP[0] + (1) Phosphate[0] + (1) GSH[0] + (1) H+[0]',\n",
       "  'deltag': -6.23,\n",
       "  'deltagerr': 0.63,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00033[0] + (1) cpd00506[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00042[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00351',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27892',\n",
       "  'name': 'gamma-L-glutamyl-L-cysteine:glycine ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); GLUTATHIONESYN-PWY (glutathione biosynthesis); PWY-4041 (&gamma;-glutamyl cycle); Reductants (Reductant Biosynthesis)',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00033:0:0:\"Glycine\";-1:cpd00506:0:0:\"gamma-Glutamylcysteine\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00042:0:0:\"GSH\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00499',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00499',\n",
       "   'MetaCyc: S-SUCCINYLGLUTATHIONE-HYDROLASE-RXN',\n",
       "   'Name: S-Succinylglutathione hydrolase; S-succinylglutathione hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02029[0] <=> (1) cpd00036[0] + (1) cpd00042[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00036;cpd00042;cpd00067;cpd02029',\n",
       "  'definition': '(1) H2O[0] + (1) S-Succinylglutathione[0] => (1) Succinate[0] + (1) GSH[0] + (1) H+[0]',\n",
       "  'deltag': -7.12,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.13'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02029[0] => (1) cpd00036[0] + (1) cpd00042[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00352',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Succinylglutathione hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02029:0:0:\"S-Succinylglutathione\";1:cpd00036:0:0:\"Succinate\";1:cpd00042:0:0:\"GSH\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'rxn00353',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(2) cpd00042[0] <=> (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00042;cpd00067;cpd00111',\n",
       "  'definition': '(2) GSH[0] <=> (2) H+[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': -11.56,\n",
       "  'deltagerr': 0.24,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.4.-'],\n",
       "  'equation': '(2) cpd00042[0] <=> (2) cpd00067[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00353',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00353',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-2:cpd00042:0:0:\"GSH\";2:cpd00067:0:0:\"H+\";1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'R00501',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_UDPGALOR',\n",
       "   'KEGG: R00501',\n",
       "   'Name: UDP-D-galactose:NAD+ 6-oxidoreductase; UDP-alpha-D-galactose:NAD+ 6-oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00043[0] <=> (2) cpd00004[0] + (1) cpd00472[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00043;cpd00067;cpd00472',\n",
       "  'definition': '(1) H2O[0] + (2) NAD[0] + (1) UDP-galactose[0] <=> (2) NADH[0] + (3) H+[0] + (1) UDPgalacturonate[0]',\n",
       "  'deltag': -4.54,\n",
       "  'deltagerr': 0.62,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.-'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00043[0] <=> (2) cpd00004[0] + (3) cpd00067[0] + (1) cpd00472[0]',\n",
       "  'id': 'rxn00354',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-D-galactose:NAD+ 6-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00003:0:0:\"NAD\";-1:cpd00043:0:0:\"UDP-galactose\";2:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";1:cpd00472:0:0:\"UDPgalacturonate\"'},\n",
       " {'abbreviation': 'R00502',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UTPHEXPURIDYLYLTRANS-RXN',\n",
       "   'BiGG: GAL7_1; GALT; GALTi',\n",
       "   'BrachyCyc: UTPHEXPURIDYLYLTRANS-RXN',\n",
       "   'KEGG: R00502',\n",
       "   'MetaCyc: UTPHEXPURIDYLYLTRANS-RXN',\n",
       "   'Name: Galactose-1-phosphate uridylyltransferase; UDPgalactose pyrophosphorylase; UTP--hexose-1-phosphate uridylyltransferase; UTP-hexose-1-phosphate uridylyltransferase; UTP:alpha-D-galactose-1-phosphate uridylyltransferase; UTP:alpha-D-hexose-1-phosphate uridylyltransferase; alpha-D-galactose 1-phosphate uridylyltransferase; galactose 1-phosphate uridyltransferase; galactose 1-phosphate uridylyltransferase; galactose-1-phosphate uridylyltransferase; galactose-1-phosphateuridylyltransferase-irreversible; uridine diphosphate galactose pyrophosphorylase; uridine diphosphogalactose pyrophosphorylase'],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd00348[0] <=> (1) cpd00012[0] + (1) cpd00043[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00043;cpd00062;cpd00348',\n",
       "  'definition': '(1) UTP[0] + (1) D-Galactose 1-phosphate[0] <=> (1) PPi[0] + (1) UDP-galactose[0]',\n",
       "  'deltag': 0.03,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.10'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd00348[0] <=> (1) cpd00012[0] + (1) cpd00043[0]',\n",
       "  'id': 'rxn00355',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29945',\n",
       "  'name': 'UTP:alpha-D-galactose-1-phosphate uridylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); PWY-3821 (D-galactose detoxification); PWY-6527 (stachyose degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00052 (Galactose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd00348:0:0:\"D-Galactose 1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00043:0:0:\"UDP-galactose\"'},\n",
       " {'abbreviation': 'R00503',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00503; R06067',\n",
       "   'Name: UDP-alpha-D-galactose:D-glucose 4-beta-D-galactosyltransferase; UDP-galactose:D-glucose 4-beta-D-galactosyltransferase; UDP-galactose:D-glucose 4-beta-D-galactotransferase; UDPgalactose:D-glucose 4-beta-D-galactosyltransferase'],\n",
       "  'code': '(1) cpd00027[0] + (1) cpd00043[0] <=> (1) cpd00014[0] + (1) cpd00208[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00027;cpd00043;cpd00208',\n",
       "  'definition': '(1) D-Glucose[0] + (1) UDP-galactose[0] <=> (1) UDP[0] + (1) LACT[0]',\n",
       "  'deltag': -2.65,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.22'],\n",
       "  'equation': '(1) cpd00027[0] + (1) cpd00043[0] <=> (1) cpd00014[0] + (1) cpd00208[0]',\n",
       "  'id': 'rxn00356',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-galactose:D-glucose 4-beta-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00027:0:0:\"D-Glucose\";-1:cpd00043:0:0:\"UDP-galactose\";1:cpd00014:0:0:\"UDP\";1:cpd00208:0:0:\"LACT\"'},\n",
       " {'abbreviation': 'R00504',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00504',\n",
       "   'MetaCyc: 2.7.8.18-RXN',\n",
       "   'Name: UDP-alpha-D-galactose:UDP-N-acetyl-alpha-D-glucosamine galactose phosphotransferase; UDP-galactose-UDP-N-acetylglucosamine galactose phosphotransferase; UDP-galactose:UDP-N-acetyl-D-glucosamine galactose phosphotransferase; UDPgalactose:UDP-N-acetyl-D-glucosamine galactosephosphotransferase; galactose-1-phosphotransferase; galactosyl phosphotransferase; uridine diphosphogalactose-uridine diphosphoacetylglucosamine galactose-1-phosphotransferase'],\n",
       "  'code': '(1) cpd00037[0] + (1) cpd00043[0] <=> (1) cpd00091[0] + (1) cpd02887[0]',\n",
       "  'compound_ids': 'cpd00037;cpd00043;cpd00067;cpd00091;cpd02887',\n",
       "  'definition': '(1) UDP-N-acetylglucosamine[0] + (1) UDP-galactose[0] => (1) H+[0] + (1) UMP[0] + (1) UDP-N-acetyl-6-(D-galactose-1-phospho)-D-glucosamine[0]',\n",
       "  'deltag': -14.03,\n",
       "  'deltagerr': 6.7,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.8.18'],\n",
       "  'equation': '(1) cpd00037[0] + (1) cpd00043[0] => (1) cpd00067[0] + (1) cpd00091[0] + (1) cpd02887[0]',\n",
       "  'id': 'rxn00357',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-galactose:UDP-N-acetyl-D-glucosamine galactose phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";-1:cpd00043:0:0:\"UDP-galactose\";1:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\";1:cpd02887:0:0:\"UDP-N-acetyl-6-(D-galactose-1-phospho)-D-glucosamine\"'},\n",
       " {'abbreviation': 'R00505',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: UDPGALM',\n",
       "   'EcoCyc: GALPMUT-RXN',\n",
       "   'KEGG: R00505',\n",
       "   'MetaCyc: GALPMUT-RXN',\n",
       "   'RiceCyc: GALPMUT-RXN',\n",
       "   'Name: UDP-D-galactopyranose furanomutase; UDP-alpha-D-galactopyranose furanomutase; UDP-galactopyranose mutase; UDPgalactopyranose mutase'],\n",
       "  'code': '(1) cpd00043[0] <=> (1) cpd02338[0]',\n",
       "  'compound_ids': 'cpd00043;cpd02338',\n",
       "  'definition': '(1) UDP-galactose[0] <=> (1) UDP-D-galacto-1,4-furanose[0]',\n",
       "  'deltag': 0.59,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.99.9'],\n",
       "  'equation': '(1) cpd00043[0] <=> (1) cpd02338[0]',\n",
       "  'id': 'rxn00358',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-D-galactopyranose furanomutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Metabolic-Clusters (Metabolic Clusters); OANTIGEN-PWY (<i>O</i>-antigen building blocks biosynthesis (<i>E. coli</i>)); PWY-6397 (mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY-6404 (superpathway of mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY-7328 (superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis); PWY-7622 (UDP-&alpha;-D-galactofuranose biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00043:0:0:\"UDP-galactose\";1:cpd02338:0:0:\"UDP-D-galacto-1,4-furanose\"'},\n",
       " {'abbreviation': 'R00507',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00507_c',\n",
       "   'AraGEM: R_R00507_c',\n",
       "   'KEGG: R00507',\n",
       "   'MetaCyc: PHOSPHOADENYLYLSULFATASE-RXN',\n",
       "   'TS_Athaliana: rxn579_p',\n",
       "   \"Name: 3'-Phosphoadenylylsulfate nucleotidohydrolase; 3'-phosphoadenylyl-sulfate nucleotidohydrolase; 3'-phosphoadenylyl-sulfate sulfohydrolase; 3'-phosphoadenylyl-sulfate sulfohydrolase;; 3'-phosphoadenylylsulfate sulfohydrolase; 3-phosphoadenosine 5-phosphosulfate sulfatase; 3-phosphoadenylyl sulfatase; PAPS sulfatase; phosphoadenylylsulfatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00044[0] <=> (1) cpd00045[0] + (1) cpd00048[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00044;cpd00045;cpd00048;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) 3-phosphoadenylylsulfate[0] => (1) Adenosine 3-5-bisphosphate[0] + (1) Sulfate[0] + (2) H+[0]',\n",
       "  'deltag': -19.97,\n",
       "  'deltagerr': 1.38,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.9', '3.6.2.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00044[0] => (1) cpd00045[0] + (1) cpd00048[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00359',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32779',\n",
       "  'name': \"3'-phosphoadenylyl-sulfate sulfohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism); rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";1:cpd00045:0:0:\"Adenosine 3-5-bisphosphate\";1:cpd00048:0:0:\"Sulfate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00508',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00508_c',\n",
       "   'AraGEM: R_R00508_c',\n",
       "   'JP_Creinhardtii_MSB: R_BPNT2',\n",
       "   'KEGG: R00508',\n",
       "   'MetaCyc: R163-RXN',\n",
       "   \"Name: 3'-Phospho-5'-adenylyl sulfate 3'(2')-phosphohydrolase; 3'-Phospho-5'-adenylyl sulfate 3'-phosphohydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00044[0] <=> (1) cpd00009[0] + (1) cpd00193[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00044;cpd00193',\n",
       "  'definition': '(1) H2O[0] + (1) 3-phosphoadenylylsulfate[0] <=> (1) Phosphate[0] + (1) APS[0]',\n",
       "  'deltag': -2.75,\n",
       "  'deltagerr': 1.42,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00044[0] <=> (1) cpd00009[0] + (1) cpd00193[0]',\n",
       "  'id': 'rxn00360',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33222',\n",
       "  'name': \"3'-Phospho-5'-adenylyl sulfate 3'-phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00193:0:0:\"APS\"'},\n",
       " {'abbreviation': 'R00509',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENYLYLSULFKIN-RXN',\n",
       "   'BiGG: ADSK; ADSKr; MET14',\n",
       "   'BrachyCyc: ADENYLYLSULFKIN-RXN',\n",
       "   'KEGG: R00509',\n",
       "   'MetaCyc: ADENYLYLSULFKIN-RXN',\n",
       "   \"Name: 5'-phosphoadenosine sulfate kinase; APS kinase; ATP:adenylyl-sulfate 3'-phosphotransferase; ATP:adenylylsulfate 3'-phosphotransferase; Adenosine-5'phosphosulfate kinase; Adenylylsulfate kinase; adenosine 5'-phosphosulfate kinase; adenosine phosphosulfate kinase; adenosine phosphosulfokinase; adenosine-5'-phosphosulfate-3'-phosphokinase; adenylyl-sulfate kinase; adenylyl-sulfatekinase; adenylylsulfate kinase (phosphorylating)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00193[0] <=> (1) cpd00008[0] + (1) cpd00044[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00044;cpd00067;cpd00193',\n",
       "  'definition': '(1) ATP[0] + (1) APS[0] => (1) ADP[0] + (1) 3-phosphoadenylylsulfate[0] + (1) H+[0]',\n",
       "  'deltag': -3.41,\n",
       "  'deltagerr': 1.42,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.25'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00193[0] => (1) cpd00008[0] + (1) cpd00044[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00361',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19075;rxn27688',\n",
       "  'name': \"ATP:adenylylsulfate 3'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation (Activation); Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Assimilatory-Sulfate-Reduction (Assimilatory Sulfate Reduction); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5340 (sulfate activation for sulfonation); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-821 (superpathway of sulfur amino acid biosynthesis (<i>Saccharomyces cerevisiae</i>)); SO4ASSIM-PWY (assimilatory sulfate reduction I); SULFATE-CYS-PWY (superpathway of sulfate assimilation and cysteine biosynthesis); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00193:0:0:\"APS\";1:cpd00008:0:0:\"ADP\";1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00510',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00510',\n",
       "   'MetaCyc: RXN-14065',\n",
       "   \"Name: Cytidine-5'-monophosphate phosphoribo(deoxyribo)hydrolase; Cytidine-5'-monophosphate phosphoribohydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00046[0] <=> (1) cpd00101[0] + (1) cpd00307[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00046;cpd00101;cpd00307',\n",
       "  'definition': '(1) H2O[0] + (1) CMP[0] <=> (1) ribose-5-phosphate[0] + (1) Cytosine[0]',\n",
       "  'deltag': -3.11,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00046[0] <=> (1) cpd00101[0] + (1) cpd00307[0]',\n",
       "  'id': 'rxn00362',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"Cytidine-5'-monophosphate phosphoribohydrolase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); PWY-7569 (arginomycin biosynthesis); PWY-7570 (blasticidin S biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00046:0:0:\"CMP\";1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00307:0:0:\"Cytosine\"'},\n",
       " {'abbreviation': 'R00511',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5-NUCLEOTID-RXN',\n",
       "   'BiGG: NTD4; NTD4pp; U29',\n",
       "   'ChlamyCyc: 5-NUCLEOTID-RXN',\n",
       "   'KEGG: R00511',\n",
       "   'MetaCyc: 5-NUCLEOTID-RXN; RXN-14026',\n",
       "   \"Name: 5'-AMP nucleotidase; 5'-AMPase; 5'-adenylic phosphatase; 5'-mononucleotidase; 5'-nucleotidase; 5'-nucleotidase (CMP); 5'-ribonucleotide phosphohydrolase; AMP phosphatase; AMP phosphohydrolase; AMPase; Cytidine-5'-monophosphate phosphohydrolase; IMP 5'-nucleotidase; UMPase; adenosine 5'-phosphatase; adenosine monophosphatase; cytidine-5'-monophosphate phosphohydrolase; snake venom 5'-nucleotidase; thimidine monophosphate nucleotidase; uridine 5'-nucleotidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00046[0] <=> (1) cpd00009[0] + (1) cpd00367[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00046;cpd00367',\n",
       "  'definition': '(1) H2O[0] + (1) CMP[0] <=> (1) Phosphate[0] + (1) Cytidine[0]',\n",
       "  'deltag': -3.34,\n",
       "  'deltagerr': 0.67,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.5', '3.1.3.91'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00046[0] <=> (1) cpd00009[0] + (1) cpd00367[0]',\n",
       "  'id': 'rxn00363',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18972;rxn32982',\n",
       "  'name': \"Cytidine-5'-monophosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-7185 (UTP and CTP dephosphorylation I); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation); UTP-CTP-Dephosphorylation (UTP and CTP Dephosphorylation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00046:0:0:\"CMP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00367:0:0:\"Cytidine\"'},\n",
       " {'abbreviation': 'R00512',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CMPKI-RXN; RXN-11832',\n",
       "   'BiGG: CYTK1; U18',\n",
       "   'BrachyCyc: CMPKI-RXN',\n",
       "   'KEGG: R00512',\n",
       "   'MetaCyc: CMPKI-RXN; RXN-11832',\n",
       "   'Name: (d)CMP kinase; ATP:(d)CMP phosphotransferase; ATP:CMP phosphotransferase; Cytidylate kinase; Deoxycytidylate kinase; cmp kinase; cytidine monophosphate kinase; cytidylate kinase (CMP); cytidylatekinase-CMP; dCMP kinase; deoxycytidine monophosphokinase; deoxycytidylate kinase; prokaryotic cytidylate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00046[0] <=> (1) cpd00008[0] + (1) cpd00096[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00046;cpd00067;cpd00096',\n",
       "  'definition': '(1) ATP[0] + (1) CMP[0] + (1) H+[0] <=> (1) ADP[0] + (1) CDP[0]',\n",
       "  'deltag': -0.58,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.14', '2.7.4.25'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00046[0] + (1) cpd00067[0] <=> (1) cpd00008[0] + (1) cpd00096[0]',\n",
       "  'id': 'rxn00364',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27766',\n",
       "  'name': 'ATP:CMP phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PWY-7205 (CMP phosphorylation); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00046:0:0:\"CMP\";-1:cpd00067:0:0:\"H+\";1:cpd00008:0:0:\"ADP\";1:cpd00096:0:0:\"CDP\"'},\n",
       " {'abbreviation': 'R00513',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CYTDK1',\n",
       "   'KEGG: R00513',\n",
       "   'MetaCyc: CYTIKIN-RXN',\n",
       "   'RiceCyc: CYTIKIN-RXN',\n",
       "   'SorghumCyc: CYTIKIN-RXN',\n",
       "   \"Name: ATP:cytidine 5'-phosphotransferase; cytidine kinase (ATP)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00367[0] <=> (1) cpd00008[0] + (1) cpd00046[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00046;cpd00067;cpd00367',\n",
       "  'definition': '(1) ATP[0] + (1) Cytidine[0] => (1) ADP[0] + (1) CMP[0] + (1) H+[0]',\n",
       "  'deltag': -2.82,\n",
       "  'deltagerr': 0.67,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.-', '2.7.1.213', '2.7.1.48'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00367[0] => (1) cpd00008[0] + (1) cpd00046[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00365',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"ATP:cytidine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-5532 (nucleoside and nucleotide degradation (archaea)); PWY-7193 (pyrimidine ribonucleosides salvage I); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00367:0:0:\"Cytidine\";1:cpd00008:0:0:\"ADP\";1:cpd00046:0:0:\"CMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00514',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-12198',\n",
       "   'CornCyc: RXN-12198',\n",
       "   'EcoCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-12198',\n",
       "   'KEGG: R00514',\n",
       "   'MetaCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-12198',\n",
       "   'Name: CDP diphosphohydrolase; CDP phosphohydrolase; nucleoside-diphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00096[0] <=> (1) cpd00009[0] + (1) cpd00046[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00046;cpd00067;cpd00096',\n",
       "  'definition': '(1) H2O[0] + (1) CDP[0] => (1) Phosphate[0] + (1) CMP[0] + (2) H+[0]',\n",
       "  'deltag': -5.58,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.5', '3.6.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00096[0] => (1) cpd00009[0] + (1) cpd00046[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00366',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CDP phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00096:0:0:\"CDP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00046:0:0:\"CMP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00515',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.1.19-RXN',\n",
       "   'BiGG: NTPP4',\n",
       "   'BrachyCyc: 3.6.1.19-RXN',\n",
       "   'KEGG: R00515',\n",
       "   'MetaCyc: 3.6.1.19-RXN; RXN0-383',\n",
       "   'Name: CTP diphosphohydrolase (diphosphate-forming); CTP pyrophosphohydrolase; Nucleoside triphosphate pyrophosphorylase (ctp); ctp pyrophosphorylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00052[0] <=> (1) cpd00012[0] + (1) cpd00046[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00046;cpd00052;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) CTP[0] => (1) PPi[0] + (1) CMP[0] + (2) H+[0]',\n",
       "  'deltag': -8.0,\n",
       "  'deltagerr': 0.56,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.-', '3.6.1.65', '3.6.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00052[0] => (1) cpd00012[0] + (1) cpd00046[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00367',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CTP diphosphohydrolase (diphosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00052:0:0:\"CTP\";1:cpd00012:0:0:\"PPi\";1:cpd00046:0:0:\"CMP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00516',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00516_c',\n",
       "   'AraGEM: R_R00516_c',\n",
       "   'JP_Creinhardtii_MSB: R_UTCY',\n",
       "   'KEGG: R00516',\n",
       "   'iRS1597: R00516_c',\n",
       "   \"Name: UTP:cytidine 5'-phosphotransferase\"],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd00367[0] <=> (1) cpd00014[0] + (1) cpd00046[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00046;cpd00062;cpd00067;cpd00367',\n",
       "  'definition': '(1) UTP[0] + (1) Cytidine[0] => (1) UDP[0] + (1) CMP[0] + (1) H+[0]',\n",
       "  'deltag': -2.15,\n",
       "  'deltagerr': 0.74,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.48'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd00367[0] => (1) cpd00014[0] + (1) cpd00046[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00368',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"UTP:cytidine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd00367:0:0:\"Cytidine\";1:cpd00014:0:0:\"UDP\";1:cpd00046:0:0:\"CMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00517',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CYTIDINEKIN-RXN',\n",
       "   'BiGG: CYTDK2; URK1_2',\n",
       "   'BrachyCyc: CYTIDINEKIN-RXN',\n",
       "   'KEGG: R00517',\n",
       "   'MetaCyc: CYTIDINEKIN-RXN',\n",
       "   \"Name: GTP:cytidine 5'-phosphotransferase; cytidine kinase (GTP)\"],\n",
       "  'code': '(1) cpd00038[0] + (1) cpd00367[0] <=> (1) cpd00031[0] + (1) cpd00046[0]',\n",
       "  'compound_ids': 'cpd00031;cpd00038;cpd00046;cpd00067;cpd00367',\n",
       "  'definition': '(1) GTP[0] + (1) Cytidine[0] => (1) GDP[0] + (1) CMP[0] + (1) H+[0]',\n",
       "  'deltag': -2.22,\n",
       "  'deltagerr': 0.74,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.-', '2.7.1.48'],\n",
       "  'equation': '(1) cpd00038[0] + (1) cpd00367[0] => (1) cpd00031[0] + (1) cpd00046[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00369',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"GTP:cytidine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00038:0:0:\"GTP\";-1:cpd00367:0:0:\"Cytidine\";1:cpd00031:0:0:\"GDP\";1:cpd00046:0:0:\"CMP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00518',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00518',\n",
       "   'MetaCyc: FORMATE-KINASE-RXN',\n",
       "   'Name: ATP:formate phosphotransferase; formate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00047[0] <=> (1) cpd00008[0] + (1) cpd01598[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00047;cpd01598',\n",
       "  'definition': '(1) ATP[0] + (1) Formate[0] <=> (1) ADP[0] + (1) Formyl phosphate[0]',\n",
       "  'deltag': -4.22,\n",
       "  'deltagerr': 1.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00047[0] <=> (1) cpd00008[0] + (1) cpd01598[0]',\n",
       "  'id': 'rxn00370',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:formate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00047:0:0:\"Formate\";1:cpd00008:0:0:\"ADP\";1:cpd01598:0:0:\"Formyl phosphate\"'},\n",
       " {'abbreviation': 'R00519',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.2.1.2-RXN',\n",
       "   'BiGG: FDH; FDH1; FDHr; YPL275W; YPL276W',\n",
       "   'BrachyCyc: 1.2.1.2-RXN',\n",
       "   'KEGG: R00519',\n",
       "   'MetaCyc: 1.2.1.2-RXN',\n",
       "   'Name: FDH I; FDH II; Formate dehydrogenase 1; Formate:NAD+ oxidoreductase; N-FDH; NAD+ oxidoreductase; NAD-dependent formate dehydrogenase; NAD-formate dehydrogenase; NAD-linked formate dehydrogenase; formate benzyl-viologen oxidoreductase; formate dehydrogenase; formate dehydrogenase (FDH); formate dehydrogenase (NAD); formate hydrogenlyase; formate-NAD oxidoreductase; formate:NAD+ oxidoreductase; formic acid dehydrogenase; formic hydrogen-lyase; hydrogenlyase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00047[0] <=> (1) cpd00004[0] + (1) cpd00011[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00047',\n",
       "  'definition': '(1) NAD[0] + (1) Formate[0] => (1) NADH[0] + (1) CO2[0]',\n",
       "  'deltag': -3.53,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.17.1.9', '1.2.1.2'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00047[0] => (1) cpd00004[0] + (1) cpd00011[0]',\n",
       "  'id': 'rxn00371',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31215;rxn33324;rxn33328',\n",
       "  'name': 'Formate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CHEMOAUTOTROPHIC-ENERGY-METABOLISM (Chemoautotrophic Energy Metabolism); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Oxalate-Degradation (Oxalate Degradation); P164-PWY (purine nucleobases degradation I (anaerobic)); PWY-1881 (formate oxidation to CO<sub>2</sub>); PWY-1882 (superpathway of C1 compounds oxidation to CO<sub>2</sub>); PWY-5497 (purine nucleobases degradation II (anaerobic)); PWY-6696 (oxalate degradation III); PWY-7985 (oxalate degradation VI); Purine-Degradation (Purine Nucleotide Degradation)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00047:0:0:\"Formate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\"'},\n",
       " {'abbreviation': 'R00521',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: FCOAH2',\n",
       "   'KEGG: R00521',\n",
       "   'MetaCyc: FORMYL-COA-HYDROLASE-RXN',\n",
       "   'iMEO21: FCOAH2',\n",
       "   'iNJ661: FCOAH2',\n",
       "   'Name: Formyl-CoA hydrolase; formyl coa hydrolase; formyl coenzyme A hydrolase; formyl-CoA hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00592[0] <=> (1) cpd00010[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00047;cpd00067;cpd00592',\n",
       "  'definition': '(1) H2O[0] + (1) Formyl-CoA[0] => (1) CoA[0] + (1) Formate[0] + (1) H+[0]',\n",
       "  'deltag': -0.79,\n",
       "  'deltagerr': 1.38,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00592[0] => (1) cpd00010[0] + (1) cpd00047[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00372',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Formyl-CoA hydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Fatty-Acid-Degradation (Fatty Acid Degradation); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); Oxalate-Degradation (Oxalate Degradation); PWY-6696 (oxalate degradation III); PWY-7985 (oxalate degradation VI); PWY66-387 (fatty acid &alpha;-oxidation II); PWY66-388 (fatty acid &alpha;-oxidation III)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00592:0:0:\"Formyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00047:0:0:\"Formate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00522',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: OXADC',\n",
       "   'ChlamyCyc: OXALATE-DECARBOXYLASE-RXN',\n",
       "   'JP_Creinhardtii_MSB: R_OXADCm',\n",
       "   'KEGG: R00522',\n",
       "   'MetaCyc: OXALATE-DECARBOXYLASE-RXN',\n",
       "   'Name: Oxalate carboxy-lyase; Oxalate decarboxylase; oxalate carboxy-lyase; oxalate carboxy-lyase (formate-forming); oxalate decarboxylase'],\n",
       "  'code': '(1) cpd00180[0] <=> (1) cpd00011[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00047;cpd00067;cpd00180',\n",
       "  'definition': '(1) H+[0] + (1) Oxalate[0] => (1) CO2[0] + (1) Formate[0]',\n",
       "  'deltag': -3.9,\n",
       "  'deltagerr': 1.06,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.2'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00180[0] => (1) cpd00011[0] + (1) cpd00047[0]',\n",
       "  'id': 'rxn00373',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34248',\n",
       "  'name': 'Oxalate carboxy-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Oxalate-Degradation (Oxalate Degradation); PWY-6698 (oxalate degradation V)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00180:0:0:\"Oxalate\";1:cpd00011:0:0:\"CO2\";1:cpd00047:0:0:\"Formate\"'},\n",
       " {'abbreviation': 'R00524',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FORMAMIDASE-RXN',\n",
       "   'BiGG: FORAMD',\n",
       "   'ChlamyCyc: FORMAMIDASE-RXN',\n",
       "   'KEGG: R00524',\n",
       "   'MetaCyc: FORMAMIDASE-RXN',\n",
       "   'Name: Formamidase; Formamide amidohydrolase; formamide amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00378[0] <=> (1) cpd00013[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00047;cpd00378',\n",
       "  'definition': '(1) H2O[0] + (1) Formamide[0] => (1) NH3[0] + (1) Formate[0]',\n",
       "  'deltag': -6.42,\n",
       "  'deltagerr': 1.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.49'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00378[0] => (1) cpd00013[0] + (1) cpd00047[0]',\n",
       "  'id': 'rxn00374',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30669;rxn33524;rxn34213;rxn35245;rxn35315',\n",
       "  'name': 'Formamide amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cyanide-Detoxification (Cyanide Detoxification); Detoxification (Detoxification); PWY-7142 (cyanide detoxification II)',\n",
       "   'KEGG: rn00460 (Cyanoamino acid metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00910 (Nitrogen metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00378:0:0:\"Formamide\";1:cpd00013:0:0:\"NH3\";1:cpd00047:0:0:\"Formate\"'},\n",
       " {'abbreviation': 'R00525',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NFORGLUAH',\n",
       "   'KEGG: R00525',\n",
       "   'iJN746: NFORGLUAH',\n",
       "   'Name: N-Formyl-L-glutamate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00770[0] <=> (1) cpd00023[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00023;cpd00047;cpd00770',\n",
       "  'definition': '(1) H2O[0] + (1) N-Formyl-L-glutamate[0] <=> (1) L-Glutamate[0] + (1) Formate[0]',\n",
       "  'deltag': -7.21,\n",
       "  'deltagerr': 1.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.68'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00770[0] <=> (1) cpd00023[0] + (1) cpd00047[0]',\n",
       "  'id': 'rxn00375',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Formyl-L-glutamate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00340 (Histidine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00770:0:0:\"N-Formyl-L-glutamate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00047:0:0:\"Formate\"'},\n",
       " {'abbreviation': 'R00526',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00526',\n",
       "   'MetaCyc: FORMYLASPARTATE-DEFORMYLASE-RXN',\n",
       "   'TS_Athaliana: rxn582_c; rxn582_m; rxn582_p; rxn582_x',\n",
       "   'iRS1563: R_R00526_c',\n",
       "   'Name: N-Formyl-L-aspartate amidohydrolase; N-formyl-L-aspartate amidohydrolase; formylaspartate deformylase; formylaspartic formylase (formylase I, formylase II)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00769[0] <=> (1) cpd00041[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00041;cpd00047;cpd00769',\n",
       "  'definition': '(1) H2O[0] + (1) N-Formyl-L-aspartate[0] <=> (1) L-Aspartate[0] + (1) Formate[0]',\n",
       "  'deltag': -6.8,\n",
       "  'deltagerr': 1.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.15', '3.5.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00769[0] <=> (1) cpd00041[0] + (1) cpd00047[0]',\n",
       "  'id': 'rxn00376',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32730;rxn32731;rxn32757',\n",
       "  'name': 'N-Formyl-L-aspartate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00340 (Histidine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00769:0:0:\"N-Formyl-L-aspartate\";1:cpd00041:0:0:\"L-Aspartate\";1:cpd00047:0:0:\"Formate\"'},\n",
       " {'abbreviation': 'R00527',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: S-FORMYLGLUTATHIONE-HYDROLASE-RXN',\n",
       "   'BiGG: SFGTH; SFGTHi; YJL068C',\n",
       "   'BrachyCyc: S-FORMYLGLUTATHIONE-HYDROLASE-RXN',\n",
       "   'KEGG: R00527',\n",
       "   'MetaCyc: S-FORMYLGLUTATHIONE-HYDROLASE-RXN',\n",
       "   'Name: S-Formylglutathione hydralase; S-Formylglutathione hydrolase; S-formylglutathione hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00759[0] <=> (1) cpd00042[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00042;cpd00047;cpd00067;cpd00759',\n",
       "  'definition': '(1) H2O[0] + (1) S-Formylglutathione[0] => (1) GSH[0] + (1) Formate[0] + (1) H+[0]',\n",
       "  'deltag': -0.64,\n",
       "  'deltagerr': 1.05,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.12'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00759[0] => (1) cpd00042[0] + (1) cpd00047[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00377',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32820;rxn38706',\n",
       "  'name': 'S-Formylglutathione hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Formaldehyde-Oxidation (Formaldehyde Oxidation); PWY-1801 (formaldehyde oxidation II (glutathione-dependent)); PWY-1882 (superpathway of C1 compounds oxidation to CO<sub>2</sub>)',\n",
       "   'KEGG: rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00759:0:0:\"S-Formylglutathione\";1:cpd00042:0:0:\"GSH\";1:cpd00047:0:0:\"Formate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00528',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00528',\n",
       "   'Name: Sulfite:ferricytochrome-c oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00081[0] + (2) cpd00109[0] <=> (1) cpd00048[0] + (2) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00048;cpd00067;cpd00081;cpd00109;cpd00110',\n",
       "  'definition': '(1) H2O[0] + (1) Sulfite[0] + (2) Cytochrome c3+[0] => (1) Sulfate[0] + (3) H+[0] + (2) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.8.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00081[0] + (2) cpd00109[0] => (1) cpd00048[0] + (3) cpd00067[0] + (2) cpd00110[0]',\n",
       "  'id': 'rxn00378',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Sulfite:ferricytochrome-c oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00081:0:0:\"Sulfite\";-2:cpd00109:0:0:\"Cytochrome c3+\";1:cpd00048:0:0:\"Sulfate\";3:cpd00067:0:0:\"H+\";2:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'R00529',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SULFATE-ADENYLYLTRANS-RXN',\n",
       "   'BiGG: MET3; SADT; SADT1r',\n",
       "   'BrachyCyc: SULFATE-ADENYLYLTRANS-RXN',\n",
       "   'KEGG: R00529',\n",
       "   'MetaCyc: SULFATE-ADENYLYLTRANS-RXN',\n",
       "   \"Name: ATP sulfurylase; ATP-sulfurylase; ATP:sulfate adenylyltransferase; Sulfate adenylyltransferase; adenosine-5'-triphosphate sulfurylase; adenosinetriphosphate sulfurylase; adenylylsulfate pyrophosphorylase; sulfate adenylyltransferase; sulfurylase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00048[0] <=> (1) cpd00012[0] + (1) cpd00193[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00048;cpd00193',\n",
       "  'definition': '(1) ATP[0] + (1) Sulfate[0] <=> (1) PPi[0] + (1) APS[0]',\n",
       "  'deltag': 11.38,\n",
       "  'deltagerr': 0.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.4'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00048[0] <=> (1) cpd00012[0] + (1) cpd00193[0]',\n",
       "  'id': 'rxn00379',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27259;rxn33490',\n",
       "  'name': 'ATP:sulfate adenylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROBIC-RESPIRATION (Anaerobic Respiration); Activation (Activation); Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Assimilatory-Sulfate-Reduction (Assimilatory Sulfate Reduction); CHEMOAUTOTROPHIC-ENERGY-METABOLISM (Chemoautotrophic Energy Metabolism); DISSULFRED-PWY (dissimilatory sulfate reduction I (to hydrogen sufide))); Degradation (Degradation/Utilization/Assimilation); Dissimilatory-Sulfate-Reduction (Dissimilatory Sulfate Reduction); Energy-Metabolism (Generation of Precursor Metabolite and Energy); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); P224-PWY (dissimilatory sulfate reduction II (to thiosulfate)); PWY-5278 (sulfite oxidation III); PWY-5306 (superpathway of thiosulfate metabolism (<i>Desulfovibrio sulfodismutans</i>)); PWY-5308 (superpathway of sulfur metabolism (<i>Desulfocapsa sulfoexigens</i>)); PWY-5340 (sulfate activation for sulfonation); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-6676 (superpathway of sulfide oxidation (phototrophic sulfur bacteria)); PWY-6683 (assimilatory sulfate reduction III); PWY-821 (superpathway of sulfur amino acid biosynthesis (<i>Saccharomyces cerevisiae</i>)); Respiration (); SO4ASSIM-PWY (assimilatory sulfate reduction I); SULFATE-CYS-PWY (superpathway of sulfate assimilation and cysteine biosynthesis); SULFMETII-PWY (assimilatory sulfate reduction II); Sulfide-Oxidation (Sulfide Oxidation); Sulfite-Oxidation (Sulfite Oxidation); Sulfur-Disproportionation (Sulfur Disproportionation); Sulfur-Metabolism (Sulfur Compound Metabolism); Thiosulfate-Disproportionation (Thiosulfate Disproportionation)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00048:0:0:\"Sulfate\";1:cpd00012:0:0:\"PPi\";1:cpd00193:0:0:\"APS\"'},\n",
       " {'abbreviation': 'R00530',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SULFATE-ADENYLYLTRANSFERASE-ADP-RXN',\n",
       "   'BiGG: APA1_3; SLFAT',\n",
       "   'CornCyc: SULFATE-ADENYLYLTRANSFERASE-ADP-RXN',\n",
       "   'KEGG: R00530',\n",
       "   'MetaCyc: SULFATE-ADENYLYLTRANSFERASE-ADP-RXN',\n",
       "   'Name: ADP-sulfurylase; ADP:sulfate adenylyltransferase; adenosine diphosphate sulfurylase; sulfate (adenosine diphosphate) adenylyltransferase; sulfate adenylyltransferase (ADP)'],\n",
       "  'code': '(1) cpd00008[0] + (1) cpd00048[0] <=> (1) cpd00009[0] + (1) cpd00193[0]',\n",
       "  'compound_ids': 'cpd00008;cpd00009;cpd00048;cpd00193',\n",
       "  'definition': '(1) ADP[0] + (1) Sulfate[0] <= (1) Phosphate[0] + (1) APS[0]',\n",
       "  'deltag': 14.07,\n",
       "  'deltagerr': 0.29,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.7.7.5'],\n",
       "  'equation': '(1) cpd00008[0] + (1) cpd00048[0] <= (1) cpd00009[0] + (1) cpd00193[0]',\n",
       "  'id': 'rxn00380',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27260',\n",
       "  'name': 'ADP:sulfate adenylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5279 (sulfite oxidation II); PWY-5304 (superpathway of sulfur oxidation (<i>Acidianus ambivalens</i>)); Sulfite-Oxidation (Sulfite Oxidation); Sulfur-Metabolism (Sulfur Compound Metabolism); Sulfur-Oxidation (Sulfur Oxidation)',\n",
       "   'KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00008:0:0:\"ADP\";-1:cpd00048:0:0:\"Sulfate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00193:0:0:\"APS\"'},\n",
       " {'abbreviation': 'R00531',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENYLYLSULFATASE-RXN',\n",
       "   'BrachyCyc: ADENYLYLSULFATASE-RXN',\n",
       "   'CornCyc: ADENYLYLSULFATASE-RXN',\n",
       "   'KEGG: R00531',\n",
       "   'MetaCyc: ADENYLYLSULFATASE-RXN',\n",
       "   'Name: Adenylylsulfatase; Adenylylsulfate sulfohydrolase; adenosine 5-phosphosulfate sulfohydrolase; adenylyl-sulfate sulfohydrolase; adenylylsulfate sulfohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00193[0] <=> (1) cpd00018[0] + (1) cpd00048[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00018;cpd00048;cpd00067;cpd00193',\n",
       "  'definition': '(1) H2O[0] + (1) APS[0] => (1) AMP[0] + (1) Sulfate[0] + (2) H+[0]',\n",
       "  'deltag': -19.81,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00193[0] => (1) cpd00018[0] + (1) cpd00048[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00381',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19073;rxn32985',\n",
       "  'name': 'Adenylylsulfate sulfohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00193:0:0:\"APS\";1:cpd00018:0:0:\"AMP\";1:cpd00048:0:0:\"Sulfate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00532',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00532',\n",
       "   'MetaCyc: SERINE-SULFATE-AMMONIA-LYASE-RXN',\n",
       "   'Name: (L-SOS)lyase; L-Serine-O-sulfate ammonia-lyase (pyruvate-forming); L-serine-O-sulfate ammonia-lyase (pyruvate-forming); serine-sulfate ammonia-lyase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01750[0] <=> (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00048[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00048;cpd00067;cpd01750',\n",
       "  'definition': '(1) H2O[0] + (1) L-Serine O-sulfate[0] => (1) NH3[0] + (1) Pyruvate[0] + (1) Sulfate[0] + (1) H+[0]',\n",
       "  'deltag': -23.82,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01750[0] => (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00048[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00382',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Serine-O-sulfate ammonia-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01750:0:0:\"L-Serine O-sulfate\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00048:0:0:\"Sulfate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00533',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SULFITE-OXIDASE-RXN',\n",
       "   'ChlamyCyc: SULFITE-OXIDASE-RXN',\n",
       "   'CornCyc: SULFITE-OXIDASE-RXN',\n",
       "   'KEGG: R00533',\n",
       "   'MetaCyc: SULFITE-OXIDASE-RXN',\n",
       "   'Name: Sulfite oxidase; Sulfite:oxygen oxidoreductase; sulfite oxidase; sulfite:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00081[0] <=> (1) cpd00025[0] + (1) cpd00048[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00048;cpd00067;cpd00081',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Sulfite[0] => (1) H2O2[0] + (1) Sulfate[0] + (1) H+[0]',\n",
       "  'deltag': -41.16,\n",
       "  'deltagerr': 1.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.8.3.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00081[0] => (1) cpd00025[0] + (1) cpd00048[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00383',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27265;rxn32966',\n",
       "  'name': 'Sulfite:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); METHIONINE-DEG (L-methionine Degradation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5326 (sulfite oxidation IV); PWY-5328 (superpathway of L-methionine salvage and degradation); PWY-7927 (sulfide oxidation IV (metazoa)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Sulfide-Oxidation (Sulfide Oxidation); Sulfite-Oxidation (Sulfite Oxidation); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00081:0:0:\"Sulfite\";1:cpd00025:0:0:\"H2O2\";1:cpd00048:0:0:\"Sulfate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00534',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00534',\n",
       "   'MetaCyc: GLYCOSULFATASE-RXN',\n",
       "   'Name: D-Glucose 6-sulfate sulfohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01829[0] <=> (1) cpd00027[0] + (1) cpd00048[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00048;cpd00067;cpd01829',\n",
       "  'definition': '(1) H2O[0] + (1) D-Glucose 6-sulfate[0] => (1) D-Glucose[0] + (1) Sulfate[0] + (1) H+[0]',\n",
       "  'deltag': -16.31,\n",
       "  'deltagerr': 2.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.6.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01829[0] => (1) cpd00027[0] + (1) cpd00048[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00384',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose 6-sulfate sulfohydrolase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01829:0:0:\"D-Glucose 6-sulfate\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00048:0:0:\"Sulfate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00535',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00535',\n",
       "   \"Name: 3'-Phosphoadenylylsulfate:17beta-estradiol 3-sulfotransferase\"],\n",
       "  'code': '(1) cpd00044[0] + (1) cpd00702[0] <=> (1) cpd00045[0] + (1) cpd05271[0]',\n",
       "  'compound_ids': 'cpd00044;cpd00045;cpd00067;cpd00702;cpd05271',\n",
       "  'definition': '(1) 3-phosphoadenylylsulfate[0] + (1) Estradiol[0] => (1) Adenosine 3-5-bisphosphate[0] + (1) H+[0] + (1) Estradiol-17beta 3-sulfate[0]',\n",
       "  'deltag': -14.03,\n",
       "  'deltagerr': 7.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.8.2.15'],\n",
       "  'equation': '(1) cpd00044[0] + (1) cpd00702[0] => (1) cpd00045[0] + (1) cpd00067[0] + (1) cpd05271[0]',\n",
       "  'id': 'rxn00385',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"3'-Phosphoadenylylsulfate:17beta-estradiol 3-sulfotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00140 (Steroid hormone biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";-1:cpd00702:0:0:\"Estradiol\";1:cpd00045:0:0:\"Adenosine 3-5-bisphosphate\";1:cpd00067:0:0:\"H+\";1:cpd05271:0:0:\"Estradiol-17beta 3-sulfate\"'},\n",
       " {'abbreviation': 'R00536',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00536',\n",
       "   \"Name: 3'-Phosphoadenylylsulfate:phenolic-steroid sulfotransferase\"],\n",
       "  'code': '(1) cpd00044[0] + (1) cpd03144[0] <=> (1) cpd00045[0] + (1) cpd05272[0]',\n",
       "  'compound_ids': 'cpd00044;cpd00045;cpd00067;cpd03144;cpd05272',\n",
       "  'definition': '(1) 3-phosphoadenylylsulfate[0] + (1) 2-Methoxyestrone[0] => (1) Adenosine 3-5-bisphosphate[0] + (1) H+[0] + (1) 2-Methoxyestrone 3-sulfate[0]',\n",
       "  'deltag': -14.03,\n",
       "  'deltagerr': 7.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.8.2.15'],\n",
       "  'equation': '(1) cpd00044[0] + (1) cpd03144[0] => (1) cpd00045[0] + (1) cpd00067[0] + (1) cpd05272[0]',\n",
       "  'id': 'rxn00386',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"3'-Phosphoadenylylsulfate:phenolic-steroid sulfotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00140 (Steroid hormone biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";-1:cpd03144:0:0:\"2-Methoxyestrone\";1:cpd00045:0:0:\"Adenosine 3-5-bisphosphate\";1:cpd00067:0:0:\"H+\";1:cpd05272:0:0:\"2-Methoxyestrone 3-sulfate\"'},\n",
       " {'abbreviation': 'R00537',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00537',\n",
       "   \"Name: 3'-Phosphoadenylylsulfate:phenolic-steroid sulfotransferase\"],\n",
       "  'code': '(1) cpd00044[0] + (1) cpd03147[0] <=> (1) cpd00045[0] + (1) cpd05273[0]',\n",
       "  'compound_ids': 'cpd00044;cpd00045;cpd00067;cpd03147;cpd05273',\n",
       "  'definition': '(1) 3-phosphoadenylylsulfate[0] + (1) 2-Methoxyestradiol-17beta[0] => (1) Adenosine 3-5-bisphosphate[0] + (1) H+[0] + (1) 2-Methoxyestradiol-17beta 3-sulfate[0]',\n",
       "  'deltag': -14.03,\n",
       "  'deltagerr': 7.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.8.2.15'],\n",
       "  'equation': '(1) cpd00044[0] + (1) cpd03147[0] => (1) cpd00045[0] + (1) cpd00067[0] + (1) cpd05273[0]',\n",
       "  'id': 'rxn00387',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"3'-Phosphoadenylylsulfate:phenolic-steroid sulfotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00140 (Steroid hormone biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";-1:cpd03147:0:0:\"2-Methoxyestradiol-17beta\";1:cpd00045:0:0:\"Adenosine 3-5-bisphosphate\";1:cpd00067:0:0:\"H+\";1:cpd05273:0:0:\"2-Methoxyestradiol-17beta 3-sulfate\"'},\n",
       " {'abbreviation': 'R00540',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00540_c',\n",
       "   'AraGEM: R_R00540_c',\n",
       "   'KEGG: R00540',\n",
       "   'Maize_C4GEM: R_R00540_c',\n",
       "   'iRS1563: R_R00540_c',\n",
       "   'Name: Nitrile aminohydrolase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd00542[0] <=> (1) cpd00013[0] + (1) cpd00049[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00049;cpd00542',\n",
       "  'definition': '(2) H2O[0] + (1) R-CN[0] => (1) NH3[0] + (1) Carboxylic acid[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.5.1', '3.5.5.2', '3.5.5.5', '3.5.5.7'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd00542[0] => (1) cpd00013[0] + (1) cpd00049[0]',\n",
       "  'id': 'rxn00388',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nitrile aminohydrolase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd00542:0:0:\"R-CN\";1:cpd00013:0:0:\"NH3\";1:cpd00049:0:0:\"Carboxylic acid\"'},\n",
       " {'abbreviation': 'R00545',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00545', 'Name: N6-Acyl-L-lysine amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01611[0] <=> (1) cpd00039[0] + (1) cpd00049[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00039;cpd00049;cpd01611',\n",
       "  'definition': '(1) H2O[0] + (1) N6-Acyl-L-lysine[0] => (1) L-Lysine[0] + (1) Carboxylic acid[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.17'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01611[0] => (1) cpd00039[0] + (1) cpd00049[0]',\n",
       "  'id': 'rxn00389',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N6-Acyl-L-lysine amidohydrolase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01611:0:0:\"N6-Acyl-L-lysine\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00049:0:0:\"Carboxylic acid\"'},\n",
       " {'abbreviation': 'R00546',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00546', 'Name: N-Acyl-L-aspartate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01759[0] <=> (1) cpd00041[0] + (1) cpd00049[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00041;cpd00049;cpd01759',\n",
       "  'definition': '(1) H2O[0] + (1) N-Acyl-L-aspartate[0] => (1) L-Aspartate[0] + (1) Carboxylic acid[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.15'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01759[0] => (1) cpd00041[0] + (1) cpd00049[0]',\n",
       "  'id': 'rxn00390',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Acyl-L-aspartate amidohydrolase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01759:0:0:\"N-Acyl-L-aspartate\";1:cpd00041:0:0:\"L-Aspartate\";1:cpd00049:0:0:\"Carboxylic acid\"'},\n",
       " {'abbreviation': 'R00548',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACP1(FMN); ACP1_FMN; ACP1_FMN_; ACP1e; PHO11',\n",
       "   'EcoCyc: RXN0-5187',\n",
       "   'KEGG: R00548',\n",
       "   'MaizeCyc: RXNBWI-1; RXNBWI-2',\n",
       "   'MetaCyc: RXN0-5187',\n",
       "   'Name: FMN hydrolase; Riboflavin-5-phosphate phosphohydrolase (acid optimum); acid phosphatase (FMN); acid phosphatase, (secreted); acidphosphatase-FMN; riboflavin-5-phosphate phosphohydrolase (acid optimum)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00050[0] <=> (1) cpd00009[0] + (1) cpd00220[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00050;cpd00067;cpd00220',\n",
       "  'definition': '(1) H2O[0] + (1) FMN[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) Riboflavin[0]',\n",
       "  'deltag': -4.04,\n",
       "  'deltagerr': 0.7,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.-', '3.1.3.102', '3.1.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00050[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00220[0]',\n",
       "  'id': 'rxn00391',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30337;rxn32581;rxn38689',\n",
       "  'name': 'riboflavin-5-phosphate phosphohydrolase (acid optimum)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00740 (Riboflavin metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00050:0:0:\"FMN\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:H+;1:cpd00220:0:0:\"Riboflavin\"'},\n",
       " {'abbreviation': 'R00549',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RIBOFLAVINKIN-RXN',\n",
       "   'BiGG: FMN1_1; FMN1_2; RBFK; RBFKm',\n",
       "   'BrachyCyc: RIBOFLAVINKIN-RXN',\n",
       "   'KEGG: R00549',\n",
       "   'MetaCyc: RIBOFLAVINKIN-RXN',\n",
       "   \"Name: ATP:riboflavin 5'-phosphotransferase; FK; RFK; Riboflavin kinase; flavokinase; riboflavin kinase; riboflavinkinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00220[0] <=> (1) cpd00008[0] + (1) cpd00050[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00050;cpd00220',\n",
       "  'definition': '(1) ATP[0] + (1) Riboflavin[0] => (1) ADP[0] + (1) FMN[0]',\n",
       "  'deltag': -2.12,\n",
       "  'deltagerr': 0.7,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.26'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00220[0] => (1) cpd00008[0] + (1) cpd00050[0]',\n",
       "  'id': 'rxn00392',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20779;rxn32424',\n",
       "  'name': \"ATP:riboflavin 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenosylcobamide-Biosynthesis (Adenosylcobamide Biosynthesis); Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Cobalamin-Biosynthesis (Cobamide Biosynthesis); Cobamide-De-Novo-Biosynthesis (Cobamide <i>de novo</i> Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); DMB-Biosynthesis (5,6-Dimethylbenzimidazole Biosynthesis); De-Novo-Adenosylcobalamin-Biosynthesis (Adenosylcobalamin <i>de novo</i> Biosynthesis); Flavin-Biosynthesis (Flavin Biosynthesis); P381-PWY (adenosylcobalamin biosynthesis II (aerobic)); PWY-5507 (adenosylcobalamin biosynthesis I (anaerobic)); PWY-5523 (5,6-dimethylbenzimidazole biosynthesis I (aerobic)); PWY-6168 (flavin biosynthesis III (fungi)); PWY-7863 (roseoflavin biosynthesis); PWY66-366 (flavin biosynthesis IV (mammalian)); RIBOSYN2-PWY (flavin biosynthesis I (bacteria and plants)); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00740 (Riboflavin metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00220:0:0:\"Riboflavin\";1:cpd00008:0:0:\"ADP\";1:cpd00050:0:0:\"FMN\"'},\n",
       " {'abbreviation': 'R00550',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00550',\n",
       "   'MetaCyc: RIBOFLAVIN-PHOSPHOTRANSFERASE-RXN',\n",
       "   \"Name: D-Glucose-1-phosphate:riboflavin 5'-phosphotransferase\"],\n",
       "  'code': '(1) cpd00089[0] + (1) cpd00220[0] <=> (1) cpd00027[0] + (1) cpd00050[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00050;cpd00067;cpd00089;cpd00220',\n",
       "  'definition': '(1) H+[0] + (1) Glucose-1-phosphate[0] + (1) Riboflavin[0] <=> (1) D-Glucose[0] + (1) FMN[0]',\n",
       "  'deltag': -0.82,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.42'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00089[0] + (1) cpd00220[0] <=> (1) cpd00027[0] + (1) cpd00050[0]',\n",
       "  'id': 'rxn00393',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"D-Glucose-1-phosphate:riboflavin 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:H+;-1:cpd00089:0:0:\"Glucose-1-phosphate\";-1:cpd00220:0:0:\"Riboflavin\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00050:0:0:\"FMN\"'},\n",
       " {'abbreviation': 'R00551',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ARGINASE-RXN',\n",
       "   'BrachyCyc: ARGINASE-RXN',\n",
       "   'ChlamyCyc: ARGINASE-RXN',\n",
       "   'KEGG: R00551',\n",
       "   'MetaCyc: ARGINASE-RXN',\n",
       "   'Name: ARGN; Arginase; L-Arginine amidinohydrolase; L-arginase; L-arginine amidinohydrolase; arginine amidinase; arginine transamidinase; canavanase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00051[0] <=> (1) cpd00064[0] + (1) cpd00073[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00051;cpd00064;cpd00073',\n",
       "  'definition': '(1) H2O[0] + (1) L-Arginine[0] => (1) Ornithine[0] + (1) Urea[0]',\n",
       "  'deltag': -7.0,\n",
       "  'deltagerr': 1.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.3.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00051[0] => (1) cpd00064[0] + (1) cpd00073[0]',\n",
       "  'id': 'rxn00394',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19181;rxn30615;rxn31988',\n",
       "  'name': 'L-Arginine amidinohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG-GLU-PWY (L-arginine degradation VII (arginase 3 pathway)); ARG-PRO-PWY (L-arginine degradation VI (arginase 2 pathway)); ARGASEDEG-PWY (L-arginine degradation I (arginase pathway)); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CITRULBIO-PWY (L-citrulline biosynthesis); Citrulline-Biosynthesis (L-citrulline Biosynthesis); Degradation (Degradation/Utilization/Assimilation); GLUTAMATE-SYN (L-glutamate Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PROLINE-SYN (L-proline Biosynthesis); PWY-4984 (urea cycle); PWY-5004 (superpathway of L-citrulline metabolism); PWY-6305 (putrescine biosynthesis IV); PWY-6922 (L-<i>N<sup>&delta;</sup></i>-acetylornithine biosynthesis); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Putrescine-Biosynthesis (Putrescine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00064:0:0:\"Ornithine\";1:cpd00073:0:0:\"Urea\"'},\n",
       " {'abbreviation': 'R00552',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ARGD; ARGDE; ARGDr',\n",
       "   'BrachyCyc: ARGININE-DEIMINASE-RXN',\n",
       "   'ChlamyCyc: ARGININE-DEIMINASE-RXN',\n",
       "   'KEGG: R00552',\n",
       "   'MetaCyc: ARGININE-DEIMINASE-RXN',\n",
       "   'Name: Arginine deiminase; L-Arginine iminohydrolase; L-arginine deiminase; L-arginine iminohydrolase; arginine deiminase; arginine dihydrolase; citrulline iminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00051[0] <=> (1) cpd00013[0] + (1) cpd00274[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00051;cpd00274',\n",
       "  'definition': '(1) H2O[0] + (1) L-Arginine[0] => (1) NH3[0] + (1) Citrulline[0]',\n",
       "  'deltag': -8.34,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.3.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00051[0] => (1) cpd00013[0] + (1) cpd00274[0]',\n",
       "  'id': 'rxn00395',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30071;rxn30617;rxn33434;rxn33439;rxn33441;rxn33448',\n",
       "  'name': 'L-Arginine iminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGDEGRAD-PWY (L-arginine degradation V (arginine deiminase pathway)); ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PROLINE-SYN (L-proline Biosynthesis); PWY-4981 (L-proline biosynthesis II (from arginine)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00013:0:0:\"NH3\";1:cpd00274:0:0:\"Citrulline\"'},\n",
       " {'abbreviation': 'R00554',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ARGKr',\n",
       "   'KEGG: R00554',\n",
       "   'MetaCyc: ARGININE-KINASE-RXN',\n",
       "   'iYO844: ARGKr',\n",
       "   \"Name: ATP:L-arginine N-phosphotransferase; ATP:L-arginine N-phosphotransferasel ATP:L-arginine omega-N-phosphotransferase; ATP:L-arginine Nomega-phosphotransferase; adenosine 5'-triphosphate-arginine phosphotransferase; adenosine 5'-triphosphate: L-arginine phosphotransferase; arginine kinase; arginine phosphokinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00051[0] <=> (1) cpd00008[0] + (1) cpd03535[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00051;cpd00067;cpd03535',\n",
       "  'definition': '(1) ATP[0] + (1) L-Arginine[0] <=> (1) ADP[0] + (1) H+[0] + (1) Phosphoarginine[0]',\n",
       "  'deltag': 1.8,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.3.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00051[0] <=> (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd03535[0]',\n",
       "  'id': 'rxn00396',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:L-arginine Nomega-phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd03535:0:0:\"Phosphoarginine\"'},\n",
       " {'abbreviation': 'NAD+:L-arginine ADP-D-ribosyltransferase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: NAD+:L-arginine ADP-D-ribosyltransferase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00051[0] <=> (1) cpd00133[0] + (1) cpd00884[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00051;cpd00067;cpd00133;cpd00884',\n",
       "  'definition': '(1) NAD[0] + (1) L-Arginine[0] <=> (1) H+[0] + (1) Nicotinamide[0] + (1) N(omega)-(ADP-D-ribosyl)-L-arginine[0]',\n",
       "  'deltag': -5.68,\n",
       "  'deltagerr': 1.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.2.31'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00051[0] <=> (1) cpd00067[0] + (1) cpd00133[0] + (1) cpd00884[0]',\n",
       "  'id': 'rxn00397',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NAD+:L-arginine ADP-D-ribosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00067:0:0:\"H+\";1:cpd00133:0:0:\"Nicotinamide\";1:cpd00884:0:0:\"N(omega)-(ADP-D-ribosyl)-L-arginine\"'},\n",
       " {'abbreviation': 'ARG-OXIDATION-RXN.c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: ARG-OXIDATION-RXN',\n",
       "   'MetaCyc: ARG-OXIDATION-RXN',\n",
       "   'Name: '],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00051[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd03528[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00051;cpd03528',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Arginine[0] => (1) NH3[0] + (1) H2O2[0] + (1) 2-Oxoarginine[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.-', '1.4.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00051[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd03528[0]',\n",
       "  'id': 'rxn00398',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ARG-OXIDATION-RXN.c',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['MetaCyc: ARGDEG-IV-PWY (L-arginine degradation VIII (arginine oxidase pathway)); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd03528:0:0:\"2-Oxoarginine\"'},\n",
       " {'abbreviation': 'L-Arginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: L-Arginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming)'],\n",
       "  'code': '(1) cpd00005[0] + (3) cpd00007[0] + (2) cpd00051[0] <=> (2) cpd00001[0] + (1) cpd00006[0] + (2) cpd00274[0] + (2) cpd00418[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00051;cpd00067;cpd00274;cpd00418',\n",
       "  'definition': '(1) NADPH[0] + (3) O2[0] + (2) L-Arginine[0] <=> (2) H2O[0] + (1) NADP[0] + (1) H+[0] + (2) Citrulline[0] + (2) NO[0]',\n",
       "  'deltag': -190.6,\n",
       "  'deltagerr': 3.1,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.13.39'],\n",
       "  'equation': '(1) cpd00005[0] + (3) cpd00007[0] + (2) cpd00051[0] <=> (2) cpd00001[0] + (1) cpd00006[0] + (1) cpd00067[0] + (2) cpd00274[0] + (2) cpd00418[0]',\n",
       "  'id': 'rxn00399',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Arginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming)',\n",
       "  'notes': ['GCP', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-3:cpd00007:0:0:\"O2\";-2:cpd00051:0:0:\"L-Arginine\";2:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00067:0:0:\"H+\";2:cpd00274:0:0:\"Citrulline\";2:cpd00418:0:0:\"NO\"'},\n",
       " {'abbreviation': 'R00558',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_NOS1',\n",
       "   'KEGG: R00558',\n",
       "   'iRS1563: R_R00558_c',\n",
       "   'Name: L-Arginine,NADPH:oxygen oxidoreductase (N-(omega)-hydroxyarginine-forming); L-arginine,NADPH:oxygen oxidoreductase (N-(omega)-hydroxyarginine-forming)'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00051[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd03527[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00051;cpd00067;cpd03527',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) L-Arginine[0] + (1) H+[0] => (1) H2O[0] + (1) NADP[0] + (1) N-(omega)-Hydroxyarginine[0]',\n",
       "  'deltag': -59.15,\n",
       "  'deltagerr': 7.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.39'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00051[0] + (1) cpd00067[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd03527[0]',\n",
       "  'id': 'rxn00400',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Arginine,NADPH:oxygen oxidoreductase (N-(omega)-hydroxyarginine-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00051:0:0:\"L-Arginine\";-1:cpd00067:0:0:H+;1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd03527:0:0:\"N-(omega)-Hydroxyarginine\"'},\n",
       " {'abbreviation': 'R00559',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ARGININE-2-MONOOXYGENASE-RXN',\n",
       "   'BrachyCyc: ARGININE-2-MONOOXYGENASE-RXN',\n",
       "   'KEGG: R00559',\n",
       "   'MaizeCyc: ARGININE-2-MONOOXYGENASE-RXN',\n",
       "   'MetaCyc: ARGININE-2-MONOOXYGENASE-RXN',\n",
       "   'Name: L-Arginine:oxygen 2-oxidoreductase (decarboxylating); L-arginine:oxygen 2-oxidoreductase (decarboxylating); arginine 2-monooxygenase; arginine decarboxy-oxidase; arginine decarboxylase; arginine monooxygenase; arginine oxygenase (decarboxylating)'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00051[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01973[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00011;cpd00051;cpd01973',\n",
       "  'definition': '(1) O2[0] + (1) L-Arginine[0] => (1) H2O[0] + (1) CO2[0] + (1) 4-Guanidinobutanamide[0]',\n",
       "  'deltag': -106.64,\n",
       "  'deltagerr': 1.69,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.12.1'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00051[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01973[0]',\n",
       "  'id': 'rxn00401',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30616',\n",
       "  'name': 'L-Arginine:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGDEG-V-PWY (L-arginine degradation X (arginine monooxygenase pathway)); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd01973:0:0:\"4-Guanidinobutanamide\"'},\n",
       " {'abbreviation': 'R00562',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00562',\n",
       "   'MetaCyc: 1.5.1.11-RXN',\n",
       "   'Name: 2-N-(D-1-carboxyethyl)-L-arginine:NAD+ oxidoreductase (L-arginine-forming); D-octopine dehydrogenase; D-octopine synthase; N2-(D-1-carboxyethyl)-L-arginine:NAD+ oxidoreductase (L-arginine-forming); N2-(D-1-carboxyethyl)-L-arginine:NADP+ oxidoreductase; ODH; octopine dehydrogenase; octopine:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd02553[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00051[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00020;cpd00051;cpd00067;cpd02553',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Octopine[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (1) L-Arginine[0] + (1) H+[0]',\n",
       "  'deltag': 7.53,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd02553[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00051[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00402',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N2-(D-1-carboxyethyl)-L-arginine:NAD+ oxidoreductase (L-arginine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7351 (pyruvate fermentation to opines); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd02553:0:0:\"Octopine\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00051:0:0:\"L-Arginine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00563',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NOPD',\n",
       "   'KEGG: R00563',\n",
       "   'MetaCyc: 1.5.1.19-RXN',\n",
       "   'iSB619: NOPD',\n",
       "   'Name: 2-N-(D-1,3-dicarboxypropyl)-L-arginine:NADP+ oxidoreductase (L-arginine-forming); D-nopaline dehydrogenase; D-nopaline synthase; D-nopaline:NADP+ oxidoreductase (L-arginine-forming); N2-(D-1,3-dicarboxypropyl)-L-arginine:NADP+ oxidoreductase (L-arginine-forming); NOS; nopaline dehydrogenase; nopaline synthase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd01162[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (1) cpd00051[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00024;cpd00051;cpd00067;cpd01162',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) D-Nopaline[0] <=> (1) NADPH[0] + (1) 2-Oxoglutarate[0] + (1) L-Arginine[0] + (1) H+[0]',\n",
       "  'deltag': 3.22,\n",
       "  'deltagerr': 1.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.19'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd01162[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (1) cpd00051[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00403',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N2-(D-1,3-dicarboxypropyl)-L-arginine:NADP+ oxidoreductase (L-arginine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd01162:0:0:\"D-Nopaline\";1:cpd00005:0:0:\"NADPH\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00051:0:0:\"L-Arginine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00565',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00565',\n",
       "   'MetaCyc: GLYCINE-AMIDINOTRANSFERASE-RXN',\n",
       "   'Name: L-Arginine:glycine amidinotransferase; L-arginine:glycine amidinotransferase; arginine-glycine amidinotransferase; arginine-glycine transamidinase; glycine amidinotransferase; glycine transamidinase'],\n",
       "  'code': '(1) cpd00033[0] + (1) cpd00051[0] <=> (1) cpd00064[0] + (1) cpd00451[0]',\n",
       "  'compound_ids': 'cpd00033;cpd00051;cpd00064;cpd00451',\n",
       "  'definition': '(1) Glycine[0] + (1) L-Arginine[0] <=> (1) Ornithine[0] + (1) Glycocyamine[0]',\n",
       "  'deltag': -0.08,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.4.1'],\n",
       "  'equation': '(1) cpd00033[0] + (1) cpd00051[0] <=> (1) cpd00064[0] + (1) cpd00451[0]',\n",
       "  'id': 'rxn00404',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19855',\n",
       "  'name': 'L-Arginine:glycine amidinotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Degradation (Degradation/Utilization/Assimilation); GLYCGREAT-PWY (creatine biosynthesis); GLYCINE-DEG (Glycine Degradation); PWY-7693 (guadinomine B biosynthesis); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00033:0:0:\"Glycine\";-1:cpd00051:0:0:\"L-Arginine\";1:cpd00064:0:0:\"Ornithine\";1:cpd00451:0:0:\"Glycocyamine\"'},\n",
       " {'abbreviation': 'R00566',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ARGDECARBOX-RXN',\n",
       "   'BiGG: ARGDC; ARGDCpp',\n",
       "   'BrachyCyc: ARGDECARBOX-RXN',\n",
       "   'KEGG: R00566',\n",
       "   'MetaCyc: ARGDECARBOX-RXN',\n",
       "   'Name: Arginine decarboxylase; L-Arginine carboxy-lyase; L-arginine carboxy-lyase; L-arginine carboxy-lyase (agmatine-forming); SpeA; arginine decarboxylase; argininedecarboxylase'],\n",
       "  'code': '(1) cpd00051[0] <=> (1) cpd00011[0] + (1) cpd00152[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00051;cpd00067;cpd00152',\n",
       "  'definition': '(1) L-Arginine[0] + (1) H+[0] => (1) CO2[0] + (1) Agmatine[0]',\n",
       "  'deltag': -0.83,\n",
       "  'deltagerr': 1.48,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.19'],\n",
       "  'equation': '(1) cpd00051[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00152[0]',\n",
       "  'id': 'rxn00405',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19180;rxn32213;rxn35371',\n",
       "  'name': 'L-arginine carboxy-lyase (agmatine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGDEG-III-PWY (L-arginine degradation IV (arginine decarboxylase/agmatine deiminase pathway)); ARGDEG-PWY (superpathway of L-arginine, putrescine, and 4-aminobutanoate degradation); ARGININE-DEG (L-arginine Degradation); Acid-Resistance (Acid Resistance); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); ORNARGDEG-PWY (superpathway of L-arginine and L-ornithine degradation); POLYAMINSYN3-PWY (superpathway of polyamine biosynthesis II); POLYAMSYN-PWY (superpathway of polyamine biosynthesis I); PWY-40 (putrescine biosynthesis I); PWY-43 (putrescine biosynthesis II); PWY-6305 (putrescine biosynthesis IV); PWY-6834 (spermidine biosynthesis III); PWY0-1299 (arginine dependent acid resistance); PWY0-823 (L-arginine degradation III (arginine decarboxylase/agmatinase pathway)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Putrescine-Biosynthesis (Putrescine Biosynthesis); Spermidine-Biosynthesis (Spermidine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00051:0:0:\"L-Arginine\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00152:0:0:\"Agmatine\"'},\n",
       " {'abbreviation': 'R00567',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00567',\n",
       "   'MetaCyc: ARGININE-RACEMASE-RXN',\n",
       "   'Name: L-Arginine racemase; arginine racemase'],\n",
       "  'code': '(1) cpd00051[0] <=> (1) cpd00586[0]',\n",
       "  'compound_ids': 'cpd00051;cpd00586',\n",
       "  'definition': '(1) L-Arginine[0] <=> (1) D-Arginine[0]',\n",
       "  'deltag': 4.63,\n",
       "  'deltagerr': 1.27,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.9'],\n",
       "  'equation': '(1) cpd00051[0] <=> (1) cpd00586[0]',\n",
       "  'id': 'rxn00406',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'arginine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00472 (D-Arginine and D-ornithine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00051:0:0:\"L-Arginine\";1:cpd00586:0:0:\"D-Arginine\"'},\n",
       " {'abbreviation': 'R00568',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: DCTPD2',\n",
       "   'KEGG: R00568',\n",
       "   'MetaCyc: RXN-14118',\n",
       "   'iIT341: DCTPD2',\n",
       "   'iJN746: DCTPD2',\n",
       "   'Name: CTP aminohydrolase; dCTPdeaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00052[0] <=> (1) cpd00013[0] + (1) cpd00062[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00052;cpd00062;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) CTP[0] + (1) H+[0] => (1) NH3[0] + (1) UTP[0]',\n",
       "  'deltag': -6.03,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.13'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00052[0] + (1) cpd00067[0] => (1) cpd00013[0] + (1) cpd00062[0]',\n",
       "  'id': 'rxn00407',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30308',\n",
       "  'name': 'CTP aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00052:0:0:\"CTP\";-1:cpd00067:0:0:\"H+\";1:cpd00013:0:0:\"NH3\";1:cpd00062:0:0:\"UTP\"'},\n",
       " {'abbreviation': 'R00569',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN-12195',\n",
       "   'BiGG: NTP5',\n",
       "   'BrachyCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN',\n",
       "   'KEGG: R00569',\n",
       "   'MetaCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN-12195',\n",
       "   'Name: CTP diphosphohydrolase; CTP phosphohydrolase; nucleoside-triphosphatase (CTP); unspecific diphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00052[0] <=> (1) cpd00009[0] + (1) cpd00096[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00052;cpd00067;cpd00096',\n",
       "  'definition': '(1) H2O[0] + (1) CTP[0] => (1) Phosphate[0] + (1) H+[0] + (1) CDP[0]',\n",
       "  'deltag': -5.89,\n",
       "  'deltagerr': 0.56,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.15', '3.6.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00052[0] => (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00096[0]',\n",
       "  'id': 'rxn00408',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn22518;rxn38192;rxn38200;rxn38208;rxn38216;rxn38224;rxn38234',\n",
       "  'name': 'CTP phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-6545 (pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis III); PWY-7184 (pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis I); PWY-7198 (pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis IV); PWY-7210 (pyrimidine deoxyribonucleotides biosynthesis from CTP); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY0-166 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis (E. coli)); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00052:0:0:\"CTP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00096:0:0:\"CDP\"'},\n",
       " {'abbreviation': 'R00570',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CDPKIN-RXN; NUCLEOSIDE-DIP-KIN-RXN',\n",
       "   'BiGG: NDPK3; YNK1_2',\n",
       "   'BrachyCyc: CDPKIN-RXN; NUCLEOSIDE-DIP-KIN-RXN',\n",
       "   'KEGG: R00570',\n",
       "   'MetaCyc: CDPKIN-RXN; NUCLEOSIDE-DIP-KIN-RXN',\n",
       "   'Name: ATP:CDP phosphotransferase; cdp kinase; nucleoside-diphosphate kinase (ATP:CDP); nucleoside-diphosphatekinase-ATP-CDP'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00096[0] <=> (1) cpd00008[0] + (1) cpd00052[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00052;cpd00096',\n",
       "  'definition': '(1) ATP[0] + (1) CDP[0] <=> (1) ADP[0] + (1) CTP[0]',\n",
       "  'deltag': -0.27,\n",
       "  'deltagerr': 0.56,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00096[0] <=> (1) cpd00008[0] + (1) cpd00052[0]',\n",
       "  'id': 'rxn00409',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19276;rxn27747;rxn27748;rxn32981',\n",
       "  'name': 'ATP:CDP phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PWY-7205 (CMP phosphorylation); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00096:0:0:\"CDP\";1:cpd00008:0:0:\"ADP\";1:cpd00052:0:0:\"CTP\"'},\n",
       " {'abbreviation': 'R00571',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00571_c',\n",
       "   'AraGEM: R_R00571_c',\n",
       "   'BiGG: CTPS1; URA7_2; URA8_2',\n",
       "   'KEGG: R00571',\n",
       "   'MetaCyc: RXN-14325',\n",
       "   'Name: CTP synthase (NH3); CTP synthase 1; CTPsynthase-NH3; UTP:ammonia ligase (ADP-forming); UTP:ammonia ligase(ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00062[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00052[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00013;cpd00052;cpd00062;cpd00067',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) UTP[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) CTP[0] + (2) H+[0]',\n",
       "  'deltag': -0.13,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.3.4.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00062[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00052[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00410',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30073;rxn32019;rxn32058',\n",
       "  'name': 'UTP:ammonia ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00062:0:0:\"UTP\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00052:0:0:\"CTP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00572',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00572',\n",
       "   'Name: CTP:pyruvate 2-O-phosphotransferase; CTP:pyruvate O2-phosphotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00052[0] <=> (1) cpd00061[0] + (1) cpd00096[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00052;cpd00061;cpd00067;cpd00096',\n",
       "  'definition': '(1) Pyruvate[0] + (1) CTP[0] <=> (1) Phosphoenolpyruvate[0] + (1) H+[0] + (1) CDP[0]',\n",
       "  'deltag': 6.8,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.40'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00052[0] <=> (1) cpd00061[0] + (1) cpd00067[0] + (1) cpd00096[0]',\n",
       "  'id': 'rxn00411',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CTP:pyruvate 2-O-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00052:0:0:\"CTP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00096:0:0:\"CDP\"'},\n",
       " {'abbreviation': 'R00573',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CTPSYN-RXN',\n",
       "   'BiGG: CTPS2; URA7_1; URA8_1',\n",
       "   'BrachyCyc: CTPSYN-RXN',\n",
       "   'KEGG: R00573',\n",
       "   'MetaCyc: CTPSYN-RXN',\n",
       "   \"Name: CTP synthase; CTP synthase (glutamine); CTP synthetase; UTP--ammonia ligase; UTP-ammonia ligase; UTP:L-glutamine amido-ligase (ADP-forming); UTP:ammonia ligase (ADP-forming); UTP:ammonia ligase(ADP-forming); ctp synthase; cytidine 5'-triphosphate synthetase; cytidine triphosphate synthetase; uridine triphosphate aminase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00053[0] + (1) cpd00062[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00023[0] + (1) cpd00052[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00008;cpd00009;cpd00023;cpd00052;cpd00053;cpd00062;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) L-Glutamine[0] + (1) UTP[0] => (1) ADP[0] + (1) Phosphate[0] + (1) L-Glutamate[0] + (1) CTP[0] + (2) H+[0]',\n",
       "  'deltag': -2.95,\n",
       "  'deltagerr': 0.63,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.4.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00053[0] + (1) cpd00062[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00023[0] + (1) cpd00052[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00412',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn11276;rxn27771;rxn33242',\n",
       "  'name': 'UTP:L-glutamine amido-ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Degradation (Degradation/Utilization/Assimilation); Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-7176 (UTP and CTP <i>de novo</i> biosynthesis); PWY-7177 (UTP and CTP dephosphorylation II); PWY-7185 (UTP and CTP dephosphorylation I); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PWY-7208 (superpathway of pyrimidine nucleobases salvage); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY0-162 (superpathway of pyrimidine ribonucleotides <i>de novo</i> biosynthesis); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimid-Ribonucleot-De-Novo-Biosyn (Pyrimidine Ribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation); UTP-CTP-Dephosphorylation (UTP and CTP Dephosphorylation)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00062:0:0:\"UTP\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00052:0:0:\"CTP\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00574',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00574',\n",
       "   'MetaCyc: CYTIDYLATE-CYCLASE-RXN',\n",
       "   \"Name: 3',5'-cyclic-CMP synthase; CTP diphosphate-lyase (cyclizing); CTP diphosphate-lyase (cyclizing, 3',5'-cyclic-CMP-forming); CTP diphosphate-lyase (cyclizing; 3',5'-cyclic-CMP-forming); CTP pyriphosphate-lyase (cyclizing); cytidyl cyclase; cytidylate cyclase; cytidylyl cyclase\"],\n",
       "  'code': '(1) cpd00052[0] <=> (1) cpd00012[0] + (1) cpd00696[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00052;cpd00067;cpd00696',\n",
       "  'definition': \"(1) CTP[0] => (1) PPi[0] + (1) H+[0] + (1) 3',5'-Cyclic CMP[0]\",\n",
       "  'deltag': 0.79,\n",
       "  'deltagerr': 0.7,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.6.1.6'],\n",
       "  'equation': '(1) cpd00052[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00696[0]',\n",
       "  'id': 'rxn00413',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"CTP diphosphate-lyase (cyclizing; 3',5'-cyclic-CMP-forming)\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00052:0:0:\"CTP\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd00696:0:0:\"3\\',5\\'-Cyclic CMP\"'},\n",
       " {'abbreviation': 'R00575',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CARBPSYN-RXN',\n",
       "   'BiGG: CBPS; CBPSn',\n",
       "   'BrachyCyc: CARBPSYN-RXN',\n",
       "   'KEGG: R00575',\n",
       "   'MetaCyc: CARBPSYN-RXN',\n",
       "   'Name: CPS; Carbon-dioxide:L-glutamine amido-ligase (ADP-forming,; carbamate-phosphorylating); carbamoyl phosphate synthetase; carbamoyl-phosphate synthase (glutamine-hydrolysing); carbamoyl-phosphate synthase family protein; carbamoyl-phosphate synthetase (glutamine-hydrolysing); carbamoyl-phosphatesynthase-glutamine-hydrolysing; carbamoylphosphate synthetase II; carbamyl phosphate synthetase (glutamine); carbon-dioxide:L-glutamine amido-ligase (ADP-forming, carbamate-phosphorylating); glutamine-dependent carbamyl phosphate synthetase; hydrogen-carbonate:L-glutamine amido-ligase (ADP-forming, carbamate-phosphorylating)'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00002[0] + (1) cpd00053[0] + (1) cpd00242[0] <=> (2) cpd00008[0] + (1) cpd00009[0] + (1) cpd00023[0] + (1) cpd00146[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00008;cpd00009;cpd00023;cpd00053;cpd00067;cpd00146;cpd00242',\n",
       "  'definition': '(1) H2O[0] + (2) ATP[0] + (1) L-Glutamine[0] + (1) H2CO3[0] => (2) ADP[0] + (1) Phosphate[0] + (1) L-Glutamate[0] + (2) H+[0] + (1) Carbamoylphosphate[0]',\n",
       "  'deltag': -1.42,\n",
       "  'deltagerr': 0.64,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.5.5'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00002[0] + (1) cpd00053[0] + (1) cpd00242[0] => (2) cpd00008[0] + (1) cpd00009[0] + (1) cpd00023[0] + (2) cpd00067[0] + (1) cpd00146[0]',\n",
       "  'id': 'rxn00414',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27743;rxn30627;rxn31216;rxn38060',\n",
       "  'name': 'hydrogen-carbonate:L-glutamine amido-ligase (ADP-forming, carbamate-phosphorylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGININE-SYN (L-arginine Biosynthesis); ARGSYN-PWY (L-arginine biosynthesis I (via L-ornithine)); ARGSYNBSUB-PWY (L-arginine biosynthesis II (acetyl cycle)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-5154 (L-arginine biosynthesis III (via <i>N</i>-acetyl-L-citrulline)); PWY-5686 (UMP biosynthesis I); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY-7400 (L-arginine biosynthesis IV (archaebacteria)); PWY-7790 (UMP biosynthesis II); PWY-7791 (UMP biosynthesis III); PWY0-162 (superpathway of pyrimidine ribonucleotides <i>de novo</i> biosynthesis); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimid-Ribonucleot-De-Novo-Biosyn (Pyrimidine Ribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); UMP-Biosynthesis (UMP Biosynthesis)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism); rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00002:0:0:\"ATP\";-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00242:0:0:\"H2CO3\";2:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00023:0:0:\"L-Glutamate\";2:cpd00067:0:0:\"H+\";1:cpd00146:0:0:\"Carbamoylphosphate\"'},\n",
       " {'abbreviation': 'R00576',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00576',\n",
       "   'MetaCyc: GLUTAMINE--PYRUVATE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-Glutamine:pyruvate aminotransferase; L-glutamine transaminase L; L-glutamine:pyruvate aminotransferase; glutaminase II; glutamine-oxo-acid transaminase; glutamine-pyruvate transaminase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00053[0] <=> (1) cpd00035[0] + (1) cpd00695[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00035;cpd00053;cpd00695',\n",
       "  'definition': '(1) Pyruvate[0] + (1) L-Glutamine[0] <=> (1) L-Alanine[0] + (1) 2-Oxoglutaramate[0]',\n",
       "  'deltag': -2.75,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.15'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00053[0] <=> (1) cpd00035[0] + (1) cpd00695[0]',\n",
       "  'id': 'rxn00415',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Glutamine:pyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00053:0:0:\"L-Glutamine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd00695:0:0:\"2-Oxoglutaramate\"'},\n",
       " {'abbreviation': 'R00578',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASNSYNB-RXN',\n",
       "   'BiGG: ASN1; ASN2; ASNS1',\n",
       "   'BrachyCyc: ASNSYNB-RXN',\n",
       "   'KEGG: R00578',\n",
       "   'MetaCyc: ASNSYNB-RXN',\n",
       "   'Name: AS; AS-B; Asparagine synthase (glutamine-hydrolysing); L-aspartate:L-glutamine amido-ligase (AMP-forming); asparagine synthase (glutamine-hydrolysing); asparagine synthetase (glutamine-hydrolysing); asparagine synthetase B; glutamine-dependent asparagine synthetase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00041[0] + (1) cpd00053[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00023[0] + (1) cpd00132[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00012;cpd00018;cpd00023;cpd00041;cpd00053;cpd00067;cpd00132',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) L-Aspartate[0] + (1) L-Glutamine[0] => (1) PPi[0] + (1) AMP[0] + (1) L-Glutamate[0] + (2) H+[0] + (1) L-Asparagine[0]',\n",
       "  'deltag': -7.24,\n",
       "  'deltagerr': 0.89,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.5.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00041[0] + (1) cpd00053[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00023[0] + (2) cpd00067[0] + (1) cpd00132[0]',\n",
       "  'id': 'rxn00416',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32227;rxn38046',\n",
       "  'name': 'L-aspartate:L-glutamine amido-ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-BIOSYNTHESIS (L-asparagine biosynthesis I); ASPARAGINE-SYN (L-asparagine Biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY0-1325 (superpathway of L-asparagine biosynthesis)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00041:0:0:\"L-Aspartate\";-1:cpd00053:0:0:\"L-Glutamine\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00023:0:0:\"L-Glutamate\";2:cpd00067:0:0:\"H+\";1:cpd00132:0:0:\"L-Asparagine\"'},\n",
       " {'abbreviation': 'R00579',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00579',\n",
       "   'Name: L-Glutamine racemase; glutamine racemase'],\n",
       "  'code': '(1) cpd00053[0] <=> (1) cpd00610[0]',\n",
       "  'compound_ids': 'cpd00053;cpd00610',\n",
       "  'definition': '(1) L-Glutamine[0] <=> (1) D-Glutamine[0]',\n",
       "  'deltag': 1.69,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.10'],\n",
       "  'equation': '(1) cpd00053[0] <=> (1) cpd00610[0]',\n",
       "  'id': 'rxn00417',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glutamine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00471 (D-Glutamine and D-glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00053:0:0:\"L-Glutamine\";1:cpd00610:0:0:\"D-Glutamine\"'},\n",
       " {'abbreviation': 'ENTF-RXN.c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: SERAS; SERASr',\n",
       "   'EcoCyc: ENTF-RXN',\n",
       "   'MetaCyc: ENTF-RXN',\n",
       "   'iAF1260: SERASr',\n",
       "   'iJR904: SERASr',\n",
       "   'Name: (L-seryl)adenylate synthase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00054[0] <=> (1) cpd00012[0] + (1) cpd03452[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00054;cpd03452',\n",
       "  'definition': '(1) ATP[0] + (1) L-Serine[0] => (1) PPi[0] + (1) (L-Seryl)adenylate[0]',\n",
       "  'deltag': -11.13,\n",
       "  'deltagerr': 4.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.7.-'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00054[0] => (1) cpd00012[0] + (1) cpd03452[0]',\n",
       "  'id': 'rxn00418',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ENTF-RXN.c',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['MetaCyc: ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); ENTBACSYN-PWY (enterobactin biosynthesis); PWY-7751 (shinorine biosynthesis); PWY-7987 (vanchrobactin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Siderophores-Biosynthesis (Siderophore Biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00054:0:0:\"L-Serine\";1:cpd00012:0:0:\"PPi\";1:cpd03452:0:0:\"(L-Seryl)adenylate\"'},\n",
       " {'abbreviation': 'R00581',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00581',\n",
       "   'MetaCyc: SERINE-DEHYDROGENASE-RXN',\n",
       "   'Name: L-serine:NAD oxidoreductase (deaminating); L-serine:NAD+ 2-oxidoreductase (deaminating); L-serine:NAD+ oxidoreductase (deaminating); serine 2-dehydrogenase; serine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00054[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00145[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00054;cpd00067;cpd00145',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Serine[0] <=> (1) NADH[0] + (1) NH3[0] + (1) H+[0] + (1) Hydroxypyruvate[0]',\n",
       "  'deltag': 9.72,\n",
       "  'deltagerr': 0.2,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00054[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00145[0]',\n",
       "  'id': 'rxn00419',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-serine:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00054:0:0:\"L-Serine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00145:0:0:\"Hydroxypyruvate\"'},\n",
       " {'abbreviation': 'R00582',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PSERPHOSPHA-RXN; RXN0-5114',\n",
       "   'BiGG: PSP_L; PSP_Lpp; SER2',\n",
       "   'BrachyCyc: PSERPHOSPHA-RXN; RXN0-5114',\n",
       "   'KEGG: R00582',\n",
       "   'MetaCyc: PSERPHOSPHA-RXN; RXN0-5114',\n",
       "   'Name: L-O-Phosphoserine phosphohydrolase; O-phospho-L-serine phosphohydrolase; Phosphoserine phosphatase; phospho-L-serine phosphatase; phosphoserine phosphatase (L-serine); phosphoserinephosphatase-L-serine'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00738[0] <=> (1) cpd00009[0] + (1) cpd00054[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00054;cpd00738',\n",
       "  'definition': '(1) H2O[0] + (1) phosphoserine[0] <=> (1) Phosphate[0] + (1) L-Serine[0]',\n",
       "  'deltag': -2.46,\n",
       "  'deltagerr': 0.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00738[0] <=> (1) cpd00009[0] + (1) cpd00054[0]',\n",
       "  'id': 'rxn00420',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn26100;rxn29975;rxn30983',\n",
       "  'name': 'O-phospho-L-serine phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); SER-GLYSYN-PWY (superpathway of L-serine and glycine biosynthesis I); SERINE-BIOSYNTHESIS (L-serine Biosynthesis); SERSYN-PWY (L-serine biosynthesis I); SULFATE-CYS-PWY (superpathway of sulfate assimilation and cysteine biosynthesis); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00680 (Methane metabolism); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00738:0:0:\"phosphoserine\";1:cpd00009:0:0:\"Phosphate\";1:cpd00054:0:0:\"L-Serine\"'},\n",
       " {'abbreviation': 'R00584',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00584',\n",
       "   'MetaCyc: 2.7.1.80-RXN',\n",
       "   'Name: Pyrophosphate:L-serine O-phosphotransferase; diphosphate-serine phosphotransferase; diphosphate:L-serine O-phosphotransferase; pyrophosphate-L-serine phosphotransferase; pyrophosphate-serine phosphotransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00054[0] <=> (1) cpd00009[0] + (1) cpd00738[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00012;cpd00054;cpd00067;cpd00738',\n",
       "  'definition': '(1) PPi[0] + (1) L-Serine[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) phosphoserine[0]',\n",
       "  'deltag': -1.01,\n",
       "  'deltagerr': 0.08,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.80'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00054[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00738[0]',\n",
       "  'id': 'rxn00421',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'diphosphate:L-serine O-phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00054:0:0:\"L-Serine\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00738:0:0:\"phosphoserine\"'},\n",
       " {'abbreviation': 'R00585',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SERINE--PYRUVATE-AMINOTRANSFERASE-RXN',\n",
       "   'BiGG: SPA',\n",
       "   'BrachyCyc: SERINE--PYRUVATE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00585',\n",
       "   'MetaCyc: SERINE--PYRUVATE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-Serine:pyruvate aminotransferase; L-serine:pyruvate aminotransferase; SPT; Serine--pyruvate aminotransferase; hydroxypyruvate:L-alanine transaminase; serine-pyruvate aminotransferase,; serine-pyruvate transaminase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00054[0] <=> (1) cpd00035[0] + (1) cpd00145[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00035;cpd00054;cpd00145',\n",
       "  'definition': '(1) Pyruvate[0] + (1) L-Serine[0] <=> (1) L-Alanine[0] + (1) Hydroxypyruvate[0]',\n",
       "  'deltag': 0.94,\n",
       "  'deltagerr': 0.19,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.51'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00054[0] <=> (1) cpd00035[0] + (1) cpd00145[0]',\n",
       "  'id': 'rxn00422',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27104;rxn28984;rxn33642',\n",
       "  'name': 'L-Serine:pyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-8011 (L-serine biosynthesis II); SERINE-BIOSYNTHESIS (L-serine Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00054:0:0:\"L-Serine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd00145:0:0:\"Hydroxypyruvate\"'},\n",
       " {'abbreviation': 'R00586',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SERINE-O-ACETTRAN-RXN',\n",
       "   'BiGG: SERAT; SERATi; U48',\n",
       "   'BrachyCyc: SERINE-O-ACETTRAN-RXN',\n",
       "   'KEGG: R00586',\n",
       "   'MetaCyc: SERINE-O-ACETTRAN-RXN',\n",
       "   'Name: L-serine acetyltransferase; SATase; acetyl-CoA:L-serine O-acetyltransferase; serine O-acetyltransferase; serine O-acetyltransferase, irreversible; serine acetyltransferase; serine transacetylase; serineO-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00054[0] <=> (1) cpd00010[0] + (1) cpd00722[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00054;cpd00722',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Serine[0] => (1) CoA[0] + (1) O-Acetyl-L-serine[0]',\n",
       "  'deltag': -1.6,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.30'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00054[0] => (1) cpd00010[0] + (1) cpd00722[0]',\n",
       "  'id': 'rxn00423',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27108;rxn27109',\n",
       "  'name': 'acetyl-CoA:L-serine O-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Antibiotic-Biosynthesis (Antibiotic Biosynthesis); CYSTEINE-SYN (L-cysteine Biosynthesis); CYSTSYN-PWY (L-cysteine biosynthesis I); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-6395 (superpathway of seleno-compound metabolism); PWY-6936 (seleno-amino acid biosynthesis (plants)); PWY-7274 (D-cycloserine biosynthesis); PWY-7870 (L-cysteine biosynthesis VII (from <i>S</i>-sulfo-L-cysteine)); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SULFATE-CYS-PWY (superpathway of sulfate assimilation and cysteine biosynthesis); Selenium-Metabolism (Selenium Metabolism); Seleno-Amino-Acid-Detoxification (Seleno-Amino Acid Detoxification); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn00920 (Sulfur metabolism); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00054:0:0:\"L-Serine\";1:cpd00010:0:0:\"CoA\";1:cpd00722:0:0:\"O-Acetyl-L-serine\"'},\n",
       " {'abbreviation': 'R00588',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SERINE--GLYOXYLATE-AMINOTRANSFERASE-RXN; SERINE-GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'BrachyCyc: SERINE--GLYOXYLATE-AMINOTRANSFERASE-RXN; SERINE-GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'ChlamyCyc: SERINE-GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00588',\n",
       "   'MetaCyc: SERINE--GLYOXYLATE-AMINOTRANSFERASE-RXN; SERINE-GLYOXYLATE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-Serine:glyoxylate aminotransferase; L-serine:glyoxylate aminotransferase; SGAT; Serine--glyoxylate aminotransferase; serine-glyoxylate transaminase'],\n",
       "  'code': '(1) cpd00040[0] + (1) cpd00054[0] <=> (1) cpd00033[0] + (1) cpd00145[0]',\n",
       "  'compound_ids': 'cpd00033;cpd00040;cpd00054;cpd00145',\n",
       "  'definition': '(1) Glyoxalate[0] + (1) L-Serine[0] <=> (1) Glycine[0] + (1) Hydroxypyruvate[0]',\n",
       "  'deltag': -1.03,\n",
       "  'deltagerr': 0.42,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.45'],\n",
       "  'equation': '(1) cpd00040[0] + (1) cpd00054[0] <=> (1) cpd00033[0] + (1) cpd00145[0]',\n",
       "  'id': 'rxn00424',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28985;rxn28986;rxn33134',\n",
       "  'name': 'L-Serine:glyoxylate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Formaldehyde-Assimilation (Formaldehyde Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-181 (photorespiration); PWY-8011 (L-serine biosynthesis II); Photosynthesis (); SERINE-BIOSYNTHESIS (L-serine Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00680 (Methane metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00040:0:0:\"Glyoxalate\";-1:cpd00054:0:0:\"L-Serine\";1:cpd00033:0:0:\"Glycine\";1:cpd00145:0:0:\"Hydroxypyruvate\"'},\n",
       " {'abbreviation': 'R00589',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5.1.1.18-RXN',\n",
       "   'BrachyCyc: 5.1.1.18-RXN',\n",
       "   'CornCyc: 5.1.1.18-RXN',\n",
       "   'KEGG: R00589',\n",
       "   'MetaCyc: 5.1.1.18-RXN',\n",
       "   'Name: L-2-Amino-3-hydroxypropionic acid racemase; SRR; Serine racemase; serine racemase'],\n",
       "  'code': '(1) cpd00054[0] <=> (1) cpd00550[0]',\n",
       "  'compound_ids': 'cpd00054;cpd00550',\n",
       "  'definition': '(1) L-Serine[0] <=> (1) D-Serine[0]',\n",
       "  'deltag': -0.13,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.10', '5.1.1.18'],\n",
       "  'equation': '(1) cpd00054[0] <=> (1) cpd00550[0]',\n",
       "  'id': 'rxn00425',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18977;rxn32319',\n",
       "  'name': 'serine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Antibiotic-Resistance (Antibiotic Resistance); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); D-Amino-Acid-Degradation (D-Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); MISCELLANEOUS-DEG (Other Amino Acid Degradation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-6196 (D-serine metabolism); PWY-6455 (vancomycin resistance II); Vancomycin-Resistnace (Vancomycin Resistance)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00054:0:0:\"L-Serine\";1:cpd00550:0:0:\"D-Serine\"'},\n",
       " {'abbreviation': 'R00590',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00590',\n",
       "   'MetaCyc: RXN-15125',\n",
       "   'Name: L-Serine hydro-lyase; L-Serine hydro-lyase (deaminating); L-serine hydro-lyase'],\n",
       "  'code': '(1) cpd00054[0] <=> (1) cpd00001[0] + (1) cpd01495[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00054;cpd01495',\n",
       "  'definition': '(1) L-Serine[0] <=> (1) H2O[0] + (1) Dehydroalanine[0]',\n",
       "  'deltag': -1.44,\n",
       "  'deltagerr': 0.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.3.1.17'],\n",
       "  'equation': '(1) cpd00054[0] <=> (1) cpd00001[0] + (1) cpd01495[0]',\n",
       "  'id': 'rxn00426',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Serine hydro-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Glycine-Betaine-Degradation (Glycine Betaine Degradation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-3661 (glycine betaine degradation I); PWY-5497 (purine nucleobases degradation II (anaerobic)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Purine-Degradation (Purine Nucleotide Degradation); SERDEG-PWY (L-serine degradation); SERINE-DEG (L-serine Degradation)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00054:0:0:\"L-Serine\";1:cpd00001:0:0:\"H2O\";1:cpd01495:0:0:\"Dehydroalanine\"'},\n",
       " {'abbreviation': 'R00602',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00602_c',\n",
       "   'AraGEM: R_R00602_c',\n",
       "   'BiGG: PRDX',\n",
       "   'KEGG: R00602',\n",
       "   'MetaCyc: RXN-14189',\n",
       "   'Name: Hydorogen-peroxide:hydrogen-peroxide oxidoreductase; Peroxidase (multiple substrates); R_methanol_hydrogen_peroxide_oxidoreductase; methanol:hydrogen-peroxide oxidoreductase'],\n",
       "  'code': '(1) cpd00025[0] + (1) cpd00116[0] <=> (2) cpd00001[0] + (1) cpd00055[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00025;cpd00055;cpd00116',\n",
       "  'definition': '(1) H2O2[0] + (1) Methanol[0] => (2) H2O[0] + (1) Formaldehyde[0]',\n",
       "  'deltag': -69.76,\n",
       "  'deltagerr': 1.13,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.21', '1.11.1.6', '1.11.1.7'],\n",
       "  'equation': '(1) cpd00025[0] + (1) cpd00116[0] => (2) cpd00001[0] + (1) cpd00055[0]',\n",
       "  'id': 'rxn00427',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31397',\n",
       "  'name': 'methanol:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00025:0:0:\"H2O2\";-1:cpd00116:0:0:\"Methanol\";2:cpd00001:0:0:\"H2O\";1:cpd00055:0:0:\"Formaldehyde\"'},\n",
       " {'abbreviation': 'R00603',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00603',\n",
       "   'MetaCyc: DICHLOROMETHANE-DEHALOGENASE-RXN',\n",
       "   'Name: Dichloromethane chloride-lyase (chloride-hydrolysing); dichloromethane chloride-lyase (adding H2O, chloride-hydrolysing, formaldehyde-forming); dichloromethane chloride-lyase (adding H2O; chloride-hydrolysing; formaldehyde-forming); dichloromethane chloride-lyase (chloride-hydrolysing); dichloromethane dehalogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01530[0] <=> (1) cpd00055[0] + (2) cpd00099[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00055;cpd00067;cpd00099;cpd01530',\n",
       "  'definition': '(1) H2O[0] + (1) Dichloromethane[0] <=> (1) Formaldehyde[0] + (2) H+[0] + (2) Cl-[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.5.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01530[0] <=> (1) cpd00055[0] + (2) cpd00067[0] + (2) cpd00099[0]',\n",
       "  'id': 'rxn00428',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'dichloromethane chloride-lyase (adding H2O; chloride-hydrolysing; formaldehyde-forming)',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00625 (Chloroalkane and chloroalkene degradation)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01530:0:0:\"Dichloromethane\";1:cpd00055:0:0:\"Formaldehyde\";2:cpd00067:0:0:\"H+\";2:cpd00099:0:0:\"Cl-\"'},\n",
       " {'abbreviation': 'R00604',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FORMALDEHYDE-DEHYDROGENASE-RXN',\n",
       "   'BiGG: ALDD1; ALDD1x',\n",
       "   'KEGG: R00604',\n",
       "   'MetaCyc: FORMALDEHYDE-DEHYDROGENASE-RXN',\n",
       "   'PlantCyc: FORMALDEHYDE-DEHYDROGENASE-RXN',\n",
       "   'Name: Formaldehyde dehydrogenase; Formaldehyde:NAD+ oxidoreductase; NAD-dependent formaldehyde dehydrogenase; NAD-linked formaldehyde dehydrogenase; aldehyde dehydrogenase (formaldehyde, NAD); formaldehyde dehydrogenase; formaldehyde:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00055[0] <=> (1) cpd00004[0] + (1) cpd00047[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00047;cpd00055;cpd00067',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Formaldehyde[0] <=> (1) NADH[0] + (1) Formate[0] + (2) H+[0]',\n",
       "  'deltag': -10.51,\n",
       "  'deltagerr': 0.47,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.46'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00055[0] <=> (1) cpd00004[0] + (1) cpd00047[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00429',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33269;rxn33270;rxn33272',\n",
       "  'name': 'Formaldehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); FORMASS-PWY (formaldehyde oxidation IV (thiol-independent)); Formaldehyde-Oxidation (Formaldehyde Oxidation); PWY-1882 (superpathway of C1 compounds oxidation to CO<sub>2</sub>); PWY-7616 (methanol oxidation to carbon dioxide)',\n",
       "   'KEGG: rn00625 (Chloroalkane and chloroalkene degradation); rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00055:0:0:\"Formaldehyde\";1:cpd00004:0:0:\"NADH\";1:cpd00047:0:0:\"Formate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00605',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['CornCyc: ALCOHOL-DEHYDROGENASE-NADP+-RXN',\n",
       "   'KEGG: R00605',\n",
       "   'MaizeCyc: METHANOL-DEHYDROGENASE-RXN',\n",
       "   'MetaCyc: ALCOHOL-DEHYDROGENASE-NADP+-RXN; METHANOL-DEHYDROGENASE-RXN',\n",
       "   'Name: alcohol dehydrogenase (methanol); methanol dehydrogenase; methanol:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00116[0] <=> (1) cpd00004[0] + (1) cpd00055[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00055;cpd00067;cpd00116',\n",
       "  'definition': '(1) NAD[0] + (1) Methanol[0] <=> (1) NADH[0] + (1) Formaldehyde[0] + (1) H+[0]',\n",
       "  'deltag': 8.02,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.244'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00116[0] <=> (1) cpd00004[0] + (1) cpd00055[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00430',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38171',\n",
       "  'name': 'methanol:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); Methanol-Oxidation (Methanol Oxidation); PWY-6510 (methanol oxidation to formaldehyde II); PWY-7616 (methanol oxidation to carbon dioxide)',\n",
       "   'KEGG: rn00680 (Methane metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00116:0:0:\"Methanol\";1:cpd00004:0:0:\"NADH\";1:cpd00055:0:0:\"Formaldehyde\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'Methylamine:(acceptor) oxidoreductase (deaminating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Methylamine:(acceptor) oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00187[0] <=> (1) cpd00013[0] + (1) cpd00055[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00055;cpd00067;cpd00187',\n",
       "  'definition': '(1) H2O[0] + (1) Methanamine[0] <=> (1) NH3[0] + (1) Formaldehyde[0] + (2) H+[0]',\n",
       "  'deltag': -2.29,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.99.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00187[0] <=> (1) cpd00013[0] + (1) cpd00055[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00431',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Methylamine:(acceptor) oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00187:0:0:\"Methanamine\";1:cpd00013:0:0:\"NH3\";1:cpd00055:0:0:\"Formaldehyde\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00608',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00608',\n",
       "   'MaizeCyc: METHANOL-OXIDASE-RXN',\n",
       "   'MetaCyc: METHANOL-OXIDASE-RXN',\n",
       "   'Name: Methanol oxidase; Methanol:oxygen oxidereductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00116[0] <=> (1) cpd00025[0] + (1) cpd00055[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00055;cpd00116',\n",
       "  'definition': '(1) O2[0] + (1) Methanol[0] => (1) H2O2[0] + (1) Formaldehyde[0]',\n",
       "  'deltag': -23.7,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.13'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00116[0] => (1) cpd00025[0] + (1) cpd00055[0]',\n",
       "  'id': 'rxn00432',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20190',\n",
       "  'name': 'Methanol:oxygen oxidereductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Methanol-Oxidation (Methanol Oxidation); PWY-5506 (methanol oxidation to formaldehyde IV)',\n",
       "   'KEGG: rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00116:0:0:\"Methanol\";1:cpd00025:0:0:\"H2O2\";1:cpd00055:0:0:\"Formaldehyde\"'},\n",
       " {'abbreviation': 'R00610',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SARCOX-RXN',\n",
       "   'BiGG: SARCO; SARCOX',\n",
       "   'BrachyCyc: SARCOX-RXN',\n",
       "   'KEGG: R00610',\n",
       "   'MetaCyc: SARCOX-RXN',\n",
       "   'Name: Sarcosine:oxygen oxidoreductase (demethylating); sarcosine oxidase; sarcosine:oxygen oxidoreductase (demethylating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00183[0] <=> (1) cpd00025[0] + (1) cpd00033[0] + (1) cpd00055[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00033;cpd00055;cpd00183',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Sarcosine[0] => (1) H2O2[0] + (1) Glycine[0] + (1) Formaldehyde[0]',\n",
       "  'deltag': -17.34,\n",
       "  'deltagerr': 1.53,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.3.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00183[0] => (1) cpd00025[0] + (1) cpd00033[0] + (1) cpd00055[0]',\n",
       "  'id': 'rxn00433',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31075',\n",
       "  'name': 'sarcosine:oxygen oxidoreductase (demethylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); CRNFORCAT-PWY (creatinine degradation I); Creatinine-Degradation (Creatinine Degradation); Degradation (Degradation/Utilization/Assimilation); Glycine-Betaine-Degradation (Glycine Betaine Degradation); PWY-3661 (glycine betaine degradation I); PWY-4722 (creatinine degradation II)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00183:0:0:\"Sarcosine\";1:cpd00025:0:0:\"H2O2\";1:cpd00033:0:0:\"Glycine\";1:cpd00055:0:0:\"Formaldehyde\"'},\n",
       " {'abbreviation': 'R00612',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00612',\n",
       "   'MetaCyc: N6-METHYL-LYSINE-OXIDASE-RXN',\n",
       "   'Name: 6-N-methyl-L-lysine:oxygen oxidoreductase (demethylating); N6-Methyl-L-lysine:oxygen oxidoreductase (demethylating); N6-methyl-L-lysine:oxygen oxidoreductase (demethylating); N6-methyl-lysine oxidase; N6-methyllysine oxidase; epsilon-N-methyllysine demethylase; epsilon-alkyl-L-lysine:oxygen oxidoreductase; epsilon-alkyllysinase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd01771[0] <=> (1) cpd00025[0] + (1) cpd00039[0] + (1) cpd00055[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00039;cpd00055;cpd01771',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) N6-Methyl-L-lysine[0] => (1) H2O2[0] + (1) L-Lysine[0] + (1) Formaldehyde[0]',\n",
       "  'deltag': -15.69,\n",
       "  'deltagerr': 1.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.3.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd01771[0] => (1) cpd00025[0] + (1) cpd00039[0] + (1) cpd00055[0]',\n",
       "  'id': 'rxn00434',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N6-Methyl-L-lysine:oxygen oxidoreductase (demethylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd01771:0:0:\"N6-Methyl-L-lysine\";1:cpd00025:0:0:\"H2O2\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00055:0:0:\"Formaldehyde\"'},\n",
       " {'abbreviation': 'R00614',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00614',\n",
       "   'MetaCyc: RXN-11453',\n",
       "   'Name: Formaldehyde:formaldehyde oxidoreductase; aldehyde dismutase; cannizzanase; formaldehyde dismutase; formaldehyde:formaldehyde oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00055[0] <=> (1) cpd00047[0] + (1) cpd00116[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00047;cpd00055;cpd00067;cpd00116',\n",
       "  'definition': '(1) H2O[0] + (2) Formaldehyde[0] => (1) Formate[0] + (1) H+[0] + (1) Methanol[0]',\n",
       "  'deltag': -18.53,\n",
       "  'deltagerr': 1.04,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.98.1', '1.2.99.4'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00055[0] => (1) cpd00047[0] + (1) cpd00067[0] + (1) cpd00116[0]',\n",
       "  'id': 'rxn00435',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Formaldehyde:formaldehyde oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00625 (Chloroalkane and chloroalkene degradation); rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00055:0:0:\"Formaldehyde\";1:cpd00047:0:0:\"Formate\";1:cpd00067:0:0:\"H+\";1:cpd00116:0:0:\"Methanol\"'},\n",
       " {'abbreviation': 'R00615',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00615_c',\n",
       "   'BiGG: TDP',\n",
       "   'EcoCyc: RXN0-3542',\n",
       "   'KEGG: R00615',\n",
       "   'MetaCyc: RXN0-3542',\n",
       "   'Name: Thiamin diphosphate phosphohydrolase; thiamin pyrophosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00056[0] <=> (1) cpd00009[0] + (1) cpd00793[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00056;cpd00067;cpd00793',\n",
       "  'definition': '(1) H2O[0] + (1) TPP[0] => (1) Phosphate[0] + (1) H+[0] + (1) Thiamine phosphate[0]',\n",
       "  'deltag': 6.36,\n",
       "  'deltagerr': 0.98,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.15'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00056[0] => (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00793[0]',\n",
       "  'id': 'rxn00436',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31790',\n",
       "  'name': 'Thiamin diphosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00056:0:0:\"TPP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00793:0:0:\"Thiamine phosphate\"'},\n",
       " {'abbreviation': 'R00616',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00616_c',\n",
       "   'AraCyc: THIAMIN-DIPHOSPHATE-KINASE-RXN',\n",
       "   'BiGG: THI80_2; TMDPPK',\n",
       "   'KEGG: R00616',\n",
       "   'MetaCyc: THIAMIN-DIPHOSPHATE-KINASE-RXN',\n",
       "   'Name: ATP:thiamin-diphosphate phosphotransferase; ATP:thiamine-diphosphate phosphotransferase; TDP kinase; Thiamine pyrophosphokinase; protein bound thiamin diphosphate:ATP phosphoryltransferase; thiamin diphosphate kinase; thiamin diphosphate phosphotransferase; thiamin pyrophosphate kinase; thiamine diphosphate kinase; thiamine-diphosphate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00056[0] <=> (1) cpd00008[0] + (1) cpd01937[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00056;cpd01937',\n",
       "  'definition': '(1) ATP[0] + (1) TPP[0] <=> (1) ADP[0] + (1) Thiamin triphosphate[0]',\n",
       "  'deltag': 1.05,\n",
       "  'deltagerr': 0.3,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.15', '2.7.6.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00056[0] <=> (1) cpd00008[0] + (1) cpd01937[0]',\n",
       "  'id': 'rxn00437',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:thiamine-diphosphate phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-7369 (thiamine triphosphate metabolism); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00056:0:0:\"TPP\";1:cpd00008:0:0:\"ADP\";1:cpd01937:0:0:\"Thiamin triphosphate\"'},\n",
       " {'abbreviation': 'R00617',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: THI-P-KIN-RXN',\n",
       "   'BiGG: TMPK; TMPKr; U76',\n",
       "   'CornCyc: THI-P-KIN-RXN',\n",
       "   'KEGG: R00617',\n",
       "   'MetaCyc: THI-P-KIN-RXN',\n",
       "   'Name: ATP:thiamin-phosphate phosphotransferase; ATP:thiamine-phosphate phosphotransferase; thiamin monophosphatase; thiamin monophosphokinase; thiamin-monophosphate kinase; thiamine-phosphate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00793[0] <=> (1) cpd00008[0] + (1) cpd00056[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00056;cpd00793',\n",
       "  'definition': '(1) ATP[0] + (1) Thiamine phosphate[0] <=> (1) ADP[0] + (1) TPP[0]',\n",
       "  'deltag': -12.52,\n",
       "  'deltagerr': 0.98,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.16'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00793[0] <=> (1) cpd00008[0] + (1) cpd00056[0]',\n",
       "  'id': 'rxn00438',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38718',\n",
       "  'name': 'ATP:thiamin-phosphate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-6893 (thiamine diphosphate biosynthesis II (Bacillus)); PWY-6894 (thiamine diphosphate biosynthesis I (E. coli)); PWY-6895 (superpathway of thiamine diphosphate biosynthesis II); PWY-6896 (thiamine salvage I); PWY-6897 (thiamine salvage II); THISYN-PWY (superpathway of thiamine diphosphate biosynthesis I); Thiamin-Diphosphate-Biosynthesis (Thiamine Diphosphate Biosynthesis); Thiamin-Salvage (Thiamine Salvage); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00793:0:0:\"Thiamine phosphate\";1:cpd00008:0:0:\"ADP\";1:cpd00056:0:0:\"TPP\"'},\n",
       " {'abbreviation': 'R00618',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00618',\n",
       "   'MetaCyc: THIAMIN-TRIPHOSPHATASE-RXN',\n",
       "   'TS_Athaliana: rxn587_g; rxn587_m',\n",
       "   'Name: Thiamin-triphosphate phosphohydrolase; thiamine-triphosphatase; thiamine-triphosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01937[0] <=> (1) cpd00009[0] + (1) cpd00056[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00056;cpd00067;cpd01937',\n",
       "  'definition': '(1) H2O[0] + (1) Thiamin triphosphate[0] => (1) Phosphate[0] + (1) TPP[0] + (1) H+[0]',\n",
       "  'deltag': -7.21,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.28'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01937[0] => (1) cpd00009[0] + (1) cpd00056[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00439',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32612;rxn32616',\n",
       "  'name': 'Thiamin-triphosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-7369 (thiamine triphosphate metabolism); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01937:0:0:\"Thiamin triphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00056:0:0:\"TPP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00619',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: THIAMIN-PYROPHOSPHOKINASE-RXN',\n",
       "   'BiGG: THI80_1; TMDPK',\n",
       "   'BrachyCyc: THIAMIN-PYROPHOSPHOKINASE-RXN',\n",
       "   'KEGG: R00619',\n",
       "   'MetaCyc: THIAMIN-PYROPHOSPHOKINASE-RXN',\n",
       "   'Name: ATP:thiamin pyrophosphotransferase; ATP:thiamine diphosphotransferase; TPTase; Thiamine pyrophosphokinase; thiamin kinase; thiamin pyrophosphokinase; thiamin pyrophosphotransferase; thiamin:ATP pyrophosphotransferase; thiamine diphosphokinase; thiamine pyrophosphokinase; thiaminokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00305[0] <=> (1) cpd00018[0] + (1) cpd00056[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00018;cpd00056;cpd00067;cpd00305',\n",
       "  'definition': '(1) ATP[0] + (1) Thiamin[0] => (1) AMP[0] + (1) TPP[0] + (1) H+[0]',\n",
       "  'deltag': -1.16,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.6.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00305[0] => (1) cpd00018[0] + (1) cpd00056[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00440',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32226',\n",
       "  'name': 'ATP:thiamine diphosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PWY-6898 (thiamine salvage III); PWY-6907 (thiamine diphosphate biosynthesis III (Staphylococcus)); PWY-6908 (thiamine diphosphate biosynthesis IV (eukaryotes)); PWY-7356 (thiamine salvage IV (yeast)); THISYNARA-PWY (superpathway of thiamine diphosphate biosynthesis III (eukaryotes)); Thiamin-Diphosphate-Biosynthesis (Thiamine Diphosphate Biosynthesis); Thiamin-Salvage (Thiamine Salvage); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00730 (Thiamine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00305:0:0:\"Thiamin\";1:cpd00018:0:0:\"AMP\";1:cpd00056:0:0:\"TPP\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00621',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00621_m',\n",
       "   'AraGEM: R_R00621_m',\n",
       "   'KEGG: R00621',\n",
       "   'Maize_C4GEM: R_R00621_m',\n",
       "   'iRS1563: R_R00621_m',\n",
       "   'Name: 2-Oxoglutarate:Thiamin diphosphate 2-oxidoreductase(decarboxylating; 2-oxoglutarate dehydrogenase E1 component, putative / oxoglutarate decarboxylase, putative / alpha-ketoglutaric dehydrogenase, putative; and acceptor-succinylating)'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00056[0] <=> (1) cpd00011[0] + (1) cpd03189[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00024;cpd00056;cpd00067;cpd03189',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) TPP[0] + (1) H+[0] => (1) CO2[0] + (1) 3-Carboxy-1-hydroxypropyl-ThPP[0]',\n",
       "  'deltag': -6.56,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.4.2'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00056[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd03189[0]',\n",
       "  'id': 'rxn00441',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31217',\n",
       "  'name': 'R00621',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00020 (Citrate cycle (TCA cycle))'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00056:0:0:\"TPP\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd03189:0:0:\"3-Carboxy-1-hydroxypropyl-ThPP\"'},\n",
       " {'abbreviation': '2-OXOGLUT-DECARBOX-RXN.c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2-OXOGLUT-DECARBOX-RXN',\n",
       "   'BiGG: OXGDC2',\n",
       "   'DF_Athaliana: reac_43',\n",
       "   'RiceCyc: 2-OXOGLUT-DECARBOX-RXN',\n",
       "   'SorghumCyc: 2-OXOGLUT-DECARBOX-RXN',\n",
       "   'Name: 2-oxoglutarate decarboxylase; 2-oxoglutaratedecarboxylase; alpha-ketoglutarate decarboxylase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00056[0] <=> (1) cpd00011[0] + (1) cpd03450[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00024;cpd00056;cpd00067;cpd03450',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) TPP[0] + (1) H+[0] <= (1) CO2[0] + (1) Succinatesemialdehyde-thiamindiphosphateanion[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['4.1.1.71'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00056[0] + (1) cpd00067[0] <= (1) cpd00011[0] + (1) cpd03450[0]',\n",
       "  'id': 'rxn00442',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'alpha-ketoglutarate decarboxylase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Secondary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00056:0:0:\"TPP\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd03450:0:0:\"Succinatesemialdehyde-thiamindiphosphateanion\"'},\n",
       " {'abbreviation': 'R00640',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00640_c',\n",
       "   'AraGEM: R_R00640_c',\n",
       "   'KEGG: R00640',\n",
       "   'Maize_C4GEM: R_R00640_c',\n",
       "   'iRS1597: R00640_c',\n",
       "   'Name: L-Galactono-1,4-lactone:ferricytochrome-c oxidoreductase'],\n",
       "  'code': '(2) cpd00109[0] + (1) cpd00820[0] <=> (1) cpd00059[0] + (2) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00059;cpd00067;cpd00109;cpd00110;cpd00820',\n",
       "  'definition': '(2) Cytochrome c3+[0] + (1) L-Galactono-1,4-lactone[0] <=> (1) L-Ascorbate[0] + (3) H+[0] + (2) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.2.3'],\n",
       "  'equation': '(2) cpd00109[0] + (1) cpd00820[0] <=> (1) cpd00059[0] + (3) cpd00067[0] + (2) cpd00110[0]',\n",
       "  'id': 'rxn00443',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Galactono-1,4-lactone:ferricytochrome-c oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00053 (Ascorbate and aldarate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00109:0:0:\"Cytochrome c3+\";-1:cpd00820:0:0:\"L-Galactono-1,4-lactone\";1:cpd00059:0:0:\"L-Ascorbate\";3:cpd00067:0:0:\"H+\";2:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'R00644',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: L-ASCORBATE-PEROXIDASE-RXN; RXN-12440',\n",
       "   'BrachyCyc: L-ASCORBATE-PEROXIDASE-RXN',\n",
       "   'ChlamyCyc: L-ASCORBATE-PEROXIDASE-RXN',\n",
       "   'KEGG: R00644',\n",
       "   'MetaCyc: RXN-12440',\n",
       "   'Name: Ascorbic acid peroxidase; L-Ascorbate:hydrogen-peroxide oxidoreductase; L-ascorbate peroxidase; L-ascorbate:hydrogen-peroxide oxidoreductase; L-ascorbic acid peroxidase; L-ascorbic acid-specific peroxidase; ascorbate peroxidase; ascorbic acid peroxidase'],\n",
       "  'code': '(1) cpd00025[0] + (1) cpd00059[0] <=> (2) cpd00001[0] + (1) cpd00335[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00025;cpd00059;cpd00067;cpd00335',\n",
       "  'definition': '(1) H2O2[0] + (1) L-Ascorbate[0] + (1) H+[0] => (2) H2O[0] + (1) Dehydroascorbate[0]',\n",
       "  'deltag': -74.63,\n",
       "  'deltagerr': 2.18,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.11'],\n",
       "  'equation': '(1) cpd00025[0] + (1) cpd00059[0] + (1) cpd00067[0] => (2) cpd00001[0] + (1) cpd00335[0]',\n",
       "  'id': 'rxn00444',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27962;rxn33639',\n",
       "  'name': 'L-ascorbate:hydrogen-peroxide oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Ascorbate-Degradation (L-Ascorbate Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-6959 (L-ascorbate degradation V); PWY-6960 (L-ascorbate degradation III); PWY-6961 (L-ascorbate degradation II (bacterial, aerobic))',\n",
       "   'KEGG: rn00053 (Ascorbate and aldarate metabolism); rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00025:0:0:\"H2O2\";-1:cpd00059:0:0:\"L-Ascorbate\";-1:cpd00067:0:0:H+;2:cpd00001:0:0:\"H2O\";1:cpd00335:0:0:\"Dehydroascorbate\"'},\n",
       " {'abbreviation': 'R00643',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00643',\n",
       "   'MetaCyc: 1.3.3.12-RXN',\n",
       "   'iRS1563: R_R00643_c',\n",
       "   'Name: L-Galactono-1,4-lactone:oxygen 3-oxidoreductase; L-galactono-1,4-lactone oxidase; L-galactono-1,4-lactone:oxygen 3-oxidoreductase; L-galactonolactone oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00820[0] <=> (1) cpd00025[0] + (1) cpd00059[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00059;cpd00067;cpd00820',\n",
       "  'definition': '(1) O2[0] + (1) L-Galactono-1,4-lactone[0] => (1) H2O2[0] + (1) L-Ascorbate[0] + (1) H+[0]',\n",
       "  'deltag': -32.0,\n",
       "  'deltagerr': 1.07,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.3.3.12'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00820[0] => (1) cpd00025[0] + (1) cpd00059[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00445',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Galactono-1,4-lactone:oxygen 3-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00053 (Ascorbate and aldarate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00820:0:0:\"L-Galactono-1,4-lactone\";1:cpd00025:0:0:\"H2O2\";1:cpd00059:0:0:\"L-Ascorbate\";1:cpd00067:0:0:H+'},\n",
       " {'abbreviation': 'R_R00645_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['iRS1563: R_R00645_c',\n",
       "   'iRS1597: R00645_c',\n",
       "   'Name: R_ascorbate_acceptor_oxidoreductase'],\n",
       "  'code': '(1) cpd00059[0] <=> (1) cpd00766[0]',\n",
       "  'compound_ids': 'cpd00059;cpd00067;cpd00766',\n",
       "  'definition': '(1) L-Ascorbate[0] <=> (2) H+[0] + (1) Ascorbate radical[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.-.-'],\n",
       "  'equation': '(1) cpd00059[0] <=> (2) cpd00067[0] + (1) cpd00766[0]',\n",
       "  'id': 'rxn00447',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R_ascorbate_acceptor_oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'CI:1',\n",
       "  'stoichiometry': '-1:cpd00059:0:0:\"L-Ascorbate\";2:cpd00067:0:0:\"H+\";1:cpd00766:0:0:\"Ascorbate radical\"'},\n",
       " {'abbreviation': 'R00646',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASCORBATE-23-DIOXYGENASE-RXN',\n",
       "   'ChlamyCyc: ASCORBATE-23-DIOXYGENASE-RXN',\n",
       "   'KEGG: R00646',\n",
       "   'MetaCyc: ASCORBATE-23-DIOXYGENASE-RXN',\n",
       "   'Name: AAoxygenase; Ascorbate:oxygen 2,3-oxidoreductase (bond-cleaving); ascorbate 2,3-dioxygenase; ascorbate:oxygen 2,3-oxidoreductase (bond-cleaving)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00059[0] <=> (1) cpd00180[0] + (1) cpd01138[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00059;cpd00067;cpd00180;cpd01138',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Ascorbate[0] => (2) H+[0] + (1) Oxalate[0] + (1) Threonate[0]',\n",
       "  'deltag': -120.46,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.13'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00059[0] => (2) cpd00067[0] + (1) cpd00180[0] + (1) cpd01138[0]',\n",
       "  'id': 'rxn00448',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R00646',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00053 (Ascorbate and aldarate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00059:0:0:\"L-Ascorbate\";2:cpd00067:0:0:\"H+\";1:cpd00180:0:0:\"Oxalate\";1:cpd01138:0:0:\"Threonate\"'},\n",
       " {'abbreviation': 'R00647',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00647', 'Name: '],\n",
       "  'code': '(1) cpd00059[0] <=> (1) cpd02098[0]',\n",
       "  'compound_ids': 'cpd00059;cpd00067;cpd02098',\n",
       "  'definition': '(1) L-Ascorbate[0] + (1) H+[0] => (1) L-xylo-Hexulonolactone[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 0.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.8'],\n",
       "  'equation': '(1) cpd00059[0] + (1) cpd00067[0] => (1) cpd02098[0]',\n",
       "  'id': 'rxn00449',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R00647',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00053 (Ascorbate and aldarate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00059:0:0:\"L-Ascorbate\";-1:cpd00067:0:0:H+;1:cpd02098:0:0:\"L-xylo-Hexulonolactone\"'},\n",
       " {'abbreviation': 'R00648',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: L-AMINO-ACID-OXIDASE-RXN',\n",
       "   'KEGG: R00648',\n",
       "   'Name: L-Methionine:oxygen oxidoreductase (deaminating); L-amino acid oxidase; Ophio-amino-acid oxidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00060[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00869[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00060;cpd00869',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Methionine[0] => (1) NH3[0] + (1) H2O2[0] + (1) 4-methylthio 2-oxobutyrate[0]',\n",
       "  'deltag': -25.52,\n",
       "  'deltagerr': 1.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00060[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00869[0]',\n",
       "  'id': 'rxn00450',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38887',\n",
       "  'name': 'L-Methionine:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00060:0:0:\"L-Methionine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00869:0:0:\"4-methylthio 2-oxobutyrate\"'},\n",
       " {'abbreviation': 'R00649',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: METHIONINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'BrachyCyc: METHIONINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'CornCyc: METHIONINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'KEGG: R00649',\n",
       "   'MetaCyc: METHIONINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine:L-methionine S-methyltransferase; S-adenosyl methionine:methionine methyl transferase; S-adenosyl-L-methionine:L-methionine S-methyltransferase; S-adenosylmethionine transmethylase; S-adenosylmethionine-methionine methyltransferase; methionine S-methyltransferase; methionine methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00060[0] <=> (1) cpd00019[0] + (1) cpd02027[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00060;cpd02027',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) L-Methionine[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) S-Methyl-L-methionine[0]',\n",
       "  'deltag': -0.16,\n",
       "  'deltagerr': 0.68,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.12'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00060[0] <=> (1) cpd00019[0] + (1) cpd02027[0]',\n",
       "  'id': 'rxn00451',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:L-methionine S-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Dimethylsulfoniopropionate-Biosynthesis (Dimethylsulfoniopropanoate Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Metabolic-Regulators (Metabolic Regulator Biosynthesis); Methionine-Salvage (L-methionine Salvage); Organic-solutes-Biosynthesis (Organic Solute Biosynthesis); PWY-5441 (S-methyl-L-methionine cycle); PWY-6054 (dimethylsulfoniopropanoate biosynthesis I (Wollastonia)); PWY-6055 (dimethylsulfoniopropanoate biosynthesis II (Spartina))'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00060:0:0:\"L-Methionine\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd02027:0:0:\"S-Methyl-L-methionine\"'},\n",
       " {'abbreviation': 'R00650',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HOMOCYSTEINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'BiGG: HCYSMT; MHT1; SAM4',\n",
       "   'BrachyCyc: HOMOCYSTEINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'KEGG: R00650',\n",
       "   'MetaCyc: HOMOCYSTEINE-S-METHYLTRANSFERASE-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine:L-homocysteine S-methyltransferase; S-adenosyl-L-methionine:L-homocysteine S-methyltransferase; homocysteine S-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00135[0] <=> (1) cpd00019[0] + (1) cpd00060[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00060;cpd00067;cpd00135',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Homocysteine[0] => (1) S-Adenosyl-homocysteine[0] + (1) L-Methionine[0] + (1) H+[0]',\n",
       "  'deltag': -13.15,\n",
       "  'deltagerr': 4.96,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.1.1.10'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00135[0] => (1) cpd00019[0] + (1) cpd00060[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00452',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:L-homocysteine S-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00135:0:0:\"Homocysteine\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00060:0:0:\"L-Methionine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00651',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AHSERL; MET17_1',\n",
       "   'KEGG: R00651',\n",
       "   'MetaCyc: O-ACETYLHOMOSERINE-THIOL-LYASE-RXN',\n",
       "   'TS_Athaliana: rxn588_c; rxn588_p',\n",
       "   'iAO358: rll_101',\n",
       "   'Name: O-Acetyl-L-homoserine acetate-lyase (adding methanethiol); O-acetyl-L-homoserine acetate-lyase (adding methanethiol); O-acetyl-L-homoserine sulfhydrolase; O-acetyl-L-homoserine:methanethiol; O-acetyl-L-homoserine:methanethiol 3-amino-3-carboxypropyltransferase; O-acetylhomoserine (thiol)-lyase; O-acetylhomoserine aminocarboxypropyltransferase; O-acetylhomoserine sulfhydrolase; O-acetylhomoserine sulfhydrylase; methionine synthase'],\n",
       "  'code': '(1) cpd00324[0] + (1) cpd00790[0] <=> (1) cpd00029[0] + (1) cpd00060[0]',\n",
       "  'compound_ids': 'cpd00029;cpd00060;cpd00067;cpd00324;cpd00790',\n",
       "  'definition': '(1) MTTL[0] + (1) O-Acetyl-L-homoserine[0] <=> (1) Acetate[0] + (1) L-Methionine[0] + (1) H+[0]',\n",
       "  'deltag': -13.64,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.5.1.49', '4.2.99.10'],\n",
       "  'equation': '(1) cpd00324[0] + (1) cpd00790[0] <=> (1) cpd00029[0] + (1) cpd00060[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00453',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32345',\n",
       "  'name': 'O-acetyl-L-homoserine:methanethiol 3-amino-3-carboxypropyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); MTR-1P-Degradation (<i>S</i>-methyl-5-thio-&alpha;-D-ribose 1-phosphate degradation); Methionine-Salvage (L-methionine Salvage); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-7174 (<i>S</i>-methyl-5-thio-&alpha;-D-ribose 1-phosphate degradation II)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00324:0:0:\"MTTL\";-1:cpd00790:0:0:\"O-Acetyl-L-homoserine\";1:cpd00029:0:0:\"Acetate\";1:cpd00060:0:0:\"L-Methionine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00652',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: R15-RXN',\n",
       "   'CornCyc: R15-RXN',\n",
       "   'KEGG: R00652',\n",
       "   'MaizeCyc: R15-RXN',\n",
       "   'MetaCyc: METHIONINE--GLYOXYLATE-TRANSAMINASE-RXN',\n",
       "   'Name: L-Methionine:glyoxylate aminotransferase; L-methionine:glyoxylate aminotransferase; MGAT; methionine-glyoxylate aminotransferase; methionine-glyoxylate transaminase'],\n",
       "  'code': '(1) cpd00040[0] + (1) cpd00060[0] <=> (1) cpd00033[0] + (1) cpd00869[0]',\n",
       "  'compound_ids': 'cpd00033;cpd00040;cpd00060;cpd00869',\n",
       "  'definition': '(1) Glyoxalate[0] + (1) L-Methionine[0] <=> (1) Glycine[0] + (1) 4-methylthio 2-oxobutyrate[0]',\n",
       "  'deltag': -4.55,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.73'],\n",
       "  'equation': '(1) cpd00040[0] + (1) cpd00060[0] <=> (1) cpd00033[0] + (1) cpd00869[0]',\n",
       "  'id': 'rxn00454',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Methionine:glyoxylate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00040:0:0:\"Glyoxalate\";-1:cpd00060:0:0:\"L-Methionine\";1:cpd00033:0:0:\"Glycine\";1:cpd00869:0:0:\"4-methylthio 2-oxobutyrate\"'},\n",
       " {'abbreviation': 'R00653',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: FMETDF',\n",
       "   'KEGG: R00653',\n",
       "   'MetaCyc: FORMYLMETHIONINE-DEFORMYLASE-RXN',\n",
       "   'SorghumCyc: FORMYLMETHIONINE-DEFORMYLASE-RXN',\n",
       "   'iMO1053-PAO1: FMETDF',\n",
       "   'Name: N-Formyl-L-methionine amidohydrolase; N-formyl-L-methionine amidohydrolase; formylmethionine deformylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02012[0] <=> (1) cpd00047[0] + (1) cpd00060[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00047;cpd00060;cpd02012',\n",
       "  'definition': '(1) H2O[0] + (1) N-Formyl-L-methionine[0] <=> (1) Formate[0] + (1) L-Methionine[0]',\n",
       "  'deltag': -5.28,\n",
       "  'deltagerr': 1.38,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.31'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02012[0] <=> (1) cpd00047[0] + (1) cpd00060[0]',\n",
       "  'id': 'rxn00455',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Formyl-L-methionine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02012:0:0:\"N-Formyl-L-methionine\";1:cpd00047:0:0:\"Formate\";1:cpd00060:0:0:\"L-Methionine\"'},\n",
       " {'abbreviation': 'R00654',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: METHIONINE-GAMMA-LYASE-RXN',\n",
       "   'BiGG: METGL',\n",
       "   'BrachyCyc: METHIONINE-GAMMA-LYASE-RXN',\n",
       "   'KEGG: R00654',\n",
       "   'MetaCyc: METHIONINE-GAMMA-LYASE-RXN',\n",
       "   'Name: L-Methionine methanethiol-lyase (deaminating); L-methioninase; L-methionine gamma-lyase; L-methionine methanethiol-lyase (deaminating); L-methionine methanethiol-lyase (deaminating, 2-oxobutanoate-forming); L-methionine methanethiol-lyase (deaminating;2-oxobutanoate-forming); methioninase; methionine dethiomethylase; methionine g-lyase; methionine gamma-lyase; methionine lyase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00060[0] <=> (1) cpd00013[0] + (1) cpd00094[0] + (1) cpd00324[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00060;cpd00094;cpd00324',\n",
       "  'definition': '(1) H2O[0] + (1) L-Methionine[0] => (1) NH3[0] + (1) 2-Oxobutyrate[0] + (1) MTTL[0]',\n",
       "  'deltag': -2.23,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.4.1.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00060[0] => (1) cpd00013[0] + (1) cpd00094[0] + (1) cpd00324[0]',\n",
       "  'id': 'rxn00456',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30078;rxn30731',\n",
       "  'name': 'L-methionine methanethiol-lyase (deaminating;2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-7793 (dimethyl sulfide biosynthesis from methionine); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00060:0:0:\"L-Methionine\";1:cpd00013:0:0:\"NH3\";1:cpd00094:0:0:\"2-Oxobutyrate\";1:cpd00324:0:0:\"MTTL\"'},\n",
       " {'abbreviation': 'R00655',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00655',\n",
       "   'MetaCyc: METHIONINE-RACEMASE-RXN',\n",
       "   'Name: L-Methionine racemase; methionine racemase'],\n",
       "  'code': '(1) cpd00060[0] <=> (1) cpd00637[0]',\n",
       "  'compound_ids': 'cpd00060;cpd00637',\n",
       "  'definition': '(1) L-Methionine[0] <=> (1) D-Methionine[0]',\n",
       "  'deltag': -0.64,\n",
       "  'deltagerr': 0.3,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.2'],\n",
       "  'equation': '(1) cpd00060[0] <=> (1) cpd00637[0]',\n",
       "  'id': 'rxn00457',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'methionine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00060:0:0:\"L-Methionine\";1:cpd00637:0:0:\"D-Methionine\"'},\n",
       " {'abbreviation': 'R00656',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00656',\n",
       "   'MetaCyc: METHIONINE-DECARBOXYLASE-RXN',\n",
       "   'Name: L-Methionine carboxy-lyase; L-methionine carboxy-lyase; L-methionine carboxy-lyase (3-methylthiopropanamine-forming); L-methionine decarboxylase; methionine decarboxylase'],\n",
       "  'code': '(1) cpd00060[0] <=> (1) cpd00011[0] + (1) cpd02131[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00060;cpd00067;cpd02131',\n",
       "  'definition': '(1) L-Methionine[0] + (1) H+[0] => (1) CO2[0] + (1) 3-Methylthiopropanamine[0]',\n",
       "  'deltag': -5.72,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.57'],\n",
       "  'equation': '(1) cpd00060[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd02131[0]',\n",
       "  'id': 'rxn00458',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-methionine carboxy-lyase (3-methylthiopropanamine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00060:0:0:\"L-Methionine\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd02131:0:0:\"3-Methylthiopropanamine\"'},\n",
       " {'abbreviation': 'R00658',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2PGADEHYDRAT-RXN',\n",
       "   'BiGG: ENO; ENO1; ENO2; ERR1_1; ERR1_2; ERR2',\n",
       "   'BrachyCyc: 2PGADEHYDRAT-RXN',\n",
       "   'KEGG: R00658',\n",
       "   'MetaCyc: 2PGADEHYDRAT-RXN',\n",
       "   'Name: 14-3-2-protein; 2-Phospho-D-glucerate hydro-lyase; 2-Phospho-D-glycerate hydro-lyase; 2-phospho-D-glycerate hydro-lyase; 2-phospho-D-glycerate hydro-lyase (phosphoenolpyruvate-forming); 2-phosphoglycerate dehydratase; 2-phosphoglycerate enolase; 2-phosphoglyceric dehydratase; Phosphopyruvate hydratase; enolase; gamma-enolase; nervous-system specific enolase; phosphoenolpyruvate hydratase; phosphopyruvate hydratase'],\n",
       "  'code': '(1) cpd00482[0] <=> (1) cpd00001[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00061;cpd00482',\n",
       "  'definition': '(1) 2-Phospho-D-glycerate[0] <=> (1) H2O[0] + (1) Phosphoenolpyruvate[0]',\n",
       "  'deltag': -0.98,\n",
       "  'deltagerr': 0.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.11'],\n",
       "  'equation': '(1) cpd00482[0] <=> (1) cpd00001[0] + (1) cpd00061[0]',\n",
       "  'id': 'rxn00459',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17748;rxn30576;rxn33498;rxn35218;rxn37952;rxn37953',\n",
       "  'name': '2-phospho-D-glycerate hydro-lyase (phosphoenolpyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROFRUCAT-PWY (homolactic fermentation); ANAGLYCOLYSIS-PWY (glycolysis III (from glucose)); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Entner-Duodoroff-Pathways (Entner-Duodoroff Pathways); Ethylene-Biosynthesis (Ethylene Biosynthesis); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Formaldehyde-Assimilation (Formaldehyde Assimilation); GLUCONEO-PWY (gluconeogenesis I); GLYCOLYSIS (glycolysis I (from glucose 6-phosphate)); GLYCOLYSIS-E-D (superpathway of glycolysis and the Entner-Doudoroff pathway); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYCOLYSIS-VARIANTS (Glycolysis); Gluconeogenesis (Gluconeogenesis); HEXITOLDEGSUPER-PWY (superpathway of hexitol degradation (bacteria)); HORMONE-SYN (Hormone Biosynthesis); NPGLUCAT-PWY (Entner-Doudoroff pathway II (non-phosphorylative)); P122-PWY (heterolactic fermentation); P124-PWY (Bifidobacterium shunt); P341-PWY (glycolysis V (Pyrococcus)); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-1042 (glycolysis IV (plant cytosol)); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-2221 (Entner-Doudoroff pathway III (semi-phosphorylative)); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-5484 (glycolysis II (from fructose 6-phosphate)); PWY-5723 (Rubisco shunt); PWY-6142 (gluconeogenesis II (<i>Methanobacterium thermoautotrophicum</i>)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-6901 (superpathway of glucose and xylose degradation); PWY-7003 (glycerol degradation to butanol); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY-8004 (Entner-Doudoroff pathway I); PWY66-399 (gluconeogenesis III); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00680 (Methane metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00482:0:0:\"2-Phospho-D-glycerate\";1:cpd00001:0:0:\"H2O\";1:cpd00061:0:0:\"Phosphoenolpyruvate\"'},\n",
       " {'abbreviation': 'R00659',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00659',\n",
       "   'Name: UTP:pyruvate 2-O-phosphotransferase; UTP:pyruvate O2-phosphotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00062[0] <=> (1) cpd00014[0] + (1) cpd00061[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00020;cpd00061;cpd00062;cpd00067',\n",
       "  'definition': '(1) Pyruvate[0] + (1) UTP[0] <=> (1) UDP[0] + (1) Phosphoenolpyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 7.21,\n",
       "  'deltagerr': 0.33,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.40'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00062[0] <=> (1) cpd00014[0] + (1) cpd00061[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00460',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UTP:pyruvate 2-O-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00062:0:0:\"UTP\";1:cpd00014:0:0:\"UDP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00660',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: UAGCVT',\n",
       "   'BrachyCyc: UDPNACETYLGLUCOSAMENOLPYRTRANS-RXN',\n",
       "   'EcoCyc: UDPNACETYLGLUCOSAMENOLPYRTRANS-RXN',\n",
       "   'KEGG: R00660',\n",
       "   'MetaCyc: UDPNACETYLGLUCOSAMENOLPYRTRANS-RXN',\n",
       "   \"Name: 1-carboxyvinyl-transferase; MurA transferase; Phosphoenolpyruvate:UDP-N-acetyl-D-glucosamine; Phosphoenolpyruvate:UDP-N-acetyl-D-glucosamine 1-carboxyvinyl-transferase; UDP-N-acetylglucosamine 1-carboxyvinyl-transferase; UDP-N-acetylglucosamine 1-carboxyvinyltransferase; UDP-N-acetylglucosamine enoylpyruvyltransferase; UDP-N-acetylglucosamine1-carboxyvinyltransferase; enoylpyruvate transferase; phosphoenolpyruvate-UDP-acetylglucosamine-3-enolpyruvyltransferase; phosphoenolpyruvate:UDP-2-acetamido-2-deoxy-D-glucose 2-enoyl-1-carboxyethyltransferase; phosphoenolpyruvate:UDP-N-acetyl-D-glucosamine 1-carboxyvinyltransferase; phosphoenolpyruvate:uridine diphosphate N-acetylglucosamine enolpyruvyltransferase; phosphoenolpyruvate:uridine-5'-diphospho-N-acetyl-2-amino-2-deoxyglucose 3-enolpyruvyltransferase; phosphopyruvate-uridine diphosphoacetylglucosamine pyruvatetransferase; pyruvate-UDP-acetylglucosamine transferase; pyruvate-uridine diphospho-N-acetyl-glucosamine transferase; pyruvate-uridine diphospho-N-acetylglucosamine transferase; pyruvic-uridine diphospho-N-acetylglucosaminyltransferase\"],\n",
       "  'code': '(1) cpd00037[0] + (1) cpd00061[0] <=> (1) cpd00009[0] + (1) cpd02820[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00037;cpd00061;cpd02820',\n",
       "  'definition': '(1) UDP-N-acetylglucosamine[0] + (1) Phosphoenolpyruvate[0] <= (1) Phosphate[0] + (1) UDP-N-acetylglucosamine enolpyruvate[0]',\n",
       "  'deltag': -4.1,\n",
       "  'deltagerr': 0.89,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.5.1.7'],\n",
       "  'equation': '(1) cpd00037[0] + (1) cpd00061[0] <= (1) cpd00009[0] + (1) cpd02820[0]',\n",
       "  'id': 'rxn00461',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phosphoenolpyruvate:UDP-N-acetyl-D-glucosamine 1-carboxyvinyl-transferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetylmuramoyl-Pentapeptide-Biosynthesis (UDP-<i>N</i>-Acetylmuramoyl-Pentapeptide Biosynthesis); Anhydromuropeptides-Recycling (Anhydromuropeptide Recycling); Antibiotic-Resistance (Antibiotic Resistance); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); PEPTIDOGLYCANSYN-PWY (peptidoglycan biosynthesis I (<I>meso</I>-diaminopimelate containing)); PWY-5265 (peptidoglycan biosynthesis II (staphylococci)); PWY-6385 (peptidoglycan biosynthesis III (mycobacteria)); PWY-6386 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis II (lysine-containing)); PWY-6387 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis I (<i>meso</i>-diaminopimelate containing)); PWY-6470 (peptidoglycan biosynthesis V (&beta;-lactam resistance)); PWY-6471 (peptidoglycan biosynthesis IV (Enterococcus faecium)); PWY-7953 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis III (<i>meso</i>-diaminopimelate containing)); PWY0-1261 (anhydromuropeptides recycling I); Peptidoglycan-Biosynthesis (Peptidoglycan Biosynthesis); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); Sugar-Derivatives (Sugar Derivative Degradation)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn00550 (Peptidoglycan biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00037:0:0:\"UDP-N-acetylglucosamine\";-1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00009:0:0:\"Phosphate\";1:cpd02820:0:0:\"UDP-N-acetylglucosamine enolpyruvate\"'},\n",
       " {'abbreviation': 'R00661',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00661',\n",
       "   'MetaCyc: PHOSPHOENOLPYRUVATE-MUTASE-RXN',\n",
       "   'Name: PEP phosphomutase; PEPPM; Phosphoenolpyruvate 2,3-phosphonomutase; phosphoenolpyruvate 2,3-phosphonomutase; phosphoenolpyruvate mutase; phosphoenolpyruvate phosphomutase; phosphoenolpyruvate-phosphonopyruvate phosphomutase'],\n",
       "  'code': '(1) cpd00061[0] <=> (1) cpd01807[0]',\n",
       "  'compound_ids': 'cpd00061;cpd00067;cpd01807',\n",
       "  'definition': '(1) Phosphoenolpyruvate[0] + (1) H+[0] <= (1) 3-Phosphonopyruvate[0]',\n",
       "  'deltag': 3.79,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['5.4.2.9'],\n",
       "  'equation': '(1) cpd00061[0] + (1) cpd00067[0] <= (1) cpd01807[0]',\n",
       "  'id': 'rxn00462',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'phosphoenolpyruvate 2,3-phosphonomutase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); PWY-5757 (fosfomycin biosynthesis); PWY-6322 (phosphinothricin tripeptide biosynthesis); PWY-6682 (dehydrophos biosynthesis); PWY-6839 (2-aminoethylphosphonate biosynthesis); PWY-7419 (FR-900098 and FR-33289 antibiotics biosynthesis); PWY-7510 (rhizocticin A and B biosynthesis); PWY-7769 (phosalacine biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00440 (Phosphonate and phosphinate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00061:0:0:\"Phosphoenolpyruvate\";-1:cpd00067:0:0:H+;1:cpd01807:0:0:\"3-Phosphonopyruvate\"'},\n",
       " {'abbreviation': 'R00662',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.1.19-RXN',\n",
       "   'BiGG: NTPP8',\n",
       "   'BrachyCyc: 3.6.1.19-RXN',\n",
       "   'KEGG: R00662',\n",
       "   'MetaCyc: 3.6.1.19-RXN; RXN-14139',\n",
       "   'Name: Nucleoside triphosphate pyrophosphorylase (utp); Uridine triphosphate pyrophosphohydrolase; nucleoside-triphosphate diphosphatase; nucleoside-triphosphate diphosphohydrolase; nucleoside-triphosphate pyrophosphatase; utp pyrophosphorylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00062[0] <=> (1) cpd00012[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00062;cpd00067;cpd00091',\n",
       "  'definition': '(1) H2O[0] + (1) UTP[0] => (1) PPi[0] + (2) H+[0] + (1) UMP[0]',\n",
       "  'deltag': -7.05,\n",
       "  'deltagerr': 0.94,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.19', '3.6.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00062[0] => (1) cpd00012[0] + (2) cpd00067[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00463',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Uridine triphosphate pyrophosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); PWY-7821 (tunicamycin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00062:0:0:\"UTP\";1:cpd00012:0:0:\"PPi\";2:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00664',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00664',\n",
       "   'MetaCyc: ORNITHINE-N-BENZOYLTRANSFERASE-RXN',\n",
       "   'Name: Bezoyl-CoA:L-ornithine N-benzoyltransferase; benzoyl-CoA:L-ornithine N-benzoyltransferase; ornithine N-acyltransferase; ornithine N-benzoyltransferase'],\n",
       "  'code': '(1) cpd00064[0] + (2) cpd00401[0] <=> (2) cpd00010[0] + (1) cpd02329[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00064;cpd00067;cpd00401;cpd02329',\n",
       "  'definition': '(1) Ornithine[0] + (2) Benzoyl-CoA[0] => (2) CoA[0] + (2) H+[0] + (1) L-Ornithuric acid[0]',\n",
       "  'deltag': -14.87,\n",
       "  'deltagerr': 1.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.127'],\n",
       "  'equation': '(1) cpd00064[0] + (2) cpd00401[0] => (2) cpd00010[0] + (2) cpd00067[0] + (1) cpd02329[0]',\n",
       "  'id': 'rxn00464',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Bezoyl-CoA:L-ornithine N-benzoyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00064:0:0:\"Ornithine\";-2:cpd00401:0:0:\"Benzoyl-CoA\";2:cpd00010:0:0:\"CoA\";2:cpd00067:0:0:\"H+\";1:cpd02329:0:0:\"L-Ornithuric acid\"'},\n",
       " {'abbreviation': 'R00665',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BrachyCyc: CITRULLINASE-RXN',\n",
       "   'ChlamyCyc: CITRULLINASE-RXN',\n",
       "   'KEGG: R00665',\n",
       "   'MaizeCyc: CITRULLINASE-RXN',\n",
       "   'MetaCyc: CITRULLINASE-RXN',\n",
       "   'Name: Citrullinase; L-Citrulline N5-carbamoyldihydrolase; L-citrulline 5-N-carbamoyldihydrolase; L-citrulline N5-carbamoyldihydrolase; citrulline hydrolase; citrulline ureidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00274[0] <=> (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00064[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00064;cpd00067;cpd00274',\n",
       "  'definition': '(1) H2O[0] + (2) H+[0] + (1) Citrulline[0] => (1) CO2[0] + (1) NH3[0] + (1) Ornithine[0]',\n",
       "  'deltag': -5.77,\n",
       "  'deltagerr': 1.2,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.20'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00067[0] + (1) cpd00274[0] => (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00064[0]',\n",
       "  'id': 'rxn00465',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32164;rxn32350',\n",
       "  'name': 'L-Citrulline N5-carbamoyldihydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PROLINE-SYN (L-proline Biosynthesis); PWY-4981 (L-proline biosynthesis II (from arginine))'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00067:0:0:\"H+\";-1:cpd00274:0:0:\"Citrulline\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00064:0:0:\"Ornithine\"'},\n",
       " {'abbreviation': 'R00666',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00666',\n",
       "   'MetaCyc: 1.5.1.24-RXN',\n",
       "   'iAO358: rll_88',\n",
       "   'Name: (L-ornithine-dorming); 5-N-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase (L-ornithine-forming); N5-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase; N5-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase (L-ornithine-dorming); N5-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase (L-ornithine-forming); N5-(carboxyethyl)ornithine synthase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd02587[0] <=> (1) cpd00005[0] + (1) cpd00020[0] + (1) cpd00064[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00020;cpd00064;cpd00067;cpd02587',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) N5-(L-1-Carboxyethyl)-L-ornithine[0] <=> (1) NADPH[0] + (1) Pyruvate[0] + (1) Ornithine[0] + (1) H+[0]',\n",
       "  'deltag': 7.4,\n",
       "  'deltagerr': 6.23,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.24'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd02587[0] <=> (1) cpd00005[0] + (1) cpd00020[0] + (1) cpd00064[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00466',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N5-(L-1-carboxyethyl)-L-ornithine:NADP+ oxidoreductase (L-ornithine-dorming)',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd02587:0:0:\"N5-(L-1-Carboxyethyl)-L-ornithine\";1:cpd00005:0:0:\"NADPH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00064:0:0:\"Ornithine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00667',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ORNITHINE-GLU-AMINOTRANSFORASE-RXN; RXN-7562',\n",
       "   'BrachyCyc: ORNITHINE-GLU-AMINOTRANSFORASE-RXN',\n",
       "   'ChlamyCyc: ORNITHINE-GLU-AMINOTRANSFORASE-RXN',\n",
       "   'KEGG: R00667',\n",
       "   'MetaCyc: ORNITHINE-GLU-AMINOTRANSFERASE-RXN; ORNITHINE-GLU-AMINOTRANSFORASE-RXN',\n",
       "   'Name: L-Ornithine:2-oxo-acid aminotransferase; ORNTA; Ornithine--oxo-glutarate aminotransferase; ornithinetransaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00064[0] <=> (1) cpd00023[0] + (1) cpd00858[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00064;cpd00858',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) Ornithine[0] <=> (1) L-Glutamate[0] + (1) L-Glutamate5-semialdehyde[0]',\n",
       "  'deltag': -1.37,\n",
       "  'deltagerr': 1.09,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.13'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00064[0] <=> (1) cpd00023[0] + (1) cpd00858[0]',\n",
       "  'id': 'rxn00467',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20496;rxn33315',\n",
       "  'name': 'L-Ornithine:2-oxo-acid aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG-PRO-PWY (L-arginine degradation VI (arginase 2 pathway)); ARGASEDEG-PWY (L-arginine degradation I (arginase pathway)); ARGININE-DEG (L-arginine Degradation); ARGININE-SYN4-PWY (L-ornithine biosynthesis II); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CITRULBIO-PWY (L-citrulline biosynthesis); Citrulline-Biosynthesis (L-citrulline Biosynthesis); Degradation (Degradation/Utilization/Assimilation); GLUTAMATE-SYN (L-glutamate Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); MISCELLANEOUS-DEG (Other Amino Acid Degradation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PROLINE-SYN (L-proline Biosynthesis); PWY-3341 (L-proline biosynthesis III); PWY-4981 (L-proline biosynthesis II (from arginine)); PWY-5004 (superpathway of L-citrulline metabolism); PWY-6344 (L-ornithine degradation II (Stickland reaction)); PWY-6922 (L-<i>N<sup>&delta;</sup></i>-acetylornithine biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00064:0:0:\"Ornithine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00858:0:0:\"L-Glutamate5-semialdehyde\"'},\n",
       " {'abbreviation': 'R00668',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00668',\n",
       "   'MetaCyc: ORNITHINELYSINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-ornithine(L-lysine):2-oxoglutarate-aminotransferase; L-ornithine:2-oxoglutarate aminotransferase; L-ornithine:2-oxoglutarate-aminotransferase; lysine/ornithine:2-oxoglutarate aminotransferase; ornithine(lysine) aminotransferase; ornithine(lysine) transaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00064[0] <=> (1) cpd00001[0] + (1) cpd00023[0] + (1) cpd02651[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00023;cpd00024;cpd00064;cpd00067;cpd02651',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) Ornithine[0] => (1) H2O[0] + (1) L-Glutamate[0] + (1) H+[0] + (1) 3,4-Dihydro-2H-Pyrrole-2-carboxylate[0]',\n",
       "  'deltag': -9.44,\n",
       "  'deltagerr': 1.27,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.6.1.13', '2.6.1.68'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00064[0] => (1) cpd00001[0] + (1) cpd00023[0] + (1) cpd00067[0] + (1) cpd02651[0]',\n",
       "  'id': 'rxn00468',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-ornithine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00064:0:0:\"Ornithine\";1:cpd00001:0:0:\"H2O\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00067:0:0:\"H+\";1:cpd02651:0:0:\"3,4-Dihydro-2H-Pyrrole-2-carboxylate\"'},\n",
       " {'abbreviation': 'R00669',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETYLORNDEACET-RXN',\n",
       "   'ChlamyCyc: ACETYLORNDEACET-RXN',\n",
       "   'CornCyc: ACETYLORNDEACET-RXN',\n",
       "   'KEGG: R00669',\n",
       "   'MetaCyc: ACETYLORNDEACET-RXN',\n",
       "   'Name: 2-N-acetyl-L-ornithine amidohydrolase; Acetylornithinase; Acetylornithine deacetylase; N-acetylornithinase; N2-Acetyl-L-ornithine amidohydrolase; N2-acetyl-L-ornithine amidohydrolase; acetylornithine deacetylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00342[0] <=> (1) cpd00029[0] + (1) cpd00064[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00064;cpd00342',\n",
       "  'definition': '(1) H2O[0] + (1) N-Acetylornithine[0] <=> (1) Acetate[0] + (1) Ornithine[0]',\n",
       "  'deltag': -1.62,\n",
       "  'deltagerr': 0.84,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.14', '3.5.1.16'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00342[0] <=> (1) cpd00029[0] + (1) cpd00064[0]',\n",
       "  'id': 'rxn00469',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27670;rxn30596',\n",
       "  'name': 'N2-Acetyl-L-ornithine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGININE-SYN (L-arginine Biosynthesis); ARGSYN-PWY (L-arginine biosynthesis I (via L-ornithine)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); GLUTORN-PWY (L-ornithine biosynthesis I); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); L-Ornithine-Biosynthesis (L-Ornithine Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00342:0:0:\"N-Acetylornithine\";1:cpd00029:0:0:\"Acetate\";1:cpd00064:0:0:\"Ornithine\"'},\n",
       " {'abbreviation': 'R00670',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ORNDECARBOX-RXN',\n",
       "   'BiGG: or NDC',\n",
       "   'BrachyCyc: ORNDECARBOX-RXN',\n",
       "   'KEGG: R00670',\n",
       "   'MetaCyc: ORNDECARBOX-RXN',\n",
       "   'Name: L-Ornithine carboxy-lyase; L-ornithine carboxy-lyase; L-ornithine carboxy-lyase (putrescine-forming); Ornithine decarboxylase; SpeC; or nithine decarboxylase; ornithine decarboxylase'],\n",
       "  'code': '(1) cpd00064[0] <=> (1) cpd00011[0] + (1) cpd00118[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00064;cpd00067;cpd00118',\n",
       "  'definition': '(1) Ornithine[0] + (1) H+[0] => (1) CO2[0] + (1) Putrescine[0]',\n",
       "  'deltag': -7.26,\n",
       "  'deltagerr': 1.32,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.17'],\n",
       "  'equation': '(1) cpd00064[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00118[0]',\n",
       "  'id': 'rxn00470',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31975;rxn31977',\n",
       "  'name': 'L-ornithine carboxy-lyase (putrescine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); ORNARGDEG-PWY (superpathway of L-arginine and L-ornithine degradation); ORNDEG-PWY (superpathway of ornithine degradation); POLYAMINSYN3-PWY (superpathway of polyamine biosynthesis II); POLYAMSYN-PWY (superpathway of polyamine biosynthesis I); PWY-46 (putrescine biosynthesis III); PWY-6305 (putrescine biosynthesis IV); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Putrescine-Biosynthesis (Putrescine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn00480 (Glutathione metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00064:0:0:\"Ornithine\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00118:0:0:\"Putrescine\"'},\n",
       " {'abbreviation': 'R00671',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00671',\n",
       "   'MetaCyc: ORNITHINE-CYCLODEAMINASE-RXN',\n",
       "   'RiceCyc: ORNITHINE-CYCLODEAMINASE-RXN',\n",
       "   'SorghumCyc: ORNITHINE-CYCLODEAMINASE-RXN',\n",
       "   'Name: L-Ornithine ammonia-lyase(cyclizing); L-ornithine ammonia-lyase (cyclizing); L-ornithine ammonia-lyase (cyclizing, L-proline-forming); L-ornithine ammonia-lyase (cyclizing; L-proline-forming); ornithine cyclase; ornithine cyclase (deaminating); ornithine cyclodeaminase'],\n",
       "  'code': '(1) cpd00064[0] <=> (1) cpd00013[0] + (1) cpd00129[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00064;cpd00129',\n",
       "  'definition': '(1) Ornithine[0] <=> (1) NH3[0] + (1) L-Proline[0]',\n",
       "  'deltag': -5.45,\n",
       "  'deltagerr': 1.01,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.3.1.12'],\n",
       "  'equation': '(1) cpd00064[0] <=> (1) cpd00013[0] + (1) cpd00129[0]',\n",
       "  'id': 'rxn00471',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20495',\n",
       "  'name': 'L-ornithine ammonia-lyase (cyclizing; L-proline-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00064:0:0:\"Ornithine\";1:cpd00013:0:0:\"NH3\";1:cpd00129:0:0:\"L-Proline\"'},\n",
       " {'abbreviation': 'R00672',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00672',\n",
       "   'MetaCyc: ORNITHINE-RACEMASE-RXN',\n",
       "   'RiceCyc: ORNITHINE-RACEMASE-RXN',\n",
       "   'Name: L-Ornithine racemase; ornithine racemase'],\n",
       "  'code': '(1) cpd00064[0] <=> (1) cpd00404[0]',\n",
       "  'compound_ids': 'cpd00064;cpd00404',\n",
       "  'definition': '(1) Ornithine[0] <=> (1) D-Ornithine[0]',\n",
       "  'deltag': -2.31,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.10', '5.1.1.12', '5.1.1.9'],\n",
       "  'equation': '(1) cpd00064[0] <=> (1) cpd00404[0]',\n",
       "  'id': 'rxn00472',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ornithine racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); MISCELLANEOUS-DEG (Other Amino Acid Degradation); PWY-6344 (L-ornithine degradation II (Stickland reaction)); PWY-7990 (staphyloferrin A biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Siderophores-Biosynthesis (Siderophore Biosynthesis)',\n",
       "   'KEGG: rn00472 (D-Arginine and D-ornithine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00064:0:0:\"Ornithine\";1:cpd00404:0:0:\"D-Ornithine\"'},\n",
       " {'abbreviation': 'R00673',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: TRPAS2',\n",
       "   'EcoCyc: TRYPTOPHAN-RXN',\n",
       "   'KEGG: R00673',\n",
       "   'MetaCyc: TRYPTOPHAN-RXN',\n",
       "   'iAF1260: TRPAS2',\n",
       "   'Name: L-Tryotophan indole-lyase (deaminating); L-tryptophan indole-lyase (deaminating); L-tryptophan indole-lyase (deaminating, pyruvate-forming); L-tryptophan indole-lyase (deaminating; pyruvate-forming); L-tryptophanase; Tryptophanase (L-tryptophan); tryptophanase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00065[0] <=> (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00359[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00065;cpd00359',\n",
       "  'definition': '(1) H2O[0] + (1) L-Tryptophan[0] => (1) NH3[0] + (1) Pyruvate[0] + (1) indol[0]',\n",
       "  'deltag': 4.95,\n",
       "  'deltagerr': 0.16,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.99.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00065[0] => (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00359[0]',\n",
       "  'id': 'rxn00473',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30065',\n",
       "  'name': 'L-tryptophan indole-lyase (deaminating; pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00359:0:0:\"indol\"'},\n",
       " {'abbreviation': 'R00674',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN0-2382',\n",
       "   'BiGG: TRPS2',\n",
       "   'BrachyCyc: RXN0-2382',\n",
       "   'KEGG: R00674',\n",
       "   'MetaCyc: RXN0-2382',\n",
       "   'Name: L-Serine hydro-lyase (adding indoleglycerol-phosphate); L-serine hydro-lyase (adding indole; L-tryptophan-forming); L-serine hydro-lyase [adding indole, L-tryptophan-forming]; tryptophan synthase (indole); tryptophan synthase (indole-salvaging); tryptophan synthase beta chain; tryptophan synthase beta2; tryptophansynthase-indole'],\n",
       "  'code': '(1) cpd00054[0] + (1) cpd00359[0] <=> (1) cpd00001[0] + (1) cpd00065[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00054;cpd00065;cpd00359',\n",
       "  'definition': '(1) L-Serine[0] + (1) indol[0] <=> (1) H2O[0] + (1) L-Tryptophan[0]',\n",
       "  'deltag': -11.88,\n",
       "  'deltagerr': 0.27,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.122', '4.2.1.20'],\n",
       "  'equation': '(1) cpd00054[0] + (1) cpd00359[0] <=> (1) cpd00001[0] + (1) cpd00065[0]',\n",
       "  'id': 'rxn00474',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn26013;rxn30979',\n",
       "  'name': 'L-serine hydro-lyase (adding indole; L-tryptophan-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); COMPLETE-ARO-PWY (superpathway of aromatic amino acid biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-6629 (superpathway of L-tryptophan biosynthesis); TRPSYN-PWY (L-tryptophan biosynthesis); TRYPTOPHAN-BIOSYNTHESIS (L-tryptophan Biosynthesis)',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00054:0:0:\"L-Serine\";-1:cpd00359:0:0:\"indol\";1:cpd00001:0:0:\"H2O\";1:cpd00065:0:0:\"L-Tryptophan\"'},\n",
       " {'abbreviation': 'R00675',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00675',\n",
       "   'MetaCyc: TRYPTOPHAN-DEHYDROGENASE-RXN',\n",
       "   'Name: L-Trp-dehydrogenase; L-Tryptophan:NAD+ oxidoreductase (deaminating); L-tryptophan dehydrogenase; L-tryptophan:NAD(P)+ oxidoreductase (deaminating); NAD(P)+-L-tryptophan dehydrogenase; TDH; tryptophan dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00065[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00278[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00065;cpd00067;cpd00278',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Tryptophan[0] <=> (1) NADH[0] + (1) NH3[0] + (1) H+[0] + (1) Indolepyruvate[0]',\n",
       "  'deltag': -0.89,\n",
       "  'deltagerr': 1.21,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.19'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00065[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00278[0]',\n",
       "  'id': 'rxn00475',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tryptophan:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00278:0:0:\"Indolepyruvate\"'},\n",
       " {'abbreviation': 'R00676',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00676',\n",
       "   'MetaCyc: TRYPTOPHAN-DEHYDROGENASE-RXN',\n",
       "   'Name: L-Trp-dehydrogenase; L-Tryptophan:NADP+ oxidoreductase (deaminating); L-tryptophan dehydrogenase; L-tryptophan:NAD(P)+ oxidoreductase (deaminating); NAD(P)+-L-tryptophan dehydrogenase; TDH; tryptophan dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00065[0] <=> (1) cpd00005[0] + (1) cpd00013[0] + (1) cpd00278[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00013;cpd00065;cpd00067;cpd00278',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) L-Tryptophan[0] <=> (1) NADPH[0] + (1) NH3[0] + (1) H+[0] + (1) Indolepyruvate[0]',\n",
       "  'deltag': -0.89,\n",
       "  'deltagerr': 1.21,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.19'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00065[0] <=> (1) cpd00005[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00278[0]',\n",
       "  'id': 'rxn00476',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tryptophan:NADP+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00005:0:0:\"NADPH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00278:0:0:\"Indolepyruvate\"'},\n",
       " {'abbreviation': 'R00677',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00677',\n",
       "   'Name: L-Tryptophan:oxygen oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00065[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00278[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00065;cpd00278',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Tryptophan[0] => (1) NH3[0] + (1) H2O2[0] + (1) Indolepyruvate[0]',\n",
       "  'deltag': -32.61,\n",
       "  'deltagerr': 1.57,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00065[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00278[0]',\n",
       "  'id': 'rxn00477',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tryptophan:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00278:0:0:\"Indolepyruvate\"'},\n",
       " {'abbreviation': 'R00678',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00678_c',\n",
       "   'BiGG: TRPO2; YJR078W',\n",
       "   'JP_Creinhardtii_MSB: R_TRP23OR',\n",
       "   'KEGG: R00678',\n",
       "   'iIN800: YJR078W',\n",
       "   'Name: Indoleamine 2,3-dioxygenase family protein; L-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00065[0] <=> (1) cpd01749[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00065;cpd01749',\n",
       "  'definition': '(1) O2[0] + (1) L-Tryptophan[0] => (1) N-Formylkynurenine[0]',\n",
       "  'deltag': -82.24,\n",
       "  'deltagerr': 1.58,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.11', '1.13.11.52'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00065[0] => (1) cpd01749[0]',\n",
       "  'id': 'rxn00478',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-tryptophan:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd01749:0:0:\"N-Formylkynurenine\"'},\n",
       " {'abbreviation': 'R00679',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00679',\n",
       "   'MetaCyc: TRYPTOPHAN-2-MONOOXYGENASE-RXN',\n",
       "   'RiceCyc: TRYPTOPHAN-2-MONOOXYGENASE-RXN',\n",
       "   'SorghumCyc: TRYPTOPHAN-2-MONOOXYGENASE-RXN',\n",
       "   'Name: L-Tryptophan:oxygen 2-oxidoreductase(decarboxylating); L-tryptophan:oxygen 2-oxidoreductase (decarboxylating); tryptophan 2-monooxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00065[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01747[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00011;cpd00065;cpd01747',\n",
       "  'definition': '(1) O2[0] + (1) L-Tryptophan[0] => (1) H2O[0] + (1) CO2[0] + (1) Indole-3-acetamide[0]',\n",
       "  'deltag': -115.02,\n",
       "  'deltagerr': 1.6,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.12.3'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00065[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01747[0]',\n",
       "  'id': 'rxn00479',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-tryptophan:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Auxin-Biosynthesis (Auxin Biosynthesis); HORMONE-SYN (Hormone Biosynthesis); PWY-3161 (indole-3-acetate biosynthesis III (bacteria)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis)',\n",
       "   'KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd01747:0:0:\"Indole-3-acetamide\"'},\n",
       " {'abbreviation': 'R00681',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00681',\n",
       "   'MetaCyc: TRYPTOPHAN-2-DIOXYGENASE-RXN',\n",
       "   \"RiceCyc: TRYPTOPHAN-2'-DIOXYGENASE-RXN\",\n",
       "   'SorghumCyc: TRYPTOPHAN-2-DIOXYGENASE-RXN',\n",
       "   \"Name: L-Tryptophan:oxygen 2'-oxidoreductase (side-chain-cleaving); L-tryptophan:oxygen 2'-oxidoreductase (side-chain-cleaving); TSO; TSO I; TSO II; indole-3-alkane alpha-hydroxylase; indolyl-3-alkan alpha-hydroxylase; tryptophan 2'-dioxygenase; tryptophan side chain oxidase; tryptophan side chain oxidase II; tryptophan side chain oxidase type I; tryptophan side-chain alpha,beta-oxidase; tryptophan side-chain oxidase\"],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00065[0] <=> (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd02067[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00013;cpd00065;cpd00067;cpd02067',\n",
       "  'definition': '(1) O2[0] + (1) L-Tryptophan[0] + (1) H+[0] => (1) CO2[0] + (1) NH3[0] + (1) 3-Indoleglycolaldehyde[0]',\n",
       "  'deltag': -94.74,\n",
       "  'deltagerr': 1.55,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.99.3'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00065[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd02067[0]',\n",
       "  'id': 'rxn00480',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"L-Tryptophan:oxygen 2'-oxidoreductase (side-chain-cleaving)\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00065:0:0:\"L-Tryptophan\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd02067:0:0:\"3-Indoleglycolaldehyde\"'},\n",
       " {'abbreviation': 'R00682',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00682',\n",
       "   'MetaCyc: TRYPTOPHANAMIDASE-RXN',\n",
       "   'Name: L-Tryptophanamide amidohydrolase; L-tryptophan aminopeptidase; L-tryptophanamide amidohydrolase; tryptophan aminopeptidase; tryptophanamidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00720[0] <=> (1) cpd00013[0] + (1) cpd00065[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00065;cpd00720',\n",
       "  'definition': '(1) H2O[0] + (1) L-Tryptophanamide[0] => (1) NH3[0] + (1) L-Tryptophan[0]',\n",
       "  'deltag': 1.3,\n",
       "  'deltagerr': 1.31,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.57'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00720[0] => (1) cpd00013[0] + (1) cpd00065[0]',\n",
       "  'id': 'rxn00481',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tryptophanamide amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00720:0:0:\"L-Tryptophanamide\";1:cpd00013:0:0:\"NH3\";1:cpd00065:0:0:\"L-Tryptophan\"'},\n",
       " {'abbreviation': 'R00683',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00683',\n",
       "   'Name: S-Adenosyl-L-methionine:L-tryptophan N-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00065[0] <=> (1) cpd00019[0] + (1) cpd01911[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00065;cpd00067;cpd01911',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) L-Tryptophan[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) Abrine[0]',\n",
       "  'deltag': -10.46,\n",
       "  'deltagerr': 5.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00065[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd01911[0]',\n",
       "  'id': 'rxn00482',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:L-tryptophan N-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd01911:0:0:\"Abrine\"'},\n",
       " {'abbreviation': 'R00684',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: TRYPTOPHAN-AMINOTRANSFERASE-RXN',\n",
       "   'BiGG: TRPTA',\n",
       "   'BrachyCyc: TRYPTOPHAN-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00684',\n",
       "   'MetaCyc: TRYPTOPHAN-AMINOTRANSFERASE-RXN',\n",
       "   'Name: 5-hydroxytryptophan-ketoglutaric transaminase; L-Tryptophan:2-oxoglutarate aminotransferase; L-phenylalanine-2-oxoglutarate aminotransferase; L-tryptophan aminotransferase; L-tryptophan transaminase; L-tryptophan:2-oxoglutarate aminotransferase; aminotransferase; hydroxytryptophan aminotransferase; tryptophan aminotransferase; tryptophan transaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00065[0] <=> (1) cpd00023[0] + (1) cpd00278[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00065;cpd00278',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Tryptophan[0] <=> (1) L-Glutamate[0] + (1) Indolepyruvate[0]',\n",
       "  'deltag': -9.6,\n",
       "  'deltagerr': 1.21,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.27'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00065[0] <=> (1) cpd00023[0] + (1) cpd00278[0]',\n",
       "  'id': 'rxn00483',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38818',\n",
       "  'name': 'L-Tryptophan:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Auxin-Biosynthesis (Auxin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); HORMONE-SYN (Hormone Biosynthesis); PWY-5081 (L-tryptophan degradation VIII (to tryptophol)); PWY-581 (indole-3-acetate biosynthesis II); PWY-8017 (L-tryptophan degradation XIII (Stickland reaction)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TRPIAACAT-PWY (indole-3-acetate biosynthesis VI (bacteria)); TRPKYNCAT-PWY (L-tryptophan degradation IV (via indole-3-lactate)); TRYPTOPHAN-DEG (L-tryptophan Degradation)',\n",
       "   'KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00065:0:0:\"L-Tryptophan\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00278:0:0:\"Indolepyruvate\"'},\n",
       " {'abbreviation': 'R00685',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: AROMATIC-L-AMINO-ACID-DECARBOXYLASE-RXN',\n",
       "   'BrachyCyc: AROMATIC-L-AMINO-ACID-DECARBOXYLASE-RXN',\n",
       "   'ChlamyCyc: AROMATIC-L-AMINO-ACID-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R00685',\n",
       "   'MetaCyc: AROMATIC-L-AMINO-ACID-DECARBOXYLASE-RXN',\n",
       "   'Name: Aromatic-L-amino-acid decarboxylase; DOPA decarboxylase; Hydroxytryptophan decarboxylase; L-Tryptophan decarboxy-lyase; Tryptophan decarboxylase'],\n",
       "  'code': '(1) cpd00065[0] <=> (1) cpd00011[0] + (1) cpd00318[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00065;cpd00067;cpd00318',\n",
       "  'definition': '(1) L-Tryptophan[0] + (1) H+[0] => (1) CO2[0] + (1) Tryptamine[0]',\n",
       "  'deltag': -8.82,\n",
       "  'deltagerr': 1.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.105', '4.1.1.28'],\n",
       "  'equation': '(1) cpd00065[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd00318[0]',\n",
       "  'id': 'rxn00484',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32690',\n",
       "  'name': 'L-Tryptophan decarboxy-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Auxin-Biosynthesis (Auxin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); HORMONE-SYN (Hormone Biosynthesis); INDOLE-ALKALOIDS (Indole Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-3181 (L-tryptophan degradation VI (via tryptamine)); PWY-5290 (secologanin and strictosidine biosynthesis); PWY-5473 (hydroxycinnamic acid serotonin amides biosynthesis); PWY-581 (indole-3-acetate biosynthesis II); PWY-5877 (beta-carboline biosynthesis); PWY-6307 (L-tryptophan degradation X (mammalian, via tryptamine)); PWY-7936 (psilocybin biosynthesis); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TERPENOID-ALKALOIDS (Terpenoid Alkaloid Biosynthesis); TRYPTOPHAN-DEG (L-tryptophan Degradation); Terpenoid-Biosynthesis (Terpenoid Biosynthesis)',\n",
       "   'KEGG: rn00380 (Tryptophan metabolism); rn00901 (Indole alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00065:0:0:\"L-Tryptophan\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd00318:0:0:\"Tryptamine\"'},\n",
       " {'abbreviation': 'R00686',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5.1.1.11-RXN',\n",
       "   'KEGG: R00686',\n",
       "   'MetaCyc: 5.1.1.11-RXN',\n",
       "   'SoyCyc: 5.1.1.11-RXN',\n",
       "   'Name: L-Phenylalanine racemase (ATP-hydrolysing); gramicidin S synthetase I; phenylalanine racemase; phenylalanine racemase (ATP-hydrolysing); phenylalanine racemase (adenosine triphosphate-hydrolysing)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00066[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd01526[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00012;cpd00018;cpd00066;cpd00067;cpd01526',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) L-Phenylalanine[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0] + (1) D-Phenylalanine[0]',\n",
       "  'deltag': -10.07,\n",
       "  'deltagerr': 0.96,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['5.1.1.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00066[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd01526[0]',\n",
       "  'id': 'rxn00485',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Phenylalanine racemase (ATP-hydrolysing)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd01526:0:0:\"D-Phenylalanine\"'},\n",
       " {'abbreviation': 'R00687',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00687',\n",
       "   'MetaCyc: PHENYLALANINE-ADENYLYLTRANSFERASE-RXN',\n",
       "   'Name: ATP:L-phenylalanine adenylyltransferase; L-phenylalanine adenylyltransferase; phenylalanine adenylyltransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00066[0] <=> (1) cpd00012[0] + (1) cpd02326[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00066;cpd00067;cpd02326',\n",
       "  'definition': '(1) ATP[0] + (1) L-Phenylalanine[0] => (1) PPi[0] + (2) H+[0] + (1) N-Adenylyl-L-phenylalanine[0]',\n",
       "  'deltag': -21.24,\n",
       "  'deltagerr': 4.5,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.7.54'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00066[0] => (1) cpd00012[0] + (2) cpd00067[0] + (1) cpd02326[0]',\n",
       "  'id': 'rxn00486',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:L-phenylalanine adenylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00012:0:0:\"PPi\";2:cpd00067:0:0:\"H+\";1:cpd02326:0:0:\"N-Adenylyl-L-phenylalanine\"'},\n",
       " {'abbreviation': 'R00688',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00688',\n",
       "   'MetaCyc: PHENYLALANINE-DEHYDROGENASE-RXN',\n",
       "   'Name: L-phenylalanine dehydrogenase; L-phenylalanine:NAD+ oxidoreductase (deaminating); PHD; phenylalanine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00066[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00066;cpd00067;cpd00143',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Phenylalanine[0] <=> (1) NADH[0] + (1) NH3[0] + (1) H+[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': 5.2,\n",
       "  'deltagerr': 1.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.20'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00066[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn00487',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-phenylalanine:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism); rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R00689',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00689',\n",
       "   'MetaCyc: RXN-17130',\n",
       "   'Name: L-Phenylalanine:oxygen oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00066[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00066;cpd00143',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Phenylalanine[0] => (1) NH3[0] + (1) H2O2[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': -26.52,\n",
       "  'deltagerr': 1.41,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00066[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn00488',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Phenylalanine:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism); rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R00690',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00690; R00698',\n",
       "   'MetaCyc: PHENYLALANINE-2-MONOOXYGENASE-RXN',\n",
       "   'TS_Athaliana: rxn592_c; rxn592_v',\n",
       "   'iRS1563: R_R00690_c; R_R00698_c',\n",
       "   'Name: L-Phenylalanine:oxygen 2-oxidoreductase (decarboxylating); L-phenylalanine oxidase (deaminating and decarboxylating); L-phenylalanine:oxygen 2-oxidoreductase (decarboxylating); L-phenylalanine:oxygen oxidoreductase (decarboxylating); phenylalanine (deaminating, decarboxylating)oxidase; phenylalanine 2-monooxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00066[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01647[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00011;cpd00066;cpd01647',\n",
       "  'definition': '(1) O2[0] + (1) L-Phenylalanine[0] => (1) H2O[0] + (1) CO2[0] + (1) 2-Phenylacetamide[0]',\n",
       "  'deltag': -112.92,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.11.1.21', '1.11.1.7', '1.13.12.9'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00066[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01647[0]',\n",
       "  'id': 'rxn00489',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn14316;rxn32630',\n",
       "  'name': 'L-Phenylalanine:oxygen 2-oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd01647:0:0:\"2-Phenylacetamide\"'},\n",
       " {'abbreviation': 'R00691',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CARBOXYCYCLOHEXADIENYL-DEHYDRATASE-RXN',\n",
       "   'BrachyCyc: CARBOXYCYCLOHEXADIENYL-DEHYDRATASE-RXN',\n",
       "   'ChlamyCyc: CARBOXYCYCLOHEXADIENYL-DEHYDRATASE-RXN',\n",
       "   'KEGG: R00691',\n",
       "   'MetaCyc: CARBOXYCYCLOHEXADIENYL-DEHYDRATASE-RXN',\n",
       "   'Name: L-Arogenate hydro-lyase (decarboxylating); L-arogenate hydro-lyase (decarboxylating); L-arogenate hydro-lyase (decarboxylating, L-phenylalanine-forming); L-arogenate hydro-lyase (decarboxylating; L-phenylalanine-forming); arogenate dehydratase; carboxycyclohexadienyl dehydratase'],\n",
       "  'code': '(1) cpd00616[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00066[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00066;cpd00067;cpd00616',\n",
       "  'definition': '(1) H+[0] + (1) Pretyrosine[0] => (1) H2O[0] + (1) CO2[0] + (1) L-Phenylalanine[0]',\n",
       "  'deltag': -19.05,\n",
       "  'deltagerr': 2.64,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.51', '4.2.1.91'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00616[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00066[0]',\n",
       "  'id': 'rxn00490',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19253',\n",
       "  'name': 'L-arogenate hydro-lyase (decarboxylating; L-phenylalanine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PHENYLALANINE-SYN (L-phenylalanine Biosynthesis); PWY-3462 (L-phenylalanine biosynthesis II); PWY-3481 (superpathway of L-phenylalanine and L-tyrosine biosynthesis)',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00616:0:0:\"Pretyrosine\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd00066:0:0:\"L-Phenylalanine\"'},\n",
       " {'abbreviation': 'R00692',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.6.1.58-RXN',\n",
       "   'BrachyCyc: 2.6.1.58-RXN',\n",
       "   'KEGG: R00692',\n",
       "   'MaizeCyc: 2.6.1.58-RXN',\n",
       "   'MetaCyc: 2.6.1.58-RXN',\n",
       "   'Name: L-Phenylalanine(L-histidine):pyruvate aminotransferase; L-phenylalanine(L-histidine):pyruvate aminotransferase; L-phenylalanine:pyruvate aminotransferase; histidine:pyruvate aminotransferase; phenylalanine (histidine) aminotransferase; phenylalanine(histidine) transaminase; phenylalanine(histidine):pyruvate aminotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00066[0] <=> (1) cpd00035[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00035;cpd00066;cpd00143',\n",
       "  'definition': '(1) Pyruvate[0] + (1) L-Phenylalanine[0] <=> (1) L-Alanine[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': -3.58,\n",
       "  'deltagerr': 1.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.58'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00066[0] <=> (1) cpd00035[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn00491',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn37938',\n",
       "  'name': 'L-phenylalanine:pyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PHENYLALANINE-DEG (L-phenylalanine Degradation); PWY-5079 (L-phenylalanine degradation III); PWY-7275 (L-homophenylalanine biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism); rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R00693',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00693',\n",
       "   'MetaCyc: PHENYLALANINE-N-ACETYLTRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:L-phenylalanine N-acetyltransferase; acetyl-CoA-L-phenylalanine alpha-N-acetyltransferase; acetyl-CoA:L-phenylalanine N-acetyltransferase; phenylalanine N-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00066[0] <=> (1) cpd00010[0] + (1) cpd02218[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00066;cpd00067;cpd02218',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Phenylalanine[0] => (1) CoA[0] + (1) H+[0] + (1) N-Acetyl-L-phenylalanine[0]',\n",
       "  'deltag': -6.96,\n",
       "  'deltagerr': 1.04,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.53'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00066[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd02218[0]',\n",
       "  'id': 'rxn00492',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:L-phenylalanine N-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd02218:0:0:\"N-Acetyl-L-phenylalanine\"'},\n",
       " {'abbreviation': 'R00694',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHEAMINOTRANS-RXN; RXN-10814',\n",
       "   'BiGG: ARO9_1; PHETA1',\n",
       "   'BrachyCyc: PHEAMINOTRANS-RXN',\n",
       "   'KEGG: R00694',\n",
       "   'MetaCyc: PHEAMINOTRANS-RXN; RXN-10814',\n",
       "   'Name: L-Phenylalanine:2-oxoglutarate aminotransferase; L-phenylalanine aminotransferase; aminotransferase; phenylalanine aminotransferase; phenylalanine transaminase; phenylalaninetransaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00066[0] <=> (1) cpd00023[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00066;cpd00143',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Phenylalanine[0] <=> (1) L-Glutamate[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': -3.51,\n",
       "  'deltagerr': 1.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.1', '2.6.1.5', '2.6.1.57', '2.6.1.58', '2.6.1.9'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00066[0] <=> (1) cpd00023[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn00493',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn21134;rxn28060;rxn38254',\n",
       "  'name': 'L-Phenylalanine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); ANAPHENOXI-PWY (L-phenylalanine degradation II (anaerobic)); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); BENZCOA-PWY (anaerobic aromatic compound degradation (<i>Thauera aromatica</i>)); COMPLETE-ARO-PWY (superpathway of aromatic amino acid biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PHENYLALANINE-DEG (L-phenylalanine Degradation); PHENYLALANINE-SYN (L-phenylalanine Biosynthesis); PHESYN (L-phenylalanine biosynthesis I); PWY-5079 (L-phenylalanine degradation III); PWY-6318 (L-phenylalanine degradation IV (mammalian, via side chain)); PWY-6628 (superpathway of L-phenylalanine biosynthesis); PWY-8014 (L-phenylalanine degradation VI (Stickland reaction)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism); rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R00695',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00695',\n",
       "   'MetaCyc: 2.6.1.70-RXN',\n",
       "   'Name: L-Aspartate:phenylpyruvate aminotransferase; L-aspartate:phenylpyruvate aminotransferase; aspartate-phenylpyruvate aminotransferase; aspartate-phenylpyruvate transaminase'],\n",
       "  'code': '(1) cpd00041[0] + (1) cpd00143[0] <=> (1) cpd00032[0] + (1) cpd00066[0]',\n",
       "  'compound_ids': 'cpd00032;cpd00041;cpd00066;cpd00143',\n",
       "  'definition': '(1) L-Aspartate[0] + (1) Phenylpyruvate[0] <=> (1) Oxaloacetate[0] + (1) L-Phenylalanine[0]',\n",
       "  'deltag': 4.2,\n",
       "  'deltagerr': 1.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.70'],\n",
       "  'equation': '(1) cpd00041[0] + (1) cpd00143[0] <=> (1) cpd00032[0] + (1) cpd00066[0]',\n",
       "  'id': 'rxn00494',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Aspartate:phenylpyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";-1:cpd00143:0:0:\"Phenylpyruvate\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00066:0:0:\"L-Phenylalanine\"'},\n",
       " {'abbreviation': 'R00697',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHENYLALANINE-AMMONIA-LYASE-RXN',\n",
       "   'BiGG: PHEAL',\n",
       "   'BrachyCyc: PHENYLALANINE-AMMONIA-LYASE-RXN',\n",
       "   'KEGG: R00697',\n",
       "   'MetaCyc: 4.3.1.25-RXN; PHENYLALANINE-AMMONIA-LYASE-RXN',\n",
       "   'Name: L-Phenylalanine ammonia-lyase; L-phenylalanine ammonia-lyase; L-phenylalanine ammonia-lyase (trans-cinnamate-forming); PAL; Phe ammonia-lyase; Phenylalanine ammonia-lyase; phenylalanine ammonia-lyase; phenylalanine ammonium-lyase; phenylalanine deaminase'],\n",
       "  'code': '(1) cpd00066[0] <=> (1) cpd00013[0] + (1) cpd00333[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00066;cpd00333',\n",
       "  'definition': '(1) L-Phenylalanine[0] => (1) NH3[0] + (1) (E)-Cinnamate[0]',\n",
       "  'deltag': -0.89,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.24', '4.3.1.25', '4.3.1.5'],\n",
       "  'equation': '(1) cpd00066[0] => (1) cpd00013[0] + (1) cpd00333[0]',\n",
       "  'id': 'rxn00495',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30552;rxn32185;rxn32187;rxn32189;rxn32193',\n",
       "  'name': 'L-phenylalanine ammonia-lyase (trans-cinnamate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); Benzoate-Biosynthesis (Benzoate Biosynthesis); CINNAMATE-SYN (Cinnamate Biosynthesis); COUMARIN-SYN (Coumarin Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Metabolic-Clusters (Metabolic Clusters); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PLANT-CELL-STRUCTURE (Plant Cell Structures); POLYKETIDE-SYN (Polyketide Biosynthesis); PWY-1121 (suberin monomers biosynthesis); PWY-5048 (rosmarinic acid biosynthesis I); PWY-5071 (superpathway of rosmarinic acid biosynthesis); PWY-5883 (ephedrine biosynthesis); PWY-6432 (curcuminoid biosynthesis); PWY-6443 (benzoate biosynthesis I (CoA-dependent, &beta;-oxidative)); PWY-6444 (benzoate biosynthesis II (CoA-independent, non-&beta;-oxidative)); PWY-6446 (benzoate biosynthesis III (CoA-dependent, non-&beta;-oxidative)); PWY-6457 (<i>trans</i>-cinnamoyl-CoA biosynthesis); PWY-7169 (hyperxanthone E biosynthesis); PWY-7186 (superpathway of scopolin and esculin biosynthesis); PWY1F-467 (phenylpropanoid biosynthesis, initial reactions); Rosmarinate-Biosynthesis (Rosmarinate Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); XANTHONE-SYN (Xanthone Biosynthesis)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism); rn00940 (Phenylpropanoid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00013:0:0:\"NH3\";1:cpd00333:0:0:\"(E)-Cinnamate\"'},\n",
       " {'abbreviation': 'R_R00698_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00698_c',\n",
       "   'AraGEM: R_R00698_c',\n",
       "   'Maize_C4GEM: R_R00698_c',\n",
       "   'iRS1597: R00698_c',\n",
       "   'Name: Donor:hydrogen-peroxide oxidoreductase; peroxidase, putative'],\n",
       "  'code': '(1) cpd00066[0] <=> (1) cpd01647[0]',\n",
       "  'compound_ids': 'cpd00066;cpd01647',\n",
       "  'definition': '(1) L-Phenylalanine[0] <= (1) 2-Phenylacetamide[0]',\n",
       "  'deltag': 20.8,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.11.1.7'],\n",
       "  'equation': '(1) cpd00066[0] <= (1) cpd01647[0]',\n",
       "  'id': 'rxn00496',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'peroxidase, putative',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'MI:C:-1/H:-2/O:-1',\n",
       "  'stoichiometry': '-1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd01647:0:0:\"2-Phenylacetamide\"'},\n",
       " {'abbreviation': 'R00699',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHENYLALANINE-DECARBOXYLASE-RXN',\n",
       "   'BiGG: PHYCBOXL',\n",
       "   'BrachyCyc: PHENYLALANINE-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R00699',\n",
       "   'MetaCyc: PHENYLALANINE-DECARBOXYLASE-RXN',\n",
       "   'Name: L-Phenylalanine carboxy-lyase; L-phenylalanine carboxy-lyase; L-phenylalanine carboxy-lyase (phenylethylamine-forming); L-phenylalanine decarboxylase; aromatic L-amino acid decarboxylase; phenylalanine decarboxylase'],\n",
       "  'code': '(1) cpd00066[0] <=> (1) cpd00011[0] + (1) cpd03161[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00066;cpd00067;cpd03161',\n",
       "  'definition': '(1) L-Phenylalanine[0] + (1) H+[0] => (1) CO2[0] + (1) Phenethylamine[0]',\n",
       "  'deltag': -6.72,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.28', '4.1.1.53'],\n",
       "  'equation': '(1) cpd00066[0] + (1) cpd00067[0] => (1) cpd00011[0] + (1) cpd03161[0]',\n",
       "  'id': 'rxn00497',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-phenylalanine carboxy-lyase (phenylethylamine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PHENYLALANINE-DEG (L-phenylalanine Degradation); PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PWY-5751 (phenylethanol biosynthesis); PWY-6318 (L-phenylalanine degradation IV (mammalian, via side chain)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00066:0:0:\"L-Phenylalanine\";-1:cpd00067:0:0:\"H+\";1:cpd00011:0:0:\"CO2\";1:cpd03161:0:0:\"Phenethylamine\"'},\n",
       " {'abbreviation': 'R00702',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.5.1.21-RXN; RXN-12263',\n",
       "   'BiGG: ERG9_1',\n",
       "   'BrachyCyc: 2.5.1.21-RXN',\n",
       "   'KEGG: R00702',\n",
       "   'MetaCyc: RXN-12263',\n",
       "   'Name: Farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase; SQS; Squalene synthetase; farnesyl-diphosphate farnesyltransferase; farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase; farnesyltransferase; presqualene diphosphate; presqualene synthase; presqualene-diphosphate synthase; squalene synthetase'],\n",
       "  'code': '(2) cpd00350[0] <=> (1) cpd00012[0] + (1) cpd02169[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00067;cpd00350;cpd02169',\n",
       "  'definition': '(2) Farnesyldiphosphate[0] => (1) PPi[0] + (1) H+[0] + (1) Presqualene diphosphate[0]',\n",
       "  'deltag': -27.33,\n",
       "  'deltagerr': 4.38,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.5.1.-', '2.5.1.103', '2.5.1.21', '2.5.1.96'],\n",
       "  'equation': '(2) cpd00350[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd02169[0]',\n",
       "  'id': 'rxn00498',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn22724;rxn37933',\n",
       "  'name': 'farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': [\"MetaCyc: C30-Carotenoids-Biosynthesis (C<sub>30</sub> Carotenoid Biosynthesis); CAROTENOID-SYN (Carotenoid Biosynthesis); PWY-5875 (staphyloxanthin biosynthesis); PWY-6105 (botryococcenes and methylated squalene biosynthesis); PWY-6767 (4,4'-diapolycopenedioate biosynthesis); PWY-7072 (hopanoid biosynthesis (bacteria)); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TRITERPENOID-SYN (Triterpenoid Biosynthesis); Terpenoid-Biosynthesis (Terpenoid Biosynthesis)\",\n",
       "   'KEGG: rn00100 (Steroid biosynthesis); rn00906 (Carotenoid biosynthesis); rn00909 (Sesquiterpenoid and triterpenoid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00350:0:0:\"Farnesyldiphosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd02169:0:0:\"Presqualene diphosphate\"'},\n",
       " {'abbreviation': 'R00703',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: L-LACTATE-DEHYDROGENASE-RXN',\n",
       "   'BiGG: LDH_L',\n",
       "   'BrachyCyc: L-LACTATE-DEHYDROGENASE-RXN',\n",
       "   'KEGG: R00703',\n",
       "   'MetaCyc: L-LACTATE-DEHYDROGENASE-RXN',\n",
       "   'Name: (S)-Lactate:NAD+ oxidoreductase; (S)-lactate:NAD+ oxidoreductase; L(+)-nLDH; L-(+)-lactate dehydrogenase; L-lactate dehydrogenase; L-lactic acid dehydrogenase; L-lactic dehydrogenase; NAD-lactate dehydrogenase; lactate dehydrogenase; lactate dehydrogenase NAD-dependent; lactic acid dehydrogenase; lactic dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00159[0] <=> (1) cpd00004[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00020;cpd00067;cpd00159',\n",
       "  'definition': '(1) NAD[0] + (1) L-Lactate[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 6.43,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.27'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00159[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00499',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34239;rxn34240',\n",
       "  'name': '(S)-Lactate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROFRUCAT-PWY (homolactic fermentation); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); P122-PWY (heterolactic fermentation); P124-PWY (Bifidobacterium shunt); P41-PWY (pyruvate fermentation to acetate and lactate I); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-5100 (pyruvate fermentation to acetate and lactate II); PWY-5481 (pyruvate fermentation to lactate); PWY-6901 (superpathway of glucose and xylose degradation); PWY-7345 (superpathway of anaerobic sucrose degradation); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Pyruvate-Lactate-Fermentation (pyruvate fermentation to lactate); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00159:0:0:\"L-Lactate\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00704',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DLACTDEHYDROGNAD-RXN',\n",
       "   'BiGG: LDH_D; LDH_Dir',\n",
       "   'ChlamyCyc: DLACTDEHYDROGNAD-RXN',\n",
       "   'KEGG: R00704',\n",
       "   'MetaCyc: DLACTDEHYDROGNAD-RXN',\n",
       "   'Name: (R)-Lactate:NAD+ oxidoreductase; (R)-lactate:NAD+ oxidoreductase; D-(-)-lactate dehydrogenase (NAD); D-lactate dehydrogenase; D-lactic acid dehydrogenase; D-lactic dehydrogenase; D-specific lactic dehydrogenase; d-lactate dehydrogenase, nad; lactic acid dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00221[0] <=> (1) cpd00004[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00020;cpd00067;cpd00221',\n",
       "  'definition': '(1) NAD[0] + (1) D-Lactate[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 5.68,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.28'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00221[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00500',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34235',\n",
       "  'name': '(R)-Lactate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALACAT2-PWY (L-alanine degradation II (to D-lactate)); ALANINE-DEG (L-alanine Degradation); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Degradation (Amino Acid Degradation); Antibiotic-Resistance (Antibiotic Resistance); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); Energy-Metabolism (Generation of Precursor Metabolite and Energy); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Fermentation-to-propanoate (Fermentation to Propanoate); P122-PWY (heterolactic fermentation); PROPFERM-PWY (L-alanine fermentation to propanoate and acetate); PWY-6454 (vancomycin resistance I); PWY-6901 (superpathway of glucose and xylose degradation); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); Vancomycin-Resistnace (Vancomycin Resistance)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00221:0:0:\"D-Lactate\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00705',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00705_c',\n",
       "   'AraCyc: RXN-9958',\n",
       "   'BiGG: MMSAD3',\n",
       "   'KEGG: R00705',\n",
       "   'MetaCyc: RXN-2902; RXN-9958',\n",
       "   'Name: 3-Oxopropanoate:NAD+ oxidoreductase (decarboxylating,; 3-Oxopropanoate:NAD+ oxidoreductase (decarboxylating, CoA-acetylating); CoA-acetylating); methylmalonate-semialdehyde dehydrogenase (malonic semialdehyde)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00191[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00011;cpd00022;cpd00191',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) 3-Oxopropanoate[0] => (1) NADH[0] + (1) CO2[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': -5.96,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.18', '1.2.1.27'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00191[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00501',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn23817',\n",
       "  'name': '3-Oxopropanoate:NAD+ oxidoreductase (decarboxylating, CoA-acetylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); BETA-ALA-DEGRADATION-I-PWY (&beta;-alanine degradation I); Beta-Alanine-Degradation (&beta;-Alanine Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); MISCELLANEOUS-DEG (Other Amino Acid Degradation); P562-PWY (<I>myo</I>-inositol degradation I); PWY-1781 (&beta;-alanine degradation II); PWY-6373 (acrylate degradation); PWY-7237 (<i>myo</i>-, <i>chiro</i>- and <i>scyllo</i>-inositol degradation); PWY-7574 (propanoyl-CoA degradation II); Propionate-Degradation (Propanoate Degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation)',\n",
       "   'KEGG: rn00410 (beta-Alanine metabolism); rn00562 (Inositol phosphate metabolism); rn00640 (Propanoate metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd00191:0:0:\"3-Oxopropanoate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00706',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-9958',\n",
       "   'KEGG: R00706',\n",
       "   'MetaCyc: 1.2.1.18-RXN; RXN-9958',\n",
       "   'iRS1563: R_R00706_c',\n",
       "   'Name: 3-Oxopropanoate:NADP+ oxidoreductase (decarboxylating,; 3-Oxopropanoate:NADP+ oxidoreductase (decarboxylating, CoA-acetylating); CoA-acetylating); Malonate-semialdehyde dehydrogenase (acetylating)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00191[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00010;cpd00011;cpd00022;cpd00191',\n",
       "  'definition': '(1) NADP[0] + (1) CoA[0] + (1) 3-Oxopropanoate[0] => (1) NADPH[0] + (1) CO2[0] + (1) Acetyl-CoA[0]',\n",
       "  'deltag': -5.96,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.18'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00191[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00022[0]',\n",
       "  'id': 'rxn00502',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn37807',\n",
       "  'name': '3-Oxopropanoate:NADP+ oxidoreductase (decarboxylating, CoA-acetylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-6373 (acrylate degradation)',\n",
       "   'KEGG: rn00410 (beta-Alanine metabolism); rn00562 (Inositol phosphate metabolism); rn00640 (Propanoate metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00191:0:0:\"3-Oxopropanoate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00022:0:0:\"Acetyl-CoA\"'},\n",
       " {'abbreviation': 'R00707',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRROLINECARBDEHYDROG-RXN; RXN-11632',\n",
       "   'BiGG: P5CD; P5CDm; U41',\n",
       "   'BrachyCyc: PYRROLINECARBDEHYDROG-RXN',\n",
       "   'KEGG: R00707',\n",
       "   'MetaCyc: PYRROLINECARBDEHYDROG-RXN; RXN-11632',\n",
       "   'Name: (S)-1-pyrroline-5-carboxylate:NAD+ oxidoreductase; 1-pyrroline-5-carboxylate dehydrogenase; 1-pyrroline-5-carboxylatedehydrogenase; Hypothetical enzyme; L-1-Pyrroline-5-carboxylate:NAD+ oxidoreductase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd00003[0] + (1) cpd02431[0] <=> (1) cpd00004[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00023;cpd00067;cpd02431',\n",
       "  'definition': '(2) H2O[0] + (1) NAD[0] + (1) 1-Pyrroline-5-carboxylate[0] <=> (1) NADH[0] + (1) L-Glutamate[0] + (1) H+[0]',\n",
       "  'deltag': -2.27,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.12'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd00003[0] + (1) cpd02431[0] <=> (1) cpd00004[0] + (1) cpd00023[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00503',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20649;rxn32901',\n",
       "  'name': '(S)-1-pyrroline-5-carboxylate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd02431:0:0:\"1-Pyrroline-5-carboxylate\";1:cpd00004:0:0:\"NADH\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00708',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-11632',\n",
       "   'CornCyc: RXN-11632',\n",
       "   'KEGG: R00708',\n",
       "   'MaizeCyc: RXN-11632',\n",
       "   'MetaCyc: RXN-11632',\n",
       "   'Name: (S)-1-pyrroline-5-carboxylate:NAD+ oxidoreductase; (S)-1-pyrroline-5-carboxylate:NADP+ oxidoreductase; 1-pyrroline dehydrogenase; 1-pyrroline-5-carboxylate dehydrogenase; 1-pyrroline-5-carboxylate:NAD+ oxidoreductase; L-1-Pyrroline-5-carboxylate:NAD+ oxidoreductase; L-pyrroline-5-carboxylate-NAD+ oxidoreductase; delta1-pyrroline-5-carboxylate dehydrogenase; delta1-pyrroline-5-carboxylic acid dehydrogenase; pyrroline-5-carboxylate dehydrogenase; pyrroline-5-carboxylic acid dehydrogenase'],\n",
       "  'code': '(2) cpd00001[0] + (1) cpd00006[0] + (1) cpd02431[0] <=> (1) cpd00005[0] + (1) cpd00023[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00023;cpd00067;cpd02431',\n",
       "  'definition': '(2) H2O[0] + (1) NADP[0] + (1) 1-Pyrroline-5-carboxylate[0] <=> (1) NADPH[0] + (1) L-Glutamate[0] + (1) H+[0]',\n",
       "  'deltag': -2.26,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.12'],\n",
       "  'equation': '(2) cpd00001[0] + (1) cpd00006[0] + (1) cpd02431[0] <=> (1) cpd00005[0] + (1) cpd00023[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00504',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28199;rxn33563',\n",
       "  'name': '(S)-1-pyrroline-5-carboxylate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd02431:0:0:\"1-Pyrroline-5-carboxylate\";1:cpd00005:0:0:\"NADPH\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00709',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00709_m',\n",
       "   'BiGG: ICDHxi; ICDHxm; IDH1',\n",
       "   'JM_Creinhardtii: JM_Cre_47; JM_Cre_48',\n",
       "   'JP_Creinhardtii_MSB: R_ICDHm',\n",
       "   'KEGG: R00709',\n",
       "   'Name: Isocitrate dehydrogenase (NAD+); Isocitrate dehydrogenase [NAD] subunit 1; Isocitrate:NAD+ oxidoreductase (decarboxylating); isocitrate dehydrogenase (NAD); isocitrate:NAD+ oxidoreductase (decarboxylating)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00260[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00024[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00024;cpd00260',\n",
       "  'definition': '(1) NAD[0] + (1) Isocitrate[0] => (1) NADH[0] + (1) CO2[0] + (1) 2-Oxoglutarate[0]',\n",
       "  'deltag': 1.41,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.1.286', '1.1.1.41'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00260[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00024[0]',\n",
       "  'id': 'rxn00505',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30141',\n",
       "  'name': 'Isocitrate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00020 (Citrate cycle (TCA cycle)); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00260:0:0:\"Isocitrate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00024:0:0:\"2-Oxoglutarate\"'},\n",
       " {'abbreviation': 'R00710',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN66-3',\n",
       "   'BiGG: ALD2; ALD3; ALD4_1; ALDD2x; ALDD2xm; ALDD2xr',\n",
       "   'BrachyCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN; RXN66-3',\n",
       "   'KEGG: R00710',\n",
       "   'MetaCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN; RXN66-3',\n",
       "   'Name: Acetaldehyde:NAD+ oxidoreductase; Aldehyde dehydrogenase [NAD(P)+] 1; Potassium-activated aldehyde dehydrogenase; acetaldehyde dehydrogenase (NAD+); aldehyde dehydrogenase (acetaldehyde, NAD); aldehyde dehydrogenase (acetylaldehyde, NAD); aldehyde dehydrogenase, putative (ALDH)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00071[0] <=> (1) cpd00004[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00029;cpd00067;cpd00071',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Acetaldehyde[0] => (1) NADH[0] + (1) Acetate[0] + (2) H+[0]',\n",
       "  'deltag': -11.48,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.3', '1.2.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00071[0] => (1) cpd00004[0] + (1) cpd00029[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00506',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19123;rxn31362',\n",
       "  'name': 'Acetaldehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); Degradation (Degradation/Utilization/Assimilation); Ethanol-Degradation (Ethanol Degradation); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-7387 (hypotaurine degradation); PWY66-161 (ethanol degradation III); PWY66-162 (ethanol degradation IV); PWY66-21 (ethanol degradation II); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00071:0:0:\"Acetaldehyde\";1:cpd00004:0:0:\"NADH\";1:cpd00029:0:0:\"Acetate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00711',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ALD4_2; ALD5_1; ALD6; ALDD2y; ALDD2ym',\n",
       "   'BrachyCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN',\n",
       "   'EcoCyc: RXN0-3962',\n",
       "   'KEGG: R00711',\n",
       "   'MetaCyc: ALDEHYDE-DEHYDROGENASE-NADORNOP+-RXN; RXN0-3962',\n",
       "   'Name: Acetaldehyde:NADP+ oxidoreductase; Magnesium-activated aldehyde dehydrogenase; Potassium-activated aldehyde dehydrogenase; aldehyde dehydrogenase (acetaldehyde, NADP); aldehyde dehydrogenase (acetylaldehyde, NADP); aldehyde dehydrogenase, putative / antiquitin, putative'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00071[0] <=> (1) cpd00005[0] + (1) cpd00029[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00029;cpd00067;cpd00071',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) Acetaldehyde[0] => (1) NADPH[0] + (1) Acetate[0] + (2) H+[0]',\n",
       "  'deltag': -11.48,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.3', '1.2.1.4', '1.2.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00071[0] => (1) cpd00005[0] + (1) cpd00029[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00507',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30086;rxn31363',\n",
       "  'name': 'Acetaldehyde:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); CARBOXYLATES-DEG (Carboxylate Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); NAD-Metabolism (NAD Metabolism); PWY-5768 (pyruvate fermentation to acetate VIII); PWY-7245 (superpathway NAD/NADP-NADH/NADPH interconversion (yeast)); PWY-7268 (NAD/NADP-NADH/NADPH cytosolic interconversion (yeast)); Pyruvate-Acetate-Fermentation (Pyruvate Fermentation to Acetate); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00071:0:0:\"Acetaldehyde\";1:cpd00005:0:0:\"NADPH\";1:cpd00029:0:0:\"Acetate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00713',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUCCINATE-SEMIALDEHYDE-DEHYDROGENASE-RXN',\n",
       "   'BiGG: SSALx',\n",
       "   'ChlamyCyc: SUCCINATE-SEMIALDEHYDE-DEHYDROGENASE-RXN',\n",
       "   'KEGG: R00713',\n",
       "   'MetaCyc: RXN0-5293; SUCCINATE-SEMIALDEHYDE-DEHYDROGENASE-RXN',\n",
       "   'Name: Succinate-semialdehyde dehydrogenase; Succinate-semialdehyde:NAD+ oxidoreductase; succinate semialdehyde dehydrogenase (NAD+); succinate semialdehyde:NAD+ oxidoreductase; succinate-semialdehyde dehydrogenase (NAD); succinate-semialdehyde dehydrogenase (NAD+); succinate-semialdehyde:NAD+ oxidoreductase; succinic semialdehyde dehydrogenase (NAD+); succinyl semialdehyde dehydrogenase (NAD+)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00199[0] <=> (1) cpd00004[0] + (1) cpd00036[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00036;cpd00067;cpd00199',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) 4-Oxobutanoate[0] <=> (1) NADH[0] + (1) Succinate[0] + (2) H+[0]',\n",
       "  'deltag': -8.47,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.16', '1.2.1.24'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00199[0] <=> (1) cpd00004[0] + (1) cpd00036[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00508',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27251;rxn31894',\n",
       "  'name': 'Succinate-semialdehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Aminobutyraye-Degradation (4-Aminobutanoate Degradation); 4AMINOBUTMETAB-PWY (superpathway of 4-aminobutanoate degradation); AMINE-DEG (Amine and Polyamine Degradation); ARGDEG-PWY (superpathway of L-arginine, putrescine, and 4-aminobutanoate degradation); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); GLUDEG-I-PWY (GABA shunt); GLUTAMATE-DEG (L-glutamate Degradation); ORNARGDEG-PWY (superpathway of L-arginine and L-ornithine degradation); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-4321 (L-glutamate degradation IV); PWY-6473 (4-aminobutanoate degradation IV); PWY-6535 (4-aminobutanoate degradation I); PWY-6536 (4-aminobutanoate degradation III); PWY-7978 (<i>N</i>-methylpyrrolidone degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00350 (Tyrosine metabolism); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00199:0:0:\"4-Oxobutanoate\";1:cpd00004:0:0:\"NADH\";1:cpd00036:0:0:\"Succinate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00714',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUCCSEMIALDDEHYDROG-RXN',\n",
       "   'BiGG: SSALy; SSALyr; YBR006W',\n",
       "   'ChlamyCyc: SUCCSEMIALDDEHYDROG-RXN',\n",
       "   'KEGG: R00714',\n",
       "   'MetaCyc: RXN0-5293; SUCCSEMIALDDEHYDROG-RXN',\n",
       "   'Name: GabD; NADP-dependent succinate-semialdehyde dehydrogenase; Succinate-semialdehyde dehydrogenase (NAD(P)+); Succinate-semialdehyde dehydrogenase [NADP+]; Succinate-semialdehyde:NADP+ oxidoreductase; succinate semialdehyde:NADP+ oxidoreductase; succinate-semialdehyde dehydrogenase (NADP); succinate-semialdehyde dehydrogenase (NADP) reversible; succinate-semialdehyde dehydrogenase (NADP+); succinate-semialdehyde dehydrogenase (nad(p)+); succinate-semialdehyde:NADP+ oxidoreductase; succinic semialdehyde dehydrogenase (NADP+); succinyl semialdehyde dehydrogenase (NADP+)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00199[0] <=> (1) cpd00005[0] + (1) cpd00036[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00036;cpd00067;cpd00199',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) 4-Oxobutanoate[0] <=> (1) NADPH[0] + (1) Succinate[0] + (2) H+[0]',\n",
       "  'deltag': -8.47,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.16', '1.2.1.79'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00199[0] <=> (1) cpd00005[0] + (1) cpd00036[0] + (2) cpd00067[0]',\n",
       "  'id': 'rxn00509',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29002;rxn32195',\n",
       "  'name': 'Succinate-semialdehyde:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 3-HYDROXYPHENYLACETATE-DEGRADATION-PWY (4-hydroxyphenylacetate degradation); 4-Aminobutyraye-Degradation (4-Aminobutanoate Degradation); 4AMINOBUTMETAB-PWY (superpathway of 4-aminobutanoate degradation); AMINE-DEG (Amine and Polyamine Degradation); ARGDEG-PWY (superpathway of L-arginine, putrescine, and 4-aminobutanoate degradation); ARGININE-DEG (L-arginine Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Nicotine-Degradation (Nicotine Degradation); ORNARGDEG-PWY (superpathway of L-arginine and L-ornithine degradation); Other-Degradation (Degradation/Utilization/Assimilation-Other); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); P181-PWY (nicotine degradation I (pyridine pathway)); PWY-6536 (4-aminobutanoate degradation III); PWY-6537 (4-aminobutanoate degradation II); PWY-6993 (nicotine degradation II (pyrrolidine pathway)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00350 (Tyrosine metabolism); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00199:0:0:\"4-Oxobutanoate\";1:cpd00005:0:0:\"NADPH\";1:cpd00036:0:0:\"Succinate\";2:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00715',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.5.1.7-RXN',\n",
       "   'BiGG: LYS1; SACCD2',\n",
       "   'BrachyCyc: 1.5.1.7-RXN',\n",
       "   'KEGG: R00715',\n",
       "   'MetaCyc: 1.5.1.7-RXN',\n",
       "   'Name: (L-lysine-forming); 6-N-(L-1,3-dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-lysine-forming); N6-(L-1,3-Dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-lysine-forming); N6-(L-1,3-dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-lysine-forming); N6-(glutar-2-yl)-L-lysine:NAD oxidoreductase (L-lysine-forming); Saccharopine dehydrogenase [NAD+, L-lysine-forming]; dehydrogenase, saccharopine (nicotinamide adenine dinucleotide, lysine forming); epsilon-N-(L-glutaryl-2)-L-lysine:NAD oxidoreductase (L-lysine forming); lysine-2-oxoglutarate reductase; saccharopine dehydrogenase (NAD+, L-lysine-forming); saccharopine dehydrogenase (NAD, L-lysine forming)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00351[0] <=> (1) cpd00004[0] + (1) cpd00024[0] + (1) cpd00039[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00024;cpd00039;cpd00067;cpd00351',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Saccharopine[0] <=> (1) NADH[0] + (1) 2-Oxoglutarate[0] + (1) L-Lysine[0] + (1) H+[0]',\n",
       "  'deltag': 8.74,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00351[0] <=> (1) cpd00004[0] + (1) cpd00024[0] + (1) cpd00039[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00510',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N6-(L-1,3-Dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-lysine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); LYSINE-AMINOAD-PWY (L-lysine biosynthesis IV); LYSINE-SYN (L-lysine Biosynthesis)',\n",
       "   'KEGG: rn00300 (Lysine biosynthesis); rn00310 (Lysine degradation); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00351:0:0:\"Saccharopine\";1:cpd00004:0:0:\"NADH\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00716',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.5.1.8-RXN',\n",
       "   'BrachyCyc: 1.5.1.8-RXN',\n",
       "   'ChlamyCyc: 1.5.1.8-RXN',\n",
       "   'KEGG: R00716',\n",
       "   'MetaCyc: 1.5.1.8-RXN',\n",
       "   'Name: (L-lysine-forming); 6-N-(L-1,3-dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming); L-lysine-alpha-ketoglutarate reductase; N6-(L-1,3-Dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming); N6-(L-1,3-dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming); lysine-2-oxoglutarate reductase; lysine-ketoglutarate reductase; lysine:alpha-ketoglutarate:TPNH oxidoreductase (epsilon-N-[gultaryl-2]-L-lysine forming); saccharopine (nicotinamide adenine dinucleotide phosphate, lysine-forming) dehydrogenase; saccharopine dehydrogenase (NADP+, L-lysine-forming)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00351[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (1) cpd00039[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00024;cpd00039;cpd00067;cpd00351',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) Saccharopine[0] <=> (1) NADPH[0] + (1) 2-Oxoglutarate[0] + (1) L-Lysine[0] + (1) H+[0]',\n",
       "  'deltag': 8.74,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00351[0] <=> (1) cpd00005[0] + (1) cpd00024[0] + (1) cpd00039[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00511',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17246;rxn37893',\n",
       "  'name': 'N6-(L-1,3-Dicarboxypropyl)-L-lysine:NADP+ oxidoreductase (L-lysine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LYSINE-DEG (L-lysine Degradation); LYSINE-DEG1-PWY (L-lysine degradation XI (mammalian)); PWY-5327 (superpathway of L-lysine degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00310 (Lysine degradation); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00351:0:0:\"Saccharopine\";1:cpd00005:0:0:\"NADPH\";1:cpd00024:0:0:\"2-Oxoglutarate\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00717',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCOLATE-REDUCTASE-RXN',\n",
       "   'BiGG: GLYCLTDx; GLYCLTDxr',\n",
       "   'KEGG: R00717',\n",
       "   'MaizeCyc: GLYCOLATE-REDUCTASE-RXN',\n",
       "   'MetaCyc: GLYCOLATE-REDUCTASE-RXN',\n",
       "   'Name: Glycolate dehydrogenase (NAD); Glycolate:NAD+ oxidoreductase; NADH-dependent glyoxylate reductase; NADH-glyoxylate reductase; R_Glycolate_NAD_oxidoreductase; glycolate dehydrogenase; glycolate:NAD+ oxidoreductase; glyoxylate reductase; glyoxylic acid reductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00139[0] <=> (1) cpd00004[0] + (1) cpd00040[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00040;cpd00067;cpd00139',\n",
       "  'definition': '(1) NAD[0] + (1) Glycolate[0] <=> (1) NADH[0] + (1) Glyoxalate[0] + (1) H+[0]',\n",
       "  'deltag': 10.54,\n",
       "  'deltagerr': 0.21,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.26', '1.1.1.29', '1.1.99.14'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00139[0] <=> (1) cpd00004[0] + (1) cpd00040[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00512',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31400;rxn34300',\n",
       "  'name': 'Glycolate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); L-Arabinose-Degradation (L-arabinose Degradation); PWY-5257 (superpathway of pentose and pentitol degradation); PWY-7294 (D-xylose degradation IV); PWY-7295 (L-arabinose degradation IV); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); Xylose-Degradation (Xylose Degradation)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00139:0:0:\"Glycolate\";1:cpd00004:0:0:\"NADH\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'R00719',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NTP10',\n",
       "   'EcoCyc: RXN0-5073',\n",
       "   'KEGG: R00719',\n",
       "   'MetaCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN; RXN0-5073',\n",
       "   'SoyCyc: NUCLEOSIDE-TRIPHOSPHATASE-RXN',\n",
       "   'Name: ITP diphosphohydrolase; ITP phosphohydrolase; nucleoside-triphosphatase (ITP)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00068[0] <=> (1) cpd00009[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00067;cpd00068;cpd00090',\n",
       "  'definition': '(1) H2O[0] + (1) ITP[0] => (1) Phosphate[0] + (1) H+[0] + (1) IDP[0]',\n",
       "  'deltag': -7.03,\n",
       "  'deltagerr': 0.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.-', '3.6.1.15', '3.6.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00068[0] => (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00513',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP phosphohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00068:0:0:\"ITP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00720',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.1.19-RXN',\n",
       "   'BiGG: ITPASE2; NTPP9',\n",
       "   'EcoCyc: 3.6.1.19-RXN; RXN0-6382',\n",
       "   'KEGG: R00720',\n",
       "   'MetaCyc: 3.6.1.19-RXN; RXN0-6382',\n",
       "   \"Name: Inosine 5'-triphosphate pyrophosphohydrolase; Nucleoside triphosphate pyrophosphorylase (itp)\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00068[0] <=> (1) cpd00012[0] + (1) cpd00114[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00067;cpd00068;cpd00114',\n",
       "  'definition': '(1) H2O[0] + (1) ITP[0] => (1) PPi[0] + (2) H+[0] + (1) IMP[0]',\n",
       "  'deltag': -2.25,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.-', '3.6.1.19', '3.6.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00068[0] => (1) cpd00012[0] + (2) cpd00067[0] + (1) cpd00114[0]',\n",
       "  'id': 'rxn00514',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32143;rxn32148',\n",
       "  'name': \"Inosine 5'-triphosphate pyrophosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00068:0:0:\"ITP\";1:cpd00012:0:0:\"PPi\";2:cpd00067:0:0:\"H+\";1:cpd00114:0:0:\"IMP\"'},\n",
       " {'abbreviation': 'R00722',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00722_c',\n",
       "   'AraGEM: R_R00722_c',\n",
       "   'BiGG: NDPK9; YNK1_9',\n",
       "   'KEGG: R00722',\n",
       "   'MetaCyc: RXN-14120',\n",
       "   'Name: ATP:IDP phosphotransferase; nucleoside-diphophate kinase (ATP:IDP); nucleoside-diphosphate kinase (ATP:IDP)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00090[0] <=> (1) cpd00008[0] + (1) cpd00068[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00068;cpd00090',\n",
       "  'definition': '(1) ATP[0] + (1) IDP[0] <=> (1) ADP[0] + (1) ITP[0]',\n",
       "  'deltag': 0.87,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00090[0] <=> (1) cpd00008[0] + (1) cpd00068[0]',\n",
       "  'id': 'rxn00515',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33570;rxn33573',\n",
       "  'name': 'ATP:IDP phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00090:0:0:\"IDP\";1:cpd00008:0:0:\"ADP\";1:cpd00068:0:0:\"ITP\"'},\n",
       " {'abbreviation': 'R00723',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00723',\n",
       "   \"Name: Inosine 5'-tetraphosphate phosphohydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02272[0] <=> (1) cpd00009[0] + (1) cpd00068[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00067;cpd00068;cpd02272',\n",
       "  'definition': \"(1) H2O[0] + (1) Inosine 5'-tetraphosphate[0] => (1) Phosphate[0] + (1) H+[0] + (1) ITP[0]\",\n",
       "  'deltag': -6.81,\n",
       "  'deltagerr': 3.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.14'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02272[0] => (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00068[0]',\n",
       "  'id': 'rxn00516',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"Inosine 5'-tetraphosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02272:0:0:\"Inosine 5\\'-tetraphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00068:0:0:\"ITP\"'},\n",
       " {'abbreviation': 'R00724',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00724',\n",
       "   'Name: ITP:pyruvate 2-O-phosphotransferase; ITP:pyruvate O2-phosphotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00068[0] <=> (1) cpd00061[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00061;cpd00067;cpd00068;cpd00090',\n",
       "  'definition': '(1) Pyruvate[0] + (1) ITP[0] <=> (1) Phosphoenolpyruvate[0] + (1) H+[0] + (1) IDP[0]',\n",
       "  'deltag': 5.66,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.40'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00068[0] <=> (1) cpd00061[0] + (1) cpd00067[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00517',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP:pyruvate 2-O-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00068:0:0:\"ITP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00725',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00725', 'Name: ITP:D-glucose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00027[0] + (1) cpd00068[0] <=> (1) cpd00079[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00067;cpd00068;cpd00079;cpd00090',\n",
       "  'definition': '(1) D-Glucose[0] + (1) ITP[0] => (1) H+[0] + (1) D-glucose-6-phosphate[0] + (1) IDP[0]',\n",
       "  'deltag': -4.88,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1'],\n",
       "  'equation': '(1) cpd00027[0] + (1) cpd00068[0] => (1) cpd00067[0] + (1) cpd00079[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00518',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP:D-glucose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00027:0:0:\"D-Glucose\";-1:cpd00068:0:0:\"ITP\";1:cpd00067:0:0:\"H+\";1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00726',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00726',\n",
       "   'MetaCyc: RXN-12481',\n",
       "   'Name: ITP:oxaloacetate carboxy-lyase (adding ITP; phosphoenolpyruvate-forming); ITP:oxaloacetate carboxy-lyase (transphosphorylating)'],\n",
       "  'code': '(1) cpd00032[0] + (1) cpd00068[0] <=> (1) cpd00011[0] + (1) cpd00061[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00032;cpd00061;cpd00068;cpd00090',\n",
       "  'definition': '(1) Oxaloacetate[0] + (1) ITP[0] => (1) CO2[0] + (1) Phosphoenolpyruvate[0] + (1) IDP[0]',\n",
       "  'deltag': 1.7,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.32'],\n",
       "  'equation': '(1) cpd00032[0] + (1) cpd00068[0] => (1) cpd00011[0] + (1) cpd00061[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00519',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP:oxaloacetate carboxy-lyase (adding ITP; phosphoenolpyruvate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7383 (anaerobic energy metabolism (invertebrates, cytosol)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates))',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00032:0:0:\"Oxaloacetate\";-1:cpd00068:0:0:\"ITP\";1:cpd00011:0:0:\"CO2\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00727',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00727',\n",
       "   'MetaCyc: RXN-12511',\n",
       "   'TS_Athaliana: rxn597_m',\n",
       "   'Name: Succinate:CoA ligase (IDP-forming)'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00036[0] + (1) cpd00068[0] <=> (1) cpd00009[0] + (1) cpd00078[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00010;cpd00036;cpd00068;cpd00078;cpd00090',\n",
       "  'definition': '(1) CoA[0] + (1) Succinate[0] + (1) ITP[0] <=> (1) Phosphate[0] + (1) Succinyl-CoA[0] + (1) IDP[0]',\n",
       "  'deltag': -1.36,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.4'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00036[0] + (1) cpd00068[0] <=> (1) cpd00009[0] + (1) cpd00078[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00520',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33785',\n",
       "  'name': 'Succinate:CoA ligase (IDP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00020 (Citrate cycle (TCA cycle)); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00036:0:0:\"Succinate\";-1:cpd00068:0:0:\"ITP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00078:0:0:\"Succinyl-CoA\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00728',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00728',\n",
       "   'MetaCyc: TYROSINE-PHENOL-LYASE-RXN',\n",
       "   'Name: L-Tyrosine phenol-lyase (deaminating); L-tyrosine phenol-lyase (deaminating); L-tyrosine phenol-lyase (deaminating, pyruvate-forming); L-tyrosine phenol-lyase (deaminating; pyruvate-forming); beta-tyrosinase; tyrosine phenol-lyase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00069[0] <=> (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00127[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00069;cpd00127',\n",
       "  'definition': '(1) H2O[0] + (1) L-Tyrosine[0] => (1) NH3[0] + (1) Pyruvate[0] + (1) Phenol[0]',\n",
       "  'deltag': -6.32,\n",
       "  'deltagerr': 1.06,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.99.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00069[0] => (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00127[0]',\n",
       "  'id': 'rxn00521',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-tyrosine phenol-lyase (deaminating; pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00350 (Tyrosine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00127:0:0:\"Phenol\"'},\n",
       " {'abbreviation': 'R00729',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: L-AMINO-ACID-OXIDASE-RXN',\n",
       "   'KEGG: R00729',\n",
       "   'MetaCyc: RXN-12487',\n",
       "   'Name: L-Tyrosine:oxygen oxidoreductase (deaminating); L-amino acid oxidase; Ophio-amino-acid oxidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00069[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00868[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00069;cpd00868',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) L-Tyrosine[0] => (1) NH3[0] + (1) H2O2[0] + (1) p-hydroxyphenylpyruvate[0]',\n",
       "  'deltag': -22.92,\n",
       "  'deltagerr': 1.14,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00069[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00868[0]',\n",
       "  'id': 'rxn00522',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38890',\n",
       "  'name': 'L-Tyrosine:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00350 (Tyrosine metabolism); rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn00950 (Isoquinoline alkaloid biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00868:0:0:\"p-hydroxyphenylpyruvate\"'},\n",
       " {'abbreviation': 'R00730',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: TYROSINE-N-MONOOXYGENASE-RXN',\n",
       "   'CornCyc: TYROSINE-N-MONOOXYGENASE-RXN',\n",
       "   'KEGG: R00730',\n",
       "   'MetaCyc: TYROSINE-N-MONOOXYGENASE-RXN',\n",
       "   'PlantCyc: TYROSINE-N-MONOOXYGENASE-RXN',\n",
       "   'Name: CYP79A1; L-Tyrosine,NADPH:oxygen oxidoreductase (N-hydroxylating); L-tyrosine,NADPH:oxygen oxidoreductase (N-hydroxylating); Tyrosine N-hydroxylase; Tyrosine N-monooxygenase; tyrosine N-hydroxylase; tyrosine N-monooxygenase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00069[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd01927[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00069;cpd01927',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) L-Tyrosine[0] => (1) H2O[0] + (1) NADP[0] + (1) N-Hydroxy-L-tyrosine[0]',\n",
       "  'deltag': -90.63,\n",
       "  'deltagerr': 6.28,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.41'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00069[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd01927[0]',\n",
       "  'id': 'rxn00523',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27401;rxn29938;rxn33232',\n",
       "  'name': 'L-tyrosine,NADPH:oxygen oxidoreductase (N-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: CYANOGENIC-GLUCOSIDE-SYN (Cyanogenic Glucoside Biosynthesis); GLUCOSINOLATE-SYN (Glucosinolate Biosynthesis); N-CONTAINING-GLUCOSIDE-SYN (Nitrogen-Containing Glucoside Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-7088 (taxiphyllin biosynthesis); PWY-7901 (glucosinolate biosynthesis from tyrosine); PWY-861 (dhurrin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00460 (Cyanoamino acid metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd01927:0:0:\"N-Hydroxy-L-tyrosine\"'},\n",
       " {'abbreviation': 'R00731',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00731', 'Name: L-Tyrosine:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00069[0] <=> (1) cpd00001[0] + (1) cpd00291[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00067;cpd00069;cpd00291',\n",
       "  'definition': '(1) O2[0] + (2) H+[0] + (1) L-Tyrosine[0] => (1) H2O[0] + (1) L-Dopa[0]',\n",
       "  'deltag': -102.49,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.18.1'],\n",
       "  'equation': '(1) cpd00007[0] + (2) cpd00067[0] + (1) cpd00069[0] => (1) cpd00001[0] + (1) cpd00291[0]',\n",
       "  'id': 'rxn00524',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tyrosine:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00350 (Tyrosine metabolism); rn00950 (Isoquinoline alkaloid biosynthesis); rn00965 (Betalain biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'CI:-2',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00001:0:0:\"H2O\";1:cpd00291:0:0:\"L-Dopa\"'},\n",
       " {'abbreviation': 'R00732',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CYCLOHEXADIENYL-DEHYDROGENASE-RXN',\n",
       "   'BrachyCyc: CYCLOHEXADIENYL-DEHYDROGENASE-RXN',\n",
       "   'ChlamyCyc: CYCLOHEXADIENYL-DEHYDROGENASE-RXN',\n",
       "   'KEGG: R00732',\n",
       "   'MetaCyc: CYCLOHEXADIENYL-DEHYDROGENASE-RXN',\n",
       "   'Name: Arogenate dehydrogenase; Cyclohexadienyl dehydrogenase; L-arogenate:NAD+ oxidoreductase; L-arogenate:NAD+ oxidoreductase (decarboxylating); Pretyrosine dehydrogenase; arogenate dehydrogenase; arogenic dehydrogenase (ambiguous); cyclohexadienyl dehydrogenase; pretyrosine dehydrogenase (ambiguous)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00616[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00069[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00069;cpd00616',\n",
       "  'definition': '(1) NAD[0] + (1) Pretyrosine[0] => (1) NADH[0] + (1) CO2[0] + (1) L-Tyrosine[0]',\n",
       "  'deltag': -5.36,\n",
       "  'deltagerr': 2.64,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.3.1.43', '1.3.1.79'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00616[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00069[0]',\n",
       "  'id': 'rxn00525',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27772',\n",
       "  'name': 'L-arogenate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-6120 (L-tyrosine biosynthesis III); TYROSINE-SYN (L-tyrosine Biosynthesis)',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00616:0:0:\"Pretyrosine\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00069:0:0:\"L-Tyrosine\"'},\n",
       " {'abbreviation': 'R00733',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-5682',\n",
       "   'ChlamyCyc: RXN-5682',\n",
       "   'CornCyc: RXN-5682',\n",
       "   'KEGG: R00733',\n",
       "   'MetaCyc: RXN-5682',\n",
       "   'Name: L-arogenate:NADP+ oxidoreductase; L-arogenate:NADP+ oxidoreductase (decarboxylating); TyrAAT1; TyrAAT2; TyrAa; arogenate dehydrogenase (NADP+); arogenic dehydrogenase (ambiguous); pretyrosine dehydrogenase (ambiguous)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00616[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00069[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00011;cpd00069;cpd00616',\n",
       "  'definition': '(1) NADP[0] + (1) Pretyrosine[0] => (1) NADPH[0] + (1) CO2[0] + (1) L-Tyrosine[0]',\n",
       "  'deltag': -5.36,\n",
       "  'deltagerr': 2.64,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.3.1.78', '1.3.1.79'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00616[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00069[0]',\n",
       "  'id': 'rxn00526',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn24109',\n",
       "  'name': 'L-arogenate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-3461 (L-tyrosine biosynthesis II); PWY-3481 (superpathway of L-phenylalanine and L-tyrosine biosynthesis); TYROSINE-SYN (L-tyrosine Biosynthesis)',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00616:0:0:\"Pretyrosine\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00069:0:0:\"L-Tyrosine\"'},\n",
       " {'abbreviation': 'R00734',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.6.1.57-RXN; TYRAMINOTRANS-RXN; TYROSINE-AMINOTRANSFERASE-RXN',\n",
       "   'BiGG: AAT1_2; AAT2_2; ARO8; ARO9_2; TYRTA; TYRTAi; TYRTAim; TYRTAip; TYRTAm; TYRTAp',\n",
       "   'BrachyCyc: 2.6.1.57-RXN; TYRAMINOTRANS-RXN; TYROSINE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00734',\n",
       "   'MetaCyc: 2.6.1.57-RXN; TYRAMINOTRANS-RXN; TYROSINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: L-Tyrosine:2-oxoglutarate aminotransferase; L-tyrosine:2-oxoglutarate aminotransferase; Tyrosine aminotransferase; Tyrosine transaminase; aminotransferase; tyrosine aminotransferase; tyrosine transaminase; tyrosine transaminase, irreversible; tyrosinetransaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00069[0] <=> (1) cpd00023[0] + (1) cpd00868[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00069;cpd00868',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Tyrosine[0] <=> (1) L-Glutamate[0] + (1) p-hydroxyphenylpyruvate[0]',\n",
       "  'deltag': 0.09,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.1', '2.6.1.5', '2.6.1.57', '2.6.1.9'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00069[0] <=> (1) cpd00023[0] + (1) cpd00868[0]',\n",
       "  'id': 'rxn00527',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27524;rxn30444;rxn30445;rxn37937',\n",
       "  'name': 'L-tyrosine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Hydroxybenzoate-Biosynthesis (4-Hydroxybenzoate Biosynthesis); ALKALOIDS-SYN (Alkaloid Biosynthesis); ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CINNAMATE-SYN (Cinnamate Biosynthesis); COMPLETE-ARO-PWY (superpathway of aromatic amino acid biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); ISOQUINOLINE-ALKALOIDS (Isoquinoline and Benzylisoquinoline Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PWY-3581 ((<i>S</i>)-reticuline biosynthesis I); PWY-5048 (rosmarinic acid biosynthesis I); PWY-5071 (superpathway of rosmarinic acid biosynthesis); PWY-5151 (L-tyrosine degradation II); PWY-5754 (4-hydroxybenzoate biosynthesis I (eukaryotes)); PWY-5864 (superpathway of plastoquinol biosynthesis); PWY-5886 (4-hydroxyphenylpyruvate biosynthesis); PWY-6630 (superpathway of L-tyrosine biosynthesis); PWY-7514 (L-tyrosine degradation IV (to 4-methylphenol)); PWY-7518 (atromentin biosynthesis); PWY-8016 (L-tyrosine degradation V (Stickland reaction)); PWY3O-4108 (L-tyrosine degradation III); Plastoquinone-Biosynthesis (Plastoquinol Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); QUINONE-SYN (Quinone Secondary Metabolite Biosynthesis); Quinone-Biosynthesis (Quinol and Quinone Biosynthesis); Rosmarinate-Biosynthesis (Rosmarinate Biosynthesis); S-Reticuline-Biosynthesis ((<i>S</i>)-reticuline biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TYRFUMCAT-PWY (L-tyrosine degradation I); TYROSINE-DEG (L-tyrosine Degradation); TYROSINE-SYN (L-tyrosine Biosynthesis); TYRSYN (L-tyrosine biosynthesis I)',\n",
       "   'KEGG: rn00130 (Ubiquinone and other terpenoid-quinone biosynthesis); rn00350 (Tyrosine metabolism); rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn00401 (Novobiocin biosynthesis); rn00950 (Isoquinoline alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00868:0:0:\"p-hydroxyphenylpyruvate\"'},\n",
       " {'abbreviation': 'R00735',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00735',\n",
       "   'MetaCyc: TYROSINE--ARGININE-LIGASE-RXN',\n",
       "   'Name: L-Tyrosine:L-arginine ligase (AMP-forming); L-tyrosine:L-arginine ligase (AMP-forming); kyotorphin synthase; kyotorphin synthetase; kyotorphin-synthesizing enzyme; tyrosine-arginine ligase; tyrosyl-arginine synthase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00051[0] + (1) cpd00069[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd01917[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00018;cpd00051;cpd00067;cpd00069;cpd01917',\n",
       "  'definition': '(1) ATP[0] + (1) L-Arginine[0] + (1) L-Tyrosine[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0] + (1) Kyotorphin[0]',\n",
       "  'deltag': -16.37,\n",
       "  'deltagerr': 1.65,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.24'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00051[0] + (1) cpd00069[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd01917[0]',\n",
       "  'id': 'rxn00528',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tyrosine:L-arginine ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00051:0:0:\"L-Arginine\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd01917:0:0:\"Kyotorphin\"'},\n",
       " {'abbreviation': 'R00736',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: TYROSINE-DECARBOXYLASE-RXN',\n",
       "   'BrachyCyc: TYROSINE-DECARBOXYLASE-RXN',\n",
       "   'CornCyc: TYROSINE-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R00736',\n",
       "   'MetaCyc: TYROSINE-DECARBOXYLASE-RXN',\n",
       "   'Name: L-(-)-tyrosine apodecarboxylase; L-Tyrosine carboxy-lyase; L-tyrosine carboxy-lyase; L-tyrosine carboxy-lyase (tyramine-forming); L-tyrosine decarboxylase; R_L_tyrosine_carboxy_lyase_tyramine_forming_; tyrosine decarboxylase'],\n",
       "  'code': '(1) cpd00069[0] <=> (1) cpd00011[0] + (1) cpd00374[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00069;cpd00374',\n",
       "  'definition': '(1) H+[0] + (1) L-Tyrosine[0] => (1) CO2[0] + (1) Tyramine[0]',\n",
       "  'deltag': -16.86,\n",
       "  'deltagerr': 1.23,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.25', '4.1.1.28'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00069[0] => (1) cpd00011[0] + (1) cpd00374[0]',\n",
       "  'id': 'rxn00529',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31401',\n",
       "  'name': 'L-tyrosine carboxy-lyase (tyramine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); ISOQUINOLINE-ALKALOIDS (Isoquinoline and Benzylisoquinoline Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-3581 ((<i>S</i>)-reticuline biosynthesis I); PWY-5254 (methanofuran biosynthesis); PWY-5474 (hydroxycinnamic acid tyramine amides biosynthesis); PWY-6133 ((S)-reticuline biosynthesis II); PWY-6802 (salidroside biosynthesis); PWY-7297 (octopamine biosynthesis); PWY-7826 (<i>Amaryllidacea</i> alkaloids biosynthesis); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); S-Reticuline-Biosynthesis ((<i>S</i>)-reticuline biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism); rn00950 (Isoquinoline alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00011:0:0:\"CO2\";1:cpd00374:0:0:\"Tyramine\"'},\n",
       " {'abbreviation': 'R00737',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00737_c',\n",
       "   'AraGEM: R_R00737_c',\n",
       "   'KEGG: R00737',\n",
       "   'MetaCyc: 4.3.1.23-RXN; RXN-9697',\n",
       "   'SoyCyc: 4.3.1.23-RXN',\n",
       "   'Name: L-Tyrosine ammonia-lyase; L-phenylalanine(or L-tyrosine):trans-cinnamate(or trans-p-hydroxycinnamate) ammonia-lyase; L-tyrosine ammonia-lyase (trans-p-hydroxycinnamate-forming); PTAL; bifunctional PAL; phenylalanine/tyrosine ammonia-lyase'],\n",
       "  'code': '(1) cpd00069[0] <=> (1) cpd00013[0] + (1) cpd00604[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00069;cpd00604',\n",
       "  'definition': '(1) L-Tyrosine[0] => (1) NH3[0] + (1) 4-Coumarate[0]',\n",
       "  'deltag': -9.56,\n",
       "  'deltagerr': 1.06,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.23', '4.3.1.25'],\n",
       "  'equation': '(1) cpd00069[0] => (1) cpd00013[0] + (1) cpd00604[0]',\n",
       "  'id': 'rxn00530',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-tyrosine ammonia-lyase (trans-p-hydroxycinnamate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: COUMARIN-SYN (Coumarin Biosynthesis); FLAVONOID-SYN (Flavonoid Biosynthesis); PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PWY-7397 (naringenin biosynthesis (engineered)); PWY-7398 (coumarins biosynthesis (engineered)); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism); rn00940 (Phenylpropanoid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd00013:0:0:\"NH3\";1:cpd00604:0:0:\"4-Coumarate\"'},\n",
       " {'abbreviation': 'R00739',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00739',\n",
       "   'MetaCyc: TYROSINE-23-AMINOMUTASE-RXN',\n",
       "   'Name: L-Tyrosine 2,3-aminomutase; L-tyrosine 2,3-aminomutase; tyrosine 2,3-aminomutase; tyrosine alpha,beta-mutase'],\n",
       "  'code': '(1) cpd00069[0] <=> (1) cpd02674[0]',\n",
       "  'compound_ids': 'cpd00069;cpd02674',\n",
       "  'definition': '(1) L-Tyrosine[0] <=> (1) beta-Tyrosine[0]',\n",
       "  'deltag': -11.79,\n",
       "  'deltagerr': 0.97,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.3.6'],\n",
       "  'equation': '(1) cpd00069[0] <=> (1) cpd02674[0]',\n",
       "  'id': 'rxn00531',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tyrosine 2,3-aminomutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00350 (Tyrosine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00069:0:0:\"L-Tyrosine\";1:cpd02674:0:0:\"beta-Tyrosine\"'},\n",
       " {'abbreviation': 'R00740',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00740',\n",
       "   'MetaCyc: RXN-8963',\n",
       "   'iRS1563: R_R00740_c',\n",
       "   'Name: 3-Oxopropanoate:NADP+ oxidoreductase (decarboxylating,; 3-oxopropanoate:NADP+ oxidoreductase (CoA-malonylating); CoA-acetylating); NADP-dependent malonyl CoA reductase; malonate semialdehyde:NADP+ oxidoreductase (malonate semialdehyde-forming); malonyl CoA reductase (NADP); malonyl CoA reductase (malonate semialdehyde-forming)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00191[0] <=> (1) cpd00005[0] + (1) cpd00070[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00010;cpd00067;cpd00070;cpd00191',\n",
       "  'definition': '(1) NADP[0] + (1) CoA[0] + (1) 3-Oxopropanoate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Malonyl-CoA[0]',\n",
       "  'deltag': -2.02,\n",
       "  'deltagerr': 1.09,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.-', '1.2.1.18', '1.2.1.75'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd00191[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00070[0]',\n",
       "  'id': 'rxn00532',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-oxopropanoate:NADP+ oxidoreductase (CoA-malonylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5743 (3-hydroxypropanoate cycle); PWY-5744 (glyoxylate assimilation); PWY-5789 (3-hydroxypropanoate/4-hydroxybutanate cycle); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle)',\n",
       "   'KEGG: rn00640 (Propanoate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00191:0:0:\"3-Oxopropanoate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00070:0:0:\"Malonyl-CoA\"'},\n",
       " {'abbreviation': 'R00742',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETYL-COA-CARBOXYLTRANSFER-RXN',\n",
       "   'BiGG: ACC1; ACCOAC; ACCOACr; ACCOACrm',\n",
       "   'BrachyCyc: ACETYL-COA-CARBOXYLTRANSFER-RXN',\n",
       "   'KEGG: R00742',\n",
       "   'MetaCyc: ACETYL-COA-CARBOXYLTRANSFER-RXN',\n",
       "   'Name: Acetyl-CoA:carbon-dioxide ligase (ADP-forming); Acetyl-Coa carboxylase; acetyl coenzyme A carboxylase; acetyl-CoA carboxylase; acetyl-CoA carboxylase, reversible reaction; acetyl-CoA carboxylate, reversible reaction; acetyl-CoA:carbon-dioxide ligase (ADP-forming); acetyl-CoAcarboxylase-reversiblereaction'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00022[0] + (1) cpd00242[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00070[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00022;cpd00067;cpd00070;cpd00242',\n",
       "  'definition': '(1) ATP[0] + (1) Acetyl-CoA[0] + (1) H2CO3[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) Malonyl-CoA[0]',\n",
       "  'deltag': -0.95,\n",
       "  'deltagerr': 0.36,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.4.1.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00022[0] + (1) cpd00242[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00070[0]',\n",
       "  'id': 'rxn00533',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27668;rxn30331;rxn38011',\n",
       "  'name': 'Acetyl-CoA:carbon-dioxide ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Fatty-acid-biosynthesis (Fatty Acid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); PWY-6679 (jadomycin biosynthesis); PWY-7388 (octanoyl-[acyl-carrier protein] biosynthesis (mitochondria, yeast)); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00061 (Fatty acid biosynthesis); rn00253 (Tetracycline biosynthesis); rn00254 (Aflatoxin biosynthesis); rn00620 (Pyruvate metabolism); rn00640 (Propanoate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00242:0:0:\"H2CO3\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00070:0:0:\"Malonyl-CoA\"'},\n",
       " {'abbreviation': 'R00743',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00743',\n",
       "   'MetaCyc: MALONATE-COA-TRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:malonate CoA-transferase; acetyl-CoA:malonate CoA-transferase; malonate CoA-transferase; malonate coenzyme A-transferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00308[0] <=> (1) cpd00029[0] + (1) cpd00070[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00029;cpd00070;cpd00308',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Malonate[0] <=> (1) Acetate[0] + (1) Malonyl-CoA[0]',\n",
       "  'deltag': -2.63,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.3'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00308[0] <=> (1) cpd00029[0] + (1) cpd00070[0]',\n",
       "  'id': 'rxn00534',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:malonate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism); rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00308:0:0:\"Malonate\";1:cpd00029:0:0:\"Acetate\";1:cpd00070:0:0:\"Malonyl-CoA\"'},\n",
       " {'abbreviation': 'rxn00535',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd03564[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00070[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00070;cpd03564',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) 3-Oxopropionyl-CoA[0] <=> (1) H2O[0] + (1) NADP[0] + (1) Malonyl-CoA[0]',\n",
       "  'deltag': -119.68,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd03564[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00070[0]',\n",
       "  'id': 'rxn00535',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00535',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd03564:0:0:\"3-Oxopropionyl-CoA\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00070:0:0:\"Malonyl-CoA\"'},\n",
       " {'abbreviation': 'R00746',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-12484',\n",
       "   'BiGG: ALCD2; ALCD2y',\n",
       "   'ChlamyCyc: ALCOHOL-DEHYDROGENASE-NADPORNOP+-RXN',\n",
       "   'KEGG: R00746',\n",
       "   'MetaCyc: ALCOHOL-DEHYDROGENASE-NADP+-RXN; RXN-12484',\n",
       "   'Name: Ethanol:NADP+ oxidoreductase; alcohol dehydrogenase (ethanol, NADP); alcoholdehydrogenase-ethanol-NADP'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00363[0] <=> (1) cpd00005[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00071;cpd00363',\n",
       "  'definition': '(1) NADP[0] + (1) Ethanol[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': 5.03,\n",
       "  'deltagerr': 0.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.2', '1.1.1.71'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00363[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00536',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34194',\n",
       "  'name': 'Ethanol:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00363:0:0:\"Ethanol\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00747',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PALDP; PPACAH',\n",
       "   'KEGG: R00747',\n",
       "   'MetaCyc: PHOSPHONOACETALDEHYDE-HYDROLASE-RXN',\n",
       "   'TS_Athaliana: rxn598_c',\n",
       "   'iMO1053-PAO1: PPACAH',\n",
       "   'Name: 2-Phosphonoacetaldehyde phosphonohydrolase; 2-oxoethylphosphonate phosphonohydrolase; 2-phosphonoacetylaldehyde phosphonohydrolase; Phosphonoacetaldehyde hydrolase; phosphonatase; phosphonoacetaldehyde hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02024[0] <=> (1) cpd00009[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00067;cpd00071;cpd02024',\n",
       "  'definition': '(1) H2O[0] + (1) Phosphonoacetaldehyde[0] => (1) Phosphate[0] + (1) H+[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': -18.07,\n",
       "  'deltagerr': 1.77,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.11.1.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02024[0] => (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00537',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Phosphonoacetaldehyde phosphonohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 2-Aminoethylphosphonate-Degradation (); Degradation (Degradation/Utilization/Assimilation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PHOSPHONOTASE-PWY (2-aminoethylphosphonate degradation I); Phosphorus-Compounds (Phosphorus Compound Metabolism)',\n",
       "   'KEGG: rn00440 (Phosphonate and phosphinate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02024:0:0:\"Phosphonoacetaldehyde\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00748',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00748',\n",
       "   'MetaCyc: ETHANOLAMINE-PHOSPHATE-PHOSPHO-LYASE-RXN',\n",
       "   'Name: Ethanolamine-phosphate phospho-lyase (deaminating); O-phosphoethanolamine-phospholyase; O-phosphorylethanol-amine phospho-lyase; amino alcohol O-phosphate phospholyase; ethanolamine-phosphate phosphate-lyase (deaminating, acetaldehyde-forming); ethanolamine-phosphate phosphate-lyase (deaminating; acetaldehyde-forming); ethanolamine-phosphate phospho-lyase; ethanolamine-phosphate phospho-lyase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00285[0] <=> (1) cpd00009[0] + (1) cpd00013[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00013;cpd00071;cpd00285',\n",
       "  'definition': '(1) H2O[0] + (1) Phosphoethanolamine[0] => (1) Phosphate[0] + (1) NH3[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': -9.47,\n",
       "  'deltagerr': 0.64,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00285[0] => (1) cpd00009[0] + (1) cpd00013[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00538',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ethanolamine-phosphate phosphate-lyase (deaminating; acetaldehyde-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00285:0:0:\"Phosphoethanolamine\";1:cpd00009:0:0:\"Phosphate\";1:cpd00013:0:0:\"NH3\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00749',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ETHAAL',\n",
       "   'EcoCyc: ETHAMLY-RXN',\n",
       "   'KEGG: R00749',\n",
       "   'MetaCyc: ETHAMLY-RXN',\n",
       "   'iAF1260: ETHAAL',\n",
       "   'Name: Ethanolamine ammonia-lyase; ethanolamine ammonia-lyase; ethanolamine ammonia-lyase (acetaldehyde-forming); ethanolamine deaminase'],\n",
       "  'code': '(1) cpd00162[0] <=> (1) cpd00013[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00071;cpd00162',\n",
       "  'definition': '(1) Aminoethanol[0] => (1) NH3[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': -8.71,\n",
       "  'deltagerr': 0.85,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.7'],\n",
       "  'equation': '(1) cpd00162[0] => (1) cpd00013[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00539',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30017',\n",
       "  'name': 'ethanolamine ammonia-lyase (acetaldehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Degradation (Degradation/Utilization/Assimilation); PWY0-1477 (ethanolamine utilization)',\n",
       "   'KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00162:0:0:\"Aminoethanol\";1:cpd00013:0:0:\"NH3\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00750',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00750_c',\n",
       "   'BiGG: HOPNTAL; HOPNTAL3',\n",
       "   'EcoCyc: MHPELY-RXN',\n",
       "   'KEGG: R00750',\n",
       "   'MetaCyc: MHPELY-RXN',\n",
       "   'Name: 4-hydroxy-2-ketovalerate aldolase; 4-hydroxy-2-oxopentanoate aldolase; 4-hydroxy-2-oxopentanoate aldolase(m-xylene); 4-hydroxy-2-oxopentanoate pyruvate-lyase; 4-hydroxy-2-oxopentanoate pyruvate-lyase (acetaldehyde-forming); 4-hydroxy-2-oxovalerate aldolase; 4-hydroxy-2-oxovalerate pyruvate-lyase; DmpG; HOA'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00071[0] <=> (1) cpd02258[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00071;cpd02258',\n",
       "  'definition': '(1) Pyruvate[0] + (1) Acetaldehyde[0] <=> (1) 4-Hydroxy-2-oxovalerate[0]',\n",
       "  'deltag': -2.94,\n",
       "  'deltagerr': 0.42,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.39'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00071[0] <=> (1) cpd02258[0]',\n",
       "  'id': 'rxn00540',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxy-2-oxopentanoate pyruvate-lyase (acetaldehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Benzoate-Degradation (Benzoate Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Catechol-Degradation (Catechol Degradation); Degradation (Degradation/Utilization/Assimilation); MANDELATE-DEG (Mandelate Degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5162 (2-hydroxypenta-2,4-dienoate degradation); PWY-5178 (toluene degradation IV (aerobic) (via catechol)); PWY-5179 (toluene degradation V (aerobic) (<I>via</I> toluene-<I>cis</I>-diol)); PWY-5180 (toluene degradation I (aerobic) (<I>via</I> <i>o</I>-cresol)); PWY-5182 (toluene degradation II (aerobic) (via 4-methylcatechol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-5266 (<I>p</I>-cymene degradation); PWY-5273 (<i>p</i>-cumate degradation); PWY-5415 (catechol degradation I (<I>meta</I>-cleavage pathway)); PWY-5420 (catechol degradation II (<I>meta</I>-cleavage pathway)); PWY-5430 (<i>meta</i> cleavage pathway of aromatic compounds); PWY-5655 (L-tryptophan degradation IX); PWY-6505 (L-tryptophan degradation XII (Geobacillus)); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-6956 (naphthalene degradation to acetyl-CoA); PWY-6957 (mandelate degradation to acetyl-CoA); PWY0-1277 (3-phenylpropanoate and 3-(3-hydroxyphenyl)propanoate degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TOLUENE-DEG (Toluene Degradation); TRYPTOPHAN-DEG (L-tryptophan Degradation)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism); rn00362 (Benzoate degradation); rn00621 (Dioxin degradation); rn00622 (Xylene degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00071:0:0:\"Acetaldehyde\";1:cpd02258:0:0:\"4-Hydroxy-2-oxovalerate\"'},\n",
       " {'abbreviation': 'R00751',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: THREONINE-ALDOLASE-RXN',\n",
       "   'BiGG: GLY1; THRA; THRA2; THRAi; THRAr',\n",
       "   'BrachyCyc: THREONINE-ALDOLASE-RXN',\n",
       "   'KEGG: R00751',\n",
       "   'MetaCyc: THREONINE-ALDOLASE-RXN',\n",
       "   'Name: L-Threonine acetaldehyde-lyase; L-threonine acetaldehyde-lyase; L-threonine acetaldehyde-lyase (glycine-forming); L-threonine aldolase; Threonine aldolase; threonine aldolase'],\n",
       "  'code': '(1) cpd00161[0] <=> (1) cpd00033[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00033;cpd00071;cpd00161',\n",
       "  'definition': '(1) L-Threonine[0] <=> (1) Glycine[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': 2.36,\n",
       "  'deltagerr': 0.55,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.5'],\n",
       "  'equation': '(1) cpd00161[0] <=> (1) cpd00033[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00541',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32996;rxn32998;rxn35214',\n",
       "  'name': 'L-threonine acetaldehyde-lyase (glycine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); GLYCINE-SYN (Glycine Biosynthesis); GLYSYN-THR-PWY (glycine biosynthesis IV); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-5436 (L-threonine degradation IV); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); THREOCAT-PWY (superpathway of L-threonine metabolism); THREONINE-DEG (L-threonine Degradation)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00161:0:0:\"L-Threonine\";1:cpd00033:0:0:\"Glycine\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00753',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00753',\n",
       "   'MetaCyc: LACTATE-ALDOLASE-RXN',\n",
       "   'Name: (S)-Lactate acetaldehyde-lyase; (S)-lactate acetaldehyde-lyase; (S)-lactate acetaldehyde-lyase (formate-forming); lactate aldolase; lactate synthase'],\n",
       "  'code': '(1) cpd00159[0] <=> (1) cpd00047[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00047;cpd00071;cpd00159',\n",
       "  'definition': '(1) L-Lactate[0] <=> (1) Formate[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': 5.67,\n",
       "  'deltagerr': 0.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.36'],\n",
       "  'equation': '(1) cpd00159[0] <=> (1) cpd00047[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00542',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-lactate acetaldehyde-lyase (formate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00159:0:0:\"L-Lactate\";1:cpd00047:0:0:\"Formate\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00754',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ALCOHOL-DEHYDROG-RXN',\n",
       "   'BiGG: ADH1; ADH2; ADH3; ADH4; ADH5; ALCD1; ALCD2if; ALCD2ir; ALCD2irm; ALCD2m; ALCD2x; SFA1_2',\n",
       "   'BrachyCyc: ALCOHOL-DEHYDROG-RXN',\n",
       "   'KEGG: R00754',\n",
       "   'MetaCyc: ALCOHOL-DEHYDROG-RXN; ALCOHOL-DEHYDROGENASE-NADP+-RXN',\n",
       "   'Name: Alcohol dehydrogenase 1; Alcohol dehydrogenase 3; Ethanol:NAD+ oxidoreductase; alcohol dehydrogenase (ethanol); alcohol dehydrogenase, reverse rxn (acetaldehyde -> ethanol); ethanol dehydroganase; ethanol:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00363[0] <=> (1) cpd00004[0] + (1) cpd00071[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00071;cpd00363',\n",
       "  'definition': '(1) NAD[0] + (1) Ethanol[0] <=> (1) NADH[0] + (1) H+[0] + (1) Acetaldehyde[0]',\n",
       "  'deltag': 5.03,\n",
       "  'deltagerr': 0.08,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.1', '1.1.1.71'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00363[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00071[0]',\n",
       "  'id': 'rxn00543',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27700;rxn30083;rxn38033;rxn38034',\n",
       "  'name': 'ethanol:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Acetaldehyde-Biosynthesis (Acetaldehyde Biosynthesis); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); Alcohol-Degradation (Alcohol Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); ETOH-ACETYLCOA-ANA-PWY (ethanol degradation I); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethanol-Degradation (Ethanol Degradation); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); P122-PWY (heterolactic fermentation); P161-PWY (acetylene degradation); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-5480 (pyruvate fermentation to ethanol I); PWY-5486 (pyruvate fermentation to ethanol II); PWY-6333 (acetaldehyde biosynthesis I); PWY-6587 (pyruvate fermentation to ethanol III); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-7118 (chitin degradation to ethanol); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY0-1477 (ethanolamine utilization); PWY4LZ-257 (superpathway of fermentation (<i>Chlamydomonas reinhardtii</i>)); PWY66-21 (ethanol degradation II); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00363:0:0:\"Ethanol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00071:0:0:\"Acetaldehyde\"'},\n",
       " {'abbreviation': 'R00755',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00755_c',\n",
       "   'AraGEM: R_R00755_c',\n",
       "   'JP_Creinhardtii_MSB: R_PDCh; R_PDCm',\n",
       "   'JP_Creinhardtii_NMeth: R_PDCh; R_PDCm',\n",
       "   'KEGG: R00755',\n",
       "   'Name: pyruvate decarboxylase, putative'],\n",
       "  'code': '(1) cpd00056[0] + (1) cpd00071[0] <=> (1) cpd03049[0]',\n",
       "  'compound_ids': 'cpd00056;cpd00071;cpd03049',\n",
       "  'definition': '(1) TPP[0] + (1) Acetaldehyde[0] <=> (1) 2-Hydroxyethyl-ThPP[0]',\n",
       "  'deltag': -2.94,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.1.1'],\n",
       "  'equation': '(1) cpd00056[0] + (1) cpd00071[0] <=> (1) cpd03049[0]',\n",
       "  'id': 'rxn00544',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31364;rxn31365',\n",
       "  'name': 'R00755',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00010 (Glycolysis / Gluconeogenesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00056:0:0:\"TPP\";-1:cpd00071:0:0:\"Acetaldehyde\";1:cpd03049:0:0:\"2-Hydroxyethyl-ThPP\"'},\n",
       " {'abbreviation': 'R00756',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PFK; PFK1_1; PFK2',\n",
       "   'JP_Creinhardtii_NMeth: R_PFKh',\n",
       "   'KEGG: R00756',\n",
       "   'iAF1260: PFK',\n",
       "   'iAG612: rbs_171',\n",
       "   'Name: ATP:D-fructose-6-phosphate 1-phosphotransferase; phosphofructokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00072[0] <=> (1) cpd00008[0] + (1) cpd00290[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00072;cpd00290',\n",
       "  'definition': '(1) ATP[0] + (1) D-fructose-6-phosphate[0] => (1) ADP[0] + (1) H+[0] + (1) D-fructose-1,6-bisphosphate[0]',\n",
       "  'deltag': -4.64,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.11'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00072[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00290[0]',\n",
       "  'id': 'rxn00545',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34257',\n",
       "  'name': 'ATP:D-fructose-6-phosphate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\"'},\n",
       " {'abbreviation': 'R00758',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: M1PD',\n",
       "   'KEGG: R00758',\n",
       "   'iAF1260: M1PD',\n",
       "   'iAG612: rbs_269',\n",
       "   'iAO358: rll_257',\n",
       "   'Name: D-Mannitol-1-phosphate:NAD+ 5-oxidoreductase; mannitol-1-phosphate 5-dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd27436[0] <=> (1) cpd00004[0] + (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00072;cpd27436',\n",
       "  'definition': '(1) NAD[0] + (1) D-mannitol, 1-(dihydrogenphosphate)[0] <=> (1) NADH[0] + (1) H+[0] + (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': 4.41,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.17'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd27436[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00072[0]',\n",
       "  'id': 'rxn00546',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn13933;rxn13943',\n",
       "  'name': 'D-Mannitol-1-phosphate:NAD+ 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd27436:0:0:D-mannitol, 1-(dihydrogenphosphate);1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00760',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: HEX7; HXK1_4; HXK2_4',\n",
       "   'KEGG: R00760',\n",
       "   'TS_Athaliana: rxn300_c; rxn300_m; rxn300_p',\n",
       "   'iAF1260: HEX7',\n",
       "   'iAF692: HEX7',\n",
       "   'Name: ATP:D-fructose 6-phosphotransferase; Fructokinase; hexokinase (D-fructose:ATP)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00082[0] <=> (1) cpd00008[0] + (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00072;cpd00082',\n",
       "  'definition': '(1) ATP[0] + (1) D-Fructose[0] => (1) ADP[0] + (1) H+[0] + (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': -2.81,\n",
       "  'deltagerr': 0.3,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1', '2.7.1.4'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00082[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00072[0]',\n",
       "  'id': 'rxn00547',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33495;rxn33497',\n",
       "  'name': 'ATP:D-fructose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00082:0:0:\"D-Fructose\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00761',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00761',\n",
       "   'iRS1563: R_R00761_c',\n",
       "   'Name: D-Fructose-6-phosphate D-erythrose-4-phosphate-lyase; D-fructose-6-phosphate D-erythrose-4-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00072[0] <=> (1) cpd00001[0] + (1) cpd00196[0] + (1) cpd00236[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00072;cpd00196;cpd00236',\n",
       "  'definition': '(1) Phosphate[0] + (1) D-fructose-6-phosphate[0] => (1) H2O[0] + (1) Acetylphosphate[0] + (1) D-Erythrose4-phosphate[0]',\n",
       "  'deltag': -12.71,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.2.22'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00072[0] => (1) cpd00001[0] + (1) cpd00196[0] + (1) cpd00236[0]',\n",
       "  'id': 'rxn00548',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-fructose-6-phosphate D-erythrose-4-phosphate-lyase (adding phosphate; acetyl-phosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00710 (Carbon fixation in photosynthetic organisms)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00001:0:0:\"H2O\";1:cpd00196:0:0:\"Acetylphosphate\";1:cpd00236:0:0:\"D-Erythrose4-phosphate\"'},\n",
       " {'abbreviation': 'R00762',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00762_c; R_R00762_p',\n",
       "   'AraGEM: R_R00762_c; R_R00762_p',\n",
       "   'BiGG: FBP; FBP1',\n",
       "   'JP_Creinhardtii_NMeth: R_FBP; R_FBPh',\n",
       "   'KEGG: R00762',\n",
       "   'Name: D-Fructose-1,6-bisphosphate 1-phosphohydrolase; fructose-1,6-bisphosphatase; fructose-1,6-bisphosphatase, putative / D-fructose-1,6-bisphosphate 1-phosphohydrolase, putative / FBPase, putative; fructose-bisphosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00290[0] <=> (1) cpd00009[0] + (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00072;cpd00290',\n",
       "  'definition': '(1) H2O[0] + (1) D-fructose-1,6-bisphosphate[0] <=> (1) Phosphate[0] + (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': -1.52,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00290[0] <=> (1) cpd00009[0] + (1) cpd00072[0]',\n",
       "  'id': 'rxn00549',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31218',\n",
       "  'name': 'D-Fructose-1,6-bisphosphate 1-phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00680 (Methane metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00763',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: FBP26',\n",
       "   'KEGG: R00763',\n",
       "   'TS_Athaliana: rxn100_c',\n",
       "   'iIN800: FBP26',\n",
       "   'iMM904: FBP26',\n",
       "   'Name: Fructose-2,6-bisphosphatase; Fructose-2,6-bisphosphate 2-phosphatase; beta-D-fructose-2,6-bisphosphate 2-phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00503[0] <=> (1) cpd00009[0] + (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00072;cpd00503',\n",
       "  'definition': '(1) H2O[0] + (1) D-Fructose 2,6-bisphosphate[0] <=> (1) Phosphate[0] + (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': -7.26,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.46'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00503[0] <=> (1) cpd00009[0] + (1) cpd00072[0]',\n",
       "  'id': 'rxn00550',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'beta-D-fructose-2,6-bisphosphate 2-phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00503:0:0:\"D-Fructose 2,6-bisphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00764',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00764',\n",
       "   'iRS1563: R_R00764_c',\n",
       "   'Name: Pyrophosphate:D-fructose-6-phosphate 1-phosphotransferase; diphosphate:D-fructose-6-phosphate 1-phosphotransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00072[0] <=> (1) cpd00009[0] + (1) cpd00290[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00012;cpd00067;cpd00072;cpd00290',\n",
       "  'definition': '(1) PPi[0] + (1) D-fructose-6-phosphate[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) D-fructose-1,6-bisphosphate[0]',\n",
       "  'deltag': -1.95,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.90'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00072[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00290[0]',\n",
       "  'id': 'rxn00551',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'diphosphate:D-fructose-6-phosphate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\"'},\n",
       " {'abbreviation': 'R00765',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G6PDA',\n",
       "   'KEGG: R00765',\n",
       "   'iAF1260: G6PDA',\n",
       "   'iAO358: rll_272',\n",
       "   'iJR904: G6PDA',\n",
       "   'Name: D-Glucosamine-6-phosphate ketol-isomerase(deaminating); D-glucosamine-6-phosphate aminohydrolase (ketol isomerizing); D-glucosamine-6-phosphate ketol-isomerase(deaminating); glucosamine-6-P isomerase; glucosamine-6-phosphate deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00288[0] <=> (1) cpd00013[0] + (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00072;cpd00288',\n",
       "  'definition': '(1) H2O[0] + (1) D-Glucosamine phosphate[0] => (1) NH3[0] + (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': 1.17,\n",
       "  'deltagerr': 0.61,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.99.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00288[0] => (1) cpd00013[0] + (1) cpd00072[0]',\n",
       "  'id': 'rxn00552',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30021',\n",
       "  'name': 'D-glucosamine-6-phosphate aminohydrolase (ketol isomerizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00288:0:0:\"D-Glucosamine phosphate\";1:cpd00013:0:0:\"NH3\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00766',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00766; R06073',\n",
       "   'TS_Athaliana: rxn1552_c',\n",
       "   'iRS1563: R_R00766_c; R_R06073_c',\n",
       "   'Name: UDP-glucose:D-fructose-6-phosphate 2-alpha-D-glucosyltransferase; UDPglucose:D-fructose-6-phosphate 2-alpha-D-glucosyltransferase'],\n",
       "  'code': '(1) cpd00026[0] + (1) cpd00072[0] <=> (1) cpd00014[0] + (1) cpd01693[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00026;cpd00072;cpd01693',\n",
       "  'definition': '(1) UDP-glucose[0] + (1) D-fructose-6-phosphate[0] <=> (1) UDP[0] + (1) 6-Phosphosucrose[0]',\n",
       "  'deltag': -0.6,\n",
       "  'deltagerr': 0.68,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.14'],\n",
       "  'equation': '(1) cpd00026[0] + (1) cpd00072[0] <=> (1) cpd00014[0] + (1) cpd01693[0]',\n",
       "  'id': 'rxn00553',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-glucose:D-fructose-6-phosphate 2-alpha-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00014:0:0:\"UDP\";1:cpd01693:0:0:\"6-Phosphosucrose\"'},\n",
       " {'abbreviation': 'R00767',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00767',\n",
       "   'Name: CTP:D-fructose-6-phosphate 1-phosphotransferase'],\n",
       "  'code': '(1) cpd00052[0] + (1) cpd00072[0] <=> (1) cpd00096[0] + (1) cpd00290[0]',\n",
       "  'compound_ids': 'cpd00052;cpd00067;cpd00072;cpd00096;cpd00290',\n",
       "  'definition': '(1) CTP[0] + (1) D-fructose-6-phosphate[0] => (1) H+[0] + (1) CDP[0] + (1) D-fructose-1,6-bisphosphate[0]',\n",
       "  'deltag': -4.37,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.11'],\n",
       "  'equation': '(1) cpd00052[0] + (1) cpd00072[0] => (1) cpd00067[0] + (1) cpd00096[0] + (1) cpd00290[0]',\n",
       "  'id': 'rxn00554',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CTP:D-fructose-6-phosphate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00052:0:0:\"CTP\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00096:0:0:\"CDP\";1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\"'},\n",
       " {'abbreviation': 'R00768',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00768_c',\n",
       "   'AraGEM: R_R00768_c',\n",
       "   'BiGG: GF6PTA; GF6PTAr; GFA1',\n",
       "   'KEGG: R00768',\n",
       "   'Maize_C4GEM: R_R00768_c',\n",
       "   'Name: Glucosamine--fructose-6-phosphate aminotransferase (isomerizing); L-Glutamine:D-fructose-6-phosphate aminotransferase (hexose; L-glutamine:D-fructose-6-phosphate aminotransferase (hexose isomerizing); L-glutamine:D-fructose-6-phosphate isomerase (deaminating); glucosamine--fructose-6-phosphate aminotransferase (isomerizing); glutamine-fructose-6-phosphate transaminase; glutamine-fructose-6-phosphatetransaminase; isomerizing)'],\n",
       "  'code': '(1) cpd00053[0] + (1) cpd00072[0] <=> (1) cpd00023[0] + (1) cpd00288[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00053;cpd00072;cpd00288',\n",
       "  'definition': '(1) L-Glutamine[0] + (1) D-fructose-6-phosphate[0] <=> (1) L-Glutamate[0] + (1) D-Glucosamine phosphate[0]',\n",
       "  'deltag': -3.99,\n",
       "  'deltagerr': 0.64,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.16'],\n",
       "  'equation': '(1) cpd00053[0] + (1) cpd00072[0] <=> (1) cpd00023[0] + (1) cpd00288[0]',\n",
       "  'id': 'rxn00555',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32367;rxn32369;rxn32371',\n",
       "  'name': 'L-glutamine:D-fructose-6-phosphate isomerase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00288:0:0:\"D-Glucosamine phosphate\"'},\n",
       " {'abbreviation': 'R00769',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00769',\n",
       "   'iRS1563: R_R00769_c',\n",
       "   'Name: UTP:D-fructose-6-phosphate 1-phosphotransferase'],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd00072[0] <=> (1) cpd00014[0] + (1) cpd00290[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00062;cpd00067;cpd00072;cpd00290',\n",
       "  'definition': '(1) UTP[0] + (1) D-fructose-6-phosphate[0] => (1) UDP[0] + (1) H+[0] + (1) D-fructose-1,6-bisphosphate[0]',\n",
       "  'deltag': -3.97,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.11'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd00072[0] => (1) cpd00014[0] + (1) cpd00067[0] + (1) cpd00290[0]',\n",
       "  'id': 'rxn00556',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UTP:D-fructose-6-phosphate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00014:0:0:\"UDP\";1:cpd00067:0:0:\"H+\";1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\"'},\n",
       " {'abbreviation': 'R00770',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00770',\n",
       "   'iRS1563: R_R00770_c',\n",
       "   'Name: ITP:D-fructose-6-phosphate 1-phosphotransferase'],\n",
       "  'code': '(1) cpd00068[0] + (1) cpd00072[0] <=> (1) cpd00090[0] + (1) cpd00290[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00068;cpd00072;cpd00090;cpd00290',\n",
       "  'definition': '(1) ITP[0] + (1) D-fructose-6-phosphate[0] => (1) H+[0] + (1) IDP[0] + (1) D-fructose-1,6-bisphosphate[0]',\n",
       "  'deltag': -5.51,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.11'],\n",
       "  'equation': '(1) cpd00068[0] + (1) cpd00072[0] => (1) cpd00067[0] + (1) cpd00090[0] + (1) cpd00290[0]',\n",
       "  'id': 'rxn00557',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP:D-fructose-6-phosphate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00068:0:0:\"ITP\";-1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00090:0:0:\"IDP\";1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\"'},\n",
       " {'abbreviation': 'R00771',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PGI; PGI1_1',\n",
       "   'JP_Creinhardtii_NMeth: R_PGI; R_PGIh',\n",
       "   'KEGG: R00771',\n",
       "   'iAF1260: PGI',\n",
       "   'iAF692: PGI',\n",
       "   'Name: D-Glucose-6-phosphate ketol-isomerase; D-glucose-6-phosphate aldose-ketose-isomerase; glucose-6-phosphate isomerase; glucose-6-phosphateisomerase'],\n",
       "  'code': '(1) cpd00079[0] <=> (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00072;cpd00079',\n",
       "  'definition': '(1) D-glucose-6-phosphate[0] <=> (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': 1.4,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.9'],\n",
       "  'equation': '(1) cpd00079[0] <=> (1) cpd00072[0]',\n",
       "  'id': 'rxn00558',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34259',\n",
       "  'name': 'D-glucose-6-phosphate aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00772',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MAN6PI; PMI40',\n",
       "   'KEGG: R00772',\n",
       "   'TS_Athaliana: rxn446_m; rxn446_v',\n",
       "   'iAF1260: MAN6PI',\n",
       "   'iAF692: MAN6PI',\n",
       "   'Name: D-Mannose-6-phosphate ketol-isomerase; D-mannose-6-phosphate aldose-ketose-isomerase; Mannose-6-phosphate isomerase; mannose-6-phosphate isomerase; mannose-6-phosphateisomerase'],\n",
       "  'code': '(1) cpd00235[0] <=> (1) cpd00072[0]',\n",
       "  'compound_ids': 'cpd00072;cpd00235',\n",
       "  'definition': '(1) D-mannose-6-phosphate[0] <=> (1) D-fructose-6-phosphate[0]',\n",
       "  'deltag': -0.21,\n",
       "  'deltagerr': 0.17,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.8'],\n",
       "  'equation': '(1) cpd00235[0] <=> (1) cpd00072[0]',\n",
       "  'id': 'rxn00559',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33558;rxn33575',\n",
       "  'name': 'D-mannose-6-phosphate aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00235:0:0:\"D-mannose-6-phosphate\";1:cpd00072:0:0:\"D-fructose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00774',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00774_c',\n",
       "   'BiGG: UREASE',\n",
       "   'JP_Creinhardtii_MSB: R_URCB; R_URCBm',\n",
       "   'KEGG: R00774',\n",
       "   'iMM904: UREASE',\n",
       "   'Name: Urea:carbon-dioxide ligase (ADP-forming); urea carboxylase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00073[0] + (1) cpd00242[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00742[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00067;cpd00073;cpd00242;cpd00742',\n",
       "  'definition': '(1) ATP[0] + (1) Urea[0] + (1) H2CO3[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) Allophanate[0]',\n",
       "  'deltag': 8.13,\n",
       "  'deltagerr': 1.34,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00073[0] + (1) cpd00242[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00742[0]',\n",
       "  'id': 'rxn00560',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35318',\n",
       "  'name': 'Urea:carbon-dioxide ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00073:0:0:\"Urea\";-1:cpd00242:0:0:\"H2CO3\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00742:0:0:\"Allophanate\"'},\n",
       " {'abbreviation': 'R00775',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00775',\n",
       "   'MetaCyc: GUANIDINOACETASE-RXN',\n",
       "   'Name: Guanidinoacetate amidohydrolase; glycocyaminase; guanidinoacetase; guanidinoacetate amidinohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00451[0] <=> (1) cpd00033[0] + (1) cpd00073[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00033;cpd00073;cpd00451',\n",
       "  'definition': '(1) H2O[0] + (1) Glycocyamine[0] => (1) Glycine[0] + (1) Urea[0]',\n",
       "  'deltag': -6.92,\n",
       "  'deltagerr': 1.69,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00451[0] => (1) cpd00033[0] + (1) cpd00073[0]',\n",
       "  'id': 'rxn00561',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Guanidinoacetate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00451:0:0:\"Glycocyamine\";1:cpd00033:0:0:\"Glycine\";1:cpd00073:0:0:\"Urea\"'},\n",
       " {'abbreviation': 'R00776',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UREIDOGLYCOLATE-LYASE-RXN',\n",
       "   'ChlamyCyc: UREIDOGLYCOLATE-LYASE-RXN',\n",
       "   'CornCyc: UREIDOGLYCOLATE-LYASE-RXN',\n",
       "   'KEGG: R00776',\n",
       "   'MetaCyc: UREIDOGLYCOLATE-LYASE-RXN',\n",
       "   'Name: (-)-Ureidoglycolate urea-lyase; (S)-ureidoglycolate urea-lyase; (S)-ureidoglycolate urea-lyase (glyoxylate-forming); ureidoglycolase; ureidoglycolatase; ureidoglycolate hydrolase; ureidoglycolate lyase'],\n",
       "  'code': '(1) cpd00465[0] <=> (1) cpd00040[0] + (1) cpd00073[0]',\n",
       "  'compound_ids': 'cpd00040;cpd00073;cpd00465',\n",
       "  'definition': '(1) Ureidoglycolate[0] <=> (1) Glyoxalate[0] + (1) Urea[0]',\n",
       "  'deltag': 1.21,\n",
       "  'deltagerr': 0.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.3.2.3'],\n",
       "  'equation': '(1) cpd00465[0] <=> (1) cpd00040[0] + (1) cpd00073[0]',\n",
       "  'id': 'rxn00562',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27429',\n",
       "  'name': '(S)-ureidoglycolate urea-lyase (glyoxylate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALLANTOINDEG-PWY (superpathway of allantoin degradation in yeast); AMINE-DEG (Amine and Polyamine Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Allantoin-degradation (Allantoin Degradation); Degradation (Degradation/Utilization/Assimilation); P165-PWY (superpathway of purines degradation in plants); PWY-5694 (allantoin degradation to glyoxylate I); PWY-5705 (allantoin degradation to glyoxylate III); URDEGR-PWY (superpathway of allantoin degradation in plants); Urate-Degradation (Urate Degradation)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00465:0:0:\"Ureidoglycolate\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00073:0:0:\"Urea\"'},\n",
       " {'abbreviation': 'R00777',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00777',\n",
       "   'MetaCyc: AMIDINOASPARTASE-RXN',\n",
       "   'Name: N-Amidino-L-aspartate amidinohydrolase; N-amidino-L-aspartate amidinohydrolase; amidinoaspartase; amidinoaspartic amidinohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02008[0] <=> (1) cpd00041[0] + (1) cpd00073[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00041;cpd00073;cpd02008',\n",
       "  'definition': '(1) H2O[0] + (1) N-Amidino-L-aspartate[0] => (1) L-Aspartate[0] + (1) Urea[0]',\n",
       "  'deltag': -14.69,\n",
       "  'deltagerr': 1.56,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.3.14'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02008[0] => (1) cpd00041[0] + (1) cpd00073[0]',\n",
       "  'id': 'rxn00563',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Amidino-L-aspartate amidinohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02008:0:0:\"N-Amidino-L-aspartate\";1:cpd00041:0:0:\"L-Aspartate\";1:cpd00073:0:0:\"Urea\"'},\n",
       " {'abbreviation': 'R00778',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00778',\n",
       "   'MetaCyc: CYANAMIDE-HYDRATASE-RXN',\n",
       "   'Name: Urea hydro-lyase; cyanamide hydratase; urea hydro-lyase; urea hydro-lyase (cyanamide-forming)'],\n",
       "  'code': '(1) cpd00073[0] <=> (1) cpd00001[0] + (1) cpd01103[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00073;cpd01103',\n",
       "  'definition': '(1) Urea[0] <=> (1) H2O[0] + (1) Cyanamide[0]',\n",
       "  'deltag': 3.55,\n",
       "  'deltagerr': 4.98,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.69'],\n",
       "  'equation': '(1) cpd00073[0] <=> (1) cpd00001[0] + (1) cpd01103[0]',\n",
       "  'id': 'rxn00564',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'urea hydro-lyase (cyanamide-forming)',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00791 (Atrazine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00073:0:0:\"Urea\";1:cpd00001:0:0:\"H2O\";1:cpd01103:0:0:\"Cyanamide\"'},\n",
       " {'abbreviation': 'R00781',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00781', 'Name: Sulfur:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00074[0] <=> (1) cpd00081[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00067;cpd00074;cpd00081',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) S[0] => (1) H+[0] + (1) Sulfite[0]',\n",
       "  'deltag': -83.01,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.18'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00074[0] => (1) cpd00067[0] + (1) cpd00081[0]',\n",
       "  'id': 'rxn00565',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Sulfur:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00074:0:0:\"S\";1:cpd00067:0:0:\"H+\";1:cpd00081:0:0:\"Sulfite\"'},\n",
       " {'abbreviation': 'R00782',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CYSDS; TRPAS1',\n",
       "   'EcoCyc: LCYSDESULF-RXN',\n",
       "   'KEGG: R00782',\n",
       "   'MetaCyc: LCYSDESULF-RXN',\n",
       "   'PlantCyc: LCYSDESULF-RXN',\n",
       "   'Name: Cysteine Desulfhydrase; L-Cysteine L-homocysteine-lyase (deaminating); L-Cysteine hydrogen-sulfide-lyase (deaminating; pyruvate-forming); Tryptophanase (L-cysteine); cystathionine beta-lyase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00084[0] <=> (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00239[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00067;cpd00084;cpd00239',\n",
       "  'definition': '(1) H2O[0] + (1) L-Cysteine[0] => (1) NH3[0] + (1) Pyruvate[0] + (1) H+[0] + (1) H2S[0]',\n",
       "  'deltag': -0.83,\n",
       "  'deltagerr': 1.04,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.99.1', '4.4.1.1', '4.4.1.28', '4.4.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00084[0] => (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00067[0] + (1) cpd00239[0]',\n",
       "  'id': 'rxn00566',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30000;rxn33352;rxn34674',\n",
       "  'name': 'L-Cysteine hydrogen-sulfide-lyase (deaminating; pyruvate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Hydrogen-Sulfide-Biosynthesis (Hydrogen Sulfide Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY66-426 (hydrogen sulfide biosynthesis II (mammalian)); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00084:0:0:\"L-Cysteine\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00239:0:0:\"H2S\"'},\n",
       " {'abbreviation': 'R00783',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NIR',\n",
       "   'JP_Creinhardtii_MSB: R_NITR_LPAREN_NO_RPAREN_',\n",
       "   'KEGG: R00783',\n",
       "   'iMO1053-PAO1: NIR',\n",
       "   'iMO1056: NIR',\n",
       "   'Name: nitric-oxide:ferricytochrome-c oxidoreductase; nitrite reductase (NO-forming)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00109[0] + (1) cpd00418[0] <=> (1) cpd00075[0] + (1) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00075;cpd00109;cpd00110;cpd00418',\n",
       "  'definition': '(1) H2O[0] + (1) Cytochrome c3+[0] + (1) NO[0] <= (1) H+[0] + (1) Nitrite[0] + (1) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.7.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00109[0] + (1) cpd00418[0] <= (1) cpd00067[0] + (1) cpd00075[0] + (1) cpd00110[0]',\n",
       "  'id': 'rxn00567',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35316',\n",
       "  'name': 'nitric-oxide:ferricytochrome-c oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00109:0:0:\"Cytochrome c3+\";-1:cpd00418:0:0:\"NO\";1:cpd00067:0:0:\"H+\";1:cpd00075:0:0:\"Nitrite\";1:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'NIRBD-RXN.c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NTRIR2x',\n",
       "   'EcoCyc: RXN0-6377',\n",
       "   'KEGG: R00787',\n",
       "   'MetaCyc: NIRBD-RXN; RXN-13854; RXN0-6377',\n",
       "   'iAF1260: NTRIR2x',\n",
       "   'Name: Nitrite reductase (NAD(P)H); nitrite Reductase (NADH); nitrite reductase (nad(p)h)'],\n",
       "  'code': '(2) cpd00001[0] + (3) cpd00003[0] + (1) cpd00013[0] <=> (3) cpd00004[0] + (1) cpd00075[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00067;cpd00075',\n",
       "  'definition': '(2) H2O[0] + (3) NAD[0] + (1) NH3[0] => (3) NADH[0] + (4) H+[0] + (1) Nitrite[0]',\n",
       "  'deltag': 93.91,\n",
       "  'deltagerr': 0.89,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.6.6.-', '1.7.1.15', '1.7.1.4'],\n",
       "  'equation': '(2) cpd00001[0] + (3) cpd00003[0] + (1) cpd00013[0] => (3) cpd00004[0] + (4) cpd00067[0] + (1) cpd00075[0]',\n",
       "  'id': 'rxn00568',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'NIRBD-RXN.c',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitrate-Reduction (Nitrate Reduction); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5675 (nitrate reduction V (assimilatory)); PWY-723 (alkylnitronates degradation)',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-3:cpd00003:0:0:\"NAD\";-1:cpd00013:0:0:\"NH3\";3:cpd00004:0:0:\"NADH\";4:cpd00067:0:0:\"H+\";1:cpd00075:0:0:\"Nitrite\"'},\n",
       " {'abbreviation': 'RXN0-6377.c.metaexp.NADP_NADPH',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NTRIR2y; NTRIRy',\n",
       "   'EcoCyc: RXN0-6377',\n",
       "   'KEGG: R00789',\n",
       "   'MetaCyc: RXN0-6377',\n",
       "   'iAF692: NTRIR2y',\n",
       "   'Name: Nitrite reductase (NAD(P)H); ammonia:NADP+ oxidoreductase; nitrite Reductase (NADPH)'],\n",
       "  'code': '(2) cpd00001[0] + (3) cpd00006[0] + (1) cpd00013[0] <=> (3) cpd00005[0] + (1) cpd00075[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00013;cpd00067;cpd00075',\n",
       "  'definition': '(2) H2O[0] + (3) NADP[0] + (1) NH3[0] => (3) NADPH[0] + (4) H+[0] + (1) Nitrite[0]',\n",
       "  'deltag': 93.92,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.7.1.4'],\n",
       "  'equation': '(2) cpd00001[0] + (3) cpd00006[0] + (1) cpd00013[0] => (3) cpd00005[0] + (4) cpd00067[0] + (1) cpd00075[0]',\n",
       "  'id': 'rxn00569',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nitrite reductase (NAD(P)H)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitrate-Reduction (Nitrate Reduction); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5675 (nitrate reduction V (assimilatory)); PWY-723 (alkylnitronates degradation)',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00001:0:0:\"H2O\";-3:cpd00006:0:0:\"NADP\";-1:cpd00013:0:0:\"NH3\";3:cpd00005:0:0:\"NADPH\";4:cpd00067:0:0:\"H+\";1:cpd00075:0:0:\"Nitrite\"'},\n",
       " {'abbreviation': 'R00793',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00793',\n",
       "   'MetaCyc: HAONITRO-RXN; RXN-13595',\n",
       "   'Name: HAO (ambiguous; Hydroxylamine:oxygen oxidoreductase; cf. EC 1.7.99.8, hydroxylamine oxidoreductase); hydroxylamine oxidase; hydroxylamine oxidoreductase (ambiguous; hydroxylamine:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00075[0] <=> (1) cpd00007[0] + (1) cpd00165[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00075;cpd00165',\n",
       "  'definition': '(1) H2O[0] + (1) Nitrite[0] <= (1) O2[0] + (1) Hydroxylamine[0]',\n",
       "  'deltag': 30.61,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.7.3.4', '1.7.3.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00075[0] <= (1) cpd00007[0] + (1) cpd00165[0]',\n",
       "  'id': 'rxn00570',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19884',\n",
       "  'name': 'Hydroxylamine:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: AMMOXID-PWY (ammonia oxidation I (aerobic)); ANAEROBIC-RESPIRATION (Anaerobic Respiration); Ammonia-oxidation (Ammonia Oxidation); CHEMOAUTOTROPHIC-ENERGY-METABOLISM (Chemoautotrophic Energy Metabolism); Degradation (Degradation/Utilization/Assimilation); Denitrification (Denitrification); Energy-Metabolism (Generation of Precursor Metabolite and Energy); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-7082 (ammonia oxidation IV (autotrophic ammonia oxidizers)); PWY-7084 (nitrifier denitrification); Respiration ()'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00075:0:0:\"Nitrite\";1:cpd00007:0:0:\"O2\";1:cpd00165:0:0:\"Hydroxylamine\"'},\n",
       " {'abbreviation': 'R00794',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NITRATE-REDUCTASE-NADH-RXN; NITRATE-REDUCTASE-NADPORNOPH-RXN',\n",
       "   'BrachyCyc: NITRATE-REDUCTASE-NADH-RXN',\n",
       "   'ChlamyCyc: NITRATE-REDUCTASE-NADH-RXN',\n",
       "   'KEGG: R00794',\n",
       "   'MetaCyc: NITRATE-REDUCTASE-NADH-RXN; NITRATE-REDUCTASE-NADPORNOPH-RXN',\n",
       "   'Name: NADH-dependent nitrate reductase; NADH-nitrate reductase; NADH2:nitrate oxidoreductase; NADH:nitrate oxidoreductase; Nitrate Reductase; assimilatory NADH: nitrate reductase; assimilatory nitrate reductase; nitrate reductase (NADH); nitrate reductase (NADH2); nitrite:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00075[0] <=> (1) cpd00004[0] + (1) cpd00209[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00075;cpd00209',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Nitrite[0] <= (1) NADH[0] + (2) H+[0] + (1) Nitrate[0]',\n",
       "  'deltag': 34.51,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.6.6.1', '1.7.1.1', '1.7.1.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00075[0] <= (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00209[0]',\n",
       "  'id': 'rxn00571',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32166;rxn34009;rxn38182',\n",
       "  'name': 'nitrite:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitrate-Reduction (Nitrate Reduction); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-381 (nitrate reduction II (assimilatory))',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00075:0:0:\"Nitrite\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd00209:0:0:\"Nitrate\"'},\n",
       " {'abbreviation': 'R00796',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NITRATE-REDUCTASE-NADPH-RXN; NITRATE-REDUCTASE-NADPORNOPH-RXN',\n",
       "   'CornCyc: NITRATE-REDUCTASE-NADPH-RXN; NITRATE-REDUCTASE-NADPORNOPH-RXN',\n",
       "   'KEGG: R00796',\n",
       "   'MetaCyc: NITRATE-REDUCTASE-NADPH-RXN; NITRATE-REDUCTASE-NADPORNOPH-RXN',\n",
       "   'SoyCyc: NITRATE-REDUCTASE-NADPH-RXN',\n",
       "   'Name: NADPH-nitrate reductase; NADPH2:nitrate oxidoreductase; NADPH:nitrate oxidoreductase; NADPH:nitrate reductase; Nitrate reductase (NADPH); assimilatory NADPH-nitrate reductase; assimilatory nitrate reductase; assimilatory reduced nicotinamide adenine dinucleotide phosphate-nitrate reductase; nitrate reductase (NADPH); nitrate reductase (NADPH2); nitrite:NADP+ oxidoreductase; triphosphopyridine nucleotide-nitrate reductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00075[0] <=> (1) cpd00005[0] + (1) cpd00209[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00067;cpd00075;cpd00209',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) Nitrite[0] <= (1) NADPH[0] + (2) H+[0] + (1) Nitrate[0]',\n",
       "  'deltag': 34.51,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.7.1.2', '1.7.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00075[0] <= (1) cpd00005[0] + (2) cpd00067[0] + (1) cpd00209[0]',\n",
       "  'id': 'rxn00572',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32381',\n",
       "  'name': 'nitrite:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitrate-Reduction (Nitrate Reduction); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5675 (nitrate reduction V (assimilatory))',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00075:0:0:\"Nitrite\";1:cpd00005:0:0:\"NADPH\";2:cpd00067:0:0:\"H+\";1:cpd00209:0:0:\"Nitrate\"'},\n",
       " {'abbreviation': 'R00799',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00799',\n",
       "   'MetaCyc: NITROETHANE-OXIDASE-RXN',\n",
       "   'Name: Nitroethane oxidase; Nitroethane reductase; Nitroethane:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00025[0] + (1) cpd00071[0] + (1) cpd00075[0] <=> (1) cpd00001[0] + (1) cpd00007[0] + (1) cpd01263[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00071;cpd00075;cpd01263',\n",
       "  'definition': '(1) H2O2[0] + (1) Acetaldehyde[0] + (1) Nitrite[0] <=> (1) H2O[0] + (1) O2[0] + (1) Nitroethane[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.7.3.1'],\n",
       "  'equation': '(1) cpd00025[0] + (1) cpd00071[0] + (1) cpd00075[0] <=> (1) cpd00001[0] + (1) cpd00007[0] + (1) cpd01263[0]',\n",
       "  'id': 'rxn00573',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nitroethane:oxygen oxidoreductase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5355 (nitroethane degradation)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00025:0:0:\"H2O2\";-1:cpd00071:0:0:\"Acetaldehyde\";-1:cpd00075:0:0:\"Nitrite\";1:cpd00001:0:0:\"H2O\";1:cpd00007:0:0:\"O2\";1:cpd01263:0:0:\"Nitroethane\"'},\n",
       " {'abbreviation': 'rxn00574',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00075[0] <=> (1) cpd00209[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00075;cpd00209',\n",
       "  'definition': '(1) H2O[0] + (1) Nitrite[0] <=> (3) H+[0] + (1) Nitrate[0]',\n",
       "  'deltag': 19.26,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00075[0] <=> (3) cpd00067[0] + (1) cpd00209[0]',\n",
       "  'id': 'rxn00574',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00574',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00075:0:0:\"Nitrite\";3:cpd00067:0:0:\"H+\";1:cpd00209:0:0:\"Nitrate\"'},\n",
       " {'abbreviation': 'R00801',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00801_c',\n",
       "   'AraGEM: R_R00801_c',\n",
       "   'BiGG: SUC2; SUCH; SUCR; SUCRe',\n",
       "   'KEGG: R00801; R06087',\n",
       "   'Maize_C4GEM: R_R00801_c',\n",
       "   'Name: Invertase 2; alpha-D-Glucoside glucohydrolase; alpha-glucosidase; alpha-glucoside glucohydrolase; sucrose glucohydrolase; sucrose hydrolyzing enxyme; sucrose hydrolyzing enzyme'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00076[0] <=> (1) cpd00027[0] + (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00076;cpd00082',\n",
       "  'definition': '(1) H2O[0] + (1) Sucrose[0] => (1) D-Glucose[0] + (1) D-Fructose[0]',\n",
       "  'deltag': -5.72,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.10', '3.2.1.20', '3.2.1.48'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00076[0] => (1) cpd00027[0] + (1) cpd00082[0]',\n",
       "  'id': 'rxn00575',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30434;rxn33104',\n",
       "  'name': 'sucrose glucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00076:0:0:\"Sucrose\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00803',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00803; R06034',\n",
       "   'Name: Sucrose:orthophosphate alpha-D-glucosyltransferase; sucrose:phosphate alpha-D-glucosyltransferase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00076[0] <=> (1) cpd00082[0] + (1) cpd00089[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00076;cpd00082;cpd00089',\n",
       "  'definition': '(1) Phosphate[0] + (1) Sucrose[0] => (1) D-Fructose[0] + (1) Glucose-1-phosphate[0]',\n",
       "  'deltag': -0.85,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.4.1.7'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00076[0] => (1) cpd00082[0] + (1) cpd00089[0]',\n",
       "  'id': 'rxn00577',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'sucrose:phosphate alpha-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00076:0:0:\"Sucrose\";1:cpd00082:0:0:\"D-Fructose\";1:cpd00089:0:0:\"Glucose-1-phosphate\"'},\n",
       " {'abbreviation': 'R00805',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SUCROSE-PHOSPHATASE-RXN',\n",
       "   'BrachyCyc: SUCROSE-PHOSPHATASE-RXN',\n",
       "   'ChlamyCyc: SUCROSE-PHOSPHATASE-RXN',\n",
       "   'KEGG: R00805; R06211',\n",
       "   'MetaCyc: SUCROSE-PHOSPHATASE-RXN',\n",
       "   'Name: SPP; Sucrose-6-phosphate phosphohydrolase; Sucrose-phosphatase; sucrose 6-phosphate hydrolase; sucrose-6-phosphatase (incorrect); sucrose-6-phosphate phosphatase; sucrose-6F-phosphate phosphohydrolase; sucrose-phosphatase (incorrect); sucrose-phosphate hydrolase; sucrose-phosphate phosphatase; sucrose-phosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01693[0] <=> (1) cpd00009[0] + (1) cpd00076[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00076;cpd01693',\n",
       "  'definition': '(1) H2O[0] + (1) 6-Phosphosucrose[0] <=> (1) Phosphate[0] + (1) Sucrose[0]',\n",
       "  'deltag': -3.95,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.24'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01693[0] <=> (1) cpd00009[0] + (1) cpd00076[0]',\n",
       "  'id': 'rxn00578',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31083;rxn31919',\n",
       "  'name': 'sucrose-6F-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); PWY-7238 (sucrose biosynthesis II); PWY-7347 (sucrose biosynthesis III); SUCSYN-PWY (sucrose biosynthesis I (from photosynthesis)); Sucrose-Biosynthesis (Sucrose Biosynthesis)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01693:0:0:\"6-Phosphosucrose\";1:cpd00009:0:0:\"Phosphate\";1:cpd00076:0:0:\"Sucrose\"'},\n",
       " {'abbreviation': 'R00806',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraGEM: R_R00806_c',\n",
       "   'KEGG: R00806; R06036',\n",
       "   'Maize_C4GEM: R_R00806_c',\n",
       "   'TS_Athaliana: rxn1553_c; rxn1553_v',\n",
       "   'iRS1563: R_R00806_c',\n",
       "   'Name: Sucrose synthase; UDP-glucose:D-fructose 2-alpha-D-glucosyltransferase; UDPglucose:D-fructose 2-alpha-D-glucosyltransferase'],\n",
       "  'code': '(1) cpd00026[0] + (1) cpd00082[0] <=> (1) cpd00014[0] + (1) cpd00076[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00026;cpd00076;cpd00082',\n",
       "  'definition': '(1) UDP-glucose[0] + (1) D-Fructose[0] <=> (1) UDP[0] + (1) Sucrose[0]',\n",
       "  'deltag': -1.19,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.13'],\n",
       "  'equation': '(1) cpd00026[0] + (1) cpd00082[0] <=> (1) cpd00014[0] + (1) cpd00076[0]',\n",
       "  'id': 'rxn00579',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33446',\n",
       "  'name': 'UDP-glucose:D-fructose 2-alpha-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";-1:cpd00082:0:0:\"D-Fructose\";1:cpd00014:0:0:\"UDP\";1:cpd00076:0:0:\"Sucrose\"'},\n",
       " {'abbreviation': 'R00807',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00807; R06233',\n",
       "   'Name: Sucrose:(acceptor) 3-oxidoreductase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00076[0] <=> (1) cpd00982[0] + (1) cpd03412[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00076;cpd00982;cpd03412',\n",
       "  'definition': '(1) FAD[0] + (1) Sucrose[0] => (1) FADH2[0] + (1) 3-Ketosucrose[0]',\n",
       "  'deltag': -3.29,\n",
       "  'deltagerr': 1.1,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.99.13'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00076[0] => (1) cpd00982[0] + (1) cpd03412[0]',\n",
       "  'id': 'rxn00580',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Sucrose:(acceptor) 3-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00076:0:0:\"Sucrose\";1:cpd00982:0:0:\"FADH2\";1:cpd03412:0:0:\"3-Ketosucrose\"'},\n",
       " {'abbreviation': 'R00809',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00809; R06130',\n",
       "   'MetaCyc: 2.4.1.167-RXN',\n",
       "   'Name: UDP-alpha-D-galactose:sucrose 6F-alpha-D-galactosyltransferase; UDP-galactose:sucrose 6F-alpha-D-galactosyltransferase; UDPgalactose:sucrose 6F-alpha-D-galactosyltransferase; UDPgalactose:sucrose 6fru-alpha-galactosyltransferase; sucrose 6F-alpha-galactosyltransferase; sucrose 6F-alpha-galactotransferase; uridine diphosphogalactose-sucrose 6F-alpha-galactosyltransferase'],\n",
       "  'code': '(1) cpd00043[0] + (1) cpd00076[0] <=> (1) cpd00014[0] + (1) cpd02400[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00043;cpd00076;cpd02400',\n",
       "  'definition': '(1) UDP-galactose[0] + (1) Sucrose[0] <=> (1) UDP[0] + (1) Planteose[0]',\n",
       "  'deltag': -5.42,\n",
       "  'deltagerr': 1.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.167'],\n",
       "  'equation': '(1) cpd00043[0] + (1) cpd00076[0] <=> (1) cpd00014[0] + (1) cpd02400[0]',\n",
       "  'id': 'rxn00581',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-galactose:sucrose 6F-alpha-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00043:0:0:\"UDP-galactose\";-1:cpd00076:0:0:\"Sucrose\";1:cpd00014:0:0:\"UDP\";1:cpd02400:0:0:\"Planteose\"'},\n",
       " {'abbreviation': 'R00810',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ISOMS',\n",
       "   'KEGG: R00810; R06217',\n",
       "   'MetaCyc: ISOMALTULOSE-SYNTHASE-RXN',\n",
       "   'iYO844: ISOMS',\n",
       "   'Name: Sucrose glucosylmutase; isomaltulose synthase; isomaltulose synthetase; sucrose alpha-glucosyltransferase; sucrose glucosylmutase; trehalulose synthase'],\n",
       "  'code': '(1) cpd00076[0] <=> (1) cpd01200[0]',\n",
       "  'compound_ids': 'cpd00076;cpd01200',\n",
       "  'definition': '(1) Sucrose[0] <=> (1) Palatinose[0]',\n",
       "  'deltag': -4.82,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.99.11'],\n",
       "  'equation': '(1) cpd00076[0] <=> (1) cpd01200[0]',\n",
       "  'id': 'rxn00582',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Sucrose glucosylmutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00076:0:0:\"Sucrose\";1:cpd01200:0:0:\"Palatinose\"'},\n",
       " {'abbreviation': 'R00812',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00812',\n",
       "   'MetaCyc: 1.3.1.19-RXN',\n",
       "   'Name: cis-1,2-dihydrobenzene-1,2-diol dehydrogenase; cis-1,2-dihydrobenzene-1,2-diol:NAD+ oxidoreductase; cis-1,2-dihydrocyclohexa-3,5-diene (nicotinamide adenine dinucleotide) oxidoreductase,; cis-benzene glycol dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd02529[0] <=> (1) cpd00004[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00077;cpd02529',\n",
       "  'definition': '(1) NAD[0] + (1) cis-Benzeneglycol[0] <=> (1) NADH[0] + (1) H+[0] + (1) Catechol[0]',\n",
       "  'deltag': -19.68,\n",
       "  'deltagerr': 2.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.1.19'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd02529[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00583',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'cis-1,2-dihydrobenzene-1,2-diol:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5450 (benzene degradation)',\n",
       "   'KEGG: rn00361 (Chlorocyclohexane and chlorobenzene degradation); rn00362 (Benzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd02529:0:0:\"cis-Benzeneglycol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00813',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: BZDIOLDH',\n",
       "   'KEGG: R00813',\n",
       "   'MetaCyc: 1.3.1.25-RXN',\n",
       "   'iAbaylyiv4: BENZOATE-CIS-DIOL-DEHYDROGENASE-RXN',\n",
       "   'iJN746: BZDIOLDH',\n",
       "   'Name: (1R,6R)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase (decarboxylating); (1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase (decarboxylating); 1,2-dihydrodihydroxybenzoate dehydrogenase; 1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate dehydrogenase; 2-hydro-1,2-dihydroxybenzoate dehydrogenase; 3,5-cyclohexadiene-1,2-diol-1-carboxylate dehydrogenase; 3,5-cyclohexadiene-1,2-diol-1-carboxylic acid dehydrogenase; DHBDH; cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate dehydrogenase; cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate:NAD+; cis-1,2-dihydroxycyclohexa-3,5-diene-1-carboxylate:NAD+ oxidoreductase; dihydrodihydroxybenzoate dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd03762[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00077;cpd03762',\n",
       "  'definition': '(1) NAD[0] + (1) cis-1,2-dihydroxybenzoate[0] => (1) NADH[0] + (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -22.07,\n",
       "  'deltagerr': 2.56,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.3.1.25'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd03762[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00584',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(1R,6S)-1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Benzoate-Degradation (Benzoate Degradation); Degradation (Degradation/Utilization/Assimilation); MANDELATE-DEG (Mandelate Degradation); PWY-2503 (benzoate degradation I (aerobic)); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5178 (toluene degradation IV (aerobic) (via catechol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-5430 (<i>meta</i> cleavage pathway of aromatic compounds); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-6957 (mandelate degradation to acetyl-CoA); TOLUENE-DEG (Toluene Degradation)',\n",
       "   'KEGG: rn00362 (Benzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd03762:0:0:\"cis-1,2-dihydroxybenzoate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00814',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.3.1.20-RXN',\n",
       "   'ChlamyCyc: 1.3.1.20-RXN',\n",
       "   'KEGG: R00814',\n",
       "   'MetaCyc: 1.3.1.20-RXN',\n",
       "   'Name: dihydrodiol dehydrogenase; trans-1,2-dihydrobenzene-1,2-diol dehydrogenase; trans-1,2-dihydrobenzene-1,2-diol:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd02594[0] <=> (1) cpd00005[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00077;cpd02594',\n",
       "  'definition': '(1) NADP[0] + (1) trans-1,2-Dihydrobenzene-1,2-diol[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Catechol[0]',\n",
       "  'deltag': -19.68,\n",
       "  'deltagerr': 2.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.1.20'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd02594[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00585',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'trans-1,2-dihydrobenzene-1,2-diol:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd02594:0:0:\"trans-1,2-Dihydrobenzene-1,2-diol\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00815',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00815',\n",
       "   'MetaCyc: PHENOL-2-MONOOXYGENASE-RXN',\n",
       "   'Name: phenol 2-monooxygenase; phenol hydroxylase; phenol o-hydroxylase; phenol,NADPH:oxogen oxidoreductase (2-hydroxylating); phenol,NADPH:oxygen oxidoreductase (2-hydroxylating)'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00127[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00067;cpd00077;cpd00127',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) H+[0] + (1) Phenol[0] => (1) H2O[0] + (1) NADP[0] + (1) Catechol[0]',\n",
       "  'deltag': -105.97,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.7'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00067[0] + (1) cpd00127[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00586',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'phenol,NADPH:oxogen oxidoreductase (2-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-5418 (phenol degradation I (aerobic)); Phenol-Degradation (Phenol Degradation); Phenolic-Compounds-Degradation (Phenolic Compound Degradation)',\n",
       "   'KEGG: rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00067:0:0:\"H+\";-1:cpd00127:0:0:\"Phenol\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00816',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CAT23DOX',\n",
       "   'KEGG: R00816',\n",
       "   'MetaCyc: CATECHOL-23-DIOXYGENASE-RXN',\n",
       "   'TS_Athaliana: rxn604_c',\n",
       "   'iJN746: CAT23DOX',\n",
       "   'Name: 2,3-pyrocatechase; Catechol:oxygen 2,3-oxidoreductase (decyclizing); Catechol:oxygen 2,3-oxidoreductase(decyclizing); catechol 2,3-dioxygenase; catechol 2,3-oxygenase; catechol oxygenase; catechol:oxygen 2,3-oxidoreductase (decyclizing); metapyrocatechase; pyrocatechol 2,3-dioxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00077[0] <=> (1) cpd00518[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00067;cpd00077;cpd00518',\n",
       "  'definition': '(1) O2[0] + (1) Catechol[0] => (1) H+[0] + (1) 2-Hydroxymuconic semialdehyde[0]',\n",
       "  'deltag': -62.22,\n",
       "  'deltagerr': 1.39,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.2'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00077[0] => (1) cpd00067[0] + (1) cpd00518[0]',\n",
       "  'id': 'rxn00587',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Catechol:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Benzoate-Degradation (Benzoate Degradation); Catechol-Degradation (Catechol Degradation); Degradation (Degradation/Utilization/Assimilation); MANDELATE-DEG (Mandelate Degradation); P183-PWY (catechol degradation to 2-hydroxypentadienoate I); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5178 (toluene degradation IV (aerobic) (via catechol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-5415 (catechol degradation I (<I>meta</I>-cleavage pathway)); PWY-5419 (catechol degradation to 2-hydroxypentadienoate II); PWY-5420 (catechol degradation II (<I>meta</I>-cleavage pathway)); PWY-5430 (<i>meta</i> cleavage pathway of aromatic compounds); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-6956 (naphthalene degradation to acetyl-CoA); PWY-6957 (mandelate degradation to acetyl-CoA); TOLUENE-DEG (Toluene Degradation)',\n",
       "   'KEGG: rn00362 (Benzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00077:0:0:\"Catechol\";1:cpd00067:0:0:\"H+\";1:cpd00518:0:0:\"2-Hydroxymuconic semialdehyde\"'},\n",
       " {'abbreviation': 'R00817',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CATA; CATDOX',\n",
       "   'KEGG: R00817',\n",
       "   'MetaCyc: CATECHOL-12-DIOXYGENASE-RXN',\n",
       "   'iAbaylyiv4: CATECHOL-1,2-DIOXYGENASE-RXN',\n",
       "   'iJN746: CATDOX',\n",
       "   'Name: 1,2-pyrocatechase; CD I; CD II; Catechol:oxygen 1,2-oxidoreductase(decyclizing); catechase; catechol 1,2-dioxygenase; catechol 1,2-oxygenase; catechol dioxygenase; catechol-oxygen 1,2-oxidoreductase; catechol:oxygen 1,2-oxidoreductase; pyrocatechase; pyrocatechol 1,2-dioxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00077[0] <=> (1) cpd01630[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00067;cpd00077;cpd01630',\n",
       "  'definition': '(1) O2[0] + (1) Catechol[0] => (2) H+[0] + (1) cis,cis-Muconate[0]',\n",
       "  'deltag': -73.85,\n",
       "  'deltagerr': 1.47,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.1'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00077[0] => (2) cpd00067[0] + (1) cpd01630[0]',\n",
       "  'id': 'rxn00588',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Catechol:oxygen 1,2-oxidoreductase(decyclizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); CATECHOL-ORTHO-CLEAVAGE-PWY (catechol degradation to &beta;-ketoadipate); Catechol-Degradation (Catechol Degradation); Degradation (Degradation/Utilization/Assimilation); MANDELATE-DEG (Mandelate Degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5417 (catechol degradation III (ortho-cleavage pathway)); PWY-5431 (aromatic compounds degradation via &beta;-ketoadipate); PWY-6182 (superpathway of salicylate degradation); PWY-6957 (mandelate degradation to acetyl-CoA)',\n",
       "   'KEGG: rn00362 (Benzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00077:0:0:\"Catechol\";2:cpd00067:0:0:\"H+\";1:cpd01630:0:0:\"cis,cis-Muconate\"'},\n",
       " {'abbreviation': 'R00818',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00818_c',\n",
       "   'ChlamyCyc: SALICYLATE-1-MONOOXYGENASE-RXN',\n",
       "   'KEGG: R00818',\n",
       "   'MetaCyc: SALICYLATE-1-MONOOXYGENASE-RXN',\n",
       "   'iAbaylyiv4: SALICYLATE-1-MONOOXYGENASE-RXN',\n",
       "   'Name: R_salicylate_NADH_oxygen_oxidoreductase_1_hydroxylating__decarboxylating_; salicylate 1-hydroxylase; salicylate 1-monooxygenase; salicylate hydroxylase; salicylate hydroxylase (decarboxylating); salicylate monooxygenase; salicylate,NADH:oxygen oxidoreductase (1-hydroxylating,; salicylate,NADH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00599[0] <=> (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00007;cpd00011;cpd00067;cpd00077;cpd00599',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (2) H+[0] + (1) SALC[0] => (1) H2O[0] + (1) NAD[0] + (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -108.63,\n",
       "  'deltagerr': 1.19,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.1'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (2) cpd00067[0] + (1) cpd00599[0] => (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00589',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31404',\n",
       "  'name': 'salicylate,NADH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-6182 (superpathway of salicylate degradation); PWY-6183 (salicylate degradation I); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-6956 (naphthalene degradation to acetyl-CoA); Salicylate-Degradation (Salicylate Degradation)',\n",
       "   'KEGG: rn00621 (Dioxin degradation); rn00624 (Polycyclic aromatic hydrocarbon degradation); rn00626 (Naphthalene degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-1:cpd00599:0:0:\"SALC\";1:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'decarboxylating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Benzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating,; decarboxylating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00153[0] <=> (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00007;cpd00011;cpd00077;cpd00153',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (1) Benzoate[0] <=> (1) NAD[0] + (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -120.33,\n",
       "  'deltagerr': 1.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.12.10'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00153[0] <=> (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00590',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Benzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating,',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-1:cpd00153:0:0:\"Benzoate\";1:cpd00003:0:0:\"NAD\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': '(decarboxylating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: (decarboxylating); 1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd02821[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00077;cpd02821',\n",
       "  'definition': '(1) NAD[0] + (1) 1,2-Dihydroxycyclohexa-3,5-diene-1-carboxylate[0] <=> (1) NADH[0] + (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -22.07,\n",
       "  'deltagerr': 2.56,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.1.25'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd02821[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00591',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '1,6-dihydroxycyclohexa-2,4-diene-1-carboxylate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd02821:0:0:\"1,2-Dihydroxycyclohexa-3,5-diene-1-carboxylate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00821',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00821',\n",
       "   'MetaCyc: O-PYROCATECHUATE-DECARBOXYLASE-RXN',\n",
       "   'Name: 2,3-Dihydroxybenzoate carboxy-lyase; 2,3-dihydroxybenzoate carboxy-lyase; 2,3-dihydroxybenzoate carboxy-lyase (catechol-forming); o-pyrocatechuate decarboxylase'],\n",
       "  'code': '(1) cpd00168[0] <=> (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00077;cpd00168',\n",
       "  'definition': '(1) H+[0] + (1) 2,3-Dihydroxybenzoate[0] => (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -2.66,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.46'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00168[0] => (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00592',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2,3-dihydroxybenzoate carboxy-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation); rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00168:0:0:\"2,3-Dihydroxybenzoate\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00822',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00822',\n",
       "   'MetaCyc: PROTOCATECHUATE-DECARBOXYLASE-RXN',\n",
       "   'Name: 3,4-dihydrobenzoate decarboxylase; 3,4-dihydroxybenzoate carboxy-lyase (catechol-forming); Protocatechuate carboxy-lyase; protocatechuate carboxy-lyase; protocatechuate decarboxylase'],\n",
       "  'code': '(1) cpd00197[0] <=> (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00077;cpd00197',\n",
       "  'definition': '(1) H+[0] + (1) Protocatechuate[0] => (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -2.66,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.63'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00197[0] => (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00593',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3,4-dihydroxybenzoate carboxy-lyase (catechol-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00197:0:0:\"Protocatechuate\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00823',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00823',\n",
       "   'MetaCyc: 1.14.12.1-RXN',\n",
       "   'Name: Anthranilate,NADH:oxygen oxidoreductase (1,2-hydroxylating,; Anthranilate,NADH:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating); anthranilate 1,2-dioxygenase (deaminating, decarboxylating); anthranilate hydroxylase; anthranilate,NAD(P)H:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating); anthranilic acid hydroxylase; anthranilic hydroxylase; deaminating, decarboxylating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00093[0] <=> (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00007;cpd00011;cpd00013;cpd00067;cpd00077;cpd00093',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (3) H+[0] + (1) Anthranilate[0] => (1) NAD[0] + (1) CO2[0] + (1) NH3[0] + (1) Catechol[0]',\n",
       "  'deltag': -113.59,\n",
       "  'deltagerr': 1.3,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.12.1'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (3) cpd00067[0] + (1) cpd00093[0] => (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00594',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Anthranilate,NADH:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 2-Aminobenzoate-Degradation (2-Aminobenzoate Degradation); 2-Nirobenzoate-Degradation (2-Nitrobenzoate Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); AUXINS-DEGRADATION (Indole-3-acetate Degradation); Degradation (Degradation/Utilization/Assimilation); HORMONE-DEG (Hormone Degradation); Nitroaromatic-Degradation (Nitroaromatic Compound Degradation); Nitrobenzoate-Degradation (Nitrobenzoate Degradation); PLANT-HORMONE-DEG (Plant Hormone Degradation); PWY-2421 (indole-3-acetate degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5648 (2-nitrobenzoate degradation II); PWY-6079 (anthranilate degradation I (aerobic)); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate)',\n",
       "   'KEGG: rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-3:cpd00067:0:0:\"H+\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00003:0:0:\"NAD\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00825',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00825',\n",
       "   'MetaCyc: 1.14.12.1-RXN',\n",
       "   'iAbaylyiv4: 1.14.12.1-RXN',\n",
       "   'Name: Anthranilate,NADPH:oxygen oxidoreductase (1,2-hydroxylating,; Anthranilate,NADPH:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating); anthranilate 1,2-dioxygenase (deaminating, decarboxylating); anthranilate hydroxylase; anthranilate,NAD(P)H:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating); anthranilate,NADPH:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating); anthranilic acid hydroxylase; anthranilic hydroxylase; deaminating, decarboxylating)'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00093[0] <=> (1) cpd00006[0] + (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00007;cpd00011;cpd00013;cpd00067;cpd00077;cpd00093',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (3) H+[0] + (1) Anthranilate[0] => (1) NADP[0] + (1) CO2[0] + (1) NH3[0] + (1) Catechol[0]',\n",
       "  'deltag': -113.59,\n",
       "  'deltagerr': 1.3,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.12.1'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (3) cpd00067[0] + (1) cpd00093[0] => (1) cpd00006[0] + (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00595',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Anthranilate,NADPH:oxygen oxidoreductase (1,2-hydroxylating, deaminating, decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 2-Aminobenzoate-Degradation (2-Aminobenzoate Degradation); 2-Nirobenzoate-Degradation (2-Nitrobenzoate Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); AUXINS-DEGRADATION (Indole-3-acetate Degradation); Degradation (Degradation/Utilization/Assimilation); HORMONE-DEG (Hormone Degradation); Nitroaromatic-Degradation (Nitroaromatic Compound Degradation); Nitrobenzoate-Degradation (Nitrobenzoate Degradation); PLANT-HORMONE-DEG (Plant Hormone Degradation); PWY-2421 (indole-3-acetate degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5648 (2-nitrobenzoate degradation II); PWY-6079 (anthranilate degradation I (aerobic)); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate)',\n",
       "   'KEGG: rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-3:cpd00067:0:0:\"H+\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00006:0:0:\"NADP\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00827',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00827; R07330',\n",
       "   'Name: S-Adenosyl-L-methionine:catechol O-methyltransferase; S-adenosyl-L-methionine:catechol O-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00077[0] <=> (1) cpd00019[0] + (1) cpd01063[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00077;cpd01063',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Catechol[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) Guaiacol[0]',\n",
       "  'deltag': -4.36,\n",
       "  'deltagerr': 5.01,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.6'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00077[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd01063[0]',\n",
       "  'id': 'rxn00596',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:catechol O-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00077:0:0:\"Catechol\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd01063:0:0:\"Guaiacol\"'},\n",
       " {'abbreviation': '2-NITROPHENOL-2-MONOOXYGENASE-RXN.c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['MetaCyc: 2-NITROPHENOL-2-MONOOXYGENASE-RXN',\n",
       "   'Name: 2-nitrophenol 2-monooxygenase; 2-nitrophenol oxygenase; 2-nitrophenol,NADPH:oxygen 2-oxidoreductase (2-hydroxylating,; 2-nitrophenol,NADPH:oxygen 2-oxidoreductase (2-hydroxylating, nitrite-forming); nitrite-forming); nitrophenol oxygenase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd01361[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00075[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00067;cpd00075;cpd00077;cpd01361',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (4) H+[0] + (1) 2-Nitrophenol[0] <=> (1) H2O[0] + (1) NADP[0] + (1) Nitrite[0] + (1) Catechol[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.13.31'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (4) cpd00067[0] + (1) cpd01361[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00075[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00597',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'nitrophenol oxygenase',\n",
       "  'notes': ['HB', 'GCP', 'EQC'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); Nitroaromatic-Degradation (Nitroaromatic Compound Degradation); Nitrophenol-Degradation (Nitrophenol Degradation); PWY-5636 (2-nitrophenol degradation); Phenolic-Compounds-Degradation (Phenolic Compound Degradation); o-Nitrophenol-Degradation (2-Nitrophenol Degradation)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'CI:-2',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-4:cpd00067:0:0:\"H+\";-1:cpd01361:0:0:\"2-Nitrophenol\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00075:0:0:\"Nitrite\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R00829',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00829_c',\n",
       "   'BiGG: 3OXACOA; 3OXCOAT',\n",
       "   'EcoCyc: RXN-3641',\n",
       "   'KEGG: R00829',\n",
       "   'MetaCyc: RXN-3641',\n",
       "   'Name: 3-oxoadipyl-CoA thiolase; Succinyl-CoA:acetyl-CoA C-acyltransferase; beta-ketoadipyl coa thiolase; beta-ketoadipyl-CoA thiolase; succinyl-CoA:acetyl-CoA C-succinyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00078[0] <=> (1) cpd00010[0] + (1) cpd01507[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00078;cpd01507',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Succinyl-CoA[0] <= (1) CoA[0] + (1) 3-Oxoadipyl-CoA[0]',\n",
       "  'deltag': 6.74,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.1.-', '2.3.1.16', '2.3.1.174'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00078[0] <= (1) cpd00010[0] + (1) cpd01507[0]',\n",
       "  'id': 'rxn00598',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Succinyl-CoA:acetyl-CoA C-acyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Benzoyl-CoA-Degradation (Benzoyl-CoA Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Catechol-Degradation (Catechol Degradation); Degradation (Degradation/Utilization/Assimilation); MANDELATE-DEG (Mandelate Degradation); PWY-1361 (benzoyl-CoA degradation I (aerobic)); PWY-2361 (3-oxoadipate degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5181 (toluene degradation III (aerobic) (via <i>p</i>-cresol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-5417 (catechol degradation III (ortho-cleavage pathway)); PWY-5431 (aromatic compounds degradation via &beta;-ketoadipate); PWY-6071 (superpathway of phenylethylamine degradation); PWY-6182 (superpathway of salicylate degradation); PWY-6957 (mandelate degradation to acetyl-CoA); PWY0-321 (phenylacetate degradation I (aerobic)); Phenolic-Compounds-Degradation (Phenolic Compound Degradation); Phenylacetate-Degradation (Phenylacetate Degradation); TOLUENE-DEG (Toluene Degradation)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism); rn00362 (Benzoate degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00078:0:0:\"Succinyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd01507:0:0:\"3-Oxoadipyl-CoA\"'},\n",
       " {'abbreviation': 'R00830',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ALASm; HEM1',\n",
       "   'KEGG: R00830',\n",
       "   'MetaCyc: 5-AMINOLEVULINIC-ACID-SYNTHASE-RXN',\n",
       "   'iIN800: HEM1',\n",
       "   'iMM904: ALASm',\n",
       "   'Name: 5-aminolevulinate synthase; 5-aminolevulinate synthetase; 5-aminolevulinic acid synthetase; ALA synthase; ALA synthetase; ALAS; alpha-aminolevulinic acid synthase; aminolevulinate synthase; aminolevulinate synthetase; aminolevulinic acid synthase; aminolevulinic acid synthetase; aminolevulinic synthetase; delta-aminolevulinate synthase; delta-aminolevulinate synthetase; delta-aminolevulinic acid synthase; delta-aminolevulinic acid synthetase; delta-aminolevulinic synthetase; succinyl-CoA:glycine C-succinyltransferase (decarboxylating)'],\n",
       "  'code': '(1) cpd00033[0] + (1) cpd00078[0] <=> (1) cpd00010[0] + (1) cpd00011[0] + (1) cpd00338[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00011;cpd00033;cpd00067;cpd00078;cpd00338',\n",
       "  'definition': '(1) Glycine[0] + (1) H+[0] + (1) Succinyl-CoA[0] => (1) CoA[0] + (1) CO2[0] + (1) 5-Aminolevulinate[0]',\n",
       "  'deltag': 3.81,\n",
       "  'deltagerr': 1.01,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.37'],\n",
       "  'equation': '(1) cpd00033[0] + (1) cpd00067[0] + (1) cpd00078[0] => (1) cpd00010[0] + (1) cpd00011[0] + (1) cpd00338[0]',\n",
       "  'id': 'rxn00599',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30135',\n",
       "  'name': 'succinyl-CoA:glycine C-succinyltransferase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Chlorophyll-Biosynthesis (Chlorophyll Biosynthesis); Chlorophyll-a-Biosynthesis (Chlorophyll <i>a</i> Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); HEME-SYN (Heme Biosynthesis); Heme-b-Biosynthesis (Heme <i>b</i> Biosynthesis); PWY-5189 (tetrapyrrole biosynthesis II (from glycine)); PWY-5529 (superpathway of bacteriochlorophyll <i>a</i> biosynthesis); PWY-5920 (superpathway of <i>b</i> heme biosynthesis from glycine); PWY-7536 (2-amino-3-hydroxycyclopent-2-enone biosynthesis); Porphyrin-Compounds-Biosynthesis (Porphyrin Compound Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Tetrapyrrole-Biosynthesis (Tetrapyrrole Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00860 (Porphyrin and chlorophyll metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00033:0:0:\"Glycine\";-1:cpd00067:0:0:\"H+\";-1:cpd00078:0:0:\"Succinyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00011:0:0:\"CO2\";1:cpd00338:0:0:\"5-Aminolevulinate\"'},\n",
       " {'abbreviation': 'Succinyl-CoA:glycine C-succinyl-transferase(decarboxylating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Succinyl-CoA:glycine C-succinyl-transferase(decarboxylating)'],\n",
       "  'code': '(1) cpd00033[0] + (1) cpd00078[0] <=> (1) cpd00010[0] + (1) cpd03283[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00033;cpd00078;cpd03283',\n",
       "  'definition': '(1) Glycine[0] + (1) Succinyl-CoA[0] <=> (1) CoA[0] + (1) 2-Amino-3-oxoadipate[0]',\n",
       "  'deltag': 7.97,\n",
       "  'deltagerr': 0.74,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.37'],\n",
       "  'equation': '(1) cpd00033[0] + (1) cpd00078[0] <=> (1) cpd00010[0] + (1) cpd03283[0]',\n",
       "  'id': 'rxn00600',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Succinyl-CoA:glycine C-succinyl-transferase(decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00033:0:0:\"Glycine\";-1:cpd00078:0:0:\"Succinyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd03283:0:0:\"2-Amino-3-oxoadipate\"'},\n",
       " {'abbreviation': 'R00832',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AST',\n",
       "   'EcoCyc: ARGININE-N-SUCCINYLTRANSFERASE-RXN',\n",
       "   'KEGG: R00832',\n",
       "   'MetaCyc: ARGININE-N-SUCCINYLTRANSFERASE-RXN',\n",
       "   'RiceCyc: ARGININE-N-SUCCINYLTRANSFERASE-RXN',\n",
       "   'Name: AOST; Arginine succinyltransferase; AstA; arginine N-succinyltransferase; arginine and ornithine N2-succinyltransferase; arginine n-succinyltransferase; arginine succinyltransferase; succinyl-CoA:L-arginine 2-N-succinyltransferase; succinyl-CoA:L-arginine N2-succinyltransferase'],\n",
       "  'code': '(1) cpd00051[0] + (1) cpd00078[0] <=> (1) cpd00010[0] + (1) cpd02103[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00051;cpd00067;cpd00078;cpd02103',\n",
       "  'definition': '(1) L-Arginine[0] + (1) Succinyl-CoA[0] => (1) CoA[0] + (1) H+[0] + (1) N2-Succinyl-L-arginine[0]',\n",
       "  'deltag': -0.5,\n",
       "  'deltagerr': 1.4,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.109'],\n",
       "  'equation': '(1) cpd00051[0] + (1) cpd00078[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd02103[0]',\n",
       "  'id': 'rxn00601',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32653;rxn32654;rxn33032',\n",
       "  'name': 'succinyl-CoA:L-arginine N2-succinyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGININE-DEG (L-arginine Degradation); AST-PWY (L-arginine degradation II (AST pathway)); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00051:0:0:\"L-Arginine\";-1:cpd00078:0:0:\"Succinyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd02103:0:0:\"N2-Succinyl-L-arginine\"'},\n",
       " {'abbreviation': 'R00833',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MMM; MMM2',\n",
       "   'EcoCyc: METHYLMALONYL-COA-MUT-RXN',\n",
       "   'KEGG: R00833',\n",
       "   'MetaCyc: METHYLMALONYL-COA-MUT-RXN',\n",
       "   'iAF1260: MMM2',\n",
       "   'Name: (R)-2-methyl-3-oxopropanoyl-CoA CoA-carbonylmutase [incorrect]; (R)-Methylmalonyl-CoA CoA-carbonylmutase; (R)-methylmalonyl-CoA CoA-carbonylmutase; (S)-methylmalonyl-CoA mutase; Methylmalonyl-CoA mutase; methylmalonyl coenzyme A carbonylmutase; methylmalonyl coenzyme A mutase; methylmalonyl-CoA CoA-carbonyl mutase; methylmalonyl-CoA mutase'],\n",
       "  'code': '(1) cpd00891[0] <=> (1) cpd00078[0]',\n",
       "  'compound_ids': 'cpd00078;cpd00891',\n",
       "  'definition': '(1) L-methylmalonyl-CoA[0] <=> (1) Succinyl-CoA[0]',\n",
       "  'deltag': -1.81,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.99.2'],\n",
       "  'equation': '(1) cpd00891[0] <=> (1) cpd00078[0]',\n",
       "  'id': 'rxn00602',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20202',\n",
       "  'name': '(R)-Methylmalonyl-CoA CoA-carbonylmutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 2-Oxobutanoate-Degradation (2-Oxobutanoate Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); GLUTAMATE-DEG (L-glutamate Degradation); METHIONINE-DEG (L-methionine Degradation); P108-PWY (pyruvate fermentation to propanoate I); PROPIONMET-PWY (propanoyl CoA degradation I); PWY-5088 (L-glutamate degradation VIII (to propanoate)); PWY-5130 (2-oxobutanoate degradation I); PWY-5328 (superpathway of L-methionine salvage and degradation); PWY-5743 (3-hydroxypropanoate cycle); PWY-5789 (3-hydroxypropanoate/4-hydroxybutanate cycle); PWY-6728 (methylaspartate cycle); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle); PWY-7384 (anaerobic energy metabolism (invertebrates, mitochondrial)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY-7854 (crotonyl-CoA/ethylmalonyl-CoA/hydroxybutyryl-CoA cycle (engineered)); PWY0-43 (conversion of succinate to propanoate); Propionate-Degradation (Propanoate Degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Propanoate-Fermentation (Pyruvate Fermentation to Propanoate); SUCC-DEG (Succinate Degradation)',\n",
       "   'KEGG: rn00280 (Valine, leucine and isoleucine degradation); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00640 (Propanoate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00891:0:0:\"L-methylmalonyl-CoA\";1:cpd00078:0:0:\"Succinyl-CoA\"'},\n",
       " {'abbreviation': 'R00834',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00834',\n",
       "   'MetaCyc: ALDOSE-6-PHOSPHATE-REDUCTASE-NADPH-RXN',\n",
       "   'iRS1597: R00834_c',\n",
       "   'Name: D-Sorbitol 6-phosphate:NADP+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00491[0] <=> (1) cpd00005[0] + (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00079;cpd00491',\n",
       "  'definition': '(1) NADP[0] + (1) D-glucitol 6-phosphate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': 3.02,\n",
       "  'deltagerr': 0.42,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.200'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00491[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00079[0]',\n",
       "  'id': 'rxn00603',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Sorbitol 6-phosphate:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00491:0:0:D-glucitol 6-phosphate;1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00835',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G6PDH; G6PDH2; G6PDH2er; G6PDH2r; G6PDHy; ZWF1',\n",
       "   'JM_Creinhardtii: JM_Cre_100; JM_Cre_101',\n",
       "   'JP_Creinhardtii_NMeth: R_G6PDH; R_G6PDHh',\n",
       "   'KEGG: R00835',\n",
       "   'MetaCyc: GLU6PDEHYDROG-RXN',\n",
       "   'Name: D-Glucose-6-phosphate:NADP+ 1-oxoreductase; D-glucose-6-phosphate:NADP+ 1-oxidoreductase; Glucose-6-phosphate 1-dehydrogenase; glucose 6-phosphate dehydrogenase; glucose6-phosphatedehydrogenase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00079[0] <=> (1) cpd00005[0] + (1) cpd00911[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00079;cpd00911',\n",
       "  'definition': '(1) NADP[0] + (1) D-glucose-6-phosphate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) 6-phospho-D-glucono-1-5-lactone[0]',\n",
       "  'deltag': -0.95,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.363', '1.1.1.49'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00079[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00911[0]',\n",
       "  'id': 'rxn00604',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30383;rxn34218',\n",
       "  'name': 'D-glucose-6-phosphate:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Carbohydrates-Degradation (Carbohydrate Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Entner-Duodoroff-Pathways (Entner-Duodoroff Pathways); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Formaldehyde-Oxidation (Formaldehyde Oxidation); GLYCOLYSIS-E-D (superpathway of glycolysis and the Entner-Doudoroff pathway); NAD-Metabolism (NAD Metabolism); OXIDATIVEPENT-PWY (pentose phosphate pathway (oxidative branch) I); P122-PWY (heterolactic fermentation); PENTOSE-P-PWY (pentose phosphate pathway); PWY-6901 (superpathway of glucose and xylose degradation); PWY-7245 (superpathway NAD/NADP-NADH/NADPH interconversion (yeast)); PWY-7268 (NAD/NADP-NADH/NADPH cytosolic interconversion (yeast)); PWY-8004 (Entner-Doudoroff pathway I); Pentose-Phosphate-Cycle (Pentose Phosphate Pathways); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); RUMP-PWY (formaldehyde oxidation I); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00911:0:0:\"6-phospho-D-glucono-1-5-lactone\"'},\n",
       " {'abbreviation': 'R00836',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: TPS1; TPS3; TRE6PS; TSL1',\n",
       "   'KEGG: R00836; R06043',\n",
       "   'MetaCyc: TREHALOSE6PSYN-RXN',\n",
       "   'iAF1260: TRE6PS',\n",
       "   'iIN800: TPS1; TPS3; TSL1',\n",
       "   'Name: UDP-glucose:D-glucose-6-phosphate 1-alpha-D-glucosyltransferase; alpha,alpha-trehalose-phosphate synthase (UDP-forming)'],\n",
       "  'code': '(1) cpd00026[0] + (1) cpd00079[0] <=> (1) cpd00014[0] + (1) cpd00523[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00026;cpd00079;cpd00523',\n",
       "  'definition': '(1) UDP-glucose[0] + (1) D-glucose-6-phosphate[0] <=> (1) UDP[0] + (1) Trehalose 6-phosphate[0]',\n",
       "  'deltag': -4.6,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.15'],\n",
       "  'equation': '(1) cpd00026[0] + (1) cpd00079[0] <=> (1) cpd00014[0] + (1) cpd00523[0]',\n",
       "  'id': 'rxn00605',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-glucose:D-glucose-6-phosphate 1-alpha-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Metabolic-Regulators (Metabolic Regulator Biosynthesis); Organic-solutes-Biosynthesis (Organic Solute Biosynthesis); TRESYN-PWY (trehalose biosynthesis I); Trehalose-biosynthesis (Trehalose Biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";-1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00014:0:0:\"UDP\";1:cpd00523:0:0:\"Trehalose 6-phosphate\"'},\n",
       " {'abbreviation': 'R00837',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: TRE6PH',\n",
       "   'KEGG: R00837; R06113',\n",
       "   'MetaCyc: TRE6PHYDRO-RXN',\n",
       "   'iAF1260: TRE6PH',\n",
       "   'iAG612: rbs_285',\n",
       "   'Name: alpha,alpha-Trehalose-6-phosphate phosphoglucohydrolase; trehalose-6-phosphate hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00523[0] <=> (1) cpd00027[0] + (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00079;cpd00523',\n",
       "  'definition': '(1) H2O[0] + (1) Trehalose 6-phosphate[0] => (1) D-Glucose[0] + (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': -2.31,\n",
       "  'deltagerr': 0.47,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.122', '3.2.1.93'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00523[0] => (1) cpd00027[0] + (1) cpd00079[0]',\n",
       "  'id': 'rxn00606',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'alpha,alpha-Trehalose-6-phosphate phosphoglucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); TREDEGLOW-PWY (trehalose degradation I (low osmolarity)); Trehalose-Degradation (Trehalose Degradation)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00523:0:0:\"Trehalose 6-phosphate\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00838',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MAL6PG',\n",
       "   'KEGG: R00838; R06115',\n",
       "   'MetaCyc: MALTOSE-6-PHOSPHATE-GLUCOSIDASE-RXN',\n",
       "   'iAG612: rbs_268',\n",
       "   'iYO844: MAL6PG',\n",
       "   \"Name: Maltose-6'-phosphate 6-phosphoglucohydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01919[0] <=> (1) cpd00027[0] + (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00079;cpd01919',\n",
       "  'definition': '(1) H2O[0] + (1) maltose-6-phosphate[0] => (1) D-Glucose[0] + (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': -4.65,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.122'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01919[0] => (1) cpd00027[0] + (1) cpd00079[0]',\n",
       "  'id': 'rxn00607',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"Maltose-6'-phosphate 6-phosphoglucohydrolase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01919:0:0:\"maltose-6-phosphate\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00839',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PGGH',\n",
       "   'KEGG: R00839',\n",
       "   'iAG612: rbs_201',\n",
       "   'iAO358: rll_70',\n",
       "   'iRR1083: PGGH',\n",
       "   'Name: 6-Phospho-beta-D-glucosyl-(1,4)-D-gluxose glucohydrolase; 6-phospho-beta-D-glucosyl-(1->4)-D-glucose 6-phosphoglucohydrolase; 6-phospho-beta-glucosidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02760[0] <=> (1) cpd00027[0] + (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00079;cpd02760',\n",
       "  'definition': '(1) H2O[0] + (1) cellobiose 6-phoshate[0] => (1) D-Glucose[0] + (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': -4.65,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.86'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02760[0] => (1) cpd00027[0] + (1) cpd00079[0]',\n",
       "  'id': 'rxn00608',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '6-phospho-beta-D-glucosyl-(1->4)-D-glucose 6-phosphoglucohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02760:0:0:\"cellobiose 6-phoshate\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'MI1PS',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00840_c',\n",
       "   'AraGEM: R_R00840_c',\n",
       "   'BiGG: MI1PS',\n",
       "   'Maize_C4GEM: R_R00840_c',\n",
       "   'iAF692: MI1PS',\n",
       "   'Name: 1L-myo-Inositol-1-phosphate lyase (isomerizing); myo-Inositol-1-phosphate synthase'],\n",
       "  'code': '(1) cpd00867[0] <=> (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00079;cpd00867',\n",
       "  'definition': '(1) Inositol 1-phosphate[0] <=> (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': 14.37,\n",
       "  'deltagerr': 1.78,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00867[0] <=> (1) cpd00079[0]',\n",
       "  'id': 'rxn00609',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'myo-Inositol-1-phosphate synthase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Secondary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00867:0:0:\"Inositol 1-phosphate\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00841',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G3PT; HOR2; RHR2',\n",
       "   'KEGG: R00841',\n",
       "   'MetaCyc: RXN-14965',\n",
       "   'iAF1260: G3PT',\n",
       "   'iIN800: HOR2; RHR2',\n",
       "   'Name: (DL)-glycerol-3-phosphatase 2; Glycerol-3-phosphate phosphohydrolase; glycerol-3-phosphatase; sn-glycerol-3-phosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00080[0] <=> (1) cpd00009[0] + (1) cpd00100[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00080;cpd00100',\n",
       "  'definition': '(1) H2O[0] + (1) Glycerol-3-phosphate[0] <=> (1) Phosphate[0] + (1) Glycerol[0]',\n",
       "  'deltag': -2.58,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00080[0] <=> (1) cpd00009[0] + (1) cpd00100[0]',\n",
       "  'id': 'rxn00610',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'sn-glycerol-3-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7385 (1,3-propanediol biosynthesis (engineered))',\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00100:0:0:\"Glycerol\"'},\n",
       " {'abbreviation': 'R00842',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.1.1.8-RXN; GLYC3PDEHYDROGBIOSYN-RXN',\n",
       "   'BiGG: G3PD1; G3PD1ir; G3PD1irm; GPD1; GPD2',\n",
       "   'BrachyCyc: 1.1.1.8-RXN; GLYC3PDEHYDROGBIOSYN-RXN',\n",
       "   'KEGG: R00842',\n",
       "   'MetaCyc: 1.1.1.8-RXN; GLYC3PDEHYDROGBIOSYN-RXN',\n",
       "   'Name: Glycerol-3-phosphate dehydrogenase [NAD+] 1; Glycerophosphate dehydrogenase (NAD); L-alpha-glycerol phosphate dehydrogenase; L-alpha-glycerophosphate dehydrogenase; L-glycerol phosphate dehydrogenase; L-glycerol-3-phosphate dehydrogenase (ambiguous); L-glycerophosphate dehydrogenase (ambiguous); NAD+-L-glycerol-3-phosphate dehydrogenase; NAD+-alpha-glycerophosphate dehydrogenase; NAD+-dependent glycerol phosphate dehydrogenase; NAD+-dependent glycerol-3-phosphate dehydrogenase; NAD+-linked glycerol 3-phosphate dehydrogenase; NAD-dependent glycerol phosphate dehydrogenase; NAD-dependent glycerol-3-phosphate dehydrogenase; NADH-dihydroxyacetone phosphate reductase; alpha-glycerol phosphate dehydrogenase (NAD+); alpha-glycerophosphate dehydrogenase (NAD+); glycerol 1-phosphate dehydrogenase; glycerol phosphate dehydrogenase (NAD+); glycerol-3-phosphate dehydrogenase (NAD); glycerol-3-phosphate dehydrogenase (NAD+); glycerol-3-phosphatedehydrogenase-NAD; glycerophosphate dehydrogenase (NAD+); hydroglycerophosphate dehydrogenase; sn-Glycerol-3-phosphate:NAD+ 2-oxidoreductase; sn-glycerol-3-phosphate:NAD+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00080[0] <=> (1) cpd00004[0] + (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00080;cpd00095',\n",
       "  'definition': '(1) NAD[0] + (1) Glycerol-3-phosphate[0] <=> (1) NADH[0] + (1) H+[0] + (1) Glycerone-phosphate[0]',\n",
       "  'deltag': 6.14,\n",
       "  'deltagerr': 0.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.8', '1.1.1.94'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00080[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00095[0]',\n",
       "  'id': 'rxn00611',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27460;rxn30381;rxn37873',\n",
       "  'name': 'sn-Glycerol-3-phosphate:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); CDP-diacylglycerol-Biosynthesis (CDP-diacylglycerol Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Glycolipids-Biosynthesis (Glycolipid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); OTHER-ENERGY (Other); PHOSLIPSYN-PWY (superpathway of phospholipid biosynthesis I (bacteria)); PHOSLIPSYN2-PWY (superpathway of phospholipid biosynthesis II (plants)); PWY-5667 (CDP-diacylglycerol biosynthesis I); PWY-5981 (CDP-diacylglycerol biosynthesis III); PWY-6118 (glycerol-3-phosphate shuttle); PWY-7385 (1,3-propanediol biosynthesis (engineered)); PWY-7411 (phosphatidate biosynthesis (yeast)); PWY-7902 (glucosylglycerol biosynthesis); PWY0-1319 (CDP-diacylglycerol biosynthesis II); PWY4FS-7 (phosphatidylglycerol biosynthesis I (plastidic)); PWY4FS-8 (phosphatidylglycerol biosynthesis II (non-plastidic)); PhosphatidylglycerolBiosynthesis (Phosphatidylglycerol Biosynthesis); Phospholipid-Biosynthesis (Phospholipid Biosynthesis)',\n",
       "   'KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R00844',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYC3PDEHYDROGBIOSYN-RXN',\n",
       "   'BiGG: G3PD2',\n",
       "   'BrachyCyc: GLYC3PDEHYDROGBIOSYN-RXN',\n",
       "   'KEGG: R00844',\n",
       "   'MetaCyc: GLYC3PDEHYDROGBIOSYN-RXN',\n",
       "   'Name: Glycerol-3-phosphate dehydrogenase (NAD(P)+)-(NADP); L-glycerol-3-phosphate:NAD(P) oxidoreductase; NAD(P)H-dependent dihydroxyacetone-phosphate reductase; glycerol 3-phosphate dehydrogenase (NADP); glycerol phosphate dehydrogenase (nicotinamide adenine dinucleotide (phosphate)); glycerol-3-phosphate dehydrogenase (NADP); glycerol-3-phosphate dehydrogenase (nad(p)+); glycerol-3-phosphate dehydrogenase [NAD(P)+]; glycerol-3-phosphate dehydrogenase [NAD(P)]; glycerol-3-phosphatedehydrogenase-NADP; sn-Glycerol-3-phosphate:NADP+ 2-oxidoreductase; sn-glycerol-3-phosphate:NAD(P)+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00080[0] <=> (1) cpd00005[0] + (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00080;cpd00095',\n",
       "  'definition': '(1) NADP[0] + (1) Glycerol-3-phosphate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Glycerone-phosphate[0]',\n",
       "  'deltag': 6.14,\n",
       "  'deltagerr': 0.2,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.94'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00080[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00095[0]',\n",
       "  'id': 'rxn00612',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27898;rxn31786',\n",
       "  'name': 'sn-Glycerol-3-phosphate:NADP+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CDP-diacylglycerol-Biosynthesis (CDP-diacylglycerol Biosynthesis); Glycolipids-Biosynthesis (Glycolipid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); PHOSLIPSYN-PWY (superpathway of phospholipid biosynthesis I (bacteria)); PHOSLIPSYN2-PWY (superpathway of phospholipid biosynthesis II (plants)); PWY-5667 (CDP-diacylglycerol biosynthesis I); PWY-5981 (CDP-diacylglycerol biosynthesis III); PWY-7902 (glucosylglycerol biosynthesis); PWY0-1319 (CDP-diacylglycerol biosynthesis II); PWY4FS-7 (phosphatidylglycerol biosynthesis I (plastidic)); PWY4FS-8 (phosphatidylglycerol biosynthesis II (non-plastidic)); PhosphatidylglycerolBiosynthesis (Phosphatidylglycerol Biosynthesis); Phospholipid-Biosynthesis (Phospholipid Biosynthesis)',\n",
       "   'KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R00845',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00845',\n",
       "   'MetaCyc: 1.1.1.177-RXN',\n",
       "   'iRS1563: R_R00845_c',\n",
       "   'Name: L-glycerol 3-phosphate:NADP+ oxidoreductase; NADP-specific glycerol 3-phosphate 1-dehydrogenase; NADPH-dependent glycerin-3-phosphate dehydrogenase; glycerin-3-phosphate dehydrogenase; glycerol phosphate (nicotinamide adenine dinucleotide phosphate) dehydrogenase; glycerol-3-phosphate 1-dehydrogenase (NADP+); sn-glycerol-3-phosphate:NADP+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00080[0] <=> (1) cpd00005[0] + (1) cpd00102[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00080;cpd00102',\n",
       "  'definition': '(1) NADP[0] + (1) Glycerol-3-phosphate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Glyceraldehyde3-phosphate[0]',\n",
       "  'deltag': 7.45,\n",
       "  'deltagerr': 0.23,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.177'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00080[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00102[0]',\n",
       "  'id': 'rxn00613',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'sn-glycerol-3-phosphate:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\"'},\n",
       " {'abbreviation': 'R00846',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_G3PO',\n",
       "   'KEGG: R00846',\n",
       "   'MetaCyc: GLYCEROL-3-PHOSPHATE-OXIDASE-RXN',\n",
       "   'Name: L-alpha-glycerol-3-phosphate oxidase; L-alpha-glycerophosphate oxidase; alpha-glycerophosphate oxidase; glycerol phosphate oxidase; glycerol-1-phosphate oxidase; glycerol-3-phosphate oxidase; sn-Glycerol-3-phosphate:oxygen 2-oxidoreductase; sn-glycerol-3-phosphate:oxygen 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00080[0] <=> (1) cpd00025[0] + (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00080;cpd00095',\n",
       "  'definition': '(1) O2[0] + (1) Glycerol-3-phosphate[0] => (1) H2O2[0] + (1) Glycerone-phosphate[0]',\n",
       "  'deltag': -25.58,\n",
       "  'deltagerr': 1.03,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.21'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00080[0] => (1) cpd00025[0] + (1) cpd00095[0]',\n",
       "  'id': 'rxn00614',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35164',\n",
       "  'name': 'sn-Glycerol-3-phosphate:oxygen 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00025:0:0:\"H2O2\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R00847',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCEROL-KIN-RXN',\n",
       "   'BiGG: GLYK; GLYKr; GUT1',\n",
       "   'BrachyCyc: GLYCEROL-KIN-RXN',\n",
       "   'KEGG: R00847',\n",
       "   'MetaCyc: GLYCEROL-KIN-RXN',\n",
       "   'Name: ATP:glycerol 3-phosphotransferase; ATP:glycerol-3-phosphotransferase; GK; Glycerol kinase; glyceric kinase; glycerokinase; glycerol kinase; glycerol kinase (phosphorylating); glycerol kinase (reversible); glycerolkinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00100[0] <=> (1) cpd00008[0] + (1) cpd00080[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00080;cpd00100',\n",
       "  'definition': '(1) ATP[0] + (1) Glycerol[0] => (1) ADP[0] + (1) H+[0] + (1) Glycerol-3-phosphate[0]',\n",
       "  'deltag': -3.58,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.30'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00100[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00080[0]',\n",
       "  'id': 'rxn00615',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:glycerol 3-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); Degradation (Degradation/Utilization/Assimilation); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); GLYCEROL-DEG (Glycerol Degradation); PWY-4261 (glycerol degradation I); PWY0-381 (glycerol and glycerophosphodiester degradation)',\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00100:0:0:\"Glycerol\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00080:0:0:\"Glycerol-3-phosphate\"'},\n",
       " {'abbreviation': 'R00848',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-6841',\n",
       "   'BiGG: G3PD; G3PDm',\n",
       "   'ChlamyCyc: RXN-6841',\n",
       "   'KEGG: R00848',\n",
       "   'PlantCyc: RXN-6841',\n",
       "   'Name: glycerol-3-phosphate dehydrogenase (FAD); sn-Glycerol-3-phosphate:(acceptor) 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00080[0] <=> (1) cpd00095[0] + (1) cpd00982[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00080;cpd00095;cpd00982',\n",
       "  'definition': '(1) FAD[0] + (1) Glycerol-3-phosphate[0] => (1) Glycerone-phosphate[0] + (1) FADH2[0]',\n",
       "  'deltag': 2.54,\n",
       "  'deltagerr': 0.75,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.5.3', '1.1.99.5'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00080[0] => (1) cpd00095[0] + (1) cpd00982[0]',\n",
       "  'id': 'rxn00616',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28327',\n",
       "  'name': 'sn-Glycerol-3-phosphate:(acceptor) 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00095:0:0:\"Glycerone-phosphate\";1:cpd00982:0:0:\"FADH2\"'},\n",
       " {'abbreviation': 'R00850',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00850',\n",
       "   'MetaCyc: 2.7.1.142-RXN',\n",
       "   'Name: sn-Glycerol-3-phosphate:D-glucose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00027[0] + (1) cpd00080[0] <=> (1) cpd00079[0] + (1) cpd00100[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00079;cpd00080;cpd00100',\n",
       "  'definition': '(1) D-Glucose[0] + (1) Glycerol-3-phosphate[0] <=> (1) D-glucose-6-phosphate[0] + (1) Glycerol[0]',\n",
       "  'deltag': -0.43,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.142'],\n",
       "  'equation': '(1) cpd00027[0] + (1) cpd00080[0] <=> (1) cpd00079[0] + (1) cpd00100[0]',\n",
       "  'id': 'rxn00617',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'sn-Glycerol-3-phosphate:D-glucose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00027:0:0:\"D-Glucose\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00100:0:0:\"Glycerol\"'},\n",
       " {'abbreviation': 'R00853',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00853',\n",
       "   'MetaCyc: 2.4.1.137-RXN',\n",
       "   \"Name: FPS; UDP-alpha-D-galactose:sn-glycerol-3-phosphate 2-alpha-D-galactosyltransferase; UDP-galactose, sn-3-glycerol phosphate:1->2'; UDP-galactose:sn-glycerol-3-phosphate 2-alpha-D-galactosyltransferase; UDP-galactose:sn-glycerol-3-phosphate-2-D-galactosyl transferase; UDPgalactose:sn-glycerol-3-phosphate 2-alpha-D-galactosyltransferase; floridoside phosphate synthase; floridoside phosphate synthetase; floridoside-phosphate synthase; galactosyltransferase; sn-glycerol-3-phosphate 2-alpha-galactosyltransferase\"],\n",
       "  'code': '(1) cpd00043[0] + (1) cpd00080[0] <=> (1) cpd00014[0] + (1) cpd02827[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00043;cpd00080;cpd02827',\n",
       "  'definition': '(1) UDP-galactose[0] + (1) Glycerol-3-phosphate[0] <=> (1) UDP[0] + (1) 2-(alpha-D-Galactosyl)-sn-glycerol 3-phosphate[0]',\n",
       "  'deltag': -2.51,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.137'],\n",
       "  'equation': '(1) cpd00043[0] + (1) cpd00080[0] <=> (1) cpd00014[0] + (1) cpd02827[0]',\n",
       "  'id': 'rxn00618',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-galactose:sn-glycerol-3-phosphate 2-alpha-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00043:0:0:\"UDP-galactose\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00014:0:0:\"UDP\";1:cpd02827:0:0:\"2-(alpha-D-Galactosyl)-sn-glycerol 3-phosphate\"'},\n",
       " {'abbreviation': 'R00854',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00854',\n",
       "   'MetaCyc: 2.4.1.96-RXN',\n",
       "   'Name: UDP-Gal:sn-glycero-3-phosphoric acid 1-alpha-galactosyl-transferase; UDP-alpha-D-galactose:sn-glycerol-3-phosphate 1-alpha-D-galactosyltransferase; UDP-galactose:sn-glycerol-3-phosphate 1-alpha-D-galactosyltransferase; UDPgalactose:sn-glycerol-3-phosphate 1-alpha-D-galactosyltransferase; UDPgalactose:sn-glycerol-3-phosphate alpha-D-galactosyltransferase; glycerol 3-phosphate 1alpha-galactosyltransferase; isofloridoside-phosphate synthase; sn-glycerol-3-phosphate 1-galactosyltransferase; uridine diphosphogalactose-glycerol phosphate galactosyltransferase'],\n",
       "  'code': '(1) cpd00043[0] + (1) cpd00080[0] <=> (1) cpd00014[0] + (1) cpd02863[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00043;cpd00080;cpd02863',\n",
       "  'definition': \"(1) UDP-galactose[0] + (1) Glycerol-3-phosphate[0] <=> (1) UDP[0] + (1) alpha-D-Galactosyl-(1,1')-sn-glycerol 3-phosphate[0]\",\n",
       "  'deltag': -4.0,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.96'],\n",
       "  'equation': '(1) cpd00043[0] + (1) cpd00080[0] <=> (1) cpd00014[0] + (1) cpd02863[0]',\n",
       "  'id': 'rxn00619',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-galactose:sn-glycerol-3-phosphate 1-alpha-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00043:0:0:\"UDP-galactose\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00014:0:0:\"UDP\";1:cpd02863:0:0:\"alpha-D-Galactosyl-(1,1\\')-sn-glycerol 3-phosphate\"'},\n",
       " {'abbreviation': 'R00855',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CDP-GLYCEROL-PYROPHOSPHATASE-RXN',\n",
       "   'CornCyc: CDP-GLYCEROL-PYROPHOSPHATASE-RXN',\n",
       "   'KEGG: R00855',\n",
       "   'MetaCyc: CDP-GLYCEROL-PYROPHOSPHATASE-RXN',\n",
       "   'PoplarCyc: CDP-GLYCEROL-PYROPHOSPHATASE-RXN',\n",
       "   'Name: CDP-glycerol diphosphatase; CDP-glycerol phosphoglycerohydrolase; CDP-glycerol pyrophosphatase; CDPglycerol phosphoglycerohydrolase; cytidine diphosphoglycerol pyrophosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00402[0] <=> (1) cpd00046[0] + (1) cpd00080[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00046;cpd00067;cpd00080;cpd00402',\n",
       "  'definition': '(1) H2O[0] + (1) CDPglycerol[0] => (1) CMP[0] + (2) H+[0] + (1) Glycerol-3-phosphate[0]',\n",
       "  'deltag': -8.52,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.16'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00402[0] => (1) cpd00046[0] + (2) cpd00067[0] + (1) cpd00080[0]',\n",
       "  'id': 'rxn00620',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CDPglycerol phosphoglycerohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00402:0:0:\"CDPglycerol\";1:cpd00046:0:0:\"CMP\";2:cpd00067:0:0:\"H+\";1:cpd00080:0:0:\"Glycerol-3-phosphate\"'},\n",
       " {'abbreviation': 'R00856',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: G3PCT',\n",
       "   'KEGG: R00856',\n",
       "   'MetaCyc: 2.7.7.39-RXN',\n",
       "   'iAO358: rll_412; rll_413',\n",
       "   'iSB619: G3PCT',\n",
       "   'Name: CDP-glycerol pyrophosphorylase; CTP:glycerol 3-phosphate cytidylyltransferase; CTP:sn-glycerol-3-phosphate cytidylyltransferase; Gro-PCT; cytidine diphosphate glycerol pyrophosphorylase; cytidine diphosphoglycerol pyrophosphorylase; glycerol-3-phosphate cytidylyltransferase'],\n",
       "  'code': '(1) cpd00052[0] + (1) cpd00080[0] <=> (1) cpd00012[0] + (1) cpd00402[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00052;cpd00080;cpd00402',\n",
       "  'definition': '(1) CTP[0] + (1) Glycerol-3-phosphate[0] <=> (1) PPi[0] + (1) CDPglycerol[0]',\n",
       "  'deltag': 0.52,\n",
       "  'deltagerr': 0.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.39'],\n",
       "  'equation': '(1) cpd00052[0] + (1) cpd00080[0] <=> (1) cpd00012[0] + (1) cpd00402[0]',\n",
       "  'id': 'rxn00621',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CTP:sn-glycerol-3-phosphate cytidylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); PWY-7815 (poly(ribitol phosphate) wall teichoic acid biosynthesis I (<i>B. subtilis</i>)); PWY-7816 (poly(ribitol phosphate) wall teichoic acid biosynthesis II (<i>S. aureus</i>)); PWY-7819 (poly(3-O-&beta;-D-glucopyranosyl-<i>N</i>-acetylgalactosamine 1-phosphate) wall teichoic acid biosynthesis); TEICHOICACID-PWY (poly(glycerol phosphate) wall teichoic acid biosynthesis); Teichoic-Acids-Biosynthesis (Teichoic Acid Biosynthesis)',\n",
       "   'KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00052:0:0:\"CTP\";-1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00402:0:0:\"CDPglycerol\"'},\n",
       " {'abbreviation': 'R00857',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00857',\n",
       "   'Name: Glycerophosphodiester glycerophosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02000[0] <=> (1) cpd00057[0] + (1) cpd00080[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00057;cpd00067;cpd00080;cpd02000',\n",
       "  'definition': '(1) H2O[0] + (1) Glycerophosphodiester[0] => (1) Alcohol[0] + (1) H+[0] + (1) Glycerol-3-phosphate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.4.46'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02000[0] => (1) cpd00057[0] + (1) cpd00067[0] + (1) cpd00080[0]',\n",
       "  'id': 'rxn00622',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glycerophosphodiester glycerophosphohydrolase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02000:0:0:\"Glycerophosphodiester\";1:cpd00057:0:0:\"Alcohol\";1:cpd00067:0:0:\"H+\";1:cpd00080:0:0:\"Glycerol-3-phosphate\"'},\n",
       " {'abbreviation': 'R00858',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ECM17; MET10; SULR; SULRi',\n",
       "   'BrachyCyc: SULFITE-REDUCT-RXN',\n",
       "   'ChlamyCyc: SULFITE-REDUCT-RXN',\n",
       "   'KEGG: R00858',\n",
       "   'MetaCyc: SULFITE-REDUCT-RXN',\n",
       "   'Name: H2S-NADP oxidoreductase; NADPH-dependent sulfite reductase; NADPH-sulfite reductase; Sulfite reductase [NADPH] subunit beta; hydrogen-sulfide:NADP+ oxidoreductase; sulfite (reduced nicotinamide adenine dinucleotide phosphate) reductase; sulfite reductase (NADPH); sulfite reductase (NADPH2); sulfite reductase (nadph); sulfitereductase-NADPH2'],\n",
       "  'code': '(3) cpd00001[0] + (3) cpd00006[0] + (1) cpd00239[0] <=> (3) cpd00005[0] + (1) cpd00081[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00067;cpd00081;cpd00239',\n",
       "  'definition': '(3) H2O[0] + (3) NADP[0] + (1) H2S[0] <= (3) NADPH[0] + (3) H+[0] + (1) Sulfite[0]',\n",
       "  'deltag': 29.45,\n",
       "  'deltagerr': 1.14,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.8.1.2', '1.8.2.2'],\n",
       "  'equation': '(3) cpd00001[0] + (3) cpd00006[0] + (1) cpd00239[0] <= (3) cpd00005[0] + (3) cpd00067[0] + (1) cpd00081[0]',\n",
       "  'id': 'rxn00623',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'hydrogen-sulfide:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Assimilatory-Sulfate-Reduction (Assimilatory Sulfate Reduction); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-6683 (assimilatory sulfate reduction III); PWY-821 (superpathway of sulfur amino acid biosynthesis (<i>Saccharomyces cerevisiae</i>)); SO4ASSIM-PWY (assimilatory sulfate reduction I); SULFATE-CYS-PWY (superpathway of sulfate assimilation and cysteine biosynthesis); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-3:cpd00001:0:0:\"H2O\";-3:cpd00006:0:0:\"NADP\";-1:cpd00239:0:0:\"H2S\";3:cpd00005:0:0:\"NADPH\";3:cpd00067:0:0:\"H+\";1:cpd00081:0:0:\"Sulfite\"'},\n",
       " {'abbreviation': 'R00863',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CYSSADS',\n",
       "   'KEGG: R00863',\n",
       "   'iAF1260: CYSSADS',\n",
       "   'iMA945: CYSSADS',\n",
       "   'Name: 3-Sulfino-L-alanine 4-carboxy-lyase; L-cysteine sulfinic acid desulfurase'],\n",
       "  'code': '(1) cpd00467[0] <=> (1) cpd00035[0] + (1) cpd06201[0]',\n",
       "  'compound_ids': 'cpd00035;cpd00067;cpd00467;cpd06201',\n",
       "  'definition': '(1) H+[0] + (1) 3-Sulfinoalanine[0] <=> (1) L-Alanine[0] + (1) Sulfur dioxide[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.1.12'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00467[0] <=> (1) cpd00035[0] + (1) cpd06201[0]',\n",
       "  'id': 'rxn00624',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-Sulfino-L-alanine 4-carboxy-lyase',\n",
       "  'notes': ['GCP', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00467:0:0:\"3-Sulfinoalanine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd06201:0:0:\"Sulfur dioxide\"'},\n",
       " {'abbreviation': 'R00864',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_TSULe',\n",
       "   'KEGG: R00864',\n",
       "   'iRS1563: R_R00864_c',\n",
       "   'Name: Thiosulfate formation, spontaneous'],\n",
       "  'code': '(1) cpd00074[0] + (1) cpd00081[0] <=> (1) cpd00268[0]',\n",
       "  'compound_ids': 'cpd00074;cpd00081;cpd00268',\n",
       "  'definition': '(1) S[0] + (1) Sulfite[0] <= (1) H2S2O3[0]',\n",
       "  'deltag': 10.11,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00074[0] + (1) cpd00081[0] <= (1) cpd00268[0]',\n",
       "  'id': 'rxn00625',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35517',\n",
       "  'name': 'R00864',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00074:0:0:\"S\";-1:cpd00081:0:0:\"Sulfite\";1:cpd00268:0:0:\"H2S2O3\"'},\n",
       " {'abbreviation': 'R00865',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00865',\n",
       "   'Name: D-Mannitol:ferricytochrome-c 2-oxidoreductase'],\n",
       "  'code': '(2) cpd00109[0] + (1) cpd00314[0] <=> (1) cpd00082[0] + (2) cpd00110[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00082;cpd00109;cpd00110;cpd00314',\n",
       "  'definition': '(2) Cytochrome c3+[0] + (1) D-Mannitol[0] <=> (2) H+[0] + (1) D-Fructose[0] + (2) Cytochrome c2+[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.2.2'],\n",
       "  'equation': '(2) cpd00109[0] + (1) cpd00314[0] <=> (2) cpd00067[0] + (1) cpd00082[0] + (2) cpd00110[0]',\n",
       "  'id': 'rxn00626',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Mannitol:ferricytochrome-c 2-oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00109:0:0:\"Cytochrome c3+\";-1:cpd00314:0:0:\"D-Mannitol\";2:cpd00067:0:0:\"H+\";1:cpd00082:0:0:\"D-Fructose\";2:cpd00110:0:0:\"Cytochrome c2+\"'},\n",
       " {'abbreviation': 'R00866',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00866', 'Name: ATP:D-fructose 1-phosphotransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00082[0] <=> (1) cpd00008[0] + (1) cpd00802[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00082;cpd00802',\n",
       "  'definition': '(1) ATP[0] + (1) D-Fructose[0] => (1) ADP[0] + (1) H+[0] + (1) D-fructose-1-phosphate[0]',\n",
       "  'deltag': -3.8,\n",
       "  'deltagerr': 0.45,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00082[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00802[0]',\n",
       "  'id': 'rxn00627',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:D-fructose 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00082:0:0:\"D-Fructose\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00802:0:0:\"D-fructose-1-phosphate\"'},\n",
       " {'abbreviation': 'R00868',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MNLDH',\n",
       "   'KEGG: R00868',\n",
       "   'iMO1053-PAO1: MNLDH',\n",
       "   'iMO1056: MNLDH',\n",
       "   'Name: D-Mannitol:NAD+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00314[0] <=> (1) cpd00004[0] + (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00082;cpd00314',\n",
       "  'definition': '(1) NAD[0] + (1) D-Mannitol[0] <=> (1) NADH[0] + (1) H+[0] + (1) D-Fructose[0]',\n",
       "  'deltag': 2.56,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.11', '1.1.1.67'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00314[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00082[0]',\n",
       "  'id': 'rxn00629',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Mannitol:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00314:0:0:\"D-Mannitol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00870',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00870', 'Name: D-Mannitol:NADP+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00314[0] <=> (1) cpd00005[0] + (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00082;cpd00314',\n",
       "  'definition': '(1) NADP[0] + (1) D-Mannitol[0] <=> (1) NADPH[0] + (1) H+[0] + (1) D-Fructose[0]',\n",
       "  'deltag': 2.56,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.138'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00314[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00082[0]',\n",
       "  'id': 'rxn00630',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Mannitol:NADP+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00314:0:0:\"D-Mannitol\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00871',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00871; R08651',\n",
       "   'Name: D-Fructose:NADP+ 5-oxidoreductase; D-fructose:NADP+ 5-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00082[0] <=> (1) cpd00005[0] + (1) cpd00234[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00082;cpd00234',\n",
       "  'definition': '(1) NADP[0] + (1) D-Fructose[0] <=> (1) NADPH[0] + (1) H+[0] + (1) 5-Dehydro-D-fructose[0]',\n",
       "  'deltag': 4.51,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.124'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00082[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00234[0]',\n",
       "  'id': 'rxn00631',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn15775',\n",
       "  'name': 'D-Fructose:NADP+ 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00082:0:0:\"D-Fructose\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00234:0:0:\"5-Dehydro-D-fructose\"'},\n",
       " {'abbreviation': 'R00872',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00872', 'Name: D-Fructose 2-phosphate phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02085[0] <=> (1) cpd00009[0] + (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00082;cpd02085',\n",
       "  'definition': '(1) H2O[0] + (1) beta-D-Fructose 2-phosphate[0] <=> (1) Phosphate[0] + (1) D-Fructose[0]',\n",
       "  'deltag': -8.78,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.-'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02085[0] <=> (1) cpd00009[0] + (1) cpd00082[0]',\n",
       "  'id': 'rxn00632',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Fructose 2-phosphate phosphatase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02085:0:0:\"beta-D-Fructose 2-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00874',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00874', 'Name: D-Glucose:D-fructose oxidoreductase'],\n",
       "  'code': '(1) cpd00027[0] + (1) cpd00082[0] <=> (1) cpd00170[0] + (1) cpd00588[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00082;cpd00170;cpd00588',\n",
       "  'definition': '(1) D-Glucose[0] + (1) D-Fructose[0] <=> (1) Gluconolactone[0] + (1) Sorbitol[0]',\n",
       "  'deltag': -3.98,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.99.28'],\n",
       "  'equation': '(1) cpd00027[0] + (1) cpd00082[0] <=> (1) cpd00170[0] + (1) cpd00588[0]',\n",
       "  'id': 'rxn00633',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose:D-fructose oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00027:0:0:\"D-Glucose\";-1:cpd00082:0:0:\"D-Fructose\";1:cpd00170:0:0:\"Gluconolactone\";1:cpd00588:0:0:\"Sorbitol\"'},\n",
       " {'abbreviation': 'R00875',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00875_c',\n",
       "   'AraGEM: R_R00875_c',\n",
       "   'BiGG: SBTD_D; SBTD_D2; SBTD_Dr; SOR1',\n",
       "   'KEGG: R00875',\n",
       "   'Maize_C4GEM: R_R00875_c',\n",
       "   'Name: D-Glucitol:NAD+ 2-oxidoreductase; D-sorbitol dehydrogenase (D-fructose producing); D-sorbitol dehydrogenase (R); Sorbitol dehydrogenase 1'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00588[0] <=> (1) cpd00004[0] + (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00082;cpd00588',\n",
       "  'definition': '(1) NAD[0] + (1) Sorbitol[0] <=> (1) NADH[0] + (1) H+[0] + (1) D-Fructose[0]',\n",
       "  'deltag': 2.96,\n",
       "  'deltagerr': 0.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.14', '1.1.1.15'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00588[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00082[0]',\n",
       "  'id': 'rxn00634',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucitol:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00588:0:0:\"Sorbitol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00876',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00876',\n",
       "   'iRS1563: R_R00876_c',\n",
       "   'Name: ITP:D-fructose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00068[0] + (1) cpd00082[0] <=> (1) cpd00072[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00068;cpd00072;cpd00082;cpd00090',\n",
       "  'definition': '(1) ITP[0] + (1) D-Fructose[0] => (1) H+[0] + (1) D-fructose-6-phosphate[0] + (1) IDP[0]',\n",
       "  'deltag': -3.67,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1'],\n",
       "  'equation': '(1) cpd00068[0] + (1) cpd00082[0] => (1) cpd00067[0] + (1) cpd00072[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00635',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP:D-fructose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00068:0:0:\"ITP\";-1:cpd00082:0:0:\"D-Fructose\";1:cpd00067:0:0:\"H+\";1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00877',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00877',\n",
       "   'Name: D-Mannose ketol-isomerase; D-mannose aldose-ketose-isomerase'],\n",
       "  'code': '(1) cpd00138[0] <=> (1) cpd00082[0]',\n",
       "  'compound_ids': 'cpd00082;cpd00138',\n",
       "  'definition': '(1) D-Mannose[0] <=> (1) D-Fructose[0]',\n",
       "  'deltag': -0.46,\n",
       "  'deltagerr': 0.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.7'],\n",
       "  'equation': '(1) cpd00138[0] <=> (1) cpd00082[0]',\n",
       "  'id': 'rxn00636',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-mannose aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00138:0:0:\"D-Mannose\";1:cpd00082:0:0:\"D-Fructose\"'},\n",
       " {'abbreviation': 'R00880',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ALGD',\n",
       "   'KEGG: R00880',\n",
       "   'MetaCyc: GDP-MANNOSE-6-DEHYDROGENASE-RXN',\n",
       "   'iAbaylyiv4: GDP-MANNOSE-6-DEHYDROGENASE-RXN',\n",
       "   'iMO1053-PAO1: ALGD',\n",
       "   'Name: GDP-D-mannose:NAD+ 6-oxidoreductase; GDP-mannose 6-dehydrogenase; GDP-mannose dehydrogenase; gdp-mannose 6-dehydrogenase; guanosine diphospho-D-mannose dehydrogenase; guanosine diphosphomannose dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00083[0] <=> (2) cpd00004[0] + (1) cpd00719[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00083;cpd00719',\n",
       "  'definition': '(1) H2O[0] + (2) NAD[0] + (1) GDP-mannose[0] <=> (2) NADH[0] + (3) H+[0] + (1) GDP-D-mannuronate[0]',\n",
       "  'deltag': -2.77,\n",
       "  'deltagerr': 0.85,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.132'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00083[0] <=> (2) cpd00004[0] + (3) cpd00067[0] + (1) cpd00719[0]',\n",
       "  'id': 'rxn00638',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GDP-D-mannose:NAD+ 6-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alginate-Biosynthesis (Alginate Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Glycan-Biosynthesis (Glycan Biosynthesis); Glycan-Pathways (Glycan Pathways); PWY-6073 (alginate biosynthesis I (algal)); PWY-6082 (alginate biosynthesis II (bacterial)); Polysaccharides-Biosynthesis (Polysaccharide Biosynthesis)',\n",
       "   'KEGG: rn00051 (Fructose and mannose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00003:0:0:\"NAD\";-1:cpd00083:0:0:\"GDP-mannose\";2:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";1:cpd00719:0:0:\"GDP-D-mannuronate\"'},\n",
       " {'abbreviation': 'R00881',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GDPMNP',\n",
       "   'EcoCyc: RXN0-5108',\n",
       "   'KEGG: R00881',\n",
       "   'MetaCyc: RXN0-5108',\n",
       "   'iAF1260: GDPMNP',\n",
       "   'Name: GDP-mannose mannophosphohydrolase; GDP-mannose phyrophosphatase; GDPmannose sugarphosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00083[0] <=> (1) cpd00126[0] + (1) cpd00485[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00083;cpd00126;cpd00485',\n",
       "  'definition': '(1) H2O[0] + (1) GDP-mannose[0] => (2) H+[0] + (1) GMP[0] + (1) D-Mannose1-phosphate[0]',\n",
       "  'deltag': -4.74,\n",
       "  'deltagerr': 0.99,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.-', '3.6.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00083[0] => (2) cpd00067[0] + (1) cpd00126[0] + (1) cpd00485[0]',\n",
       "  'id': 'rxn00639',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GDP-mannose mannophosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00083:0:0:\"GDP-mannose\";2:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\";1:cpd00485:0:0:\"D-Mannose1-phosphate\"'},\n",
       " {'abbreviation': 'R00883',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MANNPGUANYLTRANGDP-RXN',\n",
       "   'BiGG: MAN1PT2; MAN1PT2_r; MAN1PT2r',\n",
       "   'BrachyCyc: MANNPGUANYLTRANGDP-RXN',\n",
       "   'KEGG: R00883',\n",
       "   'MetaCyc: MANNPGUANYLTRANGDP-RXN',\n",
       "   'Name: GDP mannose phosphorylase; GDP-mannose 1-phosphate guanylyltransferase; GDP:D-mannose-1-phosphate guanylyltransferase; GDP:alpha-D-mannose-1-phosphate guanylyltransferase; guanosine diphosphate-mannose 1-phosphate guanylyltransferase; guanosine diphosphomannose phosphorylase; mannose 1-phosphate (guanosine diphosphate) guanylyltransferase; mannose 1-phosphate guanylyltransferase; mannose-1-phosphate guanylyltransferase (GDP); mannose-1-phosphate guanylyltransferase (GDP) reversible; mannose-1-phosphateguanylyltransferase-GDPreversible'],\n",
       "  'code': '(1) cpd00031[0] + (1) cpd00485[0] <=> (1) cpd00009[0] + (1) cpd00083[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00031;cpd00083;cpd00485',\n",
       "  'definition': '(1) GDP[0] + (1) D-Mannose1-phosphate[0] <= (1) Phosphate[0] + (1) GDP-mannose[0]',\n",
       "  'deltag': -0.79,\n",
       "  'deltagerr': 0.94,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.7.7.22'],\n",
       "  'equation': '(1) cpd00031[0] + (1) cpd00485[0] <= (1) cpd00009[0] + (1) cpd00083[0]',\n",
       "  'id': 'rxn00640',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GDP:D-mannose-1-phosphate guanylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00051 (Fructose and mannose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00031:0:0:\"GDP\";-1:cpd00485:0:0:\"D-Mannose1-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00083:0:0:\"GDP-mannose\"'},\n",
       " {'abbreviation': 'R00885',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.7.13-RXN',\n",
       "   'BiGG: MAN1PT; PSA1',\n",
       "   'BrachyCyc: 2.7.7.13-RXN',\n",
       "   'KEGG: R00885',\n",
       "   'MetaCyc: 2.7.7.13-RXN',\n",
       "   \"Name: GDP-mannose pyrophosphorylase; GTP-mannose-1-phosphate guanylyltransferase; GTP:alpha-D-mannose-1-phosphate guanylyltransferase; Mannose-1-phosphate guanyltransferase; Mannose-1-phosphate guanylyltransferase; PIM-GMP (phosphomannose isomerase-guanosine 5'-diphospho-D-mannose pyrophosphorylase); guanosine 5'-diphospho-D-mannose pyrophosphorylase; guanosine diphosphomannose pyrophosphorylase; guanosine triphosphate-mannose 1-phosphate guanylyltransferase; mannose 1-phosphate guanylyltransferase (guanosine triphosphate); mannose-1-phosphate guanylyltransferase\"],\n",
       "  'code': '(1) cpd00038[0] + (1) cpd00485[0] <=> (1) cpd00012[0] + (1) cpd00083[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00038;cpd00083;cpd00485',\n",
       "  'definition': '(1) GTP[0] + (1) D-Mannose1-phosphate[0] <=> (1) PPi[0] + (1) GDP-mannose[0]',\n",
       "  'deltag': -2.88,\n",
       "  'deltagerr': 0.88,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.13'],\n",
       "  'equation': '(1) cpd00038[0] + (1) cpd00485[0] <=> (1) cpd00012[0] + (1) cpd00083[0]',\n",
       "  'id': 'rxn00641',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33633',\n",
       "  'name': 'GTP:alpha-D-mannose-1-phosphate guanylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Ascorbate-Biosynthesis (Ascorbate Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); COLANSYN-PWY (colanic acid building blocks biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); GDP-Sugar-Biosynthesis (GDP-sugar Biosynthesis); PWY-5659 (GDP-mannose biosynthesis); PWY-7323 (superpathway of GDP-mannose-derived O-antigen building blocks biosynthesis); PWY-882 (L-ascorbate biosynthesis I (L-galactose pathway)); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00051 (Fructose and mannose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00038:0:0:\"GTP\";-1:cpd00485:0:0:\"D-Mannose1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00083:0:0:\"GDP-mannose\"'},\n",
       " {'abbreviation': 'R00888',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GDPMANDEHYDRA-RXN',\n",
       "   'BiGG: GDPMD; GMAND',\n",
       "   'BrachyCyc: GDPMANDEHYDRA-RXN',\n",
       "   'KEGG: R00888',\n",
       "   'MetaCyc: GDPMANDEHYDRA-RXN',\n",
       "   \"Name: GDP-D-mannose 4,6-dehydratase; GDP-D-mannose dehydratase; GDP-D-mannosedehydratase; GDP-alpha-D-mannose 4,6-hydro-lyase (GDP-4-dehydro-alpha-D-rhamnose-forming); GDP-mannose 4,6-dehydratase; GDP-mannose 4,6-hydro-lyase; GDP-mannose 4,6-hydro-lyase (GDP-4-dehydro-6-deoxy-D-mannose-forming); GDPmannose 4,6-hydro-lyase; Gmd; guanosine 5'-diphosphate-D-mannose oxidoreductase; guanosine diphosphomannose 4,6-dehydratase; guanosine diphosphomannose oxidoreductase\"],\n",
       "  'code': '(1) cpd00083[0] <=> (1) cpd00001[0] + (1) cpd00900[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00083;cpd00900',\n",
       "  'definition': '(1) GDP-mannose[0] => (1) H2O[0] + (1) GDP-4-dehydro-D-rhamnose[0]',\n",
       "  'deltag': -13.88,\n",
       "  'deltagerr': 0.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.47'],\n",
       "  'equation': '(1) cpd00083[0] => (1) cpd00001[0] + (1) cpd00900[0]',\n",
       "  'id': 'rxn00642',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30677;rxn38114',\n",
       "  'name': 'GDP-mannose 4,6-hydro-lyase (GDP-4-dehydro-6-deoxy-D-mannose-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); COLANSYN-PWY (colanic acid building blocks biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); GDP-Sugar-Biosynthesis (GDP-sugar Biosynthesis); GDPRHAMSYN-PWY (GDP-D-rhamnose biosynthesis); PWY-5738 (GDP-6-deoxy-D-talose biosynthesis); PWY-5739 (GDP-D-perosamine biosynthesis); PWY-5740 (GDP-L-colitose biosynthesis); PWY-66 (GDP-L-fucose biosynthesis I (from GDP-D-mannose)); PWY-7323 (superpathway of GDP-mannose-derived O-antigen building blocks biosynthesis); PWY-7573 (GDP-mycosamine biosynthesis); PWY-7575 (superpathway of candicidin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis)',\n",
       "   'KEGG: rn00051 (Fructose and mannose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00083:0:0:\"GDP-mannose\";1:cpd00001:0:0:\"H2O\";1:cpd00900:0:0:\"GDP-4-dehydro-D-rhamnose\"'},\n",
       " {'abbreviation': 'R00889',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-1882',\n",
       "   'BrachyCyc: RXN-1882',\n",
       "   'ChlamyCyc: RXN-1882',\n",
       "   'KEGG: R00889',\n",
       "   'MetaCyc: RXN-1882',\n",
       "   \"Name: GDP-D-mannose:GDP-L-galactose epimerase; GDP-mannose 3,5-epimerase; GDPmannose 3,5-epimerase; guanosine 5'-diphosphate D-mannose:guanosine 5'-diphosphate L-galactose epimerase\"],\n",
       "  'code': '(1) cpd00083[0] <=> (1) cpd01536[0]',\n",
       "  'compound_ids': 'cpd00083;cpd01536',\n",
       "  'definition': '(1) GDP-mannose[0] <=> (1) GDP-L-galactose[0]',\n",
       "  'deltag': 0.4,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.3.18'],\n",
       "  'equation': '(1) cpd00083[0] <=> (1) cpd01536[0]',\n",
       "  'id': 'rxn00643',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31751;rxn31753',\n",
       "  'name': 'GDP-mannose 3,5-epimerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Ascorbate-Biosynthesis (Ascorbate Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); GDP-Sugar-Biosynthesis (GDP-sugar Biosynthesis); PWY-5115 (GDP-L-galactose biosynthesis); PWY-882 (L-ascorbate biosynthesis I (L-galactose pathway)); PWY4FS-12 (VTC2 cycle); PWY4FS-13 (extended VTC2 cycle); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00053 (Ascorbate and aldarate metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00083:0:0:\"GDP-mannose\";1:cpd01536:0:0:\"GDP-L-galactose\"'},\n",
       " {'abbreviation': 'R00892',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00892',\n",
       "   'iRS1563: R_R00892_c',\n",
       "   'Name: L-cysteine:NAD+ oxidoreductase; NADH:L-cystine oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (2) cpd00084[0] <=> (1) cpd00004[0] + (1) cpd00381[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00084;cpd00381',\n",
       "  'definition': '(1) NAD[0] + (2) L-Cysteine[0] <=> (1) NADH[0] + (1) H+[0] + (1) L-Cystine[0]',\n",
       "  'deltag': -1.55,\n",
       "  'deltagerr': 1.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.1.6'],\n",
       "  'equation': '(1) cpd00003[0] + (2) cpd00084[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00381[0]',\n",
       "  'id': 'rxn00644',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-cysteine:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-2:cpd00084:0:0:\"L-Cysteine\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00381:0:0:\"L-Cystine\"'},\n",
       " {'abbreviation': 'R00893',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: CYSTEINE-DIOXYGENASE-RXN',\n",
       "   'KEGG: R00893',\n",
       "   'MetaCyc: CYSTEINE-DIOXYGENASE-RXN',\n",
       "   'PlantCyc: CYSTEINE-DIOXYGENASE-RXN',\n",
       "   'RiceCyc: CYSTEINE-DIOXYGENASE-RXN',\n",
       "   'Name: L-Cysteine:oxygen oxidoreductase; L-cysteine:oxygen oxidoreductase; cysteine dioxygenase; cysteine oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00084[0] <=> (1) cpd00467[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00067;cpd00084;cpd00467',\n",
       "  'definition': '(1) O2[0] + (1) L-Cysteine[0] <=> (1) H+[0] + (1) 3-Sulfinoalanine[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.13.11.20'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00084[0] <=> (1) cpd00067[0] + (1) cpd00467[0]',\n",
       "  'id': 'rxn00645',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34672',\n",
       "  'name': 'L-Cysteine:oxygen oxidoreductase',\n",
       "  'notes': ['GCP', 'EQC'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); CYSTEINE-DEG (L-cysteine Degradation); CYSTEINE-DEG-PWY (L-cysteine degradation I); Degradation (Degradation/Utilization/Assimilation); METHIONINE-DEG (L-methionine Degradation); PWY-5328 (superpathway of L-methionine salvage and degradation); PWY-5331 (taurine biosynthesis I); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); S-CONTAINING-SECONDARY-CMPD-SYN (Sulfur-Containing Secondary Compound Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Taurine-Biosynthesis (Taurine Biosynthesis)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn00430 (Taurine and hypotaurine metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00084:0:0:\"L-Cysteine\";1:cpd00067:0:0:\"H+\";1:cpd00467:0:0:\"3-Sulfinoalanine\"'},\n",
       " {'abbreviation': 'R00894',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUTCYSLIG-RXN',\n",
       "   'BiGG: GLUCYS; GLUCYSL; GSH1',\n",
       "   'BrachyCyc: GLUTCYSLIG-RXN',\n",
       "   'KEGG: R00894',\n",
       "   'MetaCyc: GLUTCYSLIG-RXN',\n",
       "   'Name: Gamma-glutamyl-L-cysteine synthetase; Gamma-glutamylcysteine synthetase; L-Glutamate:L-cysteine gamma-ligase (ADP-forming); L-glutamate:L-cysteine gamma-ligase (ADP-forming); gamma-glutamyl-L-cysteine synthetase; gamma-glutamylcysteine synthetase; gamma-glutamylcysteinyl synthetase; glutamate--cysteine ligase; glutamate-cysteine ligase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00023[0] + (1) cpd00084[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00506[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00023;cpd00067;cpd00084;cpd00506',\n",
       "  'definition': '(1) ATP[0] + (1) L-Glutamate[0] + (1) L-Cysteine[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) gamma-Glutamylcysteine[0]',\n",
       "  'deltag': -7.43,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00023[0] + (1) cpd00084[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00506[0]',\n",
       "  'id': 'rxn00646',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27893',\n",
       "  'name': 'L-glutamate:L-cysteine gamma-ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Ergothioneine-Biosynthesis (Ergothioneine Biosynthesis); GLUTATHIONESYN-PWY (glutathione biosynthesis); PWY-4041 (&gamma;-glutamyl cycle); PWY-6840 (homoglutathione biosynthesis); PWY-7255 (ergothioneine biosynthesis I (bacteria)); Reductants (Reductant Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00023:0:0:\"L-Glutamate\";-1:cpd00084:0:0:\"L-Cysteine\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00506:0:0:\"gamma-Glutamylcysteine\"'},\n",
       " {'abbreviation': 'R00895',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ASPTA4; R00896',\n",
       "   'EcoCyc: CYSTEINE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R00895',\n",
       "   'MetaCyc: CYSTEINE-AMINOTRANSFERASE-RXN',\n",
       "   'RiceCyc: CYSTEINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: CGT; L-Cysteine:2-oxoglutarate aminotransferase; L-cysteine aminotransferase; L-cysteine:2-oxoglutarate aminotransferase; aspartate aminotransferase, cysteine forming; cysteine aminotransferase; cysteine transaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00084[0] <=> (1) cpd00023[0] + (1) cpd00706[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00084;cpd00706',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Cysteine[0] <=> (1) L-Glutamate[0] + (1) Mercaptopyruvate[0]',\n",
       "  'deltag': -4.0,\n",
       "  'deltagerr': 1.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.1', '2.6.1.3'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00084[0] <=> (1) cpd00023[0] + (1) cpd00706[0]',\n",
       "  'id': 'rxn00647',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34664;rxn34671',\n",
       "  'name': 'L-Cysteine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); CYSTEINE-DEG (L-cysteine Degradation); Degradation (Degradation/Utilization/Assimilation); Hydrogen-Sulfide-Biosynthesis (Hydrogen Sulfide Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-5329 (L-cysteine degradation III); PWY0-1534 (hydrogen sulfide biosynthesis I); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00084:0:0:\"L-Cysteine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00706:0:0:\"Mercaptopyruvate\"'},\n",
       " {'abbreviation': 'R00897',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACSERLY-RXN',\n",
       "   'BiGG: CYSS; CYSSr; YGR012W',\n",
       "   'BrachyCyc: ACSERLY-RXN',\n",
       "   'KEGG: R00897',\n",
       "   'MetaCyc: ACSERLY-RXN',\n",
       "   'Name: 3-O-acetyl-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase; Cysteine synthase; O-acetyl-L-serine sulfhydrylase; O-acetyl-L-serine sulfohydrolase; O-acetylserine (thiol)-lyase; O-acetylserine (thiol)-lyase A; O-acetylserine sulfhydrylase; O3-acetyl-L-serine acetate-lyase (adding hydrogen sulfide); O3-acetyl-L-serine acetate-lyase (adding hydrogen-sulfide); O3-acetyl-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase; S-sulfocysteine synthase; acetylserine sulfhydrylase; cysteine synthase; cysteine synthase (reversible); cysteine synthetase; cysteinesynthase'],\n",
       "  'code': '(1) cpd00239[0] + (1) cpd00722[0] <=> (1) cpd00029[0] + (1) cpd00084[0]',\n",
       "  'compound_ids': 'cpd00029;cpd00084;cpd00239;cpd00722',\n",
       "  'definition': '(1) H2S[0] + (1) O-Acetyl-L-serine[0] <=> (1) Acetate[0] + (1) L-Cysteine[0]',\n",
       "  'deltag': -11.8,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.5.1.47', '2.5.1.65'],\n",
       "  'equation': '(1) cpd00239[0] + (1) cpd00722[0] <=> (1) cpd00029[0] + (1) cpd00084[0]',\n",
       "  'id': 'rxn00649',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19041;rxn19042;rxn32343',\n",
       "  'name': 'O3-acetyl-L-serine:hydrogen-sulfide 2-amino-2-carboxyethyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); CYSTEINE-SYN (L-cysteine Biosynthesis); CYSTSYN-PWY (L-cysteine biosynthesis I); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); SULFATE-CYS-PWY (superpathway of sulfate assimilation and cysteine biosynthesis); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn00920 (Sulfur metabolism); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00239:0:0:\"H2S\";-1:cpd00722:0:0:\"O-Acetyl-L-serine\";1:cpd00029:0:0:\"Acetate\";1:cpd00084:0:0:\"L-Cysteine\"'},\n",
       " {'abbreviation': 'R00899',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-6622',\n",
       "   'BiGG: AMPTASECG; TRIA',\n",
       "   'BrachyCyc: RXN-6622',\n",
       "   'KEGG: R00899',\n",
       "   'MetaCyc: RXN-6622',\n",
       "   'Name: L-cysteinylglycine dipeptidase; alanine aminopeptidase; alanyl aminopeptidase; alanyl aminopeptidase (cys-gly)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01017[0] <=> (1) cpd00033[0] + (1) cpd00084[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00033;cpd00084;cpd01017',\n",
       "  'definition': '(1) H2O[0] + (1) Cys-Gly[0] <=> (1) Glycine[0] + (1) L-Cysteine[0]',\n",
       "  'deltag': 1.77,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.4.11.1', '3.4.11.2', '3.4.11.23', '3.4.13.-', '3.4.13.18'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01017[0] <=> (1) cpd00033[0] + (1) cpd00084[0]',\n",
       "  'id': 'rxn00650',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30878;rxn32989;rxn32991',\n",
       "  'name': 'L-cysteinylglycine dipeptidase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: COFACTOR-DEGRADATION (Cofactor, Prosthetic Group, Electron Carrier Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); PWY-4041 (&gamma;-glutamyl cycle); PWY-7559 (glutathione degradation (DUG pathway-yeast)); Reductants (Reductant Biosynthesis)',\n",
       "   'KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01017:0:0:\"Cys-Gly\";1:cpd00033:0:0:\"Glycine\";1:cpd00084:0:0:\"L-Cysteine\"'},\n",
       " {'abbreviation': 'R00901',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00901',\n",
       "   'MetaCyc: CYSTEINE-LYASE-RXN',\n",
       "   'Name: L-Cysteine hydrogen-sulfide-lyase (adding sulfite); L-cysteine hydrogen-sulfide-lyase (adding sulfite); L-cysteine hydrogen-sulfide-lyase (adding sulfite, L-cysteate-forming); L-cysteine hydrogen-sulfide-lyase (adding sulfite; L-cysteate-forming); cysteine (sulfite) lyase; cysteine lyase'],\n",
       "  'code': '(1) cpd00081[0] + (1) cpd00084[0] <=> (1) cpd00239[0] + (1) cpd00395[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00081;cpd00084;cpd00239;cpd00395',\n",
       "  'definition': '(1) Sulfite[0] + (1) L-Cysteine[0] => (1) H+[0] + (1) H2S[0] + (1) L-Cysteate[0]',\n",
       "  'deltag': -13.05,\n",
       "  'deltagerr': 1.27,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.4.1.10'],\n",
       "  'equation': '(1) cpd00081[0] + (1) cpd00084[0] => (1) cpd00067[0] + (1) cpd00239[0] + (1) cpd00395[0]',\n",
       "  'id': 'rxn00651',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-cysteine hydrogen-sulfide-lyase (adding sulfite; L-cysteate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism); rn00430 (Taurine and hypotaurine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00081:0:0:\"Sulfite\";-1:cpd00084:0:0:\"L-Cysteine\";1:cpd00067:0:0:\"H+\";1:cpd00239:0:0:\"H2S\";1:cpd00395:0:0:\"L-Cysteate\"'},\n",
       " {'abbreviation': 'R00903',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00903',\n",
       "   'TS_Athaliana: rxn609_c; rxn609_m; rxn609_p; rxn609_r',\n",
       "   'Name: 2-Amino-3-mercaptopropionic acid racemase; 2-Amino-3-mercaptopropionic acid racemase;; L-2-Amino-3-mercaptopropionic acid racemase; cysteine racemase'],\n",
       "  'code': '(1) cpd00084[0] <=> (1) cpd00587[0]',\n",
       "  'compound_ids': 'cpd00084;cpd00587',\n",
       "  'definition': '(1) L-Cysteine[0] <=> (1) D-Cysteine[0]',\n",
       "  'deltag': -2.14,\n",
       "  'deltagerr': 1.05,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.10'],\n",
       "  'equation': '(1) cpd00084[0] <=> (1) cpd00587[0]',\n",
       "  'id': 'rxn00652',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31853;rxn31854;rxn31855',\n",
       "  'name': '2-Amino-3-mercaptopropionic acid racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00084:0:0:\"L-Cysteine\";1:cpd00587:0:0:\"D-Cysteine\"'},\n",
       " {'abbreviation': 'R00904',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-12332; RXN-6382',\n",
       "   'BiGG: ALDD22x',\n",
       "   'BrachyCyc: RXN-6382',\n",
       "   'KEGG: R00904',\n",
       "   'MetaCyc: RXN-12332; RXN-6382',\n",
       "   'Name: 3-aminopropanal:NAD+ oxidoreductase; aldehyde dehydrogenase (3-aminopropanal, NAD); beta-Aminopropion aldehyde:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00085[0] <=> (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd01504[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00085;cpd01504',\n",
       "  'definition': '(1) NADH[0] + (2) H+[0] + (1) beta-Alanine[0] <=> (1) H2O[0] + (1) NAD[0] + (1) 3-Aminopropanal[0]',\n",
       "  'deltag': 9.54,\n",
       "  'deltagerr': 1.46,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.-', '1.2.1.19', '1.2.1.3', '1.2.1.5'],\n",
       "  'equation': '(1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00085[0] <=> (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd01504[0]',\n",
       "  'id': 'rxn00653',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn22781;rxn28322;rxn31895',\n",
       "  'name': '3-aminopropanal:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Beta-Alanine-Biosynthesis (&beta; Alanine Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-3981 (&beta;-alanine biosynthesis I); PWY-5760 (&beta;-alanine biosynthesis IV)',\n",
       "   'KEGG: rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-2:cpd00067:0:0:\"H+\";-1:cpd00085:0:0:\"beta-Alanine\";1:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\";1:cpd01504:0:0:\"3-Aminopropanal\"'},\n",
       " {'abbreviation': 'R00905',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: BETA-UREIDOPROPIONASE-RXN',\n",
       "   'BiGG: UPPN',\n",
       "   'BrachyCyc: BETA-UREIDOPROPIONASE-RXN',\n",
       "   'KEGG: R00905',\n",
       "   'MetaCyc: BETA-UREIDOPROPIONASE-RXN',\n",
       "   'Name: Beta-ureidopropionase; N-Carbamoyl-beta-alanine amidohydrolase; N-carbamoyl-beta-alanine amidohydrolase; b-ureidopropionase; beta-ureidopropionase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01720[0] <=> (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00067;cpd00085;cpd01720',\n",
       "  'definition': '(1) H2O[0] + (2) H+[0] + (1) 3-Ureidopropanoate[0] => (1) CO2[0] + (1) NH3[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': -13.78,\n",
       "  'deltagerr': 1.7,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00067[0] + (1) cpd01720[0] => (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00654',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33392;rxn34582',\n",
       "  'name': 'N-Carbamoyl-beta-alanine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Beta-Alanine-Biosynthesis (&beta; Alanine Biosynthesis); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-3982 (uracil degradation I (reductive)); PWY-7209 (superpathway of pyrimidine ribonucleosides degradation); Pyrimidine-Base-Degradation (Pyrimidine Nucleobase Degradation); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation); Uracil-Degradation (Uracil Degradation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism); rn00410 (beta-Alanine metabolism); rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00067:0:0:\"H+\";-1:cpd01720:0:0:\"3-Ureidopropanoate\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'R00906',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00906',\n",
       "   'MetaCyc: BETA-ALANOPINE-DEHYDROGENASE-RXN',\n",
       "   'Name: (beta-alanine-forming); N-(D-1-carboxyethyl)-beta-alanine:NAD+ oxidoreductase; N-(D-1-carboxyethyl)-beta-alanine:NAD+ oxidoreductase (beta-alanine-forming); beta-alanopine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd01489[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00020;cpd00067;cpd00085;cpd01489',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) beta-Alanopine[0] <=> (1) NADH[0] + (1) Pyruvate[0] + (1) H+[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': 8.09,\n",
       "  'deltagerr': 1.65,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.26'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd01489[0] <=> (1) cpd00004[0] + (1) cpd00020[0] + (1) cpd00067[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00655',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-(D-1-carboxyethyl)-beta-alanine:NAD+ oxidoreductase (beta-alanine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7351 (pyruvate fermentation to opines); Pyruvate-Degradation (Fermentation of Pyruvate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd01489:0:0:\"beta-Alanopine\";1:cpd00004:0:0:\"NADH\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'R00907',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.6.1.18-RXN',\n",
       "   'BiGG: APATr; BALAAT',\n",
       "   'BrachyCyc: 2.6.1.18-RXN',\n",
       "   'KEGG: R00907',\n",
       "   'MetaCyc: 2.6.1.18-RXN',\n",
       "   'Name: B-alanine-pyruvate aminotransferase; Beta-alanine--pyruvate aminotransferase; L-Alanine:3-oxopropanoate aminotransferase; L-alanine:3-oxopropanoate aminotransferase; beta-alanine-alpha-alanine transaminase; beta-alanine-pyruvate aminotransferase; beta-alanine-pyruvate transaminase'],\n",
       "  'code': '(1) cpd00035[0] + (1) cpd00191[0] <=> (1) cpd00020[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00035;cpd00085;cpd00191',\n",
       "  'definition': '(1) L-Alanine[0] + (1) 3-Oxopropanoate[0] <=> (1) Pyruvate[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': 0.96,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.18'],\n",
       "  'equation': '(1) cpd00035[0] + (1) cpd00191[0] <=> (1) cpd00020[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00656',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17652;rxn32229;rxn32234',\n",
       "  'name': 'L-Alanine:3-oxopropanoate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Beta-Alanine-Biosynthesis (&beta; Alanine Biosynthesis); Beta-Alanine-Degradation (&beta;-Alanine Degradation); CoA-Biosynthesis (Coenzyme A Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); MISCELLANEOUS-DEG (Other Amino Acid Degradation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-1781 (&beta;-alanine degradation II); PWY-3941 (&beta;-alanine biosynthesis II); PWY-4221 (superpathway of coenzyme A biosynthesis II (plants))',\n",
       "   'KEGG: rn00410 (beta-Alanine metabolism); rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00035:0:0:\"L-Alanine\";-1:cpd00191:0:0:\"3-Oxopropanoate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'R00908',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-2901',\n",
       "   'ChlamyCyc: RXN-2901',\n",
       "   'CornCyc: RXN-2901',\n",
       "   'KEGG: R00908',\n",
       "   'MetaCyc: RXN-2901',\n",
       "   'Name: 3-Aminopropanoate:2-oxoglutarate aminotransferase; beta-alanine:2-oxoglutarate aminotransferase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00085[0] <=> (1) cpd00023[0] + (1) cpd00191[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00085;cpd00191',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) beta-Alanine[0] <=> (1) L-Glutamate[0] + (1) 3-Oxopropanoate[0]',\n",
       "  'deltag': -0.89,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.19', '2.6.1.55'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00085[0] <=> (1) cpd00023[0] + (1) cpd00191[0]',\n",
       "  'id': 'rxn00657',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn23816;rxn28267',\n",
       "  'name': 'beta-alanine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); BETA-ALA-DEGRADATION-I-PWY (&beta;-alanine degradation I); Beta-Alanine-Degradation (&beta;-Alanine Degradation); Degradation (Degradation/Utilization/Assimilation); MISCELLANEOUS-DEG (Other Amino Acid Degradation)',\n",
       "   'KEGG: rn00410 (beta-Alanine metabolism); rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00085:0:0:\"beta-Alanine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00191:0:0:\"3-Oxopropanoate\"'},\n",
       " {'abbreviation': 'R00909',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00909',\n",
       "   'MetaCyc: N-ACETYL-BETA-ALANINE-DEACETYLASE-RXN',\n",
       "   'Name: N-Acetyl-beta-alanine amidohydrolase; N-acetyl-beta-alanine amidohydrolase; N-acetyl-beta-alanine deacetylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00788[0] <=> (1) cpd00029[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00085;cpd00788',\n",
       "  'definition': '(1) H2O[0] + (1) N-Acetyl-beta-alanine[0] <=> (1) Acetate[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': -3.38,\n",
       "  'deltagerr': 1.36,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00788[0] <=> (1) cpd00029[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00658',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Acetyl-beta-alanine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00788:0:0:\"N-Acetyl-beta-alanine\";1:cpd00029:0:0:\"Acetate\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'R00910',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00910',\n",
       "   'Name: L-Lysine:beta-alanine ligase (AMP-forming); L-lysine:beta-alanine ligase (AMP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00039[0] + (1) cpd00085[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd03168[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00018;cpd00039;cpd00067;cpd00085;cpd03168',\n",
       "  'definition': '(1) ATP[0] + (1) L-Lysine[0] + (1) beta-Alanine[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0] + (1) beta-Alanyl-L-lysine[0]',\n",
       "  'deltag': -8.32,\n",
       "  'deltagerr': 1.4,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.11'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00039[0] + (1) cpd00085[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd03168[0]',\n",
       "  'id': 'rxn00659',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-lysine:beta-alanine ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00039:0:0:\"L-Lysine\";-1:cpd00085:0:0:\"beta-Alanine\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd03168:0:0:\"beta-Alanyl-L-lysine\"'},\n",
       " {'abbreviation': 'R00911',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00911', 'Name: beta-Alanyl-L-lysine hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03168[0] <=> (1) cpd00039[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00039;cpd00085;cpd03168',\n",
       "  'definition': '(1) H2O[0] + (1) beta-Alanyl-L-lysine[0] <=> (1) L-Lysine[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': -0.11,\n",
       "  'deltagerr': 1.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.4.13.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03168[0] <=> (1) cpd00039[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00660',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'beta-Alanyl-L-lysine hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03168:0:0:\"beta-Alanyl-L-lysine\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'R00912',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00912',\n",
       "   'Name: L-Arginine:beta-alanine ligase (AMP-forming); L-arginine:beta-alanine ligase (AMP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00051[0] + (1) cpd00085[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd03167[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00018;cpd00051;cpd00067;cpd00085;cpd03167',\n",
       "  'definition': '(1) ATP[0] + (1) L-Arginine[0] + (1) beta-Alanine[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0] + (1) beta-Alanyl-L-arginine[0]',\n",
       "  'deltag': -3.17,\n",
       "  'deltagerr': 1.88,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.11'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00051[0] + (1) cpd00085[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd03167[0]',\n",
       "  'id': 'rxn00661',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-arginine:beta-alanine ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00051:0:0:\"L-Arginine\";-1:cpd00085:0:0:\"beta-Alanine\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd03167:0:0:\"beta-Alanyl-L-arginine\"'},\n",
       " {'abbreviation': 'R00913',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00913',\n",
       "   'MetaCyc: GUANIDINOPROPIONASE-RXN',\n",
       "   'Name: 3-Guanidinopropanoate amidinopropionase; 3-guanidinopropanoate amidinopropionase; GPH; GPase; guanidinopropionase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01962[0] <=> (1) cpd00073[0] + (1) cpd00085[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00073;cpd00085;cpd01962',\n",
       "  'definition': '(1) H2O[0] + (1) 3-Guanidinopropanoate[0] => (1) Urea[0] + (1) beta-Alanine[0]',\n",
       "  'deltag': -14.89,\n",
       "  'deltagerr': 1.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.3.17'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01962[0] => (1) cpd00073[0] + (1) cpd00085[0]',\n",
       "  'id': 'rxn00662',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-Guanidinopropanoate amidinopropionase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01962:0:0:\"3-Guanidinopropanoate\";1:cpd00073:0:0:\"Urea\";1:cpd00085:0:0:\"beta-Alanine\"'},\n",
       " {'abbreviation': 'rxn00663',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00085[0] <=> (1) cpd00001[0] + (1) cpd01560[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00067;cpd00085;cpd01560',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) beta-Alanine[0] <=> (1) H2O[0] + (1) beta-Alanyl-CoA[0]',\n",
       "  'deltag': 3.94,\n",
       "  'deltagerr': 4.98,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00085[0] <=> (1) cpd00001[0] + (1) cpd01560[0]',\n",
       "  'id': 'rxn00663',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00663',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd00085:0:0:\"beta-Alanine\";1:cpd00001:0:0:\"H2O\";1:cpd01560:0:0:\"beta-Alanyl-CoA\"'},\n",
       " {'abbreviation': 'R00915',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00915', 'iRS1563: R_R00915_c', 'Name: '],\n",
       "  'code': '(1) cpd00085[0] <=> (1) cpd02333[0]',\n",
       "  'compound_ids': 'cpd00085;cpd02333',\n",
       "  'definition': '(1) beta-Alanine[0] => (1) Quinolinate[0]',\n",
       "  'deltag': -73.51,\n",
       "  'deltagerr': 1.49,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00085[0] => (1) cpd02333[0]',\n",
       "  'id': 'rxn00664',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R00915',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:C:4/H:-4/O:2|CI:-2',\n",
       "  'stoichiometry': '-1:cpd00085:0:0:\"beta-Alanine\";1:cpd02333:0:0:\"Quinolinate\"'},\n",
       " {'abbreviation': 'R00916',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00916', 'Name: '],\n",
       "  'code': '(1) cpd00085[0] <=> (1) cpd03376[0]',\n",
       "  'compound_ids': 'cpd00085;cpd03376',\n",
       "  'definition': '(1) beta-Alanine[0] <=> (1) beta-Nitropropanoate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00085[0] <=> (1) cpd03376[0]',\n",
       "  'id': 'rxn00665',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R00916',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:H:-3/O:2|CI:-1',\n",
       "  'stoichiometry': '-1:cpd00085:0:0:\"beta-Alanine\";1:cpd03376:0:0:\"beta-Nitropropanoate\"'},\n",
       " {'abbreviation': 'rxn00666',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00085[0] <=> (1) cpd03377[0]',\n",
       "  'compound_ids': 'cpd00085;cpd03377',\n",
       "  'definition': '(1) beta-Alanine[0] <=> (1) 3-Aminopropiononitrile[0]',\n",
       "  'deltag': 92.6,\n",
       "  'deltagerr': 4.94,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00085[0] <=> (1) cpd03377[0]',\n",
       "  'id': 'rxn00666',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00666',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:N:1/O:-2|CI:1',\n",
       "  'stoichiometry': '-1:cpd00085:0:0:\"beta-Alanine\";1:cpd03377:0:0:\"3-Aminopropiononitrile\"'},\n",
       " {'abbreviation': 'Malonyl-CoA:propionyl-CoA malonyltransfersase (cyclizing)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Malonyl-CoA:propionyl-CoA malonyltransfersase (cyclizing)'],\n",
       "  'code': '(1) cpd00086[0] + (6) cpd01675[0] <=> (7) cpd00010[0] + (6) cpd00011[0] + (1) cpd02075[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00011;cpd00086;cpd01675;cpd02075',\n",
       "  'definition': '(1) Propionyl-CoA[0] + (6) Methylmalonyl-CoA[0] <=> (7) CoA[0] + (6) CO2[0] + (1) 6-Deoxyerythronolide B[0]',\n",
       "  'deltag': 83.08,\n",
       "  'deltagerr': 7.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.94'],\n",
       "  'equation': '(1) cpd00086[0] + (6) cpd01675[0] <=> (7) cpd00010[0] + (6) cpd00011[0] + (1) cpd02075[0]',\n",
       "  'id': 'rxn00667',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Malonyl-CoA:propionyl-CoA malonyltransfersase (cyclizing)',\n",
       "  'notes': ['GCP', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:H:16/O:-1|CI:6',\n",
       "  'stoichiometry': '-1:cpd00086:0:0:\"Propionyl-CoA\";-6:cpd01675:0:0:\"Methylmalonyl-CoA\";7:cpd00010:0:0:\"CoA\";6:cpd00011:0:0:\"CO2\";1:cpd02075:0:0:\"6-Deoxyerythronolide B\"'},\n",
       " {'abbreviation': 'R00919',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_PCNO; R_PCNOn',\n",
       "   'KEGG: R00919',\n",
       "   'MetaCyc: RXN-9087',\n",
       "   'TS_Athaliana: rxn610_c',\n",
       "   'iRS1563: R_R00919_c',\n",
       "   'Name: Acrylyl-CoA reductase (NADPH); acrylyl-CoA reductase (NADPH); propanoyl-CoA:NADP+ 2-oxidoreductase; propanoyl-CoA:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00663[0] <=> (1) cpd00006[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00086;cpd00663',\n",
       "  'definition': '(1) NADPH[0] + (1) H+[0] + (1) Acrylyl-CoA[0] <= (1) NADP[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': -14.14,\n",
       "  'deltagerr': 0.81,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.3.1.-', '1.3.1.84'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00663[0] <= (1) cpd00006[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00668',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35427',\n",
       "  'name': 'propanoyl-CoA:NADP+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5743 (3-hydroxypropanoate cycle); PWY-5744 (glyoxylate assimilation); PWY-5789 (3-hydroxypropanoate/4-hydroxybutanate cycle); PWY-7024 (superpathway of the 3-hydroxypropanoate cycle)',\n",
       "   'KEGG: rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00067:0:0:H+;-1:cpd00663:0:0:\"Acrylyl-CoA\";1:cpd00006:0:0:\"NADP\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00920',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACCOAL; ACCOAL2r',\n",
       "   'BrachyCyc: PROPANOATECOA-LIGASE-RXN',\n",
       "   'KEGG: R00920',\n",
       "   'MetaCyc: PROPANOATECOA-LIGASE-RXN',\n",
       "   'iAF1260: ACCOAL',\n",
       "   'Name: Propanoate:CoA ligase (ADP-forming); Propionate thiokinase; Propionate--CoA ligase (ADP-forming); Propionate-CoA synthetase (ADP-forming); acetate-CoA ligase (ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00141[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00010;cpd00086;cpd00141',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Propionate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': 3.69,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.13'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00141[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00669',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Propanoate:CoA ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00141:0:0:\"Propionate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00921',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PTA2; PTA2r; PTAr_ppa',\n",
       "   'ChlamyCyc: PTAALT-RXN',\n",
       "   'EcoCyc: PTAALT-RXN',\n",
       "   'KEGG: R00921',\n",
       "   'MetaCyc: PTAALT-RXN',\n",
       "   'Name: Phosphate acetyltransferase; propanoyl-CoA:phosphate propanoyltransferase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00086[0] <=> (1) cpd00010[0] + (1) cpd01844[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00010;cpd00086;cpd01844',\n",
       "  'definition': '(1) Phosphate[0] + (1) Propionyl-CoA[0] <=> (1) CoA[0] + (1) Propionyl phosphate[0]',\n",
       "  'deltag': 4.48,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.-', '2.3.1.222', '2.3.1.8'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00086[0] <=> (1) cpd00010[0] + (1) cpd01844[0]',\n",
       "  'id': 'rxn00670',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28106;rxn28107',\n",
       "  'name': 'propanoyl-CoA:phosphate propanoyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); PWY-5437 (L-threonine degradation I); PWY-7013 ((<i>S</i>)-propane-1,2-diol degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); THREOCAT-PWY (superpathway of L-threonine metabolism); THREONINE-DEG (L-threonine Degradation)',\n",
       "   'KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00086:0:0:\"Propionyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd01844:0:0:\"Propionyl phosphate\"'},\n",
       " {'abbreviation': 'R00922',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MMSAD1; MMTSAO',\n",
       "   'KEGG: R00922',\n",
       "   'iAG612: rbs_468',\n",
       "   'iAbaylyiv4: 1.2.1.27-RXN',\n",
       "   'iJN746: MMTSAO',\n",
       "   'Name: (CoA-propanoylating); 2-Methyl-3-oxopropanoate:NAD+ oxidoreductase (CoA-propanoylating); methylmalonate-semialdehyde dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00287[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00011;cpd00086;cpd00287',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) 3-Oxo-2-methylpropanoate[0] => (1) NADH[0] + (1) CO2[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': -6.89,\n",
       "  'deltagerr': 1.05,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.27'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00287[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00671',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Methyl-3-oxopropanoate:NAD+ oxidoreductase (CoA-propanoylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd00287:0:0:\"3-Oxo-2-methylpropanoate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00923',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MMCD',\n",
       "   'KEGG: R00923',\n",
       "   'MetaCyc: METHYLMALONYL-COA-DECARBOXYLASE-RXN; RXN-19738',\n",
       "   'iAF1260: MMCD',\n",
       "   'iJR904: MMCD',\n",
       "   'Name: (S)-2-Methyl-3-oxopropanoyl-CoA carboxy-lyase; (S)-2-methyl-3-oxopropanoyl-CoA carboxy-lyase [incorrect]; (S)-methylmalonyl-CoA carboxy-lyase; (S)-methylmalonyl-CoA carboxy-lyase (propanoyl-CoA-forming); Methylmalonyl-CoA decarboxylase; methylmalonyl-CoA decarboxylase; methylmalonyl-coenzyme A decarboxylase; propionyl coenzyme A carboxylase; propionyl-CoA carboxylase'],\n",
       "  'code': '(1) cpd00519[0] <=> (1) cpd00011[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00086;cpd00519',\n",
       "  'definition': '(1) H+[0] + (1) D-methylmalonyl-CoA[0] => (1) CO2[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': -4.15,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.41', '4.1.1.94', '7.2.4.3'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00519[0] => (1) cpd00011[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00672',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-methylmalonyl-CoA carboxy-lyase (propanoyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00519:0:0:\"D-methylmalonyl-CoA\";1:cpd00011:0:0:\"CO2\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00924',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00924_c',\n",
       "   'AraGEM: R_R00924_c',\n",
       "   'ChlamyCyc: PROPCOASYN-RXN',\n",
       "   'DF_Athaliana: reac_576',\n",
       "   'KEGG: R00924',\n",
       "   'Name: Propanoyl-CoA:(acceptor) 2,3-oxidoreductase; propanoyl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00086[0] <=> (1) cpd00663[0] + (1) cpd00982[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00086;cpd00663;cpd00982',\n",
       "  'definition': '(1) FAD[0] + (1) Propionyl-CoA[0] <=> (1) Acrylyl-CoA[0] + (1) FADH2[0]',\n",
       "  'deltag': 10.53,\n",
       "  'deltagerr': 1.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.99.3'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00086[0] <=> (1) cpd00663[0] + (1) cpd00982[0]',\n",
       "  'id': 'rxn00673',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28087;rxn32155',\n",
       "  'name': 'Propanoyl-CoA:(acceptor) 2,3-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00086:0:0:\"Propionyl-CoA\";1:cpd00663:0:0:\"Acrylyl-CoA\";1:cpd00982:0:0:\"FADH2\"'},\n",
       " {'abbreviation': 'R00925',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PROPIONATE--COA-LIGASE-RXN',\n",
       "   'BiGG: ACS2',\n",
       "   'BrachyCyc: PROPIONATE--COA-LIGASE-RXN',\n",
       "   'KEGG: R00925',\n",
       "   'MetaCyc: PROPIONATE--COA-LIGASE-RXN',\n",
       "   'Name: Propanoate:CoA ligase (AMP-forming); Propionate--CoA ligase; Propionyl-CoA synthetase; acetyl-CoA synthase (propionate); propanoate:CoA ligase (AMP-forming); propionate--coa ligase; propionate-CoA ligase; propionyl-CoA synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00141[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00086;cpd00141',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Propionate[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': 1.42,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.1', '6.2.1.17'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00141[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00674',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28088;rxn28089',\n",
       "  'name': 'Propanoate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Beta-Alanine-Biosynthesis (&beta; Alanine Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); CoA-Biosynthesis (Coenzyme A Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); ISOLEUCINE-SYN (L-isoleucine Biosynthesis); Methyl-Citrate-Cycle (2-Methylcitrate Cycle); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-3941 (&beta;-alanine biosynthesis II); PWY-4221 (superpathway of coenzyme A biosynthesis II (plants)); PWY-5104 (L-isoleucine biosynthesis IV); PWY-5747 (2-methylcitrate cycle II); PWY0-42 (2-methylcitrate cycle I); Propionate-Degradation (Propanoate Degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00141:0:0:\"Propionate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00926',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_PACPT',\n",
       "   'KEGG: R00926',\n",
       "   'TS_Athaliana: rxn611_p',\n",
       "   'iRS1597: R00926_c',\n",
       "   'Name: Propinol adenylate:CoA ligase (AMP-forming); Propionyladenylate:CoA propionyltransferase; propionyladenylate:CoA propionyltransferase'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd03560[0] <=> (1) cpd00018[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00018;cpd00067;cpd00086;cpd03560',\n",
       "  'definition': '(1) CoA[0] + (1) Propionyladenylate[0] => (1) AMP[0] + (1) H+[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': -10.42,\n",
       "  'deltagerr': 6.57,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.1', '6.2.1.17'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd03560[0] => (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00675',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31998;rxn35424',\n",
       "  'name': 'Propionyladenylate:CoA propionyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd03560:0:0:\"Propionyladenylate\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00927',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: METHYLACETOACETYLCOATHIOL-RXN',\n",
       "   'BiGG: ACACT10rr; ACOACAT; MACCOAT',\n",
       "   'BrachyCyc: METHYLACETOACETYLCOATHIOL-RXN',\n",
       "   'KEGG: R00927',\n",
       "   'MetaCyc: METHYLACETOACETYLCOATHIOL-RXN',\n",
       "   'Name: 2-Methylacetyl-CoA thiolase; 2-methylacetoacetyl-CoA thiolase; Acetyl-CoA C-acyltransferase; Acetyl_CoA_C_acyltransferase; Propanoyl-CoA:acetyl-CoA C-acyltransferase; acetyl-CoA C-acyltransferase (2-Methyl-3-acetoacetyl-CoA); acetyl-CoA:propanoyl-CoA 2-C-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00086[0] <=> (1) cpd00010[0] + (1) cpd02124[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00086;cpd02124',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Propionyl-CoA[0] <=> (1) CoA[0] + (1) 2-Methylacetoacetyl-CoA[0]',\n",
       "  'deltag': 6.98,\n",
       "  'deltagerr': 0.69,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.16', '2.3.1.9'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00086[0] <=> (1) cpd00010[0] + (1) cpd02124[0]',\n",
       "  'id': 'rxn00676',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20197;rxn33120',\n",
       "  'name': 'acetyl-CoA:propanoyl-CoA 2-C-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); ILEUDEG-PWY (L-isoleucine degradation I); ISOLEUCINE-DEG (L-isoleucine Degradation); PWY-5109 (fermentation to 2-methylbutanoate); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00280 (Valine, leucine and isoleucine degradation); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00086:0:0:\"Propionyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd02124:0:0:\"2-Methylacetoacetyl-CoA\"'},\n",
       " {'abbreviation': 'R00928',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00928',\n",
       "   'MetaCyc: PROPIONATE-COA-TRANSFERASE-RXN',\n",
       "   'RiceCyc: PROPIONATE-COA-TRANSFERASE-RXN',\n",
       "   'SorghumCyc: PROPIONATE-COA-TRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:propanoate CoA-transferase; acetyl-CoA:propanoate CoA-transferase; propionate CoA-transferase; propionate coenzyme A-transferase; propionate-CoA:lactoyl-CoA transferase; propionyl CoA:acetate CoA transferase; propionyl-CoA transferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00141[0] <=> (1) cpd00029[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00029;cpd00086;cpd00141',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Propionate[0] <=> (1) Acetate[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': 2.55,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.1'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00141[0] <=> (1) cpd00029[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00677',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:propanoate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); PROPFERM-PWY (L-alanine fermentation to propanoate and acetate); PWY-5109 (fermentation to 2-methylbutanoate); PWY-5494 (pyruvate fermentation to propanoate II (acrylate pathway)); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Propanoate-Fermentation (Pyruvate Fermentation to Propanoate)',\n",
       "   'KEGG: rn00620 (Pyruvate metabolism); rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00141:0:0:\"Propionate\";1:cpd00029:0:0:\"Acetate\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00930',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00930',\n",
       "   'MetaCyc: 2.1.3.1-RXN',\n",
       "   'Name: (S)-2-Methyl-3-oxopropanoyl-CoA:pyruvate carboxyltransferase; (S)-2-methyl-3-oxopropanoyl-CoA:pyruvate carboxyltransferase; (S)-2-methyl-3-oxopropanoyl-CoA:pyruvate carboxytransferase carboxytransferase [incorrect]; (S)-methylmalonyl-CoA:pyruvate carboxytransferase; methylmalonyl coenzyme A carboxyltransferase; methylmalonyl-CoA carboxyltransferase; methylmalonyl-CoA carboxytransferase; methylmalonyl-CoA transcarboxylase; oxalacetic transcarboxylase; transcarboxylase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00519[0] <=> (1) cpd00032[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00032;cpd00086;cpd00519',\n",
       "  'definition': '(1) Pyruvate[0] + (1) D-methylmalonyl-CoA[0] <=> (1) Oxaloacetate[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': -0.18,\n",
       "  'deltagerr': 1.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.3.1'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00519[0] <=> (1) cpd00032[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00678',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-2-Methyl-3-oxopropanoyl-CoA:pyruvate carboxyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); GLUTAMATE-DEG (L-glutamate Degradation); P108-PWY (pyruvate fermentation to propanoate I); PWY-5088 (L-glutamate degradation VIII (to propanoate)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Propanoate-Fermentation (Pyruvate Fermentation to Propanoate)',\n",
       "   'KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00519:0:0:\"D-methylmalonyl-CoA\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00931',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MCITS; MCITSm',\n",
       "   'KEGG: R00931',\n",
       "   'TS_Athaliana: rxn612_c; rxn612_m; rxn612_p; rxn612_r; rxn612_v',\n",
       "   'iAF1260: MCITS',\n",
       "   'iAG612: rbs_524',\n",
       "   'Name: 2-Methylcitrate oxaloacetate-lyase; 2-methylcitrate oxaloacetate-lyase; 2-methylcitrate synthase; propanoyl-CoA:oxaloacetate C-propanoyltransferase; propanoyl-CoA:oxaloacetate C-propanoyltransferase (thioester-hydrolysing, 1-carboxyethyl-forming)'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd01501[0] <=> (1) cpd00001[0] + (1) cpd00032[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00032;cpd00067;cpd00086;cpd01501',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) 2-Methylcitrate[0] <= (1) H2O[0] + (1) Oxaloacetate[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': 9.57,\n",
       "  'deltagerr': 0.81,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.5', '4.1.3.31'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01501[0] <= (1) cpd00001[0] + (1) cpd00032[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00679',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30407;rxn32736;rxn32738;rxn32739',\n",
       "  'name': 'propanoyl-CoA:oxaloacetate C-propanoyltransferase (thioester-hydrolysing, 1-carboxyethyl-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd01501:0:0:\"2-Methylcitrate\";1:cpd00001:0:0:\"H2O\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'R00932',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00932',\n",
       "   'iRS1563: R_R00932_c',\n",
       "   'Name: 2-Hydroxyglutarate glyoxylate-lyase (CoA-propanoylating); 2-hydroxyglutarate glyoxylate-lyase (CoA-propanoylating); propanoyl-CoA:glyoxylate C-propanoyltransferase (thioester-hydrolysing, 2-carboxyethyl-forming)'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd01709[0] <=> (1) cpd00001[0] + (1) cpd00040[0] + (1) cpd00086[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00040;cpd00067;cpd00086;cpd01709',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) 2-Hydroxyglutarate[0] <= (1) H2O[0] + (1) Glyoxalate[0] + (1) Propionyl-CoA[0]',\n",
       "  'deltag': 9.2,\n",
       "  'deltagerr': 0.85,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.11'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01709[0] <= (1) cpd00001[0] + (1) cpd00040[0] + (1) cpd00086[0]',\n",
       "  'id': 'rxn00680',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'propanoyl-CoA:glyoxylate C-propanoyltransferase (thioester-hydrolysing, 2-carboxyethyl-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00660 (C5-Branched dibasic acid metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd01709:0:0:\"2-Hydroxyglutarate\";1:cpd00001:0:0:\"H2O\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00086:0:0:\"Propionyl-CoA\"'},\n",
       " {'abbreviation': 'rxn00681',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00040[0] + (1) cpd00086[0] <=> (1) cpd01709[0]',\n",
       "  'compound_ids': 'cpd00040;cpd00086;cpd01709',\n",
       "  'definition': '(1) Glyoxalate[0] + (1) Propionyl-CoA[0] <=> (1) 2-Hydroxyglutarate[0]',\n",
       "  'deltag': 382.79,\n",
       "  'deltagerr': 2.1,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00040[0] + (1) cpd00086[0] <=> (1) cpd01709[0]',\n",
       "  'id': 'rxn00681',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00681',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:C:-21/H:-31/N:-7/O:-15/P:-3/S:-1|CI:3',\n",
       "  'stoichiometry': '-1:cpd00040:0:0:\"Glyoxalate\";-1:cpd00086:0:0:\"Propionyl-CoA\";1:cpd01709:0:0:\"2-Hydroxyglutarate\"'},\n",
       " {'abbreviation': 'R00934',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00934',\n",
       "   'Name: L-erythro-3-Methylmalyl-CoA glyoxylate-lyase (propanoyl-CoA-forming); L-erythro-3-methylmalyl-CoA glyoxylate-lyase (propanoyl-CoA-forming)'],\n",
       "  'code': '(1) cpd00040[0] + (1) cpd00086[0] <=> (1) cpd03588[0]',\n",
       "  'compound_ids': 'cpd00040;cpd00086;cpd03588',\n",
       "  'definition': '(1) Glyoxalate[0] + (1) Propionyl-CoA[0] <=> (1) L-erythro-3-Methylmalyl-CoA[0]',\n",
       "  'deltag': -5.02,\n",
       "  'deltagerr': 5.04,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.24'],\n",
       "  'equation': '(1) cpd00040[0] + (1) cpd00086[0] <=> (1) cpd03588[0]',\n",
       "  'id': 'rxn00682',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-erythro-3-Methylmalyl-CoA glyoxylate-lyase (propanoyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn00660 (C5-Branched dibasic acid metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00040:0:0:\"Glyoxalate\";-1:cpd00086:0:0:\"Propionyl-CoA\";1:cpd03588:0:0:\"L-erythro-3-Methylmalyl-CoA\"'},\n",
       " {'abbreviation': 'R00936',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00936_c',\n",
       "   'AraGEM: R_R00936_c',\n",
       "   'JP_Creinhardtii_MSB: R_DHFR2i',\n",
       "   'KEGG: R00936',\n",
       "   'Maize_C4GEM: R_R00936_c',\n",
       "   'Name: 5,6,7,8-Tetrahydrofolate:NAD+ oxidoreductase; 5,6,7,8-tetrahydrofolate:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00087[0] <=> (1) cpd00004[0] + (1) cpd00330[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00087;cpd00330',\n",
       "  'definition': '(1) NAD[0] + (1) Tetrahydrofolate[0] <=> (1) NADH[0] + (1) H+[0] + (1) Dihydrofolate[0]',\n",
       "  'deltag': 5.18,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.3'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00087[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00330[0]',\n",
       "  'id': 'rxn00684',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32434',\n",
       "  'name': '5,6,7,8-tetrahydrofolate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00670 (One carbon pool by folate); rn00790 (Folate biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00330:0:0:\"Dihydrofolate\"'},\n",
       " {'abbreviation': 'R_R00937_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00937_c',\n",
       "   'AraGEM: R_R00937_c',\n",
       "   'Maize_C4GEM: R_R00937_c',\n",
       "   'iRS1563: R_R00937_c',\n",
       "   'iRS1597: R00937_c',\n",
       "   'Name: 5,6,7,8-Tetrahydrofolate:NAD+ oxidoreductase; bifunctional dihydrofolate reductase-thymidylate synthase 1 / DHFR-TS (THY-1)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00087[0] <=> (1) cpd00004[0] + (1) cpd00393[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00087;cpd00393',\n",
       "  'definition': '(1) NAD[0] + (1) Tetrahydrofolate[0] <= (1) NADH[0] + (3) H+[0] + (1) Folate[0]',\n",
       "  'deltag': -5.75,\n",
       "  'deltagerr': 0.93,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.5.1.3'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00087[0] <= (1) cpd00004[0] + (3) cpd00067[0] + (1) cpd00393[0]',\n",
       "  'id': 'rxn00685',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'bifunctional dihydrofolate reductase-thymidylate synthase 1 / DHFR-TS (THY-1)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";1:cpd00393:0:0:\"Folate\"'},\n",
       " {'abbreviation': 'R00939',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DIHYDROFOLATEREDUCT-RXN',\n",
       "   'BiGG: DFR1_1; DFR1_2; DHFR; DHFRi; DHFRim',\n",
       "   'BrachyCyc: DIHYDROFOLATEREDUCT-RXN',\n",
       "   'KEGG: R00939',\n",
       "   'MetaCyc: DIHYDROFOLATEREDUCT-RXN; RXN-19329',\n",
       "   'Name: 5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase; 7,8-dihydrofolate reductase; Dihydrofolate reductase; NADPH-dihydrofolate reductase; Tetrahydrofolate dehydrogenase; dihydrofolate reductase; dihydrofolate reductase (irreversible); dihydrofolate reductase:thymidylate synthase; dihydrofolatereductase-irreversible; dihydrofolic acid reductase; dihydrofolic reductase; dihydropholate reductase; folic acid reductase; folic reductase; pteridine reductase:dihydrofolate reductase; tetrahydrofolate dehydrogenase; thymidylate synthetase-dihydrofolate reductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00087[0] <=> (1) cpd00005[0] + (1) cpd00330[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00087;cpd00330',\n",
       "  'definition': '(1) NADP[0] + (1) Tetrahydrofolate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Dihydrofolate[0]',\n",
       "  'deltag': 5.18,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.3'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00087[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00330[0]',\n",
       "  'id': 'rxn00686',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27793;rxn30104',\n",
       "  'name': '5,6,7,8-tetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 1CMET2-PWY (<i>N</i><sup>10</sup>-formyl-tetrahydrofolate biosynthesis); ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); FOLSYN-PWY (superpathway of tetrahydrofolate biosynthesis and salvage); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); PWY-3841 (folate transformations II); PWY-6612 (superpathway of tetrahydrofolate biosynthesis); PWY-6614 (tetrahydrofolate biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00670 (One carbon pool by folate); rn00790 (Folate biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00330:0:0:\"Dihydrofolate\"'},\n",
       " {'abbreviation': 'R_R00940_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['iRS1563: R_R00940_c',\n",
       "   'iRS1597: R00940_c',\n",
       "   'Name: 5,6,7,8-Tetrahydrofolate:NADP+ oxidoreductase; R_5_6_7_8_Tetrahydrofolate_NADP_oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00087[0] <=> (1) cpd00005[0] + (1) cpd00393[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00087;cpd00393',\n",
       "  'definition': '(1) NADP[0] + (1) Tetrahydrofolate[0] <= (1) NADPH[0] + (3) H+[0] + (1) Folate[0]',\n",
       "  'deltag': -5.75,\n",
       "  'deltagerr': 0.92,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.5.1.3'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00087[0] <= (1) cpd00005[0] + (3) cpd00067[0] + (1) cpd00393[0]',\n",
       "  'id': 'rxn00687',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R_5_6_7_8_Tetrahydrofolate_NADP__oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'CI:2',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00005:0:0:\"NADPH\";3:cpd00067:0:0:\"H+\";1:cpd00393:0:0:\"Folate\"'},\n",
       " {'abbreviation': 'R00941',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_FTHFO',\n",
       "   'KEGG: R00941',\n",
       "   'MetaCyc: FORMYLTETRAHYDROFOLATE-DEHYDROGENASE-RXN',\n",
       "   'iRS1563: R_R00941_c',\n",
       "   'Name: 10-formyl tetrahydrofolate:NADP oxidoreductase; 10-formyl-H2PtGlu:NADP oxidoreductase; 10-formyl-H4folate dehydrogenase; 10-formyltetrahydrofolate dehydrogenase; 10-formyltetrahydrofolate:NADP+ oxidoreductase; N10-formyltetrahydrofolate dehydrogenase; formyltetrahydrofolate dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00201[0] <=> (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00087[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00011;cpd00067;cpd00087;cpd00201',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) 10-Formyltetrahydrofolate[0] => (1) NADPH[0] + (1) CO2[0] + (1) H+[0] + (1) Tetrahydrofolate[0]',\n",
       "  'deltag': -7.8,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00201[0] => (1) cpd00005[0] + (1) cpd00011[0] + (1) cpd00067[0] + (1) cpd00087[0]',\n",
       "  'id': 'rxn00688',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '10-formyltetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00670 (One carbon pool by folate)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00201:0:0:\"10-Formyltetrahydrofolate\";1:cpd00005:0:0:\"NADPH\";1:cpd00011:0:0:\"CO2\";1:cpd00067:0:0:\"H+\";1:cpd00087:0:0:\"Tetrahydrofolate\"'},\n",
       " {'abbreviation': 'R00942',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-6102',\n",
       "   'BiGG: FOL3; MET7; RMA1; THFGLUS',\n",
       "   'BrachyCyc: RXN-6102',\n",
       "   'ChlamyCyc: FOLYLPOLYGLUTAMATESYNTH-RXN; RXN-6102',\n",
       "   'KEGG: R00942',\n",
       "   'Name: Folylpolyglutamate synthase; Tetrahydrofolate:L-glutamate gamma-ligase (ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00023[0] + (1) cpd00087[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd06227[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00023;cpd00067;cpd00087;cpd06227',\n",
       "  'definition': '(1) ATP[0] + (1) L-Glutamate[0] + (1) Tetrahydrofolate[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) THF-L-glutamate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.17'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00023[0] + (1) cpd00087[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd06227[0]',\n",
       "  'id': 'rxn00689',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30129',\n",
       "  'name': 'Tetrahydrofolate:L-glutamate gamma-ligase (ADP-forming)',\n",
       "  'notes': ['EQC', 'EQU', 'GCP'],\n",
       "  'pathways': ['KEGG: rn00790 (Folate biosynthesis)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:C:5/H:6/N:1/O:3|CI:-1',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00023:0:0:\"L-Glutamate\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd06227:0:0:\"THF-L-glutamate\"'},\n",
       " {'abbreviation': 'R00943',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FORMATETHFLIG-RXN',\n",
       "   'BiGG: ADE3_2; FTHFL; FTHFLi; FTHFLm; FTHFLmi; MIS1_2',\n",
       "   'BrachyCyc: FORMATETHFLIG-RXN',\n",
       "   'KEGG: R00943',\n",
       "   'MetaCyc: FORMATETHFLIG-RXN',\n",
       "   'Name: 10-formyltetrahydrofolate synthetase; Formate--tetrahydrofolate ligase; Formate:tetrahydrofolate ligase (ADP-forming); Formyltetrahydrofolate synthetase; formate-tetrahydrofolate ligase; formate-tetrahydrofolateligase; formate:tetrahydrofolate ligase (ADP-forming); formyltetrahydrofolate synthetase; tetrahydrofolate formylase; tetrahydrofolic formylase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00047[0] + (1) cpd00087[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00201[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00047;cpd00087;cpd00201',\n",
       "  'definition': '(1) ATP[0] + (1) Formate[0] + (1) Tetrahydrofolate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) 10-Formyltetrahydrofolate[0]',\n",
       "  'deltag': -1.89,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.3.4.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00047[0] + (1) cpd00087[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00201[0]',\n",
       "  'id': 'rxn00690',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30159;rxn31923',\n",
       "  'name': 'Formate:tetrahydrofolate ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); CODH-PWY (reductive acetyl coenzyme A pathway I (homoacetogenic bacteria)); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); PWY-1722 (formate assimilation into 5,10-methylenetetrahydrofolate); PWY-2161 (folate polyglutamylation); PWY-2201 (folate transformations I); PWY-3841 (folate transformations II); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00670 (One carbon pool by folate); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00047:0:0:\"Formate\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00201:0:0:\"10-Formyltetrahydrofolate\"'},\n",
       " {'abbreviation': 'R00944',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FORMYLTHFDEFORMYL-RXN',\n",
       "   'BiGG: FTHFD',\n",
       "   'BrachyCyc: FORMYLTHFDEFORMYL-RXN',\n",
       "   'KEGG: R00944',\n",
       "   'MetaCyc: FORMYLTHFDEFORMYL-RXN',\n",
       "   'Name: 10-Formyltetrahydrofolate amidohydrolase; 10-formyltetrahydrofolate amidohydrolase; Formyl-FH(4) hydrolase; Formyltetrahydrofolate deformylase; Formyltetrahydrofolate hydrolase; formyltetrahydrofolate deformylase; formyltetrahydrofolatedeformylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00201[0] <=> (1) cpd00047[0] + (1) cpd00087[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00047;cpd00067;cpd00087;cpd00201',\n",
       "  'definition': '(1) H2O[0] + (1) 10-Formyltetrahydrofolate[0] => (1) Formate[0] + (1) H+[0] + (1) Tetrahydrofolate[0]',\n",
       "  'deltag': -4.27,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00201[0] => (1) cpd00047[0] + (1) cpd00067[0] + (1) cpd00087[0]',\n",
       "  'id': 'rxn00691',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32191;rxn32194',\n",
       "  'name': '10-Formyltetrahydrofolate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); Formaldehyde-Oxidation (Formaldehyde Oxidation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); P164-PWY (purine nucleobases degradation I (anaerobic)); PWY-2201 (folate transformations I); PWY-5497 (purine nucleobases degradation II (anaerobic)); PWY-7909 (formaldehyde oxidation VII (THF pathway)); Purine-Degradation (Purine Nucleotide Degradation); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn00670 (One carbon pool by folate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00201:0:0:\"10-Formyltetrahydrofolate\";1:cpd00047:0:0:\"Formate\";1:cpd00067:0:0:\"H+\";1:cpd00087:0:0:\"Tetrahydrofolate\"'},\n",
       " {'abbreviation': 'R00945',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYOHMETRANS-RXN',\n",
       "   'BiGG: GHMT; GHMT2; GHMT2r; GHMT2rm; SHM1; SHM2',\n",
       "   'BrachyCyc: GLYOHMETRANS-RXN',\n",
       "   'KEGG: R00945',\n",
       "   'MetaCyc: GLYOHMETRANS-RXN',\n",
       "   'Name: 5,10-Methylenetetrahydrofolate:glycine hydroxymethyltransferase; 5,10-methylenetetrahydrofolate:glycine hydroxymethyltransferase; Glycine Hydroxymethyltransferase; L-serine hydroxymethyltransferase; L-threonine aldolase; Serine hydroxymethyltransferase; allothreonine aldolase; glycine hydroxymethyltransferase; glycine hydroxymethyltransferase, reversible; glycinehydroxymethyltransferase-reversible; serine aldolase; serine hydroxymethylase; serine hydroxymethyltransferase; serine transhydroxymethylase; threonine aldolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00033[0] + (1) cpd00125[0] <=> (1) cpd00054[0] + (1) cpd00087[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00033;cpd00054;cpd00087;cpd00125',\n",
       "  'definition': '(1) H2O[0] + (1) Glycine[0] + (1) 5-10-Methylenetetrahydrofolate[0] <=> (1) L-Serine[0] + (1) Tetrahydrofolate[0]',\n",
       "  'deltag': 2.48,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00033[0] + (1) cpd00125[0] <=> (1) cpd00054[0] + (1) cpd00087[0]',\n",
       "  'id': 'rxn00692',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19870;rxn27914;rxn30686;rxn33977;rxn38130',\n",
       "  'name': '5,10-Methylenetetrahydrofolate:glycine hydroxymethyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 1CMET2-PWY (<i>N</i><sup>10</sup>-formyl-tetrahydrofolate biosynthesis); AMINE-DEG (Amine and Polyamine Degradation); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); Formaldehyde-Assimilation (Formaldehyde Assimilation); GLYCINE-SYN (Glycine Biosynthesis); GLYSYN-PWY (glycine biosynthesis I); Glycine-Betaine-Degradation (Glycine Betaine Degradation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-181 (photorespiration); PWY-2161 (folate polyglutamylation); PWY-2201 (folate transformations I); PWY-3661 (glycine betaine degradation I); PWY-3661-1 (glycine betaine degradation II (mammalian)); PWY-3841 (folate transformations II); PWY-5497 (purine nucleobases degradation II (anaerobic)); Photosynthesis (); Purine-Degradation (Purine Nucleotide Degradation); SER-GLYSYN-PWY (superpathway of L-serine and glycine biosynthesis I); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00460 (Cyanoamino acid metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00670 (One carbon pool by folate); rn00680 (Methane metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00033:0:0:\"Glycine\";-1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\";1:cpd00054:0:0:\"L-Serine\";1:cpd00087:0:0:\"Tetrahydrofolate\"'},\n",
       " {'abbreviation': 'R00946',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HOMOCYSMETB12-RXN',\n",
       "   'BiGG: METS; METSMHPGLUT_b; METSr; MTHFB12D; U47',\n",
       "   'BrachyCyc: HOMOCYSMETB12-RXN',\n",
       "   'KEGG: R00946',\n",
       "   'MetaCyc: HOMOCYSMETB12-RXN',\n",
       "   'Name: 5-Methyltetrahydrofolate:L-homocysteine S-methyltransferase; 5-methionine synthase; 5-methyltetrahydrofolate--homocysteine S-methyltransferase; 5-methyltetrahydrofolate--homocysteine s-methyltransferase; 5-methyltetrahydrofolate-homocysteine S-methyltransferase; 5-methyltetrahydrofolate-homocysteine transmethylase; 5-methyltetrahydrofolate:L-homocysteine S-methyltransferase; B12 N5-methyltetrahydrofolate homocysteine methyltransferase; MetH; N-methyltetrahydrofolate:L-homocysteine methyltransferase; N5-methyltetrahydrofolate methyltransferase; N5-methyltetrahydrofolate-homocysteine cobalamin methyltransferase; N5-methyltetrahydrofolic-homocysteine vitamin B12 transmethylase; cobalamin-dependent methionine synthase; methionine synthase; methionine synthase (cobalamin-dependent); methionine synthase reversible; methyltetrahydrofolate-homocysteine vitamin B12 methyltransferase; tetrahydrofolate methyltransferase; tetrahydropteroylglutamate methyltransferase; tetrahydropteroylglutamic methyltransferase; vitamin B12 methyltransferase'],\n",
       "  'code': '(1) cpd00135[0] + (1) cpd00345[0] <=> (1) cpd00060[0] + (1) cpd00087[0]',\n",
       "  'compound_ids': 'cpd00060;cpd00087;cpd00135;cpd00345',\n",
       "  'definition': '(1) Homocysteine[0] + (1) 5-Methyltetrahydrofolate[0] <=> (1) L-Methionine[0] + (1) Tetrahydrofolate[0]',\n",
       "  'deltag': -1.81,\n",
       "  'deltagerr': 1.19,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.13'],\n",
       "  'equation': '(1) cpd00135[0] + (1) cpd00345[0] <=> (1) cpd00060[0] + (1) cpd00087[0]',\n",
       "  'id': 'rxn00693',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32052;rxn35345',\n",
       "  'name': '5-Methyltetrahydrofolate:L-homocysteine S-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 1CMET2-PWY (<i>N</i><sup>10</sup>-formyl-tetrahydrofolate biosynthesis); ADENOSYLHOMOCYSCAT-PWY (L-methionine salvage from L-homocysteine); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); HOMOSER-METSYN-PWY (L-methionine biosynthesis I); HSERMETANA-PWY (L-methionine biosynthesis III); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); MET-SAM-PWY (superpathway of <i>S</i>-adenosyl-L-methionine biosynthesis); METHIONINE-DEG (L-methionine Degradation); METHIONINE-SYN (L-methionine Biosynthesis); METSYN-PWY (superpathway of L-homoserine and L-methionine biosynthesis); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); Methionine-Salvage (L-methionine Salvage); P4-PWY (superpathway of L-lysine, L-threonine and L-methionine biosynthesis I); PWY-2201 (folate transformations I); PWY-3841 (folate transformations II); PWY-5328 (superpathway of L-methionine salvage and degradation); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-5347 (superpathway of L-methionine biosynthesis (transsulfuration)); PWY-7977 (L-methionine biosynthesis IV (archaea)); PWY0-781 (aspartate superpathway); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism); rn00670 (One carbon pool by folate); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00135:0:0:\"Homocysteine\";-1:cpd00345:0:0:\"5-Methyltetrahydrofolate\";1:cpd00060:0:0:\"L-Methionine\";1:cpd00087:0:0:\"Tetrahydrofolate\"'},\n",
       " {'abbreviation': 'R00948',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLUC1PADENYLTRANS-RXN',\n",
       "   'BiGG: GLGC; GLGCr',\n",
       "   'BrachyCyc: GLUC1PADENYLTRANS-RXN',\n",
       "   'KEGG: R00948',\n",
       "   'MetaCyc: GLUC1PADENYLTRANS-RXN; RXN-11988',\n",
       "   'Name: ADP glucose pyrophosphorylase; ADP-glucose pyrophosphorylase; ADP-glucose synthase; ADP-glucose synthetase; ADP:alpha-D-glucose-1-phosphate adenylyltransferase; ADPG pyrophosphorylase; ATP:alpha-D-glucose-1-phosphate adenyltransferase; ATP:alpha-D-glucose-1-phosphate adenylyltransferase; adenosine diphosphate glucose pyrophosphorylase; adenosine diphosphoglucose pyrophosphorylase; glucose 1-phosphate adenylyltransferase; glucose-1-phosphate adenylyltransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00387[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00089;cpd00387',\n",
       "  'definition': '(1) ATP[0] + (1) Glucose-1-phosphate[0] <=> (1) PPi[0] + (1) ADPglucose[0]',\n",
       "  'deltag': -2.78,\n",
       "  'deltagerr': 0.43,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.27'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00387[0]',\n",
       "  'id': 'rxn00695',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19776',\n",
       "  'name': 'ATP:alpha-D-glucose-1-phosphate adenylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); GLYCOGEN-BIOSYN (Glycogen and Starch Biosynthesis); GLYCOGENSYNTH-PWY (glycogen biosynthesis I (from ADP-D-Glucose)); Glycan-Biosynthesis (Glycan Biosynthesis); Glycan-Pathways (Glycan Pathways); Glycolipids-Biosynthesis (Glycolipid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); PWY-622 (starch biosynthesis); PWY-7902 (glucosylglycerol biosynthesis); Polysaccharides-Biosynthesis (Polysaccharide Biosynthesis)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00387:0:0:\"ADPglucose\"'},\n",
       " {'abbreviation': 'R00949',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00949',\n",
       "   'MetaCyc: PHOSPHOGLUCOKINASE-RXN',\n",
       "   'Name: ATP:D-glucose-1-phosphate 6-phosphotransferase; ATP:alpha-D-glucose-1-phosphate 6-phosphotransferase; glucose-phosphate kinase; phosphoglucokinase; phosphoglucokinase (phosphorylating)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00089[0] <=> (1) cpd00008[0] + (1) cpd00907[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00089;cpd00907',\n",
       "  'definition': '(1) ATP[0] + (1) H+[0] + (1) Glucose-1-phosphate[0] => (1) ADP[0] + (1) alpha-D-glucose-1,6-biphosphate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.10'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00067[0] + (1) cpd00089[0] => (1) cpd00008[0] + (1) cpd00907[0]',\n",
       "  'id': 'rxn00696',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:D-glucose-1-phosphate 6-phosphotransferase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00067:0:0:\"H+\";-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00008:0:0:\"ADP\";1:cpd00907:0:0:\"alpha-D-glucose-1,6-biphosphate\"'},\n",
       " {'abbreviation': 'R00951',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADPSUGPPHOSPHAT-RXN; RXN-10770',\n",
       "   'BrachyCyc: ADPSUGPPHOSPHAT-RXN',\n",
       "   'CornCyc: ADPSUGPPHOSPHAT-RXN',\n",
       "   'KEGG: R00951',\n",
       "   'MetaCyc: ADPSUGPPHOSPHAT-RXN; RXN-10770',\n",
       "   'Name: ADP-glucose Glucose-1-phosphohydrolase; ADP-glucose glucose-1-phosphohydrolase; ADPglucose sugarphosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00387[0] <=> (1) cpd00018[0] + (1) cpd00089[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00018;cpd00067;cpd00089;cpd00387',\n",
       "  'definition': '(1) H2O[0] + (1) ADPglucose[0] => (1) AMP[0] + (2) H+[0] + (1) Glucose-1-phosphate[0]',\n",
       "  'deltag': -5.65,\n",
       "  'deltagerr': 0.43,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00387[0] => (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd00089[0]',\n",
       "  'id': 'rxn00697',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33331',\n",
       "  'name': 'ADP-glucose Glucose-1-phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00387:0:0:\"ADPglucose\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd00089:0:0:\"Glucose-1-phosphate\"'},\n",
       " {'abbreviation': 'R00952',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00952; R06056',\n",
       "   'MetaCyc: CELLOBIOSE-PHOSPHORYLASE-RXN',\n",
       "   'Name: Cellobiose:orthophosphate alpha-D-glucosyltransferase; cellobiose:phosphate alpha-D-glucosyltransferase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00158[0] <=> (1) cpd00027[0] + (1) cpd00089[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00027;cpd00089;cpd00158',\n",
       "  'definition': '(1) Phosphate[0] + (1) CELB[0] => (1) D-Glucose[0] + (1) Glucose-1-phosphate[0]',\n",
       "  'deltag': 1.42,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.4.1.20'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00158[0] => (1) cpd00027[0] + (1) cpd00089[0]',\n",
       "  'id': 'rxn00698',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'cellobiose:phosphate alpha-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00158:0:0:\"CELB\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00089:0:0:\"Glucose-1-phosphate\"'},\n",
       " {'abbreviation': 'R00953',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00953; R06058',\n",
       "   'MetaCyc: LAMINARIBIOSE-PHOSPHORYLASE-RXN',\n",
       "   'Name: 3-beta-D-Glucosylglucose:orthophosphate alpha-D-glucosyltransferase; 3-beta-D-glucosyl-D-glucose:phosphate alpha-D-glucosyltransferase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd01397[0] <=> (1) cpd00027[0] + (1) cpd00089[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00027;cpd00089;cpd01397',\n",
       "  'definition': '(1) Phosphate[0] + (1) Laminariaceae[0] => (1) D-Glucose[0] + (1) Glucose-1-phosphate[0]',\n",
       "  'deltag': 1.52,\n",
       "  'deltagerr': 0.77,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.4.1.31'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd01397[0] => (1) cpd00027[0] + (1) cpd00089[0]',\n",
       "  'id': 'rxn00699',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-beta-D-glucosyl-D-glucose:phosphate alpha-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-6778 (laminaribiose degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd01397:0:0:\"Laminariaceae\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00089:0:0:\"Glucose-1-phosphate\"'},\n",
       " {'abbreviation': 'R00954',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.7.34-RXN',\n",
       "   'BrachyCyc: 2.7.7.34-RXN',\n",
       "   'CornCyc: 2.7.7.34-RXN',\n",
       "   'KEGG: R00954',\n",
       "   'MetaCyc: 2.7.7.34-RXN; RXN-11988',\n",
       "   'Name: GDP glucose pyrophosphorylase; GTP:alpha-D-glucose-1-phosphate guanylyltransferase; Glucose-1-phosphate guanylyltransferase; glucose-1-phosphate guanylyltransferase; guanosine diphosphoglucose pyrophosphorylase'],\n",
       "  'code': '(1) cpd00038[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00315[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00038;cpd00089;cpd00315',\n",
       "  'definition': '(1) GTP[0] + (1) Glucose-1-phosphate[0] <=> (1) PPi[0] + (1) GDP-glucose[0]',\n",
       "  'deltag': -0.4,\n",
       "  'deltagerr': 0.77,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.34'],\n",
       "  'equation': '(1) cpd00038[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00315[0]',\n",
       "  'id': 'rxn00700',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32349;rxn32351;rxn32352;rxn32355',\n",
       "  'name': 'GTP:alpha-D-glucose-1-phosphate guanylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); GDP-Sugar-Biosynthesis (GDP-sugar Biosynthesis); PWY-5661 (GDP-glucose biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00038:0:0:\"GTP\";-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00315:0:0:\"GDP-glucose\"'},\n",
       " {'abbreviation': 'R00955',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GAL7_2; UGLT',\n",
       "   'KEGG: R00955',\n",
       "   'MaizeCyc: GALACTURIDYLYLTRANS-RXN',\n",
       "   'MetaCyc: GALACTURIDYLYLTRANS-RXN',\n",
       "   'PlantCyc: GALACTURIDYLYLTRANS-RXN',\n",
       "   'Name: UDP-glucose-hexose-1-phosphate uridylyltransferase; UDP-glucose:alpha-D-galactose-1-phosphate uridylyltransferase; UDPglucose--hexose-1-phosphate uridylyltransferase; UDPglucose:alpha-D-galactose-1-phosphate uridylyltransferase; hexose 1-phosphate uridyltransferase; hexose-1-phosphate uridylyltransferase; uridyl transferase; uridyltransferase'],\n",
       "  'code': '(1) cpd00026[0] + (1) cpd00348[0] <=> (1) cpd00043[0] + (1) cpd00089[0]',\n",
       "  'compound_ids': 'cpd00026;cpd00043;cpd00089;cpd00348',\n",
       "  'definition': '(1) UDP-glucose[0] + (1) D-Galactose 1-phosphate[0] <=> (1) UDP-galactose[0] + (1) Glucose-1-phosphate[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 4.9,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.12'],\n",
       "  'equation': '(1) cpd00026[0] + (1) cpd00348[0] <=> (1) cpd00043[0] + (1) cpd00089[0]',\n",
       "  'id': 'rxn00701',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-glucose:alpha-D-galactose-1-phosphate uridylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); GALACTOSE-DEGRADATION (Galactose Degradation); PWY-6317 (D-galactose degradation I (Leloir pathway)); PWY-6527 (stachyose degradation); PWY66-422 (D-galactose degradation V (Leloir pathway)); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00052 (Galactose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";-1:cpd00348:0:0:\"D-Galactose 1-phosphate\";1:cpd00043:0:0:\"UDP-galactose\";1:cpd00089:0:0:\"Glucose-1-phosphate\"'},\n",
       " {'abbreviation': 'R00956',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CDPGLU',\n",
       "   'KEGG: R00956',\n",
       "   'MetaCyc: 2.7.7.33-RXN; RXN-11988',\n",
       "   'iRR1083: CDPGLU',\n",
       "   'Name: CDP glucose pyrophosphorylase; CTP:D-glucose-1-phosphate cytidylyltransferase; CTP:alpha-D-glucose-1-phosphate cytidylyltransferase; cytidine diphosphate glucose pyrophosphorylase; cytidine diphosphate-D-glucose pyrophosphorylase; cytidine diphosphoglucose pyrophosphorylase; glucose-1-phosphate cytidylyltransferase'],\n",
       "  'code': '(1) cpd00052[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00390[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00052;cpd00089;cpd00390',\n",
       "  'definition': '(1) CTP[0] + (1) Glucose-1-phosphate[0] <=> (1) PPi[0] + (1) CDP-glucose[0]',\n",
       "  'deltag': 1.07,\n",
       "  'deltagerr': 4.01,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.33'],\n",
       "  'equation': '(1) cpd00052[0] + (1) cpd00089[0] <=> (1) cpd00012[0] + (1) cpd00390[0]',\n",
       "  'id': 'rxn00702',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'CTP:alpha-D-glucose-1-phosphate cytidylyltransferase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CDP-Sugar-Biosynthesis (CDP-sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); PWY-5823 (superpathway of CDP-glucose-derived O-antigen building blocks biosynthesis); PWY-5833 (CDP-4-dehydro-3,6-dideoxy-D-glucose biosynthesis); PWY-5940 (streptomycin biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00052:0:0:\"CTP\";-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00390:0:0:\"CDP-glucose\"'},\n",
       " {'abbreviation': 'R00957',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00957; R06061',\n",
       "   'MetaCyc: MALTOSE-SYNTHASE-RXN',\n",
       "   'iRS1563: R_R06061_c',\n",
       "   'Name: 4-alpha-D-glucosyltransferase (dephosphorylating); alpha-D-Glucose-1-phosphate:alpha-D-glucose-1-phosphate; alpha-D-Glucose-1-phosphate:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (dephosphorylating); alpha-D-glucose-1-phosphate:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase; alpha-D-glucose-1-phosphate:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (dephosphorylating); maltose synthase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00089[0] <=> (2) cpd00009[0] + (1) cpd00179[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00089;cpd00179',\n",
       "  'definition': '(1) H2O[0] + (2) Glucose-1-phosphate[0] <= (2) Phosphate[0] + (1) Maltose[0]',\n",
       "  'deltag': -6.25,\n",
       "  'deltagerr': 0.68,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.1.139'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00089[0] <= (2) cpd00009[0] + (1) cpd00179[0]',\n",
       "  'id': 'rxn00703',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn15103;rxn15568',\n",
       "  'name': 'alpha-D-Glucose-1-phosphate:alpha-D-glucose-1-phosphate 4-alpha-D-glucosyltransferase (dephosphorylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00089:0:0:\"Glucose-1-phosphate\";2:cpd00009:0:0:\"Phosphate\";1:cpd00179:0:0:\"Maltose\"'},\n",
       " {'abbreviation': 'R00959',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHOSPHOGLUCMUT-RXN',\n",
       "   'BiGG: PGM1_2; PGM2_2; PGMT',\n",
       "   'BrachyCyc: PHOSPHOGLUCMUT-RXN',\n",
       "   'KEGG: R00959; R08639',\n",
       "   'MetaCyc: 5.4.2.5-RXN; PHOSPHOGLUCMUT-RXN; RXN-16999',\n",
       "   'Name: Phosphoglucomutase; alpha-D-Glucose 1-phosphate 1,6-phosphomutase; alpha-D-glucose 1,6-phosphomutase; glucose phosphomutase; phosphoglucose mutase'],\n",
       "  'code': '(1) cpd00089[0] <=> (1) cpd00079[0]',\n",
       "  'compound_ids': 'cpd00079;cpd00089',\n",
       "  'definition': '(1) Glucose-1-phosphate[0] <=> (1) D-glucose-6-phosphate[0]',\n",
       "  'deltag': -2.72,\n",
       "  'deltagerr': 0.31,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.2.2', '5.4.2.5'],\n",
       "  'equation': '(1) cpd00089[0] <=> (1) cpd00079[0]',\n",
       "  'id': 'rxn00704',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn15773;rxn33867',\n",
       "  'name': 'alpha-D-Glucose 1-phosphate 1,6-phosphomutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Ascorbate-Biosynthesis (Ascorbate Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); COLANSYN-PWY (colanic acid building blocks biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); GALACTOSE-DEGRADATION (Galactose Degradation); GDP-Sugar-Biosynthesis (GDP-sugar Biosynthesis); GLUCOSE1PMETAB-PWY (glucose and glucose-1-phosphate degradation); GLYCOCAT-PWY (glycogen degradation I); GLYCOGEN-BIOSYN (Glycogen and Starch Biosynthesis); GLYCOGENSYNTH-PWY (glycogen biosynthesis I (from ADP-D-Glucose)); Glycan-Biosynthesis (Glycan Biosynthesis); Glycan-Pathways (Glycan Pathways); Glycogen-Degradation (Glycogen Degradation); Glycolipids-Biosynthesis (Glycolipid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); POLYSACCHARIDES-DEG (Polysaccharide Degradation); PWY-2723 (trehalose degradation V); PWY-3801 (sucrose degradation II (sucrose synthase)); PWY-5384 (sucrose degradation IV (sucrose phosphorylase)); PWY-5661 (GDP-glucose biosynthesis); PWY-5940 (streptomycin biosynthesis); PWY-5941 (glycogen degradation II); PWY-622 (starch biosynthesis); PWY-6317 (D-galactose degradation I (Leloir pathway)); PWY-6415 (L-ascorbate biosynthesis V); PWY-6731 (starch degradation III); PWY-6737 (starch degradation V); PWY-6981 (chitin biosynthesis); PWY-7238 (sucrose biosynthesis II); PWY-7328 (superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis); PWY-7343 (UDP-&alpha;-D-glucose biosynthesis I); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY-7900 (glycogen biosynthesis III (from &alpha;-maltose 1-phosphate)); PWY-7902 (glucosylglycerol biosynthesis); PWY66-422 (D-galactose degradation V (Leloir pathway)); Polymer-Degradation (Polymeric Compound Degradation); Polysaccharides-Biosynthesis (Polysaccharide Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SUCROSE-DEG (Sucrose Degradation); SUCSYN-PWY (sucrose biosynthesis I (from photosynthesis)); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Starch-Degradation (Starch Degradation); Sucrose-Biosynthesis (Sucrose Biosynthesis); Sugars-And-Polysaccharides-Degradation (Sugar Degradation); Trehalose-Degradation (Trehalose Degradation); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00052 (Galactose metabolism); rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn00521 (Streptomycin biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00079:0:0:\"D-glucose-6-phosphate\"'},\n",
       " {'abbreviation': 'R00960',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00960',\n",
       "   'iAG612: rbs_15',\n",
       "   'Name: D-Glucose-1-phosphate:D-glucose-1-phosphate 6-phosphotransferase'],\n",
       "  'code': '(2) cpd00089[0] <=> (1) cpd00027[0] + (1) cpd00907[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00067;cpd00089;cpd00907',\n",
       "  'definition': '(2) H+[0] + (2) Glucose-1-phosphate[0] <=> (1) D-Glucose[0] + (1) alpha-D-glucose-1,6-biphosphate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.41'],\n",
       "  'equation': '(2) cpd00067[0] + (2) cpd00089[0] <=> (1) cpd00027[0] + (1) cpd00907[0]',\n",
       "  'id': 'rxn00705',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Glucose-1-phosphate:D-glucose-1-phosphate 6-phosphotransferase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00067:0:0:\"H+\";-2:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00907:0:0:\"alpha-D-glucose-1,6-biphosphate\"'},\n",
       " {'abbreviation': 'R00961',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00961_c',\n",
       "   'AraGEM: R_R00961_c',\n",
       "   'EcoCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN',\n",
       "   'KEGG: R00961',\n",
       "   'MetaCyc: NUCLEOSIDE-DIPHOSPHATASE-RXN; RXN-14003',\n",
       "   \"Name: ADPase; CDPase; GDPase; IDP diphosphohydrolase; IDP phosphohydrolase; IDPase; NDPase; UDPase; adenosine diphosphatase; adenosinepyrophosphatase; guanosine 5'-diphosphatase; guanosine diphosphatase; inosine 5'-diphosphatase; inosine diphosphatase; inosine diphosphate phosphatase; nucleoside 5'-diphosphatase; nucleoside diphosphate phosphatase; nucleoside diphosphate phosphohydrolase; nucleoside-diphosphatase; nucleoside-diphosphate phosphohydrolase; thiaminpyrophosphatase; type B nucleoside diphosphatase; type L nucleoside diphosphatase; uridine 5'-diphosphatase; uridine diphosphatase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00090[0] <=> (1) cpd00009[0] + (1) cpd00114[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00067;cpd00090;cpd00114',\n",
       "  'definition': '(1) H2O[0] + (1) IDP[0] => (1) Phosphate[0] + (2) H+[0] + (1) IMP[0]',\n",
       "  'deltag': 1.32,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.5', '3.6.1.6', '3.6.1.64'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00090[0] => (1) cpd00009[0] + (2) cpd00067[0] + (1) cpd00114[0]',\n",
       "  'id': 'rxn00706',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33662;rxn33663',\n",
       "  'name': 'IDP phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00090:0:0:\"IDP\";1:cpd00009:0:0:\"Phosphate\";2:cpd00067:0:0:\"H+\";1:cpd00114:0:0:\"IMP\"'},\n",
       " {'abbreviation': 'R00962',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00962_c',\n",
       "   'AraGEM: R_R00962_c',\n",
       "   'BiGG: CYTDK3',\n",
       "   'JP_Creinhardtii_MSB: R_ITCY',\n",
       "   'KEGG: R00962',\n",
       "   \"Name: ITP:cytidine 5'-phosphotransferase; cytidine kinase (ITP)\"],\n",
       "  'code': '(1) cpd00068[0] + (1) cpd00367[0] <=> (1) cpd00046[0] + (1) cpd00090[0]',\n",
       "  'compound_ids': 'cpd00046;cpd00067;cpd00068;cpd00090;cpd00367',\n",
       "  'definition': '(1) ITP[0] + (1) Cytidine[0] => (1) CMP[0] + (1) H+[0] + (1) IDP[0]',\n",
       "  'deltag': -3.69,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.48'],\n",
       "  'equation': '(1) cpd00068[0] + (1) cpd00367[0] => (1) cpd00046[0] + (1) cpd00067[0] + (1) cpd00090[0]',\n",
       "  'id': 'rxn00707',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"ITP:cytidine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00068:0:0:\"ITP\";-1:cpd00367:0:0:\"Cytidine\";1:cpd00046:0:0:\"CMP\";1:cpd00067:0:0:\"H+\";1:cpd00090:0:0:\"IDP\"'},\n",
       " {'abbreviation': 'R00963',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5-NUCLEOTID-RXN; RXNQT-4150',\n",
       "   'BiGG: NTD2; NTD2pp; U34',\n",
       "   'ChlamyCyc: 5-NUCLEOTID-RXN',\n",
       "   'KEGG: R00963',\n",
       "   'MetaCyc: 5-NUCLEOTID-RXN; RXN-14025',\n",
       "   \"Name: 5'-AMP nucleotidase; 5'-AMPase; 5'-adenylic phosphatase; 5'-mononucleotidase; 5'-nucleotidase; 5'-nucleotidase (UMP); 5'-ribonucleotide phosphohydrolase; AMP phosphatase; AMP phosphohydrolase; AMPase; IMP 5'-nucleotidase; UMPase; Uridine 5'-monophosphate phosphohydrolase; adenosine 5'-phosphatase; adenosine monophosphatase; snake venom 5'-nucleotidase; thimidine monophosphate nucleotidase; ump nucleotidase; uridine 5'-nucleotidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00091[0] <=> (1) cpd00009[0] + (1) cpd00249[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00091;cpd00249',\n",
       "  'definition': '(1) H2O[0] + (1) UMP[0] <=> (1) Phosphate[0] + (1) Uridine[0]',\n",
       "  'deltag': -2.05,\n",
       "  'deltagerr': 1.01,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00091[0] <=> (1) cpd00009[0] + (1) cpd00249[0]',\n",
       "  'id': 'rxn00708',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18975;rxn31059',\n",
       "  'name': \"Uridine 5'-monophosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-7185 (UTP and CTP dephosphorylation I); PWY-7821 (tunicamycin biosynthesis); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); UTP-CTP-Dephosphorylation (UTP and CTP Dephosphorylation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00091:0:0:\"UMP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00249:0:0:\"Uridine\"'},\n",
       " {'abbreviation': 'R00964',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: URIDINEKIN-RXN',\n",
       "   'BiGG: URIK1; URK1_3',\n",
       "   'BrachyCyc: URIDINEKIN-RXN',\n",
       "   'KEGG: R00964',\n",
       "   'MetaCyc: URIDINEKIN-RXN',\n",
       "   \"Name: ATP:uridine 5'-phosphotransferase; Uridine kinase; pyrimidine ribonucleoside kinase; uridine kinase; uridine kinase (ATP:Uridine); uridine kinase (phosphorylating); uridine phosphokinase; uridine-cytidine kinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00249[0] <=> (1) cpd00008[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00091;cpd00249',\n",
       "  'definition': '(1) ATP[0] + (1) Uridine[0] => (1) ADP[0] + (1) H+[0] + (1) UMP[0]',\n",
       "  'deltag': -4.11,\n",
       "  'deltagerr': 1.01,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.48'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00249[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00709',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33181;rxn38830;rxn38831',\n",
       "  'name': \"ATP:uridine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-7193 (pyrimidine ribonucleosides salvage I); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00249:0:0:\"Uridine\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00965',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: OROTPDECARB-RXN',\n",
       "   'BiGG: OMPDC; URA3',\n",
       "   'BrachyCyc: OROTPDECARB-RXN',\n",
       "   'KEGG: R00965',\n",
       "   'MetaCyc: OROTPDECARB-RXN',\n",
       "   \"Name: ODCase; OMP decarboxylase; OMP-DC; OMPdcase; Orotidine-5'-phosphate carboxy-lyase; Orotidine-5'-phosphate decarboxylase; Orotidylic decarboxylase; UMP synthase; Uridine 5'-monophosphate synthase; orotate decarboxylase; orotate monophosphate decarboxylase; orotic decarboxylase; orotidine 5'-phosphate decarboxylase; orotidine monophosphate decarboxylase; orotidine phosphate decarboxylase; orotidine-5'-monophosphate decarboxylase; orotidine-5'-phosphate carboxy-lyase; orotidine-5'-phosphate carboxy-lyase (UMP-forming); orotidine-5'-phosphate decarboxylase; orotidine-5-phosphatedecarboxylase; orotidine-phosphate decarboxylase; orotidylic acid decarboxylase; orotidylic decarboxylase; orotodylate decarboxylase\"],\n",
       "  'code': '(1) cpd00810[0] <=> (1) cpd00011[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00091;cpd00810',\n",
       "  'definition': '(1) H+[0] + (1) Orotidylic acid[0] => (1) CO2[0] + (1) UMP[0]',\n",
       "  'deltag': -6.26,\n",
       "  'deltagerr': 2.08,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.23'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00810[0] => (1) cpd00011[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00710',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28054;rxn38244',\n",
       "  'name': \"orotidine-5'-phosphate carboxy-lyase (UMP-forming)\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-5686 (UMP biosynthesis I); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY-7790 (UMP biosynthesis II); PWY-7791 (UMP biosynthesis III); PWY0-162 (superpathway of pyrimidine ribonucleotides <i>de novo</i> biosynthesis); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimid-Ribonucleot-De-Novo-Biosyn (Pyrimidine Ribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); UMP-Biosynthesis (UMP Biosynthesis)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00810:0:0:\"Orotidylic acid\";1:cpd00011:0:0:\"CO2\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00966',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: URACIL-PRIBOSYLTRANS-RXN',\n",
       "   'BiGG: FUR1; UPPRT; UPPRTr',\n",
       "   'BrachyCyc: URACIL-PRIBOSYLTRANS-RXN',\n",
       "   'KEGG: R00966',\n",
       "   'MetaCyc: URACIL-PRIBOSYLTRANS-RXN',\n",
       "   \"Name: UMP diphosphorylase; UMP pyrophosphorylase; UMP:diphosphate phospho-alpha-D-ribosyltransferase; UMP:pyrophosphate phosphoribosyltransferase; UPRTase; uracil phosphoribosyltransferase; uracil phosphoribosyltransferase (r); uracilphosphoribosyltransferase; uridine 5'-phosphate pyrophosphorylase; uridine monophosphate pyrophosphorylase; uridylate pyrophosphorylase; uridylic pyrophosphorylase\"],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00091[0] <=> (1) cpd00092[0] + (1) cpd00103[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00067;cpd00091;cpd00092;cpd00103',\n",
       "  'definition': '(1) PPi[0] + (1) H+[0] + (1) UMP[0] <= (1) Uracil[0] + (1) PRPP[0]',\n",
       "  'deltag': 3.32,\n",
       "  'deltagerr': 1.06,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.9'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00091[0] <= (1) cpd00092[0] + (1) cpd00103[0]',\n",
       "  'id': 'rxn00711',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29942',\n",
       "  'name': 'UMP:diphosphate phospho-alpha-D-ribosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-7183 (pyrimidine nucleobases salvage I); PWY-7194 (pyrimidine nucleobases salvage II); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PWY-7208 (superpathway of pyrimidine nucleobases salvage); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00067:0:0:\"H+\";-1:cpd00091:0:0:\"UMP\";1:cpd00092:0:0:\"Uracil\";1:cpd00103:0:0:\"PRPP\"'},\n",
       " {'abbreviation': 'R00967',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00967_c',\n",
       "   'AraGEM: R_R00967_c',\n",
       "   'JP_Creinhardtii_MSB: R_UTUP',\n",
       "   'KEGG: R00967',\n",
       "   'iRS1563: R_R00967_c',\n",
       "   \"Name: UTP:uridine 5'-phosphotransferase\"],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd00249[0] <=> (1) cpd00014[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00062;cpd00067;cpd00091;cpd00249',\n",
       "  'definition': '(1) UTP[0] + (1) Uridine[0] => (1) UDP[0] + (1) H+[0] + (1) UMP[0]',\n",
       "  'deltag': -3.43,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.48'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd00249[0] => (1) cpd00014[0] + (1) cpd00067[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00712',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"UTP:uridine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd00249:0:0:\"Uridine\";1:cpd00014:0:0:\"UDP\";1:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00968',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: URKI-RXN',\n",
       "   'BiGG: URIK2; URK1_1',\n",
       "   'BrachyCyc: URKI-RXN',\n",
       "   'KEGG: R00968',\n",
       "   'MetaCyc: URKI-RXN',\n",
       "   \"Name: GTP:uridine 5'-phosphotransferase; uridine kinase (GTP:Uridine)\"],\n",
       "  'code': '(1) cpd00038[0] + (1) cpd00249[0] <=> (1) cpd00031[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00031;cpd00038;cpd00067;cpd00091;cpd00249',\n",
       "  'definition': '(1) GTP[0] + (1) Uridine[0] => (1) GDP[0] + (1) H+[0] + (1) UMP[0]',\n",
       "  'deltag': -3.5,\n",
       "  'deltagerr': 1.06,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.48'],\n",
       "  'equation': '(1) cpd00038[0] + (1) cpd00249[0] => (1) cpd00031[0] + (1) cpd00067[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00713',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33201;rxn38832;rxn38833',\n",
       "  'name': \"GTP:uridine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00038:0:0:\"GTP\";-1:cpd00249:0:0:\"Uridine\";1:cpd00031:0:0:\"GDP\";1:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00969',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00969',\n",
       "   \"Name: P1,P4-bis(5'-uridyl)-tetraphosphate uridylohydrolase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03706[0] <=> (1) cpd00062[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00062;cpd00067;cpd00091;cpd03706',\n",
       "  'definition': '(1) H2O[0] + (1) UppppU[0] => (1) UTP[0] + (1) H+[0] + (1) UMP[0]',\n",
       "  'deltag': -8.33,\n",
       "  'deltagerr': 1.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.17'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03706[0] => (1) cpd00062[0] + (1) cpd00067[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00714',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"P1,P4-bis(5'-uridyl)-tetraphosphate uridylohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03706:0:0:\"UppppU\";1:cpd00062:0:0:\"UTP\";1:cpd00067:0:0:\"H+\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00970',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00970_c',\n",
       "   'AraGEM: R_R00970_c',\n",
       "   'BiGG: URIK3',\n",
       "   'JP_Creinhardtii_MSB: R_ITUP',\n",
       "   'KEGG: R00970',\n",
       "   \"Name: ITP:uridine 5'-phosphotransferase; uridine kinase (ITP:Uridine)\"],\n",
       "  'code': '(1) cpd00068[0] + (1) cpd00249[0] <=> (1) cpd00090[0] + (1) cpd00091[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00068;cpd00090;cpd00091;cpd00249',\n",
       "  'definition': '(1) ITP[0] + (1) Uridine[0] => (1) H+[0] + (1) IDP[0] + (1) UMP[0]',\n",
       "  'deltag': -4.97,\n",
       "  'deltagerr': 1.07,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.48'],\n",
       "  'equation': '(1) cpd00068[0] + (1) cpd00249[0] => (1) cpd00067[0] + (1) cpd00090[0] + (1) cpd00091[0]',\n",
       "  'id': 'rxn00715',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"ITP:uridine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00068:0:0:\"ITP\";-1:cpd00249:0:0:\"Uridine\";1:cpd00067:0:0:\"H+\";1:cpd00090:0:0:\"IDP\";1:cpd00091:0:0:\"UMP\"'},\n",
       " {'abbreviation': 'R00973',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00973',\n",
       "   'MetaCyc: URACIL-5-CARBOXYLATE-DECARBOXYLASE-RXN',\n",
       "   'TS_Athaliana: rxn617_c; rxn617_p',\n",
       "   'Name: Uracil-5-carboxylate carboxy-lyase; uracil-5-carboxylate carboxy-lyase; uracil-5-carboxylate carboxy-lyase (uracil-forming); uracil-5-carboxylate decarboxylase; uracil-5-carboxylic acid decarboxylase'],\n",
       "  'code': '(1) cpd01938[0] <=> (1) cpd00011[0] + (1) cpd00092[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00092;cpd01938',\n",
       "  'definition': '(1) H+[0] + (1) Uracil 5-carboxylate[0] => (1) CO2[0] + (1) Uracil[0]',\n",
       "  'deltag': -4.97,\n",
       "  'deltagerr': 0.94,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.66'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd01938[0] => (1) cpd00011[0] + (1) cpd00092[0]',\n",
       "  'id': 'rxn00716',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33735',\n",
       "  'name': 'uracil-5-carboxylate carboxy-lyase (uracil-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd01938:0:0:\"Uracil 5-carboxylate\";1:cpd00011:0:0:\"CO2\";1:cpd00092:0:0:\"Uracil\"'},\n",
       " {'abbreviation': 'R00974',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CYTDEAM-RXN',\n",
       "   'BiGG: CSND; FCY1',\n",
       "   'BrachyCyc: CYTDEAM-RXN',\n",
       "   'KEGG: R00974',\n",
       "   'MetaCyc: CYTDEAM-RXN',\n",
       "   'Name: Cytosine aminohydrolase; Cytosine deaminase; cytosine aminohydrolase; cytosine deaminase; isocytosine deaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00307[0] <=> (1) cpd00013[0] + (1) cpd00092[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00067;cpd00092;cpd00307',\n",
       "  'definition': '(1) H2O[0] + (1) H+[0] + (1) Cytosine[0] => (1) NH3[0] + (1) Uracil[0]',\n",
       "  'deltag': -7.27,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00067[0] + (1) cpd00307[0] => (1) cpd00013[0] + (1) cpd00092[0]',\n",
       "  'id': 'rxn00717',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29997',\n",
       "  'name': 'Cytosine aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-7194 (pyrimidine nucleobases salvage II); PWY-7195 (pyrimidine ribonucleosides salvage III); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00067:0:0:\"H+\";-1:cpd00307:0:0:\"Cytosine\";1:cpd00013:0:0:\"NH3\";1:cpd00092:0:0:\"Uracil\"'},\n",
       " {'abbreviation': 'R00975',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00975',\n",
       "   'MetaCyc: DIHYDROURACIL-OXIDASE-RXN',\n",
       "   'Name: 5,6-Dihydrouracil:oxygen oxidoreductase; 5,6-dihydrouracil:oxygen oxidoreductase; dihydrouracil oxidase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00337[0] <=> (1) cpd00025[0] + (1) cpd00092[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00092;cpd00337',\n",
       "  'definition': '(1) O2[0] + (1) Hydrouracil[0] => (1) H2O2[0] + (1) Uracil[0]',\n",
       "  'deltag': -31.11,\n",
       "  'deltagerr': 1.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.3.3.7'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00337[0] => (1) cpd00025[0] + (1) cpd00092[0]',\n",
       "  'id': 'rxn00718',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '5,6-Dihydrouracil:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00337:0:0:\"Hydrouracil\";1:cpd00025:0:0:\"H2O2\";1:cpd00092:0:0:\"Uracil\"'},\n",
       " {'abbreviation': 'R00977',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['EcoCyc: DIHYDROURACIL-DEHYDROGENASE-NAD+-RXN',\n",
       "   'KEGG: R00977',\n",
       "   'MetaCyc: DIHYDROURACIL-DEHYDROGENASE-NAD+-RXN',\n",
       "   'iRS1563: R_R00977_c',\n",
       "   'Name: 5,6-Dihydrouracil:NAD+ oxidoreductase; 5,6-dihydropyrimidine:NAD+ oxidoreductase; 5,6-dihydrouracil:NAD+ oxidoreductase; dihydropyrimidine dehydrogenase; dihydropyrimidine dehydrogenase (NAD+); dihydrothymine dehydrogenase; dihydrouracil dehydrogenase (NAD+); pyrimidine reductase; thymine reductase; uracil reductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00337[0] <=> (1) cpd00004[0] + (1) cpd00092[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00092;cpd00337',\n",
       "  'definition': '(1) NAD[0] + (1) Hydrouracil[0] <=> (1) NADH[0] + (1) H+[0] + (1) Uracil[0]',\n",
       "  'deltag': 0.61,\n",
       "  'deltagerr': 0.92,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.1.1'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00337[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00092[0]',\n",
       "  'id': 'rxn00719',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '5,6-Dihydrouracil:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00240 (Pyrimidine metabolism); rn00410 (beta-Alanine metabolism); rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00337:0:0:\"Hydrouracil\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00092:0:0:\"Uracil\"'},\n",
       " {'abbreviation': 'R00978',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.3.1.2-RXN',\n",
       "   'BiGG: DURAD',\n",
       "   'BrachyCyc: 1.3.1.2-RXN',\n",
       "   'KEGG: R00978',\n",
       "   'MetaCyc: 1.3.1.2-RXN',\n",
       "   'Name: 4,5-dihydrothymine: oxidoreductase; 5,6-Dihydrouracil:NADP+ oxidoreductase; 5,6-dihydrouracil:NADP+ 5-oxidoreductase; DHPDH; DHU dehydrogenase; DPD; Dihydrothymine dehydrogenase; Dihydrouracil dehydrogenase (NADP+); dehydrogenase, dihydrouracil (nicotinamide adenine dinucleotide phosphate); dihydropyrimidine dehydrogenase (NADP); dihydropyrimidine dehydrogenase (NADP+); dihydrothymine dehydrogenase; dihydrouracil dehydrogenase (NADP); hydropyrimidine dehydrogenase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00337[0] <=> (1) cpd00005[0] + (1) cpd00092[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00092;cpd00337',\n",
       "  'definition': '(1) NADP[0] + (1) Hydrouracil[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Uracil[0]',\n",
       "  'deltag': 0.61,\n",
       "  'deltagerr': 0.92,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.1.2'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00337[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00092[0]',\n",
       "  'id': 'rxn00720',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27484',\n",
       "  'name': '5,6-Dihydrouracil:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Beta-Alanine-Biosynthesis (&beta; Alanine Biosynthesis); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-3982 (uracil degradation I (reductive)); PWY-7209 (superpathway of pyrimidine ribonucleosides degradation); Pyrimidine-Base-Degradation (Pyrimidine Nucleobase Degradation); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation); Uracil-Degradation (Uracil Degradation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism); rn00410 (beta-Alanine metabolism); rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00337:0:0:\"Hydrouracil\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00092:0:0:\"Uracil\"'},\n",
       " {'abbreviation': 'R00979',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00979',\n",
       "   'MetaCyc: ANTHRANILATE-ADENYLYLTRANSFERASE-RXN',\n",
       "   'Name: ATP:anthranilate N-adenylyltransferase; anthranilate adenylyltransferase; anthranilic acid adenylyltransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00093[0] <=> (1) cpd00012[0] + (1) cpd02102[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00067;cpd00093;cpd02102',\n",
       "  'definition': '(1) ATP[0] + (1) Anthranilate[0] => (1) PPi[0] + (1) H+[0] + (1) N-Adenylylanthranilate[0]',\n",
       "  'deltag': -20.0,\n",
       "  'deltagerr': 4.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.7.55'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00093[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd02102[0]',\n",
       "  'id': 'rxn00721',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:anthranilate N-adenylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd02102:0:0:\"N-Adenylylanthranilate\"'},\n",
       " {'abbreviation': 'R00980',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00980',\n",
       "   'MetaCyc: 1.14.13.35-RXN',\n",
       "   'Name: Anthranilate,NADPH:oxygen oxidoreductase (3-hydroxylating,; Anthranilate,NADPH:oxygen oxidoreductase (3-hydroxylating, deaminating); anthranilate 2,3-dioxygenase (deaminating); anthranilate 2,3-hydroxylase (deaminating); anthranilate 3-monooxygenase (deaminating); anthranilate hydroxylase; anthranilate hydroxylase (deaminating); anthranilate,NADPH:oxygen oxidoreductase (3-hydroxylating, deaminating); anthranilic hydroxylase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00093[0] <=> (1) cpd00006[0] + (1) cpd00013[0] + (1) cpd00168[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00007;cpd00013;cpd00067;cpd00093;cpd00168',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (2) H+[0] + (1) Anthranilate[0] => (1) NADP[0] + (1) NH3[0] + (1) 2,3-Dihydroxybenzoate[0]',\n",
       "  'deltag': -110.93,\n",
       "  'deltagerr': 1.05,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.35'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (2) cpd00067[0] + (1) cpd00093[0] => (1) cpd00006[0] + (1) cpd00013[0] + (1) cpd00168[0]',\n",
       "  'id': 'rxn00722',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Anthranilate,NADPH:oxygen oxidoreductase (3-hydroxylating, deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00006:0:0:\"NADP\";1:cpd00013:0:0:\"NH3\";1:cpd00168:0:0:\"2,3-Dihydroxybenzoate\"'},\n",
       " {'abbreviation': 'R00982',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00982',\n",
       "   'MetaCyc: AMINOBENZCOALIG-RXN',\n",
       "   'iRS1563: R_R00982_c',\n",
       "   'Name: 2-aminobenzoate coenzyme A ligase; 2-aminobenzoate-CoA ligase; 2-aminobenzoate-coenzyme A ligase; Anthranilate:CoA ligase (AMP-forming); anthranilate-CoA ligase; anthranilate:CoA ligase (AMP-forming); anthraniloyl coenzyme A synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00093[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd01518[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00093;cpd01518',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Anthranilate[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) Anthranilyl-CoA[0]',\n",
       "  'deltag': -2.62,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.32'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00093[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd01518[0]',\n",
       "  'id': 'rxn00723',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Anthranilate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 2-Aminobenzoate-Degradation (2-Aminobenzoate Degradation); 2AMINOBENZDEG-PWY (anthranilate degradation III (anaerobic)); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Aurachin-Biosynthesis (Aurachin Biosynthesis); BENZCOA-PWY (anaerobic aromatic compound degradation (<i>Thauera aromatica</i>)); Degradation (Degradation/Utilization/Assimilation); PWY-6077 (anthranilate degradation II (aerobic)); PWY-6660 (2-heptyl-3-hydroxy-4(1<i>H</i>)-quinolone biosynthesis); PWY-6661 (4-hydroxy-2(1<i>H</i>)-quinolone biosynthesis); PWY-6662 (superpathway of quinolone and alkylquinolone biosynthesis); PWY-7405 (aurachin RE biosynthesis); PWY-7407 (aurachin A, B, C and D biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:H+;1:cpd01518:0:0:\"Anthranilyl-CoA\"'},\n",
       " {'abbreviation': 'R00983',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00983',\n",
       "   'MetaCyc: 23-DIHYDROXYINDOLE-23-DIOXYGENASE-RXN',\n",
       "   'Name: 2,3-Dihydroxyindole:oxygen 2,3-oxidoreductase (decyclizing); 2,3-dihydroxyindole 2,3-dioxygenase; 2,3-dihydroxyindole:oxygen 2,3-oxidoreductase (decyclizing)'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd01789[0] <=> (1) cpd00011[0] + (1) cpd00093[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00067;cpd00093;cpd01789',\n",
       "  'definition': '(1) O2[0] + (1) Dihydroxyindole[0] => (1) CO2[0] + (1) H+[0] + (1) Anthranilate[0]',\n",
       "  'deltag': -107.59,\n",
       "  'deltagerr': 1.89,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.23'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd01789[0] => (1) cpd00011[0] + (1) cpd00067[0] + (1) cpd00093[0]',\n",
       "  'id': 'rxn00724',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2,3-Dihydroxyindole:oxygen 2,3-oxidoreductase (decyclizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd01789:0:0:\"Dihydroxyindole\";1:cpd00011:0:0:\"CO2\";1:cpd00067:0:0:\"H+\";1:cpd00093:0:0:\"Anthranilate\"'},\n",
       " {'abbreviation': 'R00984',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00984',\n",
       "   'MetaCyc: ANTHRANILATE-N-METHYLTRANSFERASE-RXN',\n",
       "   'PlantCyc: ANTHRANILATE-N-METHYLTRANSFERASE-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine:anthranilate N-methyltransferase; S-adenosyl-L-methionine:anthranilate N-methyltransferase; anthranilate N-methyltransferase; anthranilic acid N-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00093[0] <=> (1) cpd00019[0] + (1) cpd01928[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00093;cpd01928',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Anthranilate[0] => (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) N-Methylanthranilate[0]',\n",
       "  'deltag': -14.2,\n",
       "  'deltagerr': 5.04,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.1.1.111'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00093[0] => (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd01928[0]',\n",
       "  'id': 'rxn00725',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:anthranilate N-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-5958 (acridone alkaloid biosynthesis); PWY-7463 (<i>N</i>-methylanthraniloyl-&beta;-<i>D</i>-glucopyranose biosynthesis); PWY-7476 (superpathway of avenacin A biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TRITERPENOID-SYN (Triterpenoid Biosynthesis); Terpenoid-Biosynthesis (Terpenoid Biosynthesis)',\n",
       "   'KEGG: rn01058 (Acridone alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd01928:0:0:\"N-Methylanthranilate\"'},\n",
       " {'abbreviation': 'R00985',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00985_c',\n",
       "   'AraGEM: R_R00985_c',\n",
       "   'BiGG: ANS2',\n",
       "   'JP_Creinhardtii_MSB: R_ATNS_LPAREN_nh4_RPAREN_',\n",
       "   'KEGG: R00985',\n",
       "   'Name: Anthranilate synthase, ammonia; anthranilatesynthase2; chorismate pyruvate-lyase (amino-accepting; anthranilate-forming)'],\n",
       "  'code': '(1) cpd00013[0] + (1) cpd00216[0] <=> (1) cpd00001[0] + (1) cpd00020[0] + (1) cpd00093[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00067;cpd00093;cpd00216',\n",
       "  'definition': '(1) NH3[0] + (1) Chorismate[0] => (1) H2O[0] + (1) Pyruvate[0] + (1) H+[0] + (1) Anthranilate[0]',\n",
       "  'deltag': -21.94,\n",
       "  'deltagerr': 1.79,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.3.27'],\n",
       "  'equation': '(1) cpd00013[0] + (1) cpd00216[0] => (1) cpd00001[0] + (1) cpd00020[0] + (1) cpd00067[0] + (1) cpd00093[0]',\n",
       "  'id': 'rxn00726',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32242;rxn35359',\n",
       "  'name': 'chorismate pyruvate-lyase (amino-accepting; anthranilate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00013:0:0:\"NH3\";-1:cpd00216:0:0:\"Chorismate\";1:cpd00001:0:0:\"H2O\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00093:0:0:\"Anthranilate\"'},\n",
       " {'abbreviation': 'R00986',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ANTHRANSYN-RXN',\n",
       "   'BiGG: ANS; ANS1; TRP2_1; TRP3_1',\n",
       "   'BrachyCyc: ANTHRANSYN-RXN',\n",
       "   'KEGG: R00986',\n",
       "   'MetaCyc: ANTHRANSYN-RXN',\n",
       "   'Name: Chorismate pyruvate-lyase (amino-accepting); TrpE; anthranilate synthase; anthranilate synthase component II; anthranilate synthesis; anthranilate synthetase; anthranilatesynthase; chorismate lyase; chorismate pyruvate-lyase (amino-accepting); chorismate pyruvate-lyase (amino-accepting, anthranilate-forming)'],\n",
       "  'code': '(1) cpd00053[0] + (1) cpd00216[0] <=> (1) cpd00020[0] + (1) cpd00023[0] + (1) cpd00093[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00023;cpd00053;cpd00067;cpd00093;cpd00216',\n",
       "  'definition': '(1) L-Glutamine[0] + (1) Chorismate[0] => (1) Pyruvate[0] + (1) L-Glutamate[0] + (1) H+[0] + (1) Anthranilate[0]',\n",
       "  'deltag': -24.76,\n",
       "  'deltagerr': 1.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.3.27'],\n",
       "  'equation': '(1) cpd00053[0] + (1) cpd00216[0] => (1) cpd00020[0] + (1) cpd00023[0] + (1) cpd00067[0] + (1) cpd00093[0]',\n",
       "  'id': 'rxn00727',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27709;rxn38042;rxn38043',\n",
       "  'name': 'Chorismate pyruvate-lyase (amino-accepting)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); COMPLETE-ARO-PWY (superpathway of aromatic amino acid biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-5958 (acridone alkaloid biosynthesis); PWY-6629 (superpathway of L-tryptophan biosynthesis); PWY-6661 (4-hydroxy-2(1<i>H</i>)-quinolone biosynthesis); PWY-6662 (superpathway of quinolone and alkylquinolone biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TRPSYN-PWY (L-tryptophan biosynthesis); TRYPTOPHAN-BIOSYNTHESIS (L-tryptophan Biosynthesis)',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00216:0:0:\"Chorismate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00067:0:0:\"H+\";1:cpd00093:0:0:\"Anthranilate\"'},\n",
       " {'abbreviation': 'R00987',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: KYN; KYNASE2; YLR231C_1',\n",
       "   'KEGG: R00987',\n",
       "   'MetaCyc: KYNURENINASE-RXN',\n",
       "   'iIN800: YLR231C_1',\n",
       "   'iMM904: KYN',\n",
       "   'Name: Kynureninase; L-Kynurenine hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00275[0] <=> (1) cpd00035[0] + (1) cpd00093[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00035;cpd00067;cpd00093;cpd00275',\n",
       "  'definition': '(1) H2O[0] + (1) L-Kynurenine[0] => (1) L-Alanine[0] + (1) H+[0] + (1) Anthranilate[0]',\n",
       "  'deltag': -12.1,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.7.1.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00275[0] => (1) cpd00035[0] + (1) cpd00067[0] + (1) cpd00093[0]',\n",
       "  'id': 'rxn00728',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Kynurenine hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-6309 (L-tryptophan degradation XI (mammalian, via kynurenine)); PWY-6505 (L-tryptophan degradation XII (Geobacillus)); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TRPCAT-PWY (L-tryptophan degradation I (via anthranilate)); TRYPTOPHAN-DEG (L-tryptophan Degradation)',\n",
       "   'KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00275:0:0:\"L-Kynurenine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd00067:0:0:\"H+\";1:cpd00093:0:0:\"Anthranilate\"'},\n",
       " {'abbreviation': 'R00988',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00988', 'Name: N-Formylanthranilate amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03364[0] <=> (1) cpd00047[0] + (1) cpd00093[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00047;cpd00067;cpd00093;cpd03364',\n",
       "  'definition': '(1) H2O[0] + (1) Formylanthranilate[0] <=> (1) Formate[0] + (1) H+[0] + (1) Anthranilate[0]',\n",
       "  'deltag': -7.42,\n",
       "  'deltagerr': 1.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03364[0] <=> (1) cpd00047[0] + (1) cpd00067[0] + (1) cpd00093[0]',\n",
       "  'id': 'rxn00729',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Formylanthranilate amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03364:0:0:\"Formylanthranilate\";1:cpd00047:0:0:\"Formate\";1:cpd00067:0:0:\"H+\";1:cpd00093:0:0:\"Anthranilate\"'},\n",
       " {'abbreviation': 'R00989',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00989',\n",
       "   'MetaCyc: ANTHRANILATE-N-MALONYLTRANSFERASE-RXN',\n",
       "   'Name: Malonyl-CoA:anthranilate N-malonyltransferase; anthranilate N-malonyltransferase; malonyl-CoA:anthranilate N-malonyltransferase'],\n",
       "  'code': '(1) cpd00070[0] + (1) cpd00093[0] <=> (1) cpd00010[0] + (1) cpd02014[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00070;cpd00093;cpd02014',\n",
       "  'definition': '(1) Malonyl-CoA[0] + (1) Anthranilate[0] => (1) CoA[0] + (1) N-Malonylanthranilate[0]',\n",
       "  'deltag': -3.96,\n",
       "  'deltagerr': 0.81,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.113'],\n",
       "  'equation': '(1) cpd00070[0] + (1) cpd00093[0] => (1) cpd00010[0] + (1) cpd02014[0]',\n",
       "  'id': 'rxn00730',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Malonyl-CoA:anthranilate N-malonyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00070:0:0:\"Malonyl-CoA\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00010:0:0:\"CoA\";1:cpd02014:0:0:\"N-Malonylanthranilate\"'},\n",
       " {'abbreviation': 'R00990',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00990',\n",
       "   'MetaCyc: RXN-7927',\n",
       "   'TS_Athaliana: rxn620_c; rxn620_m; rxn620_p',\n",
       "   'Name: 2-Aminobenzoate carboxy-lyase; aminobenzoate carboxy-lyase (aniline-forming); aminobenzoate decarboxylase'],\n",
       "  'code': '(1) cpd00093[0] <=> (1) cpd00011[0] + (1) cpd00245[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00093;cpd00245',\n",
       "  'definition': '(1) H+[0] + (1) Anthranilate[0] => (1) CO2[0] + (1) Aniline[0]',\n",
       "  'deltag': -2.66,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.24'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00093[0] => (1) cpd00011[0] + (1) cpd00245[0]',\n",
       "  'id': 'rxn00731',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32329;rxn32330',\n",
       "  'name': '2-Aminobenzoate carboxy-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00093:0:0:\"Anthranilate\";1:cpd00011:0:0:\"CO2\";1:cpd00245:0:0:\"Aniline\"'},\n",
       " {'abbreviation': 'R00991',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00991', 'Name: '],\n",
       "  'code': '(1) cpd00093[0] <=> (1) cpd03465[0]',\n",
       "  'compound_ids': 'cpd00093;cpd03465',\n",
       "  'definition': '(1) Anthranilate[0] => (1) 2-Formaminobenzoylacetate[0]',\n",
       "  'deltag': -35.36,\n",
       "  'deltagerr': 1.3,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00093[0] => (1) cpd03465[0]',\n",
       "  'id': 'rxn00732',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R00991',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00380 (Tryptophan metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:C:3/H:2/O:2',\n",
       "  'stoichiometry': '-1:cpd00093:0:0:\"Anthranilate\";1:cpd03465:0:0:\"2-Formaminobenzoylacetate\"'},\n",
       " {'abbreviation': 'rxn00733',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00093[0] <=> (1) cpd03466[0]',\n",
       "  'compound_ids': 'cpd00093;cpd03466',\n",
       "  'definition': '(1) Anthranilate[0] <=> (1) Indole-3-acetaldoxime[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00093[0] <=> (1) cpd03466[0]',\n",
       "  'id': 'rxn00733',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00733',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:C:3/H:4/N:1/O:-1|CI:1',\n",
       "  'stoichiometry': '-1:cpd00093:0:0:\"Anthranilate\";1:cpd03466:0:0:\"Indole-3-acetaldoxime\"'},\n",
       " {'abbreviation': 'rxn00734',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(0.5) cpd00007[0] + (1) cpd00009[0] + (1) cpd00094[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01844[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00009;cpd00011;cpd00067;cpd00094;cpd01844',\n",
       "  'definition': '(0.5) O2[0] + (1) Phosphate[0] + (1) H+[0] + (1) 2-Oxobutyrate[0] <=> (1) H2O[0] + (1) CO2[0] + (1) Propionyl phosphate[0]',\n",
       "  'deltag': -54.26,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(0.5) cpd00007[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00094[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd01844[0]',\n",
       "  'id': 'rxn00734',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00734',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-0.5:cpd00007:0:0:\"O2\";-1:cpd00009:0:0:\"Phosphate\";-1:cpd00067:0:0:\"H+\";-1:cpd00094:0:0:\"2-Oxobutyrate\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd01844:0:0:\"Propionyl phosphate\"'},\n",
       " {'abbreviation': 'R00994',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R00994_c',\n",
       "   'AraGEM: R_R00994_c',\n",
       "   'KEGG: R00994',\n",
       "   'TS_Athaliana: rxn621_p',\n",
       "   'iRS1563: R_R00994_c',\n",
       "   'Name: (2R,3S)-3-methylmalate:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00094[0] <=> (1) cpd00003[0] + (1) cpd03593[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00094;cpd03593',\n",
       "  'definition': '(1) NADH[0] + (1) CO2[0] + (1) 2-Oxobutyrate[0] <= (1) NAD[0] + (1) D-erythro-3-Methylmalate[0]',\n",
       "  'deltag': 0.8,\n",
       "  'deltagerr': 1.14,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.1.1.85'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00094[0] <= (1) cpd00003[0] + (1) cpd03593[0]',\n",
       "  'id': 'rxn00735',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32091',\n",
       "  'name': '(2R,3S)-3-methylmalate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00290 (Valine, leucine and isoleucine biosynthesis); rn00660 (C5-Branched dibasic acid metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00011:0:0:\"CO2\";-1:cpd00094:0:0:\"2-Oxobutyrate\";1:cpd00003:0:0:\"NAD\";1:cpd03593:0:0:\"D-erythro-3-Methylmalate\"'},\n",
       " {'abbreviation': 'R00995',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00995',\n",
       "   'MetaCyc: RXN-7746',\n",
       "   'Name: methyloxaloacetate carboxy-lyase (2-oxobutanoate-forming)'],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00094[0] <=> (1) cpd03591[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00094;cpd03591',\n",
       "  'definition': '(1) CO2[0] + (1) 2-Oxobutyrate[0] <= (1) H+[0] + (1) Methyloxaloacetate[0]',\n",
       "  'deltag': 5.24,\n",
       "  'deltagerr': 1.17,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['4.1.1.3'],\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00094[0] <= (1) cpd00067[0] + (1) cpd03591[0]',\n",
       "  'id': 'rxn00736',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'methyloxaloacetate carboxy-lyase (2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); ISOLEUCINE-SYN (L-isoleucine Biosynthesis); PWY-5103 (L-isoleucine biosynthesis III)',\n",
       "   'KEGG: rn00660 (C5-Branched dibasic acid metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00094:0:0:\"2-Oxobutyrate\";1:cpd00067:0:0:\"H+\";1:cpd03591:0:0:\"Methyloxaloacetate\"'},\n",
       " {'abbreviation': 'R00996',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: THREDEHYD-RXN',\n",
       "   'BiGG: CHA1_1; ILV1; THRD_L; THRD_Lm; THRD_Lr',\n",
       "   'BrachyCyc: THREDEHYD-RXN',\n",
       "   'KEGG: R00996',\n",
       "   'MetaCyc: THREDEHYD-RXN',\n",
       "   'Name: L-serine dehydratase; L-threonine ammonia-lyase; L-threonine ammonia-lyase (2-oxobutanoate-forming); L-threonine deaminase; L-threonine deaminse reversible; L-threonine dehydratase; L-threonine hydro-lyase (deaminating); L-threoninedeaminase; Serine deaminase; Threonine deaminase; Threonine dehydratase; serine deaminase; threonine ammonia-lyase; threonine deaminase; threonine dehydrase; threonine dehydratase'],\n",
       "  'code': '(1) cpd00161[0] <=> (1) cpd00013[0] + (1) cpd00094[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00094;cpd00161',\n",
       "  'definition': '(1) L-Threonine[0] => (1) NH3[0] + (1) 2-Oxobutyrate[0]',\n",
       "  'deltag': -5.92,\n",
       "  'deltagerr': 1.01,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.19'],\n",
       "  'equation': '(1) cpd00161[0] => (1) cpd00013[0] + (1) cpd00094[0]',\n",
       "  'id': 'rxn00737',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29011;rxn30064;rxn30143;rxn30440;rxn35533',\n",
       "  'name': 'L-threonine ammonia-lyase (2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PWY66-428 (L-threonine degradation V); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); THREONINE-DEG (L-threonine Degradation)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00290 (Valine, leucine and isoleucine biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00161:0:0:\"L-Threonine\";1:cpd00013:0:0:\"NH3\";1:cpd00094:0:0:\"2-Oxobutyrate\"'},\n",
       " {'abbreviation': 'R00997',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 4.1.99.4-RXN',\n",
       "   'KEGG: R00997',\n",
       "   'MetaCyc: 4.1.99.4-RXN',\n",
       "   'RiceCyc: 4.1.99.4-RXN',\n",
       "   'SorghumCyc: 4.1.99.4-RXN',\n",
       "   'Name: 1-Aminocyclopropane-1-carboxylate endolyase (deaminating); 1-aminocyclopropane-1-carboxylate aminohydrolase (isomerizing); 1-aminocyclopropane-1-carboxylate deaminase; 1-aminocyclopropane-1-carboxylate endolyase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00909[0] <=> (1) cpd00013[0] + (1) cpd00094[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00094;cpd00909',\n",
       "  'definition': '(1) H2O[0] + (1) 1-Aminocyclopropane-1-carboxylate[0] => (1) NH3[0] + (1) 2-Oxobutyrate[0]',\n",
       "  'deltag': 9.9,\n",
       "  'deltagerr': 1.88,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.99.7', '4.1.99.4-'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00909[0] => (1) cpd00013[0] + (1) cpd00094[0]',\n",
       "  'id': 'rxn00738',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30589;rxn33042;rxn35416;rxn35417',\n",
       "  'name': '1-aminocyclopropane-1-carboxylate aminohydrolase (isomerizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00909:0:0:\"1-Aminocyclopropane-1-carboxylate\";1:cpd00013:0:0:\"NH3\";1:cpd00094:0:0:\"2-Oxobutyrate\"'},\n",
       " {'abbreviation': 'R00998',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R00998',\n",
       "   'MetaCyc: 2-ETHYLMALATE-SYNTHASE-RXN',\n",
       "   'Name: (R)-2-Ethylmalate 2-oxobutanoyl-lyase (CoA-acylating); (R)-2-ethylmalate 2-oxobutanoyl-lyase (CoA-acetylating); (R)-2-ethylmalate 2-oxobutanoyl-lyase (CoA-acylating); 2-ethylmalate synthase; 2-ethylmalate-3-hydroxybutanedioate synthase; acetyl-CoA:2-oxobutanoate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming); propylmalate synthase; propylmalic synthase'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd01636[0] <=> (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00094[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00022;cpd00067;cpd00094;cpd01636',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) (R)-2-Ethylmalate[0] <= (1) H2O[0] + (1) Acetyl-CoA[0] + (1) 2-Oxobutyrate[0]',\n",
       "  'deltag': 8.92,\n",
       "  'deltagerr': 4.99,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.6'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01636[0] <= (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00094[0]',\n",
       "  'id': 'rxn00739',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'acetyl-CoA:2-oxobutanoate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming)',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd01636:0:0:\"(R)-2-Ethylmalate\";1:cpd00001:0:0:\"H2O\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00094:0:0:\"2-Oxobutyrate\"'},\n",
       " {'abbreviation': 'R00999',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: METBALT-RXN',\n",
       "   'BiGG: SHSL4; SHSL4r; YML082W',\n",
       "   'BrachyCyc: METBALT-RXN',\n",
       "   'KEGG: R00999',\n",
       "   'MetaCyc: METBALT-RXN',\n",
       "   'Name: O-Succinyl-L-homoserine succinate-lyase (adding cysteine); O-Succinyl-L-homoserine succinate-lyase (deaminating; 2-oxobutanoate-forming); O-Succinyl-L-homoserine succinate-lyase (deaminating;2-oxobutanoate-forming); O-succinylhomoserine (thiol)-lyase; O-succinylhomoserine lyase (elimination); O-succinylhomoserine lyase (elimination), reversible; O-succinylhomoserinelyase-elimination-reversible'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00822[0] <=> (1) cpd00013[0] + (1) cpd00036[0] + (1) cpd00094[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00036;cpd00067;cpd00094;cpd00822',\n",
       "  'definition': '(1) H2O[0] + (1) O-Succinyl-L-homoserine[0] => (1) NH3[0] + (1) Succinate[0] + (1) H+[0] + (1) 2-Oxobutyrate[0]',\n",
       "  'deltag': -14.01,\n",
       "  'deltagerr': 1.14,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.5.1.48', '4.3.1.-'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00822[0] => (1) cpd00013[0] + (1) cpd00036[0] + (1) cpd00067[0] + (1) cpd00094[0]',\n",
       "  'id': 'rxn00740',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27994;rxn30320;rxn34665',\n",
       "  'name': 'O-Succinyl-L-homoserine succinate-lyase (deaminating; 2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00822:0:0:\"O-Succinyl-L-homoserine\";1:cpd00013:0:0:\"NH3\";1:cpd00036:0:0:\"Succinate\";1:cpd00067:0:0:\"H+\";1:cpd00094:0:0:\"2-Oxobutyrate\"'},\n",
       " {'abbreviation': 'R01000',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01000_c',\n",
       "   'AraGEM: R_R01000_c',\n",
       "   'BiGG: 2HBO; LDH2',\n",
       "   'KEGG: R01000',\n",
       "   'Maize_C4GEM: R_R01000_c',\n",
       "   'Name: 2-Hydroxybutyrate:NAD+ oxidoreductase; L-lactate dehydrogenase (2-Hydroxybutanoic acid)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd03561[0] <=> (1) cpd00004[0] + (1) cpd00094[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00094;cpd03561',\n",
       "  'definition': '(1) NAD[0] + (1) 2-Hydroxybutyrate[0] <=> (1) NADH[0] + (1) H+[0] + (1) 2-Oxobutyrate[0]',\n",
       "  'deltag': 4.44,\n",
       "  'deltagerr': 0.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.27'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd03561[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00094[0]',\n",
       "  'id': 'rxn00741',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Hydroxybutyrate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd03561:0:0:\"2-Hydroxybutyrate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00094:0:0:\"2-Oxobutyrate\"'},\n",
       " {'abbreviation': 'R01001',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CYSTAGLY-RXN',\n",
       "   'BiGG: CYS3; CYSTGL; CYSTGLr',\n",
       "   'ChlamyCyc: CYSTAGLY-RXN',\n",
       "   'KEGG: R01001',\n",
       "   'MetaCyc: CYSTAGLY-RXN',\n",
       "   'Name: Cystathionine gamma-lyase; L-Cystathionine Lysteine-lyase (deaminating); L-cystathionine cysteine-lyase (deaminating); L-cystathionine cysteine-lyase (deaminating, 2-oxobutanoate-forming); L-cystathionine cysteine-lyase (deaminating; 2-oxobutanoate-forming); cystalysin; cystathionase; cystathionine g-lyase; cystathionine gamma-lyase; cysteine desulfhydrase; cysteine lyase; cystine desulfhydrase; gamma-CTL; gamma-cystathionase; homoserine deaminase; homoserine deaminase-cystathionase; homoserine dehydratase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00424[0] <=> (1) cpd00013[0] + (1) cpd00084[0] + (1) cpd00094[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00084;cpd00094;cpd00424',\n",
       "  'definition': '(1) H2O[0] + (1) Cystathionine[0] => (1) NH3[0] + (1) L-Cysteine[0] + (1) 2-Oxobutyrate[0]',\n",
       "  'deltag': 0.54,\n",
       "  'deltagerr': 1.48,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.4.1.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00424[0] => (1) cpd00013[0] + (1) cpd00084[0] + (1) cpd00094[0]',\n",
       "  'id': 'rxn00742',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30306;rxn33426;rxn33428;rxn33507',\n",
       "  'name': 'L-cystathionine cysteine-lyase (deaminating; 2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00270 (Cysteine and methionine metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00424:0:0:\"Cystathionine\";1:cpd00013:0:0:\"NH3\";1:cpd00084:0:0:\"L-Cysteine\";1:cpd00094:0:0:\"2-Oxobutyrate\"'},\n",
       " {'abbreviation': 'R01010',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ALKP; ALKPr',\n",
       "   'KEGG: R01010',\n",
       "   'MetaCyc: RXN0-7249',\n",
       "   'iAF692: ALKP',\n",
       "   'iRS1563: R_R01010_c',\n",
       "   'Name: Glycerone phosphate phosphohydrolase; Glycerone phosphate phosphohydrolase (alkaline optimum); alkaline phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00095[0] <=> (1) cpd00009[0] + (1) cpd00157[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00095;cpd00157',\n",
       "  'definition': '(1) H2O[0] + (1) Glycerone-phosphate[0] <=> (1) Phosphate[0] + (1) Glycerone[0]',\n",
       "  'deltag': -2.99,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.1', '3.1.3.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00095[0] <=> (1) cpd00009[0] + (1) cpd00157[0]',\n",
       "  'id': 'rxn00743',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glycerone phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00095:0:0:\"Glycerone-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00157:0:0:\"Glycerone\"'},\n",
       " {'abbreviation': 'R01011',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCERONE-KINASE-RXN',\n",
       "   'BiGG: DAK1; DAK2; DHAK',\n",
       "   'BrachyCyc: GLYCERONE-KINASE-RXN',\n",
       "   'KEGG: R01011',\n",
       "   'MetaCyc: GLYCERONE-KINASE-RXN',\n",
       "   'Name: ATP:glycerone phosphotransferase; Dihydroxyacetone kinase 1; acetol kinase; acetol kinase (phosphorylating); dihydroxyacetone kinase; glycerone kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00157[0] <=> (1) cpd00008[0] + (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00095;cpd00157',\n",
       "  'definition': '(1) ATP[0] + (1) Glycerone[0] => (1) ADP[0] + (1) H+[0] + (1) Glycerone-phosphate[0]',\n",
       "  'deltag': -3.17,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.29'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00157[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00095[0]',\n",
       "  'id': 'rxn00744',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:glycerone phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Alcohol-Degradation (Alcohol Degradation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Formaldehyde-Assimilation (Formaldehyde Assimilation); GLYCEROL-DEG (Glycerol Degradation); GOLPDLCAT-PWY (superpathway of glycerol degradation to 1,3-propanediol); P185-PWY (formaldehyde assimilation III (dihydroxyacetone cycle)); PWY-6131 (glycerol degradation II); PWY-7003 (glycerol degradation to butanol)',\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism); rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00157:0:0:\"Glycerone\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R01012',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: DHAPT',\n",
       "   'EcoCyc: 2.7.1.121-RXN',\n",
       "   'KEGG: R01012',\n",
       "   'MetaCyc: 2.7.1.121-RXN',\n",
       "   'iAF1260: DHAPT',\n",
       "   'Name: Dihydroxyacetone phosphotransferase; Phosphoenolpyruvate:glycerone phosphotransferase; phosphoenolpyruvate-glycerone phosphotransferase; phosphoenolpyruvate:glycerone phosphotransferase'],\n",
       "  'code': '(1) cpd00061[0] + (1) cpd00157[0] <=> (1) cpd00020[0] + (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00061;cpd00095;cpd00157',\n",
       "  'definition': '(1) Phosphoenolpyruvate[0] + (1) Glycerone[0] <=> (1) Pyruvate[0] + (1) Glycerone-phosphate[0]',\n",
       "  'deltag': -9.7,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.121'],\n",
       "  'equation': '(1) cpd00061[0] + (1) cpd00157[0] <=> (1) cpd00020[0] + (1) cpd00095[0]',\n",
       "  'id': 'rxn00745',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phosphoenolpyruvate:glycerone phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); Degradation (Degradation/Utilization/Assimilation); GLYCEROL-DEG (Glycerol Degradation); GLYCEROLMETAB-PWY (glycerol degradation V)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00061:0:0:\"Phosphoenolpyruvate\";-1:cpd00157:0:0:\"Glycerone\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R01014',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01014',\n",
       "   'MetaCyc: KETOTETROSE-PHOSPHATE-ALDOLASE-RXN',\n",
       "   'Name: Erythrulose-1-phosphate formaldehyde-lyase; erythrose-1-phosphate synthase; erythrulose-1-phosphate formaldehyde-lyase; erythrulose-1-phosphate formaldehyde-lyase (glycerone-phosphate-forming); erythrulose-1-phosphate synthetase; ketotetrose-phosphate aldolase; phosphoketotetrose aldolase'],\n",
       "  'code': '(1) cpd02148[0] <=> (1) cpd00055[0] + (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00055;cpd00095;cpd02148',\n",
       "  'definition': '(1) Erythrulose 1-phosphate[0] <=> (1) Formaldehyde[0] + (1) Glycerone-phosphate[0]',\n",
       "  'deltag': 6.69,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.2'],\n",
       "  'equation': '(1) cpd02148[0] <=> (1) cpd00055[0] + (1) cpd00095[0]',\n",
       "  'id': 'rxn00746',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'erythrulose-1-phosphate formaldehyde-lyase (glycerone-phosphate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02148:0:0:\"Erythrulose 1-phosphate\";1:cpd00055:0:0:\"Formaldehyde\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R01015',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: TRIOSEPISOMERIZATION-RXN',\n",
       "   'BiGG: TPI',\n",
       "   'BrachyCyc: TRIOSEPISOMERIZATION-RXN',\n",
       "   'KEGG: R01015',\n",
       "   'MetaCyc: TRIOSEPISOMERIZATION-RXN',\n",
       "   'Name: D-Glyceraldehyde-3-phosphate ketol-isomerase; D-glyceraldehyde-3-phosphate aldose-ketose-isomerase; D-glyceraldehyde-3-phosphate ketol-isomerase; Triosephosphate isomerase; phosphotriose isomerase; triose phosphate mutase; triose phosphoisomerase; triose-phosphate isomerase; triose-phosphateisomerase; triosephosphate isomerase'],\n",
       "  'code': '(1) cpd00102[0] <=> (1) cpd00095[0]',\n",
       "  'compound_ids': 'cpd00095;cpd00102',\n",
       "  'definition': '(1) Glyceraldehyde3-phosphate[0] <=> (1) Glycerone-phosphate[0]',\n",
       "  'deltag': -1.31,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.1'],\n",
       "  'equation': '(1) cpd00102[0] <=> (1) cpd00095[0]',\n",
       "  'id': 'rxn00747',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27386;rxn33035',\n",
       "  'name': 'D-glyceraldehyde-3-phosphate aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROFRUCAT-PWY (homolactic fermentation); ANAGLYCOLYSIS-PWY (glycolysis III (from glucose)); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CALVIN-PWY (Calvin-Benson-Bassham cycle); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethylene-Biosynthesis (Ethylene Biosynthesis); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Formaldehyde-Assimilation (Formaldehyde Assimilation); GLUCONEO-PWY (gluconeogenesis I); GLYCOLYSIS (glycolysis I (from glucose 6-phosphate)); GLYCOLYSIS-E-D (superpathway of glycolysis and the Entner-Doudoroff pathway); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYCOLYSIS-VARIANTS (Glycolysis); Gluconeogenesis (Gluconeogenesis); HEXITOLDEGSUPER-PWY (superpathway of hexitol degradation (bacteria)); HORMONE-SYN (Hormone Biosynthesis); P185-PWY (formaldehyde assimilation III (dihydroxyacetone cycle)); P341-PWY (glycolysis V (Pyrococcus)); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PHOTOALL-PWY (oxygenic photosynthesis); PWY-1042 (glycolysis IV (plant cytosol)); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-5484 (glycolysis II (from fructose 6-phosphate)); PWY-6142 (gluconeogenesis II (<i>Methanobacterium thermoautotrophicum</i>)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7003 (glycerol degradation to butanol); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY66-373 (sucrose degradation V (sucrose &alpha;-glucosidase)); PWY66-399 (gluconeogenesis III); Photosynthesis (); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00051 (Fructose and mannose metabolism); rn00562 (Inositol phosphate metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\";1:cpd00095:0:0:\"Glycerone-phosphate\"'},\n",
       " {'abbreviation': 'R01016',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MGSA',\n",
       "   'EcoCyc: METHGLYSYN-RXN',\n",
       "   'KEGG: R01016',\n",
       "   'MetaCyc: METHGLYSYN-RXN',\n",
       "   'iAF1260: MGSA',\n",
       "   'Name: Glycerone-phosphate phospho-lyase; glycerone-phosphate phosphate-lyase (methylglyoxal-forming); glycerone-phosphate phospho-lyase; methylglyoxal synthase; methylglyoxal synthetase'],\n",
       "  'code': '(1) cpd00095[0] <=> (1) cpd00009[0] + (1) cpd00428[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00095;cpd00428',\n",
       "  'definition': '(1) Glycerone-phosphate[0] => (1) Phosphate[0] + (1) 2-Oxopropanal[0]',\n",
       "  'deltag': -14.0,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.3.3'],\n",
       "  'equation': '(1) cpd00095[0] => (1) cpd00009[0] + (1) cpd00428[0]',\n",
       "  'id': 'rxn00748',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glycerone-phosphate phosphate-lyase (methylglyoxal-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00095:0:0:\"Glycerone-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00428:0:0:\"2-Oxopropanal\"'},\n",
       " {'abbreviation': 'R01019',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01019',\n",
       "   'Name: acetaldehyde:pyrroloquinoline-quinone oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00071[0] + (1) cpd00097[0] <=> (1) cpd00029[0] + (1) cpd00986[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00067;cpd00071;cpd00097;cpd00986',\n",
       "  'definition': '(1) H2O[0] + (1) Acetaldehyde[0] + (1) Pyrroloquinoline-quinone[0] => (1) Acetate[0] + (1) H+[0] + (1) PQQH2[0]',\n",
       "  'deltag': -27.07,\n",
       "  'deltagerr': 1.53,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.99.3'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00071[0] + (1) cpd00097[0] => (1) cpd00029[0] + (1) cpd00067[0] + (1) cpd00986[0]',\n",
       "  'id': 'rxn00749',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'acetaldehyde:pyrroloquinoline-quinone oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00071:0:0:\"Acetaldehyde\";-1:cpd00097:0:0:\"Pyrroloquinoline-quinone\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\";1:cpd00986:0:0:\"PQQH2\"'},\n",
       " {'abbreviation': 'rxn00750',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00097[0] <=> (1) cpd00023[0] + (1) cpd00612[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00067;cpd00097;cpd00612',\n",
       "  'definition': '(14) H+[0] + (1) Pyrroloquinoline-quinone[0] <=> (1) L-Glutamate[0] + (1) Dopaquinone[0]',\n",
       "  'deltag': 82.33,\n",
       "  'deltagerr': 2.67,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(14) cpd00067[0] + (1) cpd00097[0] <=> (1) cpd00023[0] + (1) cpd00612[0]',\n",
       "  'id': 'rxn00750',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00750',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:-12',\n",
       "  'stoichiometry': '-14:cpd00067:0:0:\"H+\";-1:cpd00097:0:0:\"Pyrroloquinoline-quinone\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00612:0:0:\"Dopaquinone\"'},\n",
       " {'abbreviation': 'R01021',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: CHOLINE-KINASE-RXN',\n",
       "   'BiGG: CHOLK; CKI1',\n",
       "   'BrachyCyc: CHOLINE-KINASE-RXN',\n",
       "   'KEGG: R01021',\n",
       "   'MetaCyc: CHOLINE-KINASE-RXN',\n",
       "   'Name: ATP:choline phosphotransferase; Choline kinase; choline kinase; choline kinase (phosphorylating); choline phosphokinase; choline-ethanolamine kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00098[0] <=> (1) cpd00008[0] + (1) cpd00457[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00098;cpd00457',\n",
       "  'definition': '(1) ATP[0] + (1) Choline[0] => (1) ADP[0] + (1) H+[0] + (1) Phosphocholine[0]',\n",
       "  'deltag': -3.99,\n",
       "  'deltagerr': 0.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.32'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00098[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00457[0]',\n",
       "  'id': 'rxn00751',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn38064',\n",
       "  'name': 'ATP:choline phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); Macromolecule-Modification (Macromolecule Modification); PHOSLIPSYN2-PWY (superpathway of phospholipid biosynthesis II (plants)); PWY-7782 (plasmalogen biosynthesis); PWY-7818 (type IV lipoteichoic acid biosynthesis (<i>S. pneumoniae</i>)); PWY-7886 (cell-surface glycoconjugate-linked phosphocholine biosynthesis); PWY3O-450 (phosphatidylcholine biosynthesis I); PWY4FS-5 (superpathway of phosphatidylcholine biosynthesis); PhosphatidylcholineBiosynthesis (Phosphatidylcholine Biosynthesis); Phospholipid-Biosynthesis (Phospholipid Biosynthesis); Teichoic-Acids-Biosynthesis (Teichoic Acid Biosynthesis)',\n",
       "   'KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00098:0:0:\"Choline\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00457:0:0:\"Phosphocholine\"'},\n",
       " {'abbreviation': 'R01022',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01022',\n",
       "   'MetaCyc: CHOLINE-OXIDASE-RXN',\n",
       "   'Name: Choline:oxygen 1-oxidoreductase; choline oxidase; choline:oxygen 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00098[0] <=> (1) cpd00025[0] + (1) cpd00447[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00098;cpd00447',\n",
       "  'definition': '(1) O2[0] + (1) Choline[0] => (1) H2O2[0] + (1) Betaine aldehyde[0]',\n",
       "  'deltag': -27.58,\n",
       "  'deltagerr': 1.05,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.17'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00098[0] => (1) cpd00025[0] + (1) cpd00447[0]',\n",
       "  'id': 'rxn00752',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Choline:oxygen 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Choline-Degradation (Choline Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-3721 (choline degradation II); PWY-7494 (choline degradation IV)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00098:0:0:\"Choline\";1:cpd00025:0:0:\"H2O2\";1:cpd00447:0:0:\"Betaine aldehyde\"'},\n",
       " {'abbreviation': 'R01023',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01023',\n",
       "   'MetaCyc: CHOLINE-O-ACETYLTRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:choline O-acetyltransferase; acetyl-CoA:choline O-acetyltransferase; choline O-acetyltransferase; choline acetylase; choline acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00098[0] <=> (1) cpd00010[0] + (1) cpd01367[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00098;cpd01367',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Choline[0] => (1) CoA[0] + (1) Acetylcholine[0]',\n",
       "  'deltag': -2.31,\n",
       "  'deltagerr': 0.24,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.6'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00098[0] => (1) cpd00010[0] + (1) cpd01367[0]',\n",
       "  'id': 'rxn00753',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:choline O-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00098:0:0:\"Choline\";1:cpd00010:0:0:\"CoA\";1:cpd01367:0:0:\"Acetylcholine\"'},\n",
       " {'abbreviation': 'R01026',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01026_c',\n",
       "   'ChlamyCyc: ACETYLCHOLINESTERASE-RXN',\n",
       "   'KEGG: R01026',\n",
       "   'MetaCyc: ACETYLCHOLINESTERASE-RXN',\n",
       "   'Name: AcCholE; Acetylcholine aectylhydrolase; acetyl.beta-methylcholinesterase; acetylcholine acetylhydrolase; acetylcholine hydrolase; acetylcholinesterase; acetylthiocholinesterase; choline esterase I; cholinesterase; true cholinesterase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01367[0] <=> (1) cpd00029[0] + (1) cpd00098[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00067;cpd00098;cpd01367',\n",
       "  'definition': '(1) H2O[0] + (1) Acetylcholine[0] => (1) Acetate[0] + (1) H+[0] + (1) Choline[0]',\n",
       "  'deltag': -4.99,\n",
       "  'deltagerr': 0.25,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.1.7', '3.1.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01367[0] => (1) cpd00029[0] + (1) cpd00067[0] + (1) cpd00098[0]',\n",
       "  'id': 'rxn00754',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetylcholine aectylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01367:0:0:\"Acetylcholine\";1:cpd00029:0:0:\"Acetate\";1:cpd00067:0:0:\"H+\";1:cpd00098:0:0:\"Choline\"'},\n",
       " {'abbreviation': 'R01027',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01027',\n",
       "   'MetaCyc: CHOLINE-SULFOTRANSFERASE-RXN',\n",
       "   \"Name: 3'-Phosphoadenylylsulfate:choline sulfotransferase; 3'-phosphoadenylyl-sulfate:choline sulfotransferase; choline sulfotransferase; choline sulphokinase\"],\n",
       "  'code': '(1) cpd00044[0] + (1) cpd00098[0] <=> (1) cpd00045[0] + (1) cpd00681[0]',\n",
       "  'compound_ids': 'cpd00044;cpd00045;cpd00067;cpd00098;cpd00681',\n",
       "  'definition': '(1) 3-phosphoadenylylsulfate[0] + (1) Choline[0] => (1) Adenosine 3-5-bisphosphate[0] + (1) H+[0] + (1) Choline sulfate[0]',\n",
       "  'deltag': -14.03,\n",
       "  'deltagerr': 5.21,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.8.2.6'],\n",
       "  'equation': '(1) cpd00044[0] + (1) cpd00098[0] => (1) cpd00045[0] + (1) cpd00067[0] + (1) cpd00681[0]',\n",
       "  'id': 'rxn00755',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"3'-phosphoadenylyl-sulfate:choline sulfotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00044:0:0:\"3-phosphoadenylylsulfate\";-1:cpd00098:0:0:\"Choline\";1:cpd00045:0:0:\"Adenosine 3-5-bisphosphate\";1:cpd00067:0:0:\"H+\";1:cpd00681:0:0:\"Choline sulfate\"'},\n",
       " {'abbreviation': 'R01028',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CHOLSH',\n",
       "   'KEGG: R01028',\n",
       "   'MetaCyc: CHOLINE-SULFATASE-RXN',\n",
       "   'iJN746: CHOLSH',\n",
       "   'Name: Choline sulfate sulfohydrolase; choline sulphatase; choline-sulfatase; choline-sulfate sulfohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00681[0] <=> (1) cpd00048[0] + (1) cpd00098[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00048;cpd00067;cpd00098;cpd00681',\n",
       "  'definition': '(1) H2O[0] + (1) Choline sulfate[0] => (1) Sulfate[0] + (1) H+[0] + (1) Choline[0]',\n",
       "  'deltag': -16.31,\n",
       "  'deltagerr': 2.96,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.6.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00681[0] => (1) cpd00048[0] + (1) cpd00067[0] + (1) cpd00098[0]',\n",
       "  'id': 'rxn00756',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Choline sulfate sulfohydrolase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Choline-Degradation (Choline Degradation); Degradation (Degradation/Utilization/Assimilation); P542-PWY (choline-<i>O</i>-sulfate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00681:0:0:\"Choline sulfate\";1:cpd00048:0:0:\"Sulfate\";1:cpd00067:0:0:\"H+\";1:cpd00098:0:0:\"Choline\"'},\n",
       " {'abbreviation': 'R01029',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01029', 'Name: Acylcholine acylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01226[0] <=> (1) cpd00049[0] + (1) cpd00098[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00049;cpd00067;cpd00098;cpd01226',\n",
       "  'definition': '(1) H2O[0] + (1) Acylcholine[0] <=> (1) Carboxylic acid[0] + (1) H+[0] + (1) Choline[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01226[0] <=> (1) cpd00049[0] + (1) cpd00067[0] + (1) cpd00098[0]',\n",
       "  'id': 'rxn00757',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acylcholine acylhydrolase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01226:0:0:\"Acylcholine\";1:cpd00049:0:0:\"Carboxylic acid\";1:cpd00067:0:0:\"H+\";1:cpd00098:0:0:\"Choline\"'},\n",
       " {'abbreviation': 'R01030',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01030_c',\n",
       "   'AraGEM: R_R01030_c',\n",
       "   'BiGG: GPDDA1; GPDDA1pp',\n",
       "   'KEGG: R01030',\n",
       "   'MetaCyc: 3.1.4.2-RXN',\n",
       "   'Name: Glycerophosphodiester phosphodiesterase (Glycerophosphocholine); Glycerophosphodiester phosphodiesterase (sn-Glycero-3-phosphocholine); R_sn_Glycero_3_phosphocholine_glycerophosphohydrolase; glycerolphosphorylcholine phosphodiesterase; glycerophosphinicocholine diesterase; glycerophosphocholine phosphodiesterase; glycerophosphohydrolase; glycerylphosphorylcholinediesterase; sn-Glycero-3-phosphocholine glycerophosphohydrolase; sn-glycero-3-phosphocholine glycerophosphohydrolase; sn-glycero-3-phosphorylcholine diesterase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00507[0] <=> (1) cpd00080[0] + (1) cpd00098[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00080;cpd00098;cpd00507',\n",
       "  'definition': '(1) H2O[0] + (1) Glycerophosphocholine[0] => (1) H+[0] + (1) Glycerol-3-phosphate[0] + (1) Choline[0]',\n",
       "  'deltag': -14.34,\n",
       "  'deltagerr': 2.92,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.4.2', '3.1.4.46'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00507[0] => (1) cpd00067[0] + (1) cpd00080[0] + (1) cpd00098[0]',\n",
       "  'id': 'rxn00758',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29958;rxn31410;rxn33398',\n",
       "  'name': 'sn-Glycero-3-phosphocholine glycerophosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); PWY-7367 (phosphatidylcholine resynthesis via glycerophosphocholine)',\n",
       "   'KEGG: rn00564 (Glycerophospholipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00507:0:0:\"Glycerophosphocholine\";1:cpd00067:0:0:\"H+\";1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00098:0:0:\"Choline\"'},\n",
       " {'abbreviation': '3-Chloro-D-alanine chloride-lyase (deaminating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: 3-Chloro-D-alanine chloride-lyase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01713[0] <=> (1) cpd00013[0] + (1) cpd00020[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00067;cpd01713',\n",
       "  'definition': '(1) H2O[0] + (1) 3-Chloro-D-alanine[0] <=> (1) NH3[0] + (1) Pyruvate[0] + (1) H+[0]',\n",
       "  'deltag': 10.18,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.5.1.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01713[0] <=> (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00067[0]',\n",
       "  'id': 'rxn00759',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-Chloro-D-alanine chloride-lyase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:Cl:-1|CI:1',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01713:0:0:\"3-Chloro-D-alanine\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\"'},\n",
       " {'abbreviation': 'rxn00760',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00013[0] + (1) cpd00020[0] <=> (1) cpd00001[0] + (1) cpd01714[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00067;cpd01714',\n",
       "  'definition': '(1) NH3[0] + (1) Pyruvate[0] + (1) H+[0] <=> (1) H2O[0] + (1) 3-Chloro-L-alanine[0]',\n",
       "  'deltag': -10.18,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00067[0] <=> (1) cpd00001[0] + (1) cpd01714[0]',\n",
       "  'id': 'rxn00760',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00760',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:Cl:1|CI:-1',\n",
       "  'stoichiometry': '-1:cpd00013:0:0:\"NH3\";-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00067:0:0:\"H+\";1:cpd00001:0:0:\"H2O\";1:cpd01714:0:0:\"3-Chloro-L-alanine\"'},\n",
       " {'abbreviation': 'dechlorinating, decarboxylating)',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: 2-Chlorobenzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating,; dechlorinating, decarboxylating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd01574[0] <=> (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00007;cpd00011;cpd00077;cpd01574',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (1) 2-Chlorobenzoate[0] <=> (1) NAD[0] + (1) CO2[0] + (1) Catechol[0]',\n",
       "  'deltag': -104.4,\n",
       "  'deltagerr': 1.64,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.14.12.13'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd01574[0] <=> (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00077[0]',\n",
       "  'id': 'rxn00761',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Chlorobenzoate,NADH:oxygen oxidoreductase (1,2-hydroxylating,',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:Cl:-1/H:1|CI:2',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-1:cpd01574:0:0:\"2-Chlorobenzoate\";1:cpd00003:0:0:\"NAD\";1:cpd00011:0:0:\"CO2\";1:cpd00077:0:0:\"Catechol\"'},\n",
       " {'abbreviation': 'R01034',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCDEH-RXN',\n",
       "   'BiGG: GLYCDx',\n",
       "   'BrachyCyc: GLYCDEH-RXN',\n",
       "   'KEGG: R01034',\n",
       "   'MetaCyc: GLYCDEH-RXN; RXN-11708',\n",
       "   'Name: Glycerol dehydrogenase; NAD-linked glycerol dehydrogenase; glycerin dehydrogenase; glycerol dehydrogenase; glycerol:NAD+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00100[0] <=> (1) cpd00004[0] + (1) cpd00157[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00100;cpd00157',\n",
       "  'definition': '(1) NAD[0] + (1) Glycerol[0] <=> (1) NADH[0] + (1) H+[0] + (1) Glycerone[0]',\n",
       "  'deltag': 5.73,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.6'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00100[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00157[0]',\n",
       "  'id': 'rxn00762',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glycerol:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Alcohol-Degradation (Alcohol Degradation); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); GLYCEROL-DEG (Glycerol Degradation); GLYCEROLMETAB-PWY (glycerol degradation V); GOLPDLCAT-PWY (superpathway of glycerol degradation to 1,3-propanediol); PWY-6131 (glycerol degradation II); PWY-6626 (cytidine-5'-diphosphate-glycerol biosynthesis); PWY-7003 (glycerol degradation to butanol)\",\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00100:0:0:\"Glycerol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00157:0:0:\"Glycerone\"'},\n",
       " {'abbreviation': 'R01036',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ALDEHYDE-REDUCTASE-RXN',\n",
       "   'BiGG: ALCD19',\n",
       "   'CornCyc: ALDEHYDE-REDUCTASE-RXN',\n",
       "   'KEGG: R01036',\n",
       "   'MetaCyc: RXN-11709',\n",
       "   'Name: Aldose reductase; Glycerol:NAD+ oxidoreductase; Polyol dehydrogenase (NADP+); alcohol dehydrogenase (glycerol); glyceraldehyde reductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00100[0] <=> (1) cpd00004[0] + (1) cpd00448[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00100;cpd00448',\n",
       "  'definition': '(1) NAD[0] + (1) Glycerol[0] <=> (1) NADH[0] + (1) H+[0] + (1) D-Glyceraldehyde[0]',\n",
       "  'deltag': 9.31,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.-', '1.1.1.1', '1.1.1.21', '1.1.1.72'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00100[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00448[0]',\n",
       "  'id': 'rxn00763',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn37672',\n",
       "  'name': 'Glycerol:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); PWY-6626 (cytidine-5'-diphosphate-glycerol biosynthesis)\",\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00100:0:0:\"Glycerol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00448:0:0:\"D-Glyceraldehyde\"'},\n",
       " {'abbreviation': 'R01039',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GLYCDy',\n",
       "   'KEGG: R01039',\n",
       "   'MetaCyc: GLYCEROL-2-DEHYDROGENASE-NADP+-RXN; RXN-11708',\n",
       "   'iMM904: GLYCDy',\n",
       "   'Name: DHA oxidoreductase; Glycerol dehydrogenase (NADP-dependent); Glycerol:NADP+ 2-oxidoreductase; dihydroxyacetone (reduced nicotinamide adenine dinucleotide phosphate) reductase; dihydroxyacetone reductase; dihydroxyacetone reductase (NADPH); glycerol 2-dehydrogenase (NADP); glycerol 2-dehydrogenase (NADP+); glycerol:NADP+ 2-oxidoreductase (glycerone-forming)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00100[0] <=> (1) cpd00005[0] + (1) cpd00157[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00100;cpd00157',\n",
       "  'definition': '(1) NADP[0] + (1) Glycerol[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Glycerone[0]',\n",
       "  'deltag': 5.73,\n",
       "  'deltagerr': 0.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.156'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00100[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00157[0]',\n",
       "  'id': 'rxn00764',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glycerol:NADP+ 2-oxidoreductase (glycerone-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); PWY-6626 (cytidine-5'-diphosphate-glycerol biosynthesis)\",\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00100:0:0:\"Glycerol\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00157:0:0:\"Glycerone\"'},\n",
       " {'abbreviation': 'R01041',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ALDEHYDE-REDUCTASE-RXN',\n",
       "   'BiGG: ALCD19_D; ALCD19y',\n",
       "   'CornCyc: ALDEHYDE-REDUCTASE-RXN',\n",
       "   'KEGG: R01041',\n",
       "   'MetaCyc: GLYCEROL-DEHYDROGENASE-NADP+-RXN; RXN-11709',\n",
       "   'Name: Aldose reductase; Glycerol:NADP+ oxidoreductase; Polyol dehydrogenase (NADP+); alcohol dehydrogenase (D-glyceraldehyde to glycerol, NADP); alcohol dehydrogenase (glycerol, NADP); glycerol dehydrogenase (NADP); glycerol dehydrogenase (NADP+); glycerol:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00100[0] <=> (1) cpd00005[0] + (1) cpd00448[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00100;cpd00448',\n",
       "  'definition': '(1) NADP[0] + (1) Glycerol[0] <=> (1) NADPH[0] + (1) H+[0] + (1) D-Glyceraldehyde[0]',\n",
       "  'deltag': 9.31,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.2', '1.1.1.21', '1.1.1.372', '1.1.1.72'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00100[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00448[0]',\n",
       "  'id': 'rxn00765',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn37671',\n",
       "  'name': 'Glycerol:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); PWY-6626 (cytidine-5'-diphosphate-glycerol biosynthesis)\",\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00100:0:0:\"Glycerol\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00448:0:0:\"D-Glyceraldehyde\"'},\n",
       " {'abbreviation': 'R01043',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCEROL-2-PHOSPHATASE-RXN',\n",
       "   'BiGG: G2PPpp',\n",
       "   'EcoCyc: GLYCEROL-2-PHOSPHATASE-RXN',\n",
       "   'KEGG: R01043',\n",
       "   'MetaCyc: GLYCEROL-2-PHOSPHATASE-RXN',\n",
       "   'Name: 2-glycerophosphatase; Glycerol-2-phosphate phosphohydrolase; beta-glycerophosphatase; beta-glycerophosphate phosphatase; glycerol 2-phosphate phosphatase; glycerol-2-phosphatase; glycerol-2-phosphate phosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01908[0] <=> (1) cpd00009[0] + (1) cpd00100[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00100;cpd01908',\n",
       "  'definition': '(1) H2O[0] + (1) Glycerol 2-phosphate[0] <=> (1) Phosphate[0] + (1) Glycerol[0]',\n",
       "  'deltag': -6.09,\n",
       "  'deltagerr': 0.58,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.19'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01908[0] <=> (1) cpd00009[0] + (1) cpd00100[0]',\n",
       "  'id': 'rxn00766',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29963',\n",
       "  'name': 'Glycerol-2-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01908:0:0:\"Glycerol 2-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00100:0:0:\"Glycerol\"'},\n",
       " {'abbreviation': 'R01044',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01044',\n",
       "   'MetaCyc: 2.7.1.79-RXN',\n",
       "   'Name: PPi-glycerol phosphotransferase; Pyrophosphate:glycerol 1-phosphotransferase; diphosphate-glycerol phosphotransferase; diphosphate:glycerol 1-phosphotransferase; pyrophosphate-glycerol phosphotransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00100[0] <=> (1) cpd00009[0] + (1) cpd00080[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00012;cpd00067;cpd00080;cpd00100',\n",
       "  'definition': '(1) PPi[0] + (1) Glycerol[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) Glycerol-3-phosphate[0]',\n",
       "  'deltag': -0.88,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.79'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00100[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00080[0]',\n",
       "  'id': 'rxn00767',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17678',\n",
       "  'name': 'diphosphate:glycerol 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00100:0:0:\"Glycerol\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00080:0:0:\"Glycerol-3-phosphate\"'},\n",
       " {'abbreviation': 'R01047',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01047',\n",
       "   'MetaCyc: GLYCEROL-DEHYDRATASE-RXN',\n",
       "   'Name: glycerol dehydrase; glycerol dehydratase; glycerol hydro-lyase; glycerol hydro-lyase (3-hydroxypropanal-forming)'],\n",
       "  'code': '(1) cpd00100[0] <=> (1) cpd00001[0] + (1) cpd00714[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00100;cpd00714',\n",
       "  'definition': '(1) Glycerol[0] => (1) H2O[0] + (1) 3-Hydroxypropanal[0]',\n",
       "  'deltag': -8.29,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.30'],\n",
       "  'equation': '(1) cpd00100[0] => (1) cpd00001[0] + (1) cpd00714[0]',\n",
       "  'id': 'rxn00768',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'glycerol hydro-lyase (3-hydroxypropanal-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Alcohol-Degradation (Alcohol Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); GLYCEROL-DEG (Glycerol Degradation); GOLPDLCAT-PWY (superpathway of glycerol degradation to 1,3-propanediol); PWY-6130 (glycerol degradation III); PWY-7385 (1,3-propanediol biosynthesis (engineered))',\n",
       "   'KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00100:0:0:\"Glycerol\";1:cpd00001:0:0:\"H2O\";1:cpd00714:0:0:\"3-Hydroxypropanal\"'},\n",
       " {'abbreviation': 'R01048',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01048', 'Name: Glycerol hydro-lyase'],\n",
       "  'code': '(1) cpd00100[0] <=> (1) cpd00001[0] + (1) cpd03571[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00100;cpd03571',\n",
       "  'definition': '(1) Glycerol[0] => (1) H2O[0] + (1) Lactaldehyde[0]',\n",
       "  'deltag': -9.24,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.30'],\n",
       "  'equation': '(1) cpd00100[0] => (1) cpd00001[0] + (1) cpd03571[0]',\n",
       "  'id': 'rxn00769',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glycerol hydro-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00100:0:0:\"Glycerol\";1:cpd00001:0:0:\"H2O\";1:cpd03571:0:0:\"Lactaldehyde\"'},\n",
       " {'abbreviation': 'R01049',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PRPPSYN-RXN',\n",
       "   'BrachyCyc: PRPPSYN-RXN',\n",
       "   'ChlamyCyc: PRPPSYN-RXN',\n",
       "   'KEGG: R01049',\n",
       "   'MetaCyc: PRPPSYN-RXN',\n",
       "   'Name: 5-phosphoribose pyrophosphorylase; 5-phosphoribosyl-1-pyrophosphate synthetase; 5-phosphoribosyl-alpha-1-pyrophosphate synthetase; ATP:D-ribose-5-phosphate diphosphotransferase; ATP:D-ribose-5-phosphate pyrophosphotransferase; PP-ribose P synthetase; PPRibP synthetase; PRPP synthetase; PRPPS; Ribose-Phosphate pyrophosphokinase; phosphoribosyl-diphosphate synthetase; phosphoribosylpyrophosphate synthase; phosphoribosylpyrophosphate synthetase; phosphoribosylpyrophosphatesynthetase; pyrophosphoribosylphosphate synthetase; ribophosphate pyrophosphokinase; ribose-5-phosphate pyrophosphokinase; ribose-phosphate diphosphokinase; ribose-phosphate pyrophosphokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00101[0] <=> (1) cpd00018[0] + (1) cpd00103[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00018;cpd00067;cpd00101;cpd00103',\n",
       "  'definition': '(1) ATP[0] + (1) ribose-5-phosphate[0] => (1) AMP[0] + (1) H+[0] + (1) PRPP[0]',\n",
       "  'deltag': 0.19,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.6.1'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00101[0] => (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00103[0]',\n",
       "  'id': 'rxn00770',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33995;rxn38273',\n",
       "  'name': 'ATP:D-ribose-5-phosphate diphosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-6404 (superpathway of mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY0-662 (PRPP biosynthesis); Sugar-Phosphate-Biosynthesis (Sugar Phosphate Biosynthesis)',\n",
       "   'KEGG: rn00030 (Pentose phosphate pathway); rn00230 (Purine metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\";1:cpd00103:0:0:\"PRPP\"'},\n",
       " {'abbreviation': 'R01050',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01050',\n",
       "   'MetaCyc: PHOSPHORIBOKINASE-RXN',\n",
       "   'iRS1563: R_R01050_c',\n",
       "   'Name: ATP:D-ribose-5-phosphate 1-phosphotransferase; phosphoribokinase; phosphoribokinase (phosphorylating)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00101[0] <=> (1) cpd00008[0] + (1) cpd00847[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00101;cpd00847',\n",
       "  'definition': '(1) ATP[0] + (1) ribose-5-phosphate[0] => (1) ADP[0] + (1) H+[0] + (1) Ribose 1,5-bisphosphate[0]',\n",
       "  'deltag': -2.17,\n",
       "  'deltagerr': 0.33,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.18'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00101[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00847[0]',\n",
       "  'id': 'rxn00771',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:D-ribose-5-phosphate 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00847:0:0:\"Ribose 1,5-bisphosphate\"'},\n",
       " {'abbreviation': 'R01051',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01051_c',\n",
       "   'AraGEM: R_R01051_c',\n",
       "   'BiGG: RBKr',\n",
       "   'KEGG: R01051',\n",
       "   'MetaCyc: RIBOKIN-RXN',\n",
       "   'Name: ATP:D-ribose 5-phosphotransferase; Ribokinase; ribokinase reversible'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00105[0] <=> (1) cpd00008[0] + (1) cpd00101[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00101;cpd00105',\n",
       "  'definition': '(1) ATP[0] + (1) D-Ribose[0] => (1) ADP[0] + (1) H+[0] + (1) ribose-5-phosphate[0]',\n",
       "  'deltag': -6.42,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.15'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00105[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00101[0]',\n",
       "  'id': 'rxn00772',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33469',\n",
       "  'name': 'ATP:D-ribose 5-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); RIBOKIN-PWY (ribose phosphorylation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00030 (Pentose phosphate pathway)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00105:0:0:\"D-Ribose\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00101:0:0:\"ribose-5-phosphate\"'},\n",
       " {'abbreviation': 'R01052',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.7.35-RXN',\n",
       "   'BrachyCyc: 2.7.7.35-RXN',\n",
       "   'KEGG: R01052',\n",
       "   'MetaCyc: 2.7.7.35-RXN',\n",
       "   'TS_Athaliana: rxn73_c; rxn73_p',\n",
       "   'Name: ADP ribose phosphorylase; ADP:D-ribose-5-phosphate adenylyltransferase; Ribose-5-phosphate adenylyltransferase; adenosine diphosphoribose phosphorylase; ribose-5-phosphate adenylyltransferase'],\n",
       "  'code': '(1) cpd00008[0] + (1) cpd00101[0] <=> (1) cpd00009[0] + (1) cpd00251[0]',\n",
       "  'compound_ids': 'cpd00008;cpd00009;cpd00101;cpd00251',\n",
       "  'definition': '(1) ADP[0] + (1) ribose-5-phosphate[0] <= (1) Phosphate[0] + (1) ADPribose[0]',\n",
       "  'deltag': -0.25,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.7.7.35'],\n",
       "  'equation': '(1) cpd00008[0] + (1) cpd00101[0] <= (1) cpd00009[0] + (1) cpd00251[0]',\n",
       "  'id': 'rxn00773',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn14360;rxn33061',\n",
       "  'name': 'ADP:D-ribose-5-phosphate adenylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00008:0:0:\"ADP\";-1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00251:0:0:\"ADPribose\"'},\n",
       " {'abbreviation': 'R01053',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01053',\n",
       "   'MetaCyc: RIBOSE-5-PHOSPHATE--AMMONIA-LIGASE-RXN',\n",
       "   'Name: 5-phosphoribosylamine synthetase; Ribose-5-phosphate:ammonia ligase (ADP-forming); ammonia-ribose 5-phosphate aminotransferase; ribose 5-phosphate aminotransferase; ribose-5-phosphate-ammonia ligase; ribose-5-phosphate:ammonia ligase (ADP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00101[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd01982[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00013;cpd00067;cpd00101;cpd01982',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) ribose-5-phosphate[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) 5-Phosphoribosylamine[0]',\n",
       "  'deltag': -7.7,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.4.7'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00101[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd01982[0]',\n",
       "  'id': 'rxn00774',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Ribose-5-phosphate:ammonia ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd01982:0:0:\"5-Phosphoribosylamine\"'},\n",
       " {'abbreviation': 'R01054',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-9925; RXN0-1441',\n",
       "   'BiGG: ADPRDP',\n",
       "   'BrachyCyc: RXN0-1441',\n",
       "   'KEGG: R01054',\n",
       "   'MetaCyc: RXN-9925; RXN0-1441',\n",
       "   'Name: ADP-ribose diphosphatase; ADP-ribose phosphohydrolase; ADP-ribose pyrophosphatase; ADP-ribose ribophosphohydrolase; ADPR-PPase; ADPribose diphosphatase; ADPribose pyrophosphatase; ADPribose ribophosphohydrolase; Adenosine diphosphoribose pyrophosphatase; adenosine diphosphoribose pyrophosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00251[0] <=> (1) cpd00018[0] + (1) cpd00101[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00018;cpd00067;cpd00101;cpd00251',\n",
       "  'definition': '(1) H2O[0] + (1) ADPribose[0] => (1) AMP[0] + (2) H+[0] + (1) ribose-5-phosphate[0]',\n",
       "  'deltag': -5.49,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.-', '3.6.1.13', '3.6.1.21', '3.6.1.53'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00251[0] => (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd00101[0]',\n",
       "  'id': 'rxn00775',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn14038;rxn28560',\n",
       "  'name': 'ADP-ribose ribophosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00251:0:0:\"ADPribose\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd00101:0:0:\"ribose-5-phosphate\"'},\n",
       " {'abbreviation': 'R01055',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PSEUDOURIDYLATE-SYNTHASE-RXN; RXN0-5398',\n",
       "   'BrachyCyc: PSEUDOURIDYLATE-SYNTHASE-RXN',\n",
       "   'ChlamyCyc: PSEUDOURIDYLATE-SYNTHASE-RXN',\n",
       "   'KEGG: R01055',\n",
       "   'MetaCyc: RXN0-5398',\n",
       "   \"Name: 5-ribosyluracil 5-phosphate synthetase; Pseudouridine synthase; Pseudouridylate synthase; TRUB PSEUDOURIDINE (PSI) SYNTHASE HOMOLOG 1; Uracil hydro-lyase (adding D-ribose 5-phosphate); Uracil hydrolyase; pseudouridine monophosphate synthetase; pseudouridylate synthase; pseudouridylate synthetase; pseudouridylic acid synthetase; upsilonUMP synthetase; uracil hydro-lyase (adding D-ribose 5-phosphate); uracil hydro-lyase (adding D-ribose 5-phosphate, pseudouridine-5'-phosphate-forming); uracil hydro-lyase (adding D-ribose 5-phosphate;pseudouridine-5'-phosphate-forming)\"],\n",
       "  'code': '(1) cpd00092[0] + (1) cpd00101[0] <=> (1) cpd00001[0] + (1) cpd00859[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00092;cpd00101;cpd00859',\n",
       "  'definition': \"(1) Uracil[0] + (1) ribose-5-phosphate[0] <=> (1) H2O[0] + (1) Pseudouridine 5'-phosphate[0]\",\n",
       "  'deltag': -8.05,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.-.-', '4.2.1.70'],\n",
       "  'equation': '(1) cpd00092[0] + (1) cpd00101[0] <=> (1) cpd00001[0] + (1) cpd00859[0]',\n",
       "  'id': 'rxn00776',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28104;rxn28105;rxn30752',\n",
       "  'name': \"uracil hydro-lyase (adding D-ribose 5-phosphate;pseudouridine-5'-phosphate-forming)\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-6019 (pseudouridine degradation); Purine-Degradation (Purine Nucleotide Degradation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00092:0:0:\"Uracil\";-1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00001:0:0:\"H2O\";1:cpd00859:0:0:\"Pseudouridine 5\\'-phosphate\"'},\n",
       " {'abbreviation': 'R01056',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RIB5PISOM-RXN',\n",
       "   'BrachyCyc: RIB5PISOM-RXN',\n",
       "   'ChlamyCyc: RIB5PISOM-RXN',\n",
       "   'KEGG: R01056',\n",
       "   'MetaCyc: RIB5PISOM-RXN',\n",
       "   'Name: 5-phosphoribose isomerase; D-Ribose-5-phosphate ketol-isomerase; D-ribose 5-phosphate isomerase; D-ribose-5-phosphate aldose-ketose-isomerase; D-ribose-5-phosphate ketol-isomerase; Phosphopentoseisomerase; Phosphoriboisomerase; RPI; Ribose 5-phosphate epimerase; Ribose 5-phosphate isomerase; Ribose phosphate isomerase; phosphopentosisomerase; ribose 5-phosphate epimerase; ribose 5-phosphate isomerase A; ribose phosphate isomerase; ribose-5-phosphate isomerase; ribose-5-phosphateisomerase'],\n",
       "  'code': '(1) cpd00101[0] <=> (1) cpd00171[0]',\n",
       "  'compound_ids': 'cpd00101;cpd00171',\n",
       "  'definition': '(1) ribose-5-phosphate[0] <=> (1) D-Ribulose5-phosphate[0]',\n",
       "  'deltag': 1.04,\n",
       "  'deltagerr': 0.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.6'],\n",
       "  'equation': '(1) cpd00101[0] <=> (1) cpd00171[0]',\n",
       "  'id': 'rxn00777',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28141;rxn38284',\n",
       "  'name': 'D-ribose-5-phosphate aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CALVIN-PWY (Calvin-Benson-Bassham cycle); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethylene-Biosynthesis (Ethylene Biosynthesis); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Formaldehyde-Assimilation (Formaldehyde Assimilation); HORMONE-SYN (Hormone Biosynthesis); NONOXIPENT-PWY (pentose phosphate pathway (non-oxidative branch)); P124-PWY (Bifidobacterium shunt); P185-PWY (formaldehyde assimilation III (dihydroxyacetone cycle)); PENTOSE-P-PWY (pentose phosphate pathway); PHOTOALL-PWY (oxygenic photosynthesis); PWY-1861 (formaldehyde assimilation II (assimilatory RuMP Cycle)); PWY-5723 (Rubisco shunt); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-6901 (superpathway of glucose and xylose degradation); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); Pentose-Phosphate-Cycle (Pentose Phosphate Pathways); Photosynthesis (); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00030 (Pentose phosphate pathway); rn00710 (Carbon fixation in photosynthetic organisms); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00171:0:0:\"D-Ribulose5-phosphate\"'},\n",
       " {'abbreviation': 'R01057',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['EcoCyc: PPENTOMUT-RXN',\n",
       "   'KEGG: R01057',\n",
       "   'MetaCyc: PPENTOMUT-RXN',\n",
       "   'RiceCyc: PPENTOMUT-RXN',\n",
       "   'SorghumCyc: PPENTOMUT-RXN',\n",
       "   'Name: D-Ribose 1,5-phosphomutase; D-ribose 1,5-phosphomutase; PPM; Phosphopentomutase; alpha-D-Ribose 1-phosphate 1,6-phosphomutase; alpha-D-glucose-1,6-bisphosphate:deoxy-D-ribose-1-phosphate phosphotransferase; alpha-D-ribose 1,5-phosphomutase; deoxyribomutase; deoxyribose phosphomutase; phosphodeoxyribomutase; phosphoribomutase'],\n",
       "  'code': '(1) cpd00475[0] <=> (1) cpd00101[0]',\n",
       "  'compound_ids': 'cpd00101;cpd00475',\n",
       "  'definition': '(1) Ribose 1-phosphate[0] <=> (1) ribose-5-phosphate[0]',\n",
       "  'deltag': -3.38,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.2.2', '5.4.2.7'],\n",
       "  'equation': '(1) cpd00475[0] <=> (1) cpd00101[0]',\n",
       "  'id': 'rxn00778',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32821',\n",
       "  'name': 'D-Ribose 1,5-phosphomutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY0-1296 (purine ribonucleosides degradation); Purine-Degradation (Purine Nucleotide Degradation)',\n",
       "   'KEGG: rn00030 (Pentose phosphate pathway); rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00475:0:0:\"Ribose 1-phosphate\";1:cpd00101:0:0:\"ribose-5-phosphate\"'},\n",
       " {'abbreviation': 'R01058',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.2.1.9-RXN',\n",
       "   'BiGG: GAPDy1',\n",
       "   'BrachyCyc: 1.2.1.9-RXN',\n",
       "   'KEGG: R01058',\n",
       "   'MetaCyc: 1.2.1.9-RXN',\n",
       "   'Name: D-glyceraldehyde 3-phosphate:NADP+ oxidoreductase; D-glyceraldehyde-3-phosphate:NADP+ oxidoreductase; NADP-glyceraldehyde phosphate dehydrogenase; NADP-glyceraldehyde-3-phosphate dehydrogenase; dehydrogenase, glyceraldehyde phosphate (nicotinamide adenine dinucleotide phosphate); glyceraldehyde 3-phosphate dehydrogenase (NADP); glyceraldehyde phosphate dehydrogenase (NADP); glyceraldehyde-3-phosphate dehydrogenase (NADP); glyceraldehyde-3-phosphate dehydrogenase (NADP+); glyceraldehyde-3-phosphate dehydrogenase (nadp+); glyceraldehyde-3-phosphate:NADP reductase; nonphosphorylating glyceraldehyde-3-phosphate dehydrogenase; triosephosphate dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00102[0] <=> (1) cpd00005[0] + (1) cpd00169[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00067;cpd00102;cpd00169',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) Glyceraldehyde3-phosphate[0] <=> (1) NADPH[0] + (2) H+[0] + (1) 3-Phosphoglycerate[0]',\n",
       "  'deltag': -9.33,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00102[0] <=> (1) cpd00005[0] + (2) cpd00067[0] + (1) cpd00169[0]',\n",
       "  'id': 'rxn00779',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-glyceraldehyde 3-phosphate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); GLYCOLYSIS-VARIANTS (Glycolysis); PWY-1042 (glycolysis IV (plant cytosol)); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00030 (Pentose phosphate pathway); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\";1:cpd00005:0:0:\"NADPH\";2:cpd00067:0:0:\"H+\";1:cpd00169:0:0:\"3-Phosphoglycerate\"'},\n",
       " {'abbreviation': 'R01059',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01059',\n",
       "   'MetaCyc: TRIOKINASE-RXN',\n",
       "   'Name: ATP:D-glyceraldehyde 3-phosphotransferase; triokinase; triose kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00448[0] <=> (1) cpd00008[0] + (1) cpd00102[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00102;cpd00448',\n",
       "  'definition': '(1) ATP[0] + (1) D-Glyceraldehyde[0] => (1) ADP[0] + (1) H+[0] + (1) Glyceraldehyde3-phosphate[0]',\n",
       "  'deltag': -5.44,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.28'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00448[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00102[0]',\n",
       "  'id': 'rxn00780',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:D-glyceraldehyde 3-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY66-373 (sucrose degradation V (sucrose &alpha;-glucosidase)); SUCROSE-DEG (Sucrose Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00051 (Fructose and mannose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00448:0:0:\"D-Glyceraldehyde\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\"'},\n",
       " {'abbreviation': 'R01061',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GAPOXNPHOSPHN-RXN',\n",
       "   'BiGG: GAPD; GAPD_NAD',\n",
       "   'BrachyCyc: GAPDHSYNEC-RXN; GAPOXNPHOSPHN-RXN',\n",
       "   'KEGG: R01061',\n",
       "   'MetaCyc: GAPDHSYNEC-RXN; GAPOXNPHOSPHN-RXN',\n",
       "   'Name: 3-phosphoglyceraldehyde dehydrogenase; D-Glyceraldehyde-3-phosphate:NAD+ oxidoreductase(phosphorylating); D-glyceraldehyde-3-phosphate:NAD+ oxidoreductase (phosphorylating); GAPDH; Glyceraldehyde 3-phosphate dehydrogenase (phosphorylating); NAD+-dependent glyceraldehyde phosphate dehydrogenase; NAD-dependent glyceraldehyde-3-phosphate dehydrogenase; NADH-glyceraldehyde phosphate dehydrogenase; Triosephosphate dehydrogenase; dehydrogenase, glyceraldehyde phosphate; glyceraldehyde 3-phosphate dehydrogenase; glyceraldehyde 3-phosphate dehydrogenase (phosphorylating); glyceraldehyde 3-phosphate oxidation; glyceraldehyde phosphate dehydrogenase (NAD+); glyceraldehyde-3-P-dehydrogenase; glyceraldehyde-3-phosphate dehydrogenase; glyceraldehyde-3-phosphate dehydrogenase (NAD); glyceraldehyde-3-phosphate dehydrogenase (NAD+); glyceraldehyde-3-phosphate dehydrogenase (phosphorylating); glyceraldehyde-3-phosphatedehydrogenase; phosphoglyceraldehyde dehydrogenase; triosephosphate dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00009[0] + (1) cpd00102[0] <=> (1) cpd00004[0] + (1) cpd00203[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00009;cpd00067;cpd00102;cpd00203',\n",
       "  'definition': '(1) NAD[0] + (1) Phosphate[0] + (1) Glyceraldehyde3-phosphate[0] <=> (1) NADH[0] + (1) H+[0] + (1) 1,3-Bisphospho-D-glycerate[0]',\n",
       "  'deltag': 1.25,\n",
       "  'deltagerr': 0.1,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.12', '1.2.1.59'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00009[0] + (1) cpd00102[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00203[0]',\n",
       "  'id': 'rxn00781',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19762;rxn27871;rxn38113',\n",
       "  'name': 'D-glyceraldehyde-3-phosphate:NAD+ oxidoreductase (phosphorylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAEROFRUCAT-PWY (homolactic fermentation); ANAGLYCOLYSIS-PWY (glycolysis III (from glucose)); Acetate-Formation (Fermentation to Acetate); Alcohol-Biosynthesis (Fermentation to Alcohols); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Entner-Duodoroff-Pathways (Entner-Duodoroff Pathways); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-Lactate (Fermentation to Lactate); Formaldehyde-Assimilation (Formaldehyde Assimilation); GLUCONEO-PWY (gluconeogenesis I); GLYCOLYSIS (glycolysis I (from glucose 6-phosphate)); GLYCOLYSIS-E-D (superpathway of glycolysis and the Entner-Doudoroff pathway); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYCOLYSIS-VARIANTS (Glycolysis); Gluconeogenesis (Gluconeogenesis); HEXITOLDEGSUPER-PWY (superpathway of hexitol degradation (bacteria)); P122-PWY (heterolactic fermentation); P124-PWY (Bifidobacterium shunt); P185-PWY (formaldehyde assimilation III (dihydroxyacetone cycle)); P441-PWY (superpathway of <i>N</i>-acetylneuraminate degradation); P461-PWY (hexitol fermentation to lactate, formate, ethanol and acetate); PWY-1042 (glycolysis IV (plant cytosol)); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-5484 (glycolysis II (from fructose 6-phosphate)); PWY-6901 (superpathway of glucose and xylose degradation); PWY-7003 (glycerol degradation to butanol); PWY-7345 (superpathway of anaerobic sucrose degradation); PWY-8004 (Entner-Doudoroff pathway I); PWY66-399 (gluconeogenesis III); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUCROSE-DEG (Sucrose Degradation); SUCSYN-PWY (sucrose biosynthesis I (from photosynthesis)); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sucrose-Biosynthesis (Sucrose Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00710 (Carbon fixation in photosynthetic organisms); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00009:0:0:\"Phosphate\";-1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00203:0:0:\"1,3-Bisphospho-D-glycerate\"'},\n",
       " {'abbreviation': 'R01063',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.2.1.13-RXN',\n",
       "   'BiGG: GAPD(nadp); GAPD_NADP',\n",
       "   'BrachyCyc: 1.2.1.13-RXN; GAPDHSYNEC-RXN',\n",
       "   'KEGG: R01063',\n",
       "   'MetaCyc: 1.2.1.13-RXN; GAPDHSYNEC-RXN',\n",
       "   'Name: D-Glyceraldehyde-3-phosphate:NADP+ oxidoreductase(phosphorylating); D-glyceraldehyde-3-phosphate:NADP+ oxidoreductase (phosphorylating); GAPDH; NADP+-dependent glyceraldehyde phosphate dehydrogenase; NADP+-glyceraldehyde phosphate dehydrogenase; NADP+-glyceraldehyde-3-phosphate dehydrogenase; NADP+-triose phosphate dehydrogenase; dehydrogenase, glyceraldehyde phosphate (nicotinamide adenine dinucleotide phosphate) (phosphorylating); glyceraldehyde phosphate dehydrogenase (nicotinamide adenine dinucleotide phosphate) (phosphorylating); glyceraldehyde-3-phosphate dehydrogenase (NAD(P)+) (phosphorylating); glyceraldehyde-3-phosphate dehydrogenase (NADP+) (phosphorylating); triosephosphate dehydrogenase (NADP+)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00009[0] + (1) cpd00102[0] <=> (1) cpd00005[0] + (1) cpd00203[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00009;cpd00067;cpd00102;cpd00203',\n",
       "  'definition': '(1) NADP[0] + (1) Phosphate[0] + (1) Glyceraldehyde3-phosphate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) 1,3-Bisphospho-D-glycerate[0]',\n",
       "  'deltag': 1.25,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.13', '1.2.1.59'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00009[0] + (1) cpd00102[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00203[0]',\n",
       "  'id': 'rxn00782',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17199;rxn37888',\n",
       "  'name': 'D-glyceraldehyde-3-phosphate:NADP+ oxidoreductase (phosphorylating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CALVIN-PWY (Calvin-Benson-Bassham cycle); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CO2-Fixation (CO<sub>2</sub> Fixation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethylene-Biosynthesis (Ethylene Biosynthesis); Fermentation (); HORMONE-SYN (Hormone Biosynthesis); PHOTOALL-PWY (oxygenic photosynthesis); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7218 (photosynthetic 3-hydroxybutanoate biosynthesis (engineered)); Photosynthesis (); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Storage-Compounds-Biosynthesis (Storage Compound Biosynthesis)',\n",
       "   'KEGG: rn00010 (Glycolysis / Gluconeogenesis); rn00710 (Carbon fixation in photosynthetic organisms); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00009:0:0:\"Phosphate\";-1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00203:0:0:\"1,3-Bisphospho-D-glycerate\"'},\n",
       " {'abbreviation': 'R01064',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: DDPGALA',\n",
       "   'EcoCyc: DEHYDDEOXPHOSGALACT-ALDOL-RXN',\n",
       "   'KEGG: R01064',\n",
       "   'MetaCyc: DEHYDDEOXPHOSGALACT-ALDOL-RXN',\n",
       "   'iAF1260: DDPGALA',\n",
       "   'Name: (KDPGal)aldolase; 2-Dehydro-3-deoxy-D-galactonate-6-phosphate; 2-dehydro-3-deoxy-6-phospho-D-galactonate D-glyceraldehyde-3-phospho-lyase (pyruvate-forming); 2-dehydro-3-deoxy-6-phosphogalactonate aldolase; 2-dehydro-3-deoxy-D-galactonate-6-phosphate D-glyceraldehyde-3-phosphate-lyase; 2-dehydro-3-deoxy-D-galactonate-6-phosphate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming); 2-keto-3-deoxy-6-phosphogalactonic acid aldolase; 2-keto-3-deoxy-6-phosphogalactonic aldolase; 6-phospho-2-keto-3-deoxygalactonate aldolase; D-glyceraldehyde-3-phosphate-lyase; phospho-2-keto-3-deoxygalactonate aldolase; phospho-2-keto-3-deoxygalactonic aldolase'],\n",
       "  'code': '(1) cpd00945[0] <=> (1) cpd00020[0] + (1) cpd00102[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00102;cpd00945',\n",
       "  'definition': '(1) 2-Dehydro-3-deoxy-D-galactonate 6-phosphate[0] <=> (1) Pyruvate[0] + (1) Glyceraldehyde3-phosphate[0]',\n",
       "  'deltag': 3.32,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.21', '4.1.2.55'],\n",
       "  'equation': '(1) cpd00945[0] <=> (1) cpd00020[0] + (1) cpd00102[0]',\n",
       "  'id': 'rxn00783',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-dehydro-3-deoxy-D-galactonate-6-phosphate D-glyceraldehyde-3-phosphate-lyase (pyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBOXYLATES-DEG (Carboxylate Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); GALACTCAT-PWY (D-galactonate degradation); PWY-7130 (L-glucose degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ACIDS-DEG (Sugar Acid Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00945:0:0:\"2-Dehydro-3-deoxy-D-galactonate 6-phosphate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\"'},\n",
       " {'abbreviation': 'R01066',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: DRPA; DRPAr',\n",
       "   'KEGG: R01066',\n",
       "   'iAF1260: DRPA',\n",
       "   'iAG612: rbs_240',\n",
       "   'iAO358: rll_113',\n",
       "   'Name: 2-Deoxy-D-ribose-5-phosphate acetaldehyde-lyase; 2-deoxy-D-ribose-5-phosphate acetaldehyde-lyase (D-glyceraldehyde-3-phosphate-forming); deoxyribose-phosphate aldolase'],\n",
       "  'code': '(1) cpd00510[0] <=> (1) cpd00071[0] + (1) cpd00102[0]',\n",
       "  'compound_ids': 'cpd00071;cpd00102;cpd00510',\n",
       "  'definition': '(1) deoxyribose-5-phosphate[0] <=> (1) Acetaldehyde[0] + (1) Glyceraldehyde3-phosphate[0]',\n",
       "  'deltag': 0.15,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.4'],\n",
       "  'equation': '(1) cpd00510[0] <=> (1) cpd00071[0] + (1) cpd00102[0]',\n",
       "  'id': 'rxn00784',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-deoxy-D-ribose-5-phosphate acetaldehyde-lyase (D-glyceraldehyde-3-phosphate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00030 (Pentose phosphate pathway)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00510:0:0:\"deoxyribose-5-phosphate\";1:cpd00071:0:0:\"Acetaldehyde\";1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\"'},\n",
       " {'abbreviation': 'R01067',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01067_p',\n",
       "   'BiGG: TKT2',\n",
       "   'JP_Creinhardtii_NMeth: R_TKT2h',\n",
       "   'KEGG: R01067',\n",
       "   'Maize_C4GEM: R_R01067_c; R_R01067_p',\n",
       "   'Name: D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde; D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase; beta-D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate; glycolaldehyde transferase; transketolase; transketolase (erythrose-4p); transketolase, putative'],\n",
       "  'code': '(1) cpd00072[0] + (1) cpd00102[0] <=> (1) cpd00198[0] + (1) cpd00236[0]',\n",
       "  'compound_ids': 'cpd00072;cpd00102;cpd00198;cpd00236',\n",
       "  'definition': '(1) D-fructose-6-phosphate[0] + (1) Glyceraldehyde3-phosphate[0] <=> (1) D-Xylulose5-phosphate[0] + (1) D-Erythrose4-phosphate[0]',\n",
       "  'deltag': 1.6,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.2.1.1'],\n",
       "  'equation': '(1) cpd00072[0] + (1) cpd00102[0] <=> (1) cpd00198[0] + (1) cpd00236[0]',\n",
       "  'id': 'rxn00785',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31366',\n",
       "  'name': 'D-Fructose 6-phosphate:D-glyceraldehyde-3-phosphate glycolaldehyde transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00072:0:0:\"D-fructose-6-phosphate\";-1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\";1:cpd00198:0:0:\"D-Xylulose5-phosphate\";1:cpd00236:0:0:\"D-Erythrose4-phosphate\"'},\n",
       " {'abbreviation': 'R01068',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01068N_c',\n",
       "   'AraGEM: R_R01068N_p',\n",
       "   'BiGG: FBA',\n",
       "   'JP_Creinhardtii_NMeth: R_FBAf; R_FBAh',\n",
       "   'KEGG: R01068',\n",
       "   'Name: D-Fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase; D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming); beta-D-fructose 1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase; fructose-bisphosphate aldolase; fructose-bisphosphate aldolase, putative; fructose-bisphosphatealdolase'],\n",
       "  'code': '(1) cpd00290[0] <=> (1) cpd00095[0] + (1) cpd00102[0]',\n",
       "  'compound_ids': 'cpd00095;cpd00102;cpd00290',\n",
       "  'definition': '(1) D-fructose-1,6-bisphosphate[0] <=> (1) Glycerone-phosphate[0] + (1) Glyceraldehyde3-phosphate[0]',\n",
       "  'deltag': 5.12,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.13'],\n",
       "  'equation': '(1) cpd00290[0] <=> (1) cpd00095[0] + (1) cpd00102[0]',\n",
       "  'id': 'rxn00786',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31335',\n",
       "  'name': 'D-fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00680 (Methane metabolism); rn00710 (Carbon fixation in photosynthetic organisms); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00290:0:0:\"D-fructose-1,6-bisphosphate\";1:cpd00095:0:0:\"Glycerone-phosphate\";1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\"'},\n",
       " {'abbreviation': 'R01069',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: TAGAALDOL-RXN',\n",
       "   'CornCyc: TAGAALDOL-RXN',\n",
       "   'EcoCyc: TAGAALDOL-RXN',\n",
       "   'KEGG: R01069',\n",
       "   'MetaCyc: TAGAALDOL-RXN',\n",
       "   'Name: D-Tagatose 1,6-bisphosphate triosephosphate lyase; D-tagatose 1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming); D-tagatose-1,6-bisphosphate triosephosphate lyase; tagatose-bisphosphate aldolase'],\n",
       "  'code': '(1) cpd02371[0] <=> (1) cpd00095[0] + (1) cpd00102[0]',\n",
       "  'compound_ids': 'cpd00095;cpd00102;cpd02371',\n",
       "  'definition': '(1) D-Tagatose 1,6-biphosphate[0] <=> (1) Glycerone-phosphate[0] + (1) Glyceraldehyde3-phosphate[0]',\n",
       "  'deltag': 4.8,\n",
       "  'deltagerr': 0.55,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.2.40'],\n",
       "  'equation': '(1) cpd02371[0] <=> (1) cpd00095[0] + (1) cpd00102[0]',\n",
       "  'id': 'rxn00787',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-tagatose 1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase (glycerone-phosphate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); GALACTITOLCAT-PWY (galactitol degradation); GALACTOSE-DEGRADATION (Galactose Degradation); HEXITOLDEGSUPER-PWY (superpathway of hexitol degradation (bacteria)); LACTOSE-DEG (Lactose Degradation); LACTOSECAT-PWY (lactose and galactose degradation I); PWY-7077 (<i>N</i>-acetyl-D-galactosamine degradation); PWY-7395 (D-galactosamine and <i>N</i>-acetyl-D-galactosamine degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02371:0:0:\"D-Tagatose 1,6-biphosphate\";1:cpd00095:0:0:\"Glycerone-phosphate\";1:cpd00102:0:0:\"Glyceraldehyde3-phosphate\"'},\n",
       " {'abbreviation': 'R01071',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ATPPHOSPHORIBOSYLTRANS-RXN; RXN-10968',\n",
       "   'BiGG: ATPPRT; ATPPRTr; HIS1',\n",
       "   'BrachyCyc: ATPPHOSPHORIBOSYLTRANS-RXN',\n",
       "   'KEGG: R01071',\n",
       "   'MetaCyc: ATPPHOSPHORIBOSYLTRANS-RXN',\n",
       "   'Name: 1-(5-Phospho-D-ribosyl)-ATP:pyrophosphate phosphoribosyl-transferase; 1-(5-phospho-D-ribosyl)-ATP:diphosphate phospho-alpha-D-ribosyl-transferase; 1-(5-phospho-D-ribosyl)-ATP:pyrophosphate; ATP phosphoribosyltransferase; ATP phosphoribosyltransferase reversible; adenosine triphosphate phosphoribosyltransferase; phospho-alpha-D-ribosyl-transferase; phosphoribosyl ATP synthetase; phosphoribosyl ATP:pyrophosphate phosphoribosyltransferase; phosphoribosyl-ATP pyrophosphorylase; phosphoribosyl-ATP:pyrophosphate-phosphoribosyl phosphotransferase; phosphoribosyladenosine triphosphate pyrophosphorylase; phosphoribosyladenosine triphosphate synthetase; phosphoribosyladenosine triphosphate:pyrophosphate phosphoribosyltransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd01775[0] <=> (1) cpd00002[0] + (1) cpd00103[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00067;cpd00103;cpd01775',\n",
       "  'definition': '(1) PPi[0] + (1) H+[0] + (1) Phosphoribosyl-ATP[0] <= (1) ATP[0] + (1) PRPP[0]',\n",
       "  'deltag': 17.21,\n",
       "  'deltagerr': 5.64,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.17'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00067[0] + (1) cpd01775[0] <= (1) cpd00002[0] + (1) cpd00103[0]',\n",
       "  'id': 'rxn00789',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19213',\n",
       "  'name': '1-(5-phospho-D-ribosyl)-ATP:diphosphate phospho-alpha-D-ribosyl-transferase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); HISTIDINE-SYN (L-histidine Biosynthesis); HISTSYN-PWY (L-histidine biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis)',\n",
       "   'KEGG: rn00340 (Histidine metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00067:0:0:\"H+\";-1:cpd01775:0:0:\"Phosphoribosyl-ATP\";1:cpd00002:0:0:\"ATP\";1:cpd00103:0:0:\"PRPP\"'},\n",
       " {'abbreviation': 'R01072',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PRPPAMIDOTRANS-RXN',\n",
       "   'BiGG: ADE4; GLUPRT',\n",
       "   'BrachyCyc: PRPPAMIDOTRANS-RXN',\n",
       "   'KEGG: R01072',\n",
       "   'MetaCyc: PRPPAMIDOTRANS-RXN',\n",
       "   \"Name: (glutamate-amidating); 5'-phosphoribosylpyrophosphate amidotransferase; 5-Phosphoribosylamine:pyrophosphate phosphoribosyltransferase; 5-phospho-beta-D-ribosylamine:diphosphate phospho-alpha-D-ribosyltransferase (glutamate-amidating); 5-phosphoribosyl-1-pyrophosphate amidotransferase; 5-phosphoribosylamine:diphosphate phospho-alpha-D-ribosyltransferase (glutamate-amidating); 5-phosphororibosyl-1-pyrophosphate amidotransferase; Glutamine phosphoribosylpyrophosphate amidotransferase; Phosphoribosyldiphosphate 5-amidotransferase; alpha-5-phosphoribosyl-1-pyrophosphate amidotransferase; amidophosphoribosyltransferase; glutamine 5-phosphoribosylpyrophosphate amidotransferase; glutamine phosphoribosyldiphosphate amidotransferase; glutamine ribosylpyrophosphate 5-phosphate amidotransferase; glutaminephosphoribosyldiphosphateamidotransferase; phosphoribose pyrophosphate amidotransferase; phosphoribosyl pyrophosphate amidotransferase; phosphoribosyldiphosphate 5-amidotransferase; phosphoribosyldiphosphate amidotransferase; phosphoribosylpyrophosphate glutamyl amidotransferase\"],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00023[0] + (1) cpd01982[0] <=> (1) cpd00001[0] + (1) cpd00053[0] + (1) cpd00103[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00012;cpd00023;cpd00053;cpd00067;cpd00103;cpd01982',\n",
       "  'definition': '(1) PPi[0] + (1) L-Glutamate[0] + (1) H+[0] + (1) 5-Phosphoribosylamine[0] <= (1) H2O[0] + (1) L-Glutamine[0] + (1) PRPP[0]',\n",
       "  'deltag': 12.98,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.14'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00023[0] + (1) cpd00067[0] + (1) cpd01982[0] <= (1) cpd00001[0] + (1) cpd00053[0] + (1) cpd00103[0]',\n",
       "  'id': 'rxn00790',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28100',\n",
       "  'name': '5-phosphoribosylamine:diphosphate phospho-alpha-D-ribosyltransferase (glutamate-amidating)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AIR-Biosynthesis (5-Aminoimidazole Ribonucleotide Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); HMP-PP-Biosynthesis (4-amino-2-methyl-5-diphosphomethylpyrimidine biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6121 (5-aminoimidazole ribonucleotide biosynthesis I); PWY-6122 (5-aminoimidazole ribonucleotide biosynthesis II); PWY-6277 (superpathway of 5-aminoimidazole ribonucleotide biosynthesis); PWY-7282 (4-amino-2-methyl-5-diphosphomethylpyrimidine biosynthesis (yeast)); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Thiamine-Biosynthesis (Thiamine Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn00250 (Alanine, aspartate and glutamate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00023:0:0:\"L-Glutamate\";-1:cpd00067:0:0:\"H+\";-1:cpd01982:0:0:\"5-Phosphoribosylamine\";1:cpd00001:0:0:\"H2O\";1:cpd00053:0:0:\"L-Glutamine\";1:cpd00103:0:0:\"PRPP\"'},\n",
       " {'abbreviation': 'R01073',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PRTRANS-RXN',\n",
       "   'BiGG: ANPRT; TRP4',\n",
       "   'BrachyCyc: PRTRANS-RXN',\n",
       "   'KEGG: R01073',\n",
       "   'MetaCyc: PRTRANS-RXN',\n",
       "   'Name: Anthranilate phosphoribosyltransferase; N-(5-Phospho-D-ribosyl)anthranilate:pyrophosphate; N-(5-Phospho-D-ribosyl)anthranilate:pyrophosphate phosphoribosyl-transferase; N-(5-phospho-D-ribosyl)-anthranilate:diphosphate phospho-alpha-D-ribosyltransferase; PR transferase; PRT; Phosphoribosyl-anthranilate diphosphorylase; Phosphoribosyl-anthranilate pyrophosphorylase; anthranilate 5-phosphoribosylpyrophosphate phosphoribosyltransferase; anthranilate phosphoribosylpyrophosphate phosphoribosyltransferase; anthranilate phosphoribosyltransfer; anthranilate phosphoribosyltransferase; anthranilate-PP-ribose-P phosphoribosyltransferase; anthranilatephosphoribosyltransferase; phosphoribosyl-anthranilate pyrophosphorylase; phosphoribosyl-transferase; phosphoribosylanthranilate pyrophosphorylase; phosphoribosylanthranilate transferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd02642[0] <=> (1) cpd00093[0] + (1) cpd00103[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00067;cpd00093;cpd00103;cpd02642',\n",
       "  'definition': '(1) PPi[0] + (1) H+[0] + (1) N-5-phosphoribosyl-anthranilate[0] <= (1) Anthranilate[0] + (1) PRPP[0]',\n",
       "  'deltag': 13.69,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.18'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00067[0] + (1) cpd02642[0] <= (1) cpd00093[0] + (1) cpd00103[0]',\n",
       "  'id': 'rxn00791',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28101;rxn38274',\n",
       "  'name': 'N-(5-Phospho-D-ribosyl)anthranilate:pyrophosphate phosphoribosyl-transferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); COMPLETE-ARO-PWY (superpathway of aromatic amino acid biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-6629 (superpathway of L-tryptophan biosynthesis); TRPSYN-PWY (L-tryptophan biosynthesis); TRYPTOPHAN-BIOSYNTHESIS (L-tryptophan Biosynthesis)',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00067:0:0:\"H+\";-1:cpd02642:0:0:\"N-5-phosphoribosyl-anthranilate\";1:cpd00093:0:0:\"Anthranilate\";1:cpd00103:0:0:\"PRPP\"'},\n",
       " {'abbreviation': 'R01074',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01074_c',\n",
       "   'BiGG: BACCL; BTNL1',\n",
       "   'KEGG: R01074',\n",
       "   'iAF692: BACCL',\n",
       "   'iIT341: BACCL',\n",
       "   'Name: ATP:biotin adenylyltransferase; Biotin:CoA ligase (AMP-forming); Biotin:apo-[carboxylase] ligase (AMP-forming); biotin-[acetyl-CoA-carboxylase] ligase; biotin-acetyl-CoA-carboxylase-ligase; biotin:CoA ligase (AMP-forming); biotin:apo-[carboxylase] ligase (AMP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00104[0] <=> (1) cpd00012[0] + (1) cpd03517[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00104;cpd03517',\n",
       "  'definition': '(1) ATP[0] + (1) BIOT[0] => (1) PPi[0] + (1) Biotinyl-5-AMP[0]',\n",
       "  'deltag': -11.13,\n",
       "  'deltagerr': 5.39,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.11', '6.3.4.10', '6.3.4.11', '6.3.4.15', '6.3.4.9'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00104[0] => (1) cpd00012[0] + (1) cpd03517[0]',\n",
       "  'id': 'rxn00792',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'biotin:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00780 (Biotin metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00104:0:0:\"BIOT\";1:cpd00012:0:0:\"PPi\";1:cpd03517:0:0:\"Biotinyl-5-AMP\"'},\n",
       " {'abbreviation': 'R01075',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01075',\n",
       "   'MetaCyc: BIOTIN--COA-LIGASE-RXN',\n",
       "   'Name: Biotin:CoA ligase (AMP-forming); biotin CoA synthetase; biotin-CoA ligase; biotin:CoA ligase (AMP-forming); biotinyl coenzyme A synthetase; biotinyl-CoA synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00104[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd01302[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00104;cpd01302',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) BIOT[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) Biotinyl-CoA[0]',\n",
       "  'deltag': -21.55,\n",
       "  'deltagerr': 7.38,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.11'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00104[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd01302[0]',\n",
       "  'id': 'rxn00793',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'biotin:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00104:0:0:\"BIOT\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:H+;1:cpd01302:0:0:\"Biotinyl-CoA\"'},\n",
       " {'abbreviation': 'R01076',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01076',\n",
       "   'MetaCyc: BIOTINIDASE-RXN',\n",
       "   'Name: Biotin-amide amidohydrolase; amidohydrolase biotinidase; biotin-amide amidohydrolase; biotinidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01301[0] <=> (1) cpd00013[0] + (1) cpd00104[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00104;cpd01301',\n",
       "  'definition': '(1) H2O[0] + (1) Biotin amide[0] => (1) NH3[0] + (1) BIOT[0]',\n",
       "  'deltag': -2.84,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.12'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01301[0] => (1) cpd00013[0] + (1) cpd00104[0]',\n",
       "  'id': 'rxn00794',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Biotin-amide amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01301:0:0:\"Biotin amide\";1:cpd00013:0:0:\"NH3\";1:cpd00104:0:0:\"BIOT\"'},\n",
       " {'abbreviation': 'R01077',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01077', 'Name: N6-D-Biotinyl-L-lysine amidohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03293[0] <=> (1) cpd00039[0] + (1) cpd00104[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00039;cpd00104;cpd03293',\n",
       "  'definition': '(1) H2O[0] + (1) Biocytin[0] <=> (1) L-Lysine[0] + (1) BIOT[0]',\n",
       "  'deltag': 1.07,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.12'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03293[0] <=> (1) cpd00039[0] + (1) cpd00104[0]',\n",
       "  'id': 'rxn00795',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N6-D-Biotinyl-L-lysine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00780 (Biotin metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03293:0:0:\"Biocytin\";1:cpd00039:0:0:\"L-Lysine\";1:cpd00104:0:0:\"BIOT\"'},\n",
       " {'abbreviation': 'BTS3r',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: BTS; BTS3r',\n",
       "   'iAF692: BTS3r',\n",
       "   'iMEO21: BTS3r',\n",
       "   'iMO1053-PAO1: BTSr',\n",
       "   'iNJ661: BTS3r',\n",
       "   'Name: biotin synthase'],\n",
       "  'code': '(2) cpd00074[0] + (1) cpd01311[0] <=> (1) cpd00104[0] + (1) cpd00239[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00074;cpd00104;cpd00239;cpd01311',\n",
       "  'definition': '(2) S[0] + (1) Dethiobiotin[0] <=> (1) H+[0] + (1) BIOT[0] + (1) H2S[0]',\n",
       "  'deltag': 3.42,\n",
       "  'deltagerr': 2.87,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(2) cpd00074[0] + (1) cpd01311[0] <=> (1) cpd00067[0] + (1) cpd00104[0] + (1) cpd00239[0]',\n",
       "  'id': 'rxn00796',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'biotin synthase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Secondary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00074:0:0:\"S\";-1:cpd01311:0:0:\"Dethiobiotin\";1:cpd00067:0:0:\"H+\";1:cpd00104:0:0:\"BIOT\";1:cpd00239:0:0:\"H2S\"'},\n",
       " {'abbreviation': 'R01080',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01080_c',\n",
       "   'AraGEM: R_R01080_c',\n",
       "   'BiGG: PYRNS1; URIH',\n",
       "   'KEGG: R01080',\n",
       "   'MetaCyc: URIDINE-NUCLEOSIDASE-RXN',\n",
       "   'Name: Uridine hydrolase; Uridine ribohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00249[0] <=> (1) cpd00092[0] + (1) cpd00105[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00092;cpd00105;cpd00249',\n",
       "  'definition': '(1) H2O[0] + (1) Uridine[0] <=> (1) Uracil[0] + (1) D-Ribose[0]',\n",
       "  'deltag': -2.99,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.3', '3.2.2.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00249[0] <=> (1) cpd00092[0] + (1) cpd00105[0]',\n",
       "  'id': 'rxn00797',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Uridine ribohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PWY-6556 (pyrimidine ribonucleosides salvage II); PWY-7196 (superpathway of pyrimidine ribonucleosides salvage); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimidine-Degradation (Pyrimidine Nucleotide Degradation); Pyrimidine-Nucleotide-Salvage (Pyrimidine Nucleotide Salvage); Pyrimidine-Ribonucleosides-Degradation (Pyrimidine Ribonucleoside Degradation)',\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00249:0:0:\"Uridine\";1:cpd00092:0:0:\"Uracil\";1:cpd00105:0:0:\"D-Ribose\"'},\n",
       " {'abbreviation': 'R01081',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01081',\n",
       "   'iRS1563: R_R01081_c',\n",
       "   'Name: D-Ribose ketol-isomerase; D-ribose aldose-ketose-isomerase'],\n",
       "  'code': '(1) cpd00105[0] <=> (1) cpd00258[0]',\n",
       "  'compound_ids': 'cpd00105;cpd00258',\n",
       "  'definition': '(1) D-Ribose[0] <=> (1) D-Ribulose[0]',\n",
       "  'deltag': 0.65,\n",
       "  'deltagerr': 0.2,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.20'],\n",
       "  'equation': '(1) cpd00105[0] <=> (1) cpd00258[0]',\n",
       "  'id': 'rxn00798',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-ribose aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00105:0:0:\"D-Ribose\";1:cpd00258:0:0:\"D-Ribulose\"'},\n",
       " {'abbreviation': 'R01082',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FUMHYDR-RXN',\n",
       "   'BiGG: FUM; FUM1_1; FUM1_2; FUMm',\n",
       "   'BrachyCyc: FUMHYDR-RXN',\n",
       "   'KEGG: R01082',\n",
       "   'MetaCyc: FUMHYDR-RXN',\n",
       "   'Name: (S)-Malate hydro-lyase; (S)-malate hydro-lyase; (S)-malate hydro-lyase (fumarate-forming); Fumarate Hydratase; Fumarate hydratase; L-malate hydro-lyase; fumarase; fumarate hydratase; fumarate hydration; malate dehydration'],\n",
       "  'code': '(1) cpd00130[0] <=> (1) cpd00001[0] + (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00106;cpd00130',\n",
       "  'definition': '(1) L-Malate[0] <=> (1) H2O[0] + (1) Fumarate[0]',\n",
       "  'deltag': 0.82,\n",
       "  'deltagerr': 0.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.2'],\n",
       "  'equation': '(1) cpd00130[0] <=> (1) cpd00001[0] + (1) cpd00106[0]',\n",
       "  'id': 'rxn00799',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19740;rxn33856;rxn33857;rxn33858',\n",
       "  'name': '(S)-malate hydro-lyase (fumarate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Degradation (Amino Acid Degradation); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); GLUTAMATE-DEG (L-glutamate Degradation); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); P108-PWY (pyruvate fermentation to propanoate I); P23-PWY (reductive TCA cycle I); P42-PWY (incomplete reductive TCA cycle); PWY-5088 (L-glutamate degradation VIII (to propanoate)); PWY-5392 (reductive TCA cycle II); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-561 (superpathway of glyoxylate cycle and fatty acid degradation); PWY-5690 (TCA cycle II (plants and fungi)); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6728 (methylaspartate cycle); PWY-6969 (TCA cycle V (2-oxoglutarate:ferredoxin oxidoreductase)); PWY-7254 (TCA cycle VII (acetate-producers)); PWY-7384 (anaerobic energy metabolism (invertebrates, mitochondrial)); PWY-7389 (superpathway of anaerobic energy metabolism (invertebrates)); PWY66-398 (TCA cycle III (animals)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); Pyruvate-Propanoate-Fermentation (Pyruvate Fermentation to Propanoate); REDCITCYC (TCA cycle VIII (helicobacter)); Reductive-TCA-Cycles (Reductive TCA Cycles); TCA (TCA cycle I (prokaryotic)); TCA-GLYOX-BYPASS (superpathway of glyoxylate bypass and TCA); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00620 (Pyruvate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00130:0:0:\"L-Malate\";1:cpd00001:0:0:\"H2O\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R01083',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: AMPSYN-RXN',\n",
       "   'BiGG: ADE13_2; ADSL1; ADSL1r',\n",
       "   'BrachyCyc: AMPSYN-RXN',\n",
       "   'KEGG: R01083',\n",
       "   'MetaCyc: AMPSYN-RXN',\n",
       "   'Name: 6-N-(1,2-dicarboxyethyl)AMP AMP-lyase; 6-N-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming); N6-(1,2-Dicarboxyethyl)AMP AMP-lyase; N6-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming); adenylosuccinase; adenylosuccinate lyase; adenylsuccinate lyase; adenylsuccinatelyase; succino AMP-lyase'],\n",
       "  'code': '(1) cpd02375[0] <=> (1) cpd00018[0] + (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00018;cpd00106;cpd02375',\n",
       "  'definition': '(1) Adenylosuccinate[0] <=> (1) AMP[0] + (1) Fumarate[0]',\n",
       "  'deltag': 2.8,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.3.2.2'],\n",
       "  'equation': '(1) cpd02375[0] <=> (1) cpd00018[0] + (1) cpd00106[0]',\n",
       "  'id': 'rxn00800',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27708',\n",
       "  'name': 'N6-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6126 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis II); PWY-7219 (adenosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7229 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02375:0:0:\"Adenylosuccinate\";1:cpd00018:0:0:\"AMP\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R01085',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01085_c',\n",
       "   'KEGG: R01085',\n",
       "   'MetaCyc: RXN-10445',\n",
       "   'Name: 3-Fumarylpyruvate acylhydrolase; 3-fumarylpyruvate fumarylhydrolase; 3-fumarylpyruvate hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01653[0] <=> (1) cpd00020[0] + (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00020;cpd00067;cpd00106;cpd01653',\n",
       "  'definition': '(1) H2O[0] + (1) 3-Fumarylpyruvate[0] => (1) Pyruvate[0] + (1) H+[0] + (1) Fumarate[0]',\n",
       "  'deltag': -10.11,\n",
       "  'deltagerr': 0.84,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.7.1.-', '3.7.1.20', '3.7.1.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01653[0] => (1) cpd00020[0] + (1) cpd00067[0] + (1) cpd00106[0]',\n",
       "  'id': 'rxn00801',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '3-fumarylpyruvate fumarylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); Gentisate-Degradation (Gentisate Degradation); Nitroaromatic-Degradation (Nitroaromatic Compound Degradation); P281-PWY (3-phenylpropanoate degradation); PWY-6223 (gentisate degradation I); PWY-7044 (5-nitroanthranilate degradation); Phenolic-Compounds-Degradation (Phenolic Compound Degradation)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01653:0:0:\"3-Fumarylpyruvate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R01086',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ARGSUCCINLYA-RXN',\n",
       "   'BrachyCyc: ARGSUCCINLYA-RXN',\n",
       "   'ChlamyCyc: ARGSUCCINLYA-RXN',\n",
       "   'KEGG: R01086',\n",
       "   'MetaCyc: ARGSUCCINLYA-RXN',\n",
       "   'Name: 2-(Nomega-L-arginino)succinate arginine-lyase (fumarate-forming); 2-(omega-N-L-arginino)succinate arginine-lyase (fumarate-forming); ARGSL; N-(L-Argininosuccinate) arginie-lyase; N-(L-argininosuccinate) arginine-lyase; arginine-succinate lyase; argininosuccinate lyase; argininosuccinatelyase; argininosuccinic acid lyase; arginosuccinase; omega-N-(L-arginino)succinate arginine-lyase'],\n",
       "  'code': '(1) cpd02152[0] <=> (1) cpd00051[0] + (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00051;cpd00106;cpd02152',\n",
       "  'definition': '(1) L-Argininosuccinate[0] <=> (1) L-Arginine[0] + (1) Fumarate[0]',\n",
       "  'deltag': -2.64,\n",
       "  'deltagerr': 1.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.3.2.1'],\n",
       "  'equation': '(1) cpd02152[0] <=> (1) cpd00051[0] + (1) cpd00106[0]',\n",
       "  'id': 'rxn00802',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27716',\n",
       "  'name': '2-(Nomega-L-arginino)succinate arginine-lyase (fumarate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGININE-DEG (L-arginine Degradation); ARGININE-SYN (L-arginine Biosynthesis); ARGSYN-PWY (L-arginine biosynthesis I (via L-ornithine)); ARGSYNBSUB-PWY (L-arginine biosynthesis II (acetyl cycle)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Citrulline-Biosynthesis (L-citrulline Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Metabolic-Regulators (Metabolic Regulator Biosynthesis); NITROGEN-DEG (Nitrogen Compound Metabolism); Nitric-Oxide-Biosynthesis (Nitric Oxide Biosynthesis); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-4983 (nitric oxide biosynthesis II (mammals)); PWY-4984 (urea cycle); PWY-5004 (superpathway of L-citrulline metabolism); PWY-5154 (L-arginine biosynthesis III (via <i>N</i>-acetyl-L-citrulline)); PWY-7400 (L-arginine biosynthesis IV (archaebacteria)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00250 (Alanine, aspartate and glutamate metabolism); rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02152:0:0:\"L-Argininosuccinate\";1:cpd00051:0:0:\"L-Arginine\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R01087',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MALEATE-ISOMERASE-RXN',\n",
       "   'BiGG: MALEI',\n",
       "   'KEGG: R01087',\n",
       "   'MetaCyc: MALEATE-ISOMERASE-RXN',\n",
       "   'iJN746: MALEI',\n",
       "   'Name: Maleate cis-trans-isomerase; maleate cis-trans-isomerase; maleate isomerase'],\n",
       "  'code': '(1) cpd00996[0] <=> (1) cpd00106[0]',\n",
       "  'compound_ids': 'cpd00106;cpd00996',\n",
       "  'definition': '(1) Maleate[0] <=> (1) Fumarate[0]',\n",
       "  'deltag': -1.53,\n",
       "  'deltagerr': 0.77,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.2.1.1'],\n",
       "  'equation': '(1) cpd00996[0] <=> (1) cpd00106[0]',\n",
       "  'id': 'rxn00803',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Maleate cis-trans-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); Nicotinate-Degradation (Nicotinate Degradation); Nicotine-Degradation (Nicotine Degradation); Other-Degradation (Degradation/Utilization/Assimilation-Other); PWY-5033 (nicotinate degradation II); PWY-5062 (superpathway of nicotinate degradation); PWY-6993 (nicotine degradation II (pyrrolidine pathway)); PWY-7128 (nicotine degradation III (VPP pathway)); PWY-722 (nicotinate degradation I)',\n",
       "   'KEGG: rn00650 (Butanoate metabolism); rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00996:0:0:\"Maleate\";1:cpd00106:0:0:\"Fumarate\"'},\n",
       " {'abbreviation': 'R01088',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: LLEUD; LLEUDr',\n",
       "   'KEGG: R01088',\n",
       "   'MetaCyc: LEUCINE-DEHYDROGENASE-RXN',\n",
       "   'iAG612: rbs_413',\n",
       "   'iMO1053-PAO1: LLEUD; LLEUDr',\n",
       "   'Name: L-leucine dehydrogenase; L-leucine:NAD+ oxidoreductase (deaminating); L-leucine:NAD+ oxidoreductase(deaminating); L-leucine:NAD+ oxidoreductase, deaminating; LeuDH; leucine dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00107[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00200[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00013;cpd00067;cpd00107;cpd00200',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Leucine[0] <=> (1) NADH[0] + (1) NH3[0] + (1) H+[0] + (1) 4MOP[0]',\n",
       "  'deltag': 8.78,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00107[0] <=> (1) cpd00004[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00200[0]',\n",
       "  'id': 'rxn00804',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30550',\n",
       "  'name': 'L-leucine:NAD+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00280 (Valine, leucine and isoleucine degradation); rn00290 (Valine, leucine and isoleucine biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00107:0:0:\"L-Leucine\";1:cpd00004:0:0:\"NADH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00200:0:0:\"4MOP\"'},\n",
       " {'abbreviation': 'R01089',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01089',\n",
       "   'MetaCyc: LEUCINE-N-ACETYLTRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:L-leucine N-acetyltransferase; acetyl-CoA:L-leucine N-acetyltransferase; leucine N-acetyltransferase; leucine acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00107[0] <=> (1) cpd00010[0] + (1) cpd01754[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00067;cpd00107;cpd01754',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Leucine[0] => (1) CoA[0] + (1) H+[0] + (1) N-Acetyl-L-leucine[0]',\n",
       "  'deltag': -3.66,\n",
       "  'deltagerr': 0.85,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.66'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00107[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01754[0]',\n",
       "  'id': 'rxn00805',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:L-leucine N-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00107:0:0:\"L-Leucine\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd01754:0:0:\"N-Acetyl-L-leucine\"'},\n",
       " {'abbreviation': 'R01090',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: BRANCHED-CHAINAMINOTRANSFERLEU-RXN',\n",
       "   'BiGG: BAT1_1; BAT2_3; LEUTA; LEUTAi; LEUTAm',\n",
       "   'BrachyCyc: BRANCHED-CHAINAMINOTRANSFERLEU-RXN',\n",
       "   'KEGG: R01090',\n",
       "   'MetaCyc: BRANCHED-CHAINAMINOTRANSFERLEU-RXN; LEUCINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: Branched-chain-amino-acid aminotransferase; L-Leucine:2-oxoglutarate aminotransferase; L-branched chain amino acid aminotransferase; Transaminase B; branched-chain amino acid aminotransferase; branched-chain amino acid-glutamate transaminase; branched-chain aminotransferase; branched-chain-amino-acid transaminase; branched-chain-amino-acid:2-oxoglutarate aminotransferase; glutamate-branched-chain amino acid transaminase; leucine aminotransferase; leucine transaminase; leucine transaminase (irreversible); leucinetransaminase; transaminase B'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00107[0] <=> (1) cpd00023[0] + (1) cpd00200[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00107;cpd00200',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Leucine[0] <=> (1) L-Glutamate[0] + (1) 4MOP[0]',\n",
       "  'deltag': 0.07,\n",
       "  'deltagerr': 0.38,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.42', '2.6.1.6', '2.6.1.67'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00107[0] <=> (1) cpd00023[0] + (1) cpd00200[0]',\n",
       "  'id': 'rxn00806',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19241;rxn27738;rxn38054',\n",
       "  'name': 'L-Leucine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); BRANCHED-CHAIN-AA-SYN-PWY (superpathway of branched chain amino acid biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); LEU-DEG2-PWY (L-leucine degradation I); LEUCINE-DEG (L-leucine Degradation); LEUCINE-SYN (L-leucine Biosynthesis); LEUSYN-PWY (L-leucine biosynthesis); PWY-5076 (L-leucine degradation III); PWY-7767 (L-leucine degradation IV (Stickland reaction)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00280 (Valine, leucine and isoleucine degradation); rn00290 (Valine, leucine and isoleucine biosynthesis); rn00966 (Glucosinolate biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00107:0:0:\"L-Leucine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00200:0:0:\"4MOP\"'},\n",
       " {'abbreviation': 'R01091',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01091',\n",
       "   'MetaCyc: LEUCINE-23-AMINOMUTASE-RXN',\n",
       "   'Name: (2S)-alpha-Leucine 2,3-aminomutase; (2S)-alpha-leucine 2,3-aminomutase; leucine 2,3-aminomutase'],\n",
       "  'code': '(1) cpd00107[0] <=> (1) cpd01634[0]',\n",
       "  'compound_ids': 'cpd00107;cpd01634',\n",
       "  'definition': '(1) L-Leucine[0] <=> (1) L-beta-Leucine[0]',\n",
       "  'deltag': 1.68,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.3.7'],\n",
       "  'equation': '(1) cpd00107[0] <=> (1) cpd01634[0]',\n",
       "  'id': 'rxn00807',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(2S)-alpha-leucine 2,3-aminomutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); LEUCINE-DEG (L-leucine Degradation); PWY-5075 (L-leucine degradation II); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00280 (Valine, leucine and isoleucine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00107:0:0:\"L-Leucine\";1:cpd01634:0:0:\"L-beta-Leucine\"'},\n",
       " {'abbreviation': 'R01092',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01092_c',\n",
       "   'AraGEM: R_R01092_c',\n",
       "   'BiGG: GAL1; GALK; GALKr',\n",
       "   'DF_Athaliana: reac_339',\n",
       "   'KEGG: R01092',\n",
       "   'Name: ATP:D-galactose 1-phosphotransferase; galactokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00108[0] <=> (1) cpd00008[0] + (1) cpd00348[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00108;cpd00348',\n",
       "  'definition': '(1) ATP[0] + (1) Galactose[0] => (1) ADP[0] + (1) H+[0] + (1) D-Galactose 1-phosphate[0]',\n",
       "  'deltag': -1.3,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00108[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00348[0]',\n",
       "  'id': 'rxn00808',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34770',\n",
       "  'name': 'ATP:D-galactose 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00108:0:0:\"Galactose\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00348:0:0:\"D-Galactose 1-phosphate\"'},\n",
       " {'abbreviation': 'D-Galactose:NAD+ 1-oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: D-Galactose:NAD+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00108[0] <=> (1) cpd00004[0] + (1) cpd01171[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00108;cpd01171',\n",
       "  'definition': '(1) NAD[0] + (3) H+[0] + (1) Galactose[0] <=> (1) NADH[0] + (1) Dulcose[0]',\n",
       "  'deltag': 27.91,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.21'],\n",
       "  'equation': '(1) cpd00003[0] + (3) cpd00067[0] + (1) cpd00108[0] <=> (1) cpd00004[0] + (1) cpd01171[0]',\n",
       "  'id': 'rxn00809',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Galactose:NAD+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:-4',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-3:cpd00067:0:0:\"H+\";-1:cpd00108:0:0:\"Galactose\";1:cpd00004:0:0:\"NADH\";1:cpd01171:0:0:\"Dulcose\"'},\n",
       " {'abbreviation': 'R01094',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01094',\n",
       "   'MetaCyc: GALACTODEHYDROG-RXN',\n",
       "   'Name: D-Galactose:NAD+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00108[0] <=> (1) cpd00004[0] + (1) cpd02143[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00108;cpd02143',\n",
       "  'definition': '(1) NAD[0] + (1) Galactose[0] <=> (1) NADH[0] + (1) H+[0] + (1) D-Galactono-1,4-lactone[0]',\n",
       "  'deltag': -2.07,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.48'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00108[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd02143[0]',\n",
       "  'id': 'rxn00810',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Galactose:NAD+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); GALACTOSE-DEGRADATION (Galactose Degradation); GALDEG-PWY (D-galactose degradation II); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00108:0:0:\"Galactose\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd02143:0:0:\"D-Galactono-1,4-lactone\"'},\n",
       " {'abbreviation': 'D-Galactose:NADP+ 1-oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: D-Galactose:NADP+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00108[0] <=> (1) cpd00005[0] + (1) cpd01171[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00108;cpd01171',\n",
       "  'definition': '(1) NADP[0] + (3) H+[0] + (1) Galactose[0] <=> (1) NADPH[0] + (1) Dulcose[0]',\n",
       "  'deltag': 27.91,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.21'],\n",
       "  'equation': '(1) cpd00006[0] + (3) cpd00067[0] + (1) cpd00108[0] <=> (1) cpd00005[0] + (1) cpd01171[0]',\n",
       "  'id': 'rxn00811',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Galactose:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'CI:-4',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-3:cpd00067:0:0:\"H+\";-1:cpd00108:0:0:\"Galactose\";1:cpd00005:0:0:\"NADPH\";1:cpd01171:0:0:\"Dulcose\"'},\n",
       " {'abbreviation': 'R01096',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01096',\n",
       "   'MetaCyc: GALACTOSE-1-DEHYDROGENASE-NADP+-RXN',\n",
       "   'Name: D-Galactose:NADP+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00108[0] <=> (1) cpd00005[0] + (1) cpd01734[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00108;cpd01734',\n",
       "  'definition': '(1) NADP[0] + (1) Galactose[0] <=> (1) NADPH[0] + (1) H+[0] + (1) D-Galactono-1,5-lactone[0]',\n",
       "  'deltag': -3.22,\n",
       "  'deltagerr': 0.51,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.120', '1.1.1.359', '1.1.1.360', '1.1.1.48'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00108[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd01734[0]',\n",
       "  'id': 'rxn00812',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Galactose:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00108:0:0:\"Galactose\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd01734:0:0:\"D-Galactono-1,5-lactone\"'},\n",
       " {'abbreviation': 'R01097',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01097', 'Name: D-Galactose:NADP+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00108[0] <=> (1) cpd00005[0] + (1) cpd02143[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00108;cpd02143',\n",
       "  'definition': '(1) NADP[0] + (1) Galactose[0] <=> (1) NADPH[0] + (1) H+[0] + (1) D-Galactono-1,4-lactone[0]',\n",
       "  'deltag': -2.06,\n",
       "  'deltagerr': 0.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.120', '1.1.1.48'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00108[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd02143[0]',\n",
       "  'id': 'rxn00813',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Galactose:NADP+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00108:0:0:\"Galactose\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd02143:0:0:\"D-Galactono-1,4-lactone\"'},\n",
       " {'abbreviation': 'R01098',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_GALO',\n",
       "   'KEGG: R01098',\n",
       "   'Name: D-Galactose:oxygen 6-oxidoreductase; galactose oxidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00108[0] <=> (1) cpd00025[0] + (1) cpd00653[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00067;cpd00108;cpd00653',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Galactose[0] => (1) H2O2[0] + (1) H+[0] + (1) D-Galactonate[0]',\n",
       "  'deltag': -39.05,\n",
       "  'deltagerr': 1.1,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.9'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00108[0] => (1) cpd00025[0] + (1) cpd00067[0] + (1) cpd00653[0]',\n",
       "  'id': 'rxn00814',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34771',\n",
       "  'name': 'D-Galactose:oxygen 6-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00108:0:0:\"Galactose\";1:cpd00025:0:0:\"H2O2\";1:cpd00067:0:0:\"H+\";1:cpd00653:0:0:\"D-Galactonate\"'},\n",
       " {'abbreviation': 'R01099',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01099',\n",
       "   'MetaCyc: GALACTOSE-OXIDASE-RXN',\n",
       "   'Name: D-Galactose:oxygen 6-oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00108[0] <=> (1) cpd00025[0] + (1) cpd02087[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00025;cpd00108;cpd02087',\n",
       "  'definition': '(1) O2[0] + (1) Galactose[0] => (1) H2O2[0] + (1) D-galacto-Hexodialdose[0]',\n",
       "  'deltag': -26.49,\n",
       "  'deltagerr': 2.26,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.9'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00108[0] => (1) cpd00025[0] + (1) cpd02087[0]',\n",
       "  'id': 'rxn00815',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'D-Galactose:oxygen 6-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00108:0:0:\"Galactose\";1:cpd00025:0:0:\"H2O2\";1:cpd02087:0:0:\"D-galacto-Hexodialdose\"'},\n",
       " {'abbreviation': 'R01100',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: LACZ; LACZpp',\n",
       "   'KEGG: R01100; R06114',\n",
       "   'iAF1260: LACZ; LACZpp',\n",
       "   'iAO358: rll_223',\n",
       "   'iJR904: LACZ',\n",
       "   'Name: Lactose galactohydrolase; b-galactosidase; beta-galactosidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00208[0] <=> (1) cpd00027[0] + (1) cpd00108[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00108;cpd00208',\n",
       "  'definition': '(1) H2O[0] + (1) LACT[0] => (1) D-Glucose[0] + (1) Galactose[0]',\n",
       "  'deltag': 97.86,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.108', '3.2.1.23'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00208[0] => (1) cpd00027[0] + (1) cpd00108[0]',\n",
       "  'id': 'rxn00816',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn11244;rxn29965',\n",
       "  'name': 'Lactose galactohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00208:0:0:\"LACT\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00108:0:0:\"Galactose\"'},\n",
       " {'abbreviation': 'R01101',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraGEM: R_R01101_c',\n",
       "   'BiGG: DGGH; FSP2_2; GALS3; YBR184W_1; YBR184W_2',\n",
       "   'KEGG: R01101; R06091',\n",
       "   'MetaCyc: ALPHAGALACTOSID-RXN',\n",
       "   'iAF1260: GALS3',\n",
       "   'Name: a-galactosidase (melibiose); alpha-D-Glucoside glucohydrolase; alpha-D-glucoside glucohydrase; melibiose galactohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03198[0] <=> (1) cpd00027[0] + (1) cpd00108[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00027;cpd00108;cpd03198',\n",
       "  'definition': '(1) H2O[0] + (1) Melibiose[0] => (1) D-Glucose[0] + (1) Galactose[0]',\n",
       "  'deltag': 97.9,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03198[0] => (1) cpd00027[0] + (1) cpd00108[0]',\n",
       "  'id': 'rxn00817',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'melibiose galactohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY0-1301 (melibiose degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03198:0:0:\"Melibiose\";1:cpd00027:0:0:\"D-Glucose\";1:cpd00108:0:0:\"Galactose\"'},\n",
       " {'abbreviation': 'R01103',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01103_c',\n",
       "   'AraGEM: R_R01103_c',\n",
       "   'BiGG: RAFGH; YBR184W_3',\n",
       "   'JP_Creinhardtii_MSB: R_RGH',\n",
       "   'KEGG: R01103; R06070',\n",
       "   'Name: Alpha-galactosidase; Raffinose galactohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00382[0] <=> (1) cpd00076[0] + (1) cpd00108[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00076;cpd00108;cpd00382',\n",
       "  'definition': '(1) H2O[0] + (1) Melitose[0] => (1) Sucrose[0] + (1) Galactose[0]',\n",
       "  'deltag': -1.16,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00382[0] => (1) cpd00076[0] + (1) cpd00108[0]',\n",
       "  'id': 'rxn00818',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34781',\n",
       "  'name': 'Raffinose galactohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00382:0:0:\"Melitose\";1:cpd00076:0:0:\"Sucrose\";1:cpd00108:0:0:\"Galactose\"'},\n",
       " {'abbreviation': 'R01104',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01104_c',\n",
       "   'AraGEM: R_R01104_c',\n",
       "   'BiGG: GGLGH; YBR184W_6',\n",
       "   'JP_Creinhardtii_MSB: R_GGGH',\n",
       "   'KEGG: R01104',\n",
       "   'Name: Alpha-galactosidase; Galactosylglycerol galactohydrolase; galactosylglycerol galactohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03197[0] <=> (1) cpd00100[0] + (1) cpd00108[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00100;cpd00108;cpd03197',\n",
       "  'definition': '(1) H2O[0] + (1) Galactosylglycerol[0] => (1) Glycerol[0] + (1) Galactose[0]',\n",
       "  'deltag': -2.16,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03197[0] => (1) cpd00100[0] + (1) cpd00108[0]',\n",
       "  'id': 'rxn00819',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34772',\n",
       "  'name': 'Galactosylglycerol galactohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03197:0:0:\"Galactosylglycerol\";1:cpd00100:0:0:\"Glycerol\";1:cpd00108:0:0:\"Galactose\"'},\n",
       " {'abbreviation': 'Nitrite:(acceptor) oxidoreductase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: Nitrite:(acceptor) oxidoreductase'],\n",
       "  'code': '(1) cpd00075[0] + (1) cpd00109[0] <=> (1) cpd00110[0] + (1) cpd00209[0]',\n",
       "  'compound_ids': 'cpd00075;cpd00109;cpd00110;cpd00209',\n",
       "  'definition': '(1) Nitrite[0] + (1) Cytochrome c3+[0] <=> (1) Cytochrome c2+[0] + (1) Nitrate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.7.99.4'],\n",
       "  'equation': '(1) cpd00075[0] + (1) cpd00109[0] <=> (1) cpd00110[0] + (1) cpd00209[0]',\n",
       "  'id': 'rxn00820',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nitrite:(acceptor) oxidoreductase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:H:-1/O:1|CI:-2',\n",
       "  'stoichiometry': '-1:cpd00075:0:0:\"Nitrite\";-1:cpd00109:0:0:\"Cytochrome c3+\";1:cpd00110:0:0:\"Cytochrome c2+\";1:cpd00209:0:0:\"Nitrate\"'},\n",
       " {'abbreviation': 'R01108',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.8.5.1-RXN',\n",
       "   'BrachyCyc: 1.8.5.1-RXN',\n",
       "   'ChlamyCyc: 1.8.5.1-RXN',\n",
       "   'KEGG: R01108',\n",
       "   'MetaCyc: 1.8.5.1-RXN',\n",
       "   'Name: DHA reductase; GDOR; Glutathione dehydrogenase (ascorbate); Glutathione:dehydroascorbate oxidoreductase; dehydroascorbate reductase; dehydroascorbic acid reductase; dehydroascorbic reductase; glutathione dehydroascorbate reductase; glutathione dehydrogenase (ascorbate); glutathione:dehydroascorbate oxidoreductase; glutathione:dehydroascorbic acid oxidoreductase'],\n",
       "  'code': '(2) cpd00042[0] + (1) cpd00335[0] <=> (1) cpd00059[0] + (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00042;cpd00059;cpd00067;cpd00111;cpd00335',\n",
       "  'definition': '(2) GSH[0] + (1) Dehydroascorbate[0] <= (1) L-Ascorbate[0] + (1) H+[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': 0.47,\n",
       "  'deltagerr': 4.73,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.8.5.1'],\n",
       "  'equation': '(2) cpd00042[0] + (1) cpd00335[0] <= (1) cpd00059[0] + (1) cpd00067[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00821',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17276;rxn33425',\n",
       "  'name': 'glutathione:dehydroascorbate oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: COFACTOR-DEGRADATION (Cofactor, Prosthetic Group, Electron Carrier Degradation); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); PWY-2261 (ascorbate glutathione cycle); PWY-6370 (ascorbate recycling (cytosolic)); REACTIVE-OXYGEN-SPECIES-DEGRADATION (Reactive Oxygen Specie Degradation)',\n",
       "   'KEGG: rn00053 (Ascorbate and aldarate metabolism); rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00042:0:0:\"GSH\";-1:cpd00335:0:0:\"Dehydroascorbate\";1:cpd00059:0:0:\"L-Ascorbate\";1:cpd00067:0:0:H+;1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'R01109',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01109', 'Name: Glutathione:cystine oxidoreductase'],\n",
       "  'code': '(2) cpd00042[0] + (1) cpd01018[0] <=> (1) cpd00111[0] + (2) cpd00547[0]',\n",
       "  'compound_ids': 'cpd00042;cpd00111;cpd00547;cpd01018',\n",
       "  'definition': '(2) GSH[0] + (1) Cystine[0] <=> (1) Oxidized glutathione[0] + (2) Cysteine[0]',\n",
       "  'deltag': 4.18,\n",
       "  'deltagerr': 2.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.4.4'],\n",
       "  'equation': '(2) cpd00042[0] + (1) cpd01018[0] <=> (1) cpd00111[0] + (2) cpd00547[0]',\n",
       "  'id': 'rxn00822',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glutathione:cystine oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00042:0:0:\"GSH\";-1:cpd01018:0:0:\"Cystine\";1:cpd00111:0:0:\"Oxidized glutathione\";2:cpd00547:0:0:\"Cysteine\"'},\n",
       " {'abbreviation': 'R01111',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01111',\n",
       "   'MetaCyc: 1.8.4.3-RXN',\n",
       "   'Name: CoA:glutathione-disulfide oxidoreductase; coenzyme A:glutathione-disulfide oxidoreductase; coenzyme A:oxidized-glutathione oxidoreductase; glutathione coenzyme A-glutathione transhydrogenase; glutathione-CoA-glutathione transhydrogenase; glutathione-coenzyme A glutathione disulfide transhydrogenase; glutathione:coenzyme A-glutathione transhydrogenase'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd00111[0] <=> (1) cpd00042[0] + (1) cpd00682[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00042;cpd00111;cpd00682',\n",
       "  'definition': '(1) CoA[0] + (1) Oxidized glutathione[0] <=> (1) GSH[0] + (1) CoA-glutathione[0]',\n",
       "  'deltag': -0.13,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.4.3'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00111[0] <=> (1) cpd00042[0] + (1) cpd00682[0]',\n",
       "  'id': 'rxn00824',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'coenzyme A:glutathione-disulfide oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00111:0:0:\"Oxidized glutathione\";1:cpd00042:0:0:\"GSH\";1:cpd00682:0:0:\"CoA-glutathione\"'},\n",
       " {'abbreviation': 'R01113',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01113',\n",
       "   'iRS1563: R_R01113_c',\n",
       "   'Name: Glutathione:L-cystine oxidoreductase'],\n",
       "  'code': '(2) cpd00042[0] + (1) cpd00381[0] <=> (2) cpd00084[0] + (1) cpd00111[0]',\n",
       "  'compound_ids': 'cpd00042;cpd00084;cpd00111;cpd00381',\n",
       "  'definition': '(2) GSH[0] + (1) L-Cystine[0] <=> (2) L-Cysteine[0] + (1) Oxidized glutathione[0]',\n",
       "  'deltag': 5.25,\n",
       "  'deltagerr': 1.61,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.4.4'],\n",
       "  'equation': '(2) cpd00042[0] + (1) cpd00381[0] <=> (2) cpd00084[0] + (1) cpd00111[0]',\n",
       "  'id': 'rxn00826',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Glutathione:L-cystine oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00042:0:0:\"GSH\";-1:cpd00381:0:0:\"L-Cystine\";2:cpd00084:0:0:\"L-Cysteine\";1:cpd00111:0:0:\"Oxidized glutathione\"'},\n",
       " {'abbreviation': 'R01117',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01117',\n",
       "   'MetaCyc: RXN-9990',\n",
       "   'Name: CTP:N-acetylneuraminate cytidylyltransferase'],\n",
       "  'code': '(1) cpd00052[0] + (1) cpd00232[0] <=> (1) cpd00012[0] + (1) cpd00112[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00052;cpd00067;cpd00112;cpd00232',\n",
       "  'definition': '(1) CTP[0] + (1) Neu5Ac[0] => (1) PPi[0] + (1) H+[0] + (1) CMP-N-acetylneuraminate[0]',\n",
       "  'deltag': -12.96,\n",
       "  'deltagerr': 3.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.7.43'],\n",
       "  'equation': '(1) cpd00052[0] + (1) cpd00232[0] => (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00112[0]',\n",
       "  'id': 'rxn00827',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn25964;rxn25965',\n",
       "  'name': 'CTP:N-acetylneuraminate cytidylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00052:0:0:\"CTP\";-1:cpd00232:0:0:\"Neu5Ac\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd00112:0:0:\"CMP-N-acetylneuraminate\"'},\n",
       " {'abbreviation': 'rxn00828',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01118'],\n",
       "  'code': '(1) cpd00112[0] <=> (1) cpd00273[0]',\n",
       "  'compound_ids': 'cpd00112;cpd00273',\n",
       "  'definition': '(1) CMP-N-acetylneuraminate[0] <=> (1) Glycoprotein[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00112[0] <=> (1) cpd00273[0]',\n",
       "  'id': 'rxn00828',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00828',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'MI:C:8/H:18/N:1/O:2/P:-1|CI:2',\n",
       "  'stoichiometry': '-1:cpd00112:0:0:\"CMP-N-acetylneuraminate\";1:cpd00273:0:0:\"Glycoprotein\"'},\n",
       " {'abbreviation': 'R01121',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DIPHOSPHOMEVALONTE-DECARBOXYLASE-RXN',\n",
       "   'BiGG: DPMVD; MVD1',\n",
       "   'BrachyCyc: DIPHOSPHOMEVALONTE-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R01121',\n",
       "   'MetaCyc: DIPHOSPHOMEVALONTE-DECARBOXYLASE-RXN',\n",
       "   'Name: 5-pyrophosphomevalonate decarboxylase; ATP:(R)-5-diphosphomevalonate carboxy-lyase (adding ATP, isopentenyl-diphosphate-forming); ATP:(R)-5-diphosphomevalonate carboxy-lyase (adding ATP; isopentenyl-diphosphate-forming); ATP:(R)-5-diphosphomevalonate carboxy-lyase (dehydrating); diphosphomevalonate decarboxylase; mevalonate 5-diphosphate decarboxylase; mevalonate-5-pyrophosphate decarboxylase; pyrophosphomevalonate decarboxylase; pyrophosphomevalonic acid decarboxylase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00841[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00011[0] + (1) cpd00113[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00011;cpd00113;cpd00841',\n",
       "  'definition': '(1) ATP[0] + (1) 5-diphosphomevalonate[0] => (1) ADP[0] + (1) Phosphate[0] + (1) CO2[0] + (1) Isopentenyldiphosphate[0]',\n",
       "  'deltag': -7.82,\n",
       "  'deltagerr': 0.97,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.33'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00841[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00011[0] + (1) cpd00113[0]',\n",
       "  'id': 'rxn00829',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:(R)-5-diphosphomevalonate carboxy-lyase (adding ATP; isopentenyl-diphosphate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); DITERPENOID-SYN (Diterpenoid Biosynthesis); ERGOSTEROL-SYN-PWY (superpathway of ergosterol biosynthesis I); Ergosterol-Biosynthesis (Ergosterol Biosynthesis); GGPP-Biosynthesis (Geranylgeranyl Diphosphate Biosynthesis); ISOPRENOIDS (Hemiterpene Biosynthesis); Isopentenyl-Diphosphate-Biosynthesis (Isopentenyl Diphosphate Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); Mevalonate-Pathways (Mevalonate Pathways); PWY-5910 (superpathway of geranylgeranyldiphosphate biosynthesis I (via mevalonate)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-7391 (isoprene biosynthesis II (engineered)); PWY-922 (mevalonate pathway I); PWY66-5 (superpathway of cholesterol biosynthesis); Polyprenyl-Biosynthesis (Polyprenyl Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Sterol-Biosynthesis (Sterol Biosynthesis); Terpenoid-Biosynthesis (Terpenoid Biosynthesis)',\n",
       "   'KEGG: rn00900 (Terpenoid backbone biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00841:0:0:\"5-diphosphomevalonate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00011:0:0:\"CO2\";1:cpd00113:0:0:\"Isopentenyldiphosphate\"'},\n",
       " {'abbreviation': 'R01123',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: IPPISOM-RXN',\n",
       "   'BiGG: IDI1; IPDDI; IPDDIi',\n",
       "   'BrachyCyc: IPPISOM-RXN',\n",
       "   'KEGG: R01123',\n",
       "   'MetaCyc: IPPISOM-RXN',\n",
       "   'Name: Isopentenyl-diphosphate delta3-delta2-isomerase; isopentenyl-diphosphate D-isomerase; isopentenyl-diphosphate delta-isomerase; isopentenyl-diphosphate delta3-delta2-isomerase; isopentenylpyrophosphate delta-isomerase; isopentenylpyrophosphate isomerase; methylbutenylpyrophosphate isomerase'],\n",
       "  'code': '(1) cpd00113[0] <=> (1) cpd00202[0]',\n",
       "  'compound_ids': 'cpd00113;cpd00202',\n",
       "  'definition': '(1) Isopentenyldiphosphate[0] <=> (1) DMAPP[0]',\n",
       "  'deltag': -1.17,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.3.2'],\n",
       "  'equation': '(1) cpd00113[0] <=> (1) cpd00202[0]',\n",
       "  'id': 'rxn00830',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19935;rxn19936',\n",
       "  'name': 'Isopentenyl-diphosphate delta3-delta2-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); All-Trans-Farnesyl-PP-Biosynthesis (); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); DITERPENOID-SYN (Diterpenoid Biosynthesis); ERGOSTEROL-SYN-PWY (superpathway of ergosterol biosynthesis I); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ergosterol-Biosynthesis (Ergosterol Biosynthesis); GGPP-Biosynthesis (Geranylgeranyl Diphosphate Biosynthesis); ISOPRENOIDS (Hemiterpene Biosynthesis); Isopentenyl-Diphosphate-Biosynthesis (Isopentenyl Diphosphate Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); MEP-Pathways (Methylerythritol Phosphate Pathways); Mevalonate-Pathways (Mevalonate Pathways); NONMEVIPP-PWY (methylerythritol phosphate pathway I); POLYISOPRENSYN-PWY (polyisoprenoid biosynthesis (<i>E. coli</i>)); PWY-5121 (superpathway of geranylgeranyl diphosphate biosynthesis II (via MEP)); PWY-5123 (<i>trans, trans</i>-farnesyl diphosphate biosynthesis); PWY-5910 (superpathway of geranylgeranyldiphosphate biosynthesis I (via mevalonate)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-6174 (mevalonate pathway II (archaea)); PWY-6270 (isoprene biosynthesis I); PWY-6383 (mono-<i>trans</i>, poly-<i>cis</i> decaprenyl phosphate biosynthesis); PWY-6404 (superpathway of mycolyl-arabinogalactan-peptidoglycan complex biosynthesis); PWY-6859 (<i>all-trans</i>-farnesol biosynthesis); PWY-7102 (bisabolene biosynthesis (engineered)); PWY-7156 (superpathway of ergosterol biosynthesis II); PWY-7391 (isoprene biosynthesis II (engineered)); PWY-7392 (taxadiene biosynthesis (engineered)); PWY-7524 (mevalonate pathway III (archaea)); PWY-7560 (methylerythritol phosphate pathway II); PWY-922 (mevalonate pathway I); PWY66-5 (superpathway of cholesterol biosynthesis); Polyprenyl-Biosynthesis (Polyprenyl Biosynthesis); Quinone-Biosynthesis (Quinol and Quinone Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); Sterol-Biosynthesis (Sterol Biosynthesis); Terpenoid-Biosynthesis (Terpenoid Biosynthesis); UBISYN-PWY (superpathway of ubiquinol-8 biosynthesis (prokaryotic)); Ubiquinone-Biosynthesis (Ubiquinol Biosynthesis)',\n",
       "   'KEGG: rn00900 (Terpenoid backbone biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00113:0:0:\"Isopentenyldiphosphate\";1:cpd00202:0:0:\"DMAPP\"'},\n",
       " {'abbreviation': 'R01126',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5-NUCLEOTID-RXN; RXN-7607',\n",
       "   'BiGG: NTD11; NTD11pp; U32',\n",
       "   'BrachyCyc: RXN-7607',\n",
       "   'KEGG: R01126',\n",
       "   'MetaCyc: 5-NUCLEOTID-RXN; RXN-7607',\n",
       "   \"Name: 5'-AMP nucleotidase; 5'-AMPase; 5'-adenylic phosphatase; 5'-mononucleotidase; 5'-nucleotidase; 5'-nucleotidase (IMP); 5'-ribonucleotide phosphohydrolase; AMP phosphatase; AMP phosphohydrolase; AMPase; IMP 5'-nucleotidase; Inosine 5'-monophosphate phosphohydrolase; UMPase; adenosine 5'-phosphatase; adenosine monophosphatase; imp nucleotidase; snake venom 5'-nucleotidase; thimidine monophosphate nucleotidase; uridine 5'-nucleotidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00114[0] <=> (1) cpd00009[0] + (1) cpd00246[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00114;cpd00246',\n",
       "  'definition': '(1) H2O[0] + (1) IMP[0] <=> (1) Phosphate[0] + (1) Inosine[0]',\n",
       "  'deltag': -1.29,\n",
       "  'deltagerr': 0.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.5', '3.1.3.99'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00114[0] <=> (1) cpd00009[0] + (1) cpd00246[0]',\n",
       "  'id': 'rxn00831',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18974;rxn30907',\n",
       "  'name': \"Inosine 5'-monophosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Adenosine-Nucleotides-Degradation (Adenosine Nucleotide Degradation); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); P165-PWY (superpathway of purines degradation in plants); PWY-5044 (purine nucleotides degradation I (plants)); PWY-6596 (adenosine nucleotides degradation I); Purine-Degradation (Purine Nucleotide Degradation); Urate-Degradation (Urate Degradation)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00114:0:0:\"IMP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00246:0:0:\"Inosine\"'},\n",
       " {'abbreviation': 'R01127',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: IMPCYCLOHYDROLASE-RXN',\n",
       "   'BiGG: ADE16_2; ADE17_2; IMPC; IMPCir',\n",
       "   'BrachyCyc: IMPCYCLOHYDROLASE-RXN',\n",
       "   'KEGG: R01127',\n",
       "   'MetaCyc: IMPCYCLOHYDROLASE-RXN',\n",
       "   'Name: IMP 1,2-hydrolase (decyclizing); IMP cyclohydrolase; IMPcyclohydrolase; inosinate cyclohydrolase; inosinicase; phosphoribosylaminoimidazolecarboxamide formyltransferase / IMP cyclohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00114[0] <=> (1) cpd02884[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00114;cpd02884',\n",
       "  'definition': '(1) H2O[0] + (1) IMP[0] <=> (1) FAICAR[0]',\n",
       "  'deltag': 4.15,\n",
       "  'deltagerr': 1.31,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.2.3', '3.5.4.10'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00114[0] <=> (1) cpd02884[0]',\n",
       "  'id': 'rxn00832',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27945;rxn30710',\n",
       "  'name': 'IMP 1,2-hydrolase (decyclizing)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); IMP-Biosynthesis (Inosine-5'-phosphate Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6123 (inosine-5'-phosphate biosynthesis I); PWY-6124 (inosine-5'-phosphate biosynthesis II); PWY-7234 (inosine-5'-phosphate biosynthesis III); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)\",\n",
       "   'KEGG: rn00230 (Purine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00114:0:0:\"IMP\";1:cpd02884:0:0:\"FAICAR\"'},\n",
       " {'abbreviation': 'R01128',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01128',\n",
       "   'MetaCyc: INOSINATE-NUCLEOSIDASE-RXN',\n",
       "   \"Name: 5'-Inosinate phosphoribohydrolase; 5'-inosinate phosphoribohydrolase; inosinate nucleosidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00114[0] <=> (1) cpd00101[0] + (1) cpd00226[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00101;cpd00114;cpd00226',\n",
       "  'definition': '(1) H2O[0] + (1) IMP[0] <=> (1) ribose-5-phosphate[0] + (1) HYXN[0]',\n",
       "  'deltag': -2.51,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.12'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00114[0] <=> (1) cpd00101[0] + (1) cpd00226[0]',\n",
       "  'id': 'rxn00833',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"5'-Inosinate phosphoribohydrolase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00114:0:0:\"IMP\";1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00226:0:0:\"HYXN\"'},\n",
       " {'abbreviation': 'R01130',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: IMP-DEHYDROG-RXN',\n",
       "   'BiGG: FUN63; IMD3; IMD4; IMPD; PUR5; YAR075W',\n",
       "   'BrachyCyc: IMP-DEHYDROG-RXN',\n",
       "   'KEGG: R01130',\n",
       "   'MetaCyc: IMP-DEHYDROG-RXN',\n",
       "   \"Name: IMP dehydrogenase; IMP oxidoreductase; IMP:NAD+ oxidoreductase; IMPdehydrogenase; Inosinate dehydrogenase; Inosine monophosphate oxidoreductase; Inosine-5'-monophosphate dehydrogenase; Inosinic acid dehydrogenase; Putative inosine-5'-monophosphate dehydrogenase IMD1; inosinate dehydrogenase; inosine 5'-monophosphate dehydrogenase; inosine monophosphate dehydrogenase; inosine monophosphate oxidoreductase; inosine-5'-phosphate dehydrogenase; inosinic acid dehydrogenase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00114[0] <=> (1) cpd00004[0] + (1) cpd00497[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00114;cpd00497',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) IMP[0] <=> (1) NADH[0] + (1) H+[0] + (1) XMP[0]',\n",
       "  'deltag': 4.75,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.205'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00114[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00497[0]',\n",
       "  'id': 'rxn00834',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19930;rxn27944',\n",
       "  'name': 'IMP:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Adenosine-Nucleotides-Degradation (Adenosine Nucleotide Degradation); DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Degradation (Degradation/Utilization/Assimilation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); P165-PWY (superpathway of purines degradation in plants); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-5044 (purine nucleotides degradation I (plants)); PWY-5695 (inosine 5'-phosphate degradation); PWY-6125 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis II); PWY-6353 (purine nucleotides degradation II (aerobic)); PWY-6596 (adenosine nucleotides degradation I); PWY-7221 (guanosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7228 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Purine-Degradation (Purine Nucleotide Degradation); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis); URSIN-PWY (ureide biosynthesis); Urate-Degradation (Urate Degradation)\",\n",
       "   'KEGG: rn00230 (Purine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00114:0:0:\"IMP\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00497:0:0:\"XMP\"'},\n",
       " {'abbreviation': 'R01131',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: INSK; U20',\n",
       "   'BrachyCyc: INOSINEKIN-RXN',\n",
       "   'ChlamyCyc: INOSINEKIN-RXN',\n",
       "   'KEGG: R01131',\n",
       "   'MetaCyc: INOSINEKIN-RXN',\n",
       "   \"Name: ATP:inosine 5'-phosphotransferase; Inosine kinase; inosine kinase; inosine kinase (phosphorylating); inosine-guanosine kinase; insosine kinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00246[0] <=> (1) cpd00008[0] + (1) cpd00114[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00114;cpd00246',\n",
       "  'definition': '(1) ATP[0] + (1) Inosine[0] => (1) ADP[0] + (1) H+[0] + (1) IMP[0]',\n",
       "  'deltag': -4.87,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.73'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00246[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00114[0]',\n",
       "  'id': 'rxn00835',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19933;rxn32620;rxn32622;rxn32633',\n",
       "  'name': \"ATP:inosine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenine-Adenosine-Salvage (Adenine and Adenosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6611 (adenine and adenosine salvage V); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00246:0:0:\"Inosine\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00114:0:0:\"IMP\"'},\n",
       " {'abbreviation': 'R01132',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HYPOXANPRIBOSYLTRAN-RXN',\n",
       "   'BiGG: HPT1_1; HXPRT; HXPRTr',\n",
       "   'BrachyCyc: HYPOXANPRIBOSYLTRAN-RXN',\n",
       "   'KEGG: R01132',\n",
       "   'MetaCyc: HYPOXANPRIBOSYLTRAN-RXN',\n",
       "   \"Name: 6-hydroxypurine phosphoribosyltransferase; 6-mercaptopurine phosphoribosyltransferase; GMP pyrophosphorylase; GPRT; HGPRTase; HPRT; Hypoxanthine phosphoribosyltransferase; IMP pyrophosphorylase; IMP-GMP pyrophosphorylase; IMP:diphosphate phospho-D-ribosyltransferase; IMP:pyrophosphate phosphoribosyltransferase; guanine phosphoribosyltransferase; guanine-hypoxanthine phosphoribosyltransferase; guanosine 5'-phosphate pyrophosphorylase; guanosine phosphoribosyltransferase; guanylate pyrophosphorylase; guanylic pyrophosphorylase; hypoxanthine phosphoribosyltransferase; hypoxanthine phosphoribosyltransferase (Hypoxanthine); hypoxanthine-guanine phosphoribosyltransferase; hypoxanthinephosphoribosyltransferase-Hypoxanthine; hypoxantine-guanine phosphorybosyltransferase; inosinate pyrophosphorylase; inosine 5'-phosphate pyrophosphorylase; inosinic acid pyrophosphorylase; inosinic pyrophosphorylase; purine-6-thiol phosphoribosyltransferase; transphosphoribosidase\"],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00114[0] <=> (1) cpd00103[0] + (1) cpd00226[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00067;cpd00103;cpd00114;cpd00226',\n",
       "  'definition': '(1) PPi[0] + (1) H+[0] + (1) IMP[0] <= (1) PRPP[0] + (1) HYXN[0]',\n",
       "  'deltag': 6.11,\n",
       "  'deltagerr': 0.38,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.8'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00114[0] <= (1) cpd00103[0] + (1) cpd00226[0]',\n",
       "  'id': 'rxn00836',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32920;rxn32921;rxn32923;rxn32967;rxn32968;rxn32969',\n",
       "  'name': 'IMP:diphosphate phospho-D-ribosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenine-Adenosine-Salvage (Adenine and Adenosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6609 (adenine and adenosine salvage III); PWY-6610 (adenine salvage); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00067:0:0:\"H+\";-1:cpd00114:0:0:\"IMP\";1:cpd00103:0:0:\"PRPP\";1:cpd00226:0:0:\"HYXN\"'},\n",
       " {'abbreviation': 'R01134',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GMPR',\n",
       "   'EcoCyc: GMP-REDUCT-RXN',\n",
       "   'KEGG: R01134',\n",
       "   'MetaCyc: GMP-REDUCT-RXN',\n",
       "   'RiceCyc: GMP-REDUCT-RXN',\n",
       "   \"Name: GMP reductase; GMPreductase; NADH:guanosine-5'-phosphate oxidoreductase (deaminating); NADH:guanosine-5'-phosphate oxidoreductase(deaminating); NADPH2:guanosine-5'-phosphate oxidoreductase (deaminating); NADPH:GMP oxidoreductase (deaminating); guanosine 5'-monophosphate reductase; guanosine 5'-phosphate reductase; guanosine monophosphate reductase; guanylate reductase; inosine-5'-phosphate:NADP+ oxidoreductase (aminating)\"],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00013[0] + (1) cpd00114[0] <=> (1) cpd00005[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00013;cpd00067;cpd00114;cpd00126',\n",
       "  'definition': '(1) NADP[0] + (1) NH3[0] + (1) IMP[0] <= (1) NADPH[0] + (2) H+[0] + (1) GMP[0]',\n",
       "  'deltag': 10.46,\n",
       "  'deltagerr': 7.23,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.6.6.8', '1.7.1.7'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00013[0] + (1) cpd00114[0] <= (1) cpd00005[0] + (2) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00837',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30029',\n",
       "  'name': \"inosine-5'-phosphate:NADP+ oxidoreductase (aminating)\",\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00114:0:0:\"IMP\";1:cpd00005:0:0:\"NADPH\";2:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01135',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ADENYLOSUCCINATE-SYNTHASE-RXN',\n",
       "   'BiGG: ADE12; ADSS; ADSS_i',\n",
       "   'BrachyCyc: ADENYLOSUCCINATE-SYNTHASE-RXN',\n",
       "   'KEGG: R01135',\n",
       "   'MetaCyc: ADENYLOSUCCINATE-SYNTHASE-RXN',\n",
       "   'Name: IMP-aspartate ligase; IMP:L-aspartate ligase (GDP-forming); adenylosuccinate synthase; adenylosuccinate synthetase; adenylosuccinatesynthase; succino-AMP synthetase; succinoadenylic kinosynthetase'],\n",
       "  'code': '(1) cpd00038[0] + (1) cpd00041[0] + (1) cpd00114[0] <=> (1) cpd00009[0] + (1) cpd00031[0] + (1) cpd02375[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00031;cpd00038;cpd00041;cpd00067;cpd00114;cpd02375',\n",
       "  'definition': '(1) GTP[0] + (1) L-Aspartate[0] + (1) IMP[0] => (1) Phosphate[0] + (1) GDP[0] + (2) H+[0] + (1) Adenylosuccinate[0]',\n",
       "  'deltag': 2.34,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.4.4'],\n",
       "  'equation': '(1) cpd00038[0] + (1) cpd00041[0] + (1) cpd00114[0] => (1) cpd00009[0] + (1) cpd00031[0] + (2) cpd00067[0] + (1) cpd02375[0]',\n",
       "  'id': 'rxn00838',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27686',\n",
       "  'name': 'IMP:L-aspartate ligase (GDP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6126 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis II); PWY-7219 (adenosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7229 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism); rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00038:0:0:\"GTP\";-1:cpd00041:0:0:\"L-Aspartate\";-1:cpd00114:0:0:\"IMP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00031:0:0:\"GDP\";2:cpd00067:0:0:\"H+\";1:cpd02375:0:0:\"Adenylosuccinate\"'},\n",
       " {'abbreviation': 'R01137',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DADPKIN-RXN',\n",
       "   'BiGG: NDPK8; YNK1_7',\n",
       "   'BrachyCyc: DADPKIN-RXN',\n",
       "   'KEGG: R01137',\n",
       "   'MetaCyc: DADPKIN-RXN',\n",
       "   'Name: ATP:dADP phosphotransferase; dADP phosphotransferase; dadp kinase; nucleoside-diphosphate kinase (ATP:dADP); nucleoside-diphosphatekinase-ATP-dADP'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00177[0] <=> (1) cpd00008[0] + (1) cpd00115[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00115;cpd00177',\n",
       "  'definition': '(1) ATP[0] + (1) dADP[0] <=> (1) ADP[0] + (1) dATP[0]',\n",
       "  'deltag': 0.74,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.4.6'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00177[0] <=> (1) cpd00008[0] + (1) cpd00115[0]',\n",
       "  'id': 'rxn00839',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19384;rxn27777;rxn27778;rxn31847',\n",
       "  'name': 'ATP:dADP phosphotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Adenosine-Deoxy-Denovo-Bbiosynthesis (Adenosine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6126 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis II); PWY-7220 (adenosine deoxyribonucleotides <i>de novo</i> biosynthesis II); PWY-7224 (purine deoxyribonucleosides salvage); PWY-7227 (adenosine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY-7229 (superpathway of adenosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Deoxyribonuc-De-Novo-Biosynthesis (Purine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)\",\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00177:0:0:\"dADP\";1:cpd00008:0:0:\"ADP\";1:cpd00115:0:0:\"dATP\"'},\n",
       " {'abbreviation': 'R01138',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_DAPOP',\n",
       "   'KEGG: R01138',\n",
       "   'TS_Athaliana: rxn627_c; rxn627_p',\n",
       "   'iRS1563: R_R01138_c',\n",
       "   'iRS1597: R01138_c',\n",
       "   'Name: dATP:pyruvate 2-O-phosphotransferase; dATP:pyruvate O2-phosphotransferase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00115[0] <=> (1) cpd00061[0] + (1) cpd00177[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00061;cpd00067;cpd00115;cpd00177',\n",
       "  'definition': '(1) Pyruvate[0] + (1) dATP[0] <=> (1) Phosphoenolpyruvate[0] + (1) H+[0] + (1) dADP[0]',\n",
       "  'deltag': 5.79,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.40'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00115[0] <=> (1) cpd00061[0] + (1) cpd00067[0] + (1) cpd00177[0]',\n",
       "  'id': 'rxn00840',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32700;rxn35439',\n",
       "  'name': 'dATP:pyruvate 2-O-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00115:0:0:\"dATP\";1:cpd00061:0:0:\"Phosphoenolpyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00177:0:0:\"dADP\"'},\n",
       " {'abbreviation': 'R01139',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01139', 'Name: dATP:D-glucose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00027[0] + (1) cpd00115[0] <=> (1) cpd00079[0] + (1) cpd00177[0]',\n",
       "  'compound_ids': 'cpd00027;cpd00067;cpd00079;cpd00115;cpd00177',\n",
       "  'definition': '(1) D-Glucose[0] + (1) dATP[0] => (1) H+[0] + (1) D-glucose-6-phosphate[0] + (1) dADP[0]',\n",
       "  'deltag': -4.75,\n",
       "  'deltagerr': 0.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1'],\n",
       "  'equation': '(1) cpd00027[0] + (1) cpd00115[0] => (1) cpd00067[0] + (1) cpd00079[0] + (1) cpd00177[0]',\n",
       "  'id': 'rxn00841',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'dATP:D-glucose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00027:0:0:\"D-Glucose\";-1:cpd00115:0:0:\"dATP\";1:cpd00067:0:0:\"H+\";1:cpd00079:0:0:\"D-glucose-6-phosphate\";1:cpd00177:0:0:\"dADP\"'},\n",
       " {'abbreviation': 'R01140',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01140',\n",
       "   'iRS1563: R_R01140_c',\n",
       "   'Name: dATP:D-fructose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00082[0] + (1) cpd00115[0] <=> (1) cpd00072[0] + (1) cpd00177[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00072;cpd00082;cpd00115;cpd00177',\n",
       "  'definition': '(1) D-Fructose[0] + (1) dATP[0] => (1) H+[0] + (1) D-fructose-6-phosphate[0] + (1) dADP[0]',\n",
       "  'deltag': -3.55,\n",
       "  'deltagerr': 0.23,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1'],\n",
       "  'equation': '(1) cpd00082[0] + (1) cpd00115[0] => (1) cpd00067[0] + (1) cpd00072[0] + (1) cpd00177[0]',\n",
       "  'id': 'rxn00842',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'dATP:D-fructose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00082:0:0:\"D-Fructose\";-1:cpd00115:0:0:\"dATP\";1:cpd00067:0:0:\"H+\";1:cpd00072:0:0:\"D-fructose-6-phosphate\";1:cpd00177:0:0:\"dADP\"'},\n",
       " {'abbreviation': 'R01142',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01142',\n",
       "   'MetaCyc: METHANE-MONOOXYGENASE-RXN',\n",
       "   'Name: Methane,NADH:oxygen oxidoreductase (hydroxylating); methane hydroxylase; methane monooxygenase (soluble); methane,NAD(P)H:oxygen oxidoreductase (hydroxylating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd01024[0] <=> (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00116[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00007;cpd00067;cpd00116;cpd01024',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (1) H+[0] + (1) Methane[0] => (1) H2O[0] + (1) NAD[0] + (1) Methanol[0]',\n",
       "  'deltag': -90.39,\n",
       "  'deltagerr': 1.25,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.25'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00067[0] + (1) cpd01024[0] => (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00116[0]',\n",
       "  'id': 'rxn00843',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Methane,NADH:oxygen oxidoreductase (hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Denitrification (Denitrification); Methane-Oxidation (methane oxidation to methanol); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-1641 (methane oxidation to methanol I); PWY-1882 (superpathway of C1 compounds oxidation to CO<sub>2</sub>); PWY-6523 (nitrite-dependent anaerobic methane oxidation)',\n",
       "   'KEGG: rn00680 (Methane metabolism); rn01200 (Carbon metabolism); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-1:cpd00067:0:0:\"H+\";-1:cpd01024:0:0:\"Methane\";1:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\";1:cpd00116:0:0:\"Methanol\"'},\n",
       " {'abbreviation': 'R01143',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01143',\n",
       "   'MetaCyc: METHANE-MONOOXYGENASE-RXN',\n",
       "   'Name: methane hydroxylase; methane monooxygenase (soluble); methane,NAD(P)H:oxygen oxidoreductase (hydroxylating); methane,NADPH:oxygen oxidoreductase (hydroxylating)'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd01024[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00116[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00067;cpd00116;cpd01024',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) H+[0] + (1) Methane[0] => (1) H2O[0] + (1) NADP[0] + (1) Methanol[0]',\n",
       "  'deltag': -90.39,\n",
       "  'deltagerr': 1.25,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.25'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00067[0] + (1) cpd01024[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00116[0]',\n",
       "  'id': 'rxn00844',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'methane,NADPH:oxygen oxidoreductase (hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Denitrification (Denitrification); Methane-Oxidation (methane oxidation to methanol); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-1641 (methane oxidation to methanol I); PWY-1882 (superpathway of C1 compounds oxidation to CO<sub>2</sub>); PWY-6523 (nitrite-dependent anaerobic methane oxidation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00067:0:0:\"H+\";-1:cpd01024:0:0:\"Methane\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00116:0:0:\"Methanol\"'},\n",
       " {'abbreviation': 'R01144',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01144',\n",
       "   'MetaCyc: 4-METHYLOXALOACETATE-ESTERASE-RXN',\n",
       "   'Name: 4-methyloxaloacetate esterase; Oxaloacetate-4-methyl-ester oxaloacetohydrolase; oxaloacetate-4-methyl-ester oxaloacetohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02330[0] <=> (1) cpd00032[0] + (1) cpd00116[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00032;cpd00067;cpd00116;cpd02330',\n",
       "  'definition': '(1) H2O[0] + (1) Oxaloacetate 4-methyl ester[0] <=> (1) Oxaloacetate[0] + (1) H+[0] + (1) Methanol[0]',\n",
       "  'deltag': 0.17,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.44'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02330[0] <=> (1) cpd00032[0] + (1) cpd00067[0] + (1) cpd00116[0]',\n",
       "  'id': 'rxn00845',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Oxaloacetate-4-methyl-ester oxaloacetohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02330:0:0:\"Oxaloacetate 4-methyl ester\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00067:0:0:\"H+\";1:cpd00116:0:0:\"Methanol\"'},\n",
       " {'abbreviation': 'R01145',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01145',\n",
       "   'MetaCyc: MONOMETHYL-SULFATASE-RXN',\n",
       "   'Name: Monomethyl-sulfate sulfohydrolase; monomethyl-sulfatase; monomethyl-sulfate sulfohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01751[0] <=> (1) cpd00048[0] + (1) cpd00116[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00048;cpd00067;cpd00116;cpd01751',\n",
       "  'definition': '(1) H2O[0] + (1) Methyl sulfate[0] => (1) Sulfate[0] + (1) H+[0] + (1) Methanol[0]',\n",
       "  'deltag': -13.06,\n",
       "  'deltagerr': 1.12,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.6.16'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01751[0] => (1) cpd00048[0] + (1) cpd00067[0] + (1) cpd00116[0]',\n",
       "  'id': 'rxn00846',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Monomethyl-sulfate sulfohydrolase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01751:0:0:\"Methyl sulfate\";1:cpd00048:0:0:\"Sulfate\";1:cpd00067:0:0:\"H+\";1:cpd00116:0:0:\"Methanol\"'},\n",
       " {'abbreviation': 'rxn00847',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00097[0] + (1) cpd00116[0] <=> (1) cpd00055[0] + (1) cpd00986[0]',\n",
       "  'compound_ids': 'cpd00055;cpd00097;cpd00116;cpd00986',\n",
       "  'definition': '(1) Pyrroloquinoline-quinone[0] + (1) Methanol[0] <=> (1) Formaldehyde[0] + (1) PQQH2[0]',\n",
       "  'deltag': -7.56,\n",
       "  'deltagerr': 1.69,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.99.8'],\n",
       "  'equation': '(1) cpd00097[0] + (1) cpd00116[0] <=> (1) cpd00055[0] + (1) cpd00986[0]',\n",
       "  'id': 'rxn00847',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'rxn00847',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00097:0:0:\"Pyrroloquinoline-quinone\";-1:cpd00116:0:0:\"Methanol\";1:cpd00055:0:0:\"Formaldehyde\";1:cpd00986:0:0:\"PQQH2\"'},\n",
       " {'abbreviation': 'R01147',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN0-5240',\n",
       "   'BiGG: ALATA_D2',\n",
       "   'ChlamyCyc: RXN0-5240',\n",
       "   'KEGG: R01147',\n",
       "   'MetaCyc: RXN0-5240',\n",
       "   \"Name: D-alanine transaminase; Pyridoxamine-5'-phosphate:2-oxoglutarate aminotransferase\"],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00493[0] <=> (1) cpd00016[0] + (1) cpd00117[0]',\n",
       "  'compound_ids': 'cpd00016;cpd00020;cpd00117;cpd00493',\n",
       "  'definition': '(1) Pyruvate[0] + (1) Pyridoxamine phosphate[0] <=> (1) Pyridoxal phosphate[0] + (1) D-Alanine[0]',\n",
       "  'deltag': -0.43,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.54'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00493[0] <=> (1) cpd00016[0] + (1) cpd00117[0]',\n",
       "  'id': 'rxn00848',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28681;rxn33135',\n",
       "  'name': \"Pyridoxamine-5'-phosphate:2-oxoglutarate aminotransferase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00750 (Vitamin B6 metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00493:0:0:\"Pyridoxamine phosphate\";1:cpd00016:0:0:\"Pyridoxal phosphate\";1:cpd00117:0:0:\"D-Alanine\"'},\n",
       " {'abbreviation': 'R01148',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: D-ALANINE-AMINOTRANSFERASE-RXN',\n",
       "   'BiGG: ALATA_D',\n",
       "   'ChlamyCyc: D-ALANINE-AMINOTRANSFERASE-RXN',\n",
       "   'KEGG: R01148',\n",
       "   'MetaCyc: D-ALANINE-AMINOTRANSFERASE-RXN',\n",
       "   'Name: D-Alanine:2-oxoglutarate aminotransferase; D-alanine aminotransferase; D-alanine transaminase; D-alanine-D-glutamate transaminase; D-alanine:2-oxoglutarate aminotransferase; D-amino acid aminotransferase; D-amino acid transaminase; D-amino-acid transaminase; D-aspartate aminotransferase; D-aspartate transaminase; D-aspartic aminotransferase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00117[0] <=> (1) cpd00020[0] + (1) cpd00186[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00024;cpd00117;cpd00186',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) D-Alanine[0] <=> (1) Pyruvate[0] + (1) D-Glutamate[0]',\n",
       "  'deltag': 0.21,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.21'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00117[0] <=> (1) cpd00020[0] + (1) cpd00186[0]',\n",
       "  'id': 'rxn00849',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27773;rxn27774',\n",
       "  'name': 'D-Alanine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00473 (D-Alanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00117:0:0:\"D-Alanine\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00186:0:0:\"D-Glutamate\"'},\n",
       " {'abbreviation': 'R01149',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01149',\n",
       "   'MetaCyc: D-ALANINE-GAMMA-GLUTAMYLTRANSFERASE-RXN',\n",
       "   'Name: D-alanine gamma-glutamyltransferase; L-Glutamine:D-alanine gamma-glutamyltransferase; L-glutamine:D-alanine gamma-glutamyltransferase'],\n",
       "  'code': '(1) cpd00053[0] + (1) cpd00117[0] <=> (1) cpd00013[0] + (1) cpd02343[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00053;cpd00117;cpd02343',\n",
       "  'definition': '(1) L-Glutamine[0] + (1) D-Alanine[0] => (1) NH3[0] + (1) gamma-L-Glutamyl-D-alanine[0]',\n",
       "  'deltag': -1.52,\n",
       "  'deltagerr': 0.88,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.2.14'],\n",
       "  'equation': '(1) cpd00053[0] + (1) cpd00117[0] => (1) cpd00013[0] + (1) cpd02343[0]',\n",
       "  'id': 'rxn00850',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Glutamine:D-alanine gamma-glutamyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00117:0:0:\"D-Alanine\";1:cpd00013:0:0:\"NH3\";1:cpd02343:0:0:\"gamma-L-Glutamyl-D-alanine\"'},\n",
       " {'abbreviation': 'R01150',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: DALADALALIG-RXN',\n",
       "   'BiGG: ALAALA; ALAALAr',\n",
       "   'BrachyCyc: DALADALALIG-RXN',\n",
       "   'KEGG: R01150',\n",
       "   'MetaCyc: DALADALALIG-RXN',\n",
       "   'Name: D-AlanineD-Alanine ligase (ADP-forming); D-alanine--D-alanine ligase; D-alanine-D-alanine ligase; D-alanine-D-alanine ligase (reversible); D-alanine-D-alanineligase-reversible; D-alanine:D-alanine ligase (ADP-forming); MurE synthetase [ambiguous]; alanine:alanine ligase (ADP-forming); alanylalanine synthetase; d-alanine--d-alanine ligase'],\n",
       "  'code': '(1) cpd00002[0] + (2) cpd00117[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00731[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00067;cpd00117;cpd00731',\n",
       "  'definition': '(1) ATP[0] + (2) D-Alanine[0] => (1) ADP[0] + (1) Phosphate[0] + (1) H+[0] + (1) Ala-Ala[0]',\n",
       "  'deltag': -6.08,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.4'],\n",
       "  'equation': '(1) cpd00002[0] + (2) cpd00117[0] => (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00731[0]',\n",
       "  'id': 'rxn00851',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn13802;rxn31784;rxn38070',\n",
       "  'name': 'D-alanine:D-alanine ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetylmuramoyl-Pentapeptide-Biosynthesis (UDP-<i>N</i>-Acetylmuramoyl-Pentapeptide Biosynthesis); Antibiotic-Resistance (Antibiotic Resistance); Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); Cell-Wall-Biosynthesis (Cell Wall Biosynthesis); Detoxification (Detoxification); PEPTIDOGLYCANSYN-PWY (peptidoglycan biosynthesis I (<I>meso</I>-diaminopimelate containing)); PWY-5265 (peptidoglycan biosynthesis II (staphylococci)); PWY-6385 (peptidoglycan biosynthesis III (mycobacteria)); PWY-6386 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis II (lysine-containing)); PWY-6387 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis I (<i>meso</i>-diaminopimelate containing)); PWY-6470 (peptidoglycan biosynthesis V (&beta;-lactam resistance)); PWY-6471 (peptidoglycan biosynthesis IV (Enterococcus faecium)); PWY-7953 (UDP-<i>N</i>-acetylmuramoyl-pentapeptide biosynthesis III (<i>meso</i>-diaminopimelate containing)); Peptidoglycan-Biosynthesis (Peptidoglycan Biosynthesis)',\n",
       "   'KEGG: rn00473 (D-Alanine metabolism); rn00550 (Peptidoglycan biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-2:cpd00117:0:0:\"D-Alanine\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00731:0:0:\"Ala-Ala\"'},\n",
       " {'abbreviation': 'R01151',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PUTRESCINE-OXIDASE-RXN',\n",
       "   'ChlamyCyc: PUTRESCINE-OXIDASE-RXN',\n",
       "   'CornCyc: PUTRESCINE-OXIDASE-RXN',\n",
       "   'KEGG: R01151',\n",
       "   'MetaCyc: PUTRESCINE-OXIDASE-RXN',\n",
       "   'Name: Putrescine:oxygen oxidoreductase (deaminating); putrescine oxidase; putrescine:oxygen oxidoreductase; putrescine:oxygen oxidoreductase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00118[0] <=> (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00434[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00013;cpd00025;cpd00118;cpd00434',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) Putrescine[0] => (1) NH3[0] + (1) H2O2[0] + (1) 4-Aminobutanal[0]',\n",
       "  'deltag': -22.19,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.3.10', '1.4.3.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00118[0] => (1) cpd00013[0] + (1) cpd00025[0] + (1) cpd00434[0]',\n",
       "  'id': 'rxn00852',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20637;rxn20638;rxn35374',\n",
       "  'name': 'Putrescine:oxygen oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-2 (putrescine degradation IV); Putrescine-Degradation (Putrescine Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00118:0:0:\"Putrescine\";1:cpd00013:0:0:\"NH3\";1:cpd00025:0:0:\"H2O2\";1:cpd00434:0:0:\"4-Aminobutanal\"'},\n",
       " {'abbreviation': 'R01152',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: N-CARBAMOYLPUTRESCINE-AMIDASE-RXN',\n",
       "   'BiGG: CBMLPTAH',\n",
       "   'BrachyCyc: N-CARBAMOYLPUTRESCINE-AMIDASE-RXN',\n",
       "   'KEGG: R01152',\n",
       "   'MetaCyc: N-CARBAMOYLPUTRESCINE-AMIDASE-RXN',\n",
       "   'Name: N-Carbamoylputrescine amidohydrolase; N-carbamoylputrescine amidase; N-carbamoylputrescine amidohydrolase; NCP; carbamoylputrescine hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00341[0] <=> (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00118[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00013;cpd00067;cpd00118;cpd00341',\n",
       "  'definition': '(1) H2O[0] + (2) H+[0] + (1) N-Carbamoylputrescine[0] => (1) CO2[0] + (1) NH3[0] + (1) Putrescine[0]',\n",
       "  'deltag': -12.69,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.53'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00067[0] + (1) cpd00341[0] => (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00118[0]',\n",
       "  'id': 'rxn00853',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35372',\n",
       "  'name': 'N-Carbamoylputrescine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGDEG-III-PWY (L-arginine degradation IV (arginine decarboxylase/agmatine deiminase pathway)); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); POLYAMINSYN3-PWY (superpathway of polyamine biosynthesis II); PWY-43 (putrescine biosynthesis II); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Putrescine-Biosynthesis (Putrescine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00067:0:0:\"H+\";-1:cpd00341:0:0:\"N-Carbamoylputrescine\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00118:0:0:\"Putrescine\"'},\n",
       " {'abbreviation': 'R01153',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['ChlamyCyc: PUTRESCINE-N-METHYLTRANSFERASE-RXN',\n",
       "   'KEGG: R01153',\n",
       "   'MetaCyc: PUTRESCINE-N-METHYLTRANSFERASE-RXN',\n",
       "   'PlantCyc: PUTRESCINE-N-METHYLTRANSFERASE-RXN',\n",
       "   'Name: Putrescine N-methyltransferase; S-Adenosyl-L-methionine:putrescine N-methyltransferase; S-adenosyl-L-methionine:putrescine N-methyltransferase; putrescine N-methyltransferase; putrescine methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00118[0] <=> (1) cpd00019[0] + (1) cpd01766[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00118;cpd01766',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Putrescine[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) N-Methylputrescine[0]',\n",
       "  'deltag': -10.46,\n",
       "  'deltagerr': 4.97,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.53'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00118[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd01766[0]',\n",
       "  'id': 'rxn00854',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28108',\n",
       "  'name': 'S-Adenosyl-L-methionine:putrescine N-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PWY-5315 (N-methyl-&Delta;<sup>1</sup>-pyrrolinium cation biosynthesis); PWY-7341 (superpathway of hyoscyamine and scopolamine biosynthesis); PWY-7342 (superpathway of nicotine biosynthesis); PYRROLIDINE-ALKALOIDS (Pyrrolidine, Piperidine and Pyridine Alkaloid Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TROPANE-ALKALOIDS (Tropane Alkaloid Biosynthesis)',\n",
       "   'KEGG: rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00118:0:0:\"Putrescine\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd01766:0:0:\"N-Methylputrescine\"'},\n",
       " {'abbreviation': 'R01154',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01154',\n",
       "   'MetaCyc: RXN-0',\n",
       "   'Name: acetyl-CoA:putrescine N-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00118[0] <=> (1) cpd00010[0] + (1) cpd01758[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00067;cpd00118;cpd01758',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Putrescine[0] => (1) CoA[0] + (1) H+[0] + (1) N-Acetylputrescine[0]',\n",
       "  'deltag': -4.8,\n",
       "  'deltagerr': 0.43,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.57'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00118[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01758[0]',\n",
       "  'id': 'rxn00855',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'acetyl-CoA:putrescine N-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-0 (putrescine degradation III); Putrescine-Degradation (Putrescine Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00118:0:0:\"Putrescine\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd01758:0:0:\"N-Acetylputrescine\"'},\n",
       " {'abbreviation': 'R01155',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PTRCTA',\n",
       "   'EcoCyc: PUTTRANSAM-RXN',\n",
       "   'KEGG: R01155',\n",
       "   'MetaCyc: PUTTRANSAM-RXN',\n",
       "   'iAF1260: PTRCTA',\n",
       "   'Name: Putrescine Transaminase; putrescine:2-oxoglutarate aminotransferase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00118[0] <=> (1) cpd00023[0] + (1) cpd00434[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00118;cpd00434',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) Putrescine[0] <=> (1) L-Glutamate[0] + (1) 4-Aminobutanal[0]',\n",
       "  'deltag': 0.82,\n",
       "  'deltagerr': 0.47,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.29', '2.6.1.82'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00118[0] <=> (1) cpd00023[0] + (1) cpd00434[0]',\n",
       "  'id': 'rxn00856',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'putrescine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); ARGDEG-PWY (superpathway of L-arginine, putrescine, and 4-aminobutanoate degradation); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); ORNARGDEG-PWY (superpathway of L-arginine and L-ornithine degradation); ORNDEG-PWY (superpathway of ornithine degradation); PUTDEG-PWY (putrescine degradation I); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Putrescine-Degradation (Putrescine Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00118:0:0:\"Putrescine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00434:0:0:\"4-Aminobutanal\"'},\n",
       " {'abbreviation': 'R01156',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01156',\n",
       "   'MetaCyc: ACETYLPUTRESCINE-DEACETYLASE-RXN',\n",
       "   'TS_Athaliana: rxn628_c; rxn628_m; rxn628_p; rxn628_x',\n",
       "   'Name: N-Acetylputrescine acetylhydrolase; N-acetylputrescine acetylhydrolase; acetylputrescine deacetylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01758[0] <=> (1) cpd00029[0] + (1) cpd00118[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00118;cpd01758',\n",
       "  'definition': '(1) H2O[0] + (1) N-Acetylputrescine[0] <=> (1) Acetate[0] + (1) Putrescine[0]',\n",
       "  'deltag': -2.51,\n",
       "  'deltagerr': 0.44,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.62'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01758[0] <=> (1) cpd00029[0] + (1) cpd00118[0]',\n",
       "  'id': 'rxn00857',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32416;rxn32417;rxn32419',\n",
       "  'name': 'N-Acetylputrescine acetylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01758:0:0:\"N-Acetylputrescine\";1:cpd00029:0:0:\"Acetate\";1:cpd00118:0:0:\"Putrescine\"'},\n",
       " {'abbreviation': 'R01157',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: AGMATIN-RXN',\n",
       "   'BiGG: AGMT',\n",
       "   'BrachyCyc: AGMATIN-RXN',\n",
       "   'KEGG: R01157',\n",
       "   'MetaCyc: AGMATIN-RXN',\n",
       "   'Name: Agmatinase; Agmatine amidinohydrolase; SpeB; agmatine amidinohydrolase; agmatine ureohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00152[0] <=> (1) cpd00073[0] + (1) cpd00118[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00073;cpd00118;cpd00152',\n",
       "  'definition': '(1) H2O[0] + (1) Agmatine[0] => (1) Urea[0] + (1) Putrescine[0]',\n",
       "  'deltag': -13.43,\n",
       "  'deltagerr': 1.51,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.3.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00152[0] => (1) cpd00073[0] + (1) cpd00118[0]',\n",
       "  'id': 'rxn00858',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30601;rxn33451;rxn33452',\n",
       "  'name': 'Agmatine amidinohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGDEG-PWY (superpathway of L-arginine, putrescine, and 4-aminobutanoate degradation); ARGININE-DEG (L-arginine Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); ORNARGDEG-PWY (superpathway of L-arginine and L-ornithine degradation); POLYAMSYN-PWY (superpathway of polyamine biosynthesis I); PWY-40 (putrescine biosynthesis I); PWY-6305 (putrescine biosynthesis IV); PWY0-823 (L-arginine degradation III (arginine decarboxylase/agmatinase pathway)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Putrescine-Biosynthesis (Putrescine Biosynthesis)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00152:0:0:\"Agmatine\";1:cpd00073:0:0:\"Urea\";1:cpd00118:0:0:\"Putrescine\"'},\n",
       " {'abbreviation': 'R01158',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-8001',\n",
       "   'BiGG: HIS4_3; HISTD',\n",
       "   'BrachyCyc: RXN-8001',\n",
       "   'KEGG: R01158',\n",
       "   'MetaCyc: RXN-8001',\n",
       "   'Name: Histidinol dehydrogenase; L-Histidinol:NAD+ oxidoreductase; L-histidinol dehydrogenase; L-histidinol:NAD+ oxidoreductase; histidinol dehydrogenase'],\n",
       "  'code': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00641[0] <=> (2) cpd00004[0] + (1) cpd00119[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00119;cpd00641',\n",
       "  'definition': '(1) H2O[0] + (2) NAD[0] + (1) L-Histidinol[0] <=> (2) NADH[0] + (3) H+[0] + (1) L-Histidine[0]',\n",
       "  'deltag': -1.52,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.23'],\n",
       "  'equation': '(1) cpd00001[0] + (2) cpd00003[0] + (1) cpd00641[0] <=> (2) cpd00004[0] + (3) cpd00067[0] + (1) cpd00119[0]',\n",
       "  'id': 'rxn00859',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28381',\n",
       "  'name': 'L-Histidinol:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-2:cpd00003:0:0:\"NAD\";-1:cpd00641:0:0:\"L-Histidinol\";2:cpd00004:0:0:\"NADH\";3:cpd00067:0:0:\"H+\";1:cpd00119:0:0:\"L-Histidine\"'},\n",
       " {'abbreviation': 'R01159',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01159_c',\n",
       "   'AraGEM: R_R01159_c',\n",
       "   'BiGG: ARMT; HMT1',\n",
       "   'KEGG: R01159',\n",
       "   'Maize_C4GEM: R_R01159_c',\n",
       "   'Name: HNRNP arginine N-methyltransferase; S-adenosyl-L-methionine:L-histidine N-methyltransferase'],\n",
       "  'code': '(1) cpd00019[0] + (1) cpd00848[0] <=> (1) cpd00017[0] + (1) cpd00119[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00119;cpd00848',\n",
       "  'definition': '(1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) 1-Methylhistidine[0] <=> (1) S-Adenosyl-L-methionine[0] + (1) L-Histidine[0]',\n",
       "  'deltag': 6.74,\n",
       "  'deltagerr': 5.11,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.-'],\n",
       "  'equation': '(1) cpd00019[0] + (1) cpd00067[0] + (1) cpd00848[0] <=> (1) cpd00017[0] + (1) cpd00119[0]',\n",
       "  'id': 'rxn00860',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-adenosyl-L-methionine:L-histidine N-methyltransferase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00340 (Histidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";-1:cpd00067:0:0:\"H+\";-1:cpd00848:0:0:\"1-Methylhistidine\";1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";1:cpd00119:0:0:\"L-Histidine\"'},\n",
       " {'abbreviation': 'R01160',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01160',\n",
       "   'Name: Acetyl-CoA:L-histidine N-acetyltransferase; acetyl-CoA:L-histidine N-acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00119[0] <=> (1) cpd00010[0] + (1) cpd01921[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00119;cpd01921',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) L-Histidine[0] => (1) CoA[0] + (1) N-Acetyl-L-histidine[0]',\n",
       "  'deltag': -7.01,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.33'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00119[0] => (1) cpd00010[0] + (1) cpd01921[0]',\n",
       "  'id': 'rxn00861',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'acetyl-CoA:L-histidine N-acetyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00119:0:0:\"L-Histidine\";1:cpd00010:0:0:\"CoA\";1:cpd01921:0:0:\"N-Acetyl-L-histidine\"'},\n",
       " {'abbreviation': 'R01161',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01161',\n",
       "   'MetaCyc: HISTTRANSAM-RXN',\n",
       "   'Name: L-histidine:2-oxoglutarate aminotransferase; histidine aminotransferase; histidine transaminase; histidine-2-oxoglutarate aminotransferase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00119[0] <=> (1) cpd00023[0] + (1) cpd02091[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00119;cpd02091',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Histidine[0] <=> (1) L-Glutamate[0] + (1) Imidazole pyruvate[0]',\n",
       "  'deltag': -3.56,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.38'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00119[0] <=> (1) cpd00023[0] + (1) cpd02091[0]',\n",
       "  'id': 'rxn00862',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-histidine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); HISTDEG-PWY (L-histidine degradation IV); HISTIDINE-DEG (L-histidine Degradation); HISTIDINE-SYN (L-histidine Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-5029 (imidazole-lactate degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00340 (Histidine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00119:0:0:\"L-Histidine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd02091:0:0:\"Imidazole pyruvate\"'},\n",
       " {'abbreviation': 'R01163',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HISTALDEHYD-RXN',\n",
       "   'BrachyCyc: HISTALDEHYD-RXN',\n",
       "   'ChlamyCyc: HISTALDEHYD-RXN',\n",
       "   'KEGG: R01163',\n",
       "   'MetaCyc: HISTALDEHYD-RXN',\n",
       "   'Name: Histidinal dehydrogenase; L-histidinal:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd01324[0] <=> (1) cpd00004[0] + (1) cpd00119[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00119;cpd01324',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) L-Histidinal[0] <=> (1) NADH[0] + (2) H+[0] + (1) L-Histidine[0]',\n",
       "  'deltag': -6.36,\n",
       "  'deltagerr': 0.47,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.23'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd01324[0] <=> (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00119[0]',\n",
       "  'id': 'rxn00863',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19892',\n",
       "  'name': 'L-histidinal:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); HISTIDINE-SYN (L-histidine Biosynthesis); HISTSYN-PWY (L-histidine biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis)',\n",
       "   'KEGG: rn00340 (Histidine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd01324:0:0:\"L-Histidinal\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd00119:0:0:\"L-Histidine\"'},\n",
       " {'abbreviation': 'R01164',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01164',\n",
       "   'Name: L-Histidine:beta-alanine ligase (AMP-forming); L-histidine:beta-alanine ligase (AMP-forming)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00085[0] + (1) cpd00119[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00310[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00018;cpd00067;cpd00085;cpd00119;cpd00310',\n",
       "  'definition': '(1) ATP[0] + (1) beta-Alanine[0] + (1) L-Histidine[0] => (1) PPi[0] + (1) AMP[0] + (2) H+[0] + (1) Carnosine[0]',\n",
       "  'deltag': -9.6,\n",
       "  'deltagerr': 1.46,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.2.11'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00085[0] + (1) cpd00119[0] => (1) cpd00012[0] + (1) cpd00018[0] + (2) cpd00067[0] + (1) cpd00310[0]',\n",
       "  'id': 'rxn00864',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-histidine:beta-alanine ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00340 (Histidine metabolism); rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00085:0:0:\"beta-Alanine\";-1:cpd00119:0:0:\"L-Histidine\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";2:cpd00067:0:0:\"H+\";1:cpd00310:0:0:\"Carnosine\"'},\n",
       " {'abbreviation': 'R01166',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01166',\n",
       "   'MetaCyc: 3.4.13.20-RXN',\n",
       "   'Name: Nalpha-(beta-alanyl)-L-histidine hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00310[0] <=> (1) cpd00085[0] + (1) cpd00119[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00085;cpd00119;cpd00310',\n",
       "  'definition': '(1) H2O[0] + (1) Carnosine[0] <=> (1) beta-Alanine[0] + (1) L-Histidine[0]',\n",
       "  'deltag': 1.17,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.4.13.18', '3.4.13.20'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00310[0] <=> (1) cpd00085[0] + (1) cpd00119[0]',\n",
       "  'id': 'rxn00865',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Nalpha-(beta-alanyl)-L-histidine hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00340 (Histidine metabolism); rn00410 (beta-Alanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00310:0:0:\"Carnosine\";1:cpd00085:0:0:\"beta-Alanine\";1:cpd00119:0:0:\"L-Histidine\"'},\n",
       " {'abbreviation': 'R01167',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HISTIDINE-DECARBOXYLASE-RXN',\n",
       "   'BiGG: HISDC',\n",
       "   'ChlamyCyc: HISTIDINE-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R01167',\n",
       "   'MetaCyc: HISTIDINE-DECARBOXYLASE-RXN',\n",
       "   'Name: L-Histidine carboxy-lyase; L-histidine carboxy-lyase; L-histidine carboxy-lyase (histamine-forming); L-histidine decarboxylase; histidine decarboxylase'],\n",
       "  'code': '(1) cpd00119[0] <=> (1) cpd00011[0] + (1) cpd00312[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00119;cpd00312',\n",
       "  'definition': '(1) H+[0] + (1) L-Histidine[0] => (1) CO2[0] + (1) Histamine[0]',\n",
       "  'deltag': -6.96,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.22', '4.1.1.28'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00119[0] => (1) cpd00011[0] + (1) cpd00312[0]',\n",
       "  'id': 'rxn00866',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32344;rxn32347',\n",
       "  'name': 'L-histidine carboxy-lyase (histamine-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: PWY-6173 (histamine biosynthesis); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis)',\n",
       "   'KEGG: rn00340 (Histidine metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00119:0:0:\"L-Histidine\";1:cpd00011:0:0:\"CO2\";1:cpd00312:0:0:\"Histamine\"'},\n",
       " {'abbreviation': 'R01168',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: HISD; HISD1; HISD1i; HISD1r; HISDr',\n",
       "   'JP_Creinhardtii_MSB: R_HISD',\n",
       "   'KEGG: R01168',\n",
       "   'MetaCyc: HISTIDINE-AMMONIA-LYASE-RXN',\n",
       "   'iAG612: rbs_403',\n",
       "   'Name: L-Histidine ammonia-lyase; L-histidine ammonia-lyase; L-histidine ammonia-lyase (urocanate-forming); histidase; histidase_r; histidinase; histidine alpha-deaminase; histidine ammonia-lyase'],\n",
       "  'code': '(1) cpd00119[0] <=> (1) cpd00013[0] + (1) cpd00581[0]',\n",
       "  'compound_ids': 'cpd00013;cpd00119;cpd00581',\n",
       "  'definition': '(1) L-Histidine[0] => (1) NH3[0] + (1) Urocanate[0]',\n",
       "  'deltag': 1.12,\n",
       "  'deltagerr': 0.25,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.3.1.3'],\n",
       "  'equation': '(1) cpd00119[0] => (1) cpd00013[0] + (1) cpd00581[0]',\n",
       "  'id': 'rxn00867',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30486',\n",
       "  'name': 'L-histidine ammonia-lyase (urocanate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); HISDEG-PWY (L-histidine degradation I); HISHP-PWY (L-histidine degradation VI); HISTIDINE-DEG (L-histidine Degradation); PWY-5028 (L-histidine degradation II); PWY-5030 (L-histidine degradation III); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00340 (Histidine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00119:0:0:\"L-Histidine\";1:cpd00013:0:0:\"NH3\";1:cpd00581:0:0:\"Urocanate\"'},\n",
       " {'abbreviation': 'R01171',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACOAD1',\n",
       "   'KEGG: R01171',\n",
       "   'MetaCyc: RXN-12558',\n",
       "   'RiceCyc: R106-RXN',\n",
       "   'SorghumCyc: R106-RXN',\n",
       "   'Name: Butanoyl-CoA:(acceptor) 2,3-oxidoreductase; Butanoyl-CoA:NAD+ trans-2-oxidoreductase; acyl-CoA dehydrogenase (butanoyl-CoA); butanoyl-CoA:NAD+ trans-2-oxidoreductase; crotonyl-CoA reductase (NAD+)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00120[0] <=> (1) cpd00004[0] + (1) cpd00650[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00120;cpd00650',\n",
       "  'definition': '(1) NAD[0] + (1) Butyryl-CoA[0] <= (1) NADH[0] + (1) H+[0] + (1) Crotonyl-CoA[0]',\n",
       "  'deltag': 13.11,\n",
       "  'deltagerr': 0.64,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.3.1.44', '1.3.8.1'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00120[0] <= (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00650[0]',\n",
       "  'id': 'rxn00868',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn23000',\n",
       "  'name': 'Butanoyl-CoA:(acceptor) 2,3-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-6863 (pyruvate fermentation to hexanol (engineered)); PWY-6883 (pyruvate fermentation to butanol II (engineered)); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); Pyruvate-Butanol-Fermentation (Pyruvate Fermentation to Butanol); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00310 (Lysine degradation); rn00650 (Butanoate metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00120:0:0:\"Butyryl-CoA\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00650:0:0:\"Crotonyl-CoA\"'},\n",
       " {'abbreviation': 'R01172',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_BNOR; R_BNORh',\n",
       "   'KEGG: R01172',\n",
       "   'MetaCyc: BUTANAL-DEHYDROGENASE-RXN',\n",
       "   'Name: butanal dehydrogenase; butanal:NAD(P)+ oxidoreductase (CoA-acylating); butanal:NAD+ oxidoreductase (CoA-acylating); butyraldehyde:NAD+ oxidoreductase (CoA-acetylating)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd01011[0] <=> (1) cpd00004[0] + (1) cpd00120[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00067;cpd00120;cpd01011',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) Butanal[0] <=> (1) NADH[0] + (1) H+[0] + (1) Butyryl-CoA[0]',\n",
       "  'deltag': -1.93,\n",
       "  'deltagerr': 0.69,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.10', '1.2.1.57', '1.2.1.87'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd01011[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00120[0]',\n",
       "  'id': 'rxn00869',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34627',\n",
       "  'name': 'butanal:NAD+ oxidoreductase (CoA-acylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-6583 (pyruvate fermentation to butanol I); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6883 (pyruvate fermentation to butanol II (engineered)); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7003 (glycerol degradation to butanol); Pyruvate-Butanol-Fermentation (Pyruvate Fermentation to Butanol); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd01011:0:0:\"Butanal\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00120:0:0:\"Butyryl-CoA\"'},\n",
       " {'abbreviation': 'R01173',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01173',\n",
       "   'MetaCyc: BUTANAL-DEHYDROGENASE-RXN',\n",
       "   'Name: butanal dehydrogenase; butanal:NAD(P)+ oxidoreductase (CoA-acylating); butanal:NADP+ oxidoreductase (CoA-acylating)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd01011[0] <=> (1) cpd00005[0] + (1) cpd00120[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00010;cpd00067;cpd00120;cpd01011',\n",
       "  'definition': '(1) NADP[0] + (1) CoA[0] + (1) Butanal[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Butyryl-CoA[0]',\n",
       "  'deltag': -1.93,\n",
       "  'deltagerr': 0.69,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.57'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00010[0] + (1) cpd01011[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00120[0]',\n",
       "  'id': 'rxn00870',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'butanal:NADP+ oxidoreductase (CoA-acylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-6583 (pyruvate fermentation to butanol I); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6883 (pyruvate fermentation to butanol II (engineered)); PWY-6886 (1-butanol autotrophic biosynthesis (engineered)); PWY-7003 (glycerol degradation to butanol); Pyruvate-Butanol-Fermentation (Pyruvate Fermentation to Butanol); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00010:0:0:\"CoA\";-1:cpd01011:0:0:\"Butanal\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00120:0:0:\"Butyryl-CoA\"'},\n",
       " {'abbreviation': 'R01174',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PBUTT',\n",
       "   'KEGG: R01174',\n",
       "   'MetaCyc: PHOSPHATE-BUTYRYLTRANSFERASE-RXN',\n",
       "   'RiceCyc: PHOSPHATE-BUTYRYLTRANSFERASE-RXN',\n",
       "   'SorghumCyc: PHOSPHATE-BUTYRYLTRANSFERASE-RXN',\n",
       "   'Name: Butanoyl-CoA:orthophosphate butanoyltransferase; butanoyl-CoA:phosphate butanoyltransferase; phosphate butyryltransferase; phosphotransbutyrylase'],\n",
       "  'code': '(1) cpd00009[0] + (1) cpd00120[0] <=> (1) cpd00010[0] + (1) cpd01662[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00010;cpd00120;cpd01662',\n",
       "  'definition': '(1) Phosphate[0] + (1) Butyryl-CoA[0] <=> (1) CoA[0] + (1) Butanoylphosphate[0]',\n",
       "  'deltag': 5.21,\n",
       "  'deltagerr': 0.43,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.19'],\n",
       "  'equation': '(1) cpd00009[0] + (1) cpd00120[0] <=> (1) cpd00010[0] + (1) cpd01662[0]',\n",
       "  'id': 'rxn00871',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'butanoyl-CoA:phosphate butanoyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Degradation (Amino Acid Degradation); CENTFERM-PWY (pyruvate fermentation to butanoate); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GLUDEG-II-PWY (L-glutamate degradation VII (to butanoate)); GLUTAMATE-DEG (L-glutamate Degradation); PWY-6590 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00009:0:0:\"Phosphate\";-1:cpd00120:0:0:\"Butyryl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd01662:0:0:\"Butanoylphosphate\"'},\n",
       " {'abbreviation': 'R01175',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACOAD1f',\n",
       "   'JP_Creinhardtii_MSB: R_ACOA40OR',\n",
       "   'KEGG: R01175',\n",
       "   'TS_Athaliana: rxn630_x',\n",
       "   'iAF1260: ACOAD1f',\n",
       "   'Name: Butanoyl-CoA:oxygen 2-oxidoreductase; R_Butanoyl_CoA_oxygen_2_oxidoreductase; acyl-CoA dehydrogenase (butanoyl-CoA); butyryl-coa dehydrogenase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00120[0] <=> (1) cpd00650[0] + (1) cpd00982[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00120;cpd00650;cpd00982',\n",
       "  'definition': '(1) FAD[0] + (1) Butyryl-CoA[0] <=> (1) Crotonyl-CoA[0] + (1) FADH2[0]',\n",
       "  'deltag': 9.5,\n",
       "  'deltagerr': 0.95,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.3.6', '1.3.8.1', '1.3.99.-', '1.3.99.2', '1.3.99.3'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00120[0] <=> (1) cpd00650[0] + (1) cpd00982[0]',\n",
       "  'id': 'rxn00872',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31412',\n",
       "  'name': 'Butanoyl-CoA:oxygen 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00071 (Fatty acid degradation); rn00650 (Butanoate metabolism); rn01200 (Carbon metabolism); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00120:0:0:\"Butyryl-CoA\";1:cpd00650:0:0:\"Crotonyl-CoA\";1:cpd00982:0:0:\"FADH2\"'},\n",
       " {'abbreviation': 'R01176',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01176',\n",
       "   'MetaCyc: RXN-19193',\n",
       "   'SoyCyc: RXN-7904',\n",
       "   'Name: ACS3; Butanoate:CoA ligase (AMP-forming); FAA1; LCFA synthetase; acid:CoA ligase (AMP-forming); acyl coenzyme A synthetase; acyl-CoA ligase; acyl-CoA synthetase; acyl-activating enzyme; acyl-coenzyme A ligase; arachidonyl-CoA synthetase; fatty acid CoA ligase; fatty acid elongase; fatty acid thiokinase (long chain); fatty acyl-coenzyme A synthetase; lignoceroyl-CoA synthase; long chain fatty acyl-CoA synthetase; long-chain acyl CoA synthetase; long-chain acyl-CoA synthetase I; long-chain acyl-CoA synthetase II; long-chain acyl-coenzyme A synthetase; long-chain fatty acyl coenzyme A synthetase; long-chain-fatty-acid-CoA ligase; oleoyl-CoA synthetase; palmitoyl coenzyme A synthetase; palmitoyl-CoA ligase; palmitoyl-CoA synthase; pristanoyl-CoA synthetase; stearoyl-CoA synthetase; thiokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00211[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00120[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00120;cpd00211',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Butyrate[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) Butyryl-CoA[0]',\n",
       "  'deltag': -0.07,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.2', '6.2.1.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00211[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00120[0]',\n",
       "  'id': 'rxn00873',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Butanoate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00211:0:0:\"Butyrate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\";1:cpd00120:0:0:\"Butyryl-CoA\"'},\n",
       " {'abbreviation': 'R01177',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-12565',\n",
       "   'BiGG: ACACT2r; KAT2; POT1_7',\n",
       "   'CornCyc: RXN-12565',\n",
       "   'KEGG: R01177',\n",
       "   'MetaCyc: RXN-12565',\n",
       "   'Name: 3-ketoacyl-CoA thiolase; 3-oxohexanoyl-CoA thiolase; 3oxoacyl-coa thiolase (c6:0); acetyl-CoA C-acyltransferase (butanoyl-CoA); acetyl-CoA C-acyltransferase (butanoyl-CoA) (r); acetyl-CoA C-acyltransferase, putative / 3-ketoacyl-CoA thiolase, putative; butanoyl-CoA:acetyl-CoA C-butanoyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00120[0] <=> (1) cpd00010[0] + (1) cpd03123[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00120;cpd03123',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Butyryl-CoA[0] <=> (1) CoA[0] + (1) 3-Oxohexanoyl-CoA[0]',\n",
       "  'deltag': 7.4,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.3.1.16', '2.3.1.9'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00120[0] <=> (1) cpd00010[0] + (1) cpd03123[0]',\n",
       "  'id': 'rxn00874',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19968;rxn19980',\n",
       "  'name': 'butanoyl-CoA:acetyl-CoA C-butanoyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-6863 (pyruvate fermentation to hexanol (engineered)); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00062 (Fatty acid elongation); rn00071 (Fatty acid degradation); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00120:0:0:\"Butyryl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd03123:0:0:\"3-Oxohexanoyl-CoA\"'},\n",
       " {'abbreviation': 'R01179',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: BUTCT',\n",
       "   'KEGG: R01179',\n",
       "   'MetaCyc: R11-RXN',\n",
       "   'RiceCyc: R11-RXN',\n",
       "   'SorghumCyc: R11-RXN',\n",
       "   'Name: Acetyl-CoA:butyrate-CoA transferase; Butanoyl-CoA:acetate CoA-transferase; acetate coa-transferase; butyryl coenzyme A transferase; butyryl-CoA:acetate CoA-transferase'],\n",
       "  'code': '(1) cpd00029[0] + (1) cpd00120[0] <=> (1) cpd00022[0] + (1) cpd00211[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00029;cpd00120;cpd00211',\n",
       "  'definition': '(1) Acetate[0] + (1) Butyryl-CoA[0] <=> (1) Acetyl-CoA[0] + (1) Butyrate[0]',\n",
       "  'deltag': -1.06,\n",
       "  'deltagerr': 0.79,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.8'],\n",
       "  'equation': '(1) cpd00029[0] + (1) cpd00120[0] <=> (1) cpd00022[0] + (1) cpd00211[0]',\n",
       "  'id': 'rxn00875',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Butanoyl-CoA:acetate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Aminobutyraye-Degradation (4-Aminobutanoate Degradation); AMINE-DEG (Amine and Polyamine Degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GALLATE-DEG (Gallate Degradation); GLUTAMATE-DEG (L-glutamate Degradation); P162-PWY (L-glutamate degradation V (via hydroxyglutarate)); P3-PWY (gallate degradation III (anaerobic)); PWY-5022 (4-aminobutanoate degradation V); PWY-5676 (acetyl-CoA fermentation to butanoate II); PWY-5677 (succinate fermentation to butanoate); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00029:0:0:\"Acetate\";-1:cpd00120:0:0:\"Butyryl-CoA\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00211:0:0:\"Butyrate\"'},\n",
       " {'abbreviation': 'R01180',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01180',\n",
       "   'MetaCyc: 3-ETHYLMALATE-SYNTHASE-RXN',\n",
       "   'Name: 2-ethyl-3-hydroxybutanedioate synthase; 3-Ethylmalate glyoxylate-lyase (CoA-butanoylating); 3-ethylmalate glyoxylate-lyase (CoA-butanoylating); 3-ethylmalate synthase; butanoyl-CoA:glyoxylate C-butanoyltransferase (thioester-hydrolysing, 1-carboxypropyl-forming)'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd01362[0] <=> (1) cpd00001[0] + (1) cpd00040[0] + (1) cpd00120[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00040;cpd00067;cpd00120;cpd01362',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) 3-Ethylmalate[0] <= (1) H2O[0] + (1) Glyoxalate[0] + (1) Butyryl-CoA[0]',\n",
       "  'deltag': 7.94,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.7'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01362[0] <= (1) cpd00001[0] + (1) cpd00040[0] + (1) cpd00120[0]',\n",
       "  'id': 'rxn00876',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'butanoyl-CoA:glyoxylate C-butanoyltransferase (thioester-hydrolysing, 1-carboxypropyl-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); PWY-7396 (butanol and isobutanol biosynthesis (engineered))',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd01362:0:0:\"3-Ethylmalate\";1:cpd00001:0:0:\"H2O\";1:cpd00040:0:0:\"Glyoxalate\";1:cpd00120:0:0:\"Butyryl-CoA\"'},\n",
       " {'abbreviation': 'R01181',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01181',\n",
       "   'MetaCyc: ISOBUTYRYL-COA-MUTASE-RXN',\n",
       "   'Name: 2-Methylpropanoyl-CoA CoA-carbonylmutase; 2-methylpropanoyl-CoA CoA-carbonylmutase; butyryl-CoA:isobutyryl-CoA mutase; isobutyryl coenzyme A mutase; isobutyryl-CoA mutase'],\n",
       "  'code': '(1) cpd00481[0] <=> (1) cpd00120[0]',\n",
       "  'compound_ids': 'cpd00120;cpd00481',\n",
       "  'definition': '(1) Isobutyryl-CoA[0] <=> (1) Butyryl-CoA[0]',\n",
       "  'deltag': -0.53,\n",
       "  'deltagerr': 0.22,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.4.99.13'],\n",
       "  'equation': '(1) cpd00481[0] <=> (1) cpd00120[0]',\n",
       "  'id': 'rxn00877',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-Methylpropanoyl-CoA CoA-carbonylmutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00481:0:0:\"Isobutyryl-CoA\";1:cpd00120:0:0:\"Butyryl-CoA\"'},\n",
       " {'abbreviation': 'R01182',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01182', 'Name: ATP:myo-inositol 1-phosphotransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00121[0] <=> (1) cpd00008[0] + (1) cpd00867[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00121;cpd00867',\n",
       "  'definition': '(1) ATP[0] + (1) L-Inositol[0] => (1) ADP[0] + (1) H+[0] + (1) Inositol 1-phosphate[0]',\n",
       "  'deltag': -0.99,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.64'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00121[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00867[0]',\n",
       "  'id': 'rxn00878',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:myo-inositol 1-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00562 (Inositol phosphate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00867:0:0:\"Inositol 1-phosphate\"'},\n",
       " {'abbreviation': 'R01183',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: INS2D',\n",
       "   'KEGG: R01183',\n",
       "   'MetaCyc: MYO-INOSITOL-2-DEHYDROGENASE-RXN',\n",
       "   'iAF692: INS2D',\n",
       "   'iAG612: rbs_353',\n",
       "   'Name: inositol 2-dehydrogenase; myo-Inositol:NAD+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00121[0] <=> (1) cpd00004[0] + (1) cpd00524[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00121;cpd00524',\n",
       "  'definition': '(1) NAD[0] + (1) L-Inositol[0] <=> (1) NADH[0] + (1) H+[0] + (1) 2-Inosose[0]',\n",
       "  'deltag': -0.61,\n",
       "  'deltagerr': 0.4,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.18'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00121[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00524[0]',\n",
       "  'id': 'rxn00879',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'myo-Inositol:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Antibiotic-Biosynthesis (Antibiotic Biosynthesis); Degradation (Degradation/Utilization/Assimilation); P562-PWY (<I>myo</I>-inositol degradation I); PWY-5940 (streptomycin biosynthesis); PWY-7237 (<i>myo</i>-, <i>chiro</i>- and <i>scyllo</i>-inositol degradation); PWY-7241 (<I>myo</I>-inositol degradation II); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation)',\n",
       "   'KEGG: rn00521 (Streptomycin biosynthesis); rn00562 (Inositol phosphate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00524:0:0:\"2-Inosose\"'},\n",
       " {'abbreviation': 'R01184',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01184_c',\n",
       "   'AraGEM: R_R01184_c',\n",
       "   'JP_Creinhardtii_MSB: R_MIOOR',\n",
       "   'KEGG: R01184',\n",
       "   'MetaCyc: MYO-INOSITOL-OXYGENASE-RXN',\n",
       "   'Name: Myo-inositol oxygenase; myo-Inositol:oxygen oxidoreductase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00121[0] <=> (1) cpd00001[0] + (1) cpd00164[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00067;cpd00121;cpd00164',\n",
       "  'definition': '(1) O2[0] + (1) L-Inositol[0] => (1) H2O[0] + (1) H+[0] + (1) Glucuronate[0]',\n",
       "  'deltag': -95.68,\n",
       "  'deltagerr': 1.96,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.99.1'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00121[0] => (1) cpd00001[0] + (1) cpd00067[0] + (1) cpd00164[0]',\n",
       "  'id': 'rxn00880',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33148;rxn33155',\n",
       "  'name': 'myo-Inositol:oxygen oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CYCLITOLS-DEG (Cyclitol Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); PWY-4841 (UDP-&alpha;-D-glucuronate biosynthesis (from <i>myo</i>-inositol)); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00053 (Ascorbate and aldarate metabolism); rn00562 (Inositol phosphate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00001:0:0:\"H2O\";1:cpd00067:0:0:\"H+\";1:cpd00164:0:0:\"Glucuronate\"'},\n",
       " {'abbreviation': 'R01185',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-10949; RXN0-5408',\n",
       "   'BiGG: MI1PP',\n",
       "   'BrachyCyc: RXN-7252; RXN0-5408',\n",
       "   'KEGG: R01185',\n",
       "   'MetaCyc: RXN-10949; RXN0-5408',\n",
       "   'Name: 1D-myo-Inositol 1-phosphate phosphohydrolase; Myo-inositol-1(or 4)-monophosphatase; Myo-inositol-1-phosphatase; myo-Inositol 1-phosphate phosphahydrolase; myo-inositol 1-phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00867[0] <=> (1) cpd00009[0] + (1) cpd00121[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00121;cpd00867',\n",
       "  'definition': '(1) H2O[0] + (1) Inositol 1-phosphate[0] <=> (1) Phosphate[0] + (1) L-Inositol[0]',\n",
       "  'deltag': -5.17,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.25'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00867[0] <=> (1) cpd00009[0] + (1) cpd00121[0]',\n",
       "  'id': 'rxn00881',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28168;rxn30897',\n",
       "  'name': '1D-myo-Inositol 1-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CYCLITOLS-DEG (Cyclitol Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-4702 (phytate degradation I); Phytate-Degradation (Phytate Degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); Sugar-Derivatives (Sugar Derivative Degradation)',\n",
       "   'KEGG: rn00562 (Inositol phosphate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00867:0:0:\"Inositol 1-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00121:0:0:\"L-Inositol\"'},\n",
       " {'abbreviation': 'R01186',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-10949; RXN-10952',\n",
       "   'BiGG: MI4PP',\n",
       "   'CornCyc: RXN-10949; RXN-10952',\n",
       "   'KEGG: R01186',\n",
       "   'MetaCyc: RXN-10949; RXN-10952',\n",
       "   'Name: 1D-myo-Inositol 4-phosphate phosphohydrolase; myo-Inositol 4-phosphate phosphahydrolase; myo-inositol 4-phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02230[0] <=> (1) cpd00009[0] + (1) cpd00121[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00121;cpd02230',\n",
       "  'definition': '(1) H2O[0] + (1) Inositol 4-phosphate[0] <=> (1) Phosphate[0] + (1) L-Inositol[0]',\n",
       "  'deltag': -5.17,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.25'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02230[0] <=> (1) cpd00009[0] + (1) cpd00121[0]',\n",
       "  'id': 'rxn00882',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28169',\n",
       "  'name': '1D-myo-Inositol 4-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CYCLITOLS-DEG (Cyclitol Degradation); Cyclitols-Biosynthesis (Cyclitol Biosynthesis); Degradation (Degradation/Utilization/Assimilation); PWY-6358 (superpathway of D-<i>myo</i>-inositol (1,4,5)-trisphosphate metabolism); PWY-6363 (D-<i>myo</i>-inositol (1,4,5)-trisphosphate degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-DERIVS (Sugar Derivative Biosynthesis); Sugar-Alcohols-Biosynthesis (Sugar Alcohol Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation)',\n",
       "   'KEGG: rn00562 (Inositol phosphate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02230:0:0:\"Inositol 4-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00121:0:0:\"L-Inositol\"'},\n",
       " {'abbreviation': 'R01187',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: MYO-INOSITOL-1OR-4-MONOPHOSPHATASE-RXN; RXN-10949',\n",
       "   'BiGG: INM1; MI3PP; YDR287W',\n",
       "   'BrachyCyc: MYO-INOSITOL-1OR-4-MONOPHOSPHATASE-RXN',\n",
       "   'KEGG: R01187',\n",
       "   'MetaCyc: MYO-INOSITOL-1OR-4-MONOPHOSPHATASE-RXN; RXN-10949',\n",
       "   'Name: 1D-myo-Inositol 3-phosphate phosphahydrolase; 1D-myo-Inositol 3-phosphate phosphohydrolase; Inositol monophosphatase 1; Myo-inositol-1(or 4)-monophosphatase; Myo-inositol-1-phosphatase; myo-inositol 3-phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02484[0] <=> (1) cpd00009[0] + (1) cpd00121[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00121;cpd02484',\n",
       "  'definition': '(1) H2O[0] + (1) Inositol 3-phosphate[0] <=> (1) Phosphate[0] + (1) L-Inositol[0]',\n",
       "  'deltag': -5.17,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.25'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02484[0] <=> (1) cpd00009[0] + (1) cpd00121[0]',\n",
       "  'id': 'rxn00883',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28001',\n",
       "  'name': '1D-myo-Inositol 3-phosphate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cyclitols-Biosynthesis (Cyclitol Biosynthesis); PWY-2301 (<i>myo</i>-inositol biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SUGAR-DERIVS (Sugar Derivative Biosynthesis); Sugar-Alcohols-Biosynthesis (Sugar Alcohol Biosynthesis)',\n",
       "   'KEGG: rn00521 (Streptomycin biosynthesis); rn00562 (Inositol phosphate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02484:0:0:\"Inositol 3-phosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00121:0:0:\"L-Inositol\"'},\n",
       " {'abbreviation': 'R01188',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01188',\n",
       "   'MetaCyc: MYO-INOSITOL-3-O-METHYLTRANSFERASE-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine:myo-inositol 1-O-methyltransferase; S-adenosyl-L-methionine:1D-myo-inositol 1-O-methyltransferase; S-adenosyl-L-methionine:myo-inositol 3-O-methyltransferase; S-adenosylmethionine:myo-inositol 3-methyltransferase; inositol 1-methyltransferase; inositol 3-O-methyltransferase (name based on 1L-numbering system and not 1D-numbering); inositol D-1-methyltransferase; myo-inositol 3-O-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd02296[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00121;cpd02296',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) L-Inositol[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) 1-O-Methyl-myo-inositol[0]',\n",
       "  'deltag': -4.36,\n",
       "  'deltagerr': 5.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.40'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd02296[0]',\n",
       "  'id': 'rxn00884',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-adenosyl-L-methionine:1D-myo-inositol 1-O-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00562 (Inositol phosphate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd02296:0:0:\"1-O-Methyl-myo-inositol\"'},\n",
       " {'abbreviation': 'R01189',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01189',\n",
       "   'MetaCyc: MYO-INOSITOL-1-O-METHYLTRANSFERASE-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine:myo-inositol 3-O-methyltransferase; S-adenosyl-L-methionine:1D-myo-inositol 3-O-methyltransferase; S-adenosyl-L-methionine:myo-inositol 1-O-methyltransferase; S-adenosylmethionine:myo-inositol 1-methyltransferase; inositol 3-methyltransferase; inositol L-1-methyltransferase; myo-inositol 1-O-methyltransferase (name based on 1L-numbering system and not 1D-numbering); myo-inositol 1-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd02297[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00121;cpd02297',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) L-Inositol[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) 3-O-Methyl-myo-inositol[0]',\n",
       "  'deltag': -4.36,\n",
       "  'deltagerr': 5.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.39'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd02297[0]',\n",
       "  'id': 'rxn00885',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-adenosyl-L-methionine:1D-myo-inositol 3-O-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00562 (Inositol phosphate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd02297:0:0:\"3-O-Methyl-myo-inositol\"'},\n",
       " {'abbreviation': 'R01190',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01190',\n",
       "   'MetaCyc: 2.1.1.129-RXN',\n",
       "   'PlantCyc: 2.1.1.129-RXN',\n",
       "   'Name: S-Adenosyl-L-methionine:myo-inositol 4-O-methyltransferase; S-adenosyl-L-methionine:1D-myo-inositol 4-methyltransferase; S-adenosyl-L-methionine:myo-inositol 4-O-methyltransferase; inositol 4-methyltransferase; myo-inositol 4-O-methyltransferase; myo-inositol 6-O-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd03793[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00121;cpd03793',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) L-Inositol[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) 4-O-Methyl-myo-inositol[0]',\n",
       "  'deltag': -4.36,\n",
       "  'deltagerr': 5.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.129'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd03793[0]',\n",
       "  'id': 'rxn00886',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-adenosyl-L-methionine:1D-myo-inositol 4-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['MetaCyc: Cyclitols-Biosynthesis (Cyclitol Biosynthesis); PWY-6738 (pinitol biosynthesis I); Pinitol-Biosynthesis (Pinitol Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); SUGAR-DERIVS (Sugar Derivative Biosynthesis); Sugar-Alcohols-Biosynthesis (Sugar Alcohol Biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd03793:0:0:\"4-O-Methyl-myo-inositol\"'},\n",
       " {'abbreviation': 'myo-Inositol 6-O-methyltransferase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: myo-Inositol 6-O-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd03794[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00121;cpd03794',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) L-Inositol[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) 6-O-Methyl-myo-inositol[0]',\n",
       "  'deltag': -4.36,\n",
       "  'deltagerr': 5.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.129'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00121[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd03794[0]',\n",
       "  'id': 'rxn00887',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'myo-Inositol 6-O-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd03794:0:0:\"6-O-Methyl-myo-inositol\"'},\n",
       " {'abbreviation': 'R01192',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.4.1.123-RXN',\n",
       "   'BrachyCyc: 2.4.1.123-RXN',\n",
       "   'CornCyc: 2.4.1.123-RXN',\n",
       "   'KEGG: R01192; R06214',\n",
       "   'MetaCyc: 2.4.1.123-RXN',\n",
       "   'Name: GolS; Inositol 1-alpha-galactosyltransferase; UDP-D-galactose:inositol galactosyltransferase; UDP-alpha-D-galactose:myo-inositol 3-alpha-D-galactosyltransferase; UDP-galactose:myo-inositol 1-alpha-D-galactosyltransferase; UDP-galactose:myo-inositol 3-alpha-D-galactosyltransferase; UDPgalactose:myo-inositol 1-alpha-D-galactosyltransferase; galactinol synthase; inositol 1-alpha-galactosyltransferase; inositol 3-alpha-galactosyltransferase; uridine diphosphogalactose-inositol galactosyltransferase'],\n",
       "  'code': '(1) cpd00043[0] + (1) cpd00121[0] <=> (1) cpd00014[0] + (1) cpd00910[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00043;cpd00121;cpd00910',\n",
       "  'definition': '(1) UDP-galactose[0] + (1) L-Inositol[0] <=> (1) UDP[0] + (1) Galactinol[0]',\n",
       "  'deltag': -4.53,\n",
       "  'deltagerr': 0.61,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.123'],\n",
       "  'equation': '(1) cpd00043[0] + (1) cpd00121[0] <=> (1) cpd00014[0] + (1) cpd00910[0]',\n",
       "  'id': 'rxn00888',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33194;rxn33195;rxn33199',\n",
       "  'name': 'UDP-galactose:myo-inositol 3-alpha-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Oligosaccharides-Biosynthesis (Oligosaccharide Biosynthesis); PWY-5337 (stachyose biosynthesis); PWY-6524 (lychnose and isolychnose biosynthesis); PWY-6525 (stellariose and mediose biosynthesis)',\n",
       "   'KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00043:0:0:\"UDP-galactose\";-1:cpd00121:0:0:\"L-Inositol\";1:cpd00014:0:0:\"UDP\";1:cpd00910:0:0:\"Galactinol\"'},\n",
       " {'abbreviation': 'R01193',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GPDDA5; GPDDA5pp',\n",
       "   'KEGG: R01193',\n",
       "   'MetaCyc: 3.1.4.44-RXN',\n",
       "   'iAF1260: GPDDA5; GPDDA5pp',\n",
       "   'iJR904: GPDDA5',\n",
       "   'Name: 1-(sn-glycero-3-phospho)-1D-myo-inositol glycerophosphohydrolase; Glycerophosphodiester phosphodiesterase (Glycerophosphoinositol); glycerophosphoinositol glycerophosphodiesterase; sn-Glycero-3-phospho-1-inositol glycerophosphohydrolase; sn-glycero(3)phosphoinositol glycerophosphohydrolase; sn-glycero-3-phospho-1-inositol glycerophosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00902[0] <=> (1) cpd00080[0] + (1) cpd00121[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00080;cpd00121;cpd00902',\n",
       "  'definition': '(1) H2O[0] + (1) sn-glycero-3-Phospho-1-inositol[0] => (1) H+[0] + (1) Glycerol-3-phosphate[0] + (1) L-Inositol[0]',\n",
       "  'deltag': -14.34,\n",
       "  'deltagerr': 2.31,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.4.44', '3.1.4.46'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00902[0] => (1) cpd00067[0] + (1) cpd00080[0] + (1) cpd00121[0]',\n",
       "  'id': 'rxn00889',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29960',\n",
       "  'name': '1-(sn-glycero-3-phospho)-1D-myo-inositol glycerophosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00902:0:0:\"sn-glycero-3-Phospho-1-inositol\";1:cpd00067:0:0:\"H+\";1:cpd00080:0:0:\"Glycerol-3-phosphate\";1:cpd00121:0:0:\"L-Inositol\"'},\n",
       " {'abbreviation': 'R01194',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: GALIGH; YBR184W_4',\n",
       "   'JP_Creinhardtii_MSB: R_OAGMIG',\n",
       "   'KEGG: R01194; R06093',\n",
       "   'iIN800: YBR184W_4',\n",
       "   'iMM904: GALIGH',\n",
       "   'Name: 1-alpha-D-Galactosyl-myo-inositol galactohydrolase; 3-O-alpha-D-Galactosyl-1D-myo-inositol galactohydrolase; Alpha-galactosidase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00910[0] <=> (1) cpd00108[0] + (1) cpd00121[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00108;cpd00121;cpd00910',\n",
       "  'definition': '(1) H2O[0] + (1) Galactinol[0] => (1) Galactose[0] + (1) L-Inositol[0]',\n",
       "  'deltag': -2.38,\n",
       "  'deltagerr': 0.49,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00910[0] => (1) cpd00108[0] + (1) cpd00121[0]',\n",
       "  'id': 'rxn00890',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34779',\n",
       "  'name': '3-O-alpha-D-Galactosyl-1D-myo-inositol galactohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00910:0:0:\"Galactinol\";1:cpd00108:0:0:\"Galactose\";1:cpd00121:0:0:\"L-Inositol\"'},\n",
       " {'abbreviation': 'R01200',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01200',\n",
       "   'MetaCyc: N-ACETYLGLUCOSAMINE-DEACETYLASE-RXN',\n",
       "   'Name: N-Acetyl-D-glucosamine amidohydrolase; N-acetyl-D-glucosamine amidohydrolase; N-acetyl-D-glucosamine deacetylase; N-acetyl-D-glucosaminyl N-deacetylase; acetylaminodeoxyglucose acetylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00122[0] <=> (1) cpd00029[0] + (1) cpd00276[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00122;cpd00276',\n",
       "  'definition': '(1) H2O[0] + (1) N-Acetyl-D-glucosamine[0] <=> (1) Acetate[0] + (1) GLUM[0]',\n",
       "  'deltag': -2.14,\n",
       "  'deltagerr': 0.6,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.5.1.33'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00122[0] <=> (1) cpd00029[0] + (1) cpd00276[0]',\n",
       "  'id': 'rxn00891',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20227',\n",
       "  'name': 'N-Acetyl-D-glucosamine amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00029:0:0:\"Acetate\";1:cpd00276:0:0:\"GLUM\"'},\n",
       " {'abbreviation': 'R01201',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACGAMK',\n",
       "   'KEGG: R01201',\n",
       "   'MetaCyc: N-ACETYLGLUCOSAMINE-KINASE-RXN',\n",
       "   'iAF1260: ACGAMK',\n",
       "   'iMA945: ACGAMK',\n",
       "   'Name: ATP:N-acetyl-D-glucosamine 6-phosphotransferase; N-acetylglucosamine kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00122[0] <=> (1) cpd00008[0] + (1) cpd00293[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00122;cpd00293',\n",
       "  'definition': '(1) ATP[0] + (1) N-Acetyl-D-glucosamine[0] => (1) ADP[0] + (1) H+[0] + (1) N-Acetyl-D-glucosamine 6-phosphate[0]',\n",
       "  'deltag': -2.34,\n",
       "  'deltagerr': 0.63,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.59'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00122[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00293[0]',\n",
       "  'id': 'rxn00892',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:N-acetyl-D-glucosamine 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00293:0:0:\"N-Acetyl-D-glucosamine 6-phosphate\"'},\n",
       " {'abbreviation': 'R01202',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01202',\n",
       "   'Name: N-Acetyl-D-glucosamine:NAD+ 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00122[0] <=> (1) cpd00004[0] + (1) cpd00833[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00122;cpd00833',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) N-Acetyl-D-glucosamine[0] <=> (1) NADH[0] + (2) H+[0] + (1) N-Acetyl-D-glucosaminate[0]',\n",
       "  'deltag': -8.11,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.240'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00122[0] <=> (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00833[0]',\n",
       "  'id': 'rxn00893',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-Acetyl-D-glucosamine:NAD+ 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd00833:0:0:\"N-Acetyl-D-glucosaminate\"'},\n",
       " {'abbreviation': 'R01203',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01203',\n",
       "   'Name: N-Acyl-D-hexosamine:oxygen 1-oxidoreductase; N-acetyl-D-glucosamine:oxygen 1-oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00122[0] <=> (1) cpd00025[0] + (1) cpd00833[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00007;cpd00025;cpd00067;cpd00122;cpd00833',\n",
       "  'definition': '(1) H2O[0] + (1) O2[0] + (1) N-Acetyl-D-glucosamine[0] => (1) H2O2[0] + (1) H+[0] + (1) N-Acetyl-D-glucosaminate[0]',\n",
       "  'deltag': -39.83,\n",
       "  'deltagerr': 1.11,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.3.29'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00007[0] + (1) cpd00122[0] => (1) cpd00025[0] + (1) cpd00067[0] + (1) cpd00833[0]',\n",
       "  'id': 'rxn00894',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-acetyl-D-glucosamine:oxygen 1-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00007:0:0:\"O2\";-1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00025:0:0:\"H2O2\";1:cpd00067:0:0:\"H+\";1:cpd00833:0:0:\"N-Acetyl-D-glucosaminate\"'},\n",
       " {'abbreviation': 'R01204',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01204',\n",
       "   'MetaCyc: GLUCOSAMINE-N-ACETYLTRANSFERASE-RXN',\n",
       "   'Name: Acetyl-CoA:D-glucosamine N-acetyltransferase; acetyl-CoA:D-glucosamine N-acetyltransferase; glucosamine N-acetyltransferase; glucosamine acetylase; glucosamine acetyltransferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00276[0] <=> (1) cpd00010[0] + (1) cpd00122[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00022;cpd00067;cpd00122;cpd00276',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) GLUM[0] => (1) CoA[0] + (1) H+[0] + (1) N-Acetyl-D-glucosamine[0]',\n",
       "  'deltag': -5.17,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.3'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00276[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00122[0]',\n",
       "  'id': 'rxn00895',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19801',\n",
       "  'name': 'Acetyl-CoA:D-glucosamine N-acetyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00276:0:0:\"GLUM\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\"'},\n",
       " {'abbreviation': 'R01205',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01205; R06055',\n",
       "   'Name: UDP-alpha-D-galactose:N-acetyl-D-glucosamine 4-beta-D-galactosyltransferase; UDP-galactose:N-acetyl-D-glucosamine 4-beta-D-galactosyltransferase; UDPgalactose:N-acetyl-D-glucosamine 4-beta-D-galactosyltransferase'],\n",
       "  'code': '(1) cpd00043[0] + (1) cpd00122[0] <=> (1) cpd00014[0] + (1) cpd00469[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00043;cpd00122;cpd00469',\n",
       "  'definition': '(1) UDP-galactose[0] + (1) N-Acetyl-D-glucosamine[0] <=> (1) UDP[0] + (1) N-Acetyllactosamine[0]',\n",
       "  'deltag': -3.59,\n",
       "  'deltagerr': 0.68,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.90'],\n",
       "  'equation': '(1) cpd00043[0] + (1) cpd00122[0] <=> (1) cpd00014[0] + (1) cpd00469[0]',\n",
       "  'id': 'rxn00896',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-galactose:N-acetyl-D-glucosamine 4-beta-D-galactosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00043:0:0:\"UDP-galactose\";-1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00014:0:0:\"UDP\";1:cpd00469:0:0:\"N-Acetyllactosamine\"'},\n",
       " {'abbreviation': 'R01207',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACGAMK',\n",
       "   'KEGG: R01201; R01207',\n",
       "   'MetaCyc: N-ACETYLGLUCOSAMINE-KINASE-RXN; RXN-16006',\n",
       "   'iAF1260: ACGAMK',\n",
       "   'iMA945: ACGAMK',\n",
       "   'Name: ATP:N-acetyl-D-glucosamine 6-phosphotransferase; N-acetyl-D-glucosamine 2-epimerase; N-acetylglucosamine kinase'],\n",
       "  'code': '(1) cpd00122[0] <=> (1) cpd00492[0]',\n",
       "  'compound_ids': 'cpd00122;cpd00492',\n",
       "  'definition': '(1) N-Acetyl-D-glucosamine[0] <=> (1) N-Acetyl-D-mannosamine[0]',\n",
       "  'deltag': 0.0,\n",
       "  'deltagerr': 2.31,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.1.59', '5.1.3.8'],\n",
       "  'equation': '(1) cpd00122[0] <=> (1) cpd00492[0]',\n",
       "  'id': 'rxn00897',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-acetyl-D-glucosamine 2-epimerase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00492:0:0:\"N-Acetyl-D-mannosamine\"'},\n",
       " {'abbreviation': 'R01209',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01209_c',\n",
       "   'AraGEM: R_R01209_c',\n",
       "   'KEGG: R01209',\n",
       "   'Maize_C4GEM: R_R01209_c',\n",
       "   'TS_Athaliana: rxn262_p; rxn634_p',\n",
       "   'Name: 2,3-Dihydroxy-3-methylbutanoate hydro-lyase'],\n",
       "  'code': '(1) cpd02498[0] <=> (1) cpd00001[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00123;cpd02498',\n",
       "  'definition': '(1) 2,3-Dihydroxy-isovalerate[0] => (1) H2O[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': -8.44,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.9'],\n",
       "  'equation': '(1) cpd02498[0] => (1) cpd00001[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00898',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32553',\n",
       "  'name': '2,3-Dihydroxy-3-methylbutanoate hydro-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02498:0:0:\"2,3-Dihydroxy-isovalerate\";1:cpd00001:0:0:\"H2O\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01210',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 1.2.1.25-RXN',\n",
       "   'BiGG: 2LDA; OIVD2',\n",
       "   'BrachyCyc: 1.2.1.25-RXN',\n",
       "   'KEGG: R01210',\n",
       "   'MetaCyc: 1.2.1.25-RXN',\n",
       "   'Name: (CoA-mehtylpropanoylating); 2-oxoisovalerate dehydrogenase; 2-oxoisovalerate dehydrogenase (acylating 3-methyl-2-oxobutanoate); 2-oxoisovalerate dehydrogenase (acylating); 2-oxoisovalerate dehydrogenase (acylating- 3-methyl-2-oxobutanoate); 3-Methyl-2-oxobutanoate:NAD+ oxidoreductase; 3-methyl-2-oxobutanoate:NAD+ 2-oxidoreductase (CoA-methyl-propanoylating); 3-methyl-2-oxobutanoate:NAD+ oxidoreductase (CoA-mehtylpropanoylating); alpha-ketoisovalerate dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00123[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00481[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00011;cpd00123;cpd00481',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) 3-Methyl-2-oxobutanoate[0] => (1) NADH[0] + (1) CO2[0] + (1) Isobutyryl-CoA[0]',\n",
       "  'deltag': -5.64,\n",
       "  'deltagerr': 0.81,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.2.1.25'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00123[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00481[0]',\n",
       "  'id': 'rxn00899',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27481',\n",
       "  'name': '3-methyl-2-oxobutanoate:NAD+ oxidoreductase (CoA-mehtylpropanoylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); VALDEG-PWY (L-valine degradation I); VALINE-DEG (L-valine Degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00481:0:0:\"Isobutyryl-CoA\"'},\n",
       " {'abbreviation': 'R01211',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01211',\n",
       "   'MetaCyc: DIMETHYLMALATE-DEHYDROGENASE-RXN',\n",
       "   'Name: (R)-3,3-Dimethylmalate:NAD+ oxidoreductase (decarboxylating); (R)-3,3-dimethylmalate:NAD+ oxidoreductase (decarboxylating); beta,beta-dimethylmalate dehydrogenase; dimethylmalate dehydrogenase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00796[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00123;cpd00796',\n",
       "  'definition': '(1) NAD[0] + (1) (R)-3,3-Dimethylmalate[0] <=> (1) NADH[0] + (1) CO2[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': -0.07,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.84'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00796[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00900',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(R)-3,3-Dimethylmalate:NAD+ oxidoreductase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00770 (Pantothenate and CoA biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00796:0:0:\"(R)-3,3-Dimethylmalate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01212',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01212',\n",
       "   'MetaCyc: VALINE-DEHYDROGENASE-NADP+-RXN',\n",
       "   'Name: L-valine:NADP+ oxidoreductase (deaminating); valine dehydrogenase (NADP); valine dehydrogenase (NADP+); valine dehydrogenase (nicotinanide adenine dinucleotide phosphate)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00156[0] <=> (1) cpd00005[0] + (1) cpd00013[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00013;cpd00067;cpd00123;cpd00156',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) L-Valine[0] <=> (1) NADPH[0] + (1) NH3[0] + (1) H+[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': 9.23,\n",
       "  'deltagerr': 0.84,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.4.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00156[0] <=> (1) cpd00005[0] + (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00901',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-valine:NADP+ oxidoreductase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00156:0:0:\"L-Valine\";1:cpd00005:0:0:\"NADPH\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01213',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2-ISOPROPYLMALATESYN-RXN',\n",
       "   'BiGG: IPPS; IPPSm; LEU4',\n",
       "   'BrachyCyc: 2-ISOPROPYLMALATESYN-RXN',\n",
       "   'KEGG: R01213',\n",
       "   'MetaCyc: 2-ISOPROPYLMALATESYN-RXN',\n",
       "   'Name: (CoA-acetylating); 2-isopropylmalate synthase; 3-Carboxy-3-hydroxy-4-methylpentanoate 3-methyl-2-oxobutanoate-lyase; 3-Carboxy-3-hydroxy-4-methylpentanoate 3-methyl-2-oxobutanoate-lyase (CoA-acetylating); 3-carboxy-3-hydroxy-4-methylpentanoate 3-methyl-2-oxobutanoate-lyase (CoA-acetylating); Alpha-IPM synthetase; Alpha-isopropylmalate synthase; acetyl-CoA:3-methyl-2-oxobutanoate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming); alpha-IPM synthase; alpha-isopropylmalate synthase; alpha-isopropylmalate synthetase; alpha-isopropylmalic synthetase; isopropylmalate synthase; isopropylmalate synthetase'],\n",
       "  'code': '(1) cpd00010[0] + (1) cpd01646[0] <=> (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00022;cpd00067;cpd00123;cpd01646',\n",
       "  'definition': '(1) CoA[0] + (1) H+[0] + (1) 2-Isopropylmalate[0] <= (1) H2O[0] + (1) Acetyl-CoA[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': 5.96,\n",
       "  'deltagerr': 0.32,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.3.3.13', '4.1.3.12'],\n",
       "  'equation': '(1) cpd00010[0] + (1) cpd00067[0] + (1) cpd01646[0] <= (1) cpd00001[0] + (1) cpd00022[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00902',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27497;rxn30151',\n",
       "  'name': 'acetyl-CoA:3-methyl-2-oxobutanoate C-acetyltransferase (thioester-hydrolysing, carboxymethyl-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); BRANCHED-CHAIN-AA-SYN-PWY (superpathway of branched chain amino acid biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); LEUCINE-SYN (L-leucine Biosynthesis); LEUSYN-PWY (L-leucine biosynthesis); PWY-6871 (3-methylbutanol biosynthesis (engineered))',\n",
       "   'KEGG: rn00290 (Valine, leucine and isoleucine biosynthesis); rn00620 (Pyruvate metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00010:0:0:\"CoA\";-1:cpd00067:0:0:\"H+\";-1:cpd01646:0:0:\"2-Isopropylmalate\";1:cpd00001:0:0:\"H2O\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01214',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: BRANCHED-CHAINAMINOTRANSFERVAL-RXN',\n",
       "   'BiGG: BAT2_2; VALTA; VALTAim',\n",
       "   'BrachyCyc: BRANCHED-CHAINAMINOTRANSFERVAL-RXN',\n",
       "   'KEGG: R01214',\n",
       "   'MetaCyc: BRANCHED-CHAINAMINOTRANSFERVAL-RXN',\n",
       "   'Name: L-Valine:2-oxoglutarate aminotransferase; Transaminase B; Valine transaminase; valine aminotransferase; valine transaminase; valinetransaminase'],\n",
       "  'code': '(1) cpd00024[0] + (1) cpd00156[0] <=> (1) cpd00023[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00023;cpd00024;cpd00123;cpd00156',\n",
       "  'definition': '(1) 2-Oxoglutarate[0] + (1) L-Valine[0] <=> (1) L-Glutamate[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': 0.51,\n",
       "  'deltagerr': 0.84,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.42', '2.6.1.6'],\n",
       "  'equation': '(1) cpd00024[0] + (1) cpd00156[0] <=> (1) cpd00023[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00903',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19242;rxn27739;rxn38055',\n",
       "  'name': 'L-Valine:2-oxoglutarate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALANINE-SYN (L-alanine Biosynthesis); ALANINE-VALINESYN-PWY (L-alanine biosynthesis I); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); BRANCHED-CHAIN-AA-SYN-PWY (superpathway of branched chain amino acid biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-5057 (L-valine degradation II); PWY0-1061 (superpathway of L-alanine biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); VALDEG-PWY (L-valine degradation I); VALINE-BIOSYNTHESIS (L-valine Biosynthesis); VALINE-DEG (L-valine Degradation); VALSYN-PWY (L-valine biosynthesis)',\n",
       "   'KEGG: rn00280 (Valine, leucine and isoleucine degradation); rn00290 (Valine, leucine and isoleucine biosynthesis); rn00770 (Pantothenate and CoA biosynthesis); rn00966 (Glucosinolate biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00156:0:0:\"L-Valine\";1:cpd00023:0:0:\"L-Glutamate\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01215',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: VPAMT',\n",
       "   'ChlamyCyc: VALINE-PYRUVATE-AMINOTRANSFER-RXN',\n",
       "   'EcoCyc: VALINE-PYRUVATE-AMINOTRANSFER-RXN',\n",
       "   'KEGG: R01215',\n",
       "   'MetaCyc: VALINE-PYRUVATE-AMINOTRANSFER-RXN',\n",
       "   'Name: Alanine--valine transaminase; L-Valine:pyruvate aminotransferase; L-valine:pyruvate aminotransferase; Transaminase C; Valine--pyruvate aminotransferase; Valine-pyruvate aminotransferase; alanine-oxoisovalerate aminotransferase; transaminase C; valine-pyruvate aminotransferase; valine-pyruvate transaminase'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00156[0] <=> (1) cpd00035[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00035;cpd00123;cpd00156',\n",
       "  'definition': '(1) Pyruvate[0] + (1) L-Valine[0] <=> (1) L-Alanine[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': 0.44,\n",
       "  'deltagerr': 0.85,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.66'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00156[0] <=> (1) cpd00035[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00904',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29946',\n",
       "  'name': 'L-Valine:pyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALANINE-SYN (L-alanine Biosynthesis); ALANINE-VALINESYN-PWY (L-alanine biosynthesis I); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY0-1061 (superpathway of L-alanine biosynthesis)',\n",
       "   'KEGG: rn00290 (Valine, leucine and isoleucine biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00156:0:0:\"L-Valine\";1:cpd00035:0:0:\"L-Alanine\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01216',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_2DHPFALDL',\n",
       "   'KEGG: R01216',\n",
       "   'MetaCyc: KETOPANTOALDOLASE-RXN',\n",
       "   'Name: 2-Hydroxy-2-isopropylbutanedioate formaldehyde-lyase; 2-dehydropantoate aldolase; 2-dehydropantoate formaldehyde-lyase; 2-dehydropantoate formaldehyde-lyase (3-methyl-2-oxobutanoate-forming); ketopantoaldolase'],\n",
       "  'code': '(1) cpd00712[0] <=> (1) cpd00055[0] + (1) cpd00123[0]',\n",
       "  'compound_ids': 'cpd00055;cpd00123;cpd00712',\n",
       "  'definition': '(1) 2-Dehydropantoate[0] <=> (1) Formaldehyde[0] + (1) 3-Methyl-2-oxobutanoate[0]',\n",
       "  'deltag': 4.5,\n",
       "  'deltagerr': 1.22,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.2.11', '4.1.2.12'],\n",
       "  'equation': '(1) cpd00712[0] <=> (1) cpd00055[0] + (1) cpd00123[0]',\n",
       "  'id': 'rxn00905',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-dehydropantoate formaldehyde-lyase (3-methyl-2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00712:0:0:\"2-Dehydropantoate\";1:cpd00055:0:0:\"Formaldehyde\";1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\"'},\n",
       " {'abbreviation': 'R01218',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MTD1; MTHFD2; MTHFD2i',\n",
       "   'BrachyCyc: 1.5.1.15-RXN',\n",
       "   'ChlamyCyc: 1.5.1.15-RXN',\n",
       "   'KEGG: R01218',\n",
       "   'MetaCyc: 1.5.1.15-RXN',\n",
       "   'Name: 5,10-methylenetetrahydrofolate:NAD+ oxidoreductase; Methylenetetrahydrofolate dehydrogenase (NAD+); Methylenetetrahydrofolate dehydrogenase [NAD+]; methylenetetrahydrofolate dehydrogenase (NAD); methylenetetrahydrofolate dehydrogenase (NAD+)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00125[0] <=> (1) cpd00004[0] + (1) cpd00347[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00125;cpd00347',\n",
       "  'definition': '(1) NAD[0] + (1) 5-10-Methylenetetrahydrofolate[0] <=> (1) NADH[0] + (1) 5-10-Methenyltetrahydrofolate[0]',\n",
       "  'deltag': 2.04,\n",
       "  'deltagerr': 0.27,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.15'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00125[0] <=> (1) cpd00004[0] + (1) cpd00347[0]',\n",
       "  'id': 'rxn00906',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31886',\n",
       "  'name': '5,10-methylenetetrahydrofolate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); PWY-2201 (folate transformations I); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00670 (One carbon pool by folate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\";1:cpd00004:0:0:\"NADH\";1:cpd00347:0:0:\"5-10-Methenyltetrahydrofolate\"'},\n",
       " {'abbreviation': 'R01220',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: METHYLENETHFDEHYDROG-NADP-RXN',\n",
       "   'BiGG: ADE3_1; MIS1_1; MTHFD; MTHFDm',\n",
       "   'BrachyCyc: METHYLENETHFDEHYDROG-NADP-RXN',\n",
       "   'KEGG: R01220',\n",
       "   'MetaCyc: METHYLENETHFDEHYDROG-NADP-RXN',\n",
       "   'Name: 5,10-methylenetetrahydrofolate dehydrogenase; 5,10-methylenetetrahydrofolate:NADP oxidoreductase; 5,10-methylenetetrahydrofolate:NADP+ oxidoreductase; Methylenetetrahydrofolate dehydrogenase; Methylenetetrahydrofolate dehydrogenase (NADP+); N5,N10-methylenetetrahydrofolate dehydrogenase; methylenetetrahydrofolate dehydrogenase; methylenetetrahydrofolate dehydrogenase (NADP(+)) / methenyltetrahydrofolate cyclohydrolase; methylenetetrahydrofolate dehydrogenase (NADP); methylenetetrahydrofolate dehydrogenase (NADP+); methylenetetrahydrofolate dehydrogenase (nadp+); methylenetetrahydrofolatedehydrogenase-NADP'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00125[0] <=> (1) cpd00005[0] + (1) cpd00347[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00125;cpd00347',\n",
       "  'definition': '(1) NADP[0] + (1) 5-10-Methylenetetrahydrofolate[0] <=> (1) NADPH[0] + (1) 5-10-Methenyltetrahydrofolate[0]',\n",
       "  'deltag': 2.04,\n",
       "  'deltagerr': 0.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.5'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00125[0] <=> (1) cpd00005[0] + (1) cpd00347[0]',\n",
       "  'id': 'rxn00907',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30158;rxn32777',\n",
       "  'name': '5,10-methylenetetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 1CMET2-PWY (<i>N</i><sup>10</sup>-formyl-tetrahydrofolate biosynthesis); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); CODH-PWY (reductive acetyl coenzyme A pathway I (homoacetogenic bacteria)); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); Formaldehyde-Oxidation (Formaldehyde Oxidation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); PWY-1722 (formate assimilation into 5,10-methylenetetrahydrofolate); PWY-2201 (folate transformations I); PWY-3841 (folate transformations II); PWY-5497 (purine nucleobases degradation II (anaerobic)); PWY-6146 (<i>Methanobacterium thermoautotrophicum</i> biosynthetic metabolism); PWY-7909 (formaldehyde oxidation VII (THF pathway)); Purine-Degradation (Purine Nucleotide Degradation); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00670 (One carbon pool by folate); rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\";1:cpd00005:0:0:\"NADPH\";1:cpd00347:0:0:\"5-10-Methenyltetrahydrofolate\"'},\n",
       " {'abbreviation': 'R01221',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GCVMULTI-RXN',\n",
       "   'BiGG: GCV1_2; GLYCL; GLYCL_2; GLYCLm',\n",
       "   'BrachyCyc: GCVMULTI-RXN',\n",
       "   'KEGG: R01221',\n",
       "   'MetaCyc: GCVMULTI-RXN',\n",
       "   'Name: Aminomethyltransferase; Glycine Cleavage System; glycine cleavage system; glycine synthase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00033[0] + (1) cpd00087[0] <=> (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00125[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00011;cpd00013;cpd00033;cpd00087;cpd00125',\n",
       "  'definition': '(1) NAD[0] + (1) Glycine[0] + (1) Tetrahydrofolate[0] => (1) NADH[0] + (1) CO2[0] + (1) NH3[0] + (1) 5-10-Methylenetetrahydrofolate[0]',\n",
       "  'deltag': 4.36,\n",
       "  'deltagerr': 0.98,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.4.4.2', '1.8.1.4', '2.1.2.10'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00033[0] + (1) cpd00087[0] => (1) cpd00004[0] + (1) cpd00011[0] + (1) cpd00013[0] + (1) cpd00125[0]',\n",
       "  'id': 'rxn00908',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19763;rxn30025;rxn30388',\n",
       "  'name': 'glycine synthase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 1CMET2-PWY (<i>N</i><sup>10</sup>-formyl-tetrahydrofolate biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Folate-Biosynthesis (Folate Biosynthesis); Folate-Transformations (Folate Transformations); PWY-181 (photorespiration); PWY-2201 (folate transformations I); PWY-3841 (folate transformations II); Photosynthesis (); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00033:0:0:\"Glycine\";-1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00004:0:0:\"NADH\";1:cpd00011:0:0:\"CO2\";1:cpd00013:0:0:\"NH3\";1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\"'},\n",
       " {'abbreviation': 'R_R01222_c',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01222_c',\n",
       "   'AraGEM: R_R01222_c',\n",
       "   'Maize_C4GEM: R_R01222_c',\n",
       "   'iAO358: rll_250',\n",
       "   'iRS1563: R_R01222_c',\n",
       "   'Name: 5,10-methylenetetrahydrofolate reductase; 5-Methyltetrahyfrofolate:(acceptor) oxidoreductase; methylenetetrahydrofolate reductase 1 (MTHFR1)'],\n",
       "  'code': '(1) cpd00125[0] + (1) cpd00982[0] <=> (1) cpd00015[0] + (1) cpd00345[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00125;cpd00345;cpd00982',\n",
       "  'definition': '(1) 5-10-Methylenetetrahydrofolate[0] + (1) FADH2[0] <=> (1) FAD[0] + (1) 5-Methyltetrahydrofolate[0]',\n",
       "  'deltag': -5.32,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.20', '1.7.99.5'],\n",
       "  'equation': '(1) cpd00125[0] + (1) cpd00982[0] <=> (1) cpd00015[0] + (1) cpd00345[0]',\n",
       "  'id': 'rxn00909',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'methylenetetrahydrofolate reductase 1 (MTHFR1)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\";-1:cpd00982:0:0:\"FADH2\";1:cpd00015:0:0:\"FAD\";1:cpd00345:0:0:\"5-Methyltetrahydrofolate\"'},\n",
       " {'abbreviation': 'R01224',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MET12; MET13; MTHFR1; MTHFR3',\n",
       "   'BrachyCyc: 1.5.1.20-RXN',\n",
       "   'ChlamyCyc: 1.5.1.20-RXN',\n",
       "   'KEGG: R01224',\n",
       "   'SorghumCyc: 1.5.1.20-RXN',\n",
       "   'Name: 5,10-methylenetetrahydrofolate reductase (FADH)-(NADP); 5,10-methylenetetrahydrofolate reductase (NADP); 5,10-methylenetetrahydrofolatereductase (NADPH); 5-methyltetrahydrofolate:NADP+ oxidoreductase; MTHFR; Methylenetetrahydrofolate reductase (NADPH); Methylenetetrahydrofolate reductase 1'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00345[0] <=> (1) cpd00005[0] + (1) cpd00125[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00125;cpd00345',\n",
       "  'definition': '(1) NADP[0] + (1) 5-Methyltetrahydrofolate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) 5-10-Methylenetetrahydrofolate[0]',\n",
       "  'deltag': 8.93,\n",
       "  'deltagerr': 1.02,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.5.1.20'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00345[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00125[0]',\n",
       "  'id': 'rxn00910',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30157;rxn31870',\n",
       "  'name': '5-methyltetrahydrofolate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00670 (One carbon pool by folate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00345:0:0:\"5-Methyltetrahydrofolate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\"'},\n",
       " {'abbreviation': 'R01225',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01225',\n",
       "   'MetaCyc: D-ALANINE-HYDROXYMETHYLTRANSFERASE-RXN',\n",
       "   'Name: 2-methylserine hydroxymethyltransferase; 5,10-Methylenetetrahydrofolate:D-alanine 2-hydroxymethyltransferase; 5,10-methylenetetrahydrofolate:D-alanine 2-hydroxymethyltransferase; D-alanine 2-hydroxymethyltransferase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00117[0] + (1) cpd00125[0] <=> (1) cpd00087[0] + (1) cpd01432[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00087;cpd00117;cpd00125;cpd01432',\n",
       "  'definition': '(1) H2O[0] + (1) D-Alanine[0] + (1) 5-10-Methylenetetrahydrofolate[0] <=> (1) Tetrahydrofolate[0] + (1) 2-Methylserine[0]',\n",
       "  'deltag': 1.74,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.2.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00117[0] + (1) cpd00125[0] <=> (1) cpd00087[0] + (1) cpd01432[0]',\n",
       "  'id': 'rxn00911',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '5,10-Methylenetetrahydrofolate:D-alanine 2-hydroxymethyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00670 (One carbon pool by folate)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00117:0:0:\"D-Alanine\";-1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\";1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd01432:0:0:\"2-Methylserine\"'},\n",
       " {'abbreviation': 'R01226',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3-CH3-2-OXOBUTANOATE-OH-CH3-XFER-RXN',\n",
       "   'BiGG: ECM31; MOHMT',\n",
       "   'BrachyCyc: 3-CH3-2-OXOBUTANOATE-OH-CH3-XFER-RXN',\n",
       "   'KEGG: R01226',\n",
       "   'MetaCyc: 3-CH3-2-OXOBUTANOATE-OH-CH3-XFER-RXN',\n",
       "   'Name: 3-methyl-2-oxobutanoate hydroxymethyltransferase; 3-methyl-2-oxobutanoatehydroxymethyltransferase; 5,10-Methylenetetrahydrofolate:3-methyl-2-oxobutanoate; 5,10-Methylenetetrahydrofolate:3-methyl-2-oxobutanoate hydroxymethyltransferase; 5,10-methylene tetrahydrofolate:alpha-ketoisovalerate hydroxymethyltransferase; 5,10-methylenetetrahydrofolate:3-methyl-2-oxobutanoate hydroxymethyltransferase; alpha-ketoisovalerate hydroxymethyltransferase; dehydropantoate hydroxymethyltransferase; hydroxymethyltransferase; ketopantoate hydroxymethyltransferase; oxopantoate hydroxymethyltransferase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00123[0] + (1) cpd00125[0] <=> (1) cpd00087[0] + (1) cpd00712[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00087;cpd00123;cpd00125;cpd00712',\n",
       "  'definition': '(1) H2O[0] + (1) 3-Methyl-2-oxobutanoate[0] + (1) 5-10-Methylenetetrahydrofolate[0] <=> (1) Tetrahydrofolate[0] + (1) 2-Dehydropantoate[0]',\n",
       "  'deltag': 2.48,\n",
       "  'deltagerr': 1.24,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.2.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00123[0] + (1) cpd00125[0] <=> (1) cpd00087[0] + (1) cpd00712[0]',\n",
       "  'id': 'rxn00912',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17749;rxn30578',\n",
       "  'name': '5,10-Methylenetetrahydrofolate:3-methyl-2-oxobutanoate hydroxymethyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CoA-Biosynthesis (Coenzyme A Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); PANTO-PWY (phosphopantothenate biosynthesis I); PANTOSYN-PWY (superpathway of coenzyme A biosynthesis I (bacteria)); PWY-4221 (superpathway of coenzyme A biosynthesis II (plants)); Pantothenate-Biosynthesis (Phosphopantothenate Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00770 (Pantothenate and CoA biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00123:0:0:\"3-Methyl-2-oxobutanoate\";-1:cpd00125:0:0:\"5-10-Methylenetetrahydrofolate\";1:cpd00087:0:0:\"Tetrahydrofolate\";1:cpd00712:0:0:\"2-Dehydropantoate\"'},\n",
       " {'abbreviation': 'R01227',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 5-NUCLEOTID-RXN; RXN-7609',\n",
       "   'BiGG: NTD9; NTD9pp; U31',\n",
       "   'BrachyCyc: RXN-7609',\n",
       "   'KEGG: R01227',\n",
       "   'MetaCyc: 5-NUCLEOTID-RXN; RXN-7609',\n",
       "   \"Name: 5'-AMP nucleotidase; 5'-AMPase; 5'-adenylic phosphatase; 5'-mononucleotidase; 5'-nucleotidase; 5'-nucleotidase (GMP); 5'-ribonucleotide phosphohydrolase; AMP phosphatase; AMP phosphohydrolase; AMPase; GMP 5'-nucleotidase; Guanosine 5'-monophosphate phosphohydrolase; IMP 5'-nucleotidase; UMPase; adenosine 5'-phosphatase; adenosine monophosphatase; dgmp nucleotidase; snake venom 5'-nucleotidase; thimidine monophosphate nucleotidase; uridine 5'-nucleotidase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00126[0] <=> (1) cpd00009[0] + (1) cpd00311[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00126;cpd00311',\n",
       "  'definition': '(1) H2O[0] + (1) GMP[0] <=> (1) Phosphate[0] + (1) Guanosine[0]',\n",
       "  'deltag': -1.34,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.5'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00126[0] <=> (1) cpd00009[0] + (1) cpd00311[0]',\n",
       "  'id': 'rxn00913',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn18973;rxn30908',\n",
       "  'name': \"Guanosine 5'-monophosphate phosphohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); Guanosine-Nucleotides-Degradation (Guanosine Nucleotide Degradation); NUCLEO-DEG (Nucleoside and Nucleotide Degradation); P165-PWY (superpathway of purines degradation in plants); PWY-5044 (purine nucleotides degradation I (plants)); PWY-6353 (purine nucleotides degradation II (aerobic)); PWY-6595 (superpathway of guanosine nucleotides degradation (plants)); PWY-6606 (guanosine nucleotides degradation II); PWY-6607 (guanosine nucleotides degradation I); PWY-6608 (guanosine nucleotides degradation III); Purine-Degradation (Purine Nucleotide Degradation); Urate-Degradation (Urate Degradation)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00126:0:0:\"GMP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00311:0:0:\"Guanosine\"'},\n",
       " {'abbreviation': 'R01228',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GUANOSINEKIN-RXN',\n",
       "   'BiGG: GSNK; U21',\n",
       "   'BrachyCyc: GUANOSINEKIN-RXN',\n",
       "   'KEGG: R01228',\n",
       "   'MetaCyc: GUANOSINEKIN-RXN',\n",
       "   \"Name: ATP:guanosine 5'-phosphotransferase; guanosine kinase\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00311[0] <=> (1) cpd00008[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00126;cpd00311',\n",
       "  'definition': '(1) ATP[0] + (1) Guanosine[0] => (1) ADP[0] + (1) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -4.82,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.73'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00311[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00914',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19878',\n",
       "  'name': \"ATP:guanosine 5'-phosphotransferase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Guanine-Guanosine-Salvage (Guanine and Guanosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6579 (superpathway of guanine and guanosine salvage); PWY-6618 (guanine and guanosine salvage III); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00311:0:0:\"Guanosine\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01229',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GUANPRIBOSYLTRAN-RXN',\n",
       "   'BiGG: GUAPRT; GUAPRTr; HPT1_2',\n",
       "   'BrachyCyc: GUANPRIBOSYLTRAN-RXN',\n",
       "   'KEGG: R01229',\n",
       "   'MetaCyc: GUANPRIBOSYLTRAN-RXN',\n",
       "   'Name: GMP:diphosphate 5-phospho-alpha-D-ribosyltransferase; GMP:pyrophosphate phosphoribosyltransferase; diphosphate 5-phospho-alpha-D-ribosyltransferase; guanine phosphoribosyltransferase; guaninephosphoribosyltransferase; hypoxantine-guanine phosphorybosyltransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00126[0] <=> (1) cpd00103[0] + (1) cpd00207[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00067;cpd00103;cpd00126;cpd00207',\n",
       "  'definition': '(1) PPi[0] + (1) H+[0] + (1) GMP[0] <= (1) PRPP[0] + (1) Guanine[0]',\n",
       "  'deltag': 6.58,\n",
       "  'deltagerr': 0.54,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.22', '2.4.2.7', '2.4.2.8'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00126[0] <= (1) cpd00103[0] + (1) cpd00207[0]',\n",
       "  'id': 'rxn00915',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32380;rxn32382;rxn32383;rxn32385;rxn32386',\n",
       "  'name': 'GMP:diphosphate 5-phospho-alpha-D-ribosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Guanine-Guanosine-Salvage (Guanine and Guanosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6579 (superpathway of guanine and guanosine salvage); PWY-6599 (guanine and guanosine salvage II); PWY-6620 (guanine and guanosine salvage); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00067:0:0:\"H+\";-1:cpd00126:0:0:\"GMP\";1:cpd00103:0:0:\"PRPP\";1:cpd00207:0:0:\"Guanine\"'},\n",
       " {'abbreviation': 'R01230',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GMP-SYN-NH3-RXN',\n",
       "   'BiGG: GMPS',\n",
       "   'BrachyCyc: GMP-SYN-NH3-RXN',\n",
       "   'KEGG: R01230',\n",
       "   'MetaCyc: GMP-SYN-NH3-RXN',\n",
       "   \"Name: GMP synthase; GMP synthetase; XMP aminase; Xanthosine-5'-phosphate--ammonia ligase; Xanthosine-5'-phosphate:ammonia ligase (AMP-forming); guanylate synthetase; xanthosine 5'-monophosphate aminase; xanthosine-5'-phosphate-ammonia ligase; xanthosine-5'-phosphate:ammonia ligase (AMP-forming)\"],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00497[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00013;cpd00018;cpd00067;cpd00126;cpd00497',\n",
       "  'definition': '(1) ATP[0] + (1) NH3[0] + (1) XMP[0] => (1) PPi[0] + (1) AMP[0] + (3) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -18.42,\n",
       "  'deltagerr': 5.84,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.4.1', '6.3.5.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00013[0] + (1) cpd00497[0] => (1) cpd00012[0] + (1) cpd00018[0] + (3) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00916',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27919;rxn30076;rxn35441',\n",
       "  'name': \"Xanthosine-5'-phosphate:ammonia ligase (AMP-forming)\",\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00013:0:0:\"NH3\";-1:cpd00497:0:0:\"XMP\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";3:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01231',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GMP-SYN-GLUT-RXN',\n",
       "   'BiGG: GMPS2; GUA1',\n",
       "   'BrachyCyc: GMP-SYN-GLUT-RXN',\n",
       "   'KEGG: R01231',\n",
       "   'MetaCyc: GMP-SYN-GLUT-RXN',\n",
       "   \"Name: GMP synthase; GMP synthase (glutamine-hydrolysing); GMP synthetase (glutamine-hydrolysing); GMPsynthase; Glutamine amidotransferase; Xanthosine-5'-phosphate:L-glutamine amido-ligase (AMP-forming); guanosine 5'-monophosphate synthetase; guanosine monophosphate synthetase (glutamine-hydrolyzing); guanylate synthetase (glutamine-hydrolyzing); xanthosine 5'-phosphate amidotransferase; xanthosine-5'-phosphate:L-glutamine amido-ligase (AMP-forming)\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00053[0] + (1) cpd00497[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00023[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00002;cpd00012;cpd00018;cpd00023;cpd00053;cpd00067;cpd00126;cpd00497',\n",
       "  'definition': '(1) H2O[0] + (1) ATP[0] + (1) L-Glutamine[0] + (1) XMP[0] => (1) PPi[0] + (1) AMP[0] + (1) L-Glutamate[0] + (3) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -23.14,\n",
       "  'deltagerr': 5.89,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.3.5.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00002[0] + (1) cpd00053[0] + (1) cpd00497[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00023[0] + (3) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00917',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27918;rxn38133;rxn38134',\n",
       "  'name': \"Xanthosine-5'-phosphate:L-glutamine amido-ligase (AMP-forming)\",\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['MetaCyc: DENOVOPURINE2-PWY (superpathway of purine nucleotides <i>de novo</i> biosynthesis II); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6125 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis II); PWY-7221 (guanosine ribonucleotides <i>de novo</i> biosynthesis); PWY-7228 (superpathway of guanosine nucleotides <i>de novo</i> biosynthesis I); PWY-841 (superpathway of purine nucleotides <i>de novo</i> biosynthesis I); PWY66-409 (superpathway of purine nucleotide salvage); Purine-Nucleotide-De-Novo-Biosynthesis (Purine Nucleotide <i>De Novo</i> Biosynthesis); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage); Purine-Ribonuc-De-Novo-Biosynthesis (Purine Riboucleotide <i>De Novo</i> Biosynthesis)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00002:0:0:\"ATP\";-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00497:0:0:\"XMP\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00023:0:0:\"L-Glutamate\";3:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01232',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.6.1.17-RXN',\n",
       "   'CornCyc: 3.6.1.17-RXN',\n",
       "   'KEGG: R01232',\n",
       "   'MetaCyc: 3.6.1.17-RXN',\n",
       "   'PoplarCyc: 3.6.1.17-RXN',\n",
       "   \"Name: 1-P,4-P-bis(5'-nucleosyl)-tetraphosphate nucleotidohydrolase; Bis(5'-nucleosyl)-tetraphosphatase (asymmetrical); P1,P4-bis(5'-guanosyl)-tetraphosphate guanylylhydrolase; P1,P4-bis(5'-nucleosyl)-tetraphosphate nucleotidohydrolase; bis(5'-adenosyl)-tetraphosphatase; bis(5'-guanosyl)-tetraphosphatase; bis(5'-nucleosyl)-tetraphosphatase (asymmetrical); diadenosine P1,P4-tetraphosphatase; diguanosinetetraphosphatase (asymmetrical); dinucleoside tetraphosphatase; dinucleosidetetraphosphatase (asymmetrical)\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00925[0] <=> (1) cpd00038[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00038;cpd00067;cpd00126;cpd00925',\n",
       "  'definition': '(1) H2O[0] + (1) GppppG[0] => (1) GTP[0] + (1) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -10.32,\n",
       "  'deltagerr': 1.49,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.17'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00925[0] => (1) cpd00038[0] + (1) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00918',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32321;rxn32324',\n",
       "  'name': \"P1,P4-bis(5'-guanosyl)-tetraphosphate guanylylhydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00925:0:0:\"GppppG\";1:cpd00038:0:0:\"GTP\";1:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01233',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01233', 'Name: GDPglucose sugarphosphohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00315[0] <=> (1) cpd00089[0] + (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00089;cpd00126;cpd00315',\n",
       "  'definition': '(1) H2O[0] + (1) GDP-glucose[0] => (2) H+[0] + (1) Glucose-1-phosphate[0] + (1) GMP[0]',\n",
       "  'deltag': -7.22,\n",
       "  'deltagerr': 0.89,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.6.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00315[0] => (2) cpd00067[0] + (1) cpd00089[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00919',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'GDPglucose sugarphosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00500 (Starch and sucrose metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00315:0:0:\"GDP-glucose\";2:cpd00067:0:0:\"H+\";1:cpd00089:0:0:\"Glucose-1-phosphate\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01234',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01234_c',\n",
       "   'AraGEM: R_R01234_c',\n",
       "   'BiGG: PDE2_4; PDE4',\n",
       "   'KEGG: R01234',\n",
       "   'MetaCyc: 3.1.4.17-RXN; 35-CYCLIC-GMP-PHOSPHODIESTERASE-RXN',\n",
       "   \"Name: 3',5'-cyclic-GMP 5'-nucleotidohydrolase; 3',5'-cyclic-GMP phosphodiesterase; 3',5'-cyclic-nucleotide phosphodiesterase 2; Guanosine 3',5'-cyclic phosphate 5'-nucleotidohydrolase; cGMP phosphodiesterase; cGMP-PDE; cyclic 3',5'-GMP phosphodiesterase; cyclic GMP phosphodiesterase; cyclic guanosine 3',5'-monophosphate phosphodiesterase; cyclic guanosine 3',5'-phosphate phosphodiesterase; guanosine cyclic 3',5'-phosphate phosphodiesterase\"],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00697[0] <=> (1) cpd00126[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00126;cpd00697',\n",
       "  'definition': '(1) H2O[0] + (1) cGMP[0] => (1) H+[0] + (1) GMP[0]',\n",
       "  'deltag': -8.84,\n",
       "  'deltagerr': 0.76,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.4.17', '3.1.4.35'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00697[0] => (1) cpd00067[0] + (1) cpd00126[0]',\n",
       "  'id': 'rxn00920',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35440',\n",
       "  'name': \"Guanosine 3',5'-cyclic phosphate 5'-nucleotidohydrolase\",\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00697:0:0:\"cGMP\";1:cpd00067:0:0:\"H+\";1:cpd00126:0:0:\"GMP\"'},\n",
       " {'abbreviation': 'R01238',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01238',\n",
       "   'MetaCyc: 4-HYDROXYBENZOATE-DECARBOXYLASE-RXN',\n",
       "   'Name: 4-Hydroxybenzoate carboxy-lyase; 4-hydroxybenzoate carboxy-lyase; 4-hydroxybenzoate carboxy-lyase (phenol-forming); 4-hydroxybenzoate decarboxylase; p-hydroxybenzoate decarboxylase'],\n",
       "  'code': '(1) cpd00136[0] <=> (1) cpd00011[0] + (1) cpd00127[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00127;cpd00136',\n",
       "  'definition': '(1) H+[0] + (1) 4-Hydroxybenzoate[0] => (1) CO2[0] + (1) Phenol[0]',\n",
       "  'deltag': -2.66,\n",
       "  'deltagerr': 0.8,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.61'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00136[0] => (1) cpd00011[0] + (1) cpd00127[0]',\n",
       "  'id': 'rxn00921',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxybenzoate carboxy-lyase (phenol-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00627 (Aminobenzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00011:0:0:\"CO2\";1:cpd00127:0:0:\"Phenol\"'},\n",
       " {'abbreviation': 'R01239',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01239',\n",
       "   'MetaCyc: PHENOL-O-METHYLTRANSFERASE-RXN',\n",
       "   'Name: PMT; S-Adenosyl-L-methionine:phenol O-methyltransferase; S-adenosyl-L-methionine:phenol O-methyltransferase; phenol O-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00127[0] <=> (1) cpd00019[0] + (1) cpd01004[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00067;cpd00127;cpd01004',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Phenol[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) H+[0] + (1) Anisole[0]',\n",
       "  'deltag': -4.36,\n",
       "  'deltagerr': 4.98,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.25'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00127[0] <=> (1) cpd00019[0] + (1) cpd00067[0] + (1) cpd01004[0]',\n",
       "  'id': 'rxn00922',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:phenol O-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00127:0:0:\"Phenol\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd00067:0:0:\"H+\";1:cpd01004:0:0:\"Anisole\"'},\n",
       " {'abbreviation': 'R01240',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01240',\n",
       "   'Name: UDP-glucose:phenol beta-D-glucosyltransferase; UDPglucose:phenol beta-D-glucosyltransferase'],\n",
       "  'code': '(1) cpd00026[0] + (1) cpd00127[0] <=> (1) cpd00014[0] + (1) cpd01988[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00026;cpd00127;cpd01988',\n",
       "  'definition': '(1) UDP-glucose[0] + (1) Phenol[0] <=> (1) UDP[0] + (1) Aryl beta-D-glucoside[0]',\n",
       "  'deltag': -13.21,\n",
       "  'deltagerr': 1.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.35'],\n",
       "  'equation': '(1) cpd00026[0] + (1) cpd00127[0] <=> (1) cpd00014[0] + (1) cpd01988[0]',\n",
       "  'id': 'rxn00923',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-glucose:phenol beta-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";-1:cpd00127:0:0:\"Phenol\";1:cpd00014:0:0:\"UDP\";1:cpd01988:0:0:\"Aryl beta-D-glucoside\"'},\n",
       " {'abbreviation': 'R01241',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01241; R07342',\n",
       "   'Name: aryl-ester hydrolase; phenyl acetate acetylhydrolase'],\n",
       "  'code': '(1) cpd00029[0] + (1) cpd00127[0] <=> (1) cpd00001[0] + (1) cpd00430[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00029;cpd00067;cpd00127;cpd00430',\n",
       "  'definition': '(1) Acetate[0] + (1) H+[0] + (1) Phenol[0] <= (1) H2O[0] + (1) PACT[0]',\n",
       "  'deltag': 3.68,\n",
       "  'deltagerr': 1.38,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['3.1.1.2'],\n",
       "  'equation': '(1) cpd00029[0] + (1) cpd00067[0] + (1) cpd00127[0] <= (1) cpd00001[0] + (1) cpd00430[0]',\n",
       "  'id': 'rxn00924',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'phenyl acetate acetylhydrolase',\n",
       "  'notes': ['GCC', 'EQU', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00029:0:0:\"Acetate\";-1:cpd00067:0:0:\"H+\";-1:cpd00127:0:0:\"Phenol\";1:cpd00001:0:0:\"H2O\";1:cpd00430:0:0:\"PACT\"'},\n",
       " {'abbreviation': 'R01243',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_ARSO4SH',\n",
       "   'KEGG: R01243',\n",
       "   'iAbaylyiv4: ARYLSULFAT-RXN',\n",
       "   'Name: Phenol sulfate sulfohydrolase; aryl-sulfate sulfohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01471[0] <=> (1) cpd00048[0] + (1) cpd00127[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00048;cpd00067;cpd00127;cpd01471',\n",
       "  'definition': '(1) H2O[0] + (1) Arylsulfate[0] => (1) Sulfate[0] + (1) H+[0] + (1) Phenol[0]',\n",
       "  'deltag': -16.31,\n",
       "  'deltagerr': 1.49,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.6.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01471[0] => (1) cpd00048[0] + (1) cpd00067[0] + (1) cpd00127[0]',\n",
       "  'id': 'rxn00925',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'aryl-sulfate sulfohydrolase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01471:0:0:\"Arylsulfate\";1:cpd00048:0:0:\"Sulfate\";1:cpd00067:0:0:\"H+\";1:cpd00127:0:0:\"Phenol\"'},\n",
       " {'abbreviation': 'R01244',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AAH1_3; ADD',\n",
       "   'ChlamyCyc: ADENINE-DEAMINASE-RXN',\n",
       "   'EcoCyc: ADENINE-DEAMINASE-RXN',\n",
       "   'KEGG: R01244',\n",
       "   'MetaCyc: ADENINE-DEAMINASE-RXN',\n",
       "   'Name: ADase; Adenine aminohydrolase; Adenosine deaminase; adenase; adenine aminase; adenine aminohydrolase; adenine deaminase; adeninedeaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00128[0] <=> (1) cpd00013[0] + (1) cpd00226[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00067;cpd00128;cpd00226',\n",
       "  'definition': '(1) H2O[0] + (1) H+[0] + (1) Adenine[0] => (1) NH3[0] + (1) HYXN[0]',\n",
       "  'deltag': -5.3,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.4.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00067[0] + (1) cpd00128[0] => (1) cpd00013[0] + (1) cpd00226[0]',\n",
       "  'id': 'rxn00926',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn29986',\n",
       "  'name': 'Adenine aminohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenine-Adenosine-Salvage (Adenine and Adenosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6610 (adenine salvage); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00067:0:0:\"H+\";-1:cpd00128:0:0:\"Adenine\";1:cpd00013:0:0:\"NH3\";1:cpd00226:0:0:\"HYXN\"'},\n",
       " {'abbreviation': 'R01245',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01245_c',\n",
       "   'AraGEM: R_R01245_c',\n",
       "   'BiGG: ADNUC; PNS1; URH1_2',\n",
       "   'KEGG: R01245',\n",
       "   'MetaCyc: ADENOSINE-NUCLEOSIDASE-RXN',\n",
       "   'Name: Adenosine ribohydrolase; adenosine hydrolase; adenosinehydrolase; adnuc'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00182[0] <=> (1) cpd00105[0] + (1) cpd00128[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00105;cpd00128;cpd00182',\n",
       "  'definition': '(1) H2O[0] + (1) Adenosine[0] <=> (1) D-Ribose[0] + (1) Adenine[0]',\n",
       "  'deltag': -1.15,\n",
       "  'deltagerr': 0.43,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.1', '3.2.2.7', '3.2.2.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00182[0] <=> (1) cpd00105[0] + (1) cpd00128[0]',\n",
       "  'id': 'rxn00927',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Adenosine ribohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Adenine-Adenosine-Salvage (Adenine and Adenosine Salvage); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PUR-NUC-SYN (Purine Nucleotide Biosynthesis); PWY-6605 (adenine and adenosine salvage II); Purine-Nucleotides-Salvage (Purine Nucleotide Salvage)',\n",
       "   'KEGG: rn00230 (Purine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00182:0:0:\"Adenosine\";1:cpd00105:0:0:\"D-Ribose\";1:cpd00128:0:0:\"Adenine\"'},\n",
       " {'abbreviation': 'R01246',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01246', 'Name: L-proline:NAD+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00129[0] <=> (1) cpd00004[0] + (1) cpd02235[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00129;cpd02235',\n",
       "  'definition': '(1) NAD[0] + (1) L-Proline[0] => (1) NADH[0] + (1) H+[0] + (1) 1-Pyrroline-2-carboxylate[0]',\n",
       "  'deltag': 5.6,\n",
       "  'deltagerr': 0.37,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.1.1'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00129[0] => (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd02235[0]',\n",
       "  'id': 'rxn00928',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-proline:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00129:0:0:\"L-Proline\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd02235:0:0:\"1-Pyrroline-2-carboxylate\"'},\n",
       " {'abbreviation': 'R01248',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRROLINECARBREDUCT-RXN',\n",
       "   'BiGG: P5CRx; PRO1x; PRO1xm; PUT1',\n",
       "   'BrachyCyc: PYRROLINECARBREDUCT-RXN',\n",
       "   'KEGG: R01248',\n",
       "   'MetaCyc: PYRROLINECARBREDUCT-RXN',\n",
       "   'Name: 1-pyrroline-5-carboxylate reductase; L-Proline:NAD+ 5-oxidoreductase; L-proline oxidase; L-proline-NAD(P)+ 5-oxidoreductase; L-proline:NAD(P)+ 5-oxidoreductase; NADPH-L-delta1-pyrroline carboxylic acid reductase; P5CR; Proline oxidase; Pyrroline-5-carboxylate reductase; proline oxidase; proline oxidase (L-proline, NAD); proline oxidase (NAD); pyrroline-5-carboxylate reductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00129[0] <=> (1) cpd00004[0] + (1) cpd02431[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00129;cpd02431',\n",
       "  'definition': '(1) NAD[0] + (1) L-Proline[0] => (1) NADH[0] + (2) H+[0] + (1) 1-Pyrroline-5-carboxylate[0]',\n",
       "  'deltag': 4.72,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.1.2'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00129[0] => (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd02431[0]',\n",
       "  'id': 'rxn00929',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28115;rxn38910',\n",
       "  'name': 'L-Proline:NAD+ 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG-PRO-PWY (L-arginine degradation VI (arginase 2 pathway)); ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); MISCELLANEOUS-DEG (Other Amino Acid Degradation); PROLINE-SYN (L-proline Biosynthesis); PROSYN-PWY (L-proline biosynthesis I); PWY-3341 (L-proline biosynthesis III); PWY-4981 (L-proline biosynthesis II (from arginine)); PWY-6344 (L-ornithine degradation II (Stickland reaction)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00129:0:0:\"L-Proline\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd02431:0:0:\"1-Pyrroline-5-carboxylate\"'},\n",
       " {'abbreviation': 'R01249',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01249', 'Name: L-proline:NADP+ 2-oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00129[0] <=> (1) cpd00005[0] + (1) cpd02235[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00129;cpd02235',\n",
       "  'definition': '(1) NADP[0] + (1) L-Proline[0] => (1) NADPH[0] + (1) H+[0] + (1) 1-Pyrroline-2-carboxylate[0]',\n",
       "  'deltag': 5.61,\n",
       "  'deltagerr': 0.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.1.1'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00129[0] => (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd02235[0]',\n",
       "  'id': 'rxn00930',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-proline:NADP+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00129:0:0:\"L-Proline\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd02235:0:0:\"1-Pyrroline-2-carboxylate\"'},\n",
       " {'abbreviation': 'R01251',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PYRROLINECARBREDUCT-RXN',\n",
       "   'BiGG: P5CR; PRO3_1',\n",
       "   'BrachyCyc: PYRROLINECARBREDUCT-RXN',\n",
       "   'KEGG: R01251',\n",
       "   'MetaCyc: PYRROLINECARBREDUCT-RXN',\n",
       "   'Name: 1-pyrroline-5-carboxylate reductase; L-Proline:NADP+ 5-oxidoreductase; L-proline oxidase; L-proline-NAD(P)+ 5-oxidoreductase; L-proline:NAD(P)+ 5-oxidoreductase; NADPH-L-delta1-pyrroline carboxylic acid reductase; Pyrroline-5-carboxylate reductase; proline oxidase; pyrroline-5-carboxylate reductase; pyrroline-5-carboxylatereductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00129[0] <=> (1) cpd00005[0] + (1) cpd02431[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00129;cpd02431',\n",
       "  'definition': '(1) NADP[0] + (1) L-Proline[0] => (1) NADPH[0] + (2) H+[0] + (1) 1-Pyrroline-5-carboxylate[0]',\n",
       "  'deltag': 4.72,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.5.1.2'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00129[0] => (1) cpd00005[0] + (2) cpd00067[0] + (1) cpd02431[0]',\n",
       "  'id': 'rxn00931',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28114;rxn38909',\n",
       "  'name': 'L-Proline:NADP+ 5-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG-PRO-PWY (L-arginine degradation VI (arginase 2 pathway)); ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); MISCELLANEOUS-DEG (Other Amino Acid Degradation); PROLINE-SYN (L-proline Biosynthesis); PROSYN-PWY (L-proline biosynthesis I); PWY-3341 (L-proline biosynthesis III); PWY-4981 (L-proline biosynthesis II (from arginine)); PWY-6344 (L-ornithine degradation II (Stickland reaction)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00129:0:0:\"L-Proline\";1:cpd00005:0:0:\"NADPH\";2:cpd00067:0:0:\"H+\";1:cpd02431:0:0:\"1-Pyrroline-5-carboxylate\"'},\n",
       " {'abbreviation': 'R01252',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN490-3641',\n",
       "   'BrachyCyc: RXN490-3641',\n",
       "   'KEGG: R01252',\n",
       "   'MaizeCyc: RXN490-3641',\n",
       "   'MetaCyc: RXN490-3641',\n",
       "   'Name: L-Proline,2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating); Proline hydroxylase; Proline,2-oxoglutarate 4-dioxygenase; Prolyl 4-hydroxylase; Prolyl hydroxylase; Protocollagen hydroxylase; proline,2-oxoglutarate-4-dioxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00024[0] + (1) cpd00129[0] <=> (1) cpd00011[0] + (1) cpd00036[0] + (1) cpd00851[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00024;cpd00036;cpd00129;cpd00851',\n",
       "  'definition': '(1) O2[0] + (1) 2-Oxoglutarate[0] + (1) L-Proline[0] => (1) CO2[0] + (1) Succinate[0] + (1) trans-4-Hydroxy-L-proline[0]',\n",
       "  'deltag': -117.0,\n",
       "  'deltagerr': 2.28,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.11.2', '1.14.11.57'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00024[0] + (1) cpd00129[0] => (1) cpd00011[0] + (1) cpd00036[0] + (1) cpd00851[0]',\n",
       "  'id': 'rxn00932',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Proline,2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00024:0:0:\"2-Oxoglutarate\";-1:cpd00129:0:0:\"L-Proline\";1:cpd00011:0:0:\"CO2\";1:cpd00036:0:0:\"Succinate\";1:cpd00851:0:0:\"trans-4-Hydroxy-L-proline\"'},\n",
       " {'abbreviation': 'R01255',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01255',\n",
       "   'MetaCyc: PROLINE-RACEMASE-RXN',\n",
       "   'RiceCyc: PROLINE-RACEMASE-RXN',\n",
       "   'Name: L-Proline racemase; proline racemase'],\n",
       "  'code': '(1) cpd00129[0] <=> (1) cpd00567[0]',\n",
       "  'compound_ids': 'cpd00129;cpd00567',\n",
       "  'definition': '(1) L-Proline[0] <=> (1) D-Proline[0]',\n",
       "  'deltag': -7.19,\n",
       "  'deltagerr': 1.91,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.1.4'],\n",
       "  'equation': '(1) cpd00129[0] <=> (1) cpd00567[0]',\n",
       "  'id': 'rxn00933',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'proline racemase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); MISCELLANEOUS-DEG (Other Amino Acid Degradation); PWY-6344 (L-ornithine degradation II (Stickland reaction)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00129:0:0:\"L-Proline\";1:cpd00567:0:0:\"D-Proline\"'},\n",
       " {'abbreviation': 'R01256',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: MALTK',\n",
       "   'KEGG: R01256',\n",
       "   'MetaCyc: MALATE--COA-LIGASE-RXN',\n",
       "   'RiceCyc: MALATE--COA-LIGASE-RXN',\n",
       "   'iSO783: MALTK',\n",
       "   'Name: (S)-Malate:CoA ligase (ADP-forming); malate thiokinase; malate-CoA ligase; malate:CoA ligase (ADP-forming); malyl coenzyme A synthetase; malyl-CoA synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00130[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd02662[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00010;cpd00130;cpd02662',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) L-Malate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) Malyl-CoA[0]',\n",
       "  'deltag': -1.53,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.9'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00130[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd02662[0]',\n",
       "  'id': 'rxn00934',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-Malate:CoA ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Formaldehyde-Assimilation (Formaldehyde Assimilation); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-5741 (ethylmalonyl-CoA pathway)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn00680 (Methane metabolism); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd02662:0:0:\"Malyl-CoA\"'},\n",
       " {'abbreviation': 'R01257',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01257',\n",
       "   'Name: (S)-Malate:(acceptor) oxidoreductase; (S)-Malate:FAD oxidoreductase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00130[0] <=> (1) cpd00032[0] + (1) cpd00982[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00032;cpd00130;cpd00982',\n",
       "  'definition': '(1) FAD[0] + (1) L-Malate[0] => (1) Oxaloacetate[0] + (1) FADH2[0]',\n",
       "  'deltag': 3.49,\n",
       "  'deltagerr': 0.73,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.1.5.4'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00130[0] => (1) cpd00032[0] + (1) cpd00982[0]',\n",
       "  'id': 'rxn00935',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '(S)-Malate:(acceptor) oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00620 (Pyruvate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00130:0:0:\"L-Malate\";1:cpd00032:0:0:\"Oxaloacetate\";1:cpd00982:0:0:\"FADH2\"'},\n",
       " {'abbreviation': 'R01266',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01266',\n",
       "   'Name: 1-beta-Aspartyl-N-acetyl-D-glucosaminylamine L-asparaginohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02763[0] <=> (1) cpd00122[0] + (1) cpd00132[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00122;cpd00132;cpd02763',\n",
       "  'definition': '(1) H2O[0] + (1) N4-(Acetyl-beta-D-glucosaminyl)asparagine[0] <=> (1) N-Acetyl-D-glucosamine[0] + (1) L-Asparagine[0]',\n",
       "  'deltag': 5.01,\n",
       "  'deltagerr': 1.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.2.2.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02763[0] <=> (1) cpd00122[0] + (1) cpd00132[0]',\n",
       "  'id': 'rxn00936',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '1-beta-Aspartyl-N-acetyl-D-glucosaminylamine L-asparaginohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02763:0:0:\"N4-(Acetyl-beta-D-glucosaminyl)asparagine\";1:cpd00122:0:0:\"N-Acetyl-D-glucosamine\";1:cpd00132:0:0:\"L-Asparagine\"'},\n",
       " {'abbreviation': 'R01267',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3-CYANOALANINE-HYDRATASE-RXN',\n",
       "   'BrachyCyc: 3-CYANOALANINE-HYDRATASE-RXN',\n",
       "   'CornCyc: 3-CYANOALANINE-HYDRATASE-RXN',\n",
       "   'KEGG: R01267',\n",
       "   'MetaCyc: 3-CYANOALANINE-HYDRATASE-RXN',\n",
       "   'Name: 3-cyanoalanine hydratase; L-Asparagine hydro-lyase; L-asparagine hydro-lyase; L-asparagine hydro-lyase (3-cyanoalanine-forming); beta-CNA nitrilase; beta-CNAla hydrolase; beta-cyanoalanine hydratase; beta-cyanoalanine hydrolase'],\n",
       "  'code': '(1) cpd00132[0] <=> (1) cpd00001[0] + (1) cpd01651[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00132;cpd01651',\n",
       "  'definition': '(1) L-Asparagine[0] <=> (1) H2O[0] + (1) L-3-Cyanoalanine[0]',\n",
       "  'deltag': -6.53,\n",
       "  'deltagerr': 4.99,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.65'],\n",
       "  'equation': '(1) cpd00132[0] <=> (1) cpd00001[0] + (1) cpd01651[0]',\n",
       "  'id': 'rxn00937',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30579;rxn32243',\n",
       "  'name': 'L-asparagine hydro-lyase (3-cyanoalanine-forming)',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['MetaCyc: ASPARAGINE-SYN (L-asparagine Biosynthesis); ASPSYNII-PWY (cyanide detoxification I); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Cyanide-Detoxification (Cyanide Detoxification); Degradation (Degradation/Utilization/Assimilation); Detoxification (Detoxification); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); Other-Degradation (Degradation/Utilization/Assimilation-Other)',\n",
       "   'KEGG: rn00460 (Cyanoamino acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00132:0:0:\"L-Asparagine\";1:cpd00001:0:0:\"H2O\";1:cpd01651:0:0:\"L-3-Cyanoalanine\"'},\n",
       " {'abbreviation': 'R01268',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NICOTINAMID-RXN; R314-RXN',\n",
       "   'BiGG: NNAM; NNAMr; NNAMrm; PNC1_1; PNC1_2',\n",
       "   'BrachyCyc: NICOTINAMID-RXN; R314-RXN',\n",
       "   'KEGG: R01268',\n",
       "   'MetaCyc: NICOTINAMID-RXN',\n",
       "   'Name: Nicotinamidase; Nicotinamide amidohydrolase; YNDase; nicotinamidase; nicotinamidase, reversible; nicotinamide amidase; nicotinamide amidohydrolase; nicotinamide deaminase; nicotinaminase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00133[0] <=> (1) cpd00013[0] + (1) cpd00218[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00133;cpd00218',\n",
       "  'definition': '(1) H2O[0] + (1) Nicotinamide[0] => (1) NH3[0] + (1) Niacin[0]',\n",
       "  'deltag': -1.38,\n",
       "  'deltagerr': 0.78,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.5.1.19', '3.5.1.4'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00133[0] => (1) cpd00013[0] + (1) cpd00218[0]',\n",
       "  'id': 'rxn00938',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30041;rxn30187;rxn30414;rxn30738;rxn38279;rxn38280',\n",
       "  'name': 'Nicotinamide amidohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Degradation (Degradation/Utilization/Assimilation); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); P345-PWY (aldoxime degradation); PWY-3502 (superpathway of NAD biosynthesis in eukaryotes); PWY-5381 (pyridine nucleotide cycling (plants)); PWY3O-4107 (NAD salvage pathway V (PNC V cycle)); PYRIDNUCSAL-PWY (NAD salvage pathway I (PNC VI cycle))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00133:0:0:\"Nicotinamide\";1:cpd00013:0:0:\"NH3\";1:cpd00218:0:0:\"Niacin\"'},\n",
       " {'abbreviation': 'R01269',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: NNMT',\n",
       "   'KEGG: R01269',\n",
       "   'MetaCyc: NICOTINAMIDE-N-METHYLTRANSFERASE-RXN',\n",
       "   'iMM904: NNMT',\n",
       "   'Name: Nicotinamide N-methyltransferase; S-Adenosyl-L-methionine:nicotinamide N-methyltransferase; S-adenosyl-L-methionine:nicotinamide N-methyltransferase; nicotinamide N-methyltransferase; nicotinamide methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00133[0] <=> (1) cpd00019[0] + (1) cpd01866[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00133;cpd01866',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Nicotinamide[0] <=> (1) S-Adenosyl-homocysteine[0] + (1) 1-Methylnicotinamide[0]',\n",
       "  'deltag': -3.8,\n",
       "  'deltagerr': 5.3,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.1.1'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00133[0] <=> (1) cpd00019[0] + (1) cpd01866[0]',\n",
       "  'id': 'rxn00939',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:nicotinamide N-methyltransferase',\n",
       "  'notes': ['GCC', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00133:0:0:\"Nicotinamide\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd01866:0:0:\"1-Methylnicotinamide\"'},\n",
       " {'abbreviation': 'R01270',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: NMNNUCLEOSID-RXN',\n",
       "   'BrachyCyc: NMNNUCLEOSID-RXN',\n",
       "   'ChlamyCyc: NMNNUCLEOSID-RXN',\n",
       "   'KEGG: R01270',\n",
       "   'MetaCyc: NMNNUCLEOSID-RXN',\n",
       "   'Name: NMN glycohydrolase; NMN nucleosidase; NMNGhase; NMNase; Nicotinamide D-ribonucleotide phosphoribohydrolase; nicotinamide mononucleotidase; nicotinamide mononucleotide nucleosidase; nicotinamide-nucleotide phosphoribohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00355[0] <=> (1) cpd00101[0] + (1) cpd00133[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00101;cpd00133;cpd00355',\n",
       "  'definition': '(1) H2O[0] + (1) Nicotinamide ribonucleotide[0] => (1) H+[0] + (1) ribose-5-phosphate[0] + (1) Nicotinamide[0]',\n",
       "  'deltag': -5.54,\n",
       "  'deltagerr': 1.2,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.2.14', '3.2.2.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00355[0] => (1) cpd00067[0] + (1) cpd00101[0] + (1) cpd00133[0]',\n",
       "  'id': 'rxn00940',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32692;rxn32698',\n",
       "  'name': 'Nicotinamide D-ribonucleotide phosphoribohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis); PWY-5381 (pyridine nucleotide cycling (plants)); PYRIDNUCSAL-PWY (NAD salvage pathway I (PNC VI cycle))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00355:0:0:\"Nicotinamide ribonucleotide\";1:cpd00067:0:0:\"H+\";1:cpd00101:0:0:\"ribose-5-phosphate\";1:cpd00133:0:0:\"Nicotinamide\"'},\n",
       " {'abbreviation': 'R01271',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_NAPT',\n",
       "   'KEGG: R01271',\n",
       "   'MetaCyc: 2.4.2.12-RXN',\n",
       "   'PlantCyc: 2.4.2.12-RXN',\n",
       "   'iAbaylyiv4: 2.4.2.12-RXN',\n",
       "   'Name: NMN pyrophosphorylase; NMN synthetase; Nicotinamide-nucleotide:pyrophosphate; nicotinamide mononucleotide pyrophosphorylase; nicotinamide mononucleotide synthetase; nicotinamide phosphoribosyltransferase; nicotinamide-D-ribonucleotide:diphosphate phospho-alpha-D-ribosyltransferase; nicotinamide-nucleotide:diphosphate phospho-alpha-D-ribosyltransferase'],\n",
       "  'code': '(1) cpd00012[0] + (1) cpd00355[0] <=> (1) cpd00103[0] + (1) cpd00133[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00103;cpd00133;cpd00355',\n",
       "  'definition': '(1) PPi[0] + (1) Nicotinamide ribonucleotide[0] <= (1) PRPP[0] + (1) Nicotinamide[0]',\n",
       "  'deltag': 3.08,\n",
       "  'deltagerr': 1.26,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['2.4.2.12'],\n",
       "  'equation': '(1) cpd00012[0] + (1) cpd00355[0] <= (1) cpd00103[0] + (1) cpd00133[0]',\n",
       "  'id': 'rxn00941',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'nicotinamide-D-ribonucleotide:diphosphate phospho-alpha-D-ribosyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-BIOSYNTHESIS-III (NAD biosynthesis III (from nicotinamide)); NAD-Metabolism (NAD Metabolism); NAD-SYN (NAD Biosynthesis)',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00012:0:0:\"PPi\";-1:cpd00355:0:0:\"Nicotinamide ribonucleotide\";1:cpd00103:0:0:\"PRPP\";1:cpd00133:0:0:\"Nicotinamide\"'},\n",
       " {'abbreviation': 'R01273',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01273_c',\n",
       "   'KEGG: R01273',\n",
       "   'MetaCyc: RXN-8441',\n",
       "   'TS_Athaliana: rxn1360_c',\n",
       "   'Name: N-Ribosylnicotinamide ribohydrolase; N-ribosylnicotinamide ribohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02016[0] <=> (1) cpd00105[0] + (1) cpd00133[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00105;cpd00133;cpd02016',\n",
       "  'definition': '(1) H2O[0] + (1) N-Ribosylnicotinamide[0] => (1) H+[0] + (1) D-Ribose[0] + (1) Nicotinamide[0]',\n",
       "  'deltag': -6.8,\n",
       "  'deltagerr': 0.83,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.2.1'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02016[0] => (1) cpd00067[0] + (1) cpd00105[0] + (1) cpd00133[0]',\n",
       "  'id': 'rxn00942',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'N-ribosylnicotinamide ribohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); NAD-Metabolism (NAD Metabolism); PWY-5381 (pyridine nucleotide cycling (plants))',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02016:0:0:\"N-Ribosylnicotinamide\";1:cpd00067:0:0:\"H+\";1:cpd00105:0:0:\"D-Ribose\";1:cpd00133:0:0:\"Nicotinamide\"'},\n",
       " {'abbreviation': 'R01274',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PALMITOYL-COA-HYDROLASE-RXN',\n",
       "   'BiGG: FACOAE160; PTE2x',\n",
       "   'BrachyCyc: PALMITOYL-COA-HYDROLASE-RXN',\n",
       "   'KEGG: R01274',\n",
       "   'MetaCyc: PALMITOYL-COA-HYDROLASE-RXN',\n",
       "   'Name: Palmitoyl-CoA hydrolase; acyl-CoA thioesterase; acyl-coa thiolase (c16:0); fatty acyl thioesterase I; fatty-acid-CoA thioesterase (hexadecanoate); long-chain fatty-acyl-CoA hydrolase; palmitoyl coenzyme A hydrolase; palmitoyl thioesterase; palmitoyl-CoA deacylase; palmitoyl-CoA hydrolase; palmityl thioesterase; palmityl thioesterase I; palmityl-CoA deacylase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00134[0] <=> (1) cpd00010[0] + (1) cpd00214[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00067;cpd00134;cpd00214',\n",
       "  'definition': '(1) H2O[0] + (1) Palmitoyl-CoA[0] => (1) CoA[0] + (1) H+[0] + (1) Palmitate[0]',\n",
       "  'deltag': -5.8,\n",
       "  'deltagerr': 1.53,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.2', '3.1.2.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00134[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00214[0]',\n",
       "  'id': 'rxn00943',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn17810;rxn20502;rxn20503',\n",
       "  'name': 'Palmitoyl-CoA hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Cell-Structure-Biosynthesis (Cell Structure Biosynthesis); EPIDERMAL-STRUCTURE (Epidermal Structures); Fatty-acid-biosynthesis (Fatty Acid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); Metabolic-Clusters (Metabolic Clusters); PLANT-CELL-STRUCTURE (Plant Cell Structures); PWY-1121 (suberin monomers biosynthesis); PWY-321 (cutin biosynthesis); PWY-5994 (palmitate biosynthesis I (animals and fungi)); PWY-6733 (sporopollenin precursors biosynthesis); Palmitate-Biosynthesis (Palmitate Biosynthesis)',\n",
       "   'KEGG: rn00062 (Fatty acid elongation); rn01040 (Biosynthesis of unsaturated fatty acids); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00134:0:0:\"Palmitoyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd00214:0:0:\"Palmitate\"'},\n",
       " {'abbreviation': 'R01277',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01277',\n",
       "   'MetaCyc: 1.2.1.42-RXN',\n",
       "   'Name: fatty acyl-CoA reductase; hexadecanal dehydrogenase (acylating); hexadecanal:NAD+ oxidoreductase (CoA-acylating)'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00405[0] <=> (1) cpd00004[0] + (1) cpd00134[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00010;cpd00067;cpd00134;cpd00405',\n",
       "  'definition': '(1) NAD[0] + (1) CoA[0] + (1) Hexadecanal[0] <=> (1) NADH[0] + (1) H+[0] + (1) Palmitoyl-CoA[0]',\n",
       "  'deltag': -3.34,\n",
       "  'deltagerr': 0.46,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.42'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00010[0] + (1) cpd00405[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00134[0]',\n",
       "  'id': 'rxn00944',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'hexadecanal:NAD+ oxidoreductase (CoA-acylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00010:0:0:\"CoA\";-1:cpd00405:0:0:\"Hexadecanal\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00134:0:0:\"Palmitoyl-CoA\"'},\n",
       " {'abbreviation': 'R01278',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01278_c',\n",
       "   'EcoCyc: TRANSENOYLCOARED-RXN',\n",
       "   'JP_Creinhardtii_MSB: R_ACOAR7m',\n",
       "   'KEGG: R01278',\n",
       "   'MetaCyc: RXN-14194; TRANSENOYLCOARED-RXN',\n",
       "   'Name: NADPH-dependent trans-2-enoyl-CoA reductase; acyl-CoA:NADP+ trans-2-oxidoreductase; hexadecanoyl-CoA:NADP+ trans-2-oxidoreductase; hexadecanoyl-CoA:NADP+ trans-2-oxidoreductase;; reductase, trans-enoyl coenzyme A; trans-2-enoyl-CoA reductase (NADPH); trans-2-enoyl-CoA reductase (NADPH2); trans-Hexadec-2-enoyl-CoA reductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00134[0] <=> (1) cpd00005[0] + (1) cpd03126[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00134;cpd03126',\n",
       "  'definition': '(1) NADP[0] + (1) Palmitoyl-CoA[0] <= (1) NADPH[0] + (1) H+[0] + (1) (2E)-Hexadecenoyl-CoA[0]',\n",
       "  'deltag': 14.74,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['1.3.1.38', '1.3.1.8'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00134[0] <= (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd03126[0]',\n",
       "  'id': 'rxn00945',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33068;rxn33082',\n",
       "  'name': 'hexadecanoyl-CoA:NADP+ trans-2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); PWY3DJ-11470 (sphingosine and sphingosine-1-phosphate metabolism)',\n",
       "   'KEGG: rn00062 (Fatty acid elongation); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00134:0:0:\"Palmitoyl-CoA\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd03126:0:0:\"(2E)-Hexadecenoyl-CoA\"'},\n",
       " {'abbreviation': 'R01279',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACOAD7f',\n",
       "   'JP_Creinhardtii_MSB: R_ACOA160OR',\n",
       "   'KEGG: R01279',\n",
       "   'TS_Athaliana: rxn637_x',\n",
       "   'iAF1260: ACOAD7f',\n",
       "   'Name: Palmitoyl-CoA:oxygen 2-oxidoreductase; R_Palmitoyl_CoA_oxygen_2_oxidoreductase; acyl-CoA dehydrogenase (hexadecanoyl-CoA); acyl-coa dehydrogenase (c16:0); palmitoyl-CoA:electron-transfer flavoprotein 2,3-oxidoreductase'],\n",
       "  'code': '(1) cpd00015[0] + (1) cpd00134[0] <=> (1) cpd00982[0] + (1) cpd03126[0]',\n",
       "  'compound_ids': 'cpd00015;cpd00134;cpd00982;cpd03126',\n",
       "  'definition': '(1) FAD[0] + (1) Palmitoyl-CoA[0] <=> (1) FADH2[0] + (1) (2E)-Hexadecenoyl-CoA[0]',\n",
       "  'deltag': 11.14,\n",
       "  'deltagerr': 0.86,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.3.3.6', '1.3.8.9', '1.3.99.-', '1.3.99.13', '1.3.99.3'],\n",
       "  'equation': '(1) cpd00015[0] + (1) cpd00134[0] <=> (1) cpd00982[0] + (1) cpd03126[0]',\n",
       "  'id': 'rxn00946',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31416',\n",
       "  'name': 'Palmitoyl-CoA:oxygen 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00071 (Fatty acid degradation); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00015:0:0:\"FAD\";-1:cpd00134:0:0:\"Palmitoyl-CoA\";1:cpd00982:0:0:\"FADH2\";1:cpd03126:0:0:\"(2E)-Hexadecenoyl-CoA\"'},\n",
       " {'abbreviation': 'R01280',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-7904; RXN-9623',\n",
       "   'BiGG: FAA1_3; FAA3_1; FAA4_1; FACOAL160; FACOAL160p; FAT1_1',\n",
       "   'BrachyCyc: RXN-7904; RXN-9623; RXN-9917',\n",
       "   'KEGG: R01280',\n",
       "   'MetaCyc: RXN-7904; RXN-9623',\n",
       "   'Name: ACS3; FAA1; LCFA synthetase; Long-chain-fatty-acid--CoA ligase 1; Palmitate:CoA ligase (AMP-forming); acyl coa transferase (c16:0); acyl coenzyme A synthetase; acyl-CoA ligase; acyl-CoA synthetase; acyl-activating enzyme; acyl-coenzyme A ligase; arachidonyl-CoA synthetase; fatty acid CoA ligase; fatty acid elongase; fatty acid thiokinase (long chain); fatty acyl-coenzyme A synthetase; fatty-acid--CoA ligase (hexadecanoate); lignoceroyl-CoA synthase; long chain fatty acyl-CoA synthetase; long-chain acyl CoA synthetase; long-chain acyl-CoA synthetase I; long-chain acyl-CoA synthetase II; long-chain acyl-coenzyme A synthetase; long-chain fatty acid:CoA ligase (AMP-forming); long-chain fatty acyl coenzyme A synthetase; long-chain-fatty-acid-CoA ligase; oleoyl-CoA synthetase; palmitoyl coenzyme A synthetase; palmitoyl-CoA ligase; palmitoyl-CoA synthase; pristanoyl-CoA synthetase; stearoyl-CoA synthetase; thiokinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00214[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00134[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00134;cpd00214',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Palmitate[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) Palmitoyl-CoA[0]',\n",
       "  'deltag': -2.63,\n",
       "  'deltagerr': 1.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.3'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00214[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00134[0]',\n",
       "  'id': 'rxn00947',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30377;rxn31929;rxn31932;rxn37332;rxn37337',\n",
       "  'name': 'Palmitate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation (Activation); Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); Alkanes-Biosynthesis (Alkane Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fatty-acid-biosynthesis (Fatty Acid Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); PWY-5143 (long-chain fatty acid activation); PWY-5156 (superpathway of fatty acid biosynthesis II (plant)); PWY-5885 (wax esters biosynthesis II); PWY-5971 (palmitate biosynthesis II (bacteria and plants)); PWY-5972 (stearate biosynthesis I (animals and fungi)); PWY-6113 (superpathway of mycolate biosynthesis); PWY-6285 (superpathway of fatty acids biosynthesis (<i>E. coli</i>)); PWY-6803 (phosphatidylcholine acyl editing); PWY-6873 (long chain fatty acid ester synthesis (engineered)); PWY-7033 (alkane biosynthesis II); Palmitate-Biosynthesis (Palmitate Biosynthesis); Stearate-Biosynthesis (Stearate Biosynthesis)',\n",
       "   'KEGG: rn00061 (Fatty acid biosynthesis); rn00071 (Fatty acid degradation); rn01212 (Fatty acid metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00214:0:0:\"Palmitate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\";1:cpd00134:0:0:\"Palmitoyl-CoA\"'},\n",
       " {'abbreviation': 'R01281',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: SERINE-C-PALMITOYLTRANSFERASE-RXN',\n",
       "   'BiGG: LCB1; LCB2; SERPT',\n",
       "   'BrachyCyc: SERINE-C-PALMITOYLTRANSFERASE-RXN',\n",
       "   'KEGG: R01281',\n",
       "   'MetaCyc: SERINE-C-PALMITOYLTRANSFERASE-RXN',\n",
       "   'Name: 3-oxosphinganine synthetase; Palmitoyl-CoA:L-serine C-palmitoyltransferase (decarboxylating); SPT; Serine palmitoyltransferase 1; acyl-CoA:serine C-2 acyltransferase decarboxylating; palmitoyl-CoA:L-serine C-palmitoyltransferase (decarboxylating); serine C-palmitoyltransferase; serine palmitoyltransferase'],\n",
       "  'code': '(1) cpd00054[0] + (1) cpd00134[0] <=> (1) cpd00010[0] + (1) cpd00011[0] + (1) cpd01879[0]',\n",
       "  'compound_ids': 'cpd00010;cpd00011;cpd00054;cpd00067;cpd00134;cpd01879',\n",
       "  'definition': '(1) L-Serine[0] + (1) H+[0] + (1) Palmitoyl-CoA[0] => (1) CoA[0] + (1) CO2[0] + (1) 3-Dehydrosphinganine[0]',\n",
       "  'deltag': 1.74,\n",
       "  'deltagerr': 0.94,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.3.1.50'],\n",
       "  'equation': '(1) cpd00054[0] + (1) cpd00067[0] + (1) cpd00134[0] => (1) cpd00010[0] + (1) cpd00011[0] + (1) cpd01879[0]',\n",
       "  'id': 'rxn00948',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27106;rxn27107',\n",
       "  'name': 'Palmitoyl-CoA:L-serine C-palmitoyltransferase (decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); PWY-5129 (sphingolipid biosynthesis (plants)); PWY-7277 (sphingolipid biosynthesis (mammals)); PWY3DJ-12 (ceramide <i>de novo</i> biosynthesis); SPHINGOLIPID-SYN-PWY (sphingolipid biosynthesis (yeast)); Sphingolipid-Biosynthesis (Sphingolipid Biosynthesis)',\n",
       "   'KEGG: rn00600 (Sphingolipid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00054:0:0:\"L-Serine\";-1:cpd00067:0:0:\"H+\";-1:cpd00134:0:0:\"Palmitoyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00011:0:0:\"CO2\";1:cpd01879:0:0:\"3-Dehydrosphinganine\"'},\n",
       " {'abbreviation': 'R01283',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01283',\n",
       "   'MetaCyc: HOMOCYSTEINE-DESULFHYDRASE-RXN',\n",
       "   'iAG612: rbs_0',\n",
       "   'iRS1563: R_R01283_c',\n",
       "   'Name: L-Homocysteine hydrogen-sulfide-lyase (deaminating); L-homocysteine hydrogen-sulfide-lyase (deaminating); L-homocysteine hydrogen-sulfide-lyase (deaminating, 2-oxobutanoate-forming); L-homocysteine hydrogen-sulfide-lyase (deaminating; 2-oxobutanoate-forming); homocysteine desulfhydrase; homocysteine desulfurase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00135[0] <=> (1) cpd00013[0] + (1) cpd00094[0] + (1) cpd00239[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00067;cpd00094;cpd00135;cpd00239',\n",
       "  'definition': '(1) H2O[0] + (1) Homocysteine[0] => (1) NH3[0] + (1) H+[0] + (1) 2-Oxobutyrate[0] + (1) H2S[0]',\n",
       "  'deltag': -1.14,\n",
       "  'deltagerr': 1.36,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.4.1.2'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00135[0] => (1) cpd00013[0] + (1) cpd00067[0] + (1) cpd00094[0] + (1) cpd00239[0]',\n",
       "  'id': 'rxn00949',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-homocysteine hydrogen-sulfide-lyase (deaminating; 2-oxobutanoate-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Hydrogen-Sulfide-Biosynthesis (Hydrogen Sulfide Biosynthesis); Metabolic-Clusters (Metabolic Clusters); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY66-426 (hydrogen sulfide biosynthesis II (mammalian)); Sulfur-Metabolism (Sulfur Compound Metabolism)',\n",
       "   'KEGG: rn00920 (Sulfur metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00135:0:0:\"Homocysteine\";1:cpd00013:0:0:\"NH3\";1:cpd00067:0:0:\"H+\";1:cpd00094:0:0:\"2-Oxobutyrate\";1:cpd00239:0:0:\"H2S\"'},\n",
       " {'abbreviation': 'R01285',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CYSTLr; YFR055W',\n",
       "   'KEGG: R01285',\n",
       "   'TS_Athaliana: rxn229_p; rxn638_p',\n",
       "   'iAG612: rbs_212; rbs_213',\n",
       "   'iAO358: rll_248',\n",
       "   'Name: Cystathionine L-homocysteine-lyase (deaminating); Cystathionine beta-lyase; L-Cystathionine L-homocysteine-lyase (deaminating)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00424[0] <=> (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00135[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00013;cpd00020;cpd00135;cpd00424',\n",
       "  'definition': '(1) H2O[0] + (1) Cystathionine[0] => (1) NH3[0] + (1) Pyruvate[0] + (1) Homocysteine[0]',\n",
       "  'deltag': 0.85,\n",
       "  'deltagerr': 0.65,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.4.1.8'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00424[0] => (1) cpd00013[0] + (1) cpd00020[0] + (1) cpd00135[0]',\n",
       "  'id': 'rxn00950',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33157',\n",
       "  'name': 'Cystathionine L-homocysteine-lyase (deaminating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00424:0:0:\"Cystathionine\";1:cpd00013:0:0:\"NH3\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00135:0:0:\"Homocysteine\"'},\n",
       " {'abbreviation': 'R01287',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: AHSERL2; MET17_2; MET17_3; METACH',\n",
       "   'KEGG: R01287',\n",
       "   'MetaCyc: ACETYLHOMOSER-CYS-RXN',\n",
       "   'PlantCyc: ACETYLHOMOSER-CYS-RXN',\n",
       "   'RiceCyc: ACETYLHOMOSER-CYS-RXN',\n",
       "   'Name: O-acetyl-L-homoserine acetate-lyase (adding hydrogen sulfide); O-acetyl-L-homoserine:hydrogen sulfide S-(3-amino-3-carboxypropyl)transferase; O-acetylhomoserine (thiol)-lyase; O-acetylhomoserine-thiol-lyase; o-acetylhomoserine sulfhydrolase'],\n",
       "  'code': '(1) cpd00239[0] + (1) cpd00790[0] <=> (1) cpd00029[0] + (1) cpd00135[0]',\n",
       "  'compound_ids': 'cpd00029;cpd00135;cpd00239;cpd00790',\n",
       "  'definition': '(1) H2S[0] + (1) O-Acetyl-L-homoserine[0] <=> (1) Acetate[0] + (1) Homocysteine[0]',\n",
       "  'deltag': -14.73,\n",
       "  'deltagerr': 1.19,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.5.1.49', '4.2.99.10'],\n",
       "  'equation': '(1) cpd00239[0] + (1) cpd00790[0] <=> (1) cpd00029[0] + (1) cpd00135[0]',\n",
       "  'id': 'rxn00952',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn35281',\n",
       "  'name': 'O-acetyl-L-homoserine:hydrogen sulfide S-(3-amino-3-carboxypropyl)transferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); CYSTEINE-SYN (L-cysteine Biosynthesis); HSERMETANA-PWY (L-methionine biosynthesis III); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); METHIONINE-SYN (L-methionine Biosynthesis); Methionine-De-novo-Biosynthesis (L-methionine <i>De Novo</i> Biosynthesis); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PWY-5344 (L-homocysteine biosynthesis); PWY-5345 (superpathway of L-methionine biosynthesis (by sulfhydrylation)); PWY-6293 (superpathway of L-cysteine biosynthesis (fungi)); PWY-821 (superpathway of sulfur amino acid biosynthesis (<i>Saccharomyces cerevisiae</i>))',\n",
       "   'KEGG: rn00270 (Cysteine and methionine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00239:0:0:\"H2S\";-1:cpd00790:0:0:\"O-Acetyl-L-homoserine\";1:cpd00029:0:0:\"Acetate\";1:cpd00135:0:0:\"Homocysteine\"'},\n",
       " {'abbreviation': 'R01289',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CYSTS',\n",
       "   'KEGG: R01289',\n",
       "   'TS_Athaliana: rxn230_c; rxn230_v',\n",
       "   'iAO358: rll_591',\n",
       "   'iMEO21: CYSTS',\n",
       "   'Name: Cystathionine beta-synthase; L-Serine hydro-lyase (adding homocysteine); cystathionine b-synthase'],\n",
       "  'code': '(1) cpd00054[0] + (1) cpd00135[0] <=> (1) cpd00001[0] + (1) cpd00424[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00054;cpd00135;cpd00424',\n",
       "  'definition': '(1) L-Serine[0] + (1) Homocysteine[0] <=> (1) H2O[0] + (1) Cystathionine[0]',\n",
       "  'deltag': -7.79,\n",
       "  'deltagerr': 0.66,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.22'],\n",
       "  'equation': '(1) cpd00054[0] + (1) cpd00135[0] <=> (1) cpd00001[0] + (1) cpd00424[0]',\n",
       "  'id': 'rxn00953',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32102',\n",
       "  'name': 'L-Serine hydro-lyase (adding homocysteine)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00054:0:0:\"L-Serine\";-1:cpd00135:0:0:\"Homocysteine\";1:cpd00001:0:0:\"H2O\";1:cpd00424:0:0:\"Cystathionine\"'},\n",
       " {'abbreviation': 'RHCYS',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['iYO844: RHCYS',\n",
       "   'Name: 5-deoxyribos-5-ylhomocysteinase; S-(5-deoxy-D-ribos-5-yl)-L-homocysteine homocysteine-lyase; [(4S)-4,5-dihydroxypentan-2,3-dione-forming]'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd02227[0] <=> (1) cpd00105[0] + (1) cpd00135[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00105;cpd00135;cpd02227',\n",
       "  'definition': '(1) H2O[0] + (1) S-Ribosylhomocysteine[0] <=> (1) D-Ribose[0] + (1) Homocysteine[0]',\n",
       "  'deltag': 11.5,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.4.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd02227[0] <=> (1) cpd00105[0] + (1) cpd00135[0]',\n",
       "  'id': 'rxn00955',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-(5-deoxy-D-ribos-5-yl)-L-homocysteine homocysteine-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '<',\n",
       "  'source': 'Published Model',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd02227:0:0:\"S-Ribosylhomocysteine\";1:cpd00105:0:0:\"D-Ribose\";1:cpd00135:0:0:\"Homocysteine\"'},\n",
       " {'abbreviation': 'R01110',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01110; R01292',\n",
       "   'MetaCyc: 1.8.4.1-RXN',\n",
       "   'Name: Glutathione:homocystine oxidoreductase; glutathione-homocystine transhydrogenase; glutathione:homocystine oxidoreductase'],\n",
       "  'code': '(2) cpd00042[0] + (1) cpd01251[0] <=> (1) cpd00111[0] + (2) cpd00135[0]',\n",
       "  'compound_ids': 'cpd00042;cpd00111;cpd00135;cpd01251',\n",
       "  'definition': '(2) GSH[0] + (1) Homocystine[0] <=> (1) Oxidized glutathione[0] + (2) Homocysteine[0]',\n",
       "  'deltag': -0.1,\n",
       "  'deltagerr': 0.53,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.8.4.1'],\n",
       "  'equation': '(2) cpd00042[0] + (1) cpd01251[0] <=> (1) cpd00111[0] + (2) cpd00135[0]',\n",
       "  'id': 'rxn00956',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn15168',\n",
       "  'name': 'glutathione:homocystine oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00270 (Cysteine and methionine metabolism); rn00480 (Glutathione metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-2:cpd00042:0:0:\"GSH\";-1:cpd01251:0:0:\"Homocystine\";1:cpd00111:0:0:\"Oxidized glutathione\";2:cpd00135:0:0:\"Homocysteine\"'},\n",
       " {'abbreviation': 'R01293',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: VNDH-2',\n",
       "   'KEGG: R01293',\n",
       "   'MetaCyc: HYDROXYBENZALDEHYDE-OXIDATION-NAD-RXN; RXN-13394',\n",
       "   'PlantCyc: HYDROXYBENZALDEHYDE-OXIDATION-NAD-RXN',\n",
       "   'iAbaylyiv4: HYDROXYBENZALDEHYDE-OXIDATION-NAD-RXN',\n",
       "   'Name: 3-hydroxybenzaldehyde:NAD+ oxidoreductase; 4-hydroxybenzaldehyde dehydrogenase; 4-hydroxybenzaldehyde:NAD+ oxidoreductase; hydroxybenzaldehyde dehydrogenase; p-hydroxybenzaldehyde dehydrogenase; vanillin:NAD oxidoreductase (4hbald)'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00484[0] <=> (1) cpd00004[0] + (1) cpd00136[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00136;cpd00484',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) p-Hydroxybenzaldehyde[0] <=> (1) NADH[0] + (2) H+[0] + (1) 4-Hydroxybenzoate[0]',\n",
       "  'deltag': -8.29,\n",
       "  'deltagerr': 0.26,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.28', '1.2.1.64'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00484[0] <=> (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00136[0]',\n",
       "  'id': 'rxn00957',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxybenzaldehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-HYDROXYMANDELATE-DEGRADATION-PWY (4-hydroxymandelate degradation); 4-Hydroxybenzoate-Biosynthesis (4-Hydroxybenzoate Biosynthesis); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); PHENYLPROPANOID-DERIVATIVE-DEG (Phenylpropanoid Derivative Degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5754 (4-hydroxybenzoate biosynthesis I (eukaryotes)); PWY-6431 (4-hydroxybenzoate biosynthesis IV (plants)); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-7046 (4-coumarate degradation (anaerobic)); PWY-8002 (4-coumarate degradation (aerobic)); Phenolic-Compounds-Degradation (Phenolic Compound Degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation)',\n",
       "   'KEGG: rn00623 (Toluene degradation); rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00484:0:0:\"p-Hydroxybenzaldehyde\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd00136:0:0:\"4-Hydroxybenzoate\"'},\n",
       " {'abbreviation': 'R01294',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01294',\n",
       "   'MetaCyc: HYDROXYBENZALDEHYDE-OXIDATION-NADP-RXN; RXN-13394',\n",
       "   'Name: 4-Hydroxybenzaldehyde:NADP+ oxidoreductase; NADP-linked benzaldehyde dehydrogenase; benzaldehyde dehydrogenase (NADP); benzaldehyde dehydrogenase (NADP+); benzaldehyde:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00484[0] <=> (1) cpd00005[0] + (1) cpd00136[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00067;cpd00136;cpd00484',\n",
       "  'definition': '(1) H2O[0] + (1) NADP[0] + (1) p-Hydroxybenzaldehyde[0] <=> (1) NADPH[0] + (2) H+[0] + (1) 4-Hydroxybenzoate[0]',\n",
       "  'deltag': -8.29,\n",
       "  'deltagerr': 0.27,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.7'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00484[0] <=> (1) cpd00005[0] + (2) cpd00067[0] + (1) cpd00136[0]',\n",
       "  'id': 'rxn00958',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-Hydroxybenzaldehyde:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-HYDROXYMANDELATE-DEGRADATION-PWY (4-hydroxymandelate degradation); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); PHENYLPROPANOID-DERIVATIVE-DEG (Phenylpropanoid Derivative Degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5181 (toluene degradation III (aerobic) (via <i>p</i>-cresol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-7046 (4-coumarate degradation (anaerobic)); PWY-7700 (4-methylphenol degradation to protocatechuate); PWY-8002 (4-coumarate degradation (aerobic)); Phenolic-Compounds-Degradation (Phenolic Compound Degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); TOLUENE-DEG (Toluene Degradation)',\n",
       "   'KEGG: rn00623 (Toluene degradation); rn00627 (Aminobenzoate degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00006:0:0:\"NADP\";-1:cpd00484:0:0:\"p-Hydroxybenzaldehyde\";1:cpd00005:0:0:\"NADPH\";2:cpd00067:0:0:\"H+\";1:cpd00136:0:0:\"4-Hydroxybenzoate\"'},\n",
       " {'abbreviation': 'R01295',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01295',\n",
       "   'MetaCyc: BENZOATE-4-MONOOXYGENASE-RXN',\n",
       "   'Name: benzoate 4-hydroxylase; benzoate 4-monooxygenase; benzoate,NADPH:oxygen oxidoreductase (4-hydroxylating); benzoate,NADPH:oxygen oxidoreductase(4-hydroxylating); benzoate-p-hydroxylase; benzoic 4-hydroxylase; benzoic acid 4-hydroxylase; p-hydroxybenzoate hydroxylase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00153[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00136[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00067;cpd00136;cpd00153',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) H+[0] + (1) Benzoate[0] => (1) H2O[0] + (1) NADP[0] + (1) 4-Hydroxybenzoate[0]',\n",
       "  'deltag': -105.97,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.12'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00067[0] + (1) cpd00153[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00136[0]',\n",
       "  'id': 'rxn00959',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19225',\n",
       "  'name': 'benzoate,NADPH:oxygen oxidoreductase (4-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation); rn00627 (Aminobenzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00067:0:0:\"H+\";-1:cpd00153:0:0:\"Benzoate\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00136:0:0:\"4-Hydroxybenzoate\"'},\n",
       " {'abbreviation': 'R01296',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01296',\n",
       "   'MetaCyc: 1.14.13.33-RXN',\n",
       "   'Name: 4-hydroxybenzoate 3-hydroxylase; 4-hydroxybenzoate 3-monooxygenase (reduced nicotinamide adenine dinucleotide (phosphate)); 4-hydroxybenzoate 3-monooxygenase [NAD(P)H]; 4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (3-hydroxylating); 4-hydroxybenzoate,NADH:oxygen oxidoreductase (3-hydroxylating); 4-hydroxybenzoate-3-hydroxylase'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00136[0] <=> (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00197[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00007;cpd00067;cpd00136;cpd00197',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (1) H+[0] + (1) 4-Hydroxybenzoate[0] => (1) H2O[0] + (1) NAD[0] + (1) Protocatechuate[0]',\n",
       "  'deltag': -105.96,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.33'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00067[0] + (1) cpd00136[0] => (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00197[0]',\n",
       "  'id': 'rxn00960',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxybenzoate,NADH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-1:cpd00067:0:0:\"H+\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\";1:cpd00197:0:0:\"Protocatechuate\"'},\n",
       " {'abbreviation': 'R01297',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01297',\n",
       "   'MetaCyc: 1.14.13.64-RXN',\n",
       "   'Name: 4-hydroxybenzoate 1-hydroxylase; 4-hydroxybenzoate 1-monooxygenase; 4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating, decarboxylating); 4-hydroxybenzoate,NADH:oxygen oxidoreductase (1-hydroxylating,; 4-hydroxybenzoate,NADH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)'],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00007[0] + (1) cpd00136[0] <=> (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00415[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00007;cpd00011;cpd00067;cpd00136;cpd00415',\n",
       "  'definition': '(1) NADH[0] + (1) O2[0] + (2) H+[0] + (1) 4-Hydroxybenzoate[0] => (1) H2O[0] + (1) NAD[0] + (1) CO2[0] + (1) Quinol[0]',\n",
       "  'deltag': -108.62,\n",
       "  'deltagerr': 1.19,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.64'],\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00007[0] + (2) cpd00067[0] + (1) cpd00136[0] => (1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00011[0] + (1) cpd00415[0]',\n",
       "  'id': 'rxn00961',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxybenzoate,NADH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\";1:cpd00011:0:0:\"CO2\";1:cpd00415:0:0:\"Quinol\"'},\n",
       " {'abbreviation': 'R01298',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: 4HBHYOX; POBA',\n",
       "   'KEGG: R01298',\n",
       "   'MetaCyc: 1.14.13.33-RXN; 4-HYDROXYBENZOATE-3-MONOOXYGENASE-RXN',\n",
       "   'iAbaylyiv4: 4-HYDROXYBENZOATE-3-MONOOXYGENASE-RXN',\n",
       "   'iJN746: 4HBHYOX',\n",
       "   'Name: 4-hydroxybenzoate 3-hydroxylase; 4-hydroxybenzoate 3-monooxygenase; 4-hydroxybenzoate monooxygenase; 4-hydroxybenzoate,NADPH:oxygen oxidoreductase (3-hydroxylating); 4-hydroxybenzoic hydroxylase; p-hydroxybenzoate hydrolyase; p-hydroxybenzoate hydroxylase; p-hydroxybenzoate-3-hydroxylase; p-hydroxybenzoic acid hydrolase; p-hydroxybenzoic acid hydroxylase; p-hydroxybenzoic hydroxylase'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00136[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00197[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00067;cpd00136;cpd00197',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (1) H+[0] + (1) 4-Hydroxybenzoate[0] => (1) H2O[0] + (1) NADP[0] + (1) Protocatechuate[0]',\n",
       "  'deltag': -105.96,\n",
       "  'deltagerr': 0.91,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.2', '1.14.13.33'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00067[0] + (1) cpd00136[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00197[0]',\n",
       "  'id': 'rxn00962',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxybenzoate,NADPH:oxygen oxidoreductase (3-hydroxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-HYDROXYMANDELATE-DEGRADATION-PWY (4-hydroxymandelate degradation); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); CHLORINATED-COMPOUNDS-DEG (Chlorinated Compound Degradation); Chloroaromatic-Compounds-Degradation (Chloroaromatic Compound Degradation); Chlorobenzoate-Degradation (Chlorobenzoate Degradation); Degradation (Degradation/Utilization/Assimilation); Halogenated-Aromatics-Biosynthesis (Halogenated Aromatic Compound Biosynthesis); PHENYLPROPANOID-DERIVATIVE-DEG (Phenylpropanoid Derivative Degradation); PWY-2504 (superpathway of aromatic compound degradation via 3-oxoadipate); PWY-5181 (toluene degradation III (aerobic) (via <i>p</i>-cresol)); PWY-5183 (superpathway of aerobic toluene degradation); PWY-6215 (4-chlorobenzoate degradation); PWY-6954 (superpathway of aromatic compound degradation via 2-hydroxypentadienoate); PWY-7700 (4-methylphenol degradation to protocatechuate); PWY-7757 (bisphenol A degradation); PWY-7934 (polybrominated dihydroxylated diphenyl ethers biosynthesis); PWY-7935 (spongiadioxin C biosynthesis); PWY-8002 (4-coumarate degradation (aerobic)); Phenolic-Compounds-Degradation (Phenolic Compound Degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); TOLUENE-DEG (Toluene Degradation)',\n",
       "   'KEGG: rn00362 (Benzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-1:cpd00067:0:0:\"H+\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00197:0:0:\"Protocatechuate\"'},\n",
       " {'abbreviation': 'R01299',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01299',\n",
       "   'MetaCyc: 1.14.13.64-RXN',\n",
       "   'Name: 4-hydroxybenzoate 1-hydroxylase; 4-hydroxybenzoate 1-monooxygenase; 4-hydroxybenzoate,NAD(P)H:oxygen oxidoreductase (1-hydroxylating, decarboxylating); 4-hydroxybenzoate,NADPH:oxygen oxidoreductase (1-hydroxylating,; 4-hydroxybenzoate,NADPH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)'],\n",
       "  'code': '(1) cpd00005[0] + (1) cpd00007[0] + (1) cpd00136[0] <=> (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00011[0] + (1) cpd00415[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00005;cpd00006;cpd00007;cpd00011;cpd00067;cpd00136;cpd00415',\n",
       "  'definition': '(1) NADPH[0] + (1) O2[0] + (2) H+[0] + (1) 4-Hydroxybenzoate[0] => (1) H2O[0] + (1) NADP[0] + (1) CO2[0] + (1) Quinol[0]',\n",
       "  'deltag': -108.62,\n",
       "  'deltagerr': 1.2,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.14.13.64'],\n",
       "  'equation': '(1) cpd00005[0] + (1) cpd00007[0] + (2) cpd00067[0] + (1) cpd00136[0] => (1) cpd00001[0] + (1) cpd00006[0] + (1) cpd00011[0] + (1) cpd00415[0]',\n",
       "  'id': 'rxn00963',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-hydroxybenzoate,NADPH:oxygen oxidoreductase (1-hydroxylating, decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00362 (Benzoate degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00005:0:0:\"NADPH\";-1:cpd00007:0:0:\"O2\";-2:cpd00067:0:0:\"H+\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00001:0:0:\"H2O\";1:cpd00006:0:0:\"NADP\";1:cpd00011:0:0:\"CO2\";1:cpd00415:0:0:\"Quinol\"'},\n",
       " {'abbreviation': 'R01300',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01300',\n",
       "   'MetaCyc: 4-HYDROXYBENZOATE--COA-LIGASE-RXN',\n",
       "   'Name: 4-Hydroxybenzoate:CoA ligase (AMP-forming); 4-hydroxybenzoate-CoA ligase; 4-hydroxybenzoate-CoA synthetase; 4-hydroxybenzoate-coenzyme A ligase (AMP-forming); 4-hydroxybenzoate:CoA ligase (AMP-forming); 4-hydroxybenzoyl coenzyme A synthetase; 4-hydroxybenzoyl-CoA ligase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00136[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd01892[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00136;cpd01892',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) 4-Hydroxybenzoate[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) 4-Hydroxybenzoyl-CoA[0]',\n",
       "  'deltag': -2.89,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.27'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00136[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd01892[0]',\n",
       "  'id': 'rxn00964',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-Hydroxybenzoate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); BENZCOA-PWY (anaerobic aromatic compound degradation (<i>Thauera aromatica</i>)); Degradation (Degradation/Utilization/Assimilation); PHENOLDEG-PWY (phenol degradation II (anaerobic)); PHENYLPROPANOID-DERIVATIVE-DEG (Phenylpropanoid Derivative Degradation); PWY-7046 (4-coumarate degradation (anaerobic)); Phenol-Degradation (Phenol Degradation); Phenolic-Compounds-Degradation (Phenolic Compound Degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation)',\n",
       "   'KEGG: rn00362 (Benzoate degradation); rn00627 (Aminobenzoate degradation); rn01220 (Degradation of aromatic compounds)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:H+;1:cpd01892:0:0:\"4-Hydroxybenzoyl-CoA\"'},\n",
       " {'abbreviation': 'R01301',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3.1.2.23-RXN',\n",
       "   'BiGG: 4HBTE',\n",
       "   'BrachyCyc: 3.1.2.23-RXN',\n",
       "   'KEGG: R01301',\n",
       "   'MetaCyc: 3.1.2.23-RXN',\n",
       "   'Name: 4-Hydroxybenzoyl-CoA hydrolase; 4-hydroxybenzoyl-CoA hydrolase; 4-hydroxybenzoyl-CoA thioesterase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01892[0] <=> (1) cpd00010[0] + (1) cpd00136[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00067;cpd00136;cpd01892',\n",
       "  'definition': '(1) H2O[0] + (1) 4-Hydroxybenzoyl-CoA[0] => (1) CoA[0] + (1) H+[0] + (1) 4-Hydroxybenzoate[0]',\n",
       "  'deltag': -5.54,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.23'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01892[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00136[0]',\n",
       "  'id': 'rxn00965',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-Hydroxybenzoyl-CoA hydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Hydroxybenzoate-Biosynthesis (4-Hydroxybenzoate Biosynthesis); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); CHLORINATED-COMPOUNDS-DEG (Chlorinated Compound Degradation); Chloroaromatic-Compounds-Degradation (Chloroaromatic Compound Degradation); Chlorobenzoate-Degradation (Chlorobenzoate Degradation); Degradation (Degradation/Utilization/Assimilation); PWY-6215 (4-chlorobenzoate degradation); PWY-6435 (4-hydroxybenzoate biosynthesis III (plants))',\n",
       "   'KEGG: rn00130 (Ubiquinone and other terpenoid-quinone biosynthesis); rn00362 (Benzoate degradation); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01892:0:0:\"4-Hydroxybenzoyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd00136:0:0:\"4-Hydroxybenzoate\"'},\n",
       " {'abbreviation': 'R01302',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: CHRPL; TRP2_2; TRP3_3',\n",
       "   'ChlamyCyc: CHORPYRLY-RXN',\n",
       "   'EcoCyc: CHORPYRLY-RXN',\n",
       "   'KEGG: R01302',\n",
       "   'MetaCyc: CHORPYRLY-RXN',\n",
       "   'Name: CL; CPL; Chorismate pyruvate lyase; UbiC; chorismate lyase; chorismate pyruvate-lyase (4-hydroxybenzoate-forming)'],\n",
       "  'code': '(1) cpd00020[0] + (1) cpd00136[0] <=> (1) cpd00216[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00136;cpd00216',\n",
       "  'definition': '(1) Pyruvate[0] + (1) 4-Hydroxybenzoate[0] <= (1) Chorismate[0]',\n",
       "  'deltag': 26.91,\n",
       "  'deltagerr': 1.75,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['4.1.3.40'],\n",
       "  'equation': '(1) cpd00020[0] + (1) cpd00136[0] <= (1) cpd00216[0]',\n",
       "  'id': 'rxn00966',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'chorismate pyruvate-lyase (4-hydroxybenzoate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: 4-Hydroxybenzoate-Biosynthesis (4-Hydroxybenzoate Biosynthesis); ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); AROMATIC-COMPOUNDS-BIOSYN (Aromatic Compound Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Halogenated-Aromatics-Biosynthesis (Halogenated Aromatic Compound Biosynthesis); Lipid-Biosynthesis (Fatty Acid and Lipid Biosynthesis); PWY-5755 (4-hydroxybenzoate biosynthesis II (bacteria)); PWY-5870 (ubiquinol-8 biosynthesis (eukaryotic)); PWY-6148 (tetrahydromethanopterin biosynthesis); PWY-7742 (phenolphthiocerol biosynthesis); PWY-7745 (p-HBAD biosynthesis); PWY-7929 (polybrominated phenols biosynthesis); PWY-7933 (superpathway of polybrominated aromatic compound biosynthesis); PWY-7934 (polybrominated dihydroxylated diphenyl ethers biosynthesis); PWY-7935 (spongiadioxin C biosynthesis); Quinone-Biosynthesis (Quinol and Quinone Biosynthesis); UBISYN-PWY (superpathway of ubiquinol-8 biosynthesis (prokaryotic)); Ubiquinone-Biosynthesis (Ubiquinol Biosynthesis)',\n",
       "   'KEGG: rn00130 (Ubiquinone and other terpenoid-quinone biosynthesis); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00020:0:0:\"Pyruvate\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00216:0:0:\"Chorismate\"'},\n",
       " {'abbreviation': 'R01304',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01304',\n",
       "   'MetaCyc: 2.4.1.194-RXN',\n",
       "   'Name: 4-hydroxybenzoate 4-O-beta-D-glucosyltransferase; HBA glucosyltransferase; PHB glucosyltransferase; PHB-O-glucosyltransferase; UDP-glucose:4-(beta-D-glucopyranosyloxy)benzoic acid glucosyltransferase; UDP-glucose:4-hydroxybenzoate 4-O-beta-D-glucosyltransferase; UDPglucose:4-hydroxybenzoate 4-O-beta-D-glucosyltransferase; p-hydroxybenzoate glucosyltransferase; uridine diphosphoglucose-4-hydroxybenzoate glucosyltransferase'],\n",
       "  'code': '(1) cpd00026[0] + (1) cpd00136[0] <=> (1) cpd00014[0] + (1) cpd02476[0]',\n",
       "  'compound_ids': 'cpd00014;cpd00026;cpd00136;cpd02476',\n",
       "  'definition': '(1) UDP-glucose[0] + (1) 4-Hydroxybenzoate[0] <=> (1) UDP[0] + (1) 4-(beta-D-Glucosyloxy)benzoate[0]',\n",
       "  'deltag': -13.2,\n",
       "  'deltagerr': 1.07,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.4.1.194'],\n",
       "  'equation': '(1) cpd00026[0] + (1) cpd00136[0] <=> (1) cpd00014[0] + (1) cpd02476[0]',\n",
       "  'id': 'rxn00967',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-glucose:4-hydroxybenzoate 4-O-beta-D-glucosyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00130 (Ubiquinone and other terpenoid-quinone biosynthesis)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00026:0:0:\"UDP-glucose\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";1:cpd00014:0:0:\"UDP\";1:cpd02476:0:0:\"4-(beta-D-Glucosyloxy)benzoate\"'},\n",
       " {'abbreviation': 'R01305',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01305',\n",
       "   'MetaCyc: 1.13.11.41-RXN',\n",
       "   \"Name: (4-hydroxybenzoyl)methanol oxygenase; 2,4'-dihydroxyacetophenone dioxygenase; 2,4'-dihydroxyacetophenone oxidoreductase (C-C-bond-cleaving)\"],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd02300[0] <=> (1) cpd00047[0] + (1) cpd00136[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00047;cpd00067;cpd00136;cpd02300',\n",
       "  'definition': '(1) O2[0] + (1) Resacetophenone[0] => (1) Formate[0] + (2) H+[0] + (1) 4-Hydroxybenzoate[0]',\n",
       "  'deltag': -118.45,\n",
       "  'deltagerr': 1.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.41'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd02300[0] => (1) cpd00047[0] + (2) cpd00067[0] + (1) cpd00136[0]',\n",
       "  'id': 'rxn00968',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"2,4'-dihydroxyacetophenone oxidoreductase (C-C-bond-cleaving)\",\n",
       "  'notes': ['GCC', 'EQU', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00363 (Bisphenol degradation)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd02300:0:0:\"Resacetophenone\";1:cpd00047:0:0:\"Formate\";2:cpd00067:0:0:\"H+\";1:cpd00136:0:0:\"4-Hydroxybenzoate\"'},\n",
       " {'abbreviation': '4-Chlorobenzoate chlorohydrolase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['Name: 4-Chlorobenzoate chlorohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01582[0] <=> (1) cpd00136[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00136;cpd01582',\n",
       "  'definition': '(1) H2O[0] + (1) 4-Chlorobenzoate[0] <=> (1) H+[0] + (1) 4-Hydroxybenzoate[0]',\n",
       "  'deltag': 4.6,\n",
       "  'deltagerr': 0.98,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.8.1.6'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01582[0] <=> (1) cpd00067[0] + (1) cpd00136[0]',\n",
       "  'id': 'rxn00969',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '4-Chlorobenzoate chlorohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'MI:Cl:-1|CI:1',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01582:0:0:\"4-Chlorobenzoate\";1:cpd00067:0:0:\"H+\";1:cpd00136:0:0:\"4-Hydroxybenzoate\"'},\n",
       " {'abbreviation': 'R01308',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01308', 'iRS1563: R_R01308_c', 'Name: '],\n",
       "  'code': '(1) cpd00004[0] + (1) cpd00029[0] + (1) cpd00136[0] <=> (2) cpd00001[0] + (1) cpd00003[0] + (1) cpd00604[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00029;cpd00067;cpd00136;cpd00604',\n",
       "  'definition': '(1) NADH[0] + (1) Acetate[0] + (2) H+[0] + (1) 4-Hydroxybenzoate[0] <= (2) H2O[0] + (1) NAD[0] + (1) 4-Coumarate[0]',\n",
       "  'deltag': 9.87,\n",
       "  'deltagerr': 0.48,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00004[0] + (1) cpd00029[0] + (2) cpd00067[0] + (1) cpd00136[0] <= (2) cpd00001[0] + (1) cpd00003[0] + (1) cpd00604[0]',\n",
       "  'id': 'rxn00970',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R01308',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00004:0:0:\"NADH\";-1:cpd00029:0:0:\"Acetate\";-2:cpd00067:0:0:\"H+\";-1:cpd00136:0:0:\"4-Hydroxybenzoate\";2:cpd00001:0:0:\"H2O\";1:cpd00003:0:0:\"NAD\";1:cpd00604:0:0:\"4-Coumarate\"'},\n",
       " {'abbreviation': 'R01322',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01322',\n",
       "   'MetaCyc: CITRATE--COA-LIGASE-RXN',\n",
       "   'Name: Citrate:CoA ligase (ADP-forming); citrate thiokinase; citrate-CoA ligase; citrate:CoA ligase; citrate:CoA ligase (ADP-forming); citryl-CoA synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00137[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00442[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00009;cpd00010;cpd00137;cpd00442',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Citrate[0] <=> (1) ADP[0] + (1) Phosphate[0] + (1) (3S)-Citryl-CoA[0]',\n",
       "  'deltag': -1.06,\n",
       "  'deltagerr': 6.63,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['6.2.1.18'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00137[0] <=> (1) cpd00008[0] + (1) cpd00009[0] + (1) cpd00442[0]',\n",
       "  'id': 'rxn00971',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Citrate:CoA ligase (ADP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': ['MetaCyc: Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); PWY-5392 (reductive TCA cycle II); Reductive-TCA-Cycles (Reductive TCA Cycles)',\n",
       "   'KEGG: rn00720 (Carbon fixation pathways in prokaryotes); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00137:0:0:\"Citrate\";1:cpd00008:0:0:\"ADP\";1:cpd00009:0:0:\"Phosphate\";1:cpd00442:0:0:\"(3S)-Citryl-CoA\"'},\n",
       " {'abbreviation': 'R01323',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01323',\n",
       "   'MetaCyc: 2.8.3.10-RXN',\n",
       "   'Name: Acetyl-CoA:citrate CoA-transferase; acetyl-CoA:citrate CoA-transferase; citrate CoA-transferase'],\n",
       "  'code': '(1) cpd00022[0] + (1) cpd00137[0] <=> (1) cpd00029[0] + (1) cpd00442[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00029;cpd00137;cpd00442',\n",
       "  'definition': '(1) Acetyl-CoA[0] + (1) Citrate[0] <=> (1) Acetate[0] + (1) (3S)-Citryl-CoA[0]',\n",
       "  'deltag': -1.54,\n",
       "  'deltagerr': 5.1,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.10'],\n",
       "  'equation': '(1) cpd00022[0] + (1) cpd00137[0] <=> (1) cpd00029[0] + (1) cpd00442[0]',\n",
       "  'id': 'rxn00972',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetyl-CoA:citrate CoA-transferase',\n",
       "  'notes': ['GCC', 'HB', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00022:0:0:\"Acetyl-CoA\";-1:cpd00137:0:0:\"Citrate\";1:cpd00029:0:0:\"Acetate\";1:cpd00442:0:0:\"(3S)-Citryl-CoA\"'},\n",
       " {'abbreviation': 'R01324',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01324_c; R_R01324_m',\n",
       "   'BiGG: ACO1; ACONT; ACONTm; YJL200C',\n",
       "   'JM_Creinhardtii: JM_Cre_45; JM_Cre_46',\n",
       "   'JP_Creinhardtii_MSB: R_ACONT; R_ACONTm; R_ACONTx',\n",
       "   'KEGG: R01324',\n",
       "   'Name: Aconitate hydratase; aconitase; citrate hydro-lyase; citrate hydroxymutase'],\n",
       "  'code': '(1) cpd00137[0] <=> (1) cpd00260[0]',\n",
       "  'compound_ids': 'cpd00137;cpd00260',\n",
       "  'definition': '(1) Citrate[0] <=> (1) Isocitrate[0]',\n",
       "  'deltag': 1.82,\n",
       "  'deltagerr': 0.08,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.3'],\n",
       "  'equation': '(1) cpd00137[0] <=> (1) cpd00260[0]',\n",
       "  'id': 'rxn00973',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn30082;rxn34190;rxn35518',\n",
       "  'name': 'citrate hydroxymutase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn01200 (Carbon metabolism); rn01210 (2-Oxocarboxylic acid metabolism); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00137:0:0:\"Citrate\";1:cpd00260:0:0:\"Isocitrate\"'},\n",
       " {'abbreviation': 'R01325',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACONITATEDEHYDR-RXN',\n",
       "   'BiGG: ACONTa',\n",
       "   'BrachyCyc: ACONITATEDEHYDR-RXN',\n",
       "   'KEGG: R01325',\n",
       "   'MetaCyc: ACONITATEDEHYDR-RXN; CITRATE-DEHYDRATASE-RXN',\n",
       "   'Name: 2-methylaconitate hydratase; AcnB; AconDHatase; Aconitate hydratase; aconitase; aconitase (citrate hydro-lyase); aconitase (half-reaction A, Citrate hydro-lyase); aconitate hydratase; aconitate hydratase, putative / citrate hydro-lyase/aconitase, putative; cis-aconitase; citrate hydro-lyase; citrate hydro-lyase (cis-aconitate-forming); citrate(isocitrate) hydro-lyase; citrate(isocitrate) hydro-lyase (cis-aconitate-forming)'],\n",
       "  'code': '(1) cpd00137[0] <=> (1) cpd00001[0] + (1) cpd00331[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00137;cpd00331',\n",
       "  'definition': '(1) Citrate[0] <=> (1) H2O[0] + (1) cis-Aconitate[0]',\n",
       "  'deltag': 1.99,\n",
       "  'deltagerr': 0.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.1.3', '4.2.1.4'],\n",
       "  'equation': '(1) cpd00137[0] <=> (1) cpd00001[0] + (1) cpd00331[0]',\n",
       "  'id': 'rxn00974',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19039;rxn30619;rxn31223;rxn33227;rxn33231',\n",
       "  'name': 'citrate hydro-lyase (cis-aconitate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Biosynthesis (Fermentation to Alcohols); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Autotrophic-CO2-Fixation (Autotrophic CO<sub>2</sub> Fixation); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); CO2-Fixation (CO<sub>2</sub> Fixation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Ethylene-Biosynthesis (Ethylene Biosynthesis); FERMENTATION-PWY (mixed acid fermentation); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); GLUTAMINE-SYN (L-glutamine Biosynthesis); GLYCOLYSIS-TCA-GLYOX-BYPASS (superpathway of glycolysis, pyruvate dehydrogenase, TCA, and glyoxylate bypass); GLYOXYLATE-BYPASS (glyoxylate cycle); HORMONE-SYN (Hormone Biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); P105-PWY (TCA cycle IV (2-oxoglutarate decarboxylase)); P23-PWY (reductive TCA cycle I); PWY-5392 (reductive TCA cycle II); PWY-5464 (superpathway of cytosolic glycolysis (plants), pyruvate dehydrogenase and TCA cycle); PWY-561 (superpathway of glyoxylate cycle and fatty acid degradation); PWY-5690 (TCA cycle II (plants and fungi)); PWY-5913 (partial TCA cycle (obligate autotrophs)); PWY-6549 (L-glutamine biosynthesis III); PWY-6728 (methylaspartate cycle); PWY-6969 (TCA cycle V (2-oxoglutarate:ferredoxin oxidoreductase)); PWY-7124 (ethylene biosynthesis V (engineered)); PWY-7254 (TCA cycle VII (acetate-producers)); PWY66-398 (TCA cycle III (animals)); Plant-Hormone-Biosynthesis (Plant Hormone Biosynthesis); Pyruvate-Degradation (Fermentation of Pyruvate); Pyruvate-Ethanol-Fermentation (Pyruvate Fermentation to Ehanol); REDCITCYC (TCA cycle VIII (helicobacter)); Reductive-TCA-Cycles (Reductive TCA Cycles); TCA (TCA cycle I (prokaryotic)); TCA-GLYOX-BYPASS (superpathway of glyoxylate bypass and TCA); TCA-VARIANTS (TCA cycle)',\n",
       "   'KEGG: rn00020 (Citrate cycle (TCA cycle)); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00720 (Carbon fixation pathways in prokaryotes); rn01110 (Biosynthesis of secondary metabolites); rn01210 (2-Oxocarboxylic acid metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00137:0:0:\"Citrate\";1:cpd00001:0:0:\"H2O\";1:cpd00331:0:0:\"cis-Aconitate\"'},\n",
       " {'abbreviation': 'R01326',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01326_c',\n",
       "   'AraGEM: R_R01326_c',\n",
       "   'BiGG: GLK1_2; HEX4; HXK1_3; HXK2_3',\n",
       "   'KEGG: R01326',\n",
       "   'MetaCyc: MANNKIN-RXN',\n",
       "   'Name: ATP:D-mannose 6-phosphotransferase; hexokinase (D-mannose:ATP)'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00138[0] <=> (1) cpd00008[0] + (1) cpd00235[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00067;cpd00138;cpd00235',\n",
       "  'definition': '(1) ATP[0] + (1) D-Mannose[0] => (1) ADP[0] + (1) H+[0] + (1) D-mannose-6-phosphate[0]',\n",
       "  'deltag': -3.05,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1', '2.7.1.7'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00138[0] => (1) cpd00008[0] + (1) cpd00067[0] + (1) cpd00235[0]',\n",
       "  'id': 'rxn00975',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32484;rxn32488',\n",
       "  'name': 'ATP:D-mannose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CMP-Sugar-Biosynthesis (CMP-sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Carbohydrates-Degradation (Carbohydrate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-3861 (mannitol degradation II); PWY-6140 (CMP-2-keto-3-deoxy-D-<i>glycero</i>-D-<i>galacto</i>-nononate biosynthesis); PWY-6145 (superpathway of CMP-sialic acids biosynthesis); PWY-6992 (1,5-anhydrofructose degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00051 (Fructose and mannose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00138:0:0:\"D-Mannose\";1:cpd00008:0:0:\"ADP\";1:cpd00067:0:0:\"H+\";1:cpd00235:0:0:\"D-mannose-6-phosphate\"'},\n",
       " {'abbreviation': 'R01327',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01327', 'Name: ITP:D-mannose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00068[0] + (1) cpd00138[0] <=> (1) cpd00090[0] + (1) cpd00235[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00068;cpd00090;cpd00138;cpd00235',\n",
       "  'definition': '(1) ITP[0] + (1) D-Mannose[0] => (1) H+[0] + (1) IDP[0] + (1) D-mannose-6-phosphate[0]',\n",
       "  'deltag': -3.92,\n",
       "  'deltagerr': 0.52,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1'],\n",
       "  'equation': '(1) cpd00068[0] + (1) cpd00138[0] => (1) cpd00067[0] + (1) cpd00090[0] + (1) cpd00235[0]',\n",
       "  'id': 'rxn00976',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ITP:D-mannose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00068:0:0:\"ITP\";-1:cpd00138:0:0:\"D-Mannose\";1:cpd00067:0:0:\"H+\";1:cpd00090:0:0:\"IDP\";1:cpd00235:0:0:\"D-mannose-6-phosphate\"'},\n",
       " {'abbreviation': 'R01329',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01329_c',\n",
       "   'AraGEM: R_R01329_c',\n",
       "   'BiGG: EPMGH; YBR184W_5',\n",
       "   'JP_Creinhardtii_MSB: R_EMGH',\n",
       "   'KEGG: R01329; R06142',\n",
       "   'Name: Alpha-galactosidase; Epimelibiose galactohydrolase; epimelibiose galactohydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd03196[0] <=> (1) cpd00108[0] + (1) cpd00138[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00108;cpd00138;cpd03196',\n",
       "  'definition': '(1) H2O[0] + (1) Epimelibiose[0] => (1) Galactose[0] + (1) D-Mannose[0]',\n",
       "  'deltag': -3.57,\n",
       "  'deltagerr': 0.59,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.2.1.22'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd03196[0] => (1) cpd00108[0] + (1) cpd00138[0]',\n",
       "  'id': 'rxn00977',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn34769',\n",
       "  'name': 'Epimelibiose galactohydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00052 (Galactose metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd03196:0:0:\"Epimelibiose\";1:cpd00108:0:0:\"Galactose\";1:cpd00138:0:0:\"D-Mannose\"'},\n",
       " {'abbreviation': 'R01330',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01330',\n",
       "   'iRS1563: R_R01330_c',\n",
       "   'Name: dATP:D-mannose 6-phosphotransferase'],\n",
       "  'code': '(1) cpd00115[0] + (1) cpd00138[0] <=> (1) cpd00177[0] + (1) cpd00235[0]',\n",
       "  'compound_ids': 'cpd00067;cpd00115;cpd00138;cpd00177;cpd00235',\n",
       "  'definition': '(1) dATP[0] + (1) D-Mannose[0] => (1) H+[0] + (1) dADP[0] + (1) D-mannose-6-phosphate[0]',\n",
       "  'deltag': -3.79,\n",
       "  'deltagerr': 0.42,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.7.1.1'],\n",
       "  'equation': '(1) cpd00115[0] + (1) cpd00138[0] => (1) cpd00067[0] + (1) cpd00177[0] + (1) cpd00235[0]',\n",
       "  'id': 'rxn00978',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'dATP:D-mannose 6-phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00115:0:0:\"dATP\";-1:cpd00138:0:0:\"D-Mannose\";1:cpd00067:0:0:\"H+\";1:cpd00177:0:0:\"dADP\";1:cpd00235:0:0:\"D-mannose-6-phosphate\"'},\n",
       " {'abbreviation': 'R01333',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCOLALD-DEHYDROG-RXN',\n",
       "   'BiGG: GCALDD; GCALDDr',\n",
       "   'EcoCyc: GLYCOLALD-DEHYDROG-RXN',\n",
       "   'KEGG: R01333',\n",
       "   'MetaCyc: GLYCOLALD-DEHYDROG-RXN',\n",
       "   'Name: Glycolaldehyde dehydrogenase; Glycolaldehydedehydrogenase; glycol aldehyde dehydrogenase; glycolaldehyde dehydrogenase; glycolaldehyde:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00229[0] <=> (1) cpd00004[0] + (1) cpd00139[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00003;cpd00004;cpd00067;cpd00139;cpd00229',\n",
       "  'definition': '(1) H2O[0] + (1) NAD[0] + (1) Glycolaldehyde[0] <=> (1) NADH[0] + (2) H+[0] + (1) Glycolate[0]',\n",
       "  'deltag': -14.12,\n",
       "  'deltagerr': 0.57,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.2.1.21'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00003[0] + (1) cpd00229[0] <=> (1) cpd00004[0] + (2) cpd00067[0] + (1) cpd00139[0]',\n",
       "  'id': 'rxn00979',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn31785',\n",
       "  'name': 'glycolaldehyde:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); Carbohydrates-Degradation (Carbohydrate Degradation); D-Arabinose-Degradation (D-arabinose Degradation); DARABCATK12-PWY (D-arabinose degradation I); Degradation (Degradation/Utilization/Assimilation); GLYCOL-GLYOXDEG-PWY (superpathway of glycol metabolism and degradation); PWY-5257 (superpathway of pentose and pentitol degradation); PWY0-1280 (ethylene glycol degradation); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-ALCOHOLS-DEG (Sugar Alcohol Degradation); Sugar-Derivatives (Sugar Derivative Degradation); Sugars-And-Polysaccharides-Degradation (Sugar Degradation)',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00003:0:0:\"NAD\";-1:cpd00229:0:0:\"Glycolaldehyde\";1:cpd00004:0:0:\"NADH\";2:cpd00067:0:0:\"H+\";1:cpd00139:0:0:\"Glycolate\"'},\n",
       " {'abbreviation': 'R01334',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GPH-RXN',\n",
       "   'BiGG: PGLYCP',\n",
       "   'BrachyCyc: GPH-RXN',\n",
       "   'KEGG: R01334',\n",
       "   'MetaCyc: GPH-RXN',\n",
       "   'Name: 2-Phosphoglycolate phosphohydrolase; 2-phosphoglycolate phosphatase; 2-phosphoglycolate phosphohydrolase; P-glycolate phosphatase; Phosphoglycolate phosphatase; phosphoglycolate hydrolase; phosphoglycolate phosphatase; phosphoglycollate phosphatase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00727[0] <=> (1) cpd00009[0] + (1) cpd00139[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00009;cpd00139;cpd00727',\n",
       "  'definition': '(1) H2O[0] + (1) 2-Phosphoglycolate[0] <=> (1) Phosphate[0] + (1) Glycolate[0]',\n",
       "  'deltag': -7.41,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.3.18'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00727[0] <=> (1) cpd00009[0] + (1) cpd00139[0]',\n",
       "  'id': 'rxn00980',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19871;rxn30689;rxn34260;rxn35247',\n",
       "  'name': '2-Phosphoglycolate phosphohydrolase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-181 (photorespiration); Photosynthesis ()',\n",
       "   'KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00727:0:0:\"2-Phosphoglycolate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00139:0:0:\"Glycolate\"'},\n",
       " {'abbreviation': 'R01336',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01336',\n",
       "   'MetaCyc: CARBOXYMETHYLOXYSUCCINATE-LYASE-RXN',\n",
       "   'Name: Carboxymethyloxysuccinate glycolate-lyase; carbon-oxygen lyase; carboxymethyloxysuccinate glycolate-lyase; carboxymethyloxysuccinate glycolate-lyase (fumarate-forming); carboxymethyloxysuccinate lyase'],\n",
       "  'code': '(1) cpd02266[0] <=> (1) cpd00106[0] + (1) cpd00139[0]',\n",
       "  'compound_ids': 'cpd00106;cpd00139;cpd02266',\n",
       "  'definition': '(1) Carboxymethyloxysuccinate[0] <=> (1) Fumarate[0] + (1) Glycolate[0]',\n",
       "  'deltag': -4.0,\n",
       "  'deltagerr': 0.72,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.2.99.12'],\n",
       "  'equation': '(1) cpd02266[0] <=> (1) cpd00106[0] + (1) cpd00139[0]',\n",
       "  'id': 'rxn00981',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'carboxymethyloxysuccinate glycolate-lyase (fumarate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd02266:0:0:\"Carboxymethyloxysuccinate\";1:cpd00106:0:0:\"Fumarate\";1:cpd00139:0:0:\"Glycolate\"'},\n",
       " {'abbreviation': 'R01349',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01349',\n",
       "   'Name: S-Adenosyl-L-methionine:fatty-acid O-methyltransferase'],\n",
       "  'code': '(1) cpd00017[0] + (1) cpd00049[0] <=> (1) cpd00019[0] + (1) cpd02149[0]',\n",
       "  'compound_ids': 'cpd00017;cpd00019;cpd00049;cpd02149',\n",
       "  'definition': '(1) S-Adenosyl-L-methionine[0] + (1) Carboxylic acid[0] => (1) S-Adenosyl-homocysteine[0] + (1) Fatty acid methyl ester[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['2.1.1.15'],\n",
       "  'equation': '(1) cpd00017[0] + (1) cpd00049[0] => (1) cpd00019[0] + (1) cpd02149[0]',\n",
       "  'id': 'rxn00982',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'S-Adenosyl-L-methionine:fatty-acid O-methyltransferase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00017:0:0:\"S-Adenosyl-L-methionine\";-1:cpd00049:0:0:\"Carboxylic acid\";1:cpd00019:0:0:\"S-Adenosyl-homocysteine\";1:cpd02149:0:0:\"Fatty acid methyl ester\"'},\n",
       " {'abbreviation': 'R01351',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AlgaGEM: R_R01351_c',\n",
       "   'KEGG: R01351; R07293',\n",
       "   'iRS1563: R_R01351_c',\n",
       "   'Name: 1-acylglycerol acylhydrolase; Glycerol-ester acylhydrolase; monoacylglycerol acylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01297[0] <=> (1) cpd00049[0] + (1) cpd00100[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00049;cpd00067;cpd00100;cpd01297',\n",
       "  'definition': '(1) H2O[0] + (1) Glyceride[0] <=> (1) Carboxylic acid[0] + (1) H+[0] + (1) Glycerol[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.23', '3.1.1.79'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01297[0] <=> (1) cpd00049[0] + (1) cpd00067[0] + (1) cpd00100[0]',\n",
       "  'id': 'rxn00983',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn15667',\n",
       "  'name': '1-acylglycerol acylhydrolase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00561 (Glycerolipid metabolism)'],\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01297:0:0:\"Glyceride\";1:cpd00049:0:0:\"Carboxylic acid\";1:cpd00067:0:0:\"H+\";1:cpd00100:0:0:\"Glycerol\"'},\n",
       " {'abbreviation': 'R01352',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01352',\n",
       "   'iRS1563: R_R01352_c',\n",
       "   'Name: 2-acylglycerol acylhydrolase; Glycerol-ester acylhydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd01431[0] <=> (1) cpd00049[0] + (1) cpd00100[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00049;cpd00067;cpd00100;cpd01431',\n",
       "  'definition': '(1) H2O[0] + (1) 2-Glyceride[0] <=> (1) Carboxylic acid[0] + (1) H+[0] + (1) Glycerol[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['3.1.1.23'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd01431[0] <=> (1) cpd00049[0] + (1) cpd00067[0] + (1) cpd00100[0]',\n",
       "  'id': 'rxn00984',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2-acylglycerol acylhydrolase',\n",
       "  'notes': ['GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd01431:0:0:\"2-Glyceride\";1:cpd00049:0:0:\"Carboxylic acid\";1:cpd00067:0:0:\"H+\";1:cpd00100:0:0:\"Glycerol\"'},\n",
       " {'abbreviation': 'R01353',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: PPAK; PPAKr; PROPP',\n",
       "   'ChlamyCyc: PROPKIN-RXN; RXN-7958',\n",
       "   'EcoCyc: PROPKIN-RXN',\n",
       "   'KEGG: R01353',\n",
       "   'MetaCyc: PROPKIN-RXN; RXN-7958',\n",
       "   'Name: ATP:propanoate phosphotransferase; PduW; Propionate kinase; TdcD; propionate kinase; propionate/acetate kinase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00141[0] <=> (1) cpd00008[0] + (1) cpd01844[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00141;cpd01844',\n",
       "  'definition': '(1) ATP[0] + (1) Propionate[0] <=> (1) ADP[0] + (1) Propionyl phosphate[0]',\n",
       "  'deltag': 8.16,\n",
       "  'deltagerr': 0.68,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.1', '2.7.2.15'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00141[0] <=> (1) cpd00008[0] + (1) cpd01844[0]',\n",
       "  'id': 'rxn00985',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28090;rxn28091',\n",
       "  'name': 'ATP:propanoate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Alcohol-Degradation (Alcohol Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); Fermentation-to-propanoate (Fermentation to Propanoate); PWY-5437 (L-threonine degradation I); PWY-7013 ((<i>S</i>)-propane-1,2-diol degradation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); THREOCAT-PWY (superpathway of L-threonine metabolism); THREONINE-DEG (L-threonine Degradation)',\n",
       "   'KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00141:0:0:\"Propionate\";1:cpd00008:0:0:\"ADP\";1:cpd01844:0:0:\"Propionyl phosphate\"'},\n",
       " {'abbreviation': 'R01354',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['JP_Creinhardtii_MSB: R_APAT',\n",
       "   'KEGG: R01354',\n",
       "   'TS_Athaliana: rxn642_p',\n",
       "   'iRS1563: R_R01354_c',\n",
       "   'iRS1597: R01354_c',\n",
       "   'Name: ATP:propanoate adenyltransferase; ATP:propanoate adenylyltransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00141[0] <=> (1) cpd00012[0] + (1) cpd03560[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00012;cpd00141;cpd03560',\n",
       "  'definition': '(1) ATP[0] + (1) Propionate[0] => (1) PPi[0] + (1) Propionyladenylate[0]',\n",
       "  'deltag': -11.13,\n",
       "  'deltagerr': 4.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.1', '6.2.1.17'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00141[0] => (1) cpd00012[0] + (1) cpd03560[0]',\n",
       "  'id': 'rxn00986',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33029;rxn35418',\n",
       "  'name': 'ATP:propanoate adenylyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC'],\n",
       "  'pathways': ['KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00141:0:0:\"Propionate\";1:cpd00012:0:0:\"PPi\";1:cpd03560:0:0:\"Propionyladenylate\"'},\n",
       " {'abbreviation': 'R01355',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 23-DIMETHYLMALATE-LYASE-RXN',\n",
       "   'CornCyc: 23-DIMETHYLMALATE-LYASE-RXN',\n",
       "   'KEGG: R01355',\n",
       "   'MetaCyc: 23-DIMETHYLMALATE-LYASE-RXN',\n",
       "   'Name: (2R,3S)-2,3-dimethylmalate pyruvate-lyase; (2R,3S)-2,3-dimethylmalate pyruvate-lyase (propanoate-forming); 2,3-Dimethylmalate pyruvate-lyase; 2,3-dimethylmalate lyase; 2,3-dimethylmalate pyruvate-lyase'],\n",
       "  'code': '(1) cpd01705[0] <=> (1) cpd00020[0] + (1) cpd00141[0]',\n",
       "  'compound_ids': 'cpd00020;cpd00141;cpd01705',\n",
       "  'definition': '(1) 2,3-Dimethylmalate[0] <=> (1) Pyruvate[0] + (1) Propionate[0]',\n",
       "  'deltag': 0.41,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.32'],\n",
       "  'equation': '(1) cpd01705[0] <=> (1) cpd00020[0] + (1) cpd00141[0]',\n",
       "  'id': 'rxn00987',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': '2,3-Dimethylmalate pyruvate-lyase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Degradation (Degradation/Utilization/Assimilation); Nicotinate-Degradation (Nicotinate Degradation); PWY-5055 (nicotinate degradation III); PWY-5062 (superpathway of nicotinate degradation)',\n",
       "   'KEGG: rn00760 (Nicotinate and nicotinamide metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd01705:0:0:\"2,3-Dimethylmalate\";1:cpd00020:0:0:\"Pyruvate\";1:cpd00141:0:0:\"Propionate\"'},\n",
       " {'abbreviation': 'R01357',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACAC',\n",
       "   'KEGG: R01357',\n",
       "   'MetaCyc: ACETOACETATE--COA-LIGASE-RXN',\n",
       "   'iMO1053-PAO1: ACAC',\n",
       "   'iMO1056: ACAC',\n",
       "   'Name: Acetoacetate:CoA ligase (AMP-forming); acetoacetate-CoA ligase; acetoacetate:CoA ligase (AMP-forming); acetoacetyl-CoA synthetase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00142[0] <=> (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00279[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00010;cpd00012;cpd00018;cpd00067;cpd00142;cpd00279',\n",
       "  'definition': '(1) ATP[0] + (1) CoA[0] + (1) Acetoacetate[0] => (1) PPi[0] + (1) AMP[0] + (1) H+[0] + (1) Acetoacetyl-CoA[0]',\n",
       "  'deltag': 0.19,\n",
       "  'deltagerr': 0.35,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['6.2.1.16'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd00010[0] + (1) cpd00142[0] => (1) cpd00012[0] + (1) cpd00018[0] + (1) cpd00067[0] + (1) cpd00279[0]',\n",
       "  'id': 'rxn00988',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetoacetate:CoA ligase (AMP-forming)',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd00010:0:0:\"CoA\";-1:cpd00142:0:0:\"Acetoacetate\";1:cpd00012:0:0:\"PPi\";1:cpd00018:0:0:\"AMP\";1:cpd00067:0:0:\"H+\";1:cpd00279:0:0:\"Acetoacetyl-CoA\"'},\n",
       " {'abbreviation': 'R01358',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01358',\n",
       "   'MetaCyc: ACETOACETYL-COA-HYDROLASE-RXN',\n",
       "   'Name: Acetoacetyl-CoA hydrolase; acetoacetyl CoA deacylase; acetoacetyl coenzyme A deacylase; acetoacetyl coenzyme A hydrolase; acetoacetyl-CoA hydrolase'],\n",
       "  'code': '(1) cpd00001[0] + (1) cpd00279[0] <=> (1) cpd00010[0] + (1) cpd00142[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00010;cpd00067;cpd00142;cpd00279',\n",
       "  'definition': '(1) H2O[0] + (1) Acetoacetyl-CoA[0] => (1) CoA[0] + (1) H+[0] + (1) Acetoacetate[0]',\n",
       "  'deltag': -8.62,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['3.1.2.11'],\n",
       "  'equation': '(1) cpd00001[0] + (1) cpd00279[0] => (1) cpd00010[0] + (1) cpd00067[0] + (1) cpd00142[0]',\n",
       "  'id': 'rxn00989',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetoacetyl-CoA hydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); PWY-6876 (isopropanol biosynthesis (engineered))',\n",
       "   'KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00279:0:0:\"Acetoacetyl-CoA\";1:cpd00010:0:0:\"CoA\";1:cpd00067:0:0:\"H+\";1:cpd00142:0:0:\"Acetoacetate\"'},\n",
       " {'abbreviation': 'R01359',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: ACACCT',\n",
       "   'EcoCyc: ACETOACETYL-COA-TRANSFER-RXN',\n",
       "   'KEGG: R01359',\n",
       "   'MetaCyc: ACETOACETYL-COA-TRANSFER-RXN',\n",
       "   'iAF1260: ACACCT',\n",
       "   'Name: Acetoacetyl-CoA:acetate CoA-transferase; acetoacetyl-coa transferase; acetyl-CoA:acetoacetyl-CoA transferase'],\n",
       "  'code': '(1) cpd00029[0] + (1) cpd00279[0] <=> (1) cpd00022[0] + (1) cpd00142[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00029;cpd00142;cpd00279',\n",
       "  'definition': '(1) Acetate[0] + (1) Acetoacetyl-CoA[0] <=> (1) Acetyl-CoA[0] + (1) Acetoacetate[0]',\n",
       "  'deltag': -1.32,\n",
       "  'deltagerr': 0.34,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.-', '2.8.3.8'],\n",
       "  'equation': '(1) cpd00029[0] + (1) cpd00279[0] <=> (1) cpd00022[0] + (1) cpd00142[0]',\n",
       "  'id': 'rxn00990',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetoacetyl-CoA:acetate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ACETOACETATE-DEG-PWY (acetoacetate degradation (to acetyl CoA)); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); Fermentation (); PWY-6588 (pyruvate fermentation to acetone); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00029:0:0:\"Acetate\";-1:cpd00279:0:0:\"Acetoacetyl-CoA\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00142:0:0:\"Acetoacetate\"'},\n",
       " {'abbreviation': 'R01360',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HYDROXYMETHYLGLUTARYL-COA-LYASE-RXN',\n",
       "   'BiGG: HMGL',\n",
       "   'BrachyCyc: HYDROXYMETHYLGLUTARYL-COA-LYASE-RXN',\n",
       "   'KEGG: R01360',\n",
       "   'MetaCyc: HYDROXYMETHYLGLUTARYL-COA-LYASE-RXN',\n",
       "   'Name: (S)-3-Hydroxy-3-methylglutaryl-CoA acetoacetate-lyase; (S)-3-hydroxy-3-methylglutaryl-CoA acetoacetate-lyase; (S)-3-hydroxy-3-methylglutaryl-CoA acetoacetate-lyase (acetyl-CoA-forming); 3-hydroxy-3-methylglutarate-CoA lyase; 3-hydroxy-3-methylglutaryl CoA cleaving enzyme; 3-hydroxy-3-methylglutaryl coenzyme A lyase; 3-hydroxy-3-methylglutaryl-CoA lyase; HMG-CoA lyase; hydroxymethylglutaryl coenzyme A lyase; hydroxymethylglutaryl coenzyme A-cleaving enzyme; hydroxymethylglutaryl-CoA lyase'],\n",
       "  'code': '(1) cpd00292[0] <=> (1) cpd00022[0] + (1) cpd00142[0]',\n",
       "  'compound_ids': 'cpd00022;cpd00142;cpd00292',\n",
       "  'definition': '(1) HMG-CoA[0] <=> (1) Acetyl-CoA[0] + (1) Acetoacetate[0]',\n",
       "  'deltag': -4.0,\n",
       "  'deltagerr': 0.63,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['4.1.3.4'],\n",
       "  'equation': '(1) cpd00292[0] <=> (1) cpd00022[0] + (1) cpd00142[0]',\n",
       "  'id': 'rxn00991',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27937',\n",
       "  'name': '(S)-3-hydroxy-3-methylglutaryl-CoA acetoacetate-lyase (acetyl-CoA-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); LEU-DEG2-PWY (L-leucine degradation I); LEUCINE-DEG (L-leucine Degradation); OTHER-ENERGY (Other); PWY-5074 (mevalonate degradation); PWY66-367 (ketogenesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00072 (Synthesis and degradation of ketone bodies); rn00280 (Valine, leucine and isoleucine degradation); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00292:0:0:\"HMG-CoA\";1:cpd00022:0:0:\"Acetyl-CoA\";1:cpd00142:0:0:\"Acetoacetate\"'},\n",
       " {'abbreviation': 'R01361',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 3-HYDROXYBUTYRATE-DEHYDROGENASE-RXN',\n",
       "   'BiGG: 3HBUTDH; BDH',\n",
       "   'BrachyCyc: 3-HYDROXYBUTYRATE-DEHYDROGENASE-RXN',\n",
       "   'KEGG: R01361',\n",
       "   'MetaCyc: 3-HYDROXYBUTYRATE-DEHYDROGENASE-RXN',\n",
       "   'Name: (R)-3-Hydroxybutanoate:NAD+ oxidoreductase; (R)-3-hydroxybutanoate:NAD+ oxidoreductase; 3-D-hydroxybutyrate dehydrogenase; 3-Hydroxybutanoate:NAD+ oxidoreductase; 3-hydroxybutyrate dehydrogenase; D-(-)-3-hydroxybutyrate dehydrogenase; D-3-hydroxybutyrate dehydrogenase; D-beta-hydroxybutyrate dehydrogenase; NAD-beta-hydroxybutyrate dehydrogenase; beta-hydroxybutyrate dehydrogenase; beta-hydroxybutyric acid dehydrogenase; beta-hydroxybutyric dehydrogenase; hydroxybutyrate oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00797[0] <=> (1) cpd00004[0] + (1) cpd00142[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00142;cpd00797',\n",
       "  'definition': '(1) NAD[0] + (1) (R)-3-Hydroxybutanoate[0] <=> (1) NADH[0] + (1) H+[0] + (1) Acetoacetate[0]',\n",
       "  'deltag': 2.6,\n",
       "  'deltagerr': 0.18,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.30'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00797[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00142[0]',\n",
       "  'id': 'rxn00992',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27539;rxn34633',\n",
       "  'name': '(R)-3-Hydroxybutanoate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Energy-Metabolism (Generation of Precursor Metabolite and Energy); OTHER-ENERGY (Other); PWY66-367 (ketogenesis); PWY66-368 (ketolysis)',\n",
       "   'KEGG: rn00072 (Synthesis and degradation of ketone bodies); rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00797:0:0:\"(R)-3-Hydroxybutanoate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00142:0:0:\"Acetoacetate\"'},\n",
       " {'abbreviation': 'R01364',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: FUMARYLACETOACETASE-RXN',\n",
       "   'BiGG: FUMAC; FUMACA; U60',\n",
       "   'BrachyCyc: FUMARYLACETOACETASE-RXN',\n",
       "   'KEGG: R01364',\n",
       "   'MetaCyc: FUMARYLACETOACETASE-RXN',\n",
       "   'Name: 4-Fumarylacetoacetate fumarylhydrola; 4-fumarylacetoacetate fumarylhydrolase; Fumarylacetoacetase; beta-diketonase; fumarylacetoacetate hydrolase'],\n",
       "  'code': '(1) cpd00106[0] + (1) cpd00142[0] <=> (1) cpd00001[0] + (1) cpd00780[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00067;cpd00106;cpd00142;cpd00780',\n",
       "  'definition': '(1) H+[0] + (1) Fumarate[0] + (1) Acetoacetate[0] <= (1) H2O[0] + (1) Fumarylacetoacetate[0]',\n",
       "  'deltag': 16.92,\n",
       "  'deltagerr': 0.92,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['3.7.1.2'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00106[0] + (1) cpd00142[0] <= (1) cpd00001[0] + (1) cpd00780[0]',\n",
       "  'id': 'rxn00993',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33228;rxn33239;rxn38105;rxn38106',\n",
       "  'name': '4-fumarylacetoacetate fumarylhydrolase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); TYRFUMCAT-PWY (L-tyrosine degradation I); TYROSINE-DEG (L-tyrosine Degradation)',\n",
       "   'KEGG: rn00350 (Tyrosine metabolism); rn00643 (Styrene degradation)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00106:0:0:\"Fumarate\";-1:cpd00142:0:0:\"Acetoacetate\";1:cpd00001:0:0:\"H2O\";1:cpd00780:0:0:\"Fumarylacetoacetate\"'},\n",
       " {'abbreviation': 'R01365',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01365',\n",
       "   'MetaCyc: 2.8.3.9-RXN',\n",
       "   'Name: Butanoyl-CoA:acetoacetate CoA-transferase; butanoyl-CoA:acetoacetate CoA-transferase; butyrate-acetoacetate CoA-transferase; butyryl coenzyme A-acetoacetate coenzyme A-transferase; butyryl-CoA-acetoacetate CoA-transferase'],\n",
       "  'code': '(1) cpd00120[0] + (1) cpd00142[0] <=> (1) cpd00211[0] + (1) cpd00279[0]',\n",
       "  'compound_ids': 'cpd00120;cpd00142;cpd00211;cpd00279',\n",
       "  'definition': '(1) Butyryl-CoA[0] + (1) Acetoacetate[0] <=> (1) Butyrate[0] + (1) Acetoacetyl-CoA[0]',\n",
       "  'deltag': 0.25,\n",
       "  'deltagerr': 0.82,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.8.3.9'],\n",
       "  'equation': '(1) cpd00120[0] + (1) cpd00142[0] <=> (1) cpd00211[0] + (1) cpd00279[0]',\n",
       "  'id': 'rxn00994',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Butanoyl-CoA:acetoacetate CoA-transferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetate-Formation (Fermentation to Acetate); Acetyl-CoA-Butyrate (Fermentation to Butanoate); Amino-Acid-Degradation (Amino Acid Degradation); CARBOXYLATES-DEG (Carboxylate Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fermentation (); Fermentation-to-Acids (Fermentation to Short-Chain Fatty Acids); LYSINE-DEG (L-lysine Degradation); P163-PWY (L-lysine fermentation to acetate and butanoate); PWY-5327 (superpathway of L-lysine degradation); PWY-6588 (pyruvate fermentation to acetone); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00310 (Lysine degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00120:0:0:\"Butyryl-CoA\";-1:cpd00142:0:0:\"Acetoacetate\";1:cpd00211:0:0:\"Butyrate\";1:cpd00279:0:0:\"Acetoacetyl-CoA\"'},\n",
       " {'abbreviation': 'R01366',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ACETOACETATE-DECARBOXYLASE-RXN',\n",
       "   'EcoCyc: ACETOACETATE-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R01366',\n",
       "   'MetaCyc: ACETOACETATE-DECARBOXYLASE-RXN',\n",
       "   'PlantCyc: ACETOACETATE-DECARBOXYLASE-RXN',\n",
       "   'Name: Acetoacetate carboxy-lyase; Acetoacetate carboxy-lyase (acetone-forming); acetoacetate carboxy-lyase; acetoacetate carboxy-lyase (acetone-forming); acetoacetate decarboxylase; acetoacetic acid decarboxylase'],\n",
       "  'code': '(1) cpd00142[0] <=> (1) cpd00011[0] + (1) cpd00178[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00142;cpd00178',\n",
       "  'definition': '(1) H+[0] + (1) Acetoacetate[0] => (1) CO2[0] + (1) Acetone[0]',\n",
       "  'deltag': -3.91,\n",
       "  'deltagerr': 1.0,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.4'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00142[0] => (1) cpd00011[0] + (1) cpd00178[0]',\n",
       "  'id': 'rxn00995',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetoacetate carboxy-lyase (acetone-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Acetone-Degradation (Acetone Degradation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Fatty-Acid-and-Lipid-Degradation (Fatty Acid and Lipid Degradation); Fermentation (); OTHER-ENERGY (Other); PWY-5451 (acetone degradation I (to methylglyoxal)); PWY-6588 (pyruvate fermentation to acetone); PWY-6594 (superpathway of <i>Clostridium acetobutylicum</i> solventogenic fermentation); PWY-6604 (superpathway of <i>Clostridium acetobutylicum</i> acidogenic and solventogenic fermentation); PWY-6876 (isopropanol biosynthesis (engineered)); PWY-7466 (acetone degradation III (to propane-1,2-diol)); PWY66-367 (ketogenesis); Pyruvate-Degradation (Fermentation of Pyruvate)',\n",
       "   'KEGG: rn00072 (Synthesis and degradation of ketone bodies); rn00640 (Propanoate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00142:0:0:\"Acetoacetate\";1:cpd00011:0:0:\"CO2\";1:cpd00178:0:0:\"Acetone\"'},\n",
       " {'abbreviation': 'R01367',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01367',\n",
       "   'iRS1563: R_R01367_c',\n",
       "   'Name: Acetoacetate hydro-lyase; Acetoacetate hydro-lyase (3-butynoate-forming)'],\n",
       "  'code': '(1) cpd00142[0] <=> (1) cpd00001[0] + (1) cpd03663[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00142;cpd03663',\n",
       "  'definition': '(1) Acetoacetate[0] <= (1) H2O[0] + (1) 3-Butynoate[0]',\n",
       "  'deltag': 77.66,\n",
       "  'deltagerr': 5.31,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': ['4.2.1.27'],\n",
       "  'equation': '(1) cpd00142[0] <= (1) cpd00001[0] + (1) cpd03663[0]',\n",
       "  'id': 'rxn00996',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Acetoacetate hydro-lyase (3-butynoate-forming)',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00650 (Butanoate metabolism)'],\n",
       "  'reversibility': '<',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00142:0:0:\"Acetoacetate\";1:cpd00001:0:0:\"H2O\";1:cpd03663:0:0:\"3-Butynoate\"'},\n",
       " {'abbreviation': 'R01370',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01370',\n",
       "   'iAO358: rll_596',\n",
       "   'Name: Phenyllactate:NAD+ oxidoreductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd03331[0] <=> (1) cpd00004[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00143;cpd03331',\n",
       "  'definition': '(1) NAD[0] + (1) PLCT[0] <=> (1) NADH[0] + (1) H+[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': 4.44,\n",
       "  'deltagerr': 0.24,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.222', '1.1.1.237'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd03331[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn00997',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phenyllactate:NAD+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism); rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd03331:0:0:\"PLCT\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R01371',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01371', 'Name: Phenyllactate:NADP+ oxidoreductase'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd03331[0] <=> (1) cpd00005[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00143;cpd03331',\n",
       "  'definition': '(1) NADP[0] + (1) PLCT[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': 4.44,\n",
       "  'deltagerr': 0.24,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.222', '1.1.1.237'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd03331[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn00998',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phenyllactate:NADP+ oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism); rn00960 (Tropane, piperidine and pyridine alkaloid biosynthesis)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd03331:0:0:\"PLCT\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R01372',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-10815',\n",
       "   'CornCyc: RXN-10815',\n",
       "   'KEGG: R01372',\n",
       "   'MaizeCyc: RXN-10815',\n",
       "   'MetaCyc: RXN-10815',\n",
       "   'Name: 2-oxo-3-phenylpropanoate dioxygenase; Phenylpyruvate:oxygen oxidoreductase (hydroxylating,decarboxylating); phenylpyruvate dioxygenase'],\n",
       "  'code': '(1) cpd00007[0] + (1) cpd00143[0] <=> (1) cpd00011[0] + (1) cpd03480[0]',\n",
       "  'compound_ids': 'cpd00007;cpd00011;cpd00143;cpd03480',\n",
       "  'definition': '(1) O2[0] + (1) Phenylpyruvate[0] => (1) CO2[0] + (1) 2-Hydroxyphenylacetate[0]',\n",
       "  'deltag': -117.42,\n",
       "  'deltagerr': 1.16,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['1.13.11.27'],\n",
       "  'equation': '(1) cpd00007[0] + (1) cpd00143[0] => (1) cpd00011[0] + (1) cpd03480[0]',\n",
       "  'id': 'rxn00999',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33629;rxn38313;rxn38314',\n",
       "  'name': 'Phenylpyruvate:oxygen oxidoreductase (hydroxylating,decarboxylating)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); PHENYLALANINE-DEG (L-phenylalanine Degradation); PWY-6318 (L-phenylalanine degradation IV (mammalian, via side chain)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00007:0:0:\"O2\";-1:cpd00143:0:0:\"Phenylpyruvate\";1:cpd00011:0:0:\"CO2\";1:cpd03480:0:0:\"2-Hydroxyphenylacetate\"'},\n",
       " {'abbreviation': 'R01373',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PREPHENATEDEHYDRAT-RXN',\n",
       "   'BiGG: PHA2; PPNDH',\n",
       "   'BrachyCyc: PREPHENATEDEHYDRAT-RXN',\n",
       "   'KEGG: R01373',\n",
       "   'MetaCyc: PREPHENATEDEHYDRAT-RXN',\n",
       "   'Name: Prephenate dehydratase; Prephenate hydro-lyase (decarboxylating); prephenate dehydratase; prephenate hydro-lyase (decarboxylating); prephenate hydro-lyase (decarboxylating, phenylpyruvate-forming); prephenate hydro-lyase (decarboxylating; phenylpyruvate-forming); prephenatedehydratase'],\n",
       "  'code': '(1) cpd00219[0] <=> (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00143[0]',\n",
       "  'compound_ids': 'cpd00001;cpd00011;cpd00067;cpd00143;cpd00219',\n",
       "  'definition': '(1) H+[0] + (1) Prephenate[0] => (1) H2O[0] + (1) CO2[0] + (1) Phenylpyruvate[0]',\n",
       "  'deltag': -20.31,\n",
       "  'deltagerr': 2.54,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.2.1.51', '4.2.1.91'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00219[0] => (1) cpd00001[0] + (1) cpd00011[0] + (1) cpd00143[0]',\n",
       "  'id': 'rxn01000',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn28085;rxn33346',\n",
       "  'name': 'prephenate hydro-lyase (decarboxylating; phenylpyruvate-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALL-CHORISMATE-PWY (superpathway of chorismate metabolism); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); COMPLETE-ARO-PWY (superpathway of aromatic amino acid biosynthesis); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PHENYLALANINE-SYN (L-phenylalanine Biosynthesis); PHESYN (L-phenylalanine biosynthesis I); PWY-6628 (superpathway of L-phenylalanine biosynthesis); PWY-7432 (L-phenylalanine biosynthesis III (cytosolic, plants))',\n",
       "   'KEGG: rn00400 (Phenylalanine, tyrosine and tryptophan biosynthesis); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00219:0:0:\"Prephenate\";1:cpd00001:0:0:\"H2O\";1:cpd00011:0:0:\"CO2\";1:cpd00143:0:0:\"Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R01375',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['CornCyc: 2.6.1.64-RXN',\n",
       "   'KEGG: R01375',\n",
       "   'MetaCyc: 2.6.1.64-RXN',\n",
       "   'PoplarCyc: 2.6.1.64-RXN',\n",
       "   'SoyCyc: 2.6.1.64-RXN',\n",
       "   'Name: L-Glutamine:phenylpyruvate aminotransferase; L-glutamine:phenylpyruvate aminotransferase; glutamine transaminase K; glutamine-phenylpyruvate aminotransferase; glutamine-phenylpyruvate transaminase'],\n",
       "  'code': '(1) cpd00053[0] + (1) cpd00143[0] <=> (1) cpd00066[0] + (1) cpd00695[0]',\n",
       "  'compound_ids': 'cpd00053;cpd00066;cpd00143;cpd00695',\n",
       "  'definition': '(1) L-Glutamine[0] + (1) Phenylpyruvate[0] <=> (1) L-Phenylalanine[0] + (1) 2-Oxoglutaramate[0]',\n",
       "  'deltag': 0.83,\n",
       "  'deltagerr': 1.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.64'],\n",
       "  'equation': '(1) cpd00053[0] + (1) cpd00143[0] <=> (1) cpd00066[0] + (1) cpd00695[0]',\n",
       "  'id': 'rxn01001',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33389;rxn33390',\n",
       "  'name': 'L-Glutamine:phenylpyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00053:0:0:\"L-Glutamine\";-1:cpd00143:0:0:\"Phenylpyruvate\";1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00695:0:0:\"2-Oxoglutaramate\"'},\n",
       " {'abbreviation': 'R01376',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01376',\n",
       "   'MetaCyc: 2.6.1.28-RXN',\n",
       "   'iAO358: rll_21; rll_27',\n",
       "   'Name: L-Tryptophan:phenylpyruvate aminotransferase; L-tryptophan-alpha-ketoisocaproate aminotransferase; L-tryptophan:phenylpyruvate aminotransferase; tryptophan-phenylpyruvate transaminase'],\n",
       "  'code': '(1) cpd00065[0] + (1) cpd00143[0] <=> (1) cpd00066[0] + (1) cpd00278[0]',\n",
       "  'compound_ids': 'cpd00065;cpd00066;cpd00143;cpd00278',\n",
       "  'definition': '(1) L-Tryptophan[0] + (1) Phenylpyruvate[0] <=> (1) L-Phenylalanine[0] + (1) Indolepyruvate[0]',\n",
       "  'deltag': -6.09,\n",
       "  'deltagerr': 1.45,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.6.1.28'],\n",
       "  'equation': '(1) cpd00065[0] + (1) cpd00143[0] <=> (1) cpd00066[0] + (1) cpd00278[0]',\n",
       "  'id': 'rxn01002',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'L-Tryptophan:phenylpyruvate aminotransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ALKALOIDS-SYN (Alkaloid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); Degradation (Degradation/Utilization/Assimilation); INDOLE-ALKALOIDS (Indole Alkaloid Biosynthesis); N-CONTAINING-SECONDARY-CMPD-SYN (Nitrogen-Containing Secondary Compound Biosynthesis); PHENYLPROPANOID-SYN (Phenylpropanoid Derivative Biosynthesis); PWY-5081 (L-tryptophan degradation VIII (to tryptophol)); PWY-7520 (terrequinone A biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); QUINONE-SYN (Quinone Secondary Metabolite Biosynthesis); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis); TRYPTOPHAN-DEG (L-tryptophan Degradation)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00065:0:0:\"L-Tryptophan\";-1:cpd00143:0:0:\"Phenylpyruvate\";1:cpd00066:0:0:\"L-Phenylalanine\";1:cpd00278:0:0:\"Indolepyruvate\"'},\n",
       " {'abbreviation': 'R01377',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHENYLPYRUVATE-DECARBOXYLASE-RXN',\n",
       "   'BiGG: PPYRDC; R01377',\n",
       "   'KEGG: R01377',\n",
       "   'MetaCyc: PHENYLPYRUVATE-DECARBOXYLASE-RXN',\n",
       "   'PlantCyc: PHENYLPYRUVATE-DECARBOXYLASE-RXN',\n",
       "   'Name: Decarboxylase; Phenylpyruvate carboxy-lyase; phenylpyruvate carboxy-lyase; phenylpyruvate carboxy-lyase (phenylacetaldehyde-forming); phenylpyruvate decarboxylase'],\n",
       "  'code': '(1) cpd00143[0] <=> (1) cpd00011[0] + (1) cpd00464[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00143;cpd00464',\n",
       "  'definition': '(1) H+[0] + (1) Phenylpyruvate[0] => (1) CO2[0] + (1) alpha-Tolualdehyde[0]',\n",
       "  'deltag': -2.78,\n",
       "  'deltagerr': 0.75,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.-', '4.1.1.43', '4.1.4.3'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00143[0] => (1) cpd00011[0] + (1) cpd00464[0]',\n",
       "  'id': 'rxn01003',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20539',\n",
       "  'name': 'phenylpyruvate carboxy-lyase (phenylacetaldehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ANAPHENOXI-PWY (L-phenylalanine degradation II (anaerobic)); AROMATIC-COMPOUNDS-DEGRADATION (Aromatic Compound Degradation); Amino-Acid-Degradation (Amino Acid Degradation); BENZCOA-PWY (anaerobic aromatic compound degradation (<i>Thauera aromatica</i>)); Degradation (Degradation/Utilization/Assimilation); PHENYLALANINE-DEG (L-phenylalanine Degradation); PWY-5079 (L-phenylalanine degradation III); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00143:0:0:\"Phenylpyruvate\";1:cpd00011:0:0:\"CO2\";1:cpd00464:0:0:\"alpha-Tolualdehyde\"'},\n",
       " {'abbreviation': 'R01378',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: PHENYLPYRUVATE-TAUTOMERASE-RXN',\n",
       "   'ChlamyCyc: PHENYLPYRUVATE-TAUTOMERASE-RXN',\n",
       "   'CornCyc: PHENYLPYRUVATE-TAUTOMERASE-RXN',\n",
       "   'KEGG: R01378',\n",
       "   'MetaCyc: PHENYLPYRUVATE-TAUTOMERASE-RXN',\n",
       "   'Name: Phenylpyruvate keto-enol-isomerase; phenylpyruvate keto-enol-isomerase; phenylpyruvate tautomerase; phenylpyruvic keto-enol isomerase'],\n",
       "  'code': '(1) cpd00143[0] <=> (1) cpd01784[0]',\n",
       "  'compound_ids': 'cpd00143;cpd01784',\n",
       "  'definition': '(1) Phenylpyruvate[0] <=> (1) enol-Phenylpyruvate[0]',\n",
       "  'deltag': 4.95,\n",
       "  'deltagerr': 1.33,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.2.1'],\n",
       "  'equation': '(1) cpd00143[0] <=> (1) cpd01784[0]',\n",
       "  'id': 'rxn01004',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Phenylpyruvate keto-enol-isomerase',\n",
       "  'notes': ['GCC', 'EQP'],\n",
       "  'pathways': ['KEGG: rn00360 (Phenylalanine metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00143:0:0:\"Phenylpyruvate\";1:cpd01784:0:0:\"enol-Phenylpyruvate\"'},\n",
       " {'abbreviation': 'R01381',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: 2.7.7.44-RXN',\n",
       "   'BrachyCyc: 2.7.7.44-RXN',\n",
       "   'ChlamyCyc: 2.7.7.44-RXN',\n",
       "   'KEGG: R01381',\n",
       "   'MetaCyc: 2.7.7.44-RXN',\n",
       "   'Name: UDP-D-glucuronic acid pyrophosphorylase; UDP-glucuronate pyrophosphorylase; UDP-glucuronic acid pyrophosphorylase; UTP:1-phospho-alpha-D-glucuronate uridylyltransferase; glucuronate-1-phosphate uridylyltransferase; uridine diphosphoglucuronic pyrophosphorylase'],\n",
       "  'code': '(1) cpd00062[0] + (1) cpd00880[0] <=> (1) cpd00012[0] + (1) cpd00144[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00062;cpd00144;cpd00880',\n",
       "  'definition': '(1) UTP[0] + (1) 1-Phospho-alpha-D-glucuronate[0] <=> (1) PPi[0] + (1) UDPglucuronate[0]',\n",
       "  'deltag': -0.74,\n",
       "  'deltagerr': 0.39,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.7.44'],\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd00880[0] <=> (1) cpd00012[0] + (1) cpd00144[0]',\n",
       "  'id': 'rxn01005',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UTP:1-phospho-alpha-D-glucuronate uridylyltransferase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: CARBO-BIOSYNTHESIS (Sugar Biosynthesis); CYCLITOLS-DEG (Cyclitol Degradation); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Degradation (Degradation/Utilization/Assimilation); PWY-4841 (UDP-&alpha;-D-glucuronate biosynthesis (from <i>myo</i>-inositol)); SECONDARY-METABOLITE-DEGRADATION (Secondary Metabolite Degradation); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); Sugar-Derivatives (Sugar Derivative Degradation); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00040 (Pentose and glucuronate interconversions); rn00053 (Ascorbate and aldarate metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd00880:0:0:\"1-Phospho-alpha-D-glucuronate\";1:cpd00012:0:0:\"PPi\";1:cpd00144:0:0:\"UDPglucuronate\"'},\n",
       " {'abbreviation': 'UTP:1-phospho-alpha-D-glucuronate uridylyltransferase',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': None,\n",
       "  'code': '(1) cpd00062[0] + (1) cpd03191[0] <=> (1) cpd00012[0] + (1) cpd00144[0]',\n",
       "  'compound_ids': 'cpd00012;cpd00062;cpd00067;cpd00144;cpd03191',\n",
       "  'definition': '(1) UTP[0] + (1) D-Glucuronate 1-phosphate[0] <=> (1) PPi[0] + (1) H+[0] + (1) UDPglucuronate[0]',\n",
       "  'deltag': 10000000.0,\n",
       "  'deltagerr': 10000000.0,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00062[0] + (1) cpd03191[0] <=> (1) cpd00012[0] + (1) cpd00067[0] + (1) cpd00144[0]',\n",
       "  'id': 'rxn01006',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UTP:1-phospho-alpha-D-glucuronate uridylyltransferase',\n",
       "  'notes': ['HB', 'GCP', 'EQP'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '?',\n",
       "  'source': 'Orphan',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00062:0:0:\"UTP\";-1:cpd03191:0:0:\"D-Glucuronate 1-phosphate\";1:cpd00012:0:0:\"PPi\";1:cpd00067:0:0:\"H+\";1:cpd00144:0:0:\"UDPglucuronate\"'},\n",
       " {'abbreviation': 'R01384',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UDP-GLUCURONATE-DECARBOXYLASE-RXN',\n",
       "   'BrachyCyc: UDP-GLUCURONATE-DECARBOXYLASE-RXN',\n",
       "   'ChlamyCyc: UDP-GLUCURONATE-DECARBOXYLASE-RXN',\n",
       "   'KEGG: R01384',\n",
       "   'MetaCyc: UDP-GLUCURONATE-DECARBOXYLASE-RXN',\n",
       "   'Name: UDP-D-glucuronate carboxy-lyase; UDP-D-glucuronate carboxy-lyase (UDP-D-xylose-forming); UDP-glucuronate decarboxylase; uridine-diphosphoglucuronate decarboxylase'],\n",
       "  'code': '(1) cpd00144[0] <=> (1) cpd00011[0] + (1) cpd00163[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00144;cpd00163',\n",
       "  'definition': '(1) H+[0] + (1) UDPglucuronate[0] => (1) CO2[0] + (1) UDP-xylose[0]',\n",
       "  'deltag': -2.79,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.35'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00144[0] => (1) cpd00011[0] + (1) cpd00163[0]',\n",
       "  'id': 'rxn01007',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27409;rxn27410;rxn32977;rxn32978',\n",
       "  'name': 'UDP-D-glucuronate carboxy-lyase (UDP-D-xylose-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Interconversion (Interconversions); PWY-4821 (UDP-&alpha;-D-xylose biosynthesis); PWY-5114 (UDP-sugars interconversion); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism); rn01110 (Biosynthesis of secondary metabolites)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00144:0:0:\"UDPglucuronate\";1:cpd00011:0:0:\"CO2\";1:cpd00163:0:0:\"UDP-xylose\"'},\n",
       " {'abbreviation': 'R01385',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: UDP-GLUCURONATE-4-EPIMERASE-RXN',\n",
       "   'BiGG: GLCURE',\n",
       "   'BrachyCyc: UDP-GLUCURONATE-4-EPIMERASE-RXN',\n",
       "   'KEGG: R01385',\n",
       "   'MetaCyc: UDP-GLUCURONATE-4-EPIMERASE-RXN',\n",
       "   'Name: UDP glucuronic epimerase; UDP-D-galacturonic acid 4-epimerase; UDP-galacturonate 4-epimerase; UDP-glucuronate 4-epimerase; UDPglucuronate 4-epimerase; uridine diphospho-D-galacturonic acid; uridine diphosphoglucuronate epimerase; uridine diphosphoglucuronic epimerase'],\n",
       "  'code': '(1) cpd00144[0] <=> (1) cpd00472[0]',\n",
       "  'compound_ids': 'cpd00144;cpd00472',\n",
       "  'definition': '(1) UDPglucuronate[0] <=> (1) UDPgalacturonate[0]',\n",
       "  'deltag': -0.26,\n",
       "  'deltagerr': 0.41,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.3.6'],\n",
       "  'equation': '(1) cpd00144[0] <=> (1) cpd00472[0]',\n",
       "  'id': 'rxn01008',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn33244;rxn33253;rxn33255;rxn38824',\n",
       "  'name': 'UDP-glucuronate 4-epimerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); Ascorbate-Biosynthesis (Ascorbate Biosynthesis); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Cofactor-Biosynthesis (Cofactor, Prosthetic Group, Electron Carrier, and Vitamin Biosynthesis); Interconversion (Interconversions); PWY-4861 (UDP-&alpha;-D-galacturonate biosynthesis I (from UDP-D-glucuronate)); PWY-5114 (UDP-sugars interconversion); PWY-6415 (L-ascorbate biosynthesis V); PWY-7328 (superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis); Vitamin-Biosynthesis (Vitamin Biosynthesis)',\n",
       "   'KEGG: rn00500 (Starch and sucrose metabolism); rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00144:0:0:\"UDPglucuronate\";1:cpd00472:0:0:\"UDPgalacturonate\"'},\n",
       " {'abbreviation': 'R01386',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: RXN-7770',\n",
       "   'CornCyc: RXN-7770',\n",
       "   'KEGG: R01386',\n",
       "   'MetaCyc: RXN-7770',\n",
       "   'PlantCyc: RXN-7770',\n",
       "   'Name: UDP-D-glucuronate carboxy-lyase (UDP-D-apiose-forming)'],\n",
       "  'code': '(1) cpd00144[0] <=> (1) cpd00011[0] + (1) cpd01140[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00144;cpd01140',\n",
       "  'definition': '(1) H+[0] + (1) UDPglucuronate[0] => (1) CO2[0] + (1) UDP-apiose[0]',\n",
       "  'deltag': -2.99,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.-'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00144[0] => (1) cpd00011[0] + (1) cpd01140[0]',\n",
       "  'id': 'rxn01009',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'UDP-D-glucuronate carboxy-lyase (UDP-D-apiose-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Activation-Inactivation-Interconversion (Activation/Inactivation/Interconversion); CARBO-BIOSYNTHESIS (Sugar Biosynthesis); Carbohydrates-Biosynthesis (Carbohydrate Biosynthesis); Interconversion (Interconversions); PWY-5113 (UDP-D-apiose biosynthesis (from UDP-D-glucuronate)); PWY-5114 (UDP-sugars interconversion); SUGAR-NUCLEOTIDES (Sugar Nucleotide Biosynthesis); UDP-Sugar-Biosynthesis (UDP-sugar Biosynthesis)',\n",
       "   'KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00144:0:0:\"UDPglucuronate\";1:cpd00011:0:0:\"CO2\";1:cpd01140:0:0:\"UDP-apiose\"'},\n",
       " {'abbreviation': 'R01387',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01387',\n",
       "   'MetaCyc: UDP-GLUCURONATE-5-EPIMERASE-RXN',\n",
       "   \"Name: C-5-uronosyl epimerase; UDP-glucuronate 5'-epimerase; UDP-glucuronic acid 5'-epimerase; UDPglucuronate 5'-epimerase; uridine diphosphoglucuronate 5'-epimerase\"],\n",
       "  'code': '(1) cpd00144[0] <=> (1) cpd01556[0]',\n",
       "  'compound_ids': 'cpd00144;cpd01556',\n",
       "  'definition': '(1) UDPglucuronate[0] <=> (1) UDP-L-iduronate[0]',\n",
       "  'deltag': -0.6,\n",
       "  'deltagerr': 0.28,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.1.3.12'],\n",
       "  'equation': '(1) cpd00144[0] <=> (1) cpd01556[0]',\n",
       "  'id': 'rxn01010',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': \"UDP-glucuronate 5'-epimerase\",\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00520 (Amino sugar and nucleotide sugar metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00144:0:0:\"UDPglucuronate\";1:cpd01556:0:0:\"UDP-L-iduronate\"'},\n",
       " {'abbreviation': 'R01388',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: GLYCERATE-DEHYDROGENASE-RXN; HYDROXYPYRUVATE-REDUCTASE-RXN',\n",
       "   'BiGG: HPYRRx',\n",
       "   'BrachyCyc: GLYCERATE-DEHYDROGENASE-RXN; HYDROXYPYRUVATE-REDUCTASE-RXN',\n",
       "   'KEGG: R01388',\n",
       "   'MetaCyc: GLYCERATE-DEHYDROGENASE-RXN; HYDROXYPYRUVATE-REDUCTASE-RXN',\n",
       "   'Name: (R)-glycerate:NAD+ oxidoreductase; D-Glycerate:NAD+ 2-oxidoreductase; D-glycerate dehydrogenase; Hydroxypyruvate dehydrogenase; Hydroxypyruvate reductase; Hydroxypyruvate reductase (NADH); glycerate dehydrogenase; hydroxypyruvate reductase'],\n",
       "  'code': '(1) cpd00003[0] + (1) cpd00223[0] <=> (1) cpd00004[0] + (1) cpd00145[0]',\n",
       "  'compound_ids': 'cpd00003;cpd00004;cpd00067;cpd00145;cpd00223',\n",
       "  'definition': '(1) NAD[0] + (1) Glycerate[0] <=> (1) NADH[0] + (1) H+[0] + (1) Hydroxypyruvate[0]',\n",
       "  'deltag': 7.45,\n",
       "  'deltagerr': 0.1,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.26', '1.1.1.29', '1.1.1.81'],\n",
       "  'equation': '(1) cpd00003[0] + (1) cpd00223[0] <=> (1) cpd00004[0] + (1) cpd00067[0] + (1) cpd00145[0]',\n",
       "  'id': 'rxn01011',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn19849;rxn27900;rxn27943',\n",
       "  'name': 'D-Glycerate:NAD+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Energy-Metabolism (Generation of Precursor Metabolite and Energy); Formaldehyde-Assimilation (Formaldehyde Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-181 (photorespiration); PWY-8011 (L-serine biosynthesis II); Photosynthesis (); SERINE-BIOSYNTHESIS (L-serine Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism); rn00680 (Methane metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01200 (Carbon metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00003:0:0:\"NAD\";-1:cpd00223:0:0:\"Glycerate\";1:cpd00004:0:0:\"NADH\";1:cpd00067:0:0:\"H+\";1:cpd00145:0:0:\"Hydroxypyruvate\"'},\n",
       " {'abbreviation': 'R01390',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01390', 'Name: '],\n",
       "  'code': '(1) cpd00011[0] + (1) cpd00145[0] <=> (1) cpd00718[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00145;cpd00718',\n",
       "  'definition': '(1) CO2[0] + (1) Hydroxypyruvate[0] <= (1) H+[0] + (1) Dihydroxyfumarate[0]',\n",
       "  'deltag': -0.75,\n",
       "  'deltagerr': 1.0,\n",
       "  'direction': '<',\n",
       "  'ec_numbers': None,\n",
       "  'equation': '(1) cpd00011[0] + (1) cpd00145[0] <= (1) cpd00067[0] + (1) cpd00718[0]',\n",
       "  'id': 'rxn01012',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'R01390',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00011:0:0:\"CO2\";-1:cpd00145:0:0:\"Hydroxypyruvate\";1:cpd00067:0:0:\"H+\";1:cpd00718:0:0:\"Dihydroxyfumarate\"'},\n",
       " {'abbreviation': 'R01392',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: HYDROXYPYRUVATE-REDUCTASE-RXN; RXN0-300',\n",
       "   'BiGG: HPYRRy',\n",
       "   'BrachyCyc: HYDROXYPYRUVATE-REDUCTASE-RXN; RXN0-300',\n",
       "   'KEGG: R01392',\n",
       "   'MetaCyc: HYDROXYPYRUVATE-REDUCTASE-RXN; RXN0-300',\n",
       "   'Name: D-Glycerate:NADP+ 2-oxidoreductase; D-glycerate dehydrogenase; Hydroxypyruvate reductase; Hydroxypyruvate reductase (NADPH); Hydroxypyruvate reductase-(NADP); hydroxypyruvate reductase (NADP+)'],\n",
       "  'code': '(1) cpd00006[0] + (1) cpd00223[0] <=> (1) cpd00005[0] + (1) cpd00145[0]',\n",
       "  'compound_ids': 'cpd00005;cpd00006;cpd00067;cpd00145;cpd00223',\n",
       "  'definition': '(1) NADP[0] + (1) Glycerate[0] <=> (1) NADPH[0] + (1) H+[0] + (1) Hydroxypyruvate[0]',\n",
       "  'deltag': 7.46,\n",
       "  'deltagerr': 0.13,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['1.1.1.79', '1.1.1.81'],\n",
       "  'equation': '(1) cpd00006[0] + (1) cpd00223[0] <=> (1) cpd00005[0] + (1) cpd00067[0] + (1) cpd00145[0]',\n",
       "  'id': 'rxn01013',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn32531;rxn38885',\n",
       "  'name': 'D-Glycerate:NADP+ 2-oxidoreductase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); C1-COMPOUNDS (C1 Compound Utilization and Assimilation); Degradation (Degradation/Utilization/Assimilation); Formaldehyde-Assimilation (Formaldehyde Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); PWY-1622 (formaldehyde assimilation I (serine pathway)); PWY-8011 (L-serine biosynthesis II); SERINE-BIOSYNTHESIS (L-serine Biosynthesis)',\n",
       "   'KEGG: rn00260 (Glycine, serine and threonine metabolism); rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00006:0:0:\"NADP\";-1:cpd00223:0:0:\"Glycerate\";1:cpd00005:0:0:\"NADPH\";1:cpd00067:0:0:\"H+\";1:cpd00145:0:0:\"Hydroxypyruvate\"'},\n",
       " {'abbreviation': 'R01393',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01393',\n",
       "   'MetaCyc: HYDROXYPYRUVATE-DECARBOXYLASE-RXN',\n",
       "   'Name: Hydroxypyruvate carboxy-lyase; hydroxypyruvate carboxy-lyase; hydroxypyruvate carboxy-lyase (glycolaldehyde-forming); hydroxypyruvate decarboxylase'],\n",
       "  'code': '(1) cpd00145[0] <=> (1) cpd00011[0] + (1) cpd00229[0]',\n",
       "  'compound_ids': 'cpd00011;cpd00067;cpd00145;cpd00229',\n",
       "  'definition': '(1) H+[0] + (1) Hydroxypyruvate[0] => (1) CO2[0] + (1) Glycolaldehyde[0]',\n",
       "  'deltag': -4.93,\n",
       "  'deltagerr': 0.87,\n",
       "  'direction': '>',\n",
       "  'ec_numbers': ['4.1.1.40'],\n",
       "  'equation': '(1) cpd00067[0] + (1) cpd00145[0] => (1) cpd00011[0] + (1) cpd00229[0]',\n",
       "  'id': 'rxn01014',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'hydroxypyruvate carboxy-lyase (glycolaldehyde-forming)',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00067:0:0:\"H+\";-1:cpd00145:0:0:\"Hydroxypyruvate\";1:cpd00011:0:0:\"CO2\";1:cpd00229:0:0:\"Glycolaldehyde\"'},\n",
       " {'abbreviation': 'R01394',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['BiGG: HPYRI',\n",
       "   'EcoCyc: RXN0-305',\n",
       "   'KEGG: R01394',\n",
       "   'MetaCyc: RXN0-305',\n",
       "   'iAF1260: HPYRI',\n",
       "   'Name: Hydroxypyruvate ketol-isomerase; hydroxypyruvate aldose-ketose-isomerase; hydroxypyruvate isomerase'],\n",
       "  'code': '(1) cpd00145[0] <=> (1) cpd00843[0]',\n",
       "  'compound_ids': 'cpd00145;cpd00843',\n",
       "  'definition': '(1) Hydroxypyruvate[0] <=> (1) Tartronate semialdehyde[0]',\n",
       "  'deltag': 0.37,\n",
       "  'deltagerr': 0.42,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['5.3.1.22'],\n",
       "  'equation': '(1) cpd00145[0] <=> (1) cpd00843[0]',\n",
       "  'id': 'rxn01015',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'hydroxypyruvate aldose-ketose-isomerase',\n",
       "  'notes': ['GCC', 'EQC', 'EQU'],\n",
       "  'pathways': ['KEGG: rn00630 (Glyoxylate and dicarboxylate metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00145:0:0:\"Hydroxypyruvate\";1:cpd00843:0:0:\"Tartronate semialdehyde\"'},\n",
       " {'abbreviation': 'R01395',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraGEM: R_R01395_c',\n",
       "   'KEGG: R01395',\n",
       "   'Maize_C4GEM: R_R01395_c',\n",
       "   'MetaCyc: RXN-14196',\n",
       "   'iRS1597: R01395_c',\n",
       "   'Name: ATP:carbamate phosphotransferase'],\n",
       "  'code': '(1) cpd00002[0] + (1) cpd01101[0] <=> (1) cpd00008[0] + (1) cpd00146[0]',\n",
       "  'compound_ids': 'cpd00002;cpd00008;cpd00146;cpd01101',\n",
       "  'definition': '(1) ATP[0] + (1) Carbamate[0] <=> (1) ADP[0] + (1) Carbamoylphosphate[0]',\n",
       "  'deltag': 4.0,\n",
       "  'deltagerr': 0.5,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.7.2.2'],\n",
       "  'equation': '(1) cpd00002[0] + (1) cpd01101[0] <=> (1) cpd00008[0] + (1) cpd00146[0]',\n",
       "  'id': 'rxn01016',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'ATP:carbamate phosphotransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: AMINE-DEG (Amine and Polyamine Degradation); ARGDEGRAD-PWY (L-arginine degradation V (arginine deiminase pathway)); ARGININE-DEG (L-arginine Degradation); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); Allantoin-degradation (Allantoin Degradation); Amino-Acid-Degradation (Amino Acid Degradation); Antibiotic-Biosynthesis (Antibiotic Biosynthesis); CITRULLINE-DEG-PWY (L-citrulline degradation); Degradation (Degradation/Utilization/Assimilation); MISCELLANEOUS-DEG (Other Amino Acid Degradation); PWY-7693 (guadinomine B biosynthesis); PWY0-41 (allantoin degradation IV (anaerobic)); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation); SECONDARY-METABOLITE-BIOSYNTHESIS (Secondary Metabolite Biosynthesis)',\n",
       "   'KEGG: rn00910 (Nitrogen metabolism)'],\n",
       "  'reversibility': '=',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00002:0:0:\"ATP\";-1:cpd01101:0:0:\"Carbamate\";1:cpd00008:0:0:\"ADP\";1:cpd00146:0:0:\"Carbamoylphosphate\"'},\n",
       " {'abbreviation': 'R01396',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01396',\n",
       "   'MetaCyc: LYSINE-CARBAMOYLTRANSFERASE-RXN',\n",
       "   'Name: Carbamoyl-phosphate:L-lysine carbamoyltransferase; carbamoyl-phosphate:L-lysine carbamoyltransferase; lysine carbamoyltransferase; lysine transcarbamylase'],\n",
       "  'code': '(1) cpd00039[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd01606[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00039;cpd00067;cpd00146;cpd01606',\n",
       "  'definition': '(1) L-Lysine[0] + (1) Carbamoylphosphate[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) L-Homocitrulline[0]',\n",
       "  'deltag': -0.65,\n",
       "  'deltagerr': 1.16,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.3.8'],\n",
       "  'equation': '(1) cpd00039[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd01606[0]',\n",
       "  'id': 'rxn01017',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Carbamoyl-phosphate:L-lysine carbamoyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00039:0:0:\"L-Lysine\";-1:cpd00146:0:0:\"Carbamoylphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd01606:0:0:\"L-Homocitrulline\"'},\n",
       " {'abbreviation': 'R01397',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ASPCARBTRANS-RXN',\n",
       "   'BiGG: ASPCT; ASPCTn; URA2_1',\n",
       "   'BrachyCyc: ASPCARBTRANS-RXN',\n",
       "   'KEGG: R01397',\n",
       "   'MetaCyc: ASPCARBTRANS-RXN',\n",
       "   'Name: ATCase; Carbamoyl-phosphate:L-aspartate carbamoyltransferase; L-aspartate transcarbamoylase; L-aspartate transcarbamylase; aspartate carbamoyltransferase; aspartate carbamoyltransferase catalytic chain; aspartate carbamyltransferase; aspartate transcarbamoylase; aspartate transcarbamylase; aspartatecarbamoyltransferase; aspartic acid transcarbamoylase; aspartic carbamyltransferase; aspartic transcarbamylase; carbamoyl-phosphate:L-aspartate carbamoyltransferase; carbamoylaspartotranskinase; carbamylaspartotranskinase'],\n",
       "  'code': '(1) cpd00041[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00343[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00041;cpd00067;cpd00146;cpd00343',\n",
       "  'definition': '(1) L-Aspartate[0] + (1) Carbamoylphosphate[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) N-Carbamoyl-L-aspartate[0]',\n",
       "  'deltag': -5.07,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.3.2'],\n",
       "  'equation': '(1) cpd00041[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00343[0]',\n",
       "  'id': 'rxn01018',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn27725;rxn35448',\n",
       "  'name': 'carbamoyl-phosphate:L-aspartate carbamoyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': [\"MetaCyc: Deoxyribonucleotide-Biosynthesis (2'-Deoxyribonucleotide Biosynthesis); Nucleotide-Biosynthesis (Nucleoside and Nucleotide Biosynthesis); PRPP-PWY (superpathway of histidine, purine, and pyrimidine biosynthesis); PWY-5686 (UMP biosynthesis I); PWY-7211 (superpathway of pyrimidine deoxyribonucleotides <i>de novo</i> biosynthesis); PWY-7790 (UMP biosynthesis II); PWY-7791 (UMP biosynthesis III); PWY0-162 (superpathway of pyrimidine ribonucleotides <i>de novo</i> biosynthesis); PYR-NUC-SYN (Pyrimidine Nucleotide Biosynthesis); Pyrimid-Deoxyribonucleot-De-Novo-Biosyn (Pyrimidine Deoxyribonucleotide <i>De Novo</i> Biosynthesis); Pyrimid-Ribonucleot-De-Novo-Biosyn (Pyrimidine Ribonucleotide <i>De Novo</i> Biosynthesis); Pyrimidine-De-Novo-Biosyn (Pyrimidine Nucleotide <i>De Novo</i> Biosynthesis); UMP-Biosynthesis (UMP Biosynthesis)\",\n",
       "   'KEGG: rn00240 (Pyrimidine metabolism); rn00250 (Alanine, aspartate and glutamate metabolism)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00041:0:0:\"L-Aspartate\";-1:cpd00146:0:0:\"Carbamoylphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00343:0:0:\"N-Carbamoyl-L-aspartate\"'},\n",
       " {'abbreviation': 'R01398',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['AraCyc: ORNCARBAMTRANSFER-RXN; RXN-13482',\n",
       "   'BrachyCyc: ORNCARBAMTRANSFER-RXN',\n",
       "   'ChlamyCyc: ORNCARBAMTRANSFER-RXN',\n",
       "   'KEGG: R01398',\n",
       "   'MetaCyc: ORNCARBAMTRANSFER-RXN; RXN-13482',\n",
       "   'Name: Carbamoyl-phosphate:L-ornithine carbamoyltransferase; L-ornithine carbamoyltransferase; L-ornithine carbamyltransferase; L-ornithine transcarbamylase; OCBTi; OTC; carbamoyl-phosphate:L-ornithine carbamoyltransferase; carbamylphosphate-ornithine transcarbamylase; citrulline phosphorylase; ornithine carbamoyltransferase; ornithine carbamoyltransferase, irreversible; ornithine carbamyltransferase; ornithine transcarbamylase; ornithinecarbamoyltransferase-irreversible'],\n",
       "  'code': '(1) cpd00064[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00274[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00064;cpd00067;cpd00146;cpd00274',\n",
       "  'definition': '(1) Ornithine[0] + (1) Carbamoylphosphate[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) Citrulline[0]',\n",
       "  'deltag': -6.69,\n",
       "  'deltagerr': 0.71,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.3.3'],\n",
       "  'equation': '(1) cpd00064[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00274[0]',\n",
       "  'id': 'rxn01019',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': 'rxn20493;rxn28051',\n",
       "  'name': 'Carbamoyl-phosphate:L-ornithine carbamoyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': ['MetaCyc: ARG+POLYAMINE-SYN (superpathway of arginine and polyamine biosynthesis); ARGDEGRAD-PWY (L-arginine degradation V (arginine deiminase pathway)); ARGININE-DEG (L-arginine Degradation); ARGININE-SYN (L-arginine Biosynthesis); ARGORNPROST-PWY (L-arginine degradation (Stickland reaction)); ARGSYN-PWY (L-arginine biosynthesis I (via L-ornithine)); ARGSYNBSUB-PWY (L-arginine biosynthesis II (acetyl cycle)); Amino-Acid-Biosynthesis (Amino Acid Biosynthesis); Amino-Acid-Degradation (Amino Acid Degradation); CITRULBIO-PWY (L-citrulline biosynthesis); CITRULLINE-DEG-PWY (L-citrulline degradation); Citrulline-Biosynthesis (L-citrulline Biosynthesis); Degradation (Degradation/Utilization/Assimilation); IND-AMINO-ACID-SYN (Proteinogenic Amino Acid Biosynthesis); MISCELLANEOUS-DEG (Other Amino Acid Degradation); NITROGEN-DEG (Nitrogen Compound Metabolism); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); Other-Amino-Acid-Biosynthesis (Other Amino Acid Biosynthesis); PROLINE-SYN (L-proline Biosynthesis); PWY-4981 (L-proline biosynthesis II (from arginine)); PWY-4984 (urea cycle); PWY-5004 (superpathway of L-citrulline metabolism); PWY-7400 (L-arginine biosynthesis IV (archaebacteria)); Polyamine-Biosynthesis (Amine and Polyamine Biosynthesis); Proteinogenic-Amino-Acids-Degradation (Proteinogenic Amino Acid Degradation)',\n",
       "   'KEGG: rn00330 (Arginine and proline metabolism); rn01110 (Biosynthesis of secondary metabolites); rn01230 (Biosynthesis of amino acids)'],\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00064:0:0:\"Ornithine\";-1:cpd00146:0:0:\"Carbamoylphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00274:0:0:\"Citrulline\"'},\n",
       " {'abbreviation': 'R01399',\n",
       "  'abstract_reaction': None,\n",
       "  'aliases': ['KEGG: R01399',\n",
       "   'MetaCyc: PUTRESCINE-CARBAMOYLTRANSFERASE-RXN',\n",
       "   'Name: Carbamoyl-phosphate:putrescine carbamoyltransferase; PTCase; carbamoyl-phosphate:putrescine carbamoyltransferase; putrescine carbamoyltransferase; putrescine synthase; putrescine transcarbamylase'],\n",
       "  'code': '(1) cpd00118[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00341[0]',\n",
       "  'compound_ids': 'cpd00009;cpd00067;cpd00118;cpd00146;cpd00341',\n",
       "  'definition': '(1) Putrescine[0] + (1) Carbamoylphosphate[0] <=> (1) Phosphate[0] + (1) H+[0] + (1) N-Carbamoylputrescine[0]',\n",
       "  'deltag': 0.24,\n",
       "  'deltagerr': 1.08,\n",
       "  'direction': '=',\n",
       "  'ec_numbers': ['2.1.3.6'],\n",
       "  'equation': '(1) cpd00118[0] + (1) cpd00146[0] <=> (1) cpd00009[0] + (1) cpd00067[0] + (1) cpd00341[0]',\n",
       "  'id': 'rxn01020',\n",
       "  'is_obsolete': 0,\n",
       "  'is_transport': 0,\n",
       "  'linked_reaction': None,\n",
       "  'name': 'Carbamoyl-phosphate:putrescine carbamoyltransferase',\n",
       "  'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       "  'pathways': None,\n",
       "  'reversibility': '>',\n",
       "  'source': 'Primary Database',\n",
       "  'status': 'OK',\n",
       "  'stoichiometry': '-1:cpd00118:0:0:\"Putrescine\";-1:cpd00146:0:0:\"Carbamoylphosphate\";1:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\";1:cpd00341:0:0:\"N-Carbamoylputrescine\"'},\n",
       " ...]"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "parsed_json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "r = parsed_json[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'abbreviation': 'R00004',\n",
       " 'abstract_reaction': None,\n",
       " 'aliases': ['AraCyc: INORGPYROPHOSPHAT-RXN',\n",
       "  'BiGG: IPP1; PPA; PPA_1; PPAm',\n",
       "  'BrachyCyc: INORGPYROPHOSPHAT-RXN',\n",
       "  'KEGG: R00004',\n",
       "  'MetaCyc: INORGPYROPHOSPHAT-RXN',\n",
       "  'Name: Diphosphate phosphohydrolase; Inorganic diphosphatase; Inorganic pyrophosphatase; Pyrophosphate phosphohydrolase; diphosphate phosphohydrolase; inorganic diphosphatase; inorganic diphosphatase (one proton translocation); inorganicdiphosphatase; pyrophosphate phosphohydrolase'],\n",
       " 'code': '(1) cpd00001[0] + (1) cpd00012[0] <=> (2) cpd00009[0]',\n",
       " 'compound_ids': 'cpd00001;cpd00009;cpd00012;cpd00067',\n",
       " 'definition': '(1) H2O[0] + (1) PPi[0] <=> (2) Phosphate[0] + (1) H+[0]',\n",
       " 'deltag': -3.46,\n",
       " 'deltagerr': 0.05,\n",
       " 'direction': '=',\n",
       " 'ec_numbers': ['3.6.1.1'],\n",
       " 'equation': '(1) cpd00001[0] + (1) cpd00012[0] <=> (2) cpd00009[0] + (1) cpd00067[0]',\n",
       " 'id': 'rxn00001',\n",
       " 'is_obsolete': 0,\n",
       " 'is_transport': 0,\n",
       " 'linked_reaction': 'rxn27946;rxn27947;rxn27948;rxn32487;rxn38157;rxn38158',\n",
       " 'name': 'diphosphate phosphohydrolase',\n",
       " 'notes': ['GCC', 'HB', 'EQC', 'EQU'],\n",
       " 'pathways': ['MetaCyc: Degradation (Degradation/Utilization/Assimilation); Glyphosate-Degradation (glyphosate degradation); Noncarbon-Nutrients (Inorganic Nutrient Metabolism); PWY-7805 ((aminomethyl)phosphonate degradation); PWY-7807 (glyphosate degradation III); Phosphorus-Compounds (Phosphorus Compound Metabolism)'],\n",
       " 'reversibility': '>',\n",
       " 'source': 'Primary Database',\n",
       " 'status': 'OK',\n",
       " 'stoichiometry': '-1:cpd00001:0:0:\"H2O\";-1:cpd00012:0:0:\"PPi\";2:cpd00009:0:0:\"Phosphate\";1:cpd00067:0:0:\"H+\"'}"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'rxn00001'"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r['id']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['AraCyc: INORGPYROPHOSPHAT-RXN',\n",
       " 'BiGG: IPP1; PPA; PPA_1; PPAm',\n",
       " 'BrachyCyc: INORGPYROPHOSPHAT-RXN',\n",
       " 'KEGG: R00004',\n",
       " 'MetaCyc: INORGPYROPHOSPHAT-RXN',\n",
       " 'Name: Diphosphate phosphohydrolase; Inorganic diphosphatase; Inorganic pyrophosphatase; Pyrophosphate phosphohydrolase; diphosphate phosphohydrolase; inorganic diphosphatase; inorganic diphosphatase (one proton translocation); inorganicdiphosphatase; pyrophosphate phosphohydrolase']"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r['aliases']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['AraCyc', 'INORGPYROPHOSPHAT-RXN']\n",
      "['BiGG', 'IPP1; PPA; PPA_1; PPAm']\n",
      "['BrachyCyc', 'INORGPYROPHOSPHAT-RXN']\n",
      "['KEGG', 'R00004']\n",
      "['MetaCyc', 'INORGPYROPHOSPHAT-RXN']\n",
      "['Name', 'Diphosphate phosphohydrolase; Inorganic diphosphatase; Inorganic pyrophosphatase; Pyrophosphate phosphohydrolase; diphosphate phosphohydrolase; inorganic diphosphatase; inorganic diphosphatase (one proton translocation); inorganicdiphosphatase; pyrophosphate phosphohydrolase']\n"
     ]
    }
   ],
   "source": [
    "for a in r['aliases']:\n",
    "    print(a.split(': '))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [],
   "source": [
    "def _modelseed_to_dict(r):\n",
    "    res  = {\n",
    "        'id': r['id'],        \n",
    "        'is_obsolete' : r['is_obsolete']        \n",
    "    }\n",
    "    if 'aliases' in r and r['aliases'] is not None:\n",
    "        for a in r['aliases']:\n",
    "            spl = a.split(': ')\n",
    "            if len(spl) != 2 and spl[0] != 'Name':\n",
    "                print (spl)\n",
    "            res[spl[0]] = spl[1].split('; ')\n",
    "    return res\n",
    "\n",
    "dfseed = pd.DataFrame([_modelseed_to_dict(r) for r in parsed_json])\n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfseed = dfseed.loc[dfseed.is_obsolete == 0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>is_obsolete</th>\n",
       "      <th>AraCyc</th>\n",
       "      <th>BiGG</th>\n",
       "      <th>BrachyCyc</th>\n",
       "      <th>KEGG</th>\n",
       "      <th>MetaCyc</th>\n",
       "      <th>Name</th>\n",
       "      <th>ChlamyCyc</th>\n",
       "      <th>AlgaGEM</th>\n",
       "      <th>...</th>\n",
       "      <th>iAbaylyiv4</th>\n",
       "      <th>iJR904</th>\n",
       "      <th>iMEO21</th>\n",
       "      <th>iNJ661</th>\n",
       "      <th>iSB619</th>\n",
       "      <th>iRR1083</th>\n",
       "      <th>iSO783</th>\n",
       "      <th>iND750</th>\n",
       "      <th>iGT196</th>\n",
       "      <th>iPS189</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>rxn00001</td>\n",
       "      <td>0</td>\n",
       "      <td>[INORGPYROPHOSPHAT-RXN]</td>\n",
       "      <td>[IPP1, PPA, PPA_1, PPAm]</td>\n",
       "      <td>[INORGPYROPHOSPHAT-RXN]</td>\n",
       "      <td>[R00004]</td>\n",
       "      <td>[INORGPYROPHOSPHAT-RXN]</td>\n",
       "      <td>[Diphosphate phosphohydrolase, Inorganic dipho...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>rxn00002</td>\n",
       "      <td>0</td>\n",
       "      <td>[ALLOPHANATE-HYDROLASE-RXN]</td>\n",
       "      <td>[ALPHNH, DUR1_2]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R00005]</td>\n",
       "      <td>[ALLOPHANATE-HYDROLASE-RXN]</td>\n",
       "      <td>[Allophanate hydrolase, AtzF, TrzF, Urea-1-car...</td>\n",
       "      <td>[ALLOPHANATE-HYDROLASE-RXN]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>rxn00003</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[ILV2_2]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R00006]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[2-acetolactate pyruvate-lyase (carboxylating)...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R_R00006_c]</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>rxn00004</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R00008]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[4-Hydroxy-4-methyl-2-oxoglutarate pyruvate-ly...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>rxn00006</td>\n",
       "      <td>0</td>\n",
       "      <td>[CATAL-RXN]</td>\n",
       "      <td>[CAT, CATp, CTA1, CTT1]</td>\n",
       "      <td>[CATAL-RXN]</td>\n",
       "      <td>[R00009]</td>\n",
       "      <td>[CATAL-RXN, RXN-12121]</td>\n",
       "      <td>[Catalase, caperase, catalase A, catalase-pero...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43769</th>\n",
       "      <td>rxn48571</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R02164]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[Succinate:ubiquinone oxidoreductase]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43770</th>\n",
       "      <td>rxn48572</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[S-adenosyl-L-methione:demethylmenaquinone met...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R_R04993_c]</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43771</th>\n",
       "      <td>rxn48573</td>\n",
       "      <td>0</td>\n",
       "      <td>[3.5.1.88-RXN]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[3.5.1.88-RXN]</td>\n",
       "      <td>[R05635]</td>\n",
       "      <td>[3.5.1.88-RXN]</td>\n",
       "      <td>[3.5.1.88-RXN.d, Formyl-L-methionyl peptide am...</td>\n",
       "      <td>[3.5.1.88-RXN]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43772</th>\n",
       "      <td>rxn48574</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R03874]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43773</th>\n",
       "      <td>rxn48575</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[RXN1G-874]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>36197 rows × 49 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             id  is_obsolete                       AraCyc  \\\n",
       "0      rxn00001            0      [INORGPYROPHOSPHAT-RXN]   \n",
       "1      rxn00002            0  [ALLOPHANATE-HYDROLASE-RXN]   \n",
       "2      rxn00003            0                          NaN   \n",
       "3      rxn00004            0                          NaN   \n",
       "4      rxn00006            0                  [CATAL-RXN]   \n",
       "...         ...          ...                          ...   \n",
       "43769  rxn48571            0                          NaN   \n",
       "43770  rxn48572            0                          NaN   \n",
       "43771  rxn48573            0               [3.5.1.88-RXN]   \n",
       "43772  rxn48574            0                          NaN   \n",
       "43773  rxn48575            0                          NaN   \n",
       "\n",
       "                           BiGG                BrachyCyc      KEGG  \\\n",
       "0      [IPP1, PPA, PPA_1, PPAm]  [INORGPYROPHOSPHAT-RXN]  [R00004]   \n",
       "1              [ALPHNH, DUR1_2]                      NaN  [R00005]   \n",
       "2                      [ILV2_2]                      NaN  [R00006]   \n",
       "3                           NaN                      NaN  [R00008]   \n",
       "4       [CAT, CATp, CTA1, CTT1]              [CATAL-RXN]  [R00009]   \n",
       "...                         ...                      ...       ...   \n",
       "43769                       NaN                      NaN  [R02164]   \n",
       "43770                       NaN                      NaN       NaN   \n",
       "43771                       NaN           [3.5.1.88-RXN]  [R05635]   \n",
       "43772                       NaN                      NaN  [R03874]   \n",
       "43773                       NaN                      NaN       NaN   \n",
       "\n",
       "                           MetaCyc  \\\n",
       "0          [INORGPYROPHOSPHAT-RXN]   \n",
       "1      [ALLOPHANATE-HYDROLASE-RXN]   \n",
       "2                              NaN   \n",
       "3                              NaN   \n",
       "4           [CATAL-RXN, RXN-12121]   \n",
       "...                            ...   \n",
       "43769                          NaN   \n",
       "43770                          NaN   \n",
       "43771               [3.5.1.88-RXN]   \n",
       "43772                          NaN   \n",
       "43773                  [RXN1G-874]   \n",
       "\n",
       "                                                    Name  \\\n",
       "0      [Diphosphate phosphohydrolase, Inorganic dipho...   \n",
       "1      [Allophanate hydrolase, AtzF, TrzF, Urea-1-car...   \n",
       "2      [2-acetolactate pyruvate-lyase (carboxylating)...   \n",
       "3      [4-Hydroxy-4-methyl-2-oxoglutarate pyruvate-ly...   \n",
       "4      [Catalase, caperase, catalase A, catalase-pero...   \n",
       "...                                                  ...   \n",
       "43769              [Succinate:ubiquinone oxidoreductase]   \n",
       "43770  [S-adenosyl-L-methione:demethylmenaquinone met...   \n",
       "43771  [3.5.1.88-RXN.d, Formyl-L-methionyl peptide am...   \n",
       "43772                                                NaN   \n",
       "43773                                                NaN   \n",
       "\n",
       "                         ChlamyCyc       AlgaGEM  ... iAbaylyiv4 iJR904  \\\n",
       "0                              NaN           NaN  ...        NaN    NaN   \n",
       "1      [ALLOPHANATE-HYDROLASE-RXN]           NaN  ...        NaN    NaN   \n",
       "2                              NaN  [R_R00006_c]  ...        NaN    NaN   \n",
       "3                              NaN           NaN  ...        NaN    NaN   \n",
       "4                              NaN           NaN  ...        NaN    NaN   \n",
       "...                            ...           ...  ...        ...    ...   \n",
       "43769                          NaN           NaN  ...        NaN    NaN   \n",
       "43770                          NaN  [R_R04993_c]  ...        NaN    NaN   \n",
       "43771               [3.5.1.88-RXN]           NaN  ...        NaN    NaN   \n",
       "43772                          NaN           NaN  ...        NaN    NaN   \n",
       "43773                          NaN           NaN  ...        NaN    NaN   \n",
       "\n",
       "      iMEO21 iNJ661 iSB619 iRR1083 iSO783 iND750 iGT196 iPS189  \n",
       "0        NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "1        NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "2        NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "3        NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "4        NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "...      ...    ...    ...     ...    ...    ...    ...    ...  \n",
       "43769    NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "43770    NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "43771    NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "43772    NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "43773    NaN    NaN    NaN     NaN    NaN    NaN    NaN    NaN  \n",
       "\n",
       "[36197 rows x 49 columns]"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfseed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfseed = dfseed.explode('BiGG')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfseed_cmp = dfseed[['id','BiGG']].drop_duplicates(subset=['id','BiGG']).dropna(subset='BiGG')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>reaction</th>\n",
       "      <th>rid</th>\n",
       "      <th>genes</th>\n",
       "      <th>genes_number</th>\n",
       "      <th>sbo</th>\n",
       "      <th>metanetx.reaction</th>\n",
       "      <th>biocyc</th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>ec-code</th>\n",
       "      <th>rhea</th>\n",
       "      <th>kegg.reaction</th>\n",
       "      <th>reactome</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR120tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94675</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR140tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94676</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR141tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94677</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR161tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94679</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1,2 diacylglycerol transport via flipping (per...</td>\n",
       "      <td>12DGR180tipp</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94680</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2335</th>\n",
       "      <td>R_EX_xtsn_e</td>\n",
       "      <td>EX_xtsn_e</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000627</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2336</th>\n",
       "      <td>R_EX_xyl__D_e</td>\n",
       "      <td>EX_xyl__D_e</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000627</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2337</th>\n",
       "      <td>R_EX_zn2_e</td>\n",
       "      <td>EX_zn2_e</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000627</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2338</th>\n",
       "      <td>Biomass reaction</td>\n",
       "      <td>Growth</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000629</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2339</th>\n",
       "      <td>R_ATPM</td>\n",
       "      <td>ATPM</td>\n",
       "      <td>[]</td>\n",
       "      <td>0</td>\n",
       "      <td>SBO:0000630</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2340 rows × 12 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               reaction           rid genes  \\\n",
       "0     1,2 diacylglycerol transport via flipping (per...  12DGR120tipp    []   \n",
       "1     1,2 diacylglycerol transport via flipping (per...  12DGR140tipp    []   \n",
       "2     1,2 diacylglycerol transport via flipping (per...  12DGR141tipp    []   \n",
       "3     1,2 diacylglycerol transport via flipping (per...  12DGR161tipp    []   \n",
       "4     1,2 diacylglycerol transport via flipping (per...  12DGR180tipp    []   \n",
       "...                                                 ...           ...   ...   \n",
       "2335                                        R_EX_xtsn_e     EX_xtsn_e    []   \n",
       "2336                                      R_EX_xyl__D_e   EX_xyl__D_e    []   \n",
       "2337                                         R_EX_zn2_e      EX_zn2_e    []   \n",
       "2338                                   Biomass reaction        Growth    []   \n",
       "2339                                             R_ATPM          ATPM    []   \n",
       "\n",
       "      genes_number          sbo metanetx.reaction biocyc seed.reaction  \\\n",
       "0                0  SBO:0000185         MNXR94675    NaN           NaN   \n",
       "1                0  SBO:0000185         MNXR94676    NaN           NaN   \n",
       "2                0  SBO:0000185         MNXR94677    NaN           NaN   \n",
       "3                0  SBO:0000185         MNXR94679    NaN           NaN   \n",
       "4                0  SBO:0000185         MNXR94680    NaN           NaN   \n",
       "...            ...          ...               ...    ...           ...   \n",
       "2335             0  SBO:0000627               NaN    NaN           NaN   \n",
       "2336             0  SBO:0000627               NaN    NaN           NaN   \n",
       "2337             0  SBO:0000627               NaN    NaN           NaN   \n",
       "2338             0  SBO:0000629               NaN    NaN           NaN   \n",
       "2339             0  SBO:0000630               NaN    NaN           NaN   \n",
       "\n",
       "     ec-code rhea kegg.reaction reactome  \n",
       "0        NaN  NaN           NaN      NaN  \n",
       "1        NaN  NaN           NaN      NaN  \n",
       "2        NaN  NaN           NaN      NaN  \n",
       "3        NaN  NaN           NaN      NaN  \n",
       "4        NaN  NaN           NaN      NaN  \n",
       "...      ...  ...           ...      ...  \n",
       "2335     NaN  NaN           NaN      NaN  \n",
       "2336     NaN  NaN           NaN      NaN  \n",
       "2337     NaN  NaN           NaN      NaN  \n",
       "2338     NaN  NaN           NaN      NaN  \n",
       "2339     NaN  NaN           NaN      NaN  \n",
       "\n",
       "[2340 rows x 12 columns]"
      ]
     },
     "execution_count": 117,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfmerge = pd.merge(df, dfseed_cmp, left_on='rid', right_on='BiGG', how='left' )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "983"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfmerge.BiGG.isna().sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfmerge = dfmerge.explode('genes')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['reaction', 'rid', 'genes', 'genes_number', 'sbo', 'metanetx.reaction',\n",
       "       'biocyc', 'seed.reaction', 'ec-code', 'rhea', 'kegg.reaction',\n",
       "       'reactome', 'id', 'BiGG'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 121,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfmerge.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>rid</th>\n",
       "      <th>id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>12PPDRtpp</td>\n",
       "      <td>rxn07919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>12PPDStpp</td>\n",
       "      <td>rxn07921</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>13PPDH</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>13PPDH2</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>13PPDH2_1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2136</th>\n",
       "      <td>ZNabcpp</td>\n",
       "      <td>rxn05150</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2137</th>\n",
       "      <td>Zn2tex</td>\n",
       "      <td>rxn09390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2138</th>\n",
       "      <td>ilvg</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2139</th>\n",
       "      <td>r0191</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2140</th>\n",
       "      <td>r2465_1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1842 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "            rid        id\n",
       "6     12PPDRtpp  rxn07919\n",
       "8     12PPDStpp  rxn07921\n",
       "9        13PPDH       NaN\n",
       "10      13PPDH2       NaN\n",
       "11    13PPDH2_1       NaN\n",
       "...         ...       ...\n",
       "2136    ZNabcpp  rxn05150\n",
       "2137     Zn2tex  rxn09390\n",
       "2138       ilvg       NaN\n",
       "2139      r0191       NaN\n",
       "2140    r2465_1       NaN\n",
       "\n",
       "[1842 rows x 2 columns]"
      ]
     },
     "execution_count": 122,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfmerge.loc[dfmerge.genes_number >0, ['rid', 'id']].drop_duplicates()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([nan, 'G_WP_049588165_1', 'G_WP_061439201_1', ...,\n",
       "       'G_WP_061485725_1', 'G_WP_014979112_1', 'G_WP_061486830_1'],\n",
       "      dtype=object)"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfmerge.genes.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfgene2id = dfmerge[['genes', 'id']].dropna()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfgene2id['protein_id'] = dfgene2id['genes'].str.replace(r'^G_', '', regex=True).str.replace(r'_1$', '.1', regex=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [],
   "source": [
    "mapdf = pd.read_csv(os.path.join(model_dpath, 'data','HOT1A3_NCBI_gene_list.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>locus_tag</th>\n",
       "      <th>type</th>\n",
       "      <th>gene</th>\n",
       "      <th>gene_biotype</th>\n",
       "      <th>old_locus_tag</th>\n",
       "      <th>product</th>\n",
       "      <th>protein_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ACZ81_RS00010</td>\n",
       "      <td>gene</td>\n",
       "      <td>dnaN</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_00010</td>\n",
       "      <td>DNA polymerase III subunit beta</td>\n",
       "      <td>WP_012516526.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ACZ81_RS00015</td>\n",
       "      <td>gene</td>\n",
       "      <td>recF</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_00015</td>\n",
       "      <td>DNA replication/repair protein RecF</td>\n",
       "      <td>WP_061438857.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ACZ81_RS00020</td>\n",
       "      <td>gene</td>\n",
       "      <td>gyrB</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_00020</td>\n",
       "      <td>DNA topoisomerase (ATP-hydrolyzing) subunit B</td>\n",
       "      <td>WP_014977848.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ACZ81_RS00025</td>\n",
       "      <td>gene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_00025</td>\n",
       "      <td>hypothetical protein</td>\n",
       "      <td>WP_039234509.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ACZ81_RS00030</td>\n",
       "      <td>gene</td>\n",
       "      <td>glyS</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_00030</td>\n",
       "      <td>glycine--tRNA ligase subunit beta</td>\n",
       "      <td>WP_061438859.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4171</th>\n",
       "      <td>ACZ81_RS20715</td>\n",
       "      <td>pseudogene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>pseudogene</td>\n",
       "      <td>ACZ81_20715</td>\n",
       "      <td>cupredoxin domain-containing protein</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4172</th>\n",
       "      <td>ACZ81_RS20720</td>\n",
       "      <td>gene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_20720</td>\n",
       "      <td>DUF2933 domain-containing protein</td>\n",
       "      <td>WP_012516909.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4173</th>\n",
       "      <td>ACZ81_RS20725</td>\n",
       "      <td>pseudogene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>pseudogene</td>\n",
       "      <td>ACZ81_20725</td>\n",
       "      <td>isoprenylcysteine carboxylmethyltransferase fa...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4174</th>\n",
       "      <td>ACZ81_RS20730</td>\n",
       "      <td>gene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_20730</td>\n",
       "      <td>hypothetical protein</td>\n",
       "      <td>WP_012516911.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4175</th>\n",
       "      <td>ACZ81_RS21070</td>\n",
       "      <td>pseudogene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>pseudogene</td>\n",
       "      <td>NaN</td>\n",
       "      <td>cation transporter</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4176 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          locus_tag        type  gene    gene_biotype old_locus_tag  \\\n",
       "0     ACZ81_RS00010        gene  dnaN  protein_coding   ACZ81_00010   \n",
       "1     ACZ81_RS00015        gene  recF  protein_coding   ACZ81_00015   \n",
       "2     ACZ81_RS00020        gene  gyrB  protein_coding   ACZ81_00020   \n",
       "3     ACZ81_RS00025        gene   NaN  protein_coding   ACZ81_00025   \n",
       "4     ACZ81_RS00030        gene  glyS  protein_coding   ACZ81_00030   \n",
       "...             ...         ...   ...             ...           ...   \n",
       "4171  ACZ81_RS20715  pseudogene   NaN      pseudogene   ACZ81_20715   \n",
       "4172  ACZ81_RS20720        gene   NaN  protein_coding   ACZ81_20720   \n",
       "4173  ACZ81_RS20725  pseudogene   NaN      pseudogene   ACZ81_20725   \n",
       "4174  ACZ81_RS20730        gene   NaN  protein_coding   ACZ81_20730   \n",
       "4175  ACZ81_RS21070  pseudogene   NaN      pseudogene           NaN   \n",
       "\n",
       "                                                product      protein_id  \n",
       "0                       DNA polymerase III subunit beta  WP_012516526.1  \n",
       "1                   DNA replication/repair protein RecF  WP_061438857.1  \n",
       "2         DNA topoisomerase (ATP-hydrolyzing) subunit B  WP_014977848.1  \n",
       "3                                  hypothetical protein  WP_039234509.1  \n",
       "4                     glycine--tRNA ligase subunit beta  WP_061438859.1  \n",
       "...                                                 ...             ...  \n",
       "4171               cupredoxin domain-containing protein             NaN  \n",
       "4172                  DUF2933 domain-containing protein  WP_012516909.1  \n",
       "4173  isoprenylcysteine carboxylmethyltransferase fa...             NaN  \n",
       "4174                               hypothetical protein  WP_012516911.1  \n",
       "4175                                 cation transporter             NaN  \n",
       "\n",
       "[4176 rows x 7 columns]"
      ]
     },
     "execution_count": 127,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mapdf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfgene2id = pd.merge(dfgene2id, mapdf[['locus_tag', 'protein_id']], how='left', on='protein_id')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfgene2id1 =  dfgene2id[['locus_tag', 'id']].dropna().drop_duplicates()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['G_spontaneous'], dtype=object)"
      ]
     },
     "execution_count": 130,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfgene2id.loc[dfgene2id.locus_tag.isna()].genes.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "ACZ81_RS18370    40\n",
       "ACZ81_RS19805    40\n",
       "ACZ81_RS09690    29\n",
       "ACZ81_RS06240    29\n",
       "ACZ81_RS13410    26\n",
       "                 ..\n",
       "ACZ81_RS02395     1\n",
       "ACZ81_RS17550     1\n",
       "ACZ81_RS02370     1\n",
       "ACZ81_RS14480     1\n",
       "ACZ81_RS19410     1\n",
       "Name: locus_tag, Length: 867, dtype: int64"
      ]
     },
     "execution_count": 131,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfgene2id1.locus_tag.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [],
   "source": [
    "dfgene2id1.to_csv(os.path.join(model_dpath, 'data','carveme_gene2reaction.tsv'), sep='\\t', header=False, index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>genes</th>\n",
       "      <th>id</th>\n",
       "      <th>protein_id</th>\n",
       "      <th>locus_tag</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08063</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08184</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08186</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08197</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08234</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05467</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08241</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05212</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>647</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08428</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08475</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>752</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05292</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>780</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08525</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>816</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08589</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08623</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>908</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05581</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>971</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08682</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>980</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05319</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>986</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08691</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1087</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08737</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1089</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08730</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1100</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08747</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1147</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05206</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08786</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08856</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1279</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05616</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1379</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08947</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1388</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08956</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1439</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08985</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1461</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05466</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1466</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08991</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1480</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn08999</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1483</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09005</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1736</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09121</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1779</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09172</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1844</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09218</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1921</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09260</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1924</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05238</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2019</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09330</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2076</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn05245</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2098</th>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>rxn09390</td>\n",
       "      <td>WP_014951106.1</td>\n",
       "      <td>ACZ81_RS18370</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                 genes        id      protein_id      locus_tag\n",
       "162   G_WP_014951106_1  rxn08063  WP_014951106.1  ACZ81_RS18370\n",
       "411   G_WP_014951106_1  rxn08184  WP_014951106.1  ACZ81_RS18370\n",
       "414   G_WP_014951106_1  rxn08186  WP_014951106.1  ACZ81_RS18370\n",
       "426   G_WP_014951106_1  rxn08197  WP_014951106.1  ACZ81_RS18370\n",
       "450   G_WP_014951106_1  rxn08234  WP_014951106.1  ACZ81_RS18370\n",
       "455   G_WP_014951106_1  rxn05467  WP_014951106.1  ACZ81_RS18370\n",
       "460   G_WP_014951106_1  rxn08241  WP_014951106.1  ACZ81_RS18370\n",
       "471   G_WP_014951106_1  rxn05212  WP_014951106.1  ACZ81_RS18370\n",
       "647   G_WP_014951106_1  rxn08428  WP_014951106.1  ACZ81_RS18370\n",
       "735   G_WP_014951106_1  rxn08475  WP_014951106.1  ACZ81_RS18370\n",
       "752   G_WP_014951106_1  rxn05292  WP_014951106.1  ACZ81_RS18370\n",
       "780   G_WP_014951106_1  rxn08525  WP_014951106.1  ACZ81_RS18370\n",
       "816   G_WP_014951106_1  rxn08589  WP_014951106.1  ACZ81_RS18370\n",
       "846   G_WP_014951106_1  rxn08623  WP_014951106.1  ACZ81_RS18370\n",
       "908   G_WP_014951106_1  rxn05581  WP_014951106.1  ACZ81_RS18370\n",
       "971   G_WP_014951106_1  rxn08682  WP_014951106.1  ACZ81_RS18370\n",
       "980   G_WP_014951106_1  rxn05319  WP_014951106.1  ACZ81_RS18370\n",
       "986   G_WP_014951106_1  rxn08691  WP_014951106.1  ACZ81_RS18370\n",
       "1087  G_WP_014951106_1  rxn08737  WP_014951106.1  ACZ81_RS18370\n",
       "1089  G_WP_014951106_1  rxn08730  WP_014951106.1  ACZ81_RS18370\n",
       "1100  G_WP_014951106_1  rxn08747  WP_014951106.1  ACZ81_RS18370\n",
       "1147  G_WP_014951106_1  rxn05206  WP_014951106.1  ACZ81_RS18370\n",
       "1168  G_WP_014951106_1  rxn08786  WP_014951106.1  ACZ81_RS18370\n",
       "1198  G_WP_014951106_1  rxn08856  WP_014951106.1  ACZ81_RS18370\n",
       "1279  G_WP_014951106_1  rxn05616  WP_014951106.1  ACZ81_RS18370\n",
       "1379  G_WP_014951106_1  rxn08947  WP_014951106.1  ACZ81_RS18370\n",
       "1388  G_WP_014951106_1  rxn08956  WP_014951106.1  ACZ81_RS18370\n",
       "1439  G_WP_014951106_1  rxn08985  WP_014951106.1  ACZ81_RS18370\n",
       "1461  G_WP_014951106_1  rxn05466  WP_014951106.1  ACZ81_RS18370\n",
       "1466  G_WP_014951106_1  rxn08991  WP_014951106.1  ACZ81_RS18370\n",
       "1480  G_WP_014951106_1  rxn08999  WP_014951106.1  ACZ81_RS18370\n",
       "1483  G_WP_014951106_1  rxn09005  WP_014951106.1  ACZ81_RS18370\n",
       "1736  G_WP_014951106_1  rxn09121  WP_014951106.1  ACZ81_RS18370\n",
       "1779  G_WP_014951106_1  rxn09172  WP_014951106.1  ACZ81_RS18370\n",
       "1844  G_WP_014951106_1  rxn09218  WP_014951106.1  ACZ81_RS18370\n",
       "1921  G_WP_014951106_1  rxn09260  WP_014951106.1  ACZ81_RS18370\n",
       "1924  G_WP_014951106_1  rxn05238  WP_014951106.1  ACZ81_RS18370\n",
       "2019  G_WP_014951106_1  rxn09330  WP_014951106.1  ACZ81_RS18370\n",
       "2076  G_WP_014951106_1  rxn05245  WP_014951106.1  ACZ81_RS18370\n",
       "2098  G_WP_014951106_1  rxn09390  WP_014951106.1  ACZ81_RS18370"
      ]
     },
     "execution_count": 141,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfgene2id.loc[dfgene2id.locus_tag.isin(['ACZ81_RS18370'])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>reaction</th>\n",
       "      <th>rid</th>\n",
       "      <th>genes</th>\n",
       "      <th>genes_number</th>\n",
       "      <th>sbo</th>\n",
       "      <th>metanetx.reaction</th>\n",
       "      <th>biocyc</th>\n",
       "      <th>seed.reaction</th>\n",
       "      <th>ec-code</th>\n",
       "      <th>rhea</th>\n",
       "      <th>kegg.reaction</th>\n",
       "      <th>reactome</th>\n",
       "      <th>id</th>\n",
       "      <th>BiGG</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>3-methylbutanoic acid transport via diffusion ...</td>\n",
       "      <td>3MBtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR94918</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>4-hydroxybenzaldehyde transport via diffusion ...</td>\n",
       "      <td>4HBALDtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR95006</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>4-hydroxy-benzyl alcohol transport via diffusi...</td>\n",
       "      <td>4HBAtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR95007</td>\n",
       "      <td>META:TRANS-RXN0-274</td>\n",
       "      <td>rxn12376</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>4-Hydroxybenzoate transport (extracellular)</td>\n",
       "      <td>4HBZtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR95013</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn09792, rxn12787]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>Acetate transport via diffusion (extracellular...</td>\n",
       "      <td>ACtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR95431</td>\n",
       "      <td>[META:RXN0-1981, META:TRANS-RXN0-567]</td>\n",
       "      <td>[rxn08063, rxn09787, rxn09866, rxn10904]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[27814, 27815, 27816, 27817]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08063</td>\n",
       "      <td>ACtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>Arsenate transporter (extracellular to periplasm)</td>\n",
       "      <td>ASO4tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96076</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn05506</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>Butyrate transport via diffusion (extracellula...</td>\n",
       "      <td>BUTtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96352</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08184</td>\n",
       "      <td>BUTtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>Benzoate transport via diffusion (extracellula...</td>\n",
       "      <td>BZtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96370</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn12436</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[32811, 32812, 32813, 32814]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>Calcium (Ca+2) transport via diffusion (extrac...</td>\n",
       "      <td>CA2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96436</td>\n",
       "      <td>[META:TRANS-RXN-144, META:TRANS-RXN0-497]</td>\n",
       "      <td>rxn08186</td>\n",
       "      <td>3.6.3.8</td>\n",
       "      <td>[29671, 29672, 29673, 29674]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-2685505, R-ATH-418309, R-BTA-139854, R-...</td>\n",
       "      <td>rxn08186</td>\n",
       "      <td>CA2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>Cadmium (Cd+2) transport via diffusion (extrac...</td>\n",
       "      <td>CD2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96499</td>\n",
       "      <td>META:RXN0-10</td>\n",
       "      <td>[rxn08197, rxn08198]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28707, 28708, 28709, 28710]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08197</td>\n",
       "      <td>CD2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>Chloride (Cl-1) transport via diffusion (extra...</td>\n",
       "      <td>CLtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96797</td>\n",
       "      <td>META:TRANS-RXN-139</td>\n",
       "      <td>[rxn08234, rxn10473]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29823, 29824, 29825, 29826]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-2684901, R-ATH-427570, R-BTA-2684901, R...</td>\n",
       "      <td>rxn08234</td>\n",
       "      <td>CLtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>CO2 transport via diffusion (extracellular to ...</td>\n",
       "      <td>CO2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>3</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96810</td>\n",
       "      <td>META:TRANS-RXN0-545</td>\n",
       "      <td>[rxn05467, rxn08237, rxn08238, rxn09706, rxn09...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-1237042, R-ATH-1247649, R-BTA-1237042, ...</td>\n",
       "      <td>rxn05467</td>\n",
       "      <td>CO2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>Cobalt (Co+2) transport via diffusion (extrace...</td>\n",
       "      <td>COBALT2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96821</td>\n",
       "      <td>META:TRANS-RXN-141A</td>\n",
       "      <td>[rxn08241, rxn08242, rxn10474]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28578, 28579, 28580, 28581]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08241</td>\n",
       "      <td>COBALT2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>Cromate transport via diffusion (extracellular...</td>\n",
       "      <td>CRO4tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96914</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[32819, 32820, 32821, 32822]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>Copper (Cu+2) transport via diffusion (extrace...</td>\n",
       "      <td>CU2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR96951</td>\n",
       "      <td>META:RXN0-14</td>\n",
       "      <td>[rxn05212, rxn08273, rxn08274]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28703, 28704, 28705, 28706]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-BTA-437300, R-CFA-437300, R-DME-437300, R-D...</td>\n",
       "      <td>rxn05212</td>\n",
       "      <td>CU2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>785</th>\n",
       "      <td>Ethanol transport via diffusion (extracellular...</td>\n",
       "      <td>ETOHtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR97980</td>\n",
       "      <td>META:TRANS-RXN0-546</td>\n",
       "      <td>[rxn08428, rxn09683, rxn09764]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[35267, 35268, 35269, 35270]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08428</td>\n",
       "      <td>ETOHtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>862</th>\n",
       "      <td>Iron (II) transport via diffusion (extracellul...</td>\n",
       "      <td>FE2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>3</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR99505</td>\n",
       "      <td>META:TRANS-RXN-8</td>\n",
       "      <td>[rxn08475, rxn08476, rxn10963, rxn13286]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28486, 28487, 28488, 28489]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08475</td>\n",
       "      <td>FE2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>874</th>\n",
       "      <td>Iron (III) transport via diffusion (extracellu...</td>\n",
       "      <td>FE3tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR99522</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn05292, rxn08492]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34971, 34972, 34973, 34974]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-BTA-442368, R-CFA-442368, R-GGA-442368, R-H...</td>\n",
       "      <td>rxn05292</td>\n",
       "      <td>FE3tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>897</th>\n",
       "      <td>Formate transport via diffusion (extracellular...</td>\n",
       "      <td>FORtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR99620</td>\n",
       "      <td>META:TRANS-RXN-1</td>\n",
       "      <td>[rxn08525, rxn08526, rxn09682, rxn09754]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29679, 29680, 29681, 29682]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>R-HSA-6803255</td>\n",
       "      <td>rxn08525</td>\n",
       "      <td>FORtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>933</th>\n",
       "      <td>D-galacturonate transport via diffusion (extra...</td>\n",
       "      <td>GALURtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100021</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08589</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08589</td>\n",
       "      <td>GALURtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>961</th>\n",
       "      <td>D-glucuronat transport via diffusion (extracel...</td>\n",
       "      <td>GLCURtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100235</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08623</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08623</td>\n",
       "      <td>GLCURtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1014</th>\n",
       "      <td>Glycerol transport via diffusion (extracellula...</td>\n",
       "      <td>GLYCtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100343</td>\n",
       "      <td>META:TRANS-RXN-131</td>\n",
       "      <td>[rxn05581, rxn08654, rxn08658]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29675, 29676, 29677, 29678]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-BTA-432049, R-BTA-432074, R-BTA-507869, R-B...</td>\n",
       "      <td>rxn05581</td>\n",
       "      <td>GLYCtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1086</th>\n",
       "      <td>Guanine transport via diffusion (extracellular...</td>\n",
       "      <td>GUAtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100465</td>\n",
       "      <td>META:TRANS-RXN0-578</td>\n",
       "      <td>[rxn08682, rxn08683, rxn09746, rxn10142]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08682</td>\n",
       "      <td>GUAtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>H2O transport via diffusion (extracellular to ...</td>\n",
       "      <td>H2Otex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>4</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR98641</td>\n",
       "      <td>[META:TRANS-RXN-145, META:TRANS-RXN0-547]</td>\n",
       "      <td>[rxn05319, rxn08686, rxn08687, rxn09643, rxn09...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29667, 29668, 29669, 29670]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-432010, R-ATH-432054, R-ATH-432065, R-A...</td>\n",
       "      <td>rxn05319</td>\n",
       "      <td>H2Otex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1099</th>\n",
       "      <td>Hydrogen transport via diffusion (extracellula...</td>\n",
       "      <td>H2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100495</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn08691, rxn08692, rxn10542]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08691</td>\n",
       "      <td>H2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1205</th>\n",
       "      <td>Hypoxanthine transport via diffusion (extracel...</td>\n",
       "      <td>HYXNtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100749</td>\n",
       "      <td>[META:TRANS-RXN0-562, META:TRANS-RXN0-579]</td>\n",
       "      <td>[rxn08737, rxn08738, rxn10143]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08737</td>\n",
       "      <td>HYXNtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1206</th>\n",
       "      <td>Proton transport via diffusion (extracellular ...</td>\n",
       "      <td>Htex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100765</td>\n",
       "      <td>META:RXN-14452</td>\n",
       "      <td>[rxn08730, rxn11009]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34979, 34980, 34981, 34982]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-170026, R-ATH-74723, R-BTA-170026, R-BT...</td>\n",
       "      <td>rxn08730</td>\n",
       "      <td>Htex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1207</th>\n",
       "      <td>Isobutyrate transport via diffusion (extracell...</td>\n",
       "      <td>IBTtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100778</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1219</th>\n",
       "      <td>L-isoleucine transport via diffusion (extracel...</td>\n",
       "      <td>ILEtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100824</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn08747, rxn10933, rxn13323]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08747</td>\n",
       "      <td>ILEtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1285</th>\n",
       "      <td>Potassium transport via diffusion (extracellul...</td>\n",
       "      <td>Ktex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR100950</td>\n",
       "      <td>[META:TRANS-RXN-143, META:TRANS-RXN-185, META:...</td>\n",
       "      <td>[rxn05206, rxn08771]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29463, 29464, 29465, 29466]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-1296043, R-BTA-1013020, R-BTA-1296024, ...</td>\n",
       "      <td>rxn05206</td>\n",
       "      <td>Ktex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1305</th>\n",
       "      <td>L-leucine transport via diffusion (extracellul...</td>\n",
       "      <td>LEUtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101057</td>\n",
       "      <td>META:RXN66-447</td>\n",
       "      <td>[rxn08786, rxn11003]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08786</td>\n",
       "      <td>LEUtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1337</th>\n",
       "      <td>L-lysine transport via diffusion (extracellula...</td>\n",
       "      <td>LYStex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101269</td>\n",
       "      <td>META:TRANS-RXN0-569</td>\n",
       "      <td>[rxn08856, rxn10834]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08856</td>\n",
       "      <td>LYStex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1385</th>\n",
       "      <td>Methanol transport via diffusion (extracellula...</td>\n",
       "      <td>MEOHtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>3</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101464</td>\n",
       "      <td>META:TRANS-RXN0-459</td>\n",
       "      <td>rxn10570</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34871, 34872, 34873, 34874]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1402</th>\n",
       "      <td>Magnesium (Mg+2) transport via diffusion (extr...</td>\n",
       "      <td>MG2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101507</td>\n",
       "      <td>META:TRANS-RXN-141</td>\n",
       "      <td>[rxn05616, rxn08922, rxn08923]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29827, 29828, 29829, 29830]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-5336453, R-ATH-5336466, R-BTA-442661, R...</td>\n",
       "      <td>rxn05616</td>\n",
       "      <td>MG2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1450</th>\n",
       "      <td>Manganese (Mn+2) transport via diffusion (extr...</td>\n",
       "      <td>MNtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101670</td>\n",
       "      <td>[META:RXN0-16, META:TRANS-RXN0-487]</td>\n",
       "      <td>[rxn08947, rxn08952]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28699, 28700, 28701, 28702]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08947</td>\n",
       "      <td>MNtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1455</th>\n",
       "      <td>Molybdate transport via diffusion (extracellul...</td>\n",
       "      <td>MOBDtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101690</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08956</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34907, 34908, 34909, 34910]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08956</td>\n",
       "      <td>MOBDtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1503</th>\n",
       "      <td>Sodium transport via diffusion (extracellular ...</td>\n",
       "      <td>NAtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101804</td>\n",
       "      <td>META:TRANS-RXN-189</td>\n",
       "      <td>[rxn08985, rxn10899]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34963, 34964, 34965, 34966]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-BTA-2671885, R-BTA-2672334, R-BTA-2730664, ...</td>\n",
       "      <td>rxn08985</td>\n",
       "      <td>NAtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1520</th>\n",
       "      <td>Ammonia transport via diffusion (extracellular...</td>\n",
       "      <td>NH4tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>3</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101950</td>\n",
       "      <td>[META:RXN-9615, META:TRANS-RXN0-206, META:TRAN...</td>\n",
       "      <td>[rxn05466, rxn08986, rxn08987, rxn09736, rxn09...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28747, 28748, 28749, 28750]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-BTA-444416, R-CEL-444416, R-CFA-444416, R-D...</td>\n",
       "      <td>rxn05466</td>\n",
       "      <td>NH4tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1525</th>\n",
       "      <td>Nickel transport via diffusion (extracellular ...</td>\n",
       "      <td>NI2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101958</td>\n",
       "      <td>META:TRANS-RXN-141B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29831, 29832, 29833, 29834]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08991</td>\n",
       "      <td>NI2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1536</th>\n",
       "      <td>Nitrite transport via diffusion (extracellular...</td>\n",
       "      <td>NO2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101912</td>\n",
       "      <td>META:TRANS-RXN-137</td>\n",
       "      <td>[rxn08999, rxn10823]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[28983, 28984, 28985, 28986]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn08999</td>\n",
       "      <td>NO2tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1540</th>\n",
       "      <td>Nitrate transport via diffusion (extracellular...</td>\n",
       "      <td>NO3tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR101992</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn09005, rxn10935]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34923, 34924, 34925, 34926]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09005</td>\n",
       "      <td>NO3tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1600</th>\n",
       "      <td>Oxalate:Formate antiporter (extracellular to p...</td>\n",
       "      <td>OXFOtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR102238</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1722</th>\n",
       "      <td>Phosphate transport via diffusion (extracellul...</td>\n",
       "      <td>PItex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR102871</td>\n",
       "      <td>META:TRANS-RXN0-470</td>\n",
       "      <td>[rxn09121, rxn09722, rxn10838, rxn13178]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[32823, 32824, 32825, 32826]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09121</td>\n",
       "      <td>PItex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1760</th>\n",
       "      <td>Propionate transport via diffusion</td>\n",
       "      <td>PPAtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR103094</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09172</td>\n",
       "      <td>PPAtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1775</th>\n",
       "      <td>Propanol transport via diffusion (extracellula...</td>\n",
       "      <td>PPOHtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR103125</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>Pentanoate transport via diffusion (extracellu...</td>\n",
       "      <td>PTAtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR103320</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1838</th>\n",
       "      <td>Pyruvate transport via diffusion (extracellula...</td>\n",
       "      <td>PYRtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR103384</td>\n",
       "      <td>[META:TRANS-RXN0-506, META:TRANS-RXN0-570]</td>\n",
       "      <td>[rxn09218, rxn10929]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09218</td>\n",
       "      <td>PYRtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1958</th>\n",
       "      <td>Sulfite transport via diffusion (extracellular...</td>\n",
       "      <td>SO3tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR104460</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn09260, rxn12453]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09260</td>\n",
       "      <td>SO3tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1961</th>\n",
       "      <td>Sulfate transport via diffusion (extracellular...</td>\n",
       "      <td>SO4tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>3</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR104469</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn05238, rxn09261]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[34983, 34984, 34985, 34986]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-8875871, R-BTA-8875871, R-CFA-8875871, ...</td>\n",
       "      <td>rxn05238</td>\n",
       "      <td>SO4tex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2032</th>\n",
       "      <td>Toluene  outer membrane porin transport</td>\n",
       "      <td>TOLtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR104917</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[rxn12814, rxn12815]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2051</th>\n",
       "      <td>Thiosulfate transport via diffusion (extracell...</td>\n",
       "      <td>TSULtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR104966</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09330</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[32807, 32808, 32809, 32810]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn09330</td>\n",
       "      <td>TSULtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2112</th>\n",
       "      <td>L-valine transport via diffusion (extracellula...</td>\n",
       "      <td>VALtex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR105190</td>\n",
       "      <td>META:TRANS-RXN0-269</td>\n",
       "      <td>[rxn05245, rxn09375]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29703, 29704, 29705, 29706]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>rxn05245</td>\n",
       "      <td>VALtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2137</th>\n",
       "      <td>Zinc (Zn+2) transport via diffusion (extracell...</td>\n",
       "      <td>Zn2tex</td>\n",
       "      <td>G_WP_014951106_1</td>\n",
       "      <td>2</td>\n",
       "      <td>SBO:0000185</td>\n",
       "      <td>MNXR105280</td>\n",
       "      <td>META:RXN0-12</td>\n",
       "      <td>[rxn09390, rxn09391]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[29351, 29352, 29353, 29354]</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R-ATH-437085, R-ATH-437136, R-ATH-442393, R-A...</td>\n",
       "      <td>rxn09390</td>\n",
       "      <td>Zn2tex</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               reaction         rid  \\\n",
       "64    3-methylbutanoic acid transport via diffusion ...      3MBtex   \n",
       "71    4-hydroxybenzaldehyde transport via diffusion ...   4HBALDtex   \n",
       "73    4-hydroxy-benzyl alcohol transport via diffusi...     4HBAtex   \n",
       "75          4-Hydroxybenzoate transport (extracellular)     4HBZtex   \n",
       "242   Acetate transport via diffusion (extracellular...       ACtex   \n",
       "450   Arsenate transporter (extracellular to periplasm)     ASO4tex   \n",
       "490   Butyrate transport via diffusion (extracellula...      BUTtex   \n",
       "494   Benzoate transport via diffusion (extracellula...       BZtex   \n",
       "496   Calcium (Ca+2) transport via diffusion (extrac...      CA2tex   \n",
       "510   Cadmium (Cd+2) transport via diffusion (extrac...      CD2tex   \n",
       "544   Chloride (Cl-1) transport via diffusion (extra...       CLtex   \n",
       "554   CO2 transport via diffusion (extracellular to ...      CO2tex   \n",
       "557   Cobalt (Co+2) transport via diffusion (extrace...  COBALT2tex   \n",
       "569   Cromate transport via diffusion (extracellular...     CRO4tex   \n",
       "583   Copper (Cu+2) transport via diffusion (extrace...      CU2tex   \n",
       "785   Ethanol transport via diffusion (extracellular...     ETOHtex   \n",
       "862   Iron (II) transport via diffusion (extracellul...      FE2tex   \n",
       "874   Iron (III) transport via diffusion (extracellu...      FE3tex   \n",
       "897   Formate transport via diffusion (extracellular...      FORtex   \n",
       "933   D-galacturonate transport via diffusion (extra...    GALURtex   \n",
       "961   D-glucuronat transport via diffusion (extracel...    GLCURtex   \n",
       "1014  Glycerol transport via diffusion (extracellula...     GLYCtex   \n",
       "1086  Guanine transport via diffusion (extracellular...      GUAtex   \n",
       "1095  H2O transport via diffusion (extracellular to ...      H2Otex   \n",
       "1099  Hydrogen transport via diffusion (extracellula...       H2tex   \n",
       "1205  Hypoxanthine transport via diffusion (extracel...     HYXNtex   \n",
       "1206  Proton transport via diffusion (extracellular ...        Htex   \n",
       "1207  Isobutyrate transport via diffusion (extracell...      IBTtex   \n",
       "1219  L-isoleucine transport via diffusion (extracel...      ILEtex   \n",
       "1285  Potassium transport via diffusion (extracellul...        Ktex   \n",
       "1305  L-leucine transport via diffusion (extracellul...      LEUtex   \n",
       "1337  L-lysine transport via diffusion (extracellula...      LYStex   \n",
       "1385  Methanol transport via diffusion (extracellula...     MEOHtex   \n",
       "1402  Magnesium (Mg+2) transport via diffusion (extr...      MG2tex   \n",
       "1450  Manganese (Mn+2) transport via diffusion (extr...       MNtex   \n",
       "1455  Molybdate transport via diffusion (extracellul...     MOBDtex   \n",
       "1503  Sodium transport via diffusion (extracellular ...       NAtex   \n",
       "1520  Ammonia transport via diffusion (extracellular...      NH4tex   \n",
       "1525  Nickel transport via diffusion (extracellular ...      NI2tex   \n",
       "1536  Nitrite transport via diffusion (extracellular...      NO2tex   \n",
       "1540  Nitrate transport via diffusion (extracellular...      NO3tex   \n",
       "1600  Oxalate:Formate antiporter (extracellular to p...     OXFOtex   \n",
       "1722  Phosphate transport via diffusion (extracellul...       PItex   \n",
       "1760                 Propionate transport via diffusion      PPAtex   \n",
       "1775  Propanol transport via diffusion (extracellula...     PPOHtex   \n",
       "1810  Pentanoate transport via diffusion (extracellu...      PTAtex   \n",
       "1838  Pyruvate transport via diffusion (extracellula...      PYRtex   \n",
       "1958  Sulfite transport via diffusion (extracellular...      SO3tex   \n",
       "1961  Sulfate transport via diffusion (extracellular...      SO4tex   \n",
       "2032            Toluene  outer membrane porin transport      TOLtex   \n",
       "2051  Thiosulfate transport via diffusion (extracell...     TSULtex   \n",
       "2112  L-valine transport via diffusion (extracellula...      VALtex   \n",
       "2137  Zinc (Zn+2) transport via diffusion (extracell...      Zn2tex   \n",
       "\n",
       "                 genes  genes_number          sbo metanetx.reaction  \\\n",
       "64    G_WP_014951106_1             2  SBO:0000185         MNXR94918   \n",
       "71    G_WP_014951106_1             2  SBO:0000185         MNXR95006   \n",
       "73    G_WP_014951106_1             2  SBO:0000185         MNXR95007   \n",
       "75    G_WP_014951106_1             2  SBO:0000185         MNXR95013   \n",
       "242   G_WP_014951106_1             2  SBO:0000185         MNXR95431   \n",
       "450   G_WP_014951106_1             2  SBO:0000185         MNXR96076   \n",
       "490   G_WP_014951106_1             2  SBO:0000185         MNXR96352   \n",
       "494   G_WP_014951106_1             2  SBO:0000185         MNXR96370   \n",
       "496   G_WP_014951106_1             2  SBO:0000185         MNXR96436   \n",
       "510   G_WP_014951106_1             2  SBO:0000185         MNXR96499   \n",
       "544   G_WP_014951106_1             2  SBO:0000185         MNXR96797   \n",
       "554   G_WP_014951106_1             3  SBO:0000185         MNXR96810   \n",
       "557   G_WP_014951106_1             2  SBO:0000185         MNXR96821   \n",
       "569   G_WP_014951106_1             2  SBO:0000185         MNXR96914   \n",
       "583   G_WP_014951106_1             2  SBO:0000185         MNXR96951   \n",
       "785   G_WP_014951106_1             2  SBO:0000185         MNXR97980   \n",
       "862   G_WP_014951106_1             3  SBO:0000185         MNXR99505   \n",
       "874   G_WP_014951106_1             2  SBO:0000185         MNXR99522   \n",
       "897   G_WP_014951106_1             2  SBO:0000185         MNXR99620   \n",
       "933   G_WP_014951106_1             2  SBO:0000185        MNXR100021   \n",
       "961   G_WP_014951106_1             2  SBO:0000185        MNXR100235   \n",
       "1014  G_WP_014951106_1             2  SBO:0000185        MNXR100343   \n",
       "1086  G_WP_014951106_1             2  SBO:0000185        MNXR100465   \n",
       "1095  G_WP_014951106_1             4  SBO:0000185         MNXR98641   \n",
       "1099  G_WP_014951106_1             2  SBO:0000185        MNXR100495   \n",
       "1205  G_WP_014951106_1             2  SBO:0000185        MNXR100749   \n",
       "1206  G_WP_014951106_1             2  SBO:0000185        MNXR100765   \n",
       "1207  G_WP_014951106_1             2  SBO:0000185        MNXR100778   \n",
       "1219  G_WP_014951106_1             2  SBO:0000185        MNXR100824   \n",
       "1285  G_WP_014951106_1             2  SBO:0000185        MNXR100950   \n",
       "1305  G_WP_014951106_1             2  SBO:0000185        MNXR101057   \n",
       "1337  G_WP_014951106_1             2  SBO:0000185        MNXR101269   \n",
       "1385  G_WP_014951106_1             3  SBO:0000185        MNXR101464   \n",
       "1402  G_WP_014951106_1             2  SBO:0000185        MNXR101507   \n",
       "1450  G_WP_014951106_1             2  SBO:0000185        MNXR101670   \n",
       "1455  G_WP_014951106_1             2  SBO:0000185        MNXR101690   \n",
       "1503  G_WP_014951106_1             2  SBO:0000185        MNXR101804   \n",
       "1520  G_WP_014951106_1             3  SBO:0000185        MNXR101950   \n",
       "1525  G_WP_014951106_1             2  SBO:0000185        MNXR101958   \n",
       "1536  G_WP_014951106_1             2  SBO:0000185        MNXR101912   \n",
       "1540  G_WP_014951106_1             2  SBO:0000185        MNXR101992   \n",
       "1600  G_WP_014951106_1             2  SBO:0000185        MNXR102238   \n",
       "1722  G_WP_014951106_1             2  SBO:0000185        MNXR102871   \n",
       "1760  G_WP_014951106_1             2  SBO:0000185        MNXR103094   \n",
       "1775  G_WP_014951106_1             2  SBO:0000185        MNXR103125   \n",
       "1810  G_WP_014951106_1             2  SBO:0000185        MNXR103320   \n",
       "1838  G_WP_014951106_1             2  SBO:0000185        MNXR103384   \n",
       "1958  G_WP_014951106_1             2  SBO:0000185        MNXR104460   \n",
       "1961  G_WP_014951106_1             3  SBO:0000185        MNXR104469   \n",
       "2032  G_WP_014951106_1             2  SBO:0000185        MNXR104917   \n",
       "2051  G_WP_014951106_1             2  SBO:0000185        MNXR104966   \n",
       "2112  G_WP_014951106_1             2  SBO:0000185        MNXR105190   \n",
       "2137  G_WP_014951106_1             2  SBO:0000185        MNXR105280   \n",
       "\n",
       "                                                 biocyc  \\\n",
       "64                                                  NaN   \n",
       "71                                                  NaN   \n",
       "73                                  META:TRANS-RXN0-274   \n",
       "75                                                  NaN   \n",
       "242               [META:RXN0-1981, META:TRANS-RXN0-567]   \n",
       "450                                                 NaN   \n",
       "490                                                 NaN   \n",
       "494                                                 NaN   \n",
       "496           [META:TRANS-RXN-144, META:TRANS-RXN0-497]   \n",
       "510                                        META:RXN0-10   \n",
       "544                                  META:TRANS-RXN-139   \n",
       "554                                 META:TRANS-RXN0-545   \n",
       "557                                 META:TRANS-RXN-141A   \n",
       "569                                                 NaN   \n",
       "583                                        META:RXN0-14   \n",
       "785                                 META:TRANS-RXN0-546   \n",
       "862                                    META:TRANS-RXN-8   \n",
       "874                                                 NaN   \n",
       "897                                    META:TRANS-RXN-1   \n",
       "933                                                 NaN   \n",
       "961                                                 NaN   \n",
       "1014                                 META:TRANS-RXN-131   \n",
       "1086                                META:TRANS-RXN0-578   \n",
       "1095          [META:TRANS-RXN-145, META:TRANS-RXN0-547]   \n",
       "1099                                                NaN   \n",
       "1205         [META:TRANS-RXN0-562, META:TRANS-RXN0-579]   \n",
       "1206                                     META:RXN-14452   \n",
       "1207                                                NaN   \n",
       "1219                                                NaN   \n",
       "1285  [META:TRANS-RXN-143, META:TRANS-RXN-185, META:...   \n",
       "1305                                     META:RXN66-447   \n",
       "1337                                META:TRANS-RXN0-569   \n",
       "1385                                META:TRANS-RXN0-459   \n",
       "1402                                 META:TRANS-RXN-141   \n",
       "1450                [META:RXN0-16, META:TRANS-RXN0-487]   \n",
       "1455                                                NaN   \n",
       "1503                                 META:TRANS-RXN-189   \n",
       "1520  [META:RXN-9615, META:TRANS-RXN0-206, META:TRAN...   \n",
       "1525                                META:TRANS-RXN-141B   \n",
       "1536                                 META:TRANS-RXN-137   \n",
       "1540                                                NaN   \n",
       "1600                                                NaN   \n",
       "1722                                META:TRANS-RXN0-470   \n",
       "1760                                                NaN   \n",
       "1775                                                NaN   \n",
       "1810                                                NaN   \n",
       "1838         [META:TRANS-RXN0-506, META:TRANS-RXN0-570]   \n",
       "1958                                                NaN   \n",
       "1961                                                NaN   \n",
       "2032                                                NaN   \n",
       "2051                                                NaN   \n",
       "2112                                META:TRANS-RXN0-269   \n",
       "2137                                       META:RXN0-12   \n",
       "\n",
       "                                          seed.reaction  ec-code  \\\n",
       "64                                                  NaN      NaN   \n",
       "71                                                  NaN      NaN   \n",
       "73                                             rxn12376      NaN   \n",
       "75                                 [rxn09792, rxn12787]      NaN   \n",
       "242            [rxn08063, rxn09787, rxn09866, rxn10904]      NaN   \n",
       "450                                            rxn05506      NaN   \n",
       "490                                                 NaN      NaN   \n",
       "494                                            rxn12436      NaN   \n",
       "496                                            rxn08186  3.6.3.8   \n",
       "510                                [rxn08197, rxn08198]      NaN   \n",
       "544                                [rxn08234, rxn10473]      NaN   \n",
       "554   [rxn05467, rxn08237, rxn08238, rxn09706, rxn09...      NaN   \n",
       "557                      [rxn08241, rxn08242, rxn10474]      NaN   \n",
       "569                                                 NaN      NaN   \n",
       "583                      [rxn05212, rxn08273, rxn08274]      NaN   \n",
       "785                      [rxn08428, rxn09683, rxn09764]      NaN   \n",
       "862            [rxn08475, rxn08476, rxn10963, rxn13286]      NaN   \n",
       "874                                [rxn05292, rxn08492]      NaN   \n",
       "897            [rxn08525, rxn08526, rxn09682, rxn09754]      NaN   \n",
       "933                                            rxn08589      NaN   \n",
       "961                                            rxn08623      NaN   \n",
       "1014                     [rxn05581, rxn08654, rxn08658]      NaN   \n",
       "1086           [rxn08682, rxn08683, rxn09746, rxn10142]      NaN   \n",
       "1095  [rxn05319, rxn08686, rxn08687, rxn09643, rxn09...      NaN   \n",
       "1099                     [rxn08691, rxn08692, rxn10542]      NaN   \n",
       "1205                     [rxn08737, rxn08738, rxn10143]      NaN   \n",
       "1206                               [rxn08730, rxn11009]      NaN   \n",
       "1207                                                NaN      NaN   \n",
       "1219                     [rxn08747, rxn10933, rxn13323]      NaN   \n",
       "1285                               [rxn05206, rxn08771]      NaN   \n",
       "1305                               [rxn08786, rxn11003]      NaN   \n",
       "1337                               [rxn08856, rxn10834]      NaN   \n",
       "1385                                           rxn10570      NaN   \n",
       "1402                     [rxn05616, rxn08922, rxn08923]      NaN   \n",
       "1450                               [rxn08947, rxn08952]      NaN   \n",
       "1455                                           rxn08956      NaN   \n",
       "1503                               [rxn08985, rxn10899]      NaN   \n",
       "1520  [rxn05466, rxn08986, rxn08987, rxn09736, rxn09...      NaN   \n",
       "1525                                                NaN      NaN   \n",
       "1536                               [rxn08999, rxn10823]      NaN   \n",
       "1540                               [rxn09005, rxn10935]      NaN   \n",
       "1600                                                NaN      NaN   \n",
       "1722           [rxn09121, rxn09722, rxn10838, rxn13178]      NaN   \n",
       "1760                                                NaN      NaN   \n",
       "1775                                                NaN      NaN   \n",
       "1810                                                NaN      NaN   \n",
       "1838                               [rxn09218, rxn10929]      NaN   \n",
       "1958                               [rxn09260, rxn12453]      NaN   \n",
       "1961                               [rxn05238, rxn09261]      NaN   \n",
       "2032                               [rxn12814, rxn12815]      NaN   \n",
       "2051                                           rxn09330      NaN   \n",
       "2112                               [rxn05245, rxn09375]      NaN   \n",
       "2137                               [rxn09390, rxn09391]      NaN   \n",
       "\n",
       "                              rhea kegg.reaction  \\\n",
       "64                             NaN           NaN   \n",
       "71                             NaN           NaN   \n",
       "73                             NaN           NaN   \n",
       "75                             NaN           NaN   \n",
       "242   [27814, 27815, 27816, 27817]           NaN   \n",
       "450                            NaN           NaN   \n",
       "490                            NaN           NaN   \n",
       "494   [32811, 32812, 32813, 32814]           NaN   \n",
       "496   [29671, 29672, 29673, 29674]           NaN   \n",
       "510   [28707, 28708, 28709, 28710]           NaN   \n",
       "544   [29823, 29824, 29825, 29826]           NaN   \n",
       "554                            NaN           NaN   \n",
       "557   [28578, 28579, 28580, 28581]           NaN   \n",
       "569   [32819, 32820, 32821, 32822]           NaN   \n",
       "583   [28703, 28704, 28705, 28706]           NaN   \n",
       "785   [35267, 35268, 35269, 35270]           NaN   \n",
       "862   [28486, 28487, 28488, 28489]           NaN   \n",
       "874   [34971, 34972, 34973, 34974]           NaN   \n",
       "897   [29679, 29680, 29681, 29682]           NaN   \n",
       "933                            NaN           NaN   \n",
       "961                            NaN           NaN   \n",
       "1014  [29675, 29676, 29677, 29678]           NaN   \n",
       "1086                           NaN           NaN   \n",
       "1095  [29667, 29668, 29669, 29670]           NaN   \n",
       "1099                           NaN           NaN   \n",
       "1205                           NaN           NaN   \n",
       "1206  [34979, 34980, 34981, 34982]           NaN   \n",
       "1207                           NaN           NaN   \n",
       "1219                           NaN           NaN   \n",
       "1285  [29463, 29464, 29465, 29466]           NaN   \n",
       "1305                           NaN           NaN   \n",
       "1337                           NaN           NaN   \n",
       "1385  [34871, 34872, 34873, 34874]           NaN   \n",
       "1402  [29827, 29828, 29829, 29830]           NaN   \n",
       "1450  [28699, 28700, 28701, 28702]           NaN   \n",
       "1455  [34907, 34908, 34909, 34910]           NaN   \n",
       "1503  [34963, 34964, 34965, 34966]           NaN   \n",
       "1520  [28747, 28748, 28749, 28750]           NaN   \n",
       "1525  [29831, 29832, 29833, 29834]           NaN   \n",
       "1536  [28983, 28984, 28985, 28986]           NaN   \n",
       "1540  [34923, 34924, 34925, 34926]           NaN   \n",
       "1600                           NaN           NaN   \n",
       "1722  [32823, 32824, 32825, 32826]           NaN   \n",
       "1760                           NaN           NaN   \n",
       "1775                           NaN           NaN   \n",
       "1810                           NaN           NaN   \n",
       "1838                           NaN           NaN   \n",
       "1958                           NaN           NaN   \n",
       "1961  [34983, 34984, 34985, 34986]           NaN   \n",
       "2032                           NaN           NaN   \n",
       "2051  [32807, 32808, 32809, 32810]           NaN   \n",
       "2112  [29703, 29704, 29705, 29706]           NaN   \n",
       "2137  [29351, 29352, 29353, 29354]           NaN   \n",
       "\n",
       "                                               reactome        id        BiGG  \n",
       "64                                                  NaN       NaN         NaN  \n",
       "71                                                  NaN       NaN         NaN  \n",
       "73                                                  NaN       NaN         NaN  \n",
       "75                                                  NaN       NaN         NaN  \n",
       "242                                                 NaN  rxn08063       ACtex  \n",
       "450                                                 NaN       NaN         NaN  \n",
       "490                                                 NaN  rxn08184      BUTtex  \n",
       "494                                                 NaN       NaN         NaN  \n",
       "496   [R-ATH-2685505, R-ATH-418309, R-BTA-139854, R-...  rxn08186      CA2tex  \n",
       "510                                                 NaN  rxn08197      CD2tex  \n",
       "544   [R-ATH-2684901, R-ATH-427570, R-BTA-2684901, R...  rxn08234       CLtex  \n",
       "554   [R-ATH-1237042, R-ATH-1247649, R-BTA-1237042, ...  rxn05467      CO2tex  \n",
       "557                                                 NaN  rxn08241  COBALT2tex  \n",
       "569                                                 NaN       NaN         NaN  \n",
       "583   [R-BTA-437300, R-CFA-437300, R-DME-437300, R-D...  rxn05212      CU2tex  \n",
       "785                                                 NaN  rxn08428     ETOHtex  \n",
       "862                                                 NaN  rxn08475      FE2tex  \n",
       "874   [R-BTA-442368, R-CFA-442368, R-GGA-442368, R-H...  rxn05292      FE3tex  \n",
       "897                                       R-HSA-6803255  rxn08525      FORtex  \n",
       "933                                                 NaN  rxn08589    GALURtex  \n",
       "961                                                 NaN  rxn08623    GLCURtex  \n",
       "1014  [R-BTA-432049, R-BTA-432074, R-BTA-507869, R-B...  rxn05581     GLYCtex  \n",
       "1086                                                NaN  rxn08682      GUAtex  \n",
       "1095  [R-ATH-432010, R-ATH-432054, R-ATH-432065, R-A...  rxn05319      H2Otex  \n",
       "1099                                                NaN  rxn08691       H2tex  \n",
       "1205                                                NaN  rxn08737     HYXNtex  \n",
       "1206  [R-ATH-170026, R-ATH-74723, R-BTA-170026, R-BT...  rxn08730        Htex  \n",
       "1207                                                NaN       NaN         NaN  \n",
       "1219                                                NaN  rxn08747      ILEtex  \n",
       "1285  [R-ATH-1296043, R-BTA-1013020, R-BTA-1296024, ...  rxn05206        Ktex  \n",
       "1305                                                NaN  rxn08786      LEUtex  \n",
       "1337                                                NaN  rxn08856      LYStex  \n",
       "1385                                                NaN       NaN         NaN  \n",
       "1402  [R-ATH-5336453, R-ATH-5336466, R-BTA-442661, R...  rxn05616      MG2tex  \n",
       "1450                                                NaN  rxn08947       MNtex  \n",
       "1455                                                NaN  rxn08956     MOBDtex  \n",
       "1503  [R-BTA-2671885, R-BTA-2672334, R-BTA-2730664, ...  rxn08985       NAtex  \n",
       "1520  [R-BTA-444416, R-CEL-444416, R-CFA-444416, R-D...  rxn05466      NH4tex  \n",
       "1525                                                NaN  rxn08991      NI2tex  \n",
       "1536                                                NaN  rxn08999      NO2tex  \n",
       "1540                                                NaN  rxn09005      NO3tex  \n",
       "1600                                                NaN       NaN         NaN  \n",
       "1722                                                NaN  rxn09121       PItex  \n",
       "1760                                                NaN  rxn09172      PPAtex  \n",
       "1775                                                NaN       NaN         NaN  \n",
       "1810                                                NaN       NaN         NaN  \n",
       "1838                                                NaN  rxn09218      PYRtex  \n",
       "1958                                                NaN  rxn09260      SO3tex  \n",
       "1961  [R-ATH-8875871, R-BTA-8875871, R-CFA-8875871, ...  rxn05238      SO4tex  \n",
       "2032                                                NaN       NaN         NaN  \n",
       "2051                                                NaN  rxn09330     TSULtex  \n",
       "2112                                                NaN  rxn05245      VALtex  \n",
       "2137  [R-ATH-437085, R-ATH-437136, R-ATH-442393, R-A...  rxn09390      Zn2tex  "
      ]
     },
     "execution_count": 142,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfmerge.loc[dfmerge.genes.isin(['G_WP_014951106_1'])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>locus_tag</th>\n",
       "      <th>type</th>\n",
       "      <th>gene</th>\n",
       "      <th>gene_biotype</th>\n",
       "      <th>old_locus_tag</th>\n",
       "      <th>product</th>\n",
       "      <th>protein_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2565</th>\n",
       "      <td>ACZ81_RS12780</td>\n",
       "      <td>gene</td>\n",
       "      <td>adk</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>ACZ81_12770</td>\n",
       "      <td>adenylate kinase</td>\n",
       "      <td>WP_014950043.1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          locus_tag  type gene    gene_biotype old_locus_tag  \\\n",
       "2565  ACZ81_RS12780  gene  adk  protein_coding   ACZ81_12770   \n",
       "\n",
       "               product      protein_id  \n",
       "2565  adenylate kinase  WP_014950043.1  "
      ]
     },
     "execution_count": 135,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mapdf.loc[mapdf.protein_id.isin(['WP_014950043.1'])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "metadata": {},
   "outputs": [],
   "source": [
    "a,b = model.reactions.Zn2tex.genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Gene identifier</strong></td><td>WP_014951106_1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>G_WP_014951106_1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x7f5943aae430</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Functional</strong></td><td>True</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>In 53 reaction(s)</strong></td><td>\n",
       "                    NI2tex, 4HBALDtex, CA2tex, GALURtex, LEUtex, COBALT2tex, PItex, VALtex, LYStex, OXFOtex, CD2tex, GUAtex, FORtex, Ktex, PTAtex, ILEtex, 3MBtex, PPAtex, H2Otex, MOBDtex, 4HBZtex, BZtex, FE3tex,...\n",
       "                    </td>\n",
       "            </tr>\n",
       "        </table>"
      ],
      "text/plain": [
       "<Gene WP_014951106_1 at 0x7f5943aae430>"
      ]
     },
     "execution_count": 137,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "b"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Gene identifier</strong></td><td>WP_014951442_1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>G_WP_014951442_1</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x7f5943aae400</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Functional</strong></td><td>True</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>In 53 reaction(s)</strong></td><td>\n",
       "                    NI2tex, 4HBALDtex, CA2tex, GALURtex, LEUtex, COBALT2tex, PItex, VALtex, LYStex, OXFOtex, CD2tex, GUAtex, FORtex, Ktex, PTAtex, ILEtex, 3MBtex, PPAtex, H2Otex, MOBDtex, 4HBZtex, BZtex, FE3tex,...\n",
       "                    </td>\n",
       "            </tr>\n",
       "        </table>"
      ],
      "text/plain": [
       "<Gene WP_014951442_1 at 0x7f5943aae400>"
      ]
     },
     "execution_count": 138,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0    38760\n",
       "Name: is_obsolete, dtype: int64"
      ]
     },
     "execution_count": 139,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfseed.is_obsolete.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>is_obsolete</th>\n",
       "      <th>AraCyc</th>\n",
       "      <th>BiGG</th>\n",
       "      <th>BrachyCyc</th>\n",
       "      <th>KEGG</th>\n",
       "      <th>MetaCyc</th>\n",
       "      <th>Name</th>\n",
       "      <th>ChlamyCyc</th>\n",
       "      <th>AlgaGEM</th>\n",
       "      <th>...</th>\n",
       "      <th>iAbaylyiv4</th>\n",
       "      <th>iJR904</th>\n",
       "      <th>iMEO21</th>\n",
       "      <th>iNJ661</th>\n",
       "      <th>iSB619</th>\n",
       "      <th>iRR1083</th>\n",
       "      <th>iSO783</th>\n",
       "      <th>iND750</th>\n",
       "      <th>iGT196</th>\n",
       "      <th>iPS189</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>12922</th>\n",
       "      <td>rxn14090</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[R08952]</td>\n",
       "      <td>[2.5.1.71-RXN]</td>\n",
       "      <td>[LG 2''-dimethylallyltransferase, LGDT, dimeth...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows × 49 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             id  is_obsolete AraCyc BiGG BrachyCyc      KEGG         MetaCyc  \\\n",
       "12922  rxn14090            0    NaN  NaN       NaN  [R08952]  [2.5.1.71-RXN]   \n",
       "\n",
       "                                                    Name ChlamyCyc AlgaGEM  \\\n",
       "12922  [LG 2''-dimethylallyltransferase, LGDT, dimeth...       NaN     NaN   \n",
       "\n",
       "       ... iAbaylyiv4 iJR904 iMEO21 iNJ661 iSB619 iRR1083 iSO783 iND750  \\\n",
       "12922  ...        NaN    NaN    NaN    NaN    NaN     NaN    NaN    NaN   \n",
       "\n",
       "      iGT196 iPS189  \n",
       "12922    NaN    NaN  \n",
       "\n",
       "[1 rows x 49 columns]"
      ]
     },
     "execution_count": 143,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dfseed.loc[dfseed.id.isin(['rxn14090'])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
